<html lang="en-US"><!--<![endif]-->
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width">
    <title>Application | TaxonomiesHBP Software Categories | High Performance Analytics &amp; Computing Platform â€“
        Guidebook</title>
    <link rel="profile" href="http://gmpg.org/xfn/11">
    <link rel="pingback" href="https://hbp-hpc-platform.fz-juelich.de/xmlrpc.php">
    <!--[if lt IE 9]>
    <script src="https://hbp-hpc-platform.fz-juelich.de/wp-content/themes/twentythirteen/js/html5.js"></script>
    <![endif]-->
    <link rel="dns-prefetch" href="//fonts.googleapis.com">
    <link rel="dns-prefetch" href="//s.w.org">
    <link href="https://fonts.gstatic.com" crossorigin="" rel="preconnect">
    <link rel="alternate" type="application/rss+xml"
          title="High Performance Analytics &amp; Computing Platform â€“ Guidebook Â» Feed"
          href="https://hbp-hpc-platform.fz-juelich.de/?feed=rss2">
    <link rel="alternate" type="application/rss+xml"
          title="High Performance Analytics &amp; Computing Platform â€“ Guidebook Â» Comments Feed"
          href="https://hbp-hpc-platform.fz-juelich.de/?feed=comments-rss2">
    <link rel="alternate" type="text/calendar"
          title="High Performance Analytics &amp; Computing Platform â€“ Guidebook Â» iCal Feed"
          href="https://hbp-hpc-platform.fz-juelich.de?post_type=tribe_events&amp;ical=1">
    <link rel="alternate" type="application/rss+xml"
          title="High Performance Analytics &amp; Computing Platform â€“ Guidebook Â» Application HBP Software Category Feed"
          href="https://hbp-hpc-platform.fz-juelich.de/?feed=rss2&amp;hbp_software_cat=application-2">
    <script type="text/javascript">
        window._wpemojiSettings = {
            "baseUrl": "https:\/\/s.w.org\/images\/core\/emoji\/11.2.0\/72x72\/",
            "ext": ".png",
            "svgUrl": "https:\/\/s.w.org\/images\/core\/emoji\/11.2.0\/svg\/",
            "svgExt": ".svg",
            "source": {"concatemoji": "https:\/\/hbp-hpc-platform.fz-juelich.de\/wp-includes\/js\/wp-emoji-release.min.js?ver=5.1.1"}
        };
        !function (a, b, c) {
            function d(a, b) {
                var c = String.fromCharCode;
                l.clearRect(0, 0, k.width, k.height), l.fillText(c.apply(this, a), 0, 0);
                var d = k.toDataURL();
                l.clearRect(0, 0, k.width, k.height), l.fillText(c.apply(this, b), 0, 0);
                var e = k.toDataURL();
                return d === e
            }

            function e(a) {
                var b;
                if (!l || !l.fillText) return !1;
                switch (l.textBaseline = "top", l.font = "600 32px Arial", a) {
                    case"flag":
                        return !(b = d([55356, 56826, 55356, 56819], [55356, 56826, 8203, 55356, 56819])) && (b = d([55356, 57332, 56128, 56423, 56128, 56418, 56128, 56421, 56128, 56430, 56128, 56423, 56128, 56447], [55356, 57332, 8203, 56128, 56423, 8203, 56128, 56418, 8203, 56128, 56421, 8203, 56128, 56430, 8203, 56128, 56423, 8203, 56128, 56447]), !b);
                    case"emoji":
                        return b = d([55358, 56760, 9792, 65039], [55358, 56760, 8203, 9792, 65039]), !b
                }
                return !1
            }

            function f(a) {
                var c = b.createElement("script");
                c.src = a, c.defer = c.type = "text/javascript", b.getElementsByTagName("head")[0].appendChild(c)
            }

            var g, h, i, j, k = b.createElement("canvas"), l = k.getContext && k.getContext("2d");
            for (j = Array("flag", "emoji"), c.supports = {
                everything: !0,
                everythingExceptFlag: !0
            }, i = 0; i < j.length; i++) c.supports[j[i]] = e(j[i]), c.supports.everything = c.supports.everything && c.supports[j[i]], "flag" !== j[i] && (c.supports.everythingExceptFlag = c.supports.everythingExceptFlag && c.supports[j[i]]);
            c.supports.everythingExceptFlag = c.supports.everythingExceptFlag && !c.supports.flag, c.DOMReady = !1, c.readyCallback = function () {
                c.DOMReady = !0
            }, c.supports.everything || (h = function () {
                c.readyCallback()
            }, b.addEventListener ? (b.addEventListener("DOMContentLoaded", h, !1), a.addEventListener("load", h, !1)) : (a.attachEvent("onload", h), b.attachEvent("onreadystatechange", function () {
                "complete" === b.readyState && c.readyCallback()
            })), g = c.source || {}, g.concatemoji ? f(g.concatemoji) : g.wpemoji && g.twemoji && (f(g.twemoji), f(g.wpemoji)))
        }(window, document, window._wpemojiSettings);
    </script>
    <script src="https://hbp-hpc-platform.fz-juelich.de/wp-includes/js/wp-emoji-release.min.js?ver=5.1.1"
            type="text/javascript" defer=""></script>
    <style type="text/css">
        img.wp-smiley,
        img.emoji {
            display: inline !important;
            border: none !important;
            box-shadow: none !important;
            height: 1em !important;
            width: 1em !important;
            margin: 0 .07em !important;
            vertical-align: -0.1em !important;
            background: none !important;
            padding: 0 !important;
        }
    </style>
    <link rel="stylesheet" id="wp-block-library-css"
          href="https://hbp-hpc-platform.fz-juelich.de/wp-includes/css/dist/block-library/style.min.css?ver=5.1.1"
          type="text/css" media="all">
    <link rel="stylesheet" id="wp-block-library-theme-css"
          href="https://hbp-hpc-platform.fz-juelich.de/wp-includes/css/dist/block-library/theme.min.css?ver=5.1.1"
          type="text/css" media="all">
    <link rel="stylesheet" id="contact-form-7-css"
          href="https://hbp-hpc-platform.fz-juelich.de/wp-content/plugins/contact-form-7/includes/css/styles.css?ver=5.1.1"
          type="text/css" media="all">
    <link rel="stylesheet" id="searchandfilter-css"
          href="https://hbp-hpc-platform.fz-juelich.de/wp-content/plugins/search-filter/style.css?ver=1" type="text/css"
          media="all">
    <link rel="stylesheet" id="vegas-css"
          href="https://hbp-hpc-platform.fz-juelich.de/wp-content/plugins/vegas-fullscreen-background-slider/methods/../css/jquery.vegas.css"
          type="text/css" media="all">
    <link rel="stylesheet" id="mb.YTPlayer_css-css"
          href="https://hbp-hpc-platform.fz-juelich.de/wp-content/plugins/wpmbytplayer/css/mb.YTPlayer.css?ver=3.2.5"
          type="text/css" media="screen">
    <link rel="stylesheet" id="parent-style-css"
          href="https://hbp-hpc-platform.fz-juelich.de/wp-content/themes/twentythirteen/style.css?ver=5.1.1"
          type="text/css" media="all">
    <link rel="stylesheet" id="child-style-css"
          href="https://hbp-hpc-platform.fz-juelich.de/wp-content/themes/hbp-hpc/style.css?ver=5.1.1" type="text/css"
          media="all">
    <link rel="stylesheet" id="font-awesome-css"
          href="https://hbp-hpc-platform.fz-juelich.de/wp-content/themes/hbp-hpc/fonts/font-awesome/css/font-awesome.css?ver=5.1.1"
          type="text/css" media="all">
    <link rel="stylesheet" id="twentythirteen-fonts-css"
          href="https://fonts.googleapis.com/css?family=Source+Sans+Pro%3A300%2C400%2C700%2C300italic%2C400italic%2C700italic%7CBitter%3A400%2C700&amp;subset=latin%2Clatin-ext"
          type="text/css" media="all">
    <link rel="stylesheet" id="genericons-css"
          href="https://hbp-hpc-platform.fz-juelich.de/wp-content/themes/twentythirteen/genericons/genericons.css?ver=3.03"
          type="text/css" media="all">
    <link rel="stylesheet" id="twentythirteen-style-css"
          href="https://hbp-hpc-platform.fz-juelich.de/wp-content/themes/hbp-hpc/style.css?ver=2013-07-18"
          type="text/css" media="all">
    <link rel="stylesheet" id="twentythirteen-block-style-css"
          href="https://hbp-hpc-platform.fz-juelich.de/wp-content/themes/twentythirteen/css/blocks.css?ver=2018-12-30"
          type="text/css" media="all">
    <!--[if lt IE 9]>
    <link rel='stylesheet' id='twentythirteen-ie-css'
          href='https://hbp-hpc-platform.fz-juelich.de/wp-content/themes/twentythirteen/css/ie.css?ver=2013-07-18'
          type='text/css' media='all'/>
    <![endif]-->
    <link rel="stylesheet" id="msl-main-css"
          href="https://hbp-hpc-platform.fz-juelich.de/wp-content/plugins/master-slider/public/assets/css/masterslider.main.css?ver=3.5.3"
          type="text/css" media="all">
    <link rel="stylesheet" id="msl-custom-css"
          href="https://hbp-hpc-platform.fz-juelich.de/wp-content/uploads/master-slider/custom.css?ver=2.8"
          type="text/css" media="all">
    <script type="text/javascript"
            src="https://hbp-hpc-platform.fz-juelich.de/wp-includes/js/jquery/jquery.js?ver=1.12.4"></script>
    <script type="text/javascript"
            src="https://hbp-hpc-platform.fz-juelich.de/wp-includes/js/jquery/jquery-migrate.min.js?ver=1.4.1"></script>
    <link rel="https://api.w.org/" href="https://hbp-hpc-platform.fz-juelich.de/index.php?rest_route=/">
    <link rel="EditURI" type="application/rsd+xml" title="RSD"
          href="https://hbp-hpc-platform.fz-juelich.de/xmlrpc.php?rsd">
    <link rel="wlwmanifest" type="application/wlwmanifest+xml"
          href="https://hbp-hpc-platform.fz-juelich.de/wp-includes/wlwmanifest.xml">
    <meta name="generator" content="WordPress 5.1.1">
    <script>var ms_grabbing_curosr = 'https://hbp-hpc-platform.fz-juelich.de/wp-content/plugins/master-slider/public/assets/css/common/grabbing.cur',
        ms_grab_curosr = 'https://hbp-hpc-platform.fz-juelich.de/wp-content/plugins/master-slider/public/assets/css/common/grab.cur';</script>
    <meta name="generator" content="MasterSlider 3.5.3 - Responsive Touch Image Slider | avt.li/msf">
    <!-- Analytics by WP-Statistics v12.6.2 - https://wp-statistics.com/ -->
    <script>jQuery(function () {
        var a = null;
        setInterval(function () {
            jQuery(".YTPOverlay").each(function () {
                var b = jQuery(this);
                jQuery("[class*=ytp_wm_]", b).remove();
                a = "ytp_wm_" + Math.floor(1E5 * Math.random());
                var c = jQuery("<img/>").attr("src", "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACYAAAG9CAYAAAB56wSaAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAADSBJREFUeNrsXT90IjcTl/2ojtRcDTWuQ23qj357fzWpoTZ1qEO/Pam5mtRsbeqjPq7OKm+Wk+XRakZ/FmzPvMdzsge7v9WM5p9GI6WEeHTn+8LPnz/7X758OVNvWH9/Uv8Z1p8BXDrVn319j2MyYPVD9M3X9Wdb37j0fFeDWRqAbKrqzzP1JX3AFvWfCfyvfuM/sTcHUKv60/c8T4N6ooC7bwH1YIBSwJ4VXLfpDwIoBd9ZUkbsvuXfHpFrx/ptD9YLTAE0lcbwm2Bg2I8xOSsCJt3vQcAc7KqQ0Ro6hH2vAdffn8HksWkSOmJj5NoOufbg+P26EfD67w5mJOXlvcAwQT5RXwCZdQcur13ARvYFm40tI1al0Pz3DI3eRzQ8NrLYCwxSAaOMzgxjdz2yJ+LIvrQB6zmuYzef16PUgJ465GvvsJ0D5AXOIcAOjgmx8HAAm7lPxPv7WQns4ApxZdvRerQKh3x9ixH+v5jASkRPFQ45PAQDg7dfE0FtkYc9upRvKkdxAnIycLgxZQ1q6/jtzJIx7TCukgCzWDM0dFdFYQn8bgEzfcHxhrOTNvbgDd8uaQsCLFYheiwHoAFYiymIw/aqwEDG/kfxwToBBu7zjOl25wEG7GoA9WPu1UvIrkdHnNA9sEB2VT7BZylYy2GcBbBrB1bilFrzD43pzqFjiMbvEdnlcgwpdA4xQ70WQAUAGjDZNc0t/NQI+wSAtOtzpoT/uWdlBTHkLoeCDgW2hRmWzYW5D/ydnp2bmm1PudyZGFZe9FkNTodtf1McxxTAlgyvQH9nUgM8ZQcGb39A/Kg2SsZWjubvG54DB8AJ0lKHLMCQyGnGtAZH0HW7bMAs/6sAGaMa9LMipOfvUshDCJshDZoXWAibOwdGdZOuBgxxLF95KlcH5vLtfMCuFY0/qPdKdx4P9pUGD/W96nut7FnqYyXHg62UlWM1AtxX7jU1EsoZ8H51vEA0sPtblTEBJsAEmAB778A4mn+A2E/MjX6svzeODetYwBQtAzQVGRNgRBmrlJDQB6bggBdZI79EU/XnhVvFGQWMmXZ6lf/PBgwpVaCSBqUzivvkwGpQ8wQ2cM0JmO8JoIpEhnnOWc6594AaEj2KimgpyAsWPrfn/y1yo3Ooezsd4FkX78OLroNlDB6wQf6JtDAKv1+qt8vSpLLmNlZOHCqAtFoLI/kMQOxRe4iRMQwYa8UNwGEL8+MYYEPkQSH5Mew3oxhg/RT+GYzamRuccDzYHxE67JgTWJRRlmBEoiQCFUiKQFgpwD6k8C+VkJDQDeUuYFU2G9UO5CJUXYyvOWKi+T+FSSpFNwh9Ss2fkoxSQf2XtAOsi8K2pp+BmQvZuPbLdanHCvU2QeNNeXYB7CvzemfAvjOvdwZMWxA7P7b11Zjd7KwUEs3fIk86Da836D2ATGkh1zJVcooq7xjCa2/dxxpz+FbpvBqfDAxGYK7whdNLyxtI6lESeyRwvvY4UwDVRnuwhVSPl7SW1POwj7KiO2GKYaPTgo04ZVeNC9wRRnAfGrP2Eo7EhbXmtn6oNni2vhO1ljR2CO4MCriXlOgKZq6tJpIu2VTmbIIH7pCcBLa085Iz4MUML1VhHnMCUxHAlAD7tEmVtoziNueDfXtGhJUCLDfJIpeQkJgkMUkC7B2YJFLkpAL6cHYBrIk9y9TpyzZ1wS38rlIC9GV7QvpAJQHISdx1CjCkm5Zmb6HoG+2CAMbsGckKMDo5zO2vQu3akCxrTUyLkoH1EgAKakeYDZjRVpVaaqMTMOUtzcoTCP0uy6zsClBOzR8FiGIri2sAogh/cQ1AqdSFBvPNmKWcYOSQE9hUhW8/kyhJgAkJfWhK4VprWzqiaPPcUVJTg/G7w2s9Q9rgnxj7yd3APlN4BV2bgd9w94hzA96l5UY3I2Ov3g6RkdQ9ydZJgQGolfq1Q/UMo7BjOpisbs0UYGsDFOvmyM5n8sj5NrA/GTfVVQTzgM3FC4PdU0pv/lZgRh/YRoif4frQt2tQBy6QxlLwIgv1q06xsI+f4Y6Y+fA1VDw1k0DffIE9AEZ5bo4OgGs2w/d9TqIP2OWQMENxmt1LJ/YDoHeB6WJfRgcmy8nwfPnAwH/vG3495g7v7Z6uUKViss0enUafDUIrh83N60djNEzWfXMI/Nl6GXME/zH+O6hyuI9EM79Z32mzi+bMdeU2stVax3ZkjgNmyMJ3x+RwhXWmvXTZUTYw0/6NDGV5smbc1DZFoL8GDpYPqMBcAe+LJSPNbCrV66zhHHTV2Zg0tm4rHfL2wh4xa3Smli6qkBk8ho8N6lKFbvhxjaoJ7hBSGjPUtALPilaopg32xrIkfURtBOXHNoZcrEyHr+UohiNkf8zvTtSv87y0JZnHAjMrMs8ADjtfsNFx35GuNM15Sc1ozSl9oyj+mJ0mL33txa1RtZ2BdL3TEXBNExdX356JetuunNWCiXv01QKZeWdjMgwd/77KupbETBafYVTztvlCQDbH2prC/wNG7xgSsgm9OwLdd3OgCkrdUI94syaUGyEeaQXex65t9sE9/lDE1Tqq5n8iqgdUX2H3iDo4wDK+VFo24EDvzTFvN7ZMK6Q94dywFGsVuPLb87AwJOAYWKdqq6TAWoTUjKiHDgATjyyWMcBGbfJj6SRqUs55JjlHxmwleMRmHDyEYhdLSGOR9o/EbmbBwj0sfpxTnUuWgkXCfiqw4LO7Up3GiD2YvGUxde6CIujqFoGpDwmMU9J8akmE9DH10jZhYvr2YPksjokaiowJMCV9e4Q+MbVpfrse7EdoB3lYkRul0vzPSGC7sB7YR1zwF8T/GilmB7hYf2yEvMBSJdgVIZpfgAkwASbAruAovomEEPuJBRy6xAaLorKFb0lJdqQKMKEPQze5QtboMVgp69+i8OvShA3nnKMuwjdb8+vQbdNVc8YQk7QHgKdbA9aQzgYFlS7Eylh1TfmjLKTq8N6Xe00uf9TaHurOQb1kWKaQP05tT3NKXhLWRa2JOwA+AHuHOYHdB9zwABVzu5zqohcwYk1HyunNAAvYPpsXGMjVnKA2/ttVo9oXXZPoMWrZyxnURLLIqs0kPSnCTgUVsbwcykofKG2y/sxlzENysBrIOrf7w60iSCpHKYAlk6OmK7jvBXtdyZFRHNd05g0GtkpRMhraqKPXYhP3kYCa40iHXc1KH7umKcxWLxEgza7HlIa9dwV2VT7BD3UUQ7qGsN1ujms9NKY7h1ibRcmsBHZNVfiJU+ek9WMtm1V87JrmFn5OT5XLueGpAuCYWaln1y51k5dYYFmcw6jwzXAidd6CfAx3l6y86LManLarf6d0Hn2tfalegf7OpAZ4yg4M3v6A+FFtlIytHM3fNzwHDoCgGCFoXxK3r51hmrZJ9755/K8CZIxq0M8AsMwGLIbNyfNjqdjcOTCqm3Q1YIhj+cpTuTowl29HbfSoOgbo7e7W6wjICpkMUkUgwASYABNg1yBfiiAVDZIBY6QIRMYEWKiMVUpI6AMTt1JlovC0lG5AtQvdHhQEDOLCQtHWxy9npmYFhrTDpBDpDNRYBVso/rJe02sxj+a3Ok9yacxtWc7R/JMWVplC7srq6GXCQ1fA3gh3SxHSQy4Zw2TrzYzT/w+tVNkHtIYCs1OXlWem7RE5fcg1K006ENSEOIoC7N0Ak31JImMSVyIkh7cKda1gS25HLFEXAkyAJdb8Ng1ig1jL7Ul2JlfM+VsYiT8mwN7NrLycindrwE5d7d8VGRNgnxqYanGt+7c6YuuU3kSOYCTrrsAYGdN5/KYSvX+Lwt+Uys9uhZWoeVKZ91i2AdPBrY912eTvzqMuqOuUybdqUBZSOTufN6k2t3CWnqktJtp6E5tR0iIJMANgkt31OZokNKdMZT3gJNSIP6jI9ciUrnXIOfV5gTFbSWxV7iYJTGFP2s/nLoF6yNIBKWaRK6lCjRJ+g7J2OQoB1klfKA6wTjtpUYBllaNQYMlbSYDKqSgj38W+JK1u1pYe9J6o0UXAix1a/RR6eGtK+sq83hmw78zrnQErEaO+9enCTva+hcxKoQ9DubbKPkJM0IdQ7qCYTfnuGMI7sZzGCjmfa+5xv8lnKN0RR2DucK0vVZ7IeahR4HwlzfbZqC4fba3oB6aQSp57hMjIR9yD5xqdFmzEKU0RXOCOMIKuaN0bl/YSjsSFtTWbVlaQbB9gMYpJEYwdgjuDhMiyxTaauY6DCqgq5hjxypxN8MAdknTBdkG85EyqYIaXqjCPOYGpCGBKgOUmqRxGZq9Uqgiwq5JUDgsJiUkSkyTA3oFJIkVOKsEBhzmANbFn2eWyoC+lhI1eMoCUDezcNr5JAHISd50CDCkI0ewtFH1XcxDA4BxsboDRyWFue0xq071kWWtiWpQMrJcAUFA3+WzAAsppdAKmvKVZeQKh32WZlV0Byqn5owBRbGVxDUAU4S+uASiVurhscOEWjKfcYoZRzLYziZIEmNC7on8FGACaNfrF+pUsRgAAAABJRU5ErkJggg==");
                ytp_wm = jQuery("<div/>").addClass(a).html(c);
                c.attr("style", "filter:none!important;-webkit-transform:none!important;transform:none!important;padding:0!important;margin:0!important;height:100%!important; width:auto!important;display:block!important;visibility:visible!important;top:0!important;right:0!important;opacity:1!important;position:absolute!important;margin:auto!important;z-index:10000!important;");
                ytp_wm.attr("style", "filter:none!important;-webkit-transform:none!important;transform:none!important;padding:0!important;margin:0!important;display:block!important;position:absolute!important;top:0!important;bottom:0!important;right:0!important;margin:auto!important;z-index:10000!important;width:100%!important;height:100%!important;max-height:220px!important;");
                b.prepend(ytp_wm)
            })
        }, 5E3)
    });</script>
    <meta name="tec-api-version" content="v1">
    <meta name="tec-api-origin" content="https://hbp-hpc-platform.fz-juelich.de">
    <link rel="https://theeventscalendar.com/"
          href="https://hbp-hpc-platform.fz-juelich.de/index.php?rest_route=/tribe/events/v1/">
    <!-- Styles cached and displayed inline for speed. Generated by http://stylesplugin.com -->
    <style type="text/css" id="styles-plugin-css">
        .styles .searchform [type=search] {
            font-size: 19px;
            font-family: Helvetica, Arial, sans-serif
        }

        .styles #page .site-info a {
            font-size: 10px
        }
    </style>
</head>

<body class="archive tax-hbp_software_cat term-application-2 term-64 wp-embed-responsive _masterslider _ms_version_3.5.3 tribe-js styles single-author sidebar">
<div id="page" class="hfeed site">
    <header id="masthead" class="site-header" role="banner">
        <a class="home-link" href="https://hbp-hpc-platform.fz-juelich.de/"
           title="High Performance Analytics &amp; Computing Platform â€“ Guidebook" rel="home">
            <h1 class="site-title">High Performance Analytics &amp; Computing Platform â€“ Guidebook</h1>
            <h2 class="site-description">Infos and documentation of the HBP's HPAC Platform</h2>
        </a>

        <div id="navbar" class="navbar">
            <nav id="site-navigation" class="navigation main-navigation" role="navigation">
                <button class="menu-toggle">Menu</button>
                <a class="screen-reader-text skip-link" href="#content" title="Skip to content">Skip to content</a>
                <div class="menu-test-container">
                    <ul id="primary-menu" class="nav-menu">
                        <li id="menu-item-7"
                            class="menu-item menu-item-type-custom menu-item-object-custom menu-item-home menu-item-7">
                            <a href="https://hbp-hpc-platform.fz-juelich.de">Home</a></li>
                        <li id="menu-item-1118"
                            class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1118"><a
                                href="https://hbp-hpc-platform.fz-juelich.de/?page_id=1114">News</a></li>
                        <li id="menu-item-1786"
                            class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-1786">
                            <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=1783">Achievements</a>
                            <ul class="sub-menu">
                                <li id="menu-item-1790"
                                    class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1790"><a
                                        href="https://hbp-hpc-platform.fz-juelich.de/?page_id=1787">Simulation
                                    technology</a></li>
                                <li id="menu-item-1799"
                                    class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1799"><a
                                        href="https://hbp-hpc-platform.fz-juelich.de/?page_id=1795">Data-intensive
                                    supercomputing technology</a></li>
                                <li id="menu-item-1834"
                                    class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1834"><a
                                        href="https://hbp-hpc-platform.fz-juelich.de/?page_id=1800">Interactive
                                    visualisation and visual data analysis</a></li>
                                <li id="menu-item-1839"
                                    class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1839"><a
                                        href="https://hbp-hpc-platform.fz-juelich.de/?page_id=1836">Dynamic resource
                                    management</a></li>
                                <li id="menu-item-1843"
                                    class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1843"><a
                                        href="https://hbp-hpc-platform.fz-juelich.de/?page_id=1841">HPAC Platform v2</a>
                                </li>
                                <li id="menu-item-800"
                                    class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-800">
                                    <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=781">User Projects</a>
                                    <ul class="sub-menu">
                                        <li id="menu-item-803"
                                            class="menu-item menu-item-type-post_type menu-item-object-page menu-item-803">
                                            <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=787">Simulation
                                                &amp; analytics workflow: NEST â€“ Elephant â€“ SpiNNaker</a></li>
                                        <li id="menu-item-1231"
                                            class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1231">
                                            <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=1229">Simulation:
                                                Circuit simulation with Neuron</a></li>
                                        <li id="menu-item-1235"
                                            class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1235">
                                            <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=1233">Simulation:
                                                Interactive neurorobotics simulation</a></li>
                                        <li id="menu-item-801"
                                            class="menu-item menu-item-type-post_type menu-item-object-page menu-item-801">
                                            <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=797">Data analysis:
                                                Texture analysis in high-resolution images</a></li>
                                        <li id="menu-item-1240"
                                            class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1240">
                                            <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=1237">Data
                                                analytics: Characterising disease pathways using machine learning</a>
                                        </li>
                                        <li id="menu-item-802"
                                            class="menu-item menu-item-type-post_type menu-item-object-page menu-item-802">
                                            <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=794">Image
                                                processing: 3D-PLI workflow</a></li>
                                    </ul>
                                </li>
                            </ul>
                        </li>
                        <li id="menu-item-28"
                            class="menu-item menu-item-type-post_type menu-item-object-page current-menu-ancestor current-menu-parent current_page_parent current_page_ancestor menu-item-has-children menu-item-28">
                            <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=26">Application Software</a>
                            <ul class="sub-menu">
                                <li id="menu-item-29"
                                    class="menu-item menu-item-type-taxonomy menu-item-object-hbp_software_cat current-menu-item menu-item-29">
                                    <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=application-2"
                                       aria-current="page">Application</a></li>
                                <li id="menu-item-31"
                                    class="menu-item menu-item-type-taxonomy menu-item-object-hbp_software_cat menu-item-31">
                                    <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=programming-models-libraries-and-toolkits">Programming
                                        models, libraries and toolkits</a></li>
                                <li id="menu-item-30"
                                    class="menu-item menu-item-type-taxonomy menu-item-object-hbp_software_cat menu-item-30">
                                    <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data-management">Data
                                        management</a></li>
                            </ul>
                        </li>
                        <li id="menu-item-44"
                            class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-44">
                            <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=32">Infrastructure</a>
                            <ul class="sub-menu">
                                <li id="menu-item-312"
                                    class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-312">
                                    <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=309">Supercomputers</a>
                                    <ul class="sub-menu">
                                        <li id="menu-item-1034"
                                            class="menu-item menu-item-type-custom menu-item-object-custom menu-item-has-children menu-item-1034">
                                            <a>Supercomputers available</a>
                                            <ul class="sub-menu">
                                                <li id="menu-item-1002"
                                                    class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1002">
                                                    <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=933">JURECA</a>
                                                </li>
                                                <li id="menu-item-2067"
                                                    class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2067">
                                                    <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=1992">JUWELS</a>
                                                </li>
                                                <li id="menu-item-1001"
                                                    class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1001">
                                                    <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=940">Piz
                                                        Daint</a></li>
                                                <li id="menu-item-1485"
                                                    class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1485">
                                                    <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=1481">MareNostrum
                                                        4</a></li>
                                                <li id="menu-item-1178"
                                                    class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1178">
                                                    <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=1169">MARCONI</a>
                                                </li>
                                                <li id="menu-item-1000"
                                                    class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1000">
                                                    <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=942">Pico</a>
                                                </li>
                                                <li id="menu-item-2270"
                                                    class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2270">
                                                    <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=2212">Irene
                                                        (Joliot â€“ Curie)</a></li>
                                            </ul>
                                        </li>
                                        <li id="menu-item-1110"
                                            class="menu-item menu-item-type-custom menu-item-object-custom menu-item-has-children menu-item-1110">
                                            <a>PCP pilots</a>
                                            <ul class="sub-menu">
                                                <li id="menu-item-1108"
                                                    class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1108">
                                                    <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=1073">JURON
                                                        (IBM-NVIDIA pilot)</a></li>
                                            </ul>
                                        </li>
                                        <li id="menu-item-1179"
                                            class="menu-item menu-item-type-custom menu-item-object-custom menu-item-has-children menu-item-1179">
                                            <a>Old systems</a>
                                            <ul class="sub-menu">
                                                <li id="menu-item-1180"
                                                    class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1180">
                                                    <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=45">BlueBrain
                                                        IV</a></li>
                                                <li id="menu-item-101"
                                                    class="menu-item menu-item-type-post_type menu-item-object-page menu-item-101">
                                                    <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=49">FERMI</a>
                                                </li>
                                                <li id="menu-item-1109"
                                                    class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1109">
                                                    <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=1063">JULIA
                                                        (Cray pilot)</a></li>
                                                <li id="menu-item-99"
                                                    class="menu-item menu-item-type-post_type menu-item-object-page menu-item-99">
                                                    <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=34">JUQUEEN</a>
                                                </li>
                                                <li id="menu-item-102"
                                                    class="menu-item menu-item-type-post_type menu-item-object-page menu-item-102">
                                                    <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=47">MareNostrum
                                                        III</a></li>
                                            </ul>
                                        </li>
                                        <li id="menu-item-82"
                                            class="menu-item menu-item-type-post_type menu-item-object-page menu-item-82">
                                            <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=68">Access to
                                                supercomputers</a></li>
                                        <li id="menu-item-734"
                                            class="menu-item menu-item-type-post_type menu-item-object-page menu-item-734">
                                            <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=732">Working with
                                                supercomputers</a></li>
                                    </ul>
                                </li>
                                <li id="menu-item-706"
                                    class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-706">
                                    <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=676">HPC storage</a>
                                    <ul class="sub-menu">
                                        <li id="menu-item-1450"
                                            class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1450">
                                            <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=1448">Storage
                                                services</a></li>
                                    </ul>
                                </li>
                                <li id="menu-item-1328"
                                    class="menu-item menu-item-type-custom menu-item-object-custom menu-item-has-children menu-item-1328">
                                    <a>Other compute resources</a>
                                    <ul class="sub-menu">
                                        <li id="menu-item-1329"
                                            class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1329">
                                            <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=1326">KIT OpenStack
                                                Deployment</a></li>
                                    </ul>
                                </li>
                                <li id="menu-item-94"
                                    class="menu-item menu-item-type-post_type menu-item-object-page menu-item-94"><a
                                        href="https://hbp-hpc-platform.fz-juelich.de/?page_id=51">Cloud storage</a></li>
                                <li id="menu-item-93"
                                    class="menu-item menu-item-type-post_type menu-item-object-page menu-item-93"><a
                                        href="https://hbp-hpc-platform.fz-juelich.de/?page_id=53">Visualisation
                                    systems</a></li>
                                <li id="menu-item-91"
                                    class="menu-item menu-item-type-post_type menu-item-object-page menu-item-91"><a
                                        href="https://hbp-hpc-platform.fz-juelich.de/?page_id=58">Network</a></li>
                                <li id="menu-item-746"
                                    class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-746">
                                    <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=744">Platform
                                        configuration</a>
                                    <ul class="sub-menu">
                                        <li id="menu-item-90"
                                            class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-90">
                                            <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=60">UNICORE</a>
                                            <ul class="sub-menu">
                                                <li id="menu-item-96"
                                                    class="menu-item menu-item-type-post_type menu-item-object-page menu-item-96">
                                                    <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=62">UNICORE
                                                        installation</a></li>
                                                <li id="menu-item-95"
                                                    class="menu-item menu-item-type-post_type menu-item-object-page menu-item-95">
                                                    <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=64">UNICORE
                                                        Portal</a></li>
                                                <li id="menu-item-1155"
                                                    class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1155">
                                                    <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=1153">UNICORE
                                                        REST API</a></li>
                                                <li id="menu-item-769"
                                                    class="menu-item menu-item-type-post_type menu-item-object-page menu-item-769">
                                                    <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=767">UNICORE
                                                        Workflow Engine</a></li>
                                            </ul>
                                        </li>
                                        <li id="menu-item-89"
                                            class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-89">
                                            <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=66">User &amp;
                                                group management</a>
                                            <ul class="sub-menu">
                                                <li id="menu-item-448"
                                                    class="menu-item menu-item-type-post_type menu-item-object-page menu-item-448">
                                                    <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=446">Naming
                                                        scheme for users and groups</a></li>
                                                <li id="menu-item-321"
                                                    class="menu-item menu-item-type-custom menu-item-object-custom menu-item-321">
                                                    <a href="https://trac.version.fz-juelich.de/hbpaccounting">Accounting
                                                        library</a></li>
                                            </ul>
                                        </li>
                                    </ul>
                                </li>
                            </ul>
                        </li>
                        <li id="menu-item-585"
                            class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-585">
                            <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=583">Training</a>
                            <ul class="sub-menu">
                                <li id="menu-item-575"
                                    class="menu-item menu-item-type-custom menu-item-object-custom menu-item-575"><a
                                        href="https://hbp-hpc-platform.fz-juelich.de/?post_type=tribe_events&amp;tribe_event_display=month">Trainings
                                    &amp; Events</a></li>
                                <li id="menu-item-543"
                                    class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-543">
                                    <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=529">Online training</a>
                                    <ul class="sub-menu">
                                        <li id="menu-item-547"
                                            class="menu-item menu-item-type-post_type menu-item-object-page menu-item-547">
                                            <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=531">Software
                                                development, computing &amp; programming languages</a></li>
                                        <li id="menu-item-546"
                                            class="menu-item menu-item-type-post_type menu-item-object-page menu-item-546">
                                            <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=534">Parallel
                                                programming &amp; High Performance Computing</a></li>
                                        <li id="menu-item-545"
                                            class="menu-item menu-item-type-post_type menu-item-object-page menu-item-545">
                                            <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=539">Additional
                                                training offered by the HPC Platform partners</a></li>
                                        <li id="menu-item-544"
                                            class="menu-item menu-item-type-post_type menu-item-object-page menu-item-544">
                                            <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=541">PRACE videos
                                                and tutorials</a></li>
                                    </ul>
                                </li>
                                <li id="menu-item-582"
                                    class="menu-item menu-item-type-post_type menu-item-object-page menu-item-582"><a
                                        href="https://hbp-hpc-platform.fz-juelich.de/?page_id=576">SP7 partnersâ€™
                                    courses</a></li>
                                <li id="menu-item-593"
                                    class="menu-item menu-item-type-custom menu-item-object-custom menu-item-593"><a
                                        href="https://education.humanbrainproject.eu/">HBP Education Programme</a></li>
                                <li id="menu-item-590"
                                    class="menu-item menu-item-type-post_type menu-item-object-page menu-item-590"><a
                                        href="https://hbp-hpc-platform.fz-juelich.de/?page_id=588">Training request</a>
                                </li>
                            </ul>
                        </li>
                        <li id="menu-item-518"
                            class="menu-item menu-item-type-post_type menu-item-object-page menu-item-518"><a
                                href="https://hbp-hpc-platform.fz-juelich.de/?page_id=491">About us</a></li>
                        <li id="menu-item-154"
                            class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-154">
                            <a href="https://hbp-hpc-platform.fz-juelich.de/?page_id=152">Contact &amp; Support</a>
                            <ul class="sub-menu">
                                <li id="menu-item-2096"
                                    class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2096"><a
                                        href="https://hbp-hpc-platform.fz-juelich.de/?page_id=2004">Request for
                                    application performance optimisation and code migration support</a></li>
                                <li id="menu-item-772"
                                    class="menu-item menu-item-type-post_type menu-item-object-page menu-item-772"><a
                                        href="https://hbp-hpc-platform.fz-juelich.de/?page_id=442">Booking request for
                                    visualization systems</a></li>
                            </ul>
                        </li>
                    </ul>
                </div>
                <form role="search" method="get" class="search-form" action="https://hbp-hpc-platform.fz-juelich.de/">
                    <label>
                        <span class="screen-reader-text">Search for:</span>
                        <input type="search" class="search-field" placeholder="Search â€¦" value="" name="s">
                    </label>
                    <input type="submit" class="search-submit" value="Search">
                </form>
            </nav><!-- #site-navigation -->
        </div><!-- #navbar -->
    </header><!-- #masthead -->


    <div id="main" class="site-main">

        <div id="tertiary" class="sidebar-container" role="complementary">
            <div class="sidebar-inner">
                <div class="widget-area">

                    <aside id="text-3" class="widget widget_text">
                        <div class="textwidget" style="margin-left:150px;">
                            <form action="" method="post" class="searchandfilter">
                                <div>
                                    <ul>
                                        <li><h3 class="archive-title">Categories</h3>
                                            <input type="reset" value="Reset Filter" style="padding: 5px"
                                                   onclick="clearFilters()">
                                            <ul>
                                                <li class="cat-item cat-item-3">
                                                    <details>
                                                        <summary>
                                                            <label>Usage</label>
                                                        </summary>
                                                        <ul class="children">

                                                            <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="filamenttracer imagej pliviewer simple-neurite-tracer vaa3d viola visualization-toolkit">Interactive Analysis</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="filamenttracer imagej pliviewer simple-neurite-tracer vaa3d viola visualization-toolkit">Presentation</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="filamenttracer imagej pliviewer simple-neurite-tracer vaa3d visualization-toolkit">Processing</input></li>

                                                        </ul>
                                                    </details>
                                                </li>
                                                <li class="cat-item cat-item-3">
                                                    <details>
                                                        <summary>
                                                            <label>Data Type</label>
                                                        </summary>
                                                        <ul class="children">

                                                            <li class="cat-item cat-item-3">
                                                                <details>
                                                                    <summary>
                                                                        <label>Abstract</label>
                                                                    </summary>
                                                                    <ul class="children">
                                                                        <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="filamenttracer visualization-toolkit">Graph</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="visualization-toolkit">Sensor</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="filamenttracer simple-neurite-tracer viola visualization-toolkit">Statistic</input></li>
                                                                    </ul>
                                                                </details>
                                                            </li>


                                                            <li class="cat-item cat-item-3">
                                                                <details>
                                                                    <summary>
                                                                        <label>2D</label>
                                                                    </summary>
                                                                    <ul class="children">
                                                                        <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="filamenttracer imagej pliviewer simple-neurite-tracer vaa3d viola visualization-toolkit">Raster</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="filamenttracer imagej visualization-toolkit">Vector</input></li>
                                                                    </ul>
                                                                </details>
                                                            </li>


                                                            <li class="cat-item cat-item-3">
                                                                <details>
                                                                    <summary>
                                                                        <label>3D</label>
                                                                    </summary>
                                                                    <ul class="children">
                                                                        <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="filamenttracer vaa3d visualization-toolkit">Model</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="filamenttracer pliviewer simple-neurite-tracer vaa3d visualization-toolkit">Raster</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="pliviewer">Tensor</input></li>
                                                                    </ul>
                                                                </details>
                                                            </li>

                                                        </ul>
                                                    </details>
                                                </li>

                                                <li class="cat-item cat-item-3">
                                                    <details>
                                                        <summary>
                                                            <label>Data Format</label>
                                                        </summary>
                                                        <ul class="children">

                                                            <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="visualization-toolkit">3D Studio</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="3dsvm afni caret fieldtrip fmristat jist:-java-image-science-toolkit jist-resources-for-algorithm-development loni-pipeline-environment mipav mricrogl neurodebian neuroelf neurolens nidb---neuroinformatics-database nimh-data-archive-/-national-database-for-autism-research r-package-for-adaptive-dwi-analysis r-package-for-adaptive-fmri-analysis sumsdb-database surfstat">AFNI BRIK</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="3d-slicer abeat absorb-for-groupwise-registration adni advanced-normalization-tools afni amilab analyze anatomist arctic artifact-detection-tools-(art) artrepair-for-robust-fmri asa---advanced-source-analysis asl-data-processing-tool-box automatic-segmentation-tool-adapter autoseg b0-and-eddy-current-correction-for-dti bioimage-suite braindecodertoolbox brainmask-volume-processing-tool brainnetome-diffusionkit brainparser brainsconstellationdetector brain-segmentation-testing-protocol brainsfit brainsmush brainsolution brainstools brainstorm brainsuite brainvisa brainvisa-morphologist-extensions brainvoyager brede-toolbox bxh/xcede-tools caret computational-morphometry-toolkit-(cmtk) conn-:-functional-connectivity-toolbox diamond dicomconvert diffusion-mri-tool diffusion-toolkit-/-trackvis dinifti dramms-deformable-image-registration-toolbox dsi-studio dti-reg dti-tk elastix fast-nonlocal-means-for-mri-denoising fbirn-image-processing-scripts-(fips) fieldtrip fiv fmri-cpca fmripower fmristat fsl functional-imaging-birn fusion-ica-toolbox-(fit) gambit gamma gcca gimias glirt gmac:-a-matlab-toolbox-for-spectral-granger-causality-analysis-of-fmri-data group-ica-toolbox-(gift-and-eegift) hcp-harvard/mgh-ucla icn_atlas idea-lab-brain-image-processing-suite insight-toolkit instar-(infant-joint-segmentation-and-registration) itk-snap jim jist:-java-image-science-toolkit jist-resources-for-algorithm-development lddmm lead-connectome lead-dbs loni-debabeler loni-de-identification-debablet loni-inspector loni-pipeline-environment mango marsbar-region-of-interest-toolbox mask_explorer menga:-multimodal-environment-for-neuroimaging-and-genomic-analysis mipav mixed-effect-model-of-genetic-set-and-environment-interaction mousebirn-atlasing-toolkit-(mbat) mouse-connectome-project-(mcp) mricrogl mricron mricros mri-studio mrtrix multitracer-version-2 neuroelf neuroimaging-made-easy-blog neurolens nidb---neuroinformatics-database niftyreg nih-cidi-lung-segmentation-tool nih-cidi-segmentation-of-pet-images-based-on-affinity-propagation-clustering nilearn niral-utilities noddi-matlab-toolbox nordicneurolab northwestern-university-schizophrenia-data-and-software-tool-(nusdast) nptk numerical-fibre-generator-(nfg) nvm odin ogles---opengl/oiv-stereotactic-tool p-[(123)i]iodo-l-phenylalanine-(ipa)-spect-template-in-mni-coordinates panda:-a-pipeline-tool-for-diffusion-mri pestica-&-slomoco:-physio-and-motion-correction-tools ppmi pymvpa pynifti rest:-a-toolkit-for-resting-state-fmri resting-state-pediatric-imaging-template rex r-package-for-adaptive-dwi-analysis r-package-for-adaptive-fmri-analysis seg3d segmentation-validation-engine simple-medical-imaging-library-interface-(smili) slicer3-example-modules snpm spharm-mat spharm-pdm-toolbox spikecor:-fmri-tool-for-automated-correction-of-head-motion-spikes spm spm_ss---fmri-functional-localizers sri24-atlas:-normal-adult-brain-anatomy staple subject-library subject-order-independent-group-ica sumsdb-database surface-based-processing-and-analysis-of-mri surfstat tapas---translational-algorithms-for-psychiatry-advancing-science template-image-processing-library tractor:-tractography-with-r turbofire turtleseg unc-0-1-2-infant-atlases user-friendly-functional-connectivity---uf²c valmap:-simple-statistical-mapping-tool virtual-reality-brain-network-viewer vis:-visualized-statistics-toolbox-(spm5/8/12) voxbo wfu_bpm wfu_pickatlas wfu_pipeline xjview,-a-viewing-tool-for-spm">ANALYZE</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="asa---advanced-source-analysis eeglab">ANT</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="vaa3d">APO</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="analyzer-2 asa---advanced-source-analysis eeglab g.bsanalyze neuroimaging-made-easy-blog niagads---nia-genetics-of-alzheimer's-disease-data-storage-site nidb---neuroinformatics-database nimh-data-archive-/-national-database-for-autism-research nonfractal nordicneurolab population-averaged-diffusion-tensor-imaging-atlas-of-the-sprague-dawley-rat-brain slice:drop">ASCII</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="imagej">AVI</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="visualization-toolkit">AVS "UCD" format</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="fmriprep">BIDS (Brain Imaging Data Structure)</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="imagej">BMP</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="analyzer-2 asa---advanced-source-analysis eeglab the-neurophysiological-biomarker-toolbox">Brain Vision Analyzer Data Exchange Format</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="visualization-toolkit">CAD STL</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="visualization-toolkit">CGM</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="cifti-connectivity-file-format connectome-workbench hcp-wu-minn-consortium nidb---neuroinformatics-database nimh-data-archive-/-national-database-for-autism-research">CIFTI</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="analyzer-2 asa---advanced-source-analysis eeglab">CNT</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="jist:-java-image-science-toolkit jist-resources-for-algorithm-development mipav neurolens nidb---neuroinformatics-database nimh-data-archive-/-national-database-for-autism-research nvm">COR</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="filamenttracer simple-neurite-tracer">CSV</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="asa---advanced-source-analysis">CTF</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="viola">DAT</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="3d-slicer adni adolescent-brain-cognitive-development-(abcd)-study advanced-normalization-tools afni amilab analyze anatomist asa---advanced-source-analysis brainance-dti-module braincsi brainmask-volume-processing-tool brainnetome-diffusionkit brainscape brainsconstellationdetector brainsfit brainsight brainsmush brainstools brainvisa brainvoyager brainvoyager-viewer bxh/xcede-tools cbica:-cancer-imaging-phenomics-toolkit-(captk) computational-morphometry-toolkit-(cmtk) daikon data-format-tools dcm2nii dcmtk---dicom-toolkit diamond dicombrowser dicomconvert diffusional-kurtosis-estimator diffusion-toolkit-/-trackvis dinifti dsi-studio fbirn-image-processing-scripts-(fips) fieldtrip fiv functional-imaging-birn gimias gradient-non-linearity-unwarping-tool gtract hcp-harvard/mgh-ucla hcp-wu-minn-consortium imagej insight-toolkit itk-snap jim jist:-java-image-science-toolkit jist-resources-for-algorithm-development lead-connectome loni-debabeler loni-de-identification-debablet loni-inspector loni-pipeline-environment mango mcic--schizophrenic-and-matched-control-data mipav mitk-diffusion morphometry-birn mricrogl mricron mrtrix neurodebian neuroelf neuroimaging-made-easy-blog neurolens nidb---neuroinformatics-database nih-pediatric-mri-data-repository nimh-data-archive-/-national-database-for-autism-research nitrc-computational-environment-(nitrc-ce) nordicneurolab notion-researchpacs nvm oasis odin panda:-a-pipeline-tool-for-diffusion-mri papaya philips-users-community ppmi pydicom r-package-for-adaptive-dwi-analysis r-package-for-adaptive-fmri-analysis seg3d self-threat-and-self-concept-data simple-medical-imaging-library-interface-(smili) slice:drop slicer3-example-modules spm staple template-image-processing-library the-cancer-imaging-archive-(tcia) tractor:-tractography-with-r turbo-brainvoyager turbofire turtleseg vandpire visualization-toolkit vmtk-in-3d-slicer voxbo wfu_pipeline xnat">DICOM</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="analyzer-2 asa---advanced-source-analysis eeglab openvibe the-neurophysiological-biomarker-toolbox">EDF/BDF/GDF</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="brainvoyager nidb---neuroinformatics-database recorder recview">EEG Formats</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="analyzer-2 asa---advanced-source-analysis eeglab the-neurophysiological-biomarker-toolbox">EGI</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="visualization-toolkit">Ensight</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="asa---advanced-source-analysis">FIF</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="imagej">FITS</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="visualization-toolkit">Facet</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="visualization-toolkit">Fluent GAMIT ASCII</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="viola">GDF</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="imagej">GIF</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="afni brainvisa brainvoyager computational-anatomy-toolbox---cat connectome-file-format-(cff) connectome-viewer connectome-workbench gifti hcp-wu-minn-consortium jist:-java-image-science-toolkit jist-resources-for-algorithm-development mango modelgui mricros nidb---neuroinformatics-database solar-eclipse-imaging-genetics-tools spm staple surf-ice the-virtual-brain-(tvb)">GIfTI</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="visualization-toolkit">Gaussian</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="pliviewer">HDF5</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="analyze nidb---neuroinformatics-database">Interfile</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="imagej">JPEG</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="viola">JSON</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="viola">LFP</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="imagej">LUT</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="asa---advanced-source-analysis">M4D</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="nidb---neuroinformatics-database">MEG Formats</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="3d-slicer brainbox brainstools brainstorm bxh/xcede-tools conn-:-functional-connectivity-toolbox cortex freesurfer freesurfr jist:-java-image-science-toolkit jist-resources-for-algorithm-development loni-pipeline-environment mindboggle-101-manually-labeled-brains mindboggle:-open-source-software-for-analyzing-the-shapes-of-brain-structures-from-human-mri-data mipav misteri mne---minimum-norm-current-estimates mricrogl mri-defacer neuroimaging-made-easy-blog neurolens nidb---neuroinformatics-database rft_fdr slice:drop surface-based-processing-and-analysis-of-mri tractor:-tractography-with-r">MGH/MGZ</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="afni amilab anatomist brainsight brainvisa bxh/xcede-tools caret fast-nonlocal-means-for-mri-denoising fieldtrip fiv fmristat jist:-java-image-science-toolkit jist-resources-for-algorithm-development libminc---medical-image-netcdf-library loni-debabeler loni-de-identification-debablet loni-inspector loni-pipeline-environment loris minc-example-files minc-toolkit mipav mni-macaque-atlases neuroimaging-analysis-kit-(niak) neurolens nidb---neuroinformatics-database nih-pediatric-mri-data-repository nimh-data-archive-/-national-database-for-autism-research ppmi surfstat turtleseg">MINC</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="bioimage-suite brainbrowser brainsight fieldtrip insight-toolkit jist:-java-image-science-toolkit jist-resources-for-algorithm-development libminc---medical-image-netcdf-library loni-pipeline-environment loris minc-example-files minc-toolkit mipav mni-macaque-atlases neurodebian neuroimaging-analysis-kit-(niak) nih-pediatric-mri-data-repository nimh-data-archive-/-national-database-for-autism-research">MINC2</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="visualization-toolkit">Marching Cubes</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="absorb-for-groupwise-registration magpro-magnetic-stimulator-(tms) niagads---nia-genetics-of-alzheimer's-disease-data-storage-site nidb---neuroinformatics-database spanish-resting-state-network">Microsoft Word Document</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="visualization-toolkit">Movie BYU</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="1000-functional-connectomes-project 3d-brain-atlas-reconstructor 3d-slicer 3dsvm 7t-structural-mri-scans-atag adni adolescent-brain-cognitive-development-(abcd)-study advanced-normalization-tools afni ahead analyze anatomist a-robust-method-for-correcting-partial-volume-effect-in-asl-data artrepair-for-robust-fmri ashs:-automatic-segmentation-of-hippocampal-subfields asl-data-processing-tool-box automatic-registration-toolbox automatic-segmentation-tool-adapter bash4rfmri bioimage-suite brainance-dti-module brainbox brainbrowser braincsi brainfx brainmagix-spm-viewer brainmap-database brainnetome-diffusionkit brain-registration-evaluations-(2008-2011) brainsconstellationdetector brain-segmentation-testing-protocol brainsfit brainsight brainsmush brainstools brainstorm brainsuite brainvisa brainvisa-morphologist-extensions brainvoyager bravissima:-human-brain-artery-nifti-atlas broccoli bxh/xcede-tools camba candi-neuroimaging-access-point caret cbica:-cancer-imaging-phenomics-toolkit-(captk) cbica:-tutorials-(image-processing-and-machine-learning)-using-cpp,-itk,-etc. cbica:-whitestripe colin-3t/7t-high-resolution-atlas computational-anatomy-toolbox---cat computational-morphometry-toolkit-(cmtk) connectome-file-format-(cff) connectome-viewer connectome-workbench conn-:-functional-connectivity-toolbox corpus-callosum-and-brain-segmentations-from-abide-database corpus-callosum-thickness-profile-analysis-pipeline c-pac data-format-tools dcm2nii demon-deep-multi-contrast-brain-extraction,-application-to-human-&-animal-imaging diamond dicomconvert diffusional-kurtosis-estimator diffusion-mri---in-vivo-and-phantom-data diffusion-mri-tool diffusion-toolkit-/-trackvis dinifti displacement-field-viewer dramms-deformable-image-registration-toolbox dsi-studio dti-denoising dti-tk elastix enigma:-enhancing-neuro-imaging-genetics-through-meta-analysis evaluation-of-automated-brain-extraction-methods-on-multi-site-multi-disorder-data fast-nonlocal-means-for-mri-denoising fast-t2-relaxation-data-analysis-with-stimulated-echo-correction-and-non-local-spatial-regularisation fieldtrip fiv fmri-cpca fmripower fmristat forward:-accurate-finite-element-electromagnetic-head-models fsl functional-imaging-birn functional-real-time-interactive-endogenous-neuromodulation-and---decoding-(friend) functional-roi-atlas fusion-ica-toolbox-(fit) gamma gcca gimias gmac:-a-matlab-toolbox-for-spectral-granger-causality-analysis-of-fmri-data gradient-non-linearity-unwarping-tool graph-theory-glm-(gtg)-matlab-toolbox group-ica-toolbox-(gift-and-eegift) gtract haiko89:-mri-in-vivo-baboon-brain-templates-collection-(papio-anubis) hcp-harvard/mgh-ucla hcp-wu-minn-consortium high-quality-diffusion-weighted-imaging-of-parkinson's-disease histolozee---3d-histology-reconstruction-and-co-registration-with-mri icn_atlas idea-lab-brain-image-processing-suite ielectrodes inia19-primate--brain-atlas insight-toolkit iterative-dual-regression-with-sparse-prior itk-snap ixi-dataset jim jip-fmri-analysis-toolkit jist:-java-image-science-toolkit jist-resources-for-algorithm-development lead-connectome lead-dbs longitudinal-rhesus-monkey-brain-templates loni-debabeler loni-de-identification-debablet loni-inspector loni-pipeline-environment mango manually-labeled-mri-brain-scan-database mapping-population-based-structural-connectomes marsbar-region-of-interest-toolbox masked-ica-(mica)-toolbox mask_explorer menga:-multimodal-environment-for-neuroimaging-and-genomic-analysis mgh-neonatal/pediatric-adc-atlases mindboggle-101-manually-labeled-brains mindboggle:-open-source-software-for-analyzing-the-shapes-of-brain-structures-from-human-mri-data mipav misteri mitk-diffusion mni-macaque-atlases modelgui morphometry-birn mousebirn-atlasing-toolkit-(mbat) mricloud mricrogl mricron mricros mri-denoising mrtool mrtrix multi-modal-mri-reproducibility-resource multitracer-version-2 multixplore:-visual-exploration-platform-for-multimodal-neuroimaging-data ncanda:-data-integration-component neurodebian neuroelf neuroimaging-made-easy-blog neurosynth neurovault niag-addiction-data nifti niftyreg nih-pediatric-mri-data-repository niistat nilearn nimh-data-archive-/-national-database-for-autism-research nitrc-computational-environment-(nitrc-ce) noddi-matlab-toolbox nordicneurolab northwestern-university-schizophrenia-data-and-software-tool-(nusdast) nptk nrm2018-pet-grand-challenge-dataset ntu-dsi-122:-a-diffusion-spectrum-imaging-template-with-high-anatomical-matching-to-the-icbm-152-space nvm nyu-csc-testretest odin ogles---opengl/oiv-stereotactic-tool onprc-infant-macaque-brain-atlas panda:-a-pipeline-tool-for-diffusion-mri papaya penn-hippocampus-atlas philips-users-community phycaa+:-adaptive-physiological-noise-correction-for-bold-fmri population-averaged-diffusion-tensor-imaging-atlas-of-the-sprague-dawley-rat-brain progression-score-model-toolkit pronto-–-pattern-recognition-for-neuroimaging-toolbox pymvpa pynifti rapidart rapidtide rda2nifti rest:-a-toolkit-for-resting-state-fmri rex r-package-for-adaptive-dwi-analysis r-package-for-adaptive-fmri-analysis ruby-nifti s3dl-sparse-dictionary-learning-based-mr-image-and-lesion-segmentation segmentation-validation-engine serotonin-transporter-highresolution-pet-template simple-medical-imaging-library-interface-(smili) simulated-diffusion-weighted-datasets simulated-dw-mri-brain-data-sets-for-quantitative-evaluation-of-estimated-fiber-orientations slice:drop slicer3-example-modules snpm solar-eclipse-imaging-genetics-tools spanish-resting-state-network spharm-mat spharm-pdm-toolbox spikecor:-fmri-tool-for-automated-correction-of-head-motion-spikes spinal-cord-toolbox spm spm_ss---fmri-functional-localizers sri24-atlas:-normal-adult-brain-anatomy staple subject-order-independent-group-ica sumsdb-database surf-ice surfstat svv tapas---translational-algorithms-for-psychiatry-advancing-science task-based-fmri-atlases template-image-processing-library templates-for-in-vivo-mouse-brain templates-for-long-evans-rat-brain the-cancer-imaging-archive-(tcia) the-virtual-brain-(tvb) tractor:-tractography-with-r turtleseg umanitoba-jhu-functionally-defined-human-white-matter-atlas unc-cedars-infant-atlas user-friendly-functional-connectivity---uf²c valmap:-simple-statistical-mapping-tool vandpire vartbx-–-variability-toolbox virtual-digital-brain vis:-visualized-statistics-toolbox-(spm5/8/12) voxbo waxholm-space-atlas-of-the-sprague-dawley-rat-brain wfu_bpm wfu_pipeline wisconsin-white-matter-hyperintensities-segmentation-toolbox xjview,-a-viewing-tool-for-spm xnat">NIfTI-1</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="3d-slicer amilab arctic autoseg autotract brainsconstellationdetector brainsmush brainstools computational-morphometry-toolkit-(cmtk) dicomconvert diffusion-mri---in-vivo-and-phantom-data diffusion-mri-tool diffusion-tractography-with-kalman-filter displacement-field-viewer dtiprocess-toolkit dti-reg dwi/dti-quality-control-tool:-dtiprep elastix fast-nonlocal-means-for-mri-denoising finsler-tractography-module-for-slicer gambit insight-toolkit itk-snap jist:-java-image-science-toolkit jist-resources-for-algorithm-development joint-anisotropic-lmmse-filter-for-stationary-rician-noise-removal-in-dwi mipav misteri mitk-diffusion mricrogl multixplore:-visual-exploration-platform-for-multimodal-neuroimaging-data niftyreg niral-utilities seg3d simple-medical-imaging-library-interface-(smili) slice:drop spharm-pdm-toolbox sri24-atlas:-normal-adult-brain-anatomy staple the-cancer-imaging-archive-(tcia) unc-human-brain-atlas unc-primate-brain-atlas vmtk-in-3d-slicer">Nrrd</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="visualization-toolkit">OBJ</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="visualization-toolkit">Open Inventor 2.0</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="3d-brain-atlas-reconstructor 3d-slicer adni advanced-normalization-tools afni amilab analyze analyzer-2 anatomist arctic autoseg biofilmquant biosig brainbrowser brain-connectivity-toolbox braindecodertoolbox braingraph-editor-1.0-beta brainmask-volume-processing-tool brainsfit brainstools brainstorm brainvisa brainvisa-morphologist-extensions brainvoyager-brain-tutor brainvoyager-viewer brava bxh/xcede-tools caret cleave:-large-data-set-anova computational-morphometry-toolkit-(cmtk) connectome-file-format-(cff) connectome-viewer connectome-workbench dalan:-disconnectivity-atlas-for-lesion-analysis-of-networks diffusion-tractography-with-kalman-filter displacement-field-viewer dsi-studio dti-reg elastix e-prime fast-nonlocal-means-for-mri-denoising fiberviewerlight fieldtrip gambit gamma gannet---batch-analysis-of-edited-magnetic-resonance-spectroscopy-(mrs)-data gazereader g.bsanalyze gimias graph-theory-glm-(gtg)-matlab-toolbox gtract hcp-harvard/mgh-ucla histolozee---3d-histology-reconstruction-and-co-registration-with-mri human-imaging-database-(hid) idea-lab-brain-image-processing-suite iso2mesh itk-snap itools-resourceome jim jip-fmri-analysis-toolkit jist:-java-image-science-toolkit jist-resources-for-algorithm-development limo-eeg loni-pipeline-environment mango map3d meg-tools mesh-based-monte-carlo-(mmc) mindboggle-101-manually-labeled-brains mipav mitk-diffusion mne---minimum-norm-current-estimates monte-carlo-extreme-(mcx) monte-carlo-simulation-software:-tmcimg mouse-connectome-project-(mcp) mpscope mricros mrtrix nc-igt-fast-imaging-library neurodebian neuroelf neuroinformatics---the-journal neurolens neurosharelibrary nimh-data-archive-/-national-database-for-autism-research niral-utilities nvm ogles---opengl/oiv-stereotactic-tool predict pymvpa quantitative-diffusion-tools recview seg3d shapeworks slice:drop slicer3-module-rician-noise-filter spharm-mat spharm-pdm-toolbox statistics-online-computational-resource stochastic-tractography-system sumsdb-database talairach-daemon template-image-processing-library the-cancer-imaging-archive-(tcia) the-virtual-brain-(tvb) turbofire turtleseg vaa3d-and-vaa3d-neuron vmtk-in-3d-slicer voxbo xnbc">Other Format</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="visualization-toolkit">PDB</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="absorb-for-groupwise-registration connectome-file-format-(cff) ncanda:-data-integration-component neurodebian niagads---nia-genetics-of-alzheimer's-disease-data-storage-site nimh-data-archive-/-national-database-for-autism-research spanish-resting-state-network textpresso---literature-search-engine the-cancer-imaging-archive-(tcia)">PDF - Portable Document Format</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="imagej">PGM</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="visualization-toolkit">PLOT3D</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="visualization-toolkit">PLY</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="imagej">PNG</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="visualization-toolkit">Particle</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="brainsight brainvoyager elastix gannet---batch-analysis-of-edited-magnetic-resonance-spectroscopy-(mrs)-data insight-toolkit jist:-java-image-science-toolkit mricrogl neurodebian neuroelf nimh-data-archive-/-national-database-for-autism-research philips-users-community tapas---translational-algorithms-for-psychiatry-advancing-science vandpire">Philips PAR/REC</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="imagej">Raw Data</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="visualization-toolkit">Renderman</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="simple-neurite-tracer vaa3d">SWC</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="imagej vaa3d">TIFF</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="simple-neurite-tracer">TRACES</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="nimh-data-archive-/-national-database-for-autism-research">TeX</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="3d-brain-atlas-reconstructor analyze jim nimh-data-archive-/-national-database-for-autism-research">VRML</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="visualization-toolkit">VRML 2.0</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="visualization-toolkit">VTK Poly Data</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="visualization-toolkit">VTK Structured Grid</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="vaa3d">Vaa3D raw</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="visualization-toolkit">XYZ</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="3d-slicer amilab brainmask-volume-processing-tool insight-toolkit jist-resources-for-algorithm-development loni-pipeline-environment mipav mpscope mrtrix neurodebian nidb---neuroinformatics-database nimh-data-archive-/-national-database-for-autism-research nvm surf-ice">bshort/bfloat</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="nidb---neuroinformatics-database nimh-data-archive-/-national-database-for-autism-research the-virtual-brain-(tvb)">cff (Connectome File Format)</input></li>

                                                        </ul>
                                                    </details>
                                                </li>

                                                <li class="cat-item cat-item-3">
                                                    <details>
                                                        <summary>
                                                            <label>Deployment</label>
                                                        </summary>
                                                        <ul class="children">

                                                            <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="visualization-toolkit">HPC</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="visualization-toolkit">Mobile</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="1000-functional-connectomes-project [123i]-fp-cit-spect-brain-template-in-mni-space 18f-dopa-pet-and-123i-fp-cit-(ioflupane---datscan)-spect-templates-for-spm-normalisation. 2008-miccai-ms-lesion-segmentation-challenge 3d-brain-atlas-reconstructor 3-d-connective-field-modeling-toolbox 3-dimensional-diffusion-tensor-imaging-(dti)-atlas-of-the-rat-brain-in-postnatal-day-5,-14,-and-adulthood 3dmeshmetric 3d-slicer 3dslicerlupuslesionmodule 3dsvm 3t-dwi-test-retest-reliability-dataset 3t-mri-data-set-on-structural-brain-behavior-correlations 4d-atlas-construction 4d-parser-(pathological-anatomy-regression-via-segmentation-and-registration) 4d-(spatio-temporal)-dramms-registration 7t-probabilistic-atlas-of-subthalamic-nucleus-for-use-with-3t-mri 7t-structural-mri-scans-atag a-3t-probabilistic-locus-coeruleus-atlas abc-(atlas-based-classification) abeat absorb-for-groupwise-registration adaptive-neuro-fuzzy-inference-system-granger-causality-(anfisgc) adjust---eeg-automatic-artifact-removal adni adolescent-brain-cognitive-development-(abcd)-study advanced-connectivity-analysis-(aca):-a-large-scale-functional-connectivity-data-mining-environment advanced-normalization-tools advanced-segmentation-tools-(asets) afni age-ility age-related-atrophy-dataset ahead a-joint-sparse-partial-correlation-method-for-estimating-group-functional-networks a-label-fusion-using-crf allen-brain-atlas:-data-portal altered-states-database alvin---lateral-ventricle-segmentation amilab analyze analyzer-2 anatomist anima anima-scripts annovar:-functional-annotation-of-genetic-variants-from-high-throughput-sequencing-data a-novel-group-fused-sparse-partial-correlation-method-for-simultaneous-estimation-of-functional-networks-in-group-comparisons a-novel-method-for-extracting-hierarchical-functional-subnetworks-based-on-a-multi-subject-spectral-clustering-approach anticorrelated-component-analysis antsr aperture a-probabilistic-subcortical-atlas-for-pd arctic a-robust-method-for-correcting-partial-volume-effect-in-asl-data arterial-territories-of-the-neonatal-brain:-automated-classification artifact-detection-tools-(art) artrepair-for-robust-fmri asa---advanced-source-analysis a-shimadzu-to-.nirs-data-converter ashs:-automatic-segmentation-of-hippocampal-subfields asl-data-processing-tool-box asl_spm8 atlas3d atlasconstructiontoolkit atlases-of-amygdala-and-hippocampus-for-pediatric-populations atlasing-of-the-basal-ganglia atpp:-automatic-tractography-based-parcellation-pipeline augmented-reality-mirror aura-tools-:-automated-retinal-analysis-tools autism-tissue-program---atp automated-image-registration-(air) automatic-extra-axial-cerebrospinal-fluid-(auto-eacsf) automatic-registration-toolbox automatic-segmentation-tool-adapter autoseg autotract axonal-beading-and-microtubule-quantification b0-and-eddy-current-correction-for-dti bash4rfmri bayesian-connectivity-analysis-software-(bca) bayesian-longitudinal-low-rank-regression bci2000 bcilab beast-segmentation-library besa best-practices-for-software-development beta-series-correlation bic-mni-models big-gaba bioera biofilmquant bioimage-suite biomedical-informatics-research-network biomesh3d biosig bisque bluepyopt books brainance-dti-module brainbay brainbox brainbrowser braincolor:-human-brain-anatomical-labeling-protocols-and-colormaps brain-connectivity-toolbox braincsi braindecodertoolbox brain-entropy-in-space-and-time-(best) brainfx brain-genomics-superstruct-project-(gsp)-open-access-data-release braingraph-editor-1.0-beta braingraph---graph-theory-analysis-of-brain-mri-data-in-r brainhack2014 brain-hierarchical-atlas:-a-brain-atlas-where-the-regions-of-interest-are-relevant-for-both-structure-and-function brainlife.io brainliner.jp brainmagix-spm-viewer brainmap-database brainmaps.org brainmask-volume-processing-tool brainnetome-atlas-viewer brainnetome-diffusionkit brainnetome-fmri-toolkit brainnet-viewer brainnetworkconstructionanalysisplatform brainparser brain-registration-evaluations-(2008-2011) brainscape brainsconstellationdetector brain-segmentation-testing-protocol brainsfit brainsight brainsimulator brainsmush brainsolution brainspell brainsroiauto brainssurfacestats brainstem-connectome-atlas brainstools brainstorm brainsuite brainvisa brainvisa-morphologist-extensions brainvoyager brainvoyager-brain-tutor brainvoyager-viewer brainwaver bratumia-(brain-tumor-image-analysis) brava bravissima:-human-brain-artery-nifti-atlas brede-toolbox brede-wiki brian-simulator broccoli bsmac:-bayesian-spatial-model-for-brain-activation-and-connectivity btk-:-baby-brain-toolkit bxh/xcede-tools c8:-corpus-callosum-computations cagwas calatk---atlas-building-tk calcfd---calculate-the-fractal-dimensionality-of-a-3d-structure camba camino camino-trackvis candi-neuroimaging-access-point candi-share:-schizophrenia-bulletin-2008 caret carlsim:-a-gpu-accelerated-snn-simulator cavernous-hemangioma:-epilepsy-&-add caworks cbfbirn cbica:-brain-anatomy-simulator cbica:-brain-tumor-modeling---coupled-solver cbica:-brain-tumor-viewer-(btv) cbica:-build-system-and-software-implementation-standard-(basis) cbica:-cancer-imaging-phenomics-toolkit-(captk) cbica:-chimera cbica:-compare cbica:-confetti cbica:-datasets cbica:-dtidroid cbica:-geodesic-graph-based-segmentation-with-shape-priors-(grasp) cbica:-glioma-image-segmentation-and-registration-(glistr) cbica:-gondola cbica:-identification-of-sparse-connectivity-patterns-in-rsfmri-(scplearn) cbica:-laboratory-for-individualized-breast-radiodensity-assessment-(libra) cbica:-moe cbica:-multi-atlas-region-segmentation-utilizing-ensembles-of-registration-algorithms-and-parameters-(muse) cbica:-multi-atlas-skull-stripping-(mass) cbica:-odvba cbica:-phi-estimator cbica:-portr cbica:-predict cbica:-tutorials-(image-processing-and-machine-learning)-using-cpp,-itk,-etc. cbica:-whitestripe cbica:-wmls-(white-matter-lesion-segmentation) cbinifti:-matlab/octave-nifti-library cbrain cbs-high-res-brain-processing-tools ccseg---corpus-callosum-segmentation cdf-hc-pointsetreg celeritas:-fiber-optic-response-system center-for-computational-biology-(ccb) center-for-functional-neuroimaging-tech cerebellar-white-matter-atlas cerebral-blood-flow-database-and-analysis-pipeline-(cbfdap) challenge-competitions-collection chiasm---combined-histology-mri-integrated-atlas-of-the-squirrel-monkey chinese-pediatric-atlas-(chn-pd-atlas) chronos:-a-multifunctional-response-and-stimulus-device cifti-connectivity-file-format cigal civility-:-cloub-based-interactive-visualization-of-tractography-brain-connectome cleanline cleave:-large-data-set-anova clinical-toolbox-for-spm cloud-reproducibility-challenge cluster-extent-correction cluster-reporter cmfreg c-mind-database cmind_py cmrep cnn-for-3d-brain-mri-classification cocomac-brain-connectivity-database cocomac-paxinos3d-viewer cognision-(tm)-system cognitive-paradigm-ontology coins colin-3t/7t-high-resolution-atlas combining-a-patch-based-approach-with-a-non-rigid-registration-based-label-fusion-method-for-the-hippocampal-segmentation computational-anatomy-toolbox---cat computational-morphometry-toolkit-(cmtk) conferences,-workshops-and-meetings connectir connectome-file-format-(cff) connectome-viewer connectome-workbench connectopy-toolkits conn-:-functional-connectivity-toolbox convert-mni-coordinates-to-or-from-xyz conwhat---a-library-for-connectome-based-white-matter-atlas-analysis corpus-callosum-and-brain-segmentations-from-abide-database corpus-callosum-thickness-profile-analysis-pipeline cortex cost c-pac cppi-toolbox-for-fmri create-dwi-atlas cuda-accelerated-spherical-model-m/eeg curry-7 cutnii---custom-angle-slicer-for-brain-atlas-volumes cwl-eeg-fmri cwl-eeg-fmri-data-set cwx-optical-imaging-system cytoseg daikon dalan:-disconnectivity-atlas-for-lesion-analysis-of-networks databrary data-format-tools datalad datapixx dbgap dbgap_cleaner dbsnp dcm2nii dcmtk---dicom-toolkit dcp：diffusion-connectome-pipeline deformable-template-model-for-brain-analysis demon-deep-multi-contrast-brain-extraction,-application-to-human-&-animal-imaging denoising-of-redundant-mr-series density-spectral-array-of-bis-vista-monitoring-system-in-a-functional-hemispherectomy densparcorr-:-dens-based-method-for-partial-correlation-estimation-in-large-scale-brain-networks dentaltools developmental-resting-state-fmri-and-structrual-sample-10-26-yo dfbidb diamond diatrack-particle-tracking-software dicccol-predictor-(v0.1) dicombrowser dicomconvert dicom_uploadgui diffusional-kurtosis-estimator diffusion-mri-@-dkfz-heidelberg diffusion-mri---in-vivo-and-phantom-data diffusion-mri-orientation-distribution-function-in-constant-solid-angle-(csa-odf)-and-hough-transform-tractography diffusion-mri-tool diffusion-toolkit-/-trackvis diffusion-tractography-with-kalman-filter diffusion-warp dinifti disease-state-prediction displacement-field-viewer distributome done:-detection-of-outlier-neurons dots-wm-tract-segmentation dparsf dramms-deformable-image-registration-toolbox draw3d-and-meshinator dsi-studio dt_bold_calc dti-atlas-builder dtibrainscope dti-denoising dti-fiber-tract-statistics dtiprocess-toolkit dti-reg dti-template-rhesus-macaques dti-tk dual-tree-complex-wavelet-combined-with-non-local-means-for-asl-fmri-denoising dwi/dti-quality-control-tool:-dtiprep dwmri-phantom-drift dw-mri-random-walk-simulator dw-mri-registration-in-fsl dynamicbc e39-macaque-histology-and-mri-data econnectome eegbase eeg-fnirs-hybrid-smr-bci-data eeg-human-categorization-data eeglab eeg-study-data-of-rapid-serial-visual-presentation-(rsvp)-in-ess-format eeg-study-schema-(ess) eeg-toolbox eegvis eeprobe efficient-longitudinal-upload-of-depression-in-the-elderly-(elude) efficient-permutation-testing elastic-net-based-parcellation-(enpa) elastix electrophysiology-analysis-toolkit-(elephant) emory-unc-developmental-macaque-structural-and-dti-atlases emse-suite enigma-dti-pipeline enigma:-enhancing-neuro-imaging-genetics-through-meta-analysis enigma-resting-state-fmri-processing-pipeline enigma-viewer:-visualizing-effect-sizes-in-meta-analysis enumerated-auditory-response-(ear) eons epi-example-dicom epilab epilepcy-neuroimaging-data e-prime e-prime-extensions-for-fmri e-prime-extensions-for-smi e-prime-extensions-for-tobii epsilon-radial-networks erplab erp-pca-toolkit erpwavelab estimating-functional-brain-network-based-on-modularity-prior european-epilepsy-database-epilepsiae evaluation-of-automated-brain-extraction-methods-on-multi-site-multi-disorder-data ew_dmgwas:-edge-weighted-dense-module-search-for-genome-wide-association-studies-and-gene-expression-profiles exposition-packages eyeballgui eye-eeg-(combined-eye-tracking-&-eeg) faceted-search-based-ontology-visualizer factory_t1_dti fadttster false-discovery-rate-weighted fandtasia-java-applet:-dt-mri-processing fasst-–-fmri-artefact-rejection-and-sleep-scoring-toolbox fast-fourier-transform-(fft)-javalibrary fast-lesion-extraction-using-convolutional-neural-networks fast-nonlocal-means-for-mri-denoising fast-t2-relaxation-data-analysis-with-stimulated-echo-correction-and-non-local-spatial-regularisation fbirn-image-processing-scripts-(fips) fiber-tracking-based-on-finsler-distance fiber-tracking-tool fiberviewerlight fieldtrip filamenttracer fine-relational-spatial-topology finsler-tractography-module-for-slicer fitbir:-federal-interagency-traumatic-brain-injury-research fitting-oscillations-&-one-over-f fiv fmri-bold-signal-simulator fmri-classification-in-r fmri-cpca fmri-grocer fmripower fmriprep fmri-results-comparison-library fmristat fnir-devices fnirs-data-analysis-environment fnirs-during-stroop-before-and-after-acute-bout-of-aerobic-exercise fnirs-research-database fnirs-toolbox forward:-accurate-finite-element-electromagnetic-head-models framewise-integrated-real-time-mri-monitoring framework-for-open-programmatic-access freesurfer freesurfer-fsaverage freesurfr fsl fsl-extensions functional-analysis-of-diffusion-tensor functional-by-strutural-hierarchical-mapping-(fsh-mapping) functional-connectivity-analysis-tool-for-near-infrared-spectroscopy-data functional-connectivity-community functional-imaging-birn functional-mixed-processes-models functional-organization-of-the-mouse-lemur-primate-microcebus-murinus-:-from-multilevel-validation-to-comparison-with-humans functional-real-time-interactive-endogenous-neuromodulation-and---decoding-(friend) functional-regression-analysis-of-dti functional-roi-atlas fusion-ica-toolbox-(fit) fvgwas:-fast-voxelwise-genome-wide-association-analysis g-3-/-neurospaces-simulator gaain-toolset gambit gamma gannet---batch-analysis-of-edited-magnetic-resonance-spectroscopy-(mrs)-data gat-:-graph-analysis-toolbox gazereader g.bsanalyze gcca generalized-covariance-analysis-(gcva) generalized-ppi-toolbox generation-r-pediatric-mri-resources genesis-neural-simulator gengen gestr giant:-genome-scale-integrated-analysis-of-gene-networks-in-tissues gibbsconnectome gifti gimias gimme-(group-iterative-multiple-model-estimation) glirt glmdenoise:-a-fast,-automated-technique-for-denoising-task-based-fmri-data gmac:-a-matlab-toolbox-for-spectral-granger-causality-analysis-of-fmri-data gpu-based-affine-registration gradient-non-linearity-unwarping-tool grantees-meeting-for-nitrc graph-theory-glm-(gtg)-matlab-toolbox graphtools graphvar:-a-user-friendly-toolbox-for-comprehensive-graph-analyses-of-functional-brain-connectivity gretna group-ica-toolbox-(gift-and-eegift) group-information-guided-ica group-level-imputation-of-statistic-maps group-sparse-canonical-correlation-analysis groupwise-image-registration gsa-snp gtract gwas3d gwasdb gwasrap hadoop-for-data-colocation haiko89:-mri-in-vivo-baboon-brain-templates-collection-(papio-anubis) hammer-and-wml-modules-for-3d-slicer hammer:-deformable-registration hammer-suite hbm-hackathon hcp-harvard/mgh-ucla hcp-wu-minn-consortium hdbig hd-neuro-informatics hermes hfh_dti_data hfh_dti_tle_tools hierarchical-event-descriptor-(hed)-tags hierarchical-functional-networks-in-resting-state-fmri high-dynamic-range-for-local-contrast-enhancement-of-mri high-quality-diffusion-weighted-imaging-of-parkinson's-disease hint:-hierarchical-independent-component-analysis-toolbox hippocampalunfolding_manualinitialization hippocampome.org hippocampus-subregional-masks-based-on-gray-matter-volume-covariance hi-speed-software-packets histolozee---3d-histology-reconstruction-and-co-registration-with-mri hitachi2nirs hitachi-optical-topography-system homer2 hsnc human-connectome-project-(hcp) human-imaging-database-(hid) hyperion:-mri-digital-projection-system iatr ibeat ibsr ica-components-from-kessler,-et-al,-jama-psychiatry-2016 icn_atlas idea-lab-brain-image-processing-suite ielectrodes iit-human-brain-atlas-(v.4.1) imagecite imagej image-segmentation-based-on-the-local-center-of-mass-computation image-synthesis imagevis3d imcalc:-batch-image-calulations-and-transformations-(spm5/8/12) imeka-tractography-service imod---3d-reconstruction-and-analysis incf incf-neuroimaging-data-sharing individual-morphological-connectivity-using-wavelet-transform infant-macaque-mri-and-dti-templates inia19-primate--brain-atlas insight-toolkit instar-(infant-joint-segmentation-and-registration) inter-group-registration-toolbox international-imaging-genetics-conference internet-brain-volume-database-(ibvd) intervar intra--and-inter-scanner-reliability-of-rs-fmri-bold-and-asl-with-eyes-closed-vs.-eyes-open intrinsic-unscented-kalman-filter-(iukf)-tractography-software-v1.0 in-vivo-memri-based-mouse-brain-atlas in-vivo-memri-based-rat-brain-atlas invizian isc-toolbox:-inter-subject-correlation-analysis-for-fmri-in-matlab iso2mesh iterative-dual-regression-with-sparse-prior itk-snap itools-resourceome iview-x-mri-lr---eye-tracking-for-fmri ixi-dataset jhu-proj.-in-applied-medical-imaging jim jip-fmri-analysis-toolkit jist:-java-image-science-toolkit jist-resources-for-algorithm-development joint-anisotropic-lmmse-filter-for-stationary-rician-noise-removal-in-dwi journals k-space-domain-image-fusion kurtosis-imaging-network---kin kwscene:-mrml-based-atlas-and-scene-builder/reader/writer kymata-atlas laboratory-of-neuro-imaging-(loni) landman-neuroimaging-tools laplace-beltrami-filter-on-quadedge-meshes lddmm lead-connectome lead-dbs learning-based-image-segmentation-toolbox level-set-segmentation-for-slicer3 lfpy libeep libminc---medical-image-netcdf-library libsbml licensing lightweight-data-pipeline limo-eeg linear-fascicle-evaluation lipsia l-measure l-neuron local-binary-pattern-analysis-tools-for-mr-brain-images local-label-learning-for-image-segmentation local-label-learning-(lll)-segmentation local-manifold-learning-for-multi-atlas-segmentation longitudinal-multiple-sclerosis-lesion-imaging-archive longitudinal-neuroimaging-hippocampal-markers-for-diagnosing-alzheimer's-disease longitudinal-rhesus-monkey-brain-templates loni-debabeler loni-de-identification-debablet loni-inspector loni-pipeline-environment loni-provenance-editor lop-dwi loris low-resolution-brain-electromagnetic-tomography lumina-lp--400-response-system lupus-patients-with-white-matter-lesions m3 mabmis-for-slicer-4:-multi-atlas-based-multi-image-segmentation mabmis:-multi-atlas-based-segmentation mach mach2dat mach2qtl mach-admix mag-design-and-engineering magnetic-resonance-angiography-atlas-dataset magpro-magnetic-stimulator-(tms) mango manually-labeled-mri-brain-scan-database manualwmh-imagej-macros map3d mapping-population-based-structural-connectomes maps4mipav-(exploratory-jist) marsbar-region-of-interest-toolbox mars-(multi-atlas-robust-segmentation) masi_hardi_reproducibility masi-label-fusion masimatlab masked-ica-(mica)-toolbox mask_explorer massively-expedited-genome-wide-heritability-analysis-(megha) matlab-tutorial-on-diffusion-tensor-mri mayo-clinic-adult-lifespan-template-and-atlases mazesuite mcic--schizophrenic-and-matched-control-data mcml-&-conv mdr-multifactor-dimensionality-reduction measure-projection-toolbox-(mpt) measuring-eye-states-in-functional-mri medical-image-processing-and-visualization-in-virtual-environments medinria megsim meg-tools menga:-multimodal-environment-for-neuroimaging-and-genomic-analysis mesh-based-monte-carlo-(mmc) meshgen---surface-meshes-from-volumetric-segmentation-data meshvalmet:-validation-metric-for-meshes meshview-for-brain-atlases metasearch meth mga---multimodal-glioma-analysis mgdm:-multi-geometric-deformable-model mgh-neonatal/pediatric-adc-atlases mias-registration-toolkit microdraw micro-manager microstructural-correlation-toolbox mid-space-independent-deformable-image-registration minc-example-files minc-toolkit minc-toolkit-testsuite minc-toolkit-v2 mindboggle-101-manually-labeled-brains mindboggle:-open-source-software-for-analyzing-the-shapes-of-brain-structures-from-human-mri-data mindseer mipav mircen-macaca-fascicularis-brain-mri-segmentation-dataset miriad-(minimal-interval-resonance-imaging-in-alzheimer's-disease) misst---microstructure-imaging-sequence-simulation-toolbox misteri mitk-diffusion miview mixed-effect-model-of-genetic-set-and-environment-interaction mmps-docker mne---minimum-norm-current-estimates mni-macaque-atlases moca modelgui monstr-multi-contrast-brain-stripping monte-carlo-extreme-(mcx) monte-carlo-simulation-software:-tmcimg monthly-morphometry-report morphometry-birn motrak:-head-motion-tracking-system mouse-birn mousebirn-atlasing-toolkit-(mbat) mouse-connectome-project-(mcp) mousetracker movement_info mpi-leipzig-mind-brain-body-dataset mpscope mrbrains-evaluation-framework-/-challenge mr-connectome-automated-pipeline-(mrcap) mri-brain-template-and-atlas-of-the-mouse-lemur-primate-microcebus-murinus mricloud mricrogl mricron mricros mri_cvpr mri-dataset-for-hippocampus-segmentation mri-defacer mri-denoising mri-neuroanatomy-labeling-services mri-predict mri-simulator mri-studio mriwatcher mrtool mrtrix multi-atlas-skull-stripping-(mass) multicomponent-t2-estimation-with-stimulated-echo-correction multi-contrast-and-submillimetric-3-tesla-hippocampal-subfield-segmentation-protocol-and-dataset-(mni-hisub25) multi-echo-independent-component-regression-(me-icr)-group-level-connectivity-(groupincorr)-dataset multi-fiber-reconstruction-from-dw-mri multi-modality-visualization-tool multi-modal-mri-reproducibility-resource multiple-correlation-function-tool multiscale-weighted-principal-component-regression multitracer-version-2 multivariate-general-linear-models-(mglm)-on-riemannian-manifolds multixplore:-visual-exploration-platform-for-multimodal-neuroimaging-data n2a n3---minc-b0-nonuniformity-correction ncaa-dod-grand-alliance-care-consortium ncanda:-data-integration-component nc-igt-fast-imaging-library nemo:-neuron-morphological-analysis-tool neo neoseg neosegpipeline---multisegpipeline netpet:-covariance-statistics-and-network-analysis-of-brain-pet-imaging-studies net-station network-based-statistic-(nbs) network-modification-(nemo)-tool-lite neural-electromagnetic-ontologies neuritetracer neuroconstruct neurodebian neurodevelopmental-mri-database neurodocker neuroelf neuroimage-field-of-view-normalization-tool neuro-image-management-system-(nims) neuroimaging-analysis-kit-(niak) neuroimaging-data-access-group neuroimaging-made-easy-blog neuroinformatics---the-journal neurolens neurolucida neuroml neuromorpho.org neuron neuron-c neuronj neuropub-visualizer neuroscience-gateway-(nsg) neuroscope neurosharelibrary neurostars neurosynth neurovault neuroviisas neuroweb---neuroimaging-database nft:-neuroelectromag-forward-modeling niag-addiction-data niagads---nia-genetics-of-alzheimer's-disease-data-storage-site nice-sign nidb---neuroinformatics-database nifti niftyrec niftyreg nifty-sim nighres nih-cidi-lung-segmentation-tool nih-cidi-segmentation-of-pet-images-based-on-affinity-propagation-clustering nih-marmoset-brain-atlas nih-pediatric-mri-data-repository niistat nilearn nimh-data-archive-/-national-database-for-autism-research nipy-community nipy-diffusion-imaging-analysis nipy-file-io nipy-pipeline-and-interfaces nipy-structural-and-functional-analysis niral-utilities nirep nirfast nirs-spm nirx2nirs:-a-nirx-to-.nirs-data-converter nirx-nirs-neuroimaging nitime nitrc-computational-environment-(nitrc-ce) nitrc-gforge-extensions nitrc-image-repository-(nitrc-ir) noddi-matlab-toolbox nonfractal non-redundant-connectivity-(nrc)-maps non-rigid-groupwise-registration-method nordicneurolab normative-independent-component-analysis nornir northwestern-university-schizophrenia-data-and-software-tool-(nusdast) notion-researchpacs npbayes-fmri nptk nrm2018-pet-grand-challenge-dataset ntu-dsi-122:-a-diffusion-spectrum-imaging-template-with-high-anatomical-matching-to-the-icbm-152-space numerical-fibre-generator-(nfg) nutil---neuroimaging-utilities nutmeg nvm nyu-csc-testretest oasis odin oei-fmri-compatible-olfactometer offline-processing-mri ogles---opengl/oiv-stereotactic-tool online-brain-atlas-reconciliation-tool online-eeg-artifact-removal-by-adaptive-spatial-filtering onprc-infant-macaque-brain-atlas open-connectome-project openelectrophy openfmri openmeeg open-neuroimaging-laboratory open-science-framework open-vanderbilt-archive-of-the-temporal-lobe openvibe openwalnut optil.io optseq---fmri-event-scheduler organization-for-human-brain-mapping orion-i ors-visual-si p-[(123)i]iodo-l-phenylalanine-(ipa)-spect-template-in-mni-coordinates palm---permutation-analysis-of-linear-models panda:-a-pipeline-tool-for-diffusion-mri papaya paradigm parallel-graph-theoretical-analysis-toolkit parallel-stochastic-ion-channel-simulator paraview parkinson's-disease-discovery-database pattern-recognition-based-on-joint-gam-sparsity-regression-model pbrain pdbp:-parkinson's-disease-biomarkers-program pd-rs-fmri-meta-and-validation peak_nii penn-chop-age-specific-neonate-atlas penncnv penn-hippocampus-atlas personode pestica-&-slomoco:-physio-and-motion-correction-tools phenolyer philadelphia-neurodevelopmental-cohort-(pnc) philips-users-community phoneme-syllable-tsinghua-university-cs&t-2011 phycaa+:-adaptive-physiological-noise-correction-for-bold-fmri physio-toolbox picasso-skull-stripping-tool picsl-multi-atlas-segmentation-tool pipeline-neuroimaging-virtualenvironment pipeline-system-for-octave-and-matlab plink pliviewer polgui--matlab-gui-for-polhemus-fastrak population-averaged-diffusion-tensor-imaging-atlas-of-the-sprague-dawley-rat-brain pos2spm ppmi predict predicting-alzheimer’s-conversion-in-mild-cognitive-impairment-patients-using-longitudinal-neuroimaging-and-clinical-markers prediction-and-diagnosis-for-depression-and-schizophrenia preprocessed-connectomes-project presentation primatologist principal-components-analysis-of-scalar,-vector,-and-mesh-vertex-data probabilisticbiascorrection probability-associated-community-estimation proctseg profound-hearing-loss-database program-for-optimal-design-of-blocked-fmri-experiments progression-score-model-toolkit pronto-–-pattern-recognition-for-neuroimaging-toolbox propixx-dlp-led-projector prospective-acquisition-correction-(pace)-resting-state-epi-data psychopy pubbrain-database pvs_enhance pydicom pymvpa pynifti pynn pysurfer pythagorean-displacement-and-motion-regressors pyxnat qcqp qmodeling quantification-of-structural-brain-connectivity-via-a-conductance-model quantitative-diffusion-tools quicknii---serial-section-aligner-to-volumetric-atlases random-forest-for-prediction-of-ci_wrs randomized-denoising-autoencoders-for-neuroimaging rapidart rapid-permutation-testing-toolbox rapidtide rda2nifti recorder recview registry-builder-data-harmonization-and-aggregation-tool repronim:-a-center-for-reproducible-neuroimaging-computation repronim-simple-reexecutable-publication resource-ontology-discussion-group rest:-a-toolkit-for-resting-state-fmri resting-state-fmri-from-healthy-young-subjects resting-state-fmri-of-patients-with-pre-eclampsia-and-healthy-pregnant-women resting-state-hemodynamic-response-function-retrieval-and-deconvolution-(rs-hrf) resting-state-meg-data-used-in-marek-et-al.,-2018,-plos-biology resting-state-pediatric-imaging-template rex rft_fdr rncut:-functional-brain-networks-parcellation robust-biological-parametric-mapping robust-brain-extraction-(robex) rockland-download-link-script rodent-brain-extraction-tool rodent-cortical-thickness-analysis rosetta-bit r-package-for-adaptive-dwi-analysis r-package-for-adaptive-fmri-analysis rt_image ruby-nifti s3dl-sparse-dictionary-learning-based-mr-image-and-lesion-segmentation scalable-brain-atlas schizconnect:-large-scale-schizophrenia-neuroimaging-data-mediation-&-federation scikit-learn scirun score scp_taskfmri scralyze scribe seecat seed-based-d-mapping-(sdm,-formerly-signed-differential-mapping) seg3d segediting segmentation-based-multimodal-rigid-image-registration segmentation-validation-engine self-threat-and-self-concept-data serotonin-transporter-highresolution-pet-template sfmproject shapeanalysismancova---spharm-tools shapecomplexatlas shapecomplexityindex shapepopulationviewer shapetools shapeviewer shapeworks signalml sigviewer simple-neurite-tracer simple-medical-imaging-library-interface-(smili) simulated-diffusion-weighted-datasets simulated-dw-mri-brain-data-sets-for-quantitative-evaluation-of-estimated-fiber-orientations single-index-varying-coefficient-model-for-functional-responses single-subject-resting-state-fmri-reproducibility-resource sinomo-(singular-node-motifs) sinoplan sivic skullstrippingtoolkit slep:-sparse-learning-with-efficient-projections slic:-a-whole-brain-parcellation-toolbox slice:drop slicer3-example-modules slicer3-module-rician-noise-filter slicersalt smac:-spatial-multi-category-angle-based-classifier small-effect-size-leads-to-reproducibility-failure-in-resting-state-fmri-studies snpm sock solar-eclipse-imaging-genetics-tools source-information-flow-toolbox spanish-resting-state-network sparse-sampling-of-silence-database spatial-analysis-3d spatially-constrained-parcellation spatial-statistical-parametric-mapping spd-rsfmri-neu-zs spharm-mat spharm-pdm-toolbox spicodyn spikecor:-fmri-tool-for-automated-correction-of-head-motion-spikes spinal-cord-toolbox spinesegmentation-module-for-3dslicer spm spm-for-fnirs-toolbox spm_ss---fmri-functional-localizers spm-workshop-hannover spot3d:-spatial-positioning-toolbox-for-head-markers-using-3d-scans s-rep-fitting,-statistics,-and-segmentation sri24-atlas:-normal-adult-brain-anatomy sr-research-eyelink-eye-trackers stamp-atlases-for-brain-abnormality-detection standard-amygdalar-fmri-probe-tasks staple stark-cross-sectional-aging startyourrecovery.org statistical-methods-for-heterogeneous-neuroimaging-data statistical-methods-for-manifold-valued-data statistics-online-computational-resource stdpc stereo-investigator stereoscopic-atlas-of-intrinsic-brain-networks-(saibn) stochastic-tractography-system structural-asymmetries studyforrest subject-library subject-order-independent-group-ica sumsdb-database surface-based-processing-and-analysis-of-mri surf-ice surfstat svv synchronized-histological-image-view-arc talairach-daemon tapas---translational-algorithms-for-psychiatry-advancing-science tapir tarquin-mrs-analysis-package task-based-fmri-atlases task-independent-fluctuations-discussion template-image-processing-library templates-for-in-vivo-mouse-brain templates-for-long-evans-rat-brain temporal-lobe.com temporal-lobe-epilpsy-multiple-modality-data tetrametrix textpresso---literature-search-engine the-brain-catalogue the-brain-coactivation-map the-cancer-imaging-archive-(tcia) the-cognitive-atlas the-de-identification-toolbox---a-data-sharing-tool-for-neuroimaging-studies the-ebds-infant-dti-fiber-atlases the-fmri-data-center the-grid-code-analysis-toolbox-(gridcat) the-medical-image-computing-and-computer-assisted-intervention-society the-multiscale-object-oriented-simulation-environment the-neuro-bureau the-neurophysiological-biomarker-toolbox the-role-of-v6-in-the-interaction-between-dorsal-and-ventral-streams-in-near-and-far-space-processing the-striatal-subregional-voimap the-virtual-brain-(tvb) time-encoding-and-decoding-toolbox tissue-thickness-estimation-via-minimum-line-integrals tns-data-repository tns-lymphatics-dataset toads-cruise-brain-segmentation-tools toolconnect topographica topographic-tract-filtering tortoise tractography-based-on-riemannian-geodesics-and-tensor-adjugates tractor:-tractography-with-r trafic---tracts-fibers-classification-using-deep-learning trees-toolbox trentool triangle-biosystems tumorsim turbo-brainvoyager turbofire turtleseg ucla-multimodal-connectivity-database uk-biobank---multimodal-imaging ultra-high-field-atlas-for-dbs-planning umanitoba-jhu-functionally-defined-human-white-matter-atlas umeå-brain-arteries unbiased-atlas-construction-via-population-(group-wise)-dramms-registration unbiased,-deformable-spatiotemporal-atlas-of-the-fetal-brain unc-0-1-2-infant-atlases unc-cedars-infant-atlas unc-detail-preserved-longitudinal-0-3-6-9-12-months-old-atlas unc-human-brain-atlas unc-neonate-and-infant-cortical-surface-atlases unc-primate-brain-atlas unc/utah-namic-dti-fiber-analysis-framework unc-wisconsin-neurodevelopment-rhesus-mri-database unfold-toolbox-for-regression-based-eeg-analysis university-of-illinois-cognitive-neuroimaging-laboratory-optical-brain-imaging-software user-friendly-functional-connectivity---uf²c using-make-for-neuroimaging-workflow:-manual-and-examples vaa3d vaa3d-and-vaa3d-neuron validate29-squirrel-monkey-brain-atlas valmap:-simple-statistical-mapping-tool vamca-cortical-meta-analysis-toolbox vandpire vartbx-–-variability-toolbox vectorvaluedhistogramnormalizer vervet-probabilistic-atlas viewpixx viewpixx-/3d viking-family-of-viewing/annotation/analysis-tools-for-connectomics vimeapy virtual-brain-generator virtual-digital-brain virtual-electrode-recording-tool-for-extracellular-potentials-(vertex) virtual-reality-brain-network-viewer visapy vismotionqcdb vist/e visualization-toolkit visually-evoked-potential-eeg vis:-visualized-statistics-toolbox-(spm5/8/12) vmtk-in-3d-slicer volbrain:-online-mri-segmentation voxbo vutools wannovar wavelet-based-image-fusion waxholm-space-atlas-of-the-c57bl/6j-mouse-brain waxholm-space-atlas-of-the-sprague-dawley-rat-brain web-game-for-collaborative-labeling web-interfaces-for-multiscale-images wfu_bpm wfu_mmnet---multivariate-modeling-of-brain-networks-toolbox wfu_pickatlas wfu_pipeline whitetext---annotated-neuroscience-text wisconsin-cortical-thickness-analysis-(cta)-toolbox wisconsin-rhesus-brain-parcellation wisconsin-white-matter-hyperintensities-segmentation-toolbox working-memory-trainer xcede-schema xfsl:-an-fsl-toolbox xjview,-a-viewing-tool-for-spm xnat xnat-extras xnbc ymdti:-diffusion-tensor-images-of-healthy-young-males zebrafish-heart-segmentation">Standalone</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="viola visualization-toolkit">Web</input></li>

                                                        </ul>
                                                    </details>
                                                </li>


                                                <li class="cat-item cat-item-3">
                                                    <details>
                                                        <summary>
                                                            <label>Display Device</label>
                                                        </summary>
                                                        <ul class="children">

                                                            <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="1000-functional-connectomes-project [123i]-fp-cit-spect-brain-template-in-mni-space 18f-dopa-pet-and-123i-fp-cit-(ioflupane---datscan)-spect-templates-for-spm-normalisation. 2008-miccai-ms-lesion-segmentation-challenge 3d-brain-atlas-reconstructor 3-d-connective-field-modeling-toolbox 3-dimensional-diffusion-tensor-imaging-(dti)-atlas-of-the-rat-brain-in-postnatal-day-5,-14,-and-adulthood 3dmeshmetric 3d-slicer 3dslicerlupuslesionmodule 3dsvm 3t-dwi-test-retest-reliability-dataset 3t-mri-data-set-on-structural-brain-behavior-correlations 4d-atlas-construction 4d-parser-(pathological-anatomy-regression-via-segmentation-and-registration) 4d-(spatio-temporal)-dramms-registration 7t-probabilistic-atlas-of-subthalamic-nucleus-for-use-with-3t-mri 7t-structural-mri-scans-atag a-3t-probabilistic-locus-coeruleus-atlas abc-(atlas-based-classification) abeat absorb-for-groupwise-registration adaptive-neuro-fuzzy-inference-system-granger-causality-(anfisgc) adjust---eeg-automatic-artifact-removal adni adolescent-brain-cognitive-development-(abcd)-study advanced-connectivity-analysis-(aca):-a-large-scale-functional-connectivity-data-mining-environment advanced-normalization-tools advanced-segmentation-tools-(asets) afni age-ility age-related-atrophy-dataset ahead a-joint-sparse-partial-correlation-method-for-estimating-group-functional-networks a-label-fusion-using-crf allen-brain-atlas:-data-portal altered-states-database alvin---lateral-ventricle-segmentation amilab analyze analyzer-2 anatomist anima anima-scripts annovar:-functional-annotation-of-genetic-variants-from-high-throughput-sequencing-data a-novel-group-fused-sparse-partial-correlation-method-for-simultaneous-estimation-of-functional-networks-in-group-comparisons a-novel-method-for-extracting-hierarchical-functional-subnetworks-based-on-a-multi-subject-spectral-clustering-approach anticorrelated-component-analysis antsr aperture a-probabilistic-subcortical-atlas-for-pd arctic a-robust-method-for-correcting-partial-volume-effect-in-asl-data arterial-territories-of-the-neonatal-brain:-automated-classification artifact-detection-tools-(art) artrepair-for-robust-fmri asa---advanced-source-analysis a-shimadzu-to-.nirs-data-converter ashs:-automatic-segmentation-of-hippocampal-subfields asl-data-processing-tool-box asl_spm8 atlas3d atlasconstructiontoolkit atlases-of-amygdala-and-hippocampus-for-pediatric-populations atlasing-of-the-basal-ganglia atpp:-automatic-tractography-based-parcellation-pipeline augmented-reality-mirror aura-tools-:-automated-retinal-analysis-tools autism-tissue-program---atp automated-image-registration-(air) automatic-extra-axial-cerebrospinal-fluid-(auto-eacsf) automatic-registration-toolbox automatic-segmentation-tool-adapter autoseg autotract axonal-beading-and-microtubule-quantification b0-and-eddy-current-correction-for-dti bash4rfmri bayesian-connectivity-analysis-software-(bca) bayesian-longitudinal-low-rank-regression bci2000 bcilab beast-segmentation-library besa best-practices-for-software-development beta-series-correlation bic-mni-models big-gaba bioera biofilmquant bioimage-suite biomedical-informatics-research-network biomesh3d biosig bisque bluepyopt books brainance-dti-module brainbay brainbox brainbrowser braincolor:-human-brain-anatomical-labeling-protocols-and-colormaps brain-connectivity-toolbox braincsi braindecodertoolbox brain-entropy-in-space-and-time-(best) brainfx brain-genomics-superstruct-project-(gsp)-open-access-data-release braingraph-editor-1.0-beta braingraph---graph-theory-analysis-of-brain-mri-data-in-r brainhack2014 brain-hierarchical-atlas:-a-brain-atlas-where-the-regions-of-interest-are-relevant-for-both-structure-and-function brainlife.io brainliner.jp brainmagix-spm-viewer brainmap-database brainmaps.org brainmask-volume-processing-tool brainnetome-atlas-viewer brainnetome-diffusionkit brainnetome-fmri-toolkit brainnet-viewer brainnetworkconstructionanalysisplatform brainparser brain-registration-evaluations-(2008-2011) brainscape brainsconstellationdetector brain-segmentation-testing-protocol brainsfit brainsight brainsimulator brainsmush brainsolution brainspell brainsroiauto brainssurfacestats brainstem-connectome-atlas brainstools brainstorm brainsuite brainvisa brainvisa-morphologist-extensions brainvoyager brainvoyager-brain-tutor brainvoyager-viewer brainwaver bratumia-(brain-tumor-image-analysis) brava bravissima:-human-brain-artery-nifti-atlas brede-toolbox brede-wiki brian-simulator broccoli bsmac:-bayesian-spatial-model-for-brain-activation-and-connectivity btk-:-baby-brain-toolkit bxh/xcede-tools c8:-corpus-callosum-computations cagwas calatk---atlas-building-tk calcfd---calculate-the-fractal-dimensionality-of-a-3d-structure camba camino camino-trackvis candi-neuroimaging-access-point candi-share:-schizophrenia-bulletin-2008 caret carlsim:-a-gpu-accelerated-snn-simulator cavernous-hemangioma:-epilepsy-&-add caworks cbfbirn cbica:-brain-anatomy-simulator cbica:-brain-tumor-modeling---coupled-solver cbica:-brain-tumor-viewer-(btv) cbica:-build-system-and-software-implementation-standard-(basis) cbica:-cancer-imaging-phenomics-toolkit-(captk) cbica:-chimera cbica:-compare cbica:-confetti cbica:-datasets cbica:-dtidroid cbica:-geodesic-graph-based-segmentation-with-shape-priors-(grasp) cbica:-glioma-image-segmentation-and-registration-(glistr) cbica:-gondola cbica:-identification-of-sparse-connectivity-patterns-in-rsfmri-(scplearn) cbica:-laboratory-for-individualized-breast-radiodensity-assessment-(libra) cbica:-moe cbica:-multi-atlas-region-segmentation-utilizing-ensembles-of-registration-algorithms-and-parameters-(muse) cbica:-multi-atlas-skull-stripping-(mass) cbica:-odvba cbica:-phi-estimator cbica:-portr cbica:-predict cbica:-tutorials-(image-processing-and-machine-learning)-using-cpp,-itk,-etc. cbica:-whitestripe cbica:-wmls-(white-matter-lesion-segmentation) cbinifti:-matlab/octave-nifti-library cbrain cbs-high-res-brain-processing-tools ccseg---corpus-callosum-segmentation cdf-hc-pointsetreg celeritas:-fiber-optic-response-system center-for-computational-biology-(ccb) center-for-functional-neuroimaging-tech cerebellar-white-matter-atlas cerebral-blood-flow-database-and-analysis-pipeline-(cbfdap) challenge-competitions-collection chiasm---combined-histology-mri-integrated-atlas-of-the-squirrel-monkey chinese-pediatric-atlas-(chn-pd-atlas) chronos:-a-multifunctional-response-and-stimulus-device cifti-connectivity-file-format cigal civility-:-cloub-based-interactive-visualization-of-tractography-brain-connectome cleanline cleave:-large-data-set-anova clinical-toolbox-for-spm cloud-reproducibility-challenge cluster-extent-correction cluster-reporter cmfreg c-mind-database cmind_py cmrep cnn-for-3d-brain-mri-classification cocomac-brain-connectivity-database cocomac-paxinos3d-viewer cognision-(tm)-system cognitive-paradigm-ontology coins colin-3t/7t-high-resolution-atlas combining-a-patch-based-approach-with-a-non-rigid-registration-based-label-fusion-method-for-the-hippocampal-segmentation computational-anatomy-toolbox---cat computational-morphometry-toolkit-(cmtk) conferences,-workshops-and-meetings connectir connectome-file-format-(cff) connectome-viewer connectome-workbench connectopy-toolkits conn-:-functional-connectivity-toolbox convert-mni-coordinates-to-or-from-xyz conwhat---a-library-for-connectome-based-white-matter-atlas-analysis corpus-callosum-and-brain-segmentations-from-abide-database corpus-callosum-thickness-profile-analysis-pipeline cortex cost c-pac cppi-toolbox-for-fmri create-dwi-atlas cuda-accelerated-spherical-model-m/eeg curry-7 cutnii---custom-angle-slicer-for-brain-atlas-volumes cwl-eeg-fmri cwl-eeg-fmri-data-set cwx-optical-imaging-system cytoseg daikon dalan:-disconnectivity-atlas-for-lesion-analysis-of-networks databrary data-format-tools datalad datapixx dbgap dbgap_cleaner dbsnp dcm2nii dcmtk---dicom-toolkit dcp：diffusion-connectome-pipeline deformable-template-model-for-brain-analysis demon-deep-multi-contrast-brain-extraction,-application-to-human-&-animal-imaging denoising-of-redundant-mr-series density-spectral-array-of-bis-vista-monitoring-system-in-a-functional-hemispherectomy densparcorr-:-dens-based-method-for-partial-correlation-estimation-in-large-scale-brain-networks dentaltools developmental-resting-state-fmri-and-structrual-sample-10-26-yo dfbidb diamond diatrack-particle-tracking-software dicccol-predictor-(v0.1) dicombrowser dicomconvert dicom_uploadgui diffusional-kurtosis-estimator diffusion-mri-@-dkfz-heidelberg diffusion-mri---in-vivo-and-phantom-data diffusion-mri-orientation-distribution-function-in-constant-solid-angle-(csa-odf)-and-hough-transform-tractography diffusion-mri-tool diffusion-toolkit-/-trackvis diffusion-tractography-with-kalman-filter diffusion-warp dinifti disease-state-prediction displacement-field-viewer distributome done:-detection-of-outlier-neurons dots-wm-tract-segmentation dparsf dramms-deformable-image-registration-toolbox draw3d-and-meshinator dsi-studio dt_bold_calc dti-atlas-builder dtibrainscope dti-denoising dti-fiber-tract-statistics dtiprocess-toolkit dti-reg dti-template-rhesus-macaques dti-tk dual-tree-complex-wavelet-combined-with-non-local-means-for-asl-fmri-denoising dwi/dti-quality-control-tool:-dtiprep dwmri-phantom-drift dw-mri-random-walk-simulator dw-mri-registration-in-fsl dynamicbc e39-macaque-histology-and-mri-data econnectome eegbase eeg-fnirs-hybrid-smr-bci-data eeg-human-categorization-data eeglab eeg-study-data-of-rapid-serial-visual-presentation-(rsvp)-in-ess-format eeg-study-schema-(ess) eeg-toolbox eegvis eeprobe efficient-longitudinal-upload-of-depression-in-the-elderly-(elude) efficient-permutation-testing elastic-net-based-parcellation-(enpa) elastix electrophysiology-analysis-toolkit-(elephant) emory-unc-developmental-macaque-structural-and-dti-atlases emse-suite enigma-dti-pipeline enigma:-enhancing-neuro-imaging-genetics-through-meta-analysis enigma-resting-state-fmri-processing-pipeline enigma-viewer:-visualizing-effect-sizes-in-meta-analysis enumerated-auditory-response-(ear) eons epi-example-dicom epilab epilepcy-neuroimaging-data e-prime e-prime-extensions-for-fmri e-prime-extensions-for-smi e-prime-extensions-for-tobii epsilon-radial-networks erplab erp-pca-toolkit erpwavelab estimating-functional-brain-network-based-on-modularity-prior european-epilepsy-database-epilepsiae evaluation-of-automated-brain-extraction-methods-on-multi-site-multi-disorder-data ew_dmgwas:-edge-weighted-dense-module-search-for-genome-wide-association-studies-and-gene-expression-profiles exposition-packages eyeballgui eye-eeg-(combined-eye-tracking-&-eeg) faceted-search-based-ontology-visualizer factory_t1_dti fadttster false-discovery-rate-weighted fandtasia-java-applet:-dt-mri-processing fasst-–-fmri-artefact-rejection-and-sleep-scoring-toolbox fast-fourier-transform-(fft)-javalibrary fast-lesion-extraction-using-convolutional-neural-networks fast-nonlocal-means-for-mri-denoising fast-t2-relaxation-data-analysis-with-stimulated-echo-correction-and-non-local-spatial-regularisation fbirn-image-processing-scripts-(fips) fiber-tracking-based-on-finsler-distance fiber-tracking-tool fiberviewerlight fieldtrip filamenttracer fine-relational-spatial-topology finsler-tractography-module-for-slicer fitbir:-federal-interagency-traumatic-brain-injury-research fitting-oscillations-&-one-over-f fiv fmri-bold-signal-simulator fmri-classification-in-r fmri-cpca fmri-grocer fmripower fmriprep fmri-results-comparison-library fmristat fnir-devices fnirs-data-analysis-environment fnirs-during-stroop-before-and-after-acute-bout-of-aerobic-exercise fnirs-research-database fnirs-toolbox forward:-accurate-finite-element-electromagnetic-head-models framewise-integrated-real-time-mri-monitoring framework-for-open-programmatic-access freesurfer freesurfer-fsaverage freesurfr fsl fsl-extensions functional-analysis-of-diffusion-tensor functional-by-strutural-hierarchical-mapping-(fsh-mapping) functional-connectivity-analysis-tool-for-near-infrared-spectroscopy-data functional-connectivity-community functional-imaging-birn functional-mixed-processes-models functional-organization-of-the-mouse-lemur-primate-microcebus-murinus-:-from-multilevel-validation-to-comparison-with-humans functional-real-time-interactive-endogenous-neuromodulation-and---decoding-(friend) functional-regression-analysis-of-dti functional-roi-atlas fusion-ica-toolbox-(fit) fvgwas:-fast-voxelwise-genome-wide-association-analysis g-3-/-neurospaces-simulator gaain-toolset gambit gamma gannet---batch-analysis-of-edited-magnetic-resonance-spectroscopy-(mrs)-data gat-:-graph-analysis-toolbox gazereader g.bsanalyze gcca generalized-covariance-analysis-(gcva) generalized-ppi-toolbox generation-r-pediatric-mri-resources genesis-neural-simulator gengen gestr giant:-genome-scale-integrated-analysis-of-gene-networks-in-tissues gibbsconnectome gifti gimias gimme-(group-iterative-multiple-model-estimation) glirt glmdenoise:-a-fast,-automated-technique-for-denoising-task-based-fmri-data gmac:-a-matlab-toolbox-for-spectral-granger-causality-analysis-of-fmri-data gpu-based-affine-registration gradient-non-linearity-unwarping-tool grantees-meeting-for-nitrc graph-theory-glm-(gtg)-matlab-toolbox graphtools graphvar:-a-user-friendly-toolbox-for-comprehensive-graph-analyses-of-functional-brain-connectivity gretna group-ica-toolbox-(gift-and-eegift) group-information-guided-ica group-level-imputation-of-statistic-maps group-sparse-canonical-correlation-analysis groupwise-image-registration gsa-snp gtract gwas3d gwasdb gwasrap hadoop-for-data-colocation haiko89:-mri-in-vivo-baboon-brain-templates-collection-(papio-anubis) hammer-and-wml-modules-for-3d-slicer hammer:-deformable-registration hammer-suite hbm-hackathon hcp-harvard/mgh-ucla hcp-wu-minn-consortium hdbig hd-neuro-informatics hermes hfh_dti_data hfh_dti_tle_tools hierarchical-event-descriptor-(hed)-tags hierarchical-functional-networks-in-resting-state-fmri high-dynamic-range-for-local-contrast-enhancement-of-mri high-quality-diffusion-weighted-imaging-of-parkinson's-disease hint:-hierarchical-independent-component-analysis-toolbox hippocampalunfolding_manualinitialization hippocampome.org hippocampus-subregional-masks-based-on-gray-matter-volume-covariance hi-speed-software-packets histolozee---3d-histology-reconstruction-and-co-registration-with-mri hitachi2nirs hitachi-optical-topography-system homer2 hsnc human-connectome-project-(hcp) human-imaging-database-(hid) hyperion:-mri-digital-projection-system iatr ibeat ibsr ica-components-from-kessler,-et-al,-jama-psychiatry-2016 icn_atlas idea-lab-brain-image-processing-suite ielectrodes iit-human-brain-atlas-(v.4.1) imagecite imagej image-segmentation-based-on-the-local-center-of-mass-computation image-synthesis imagevis3d imcalc:-batch-image-calulations-and-transformations-(spm5/8/12) imeka-tractography-service imod---3d-reconstruction-and-analysis incf incf-neuroimaging-data-sharing individual-morphological-connectivity-using-wavelet-transform infant-macaque-mri-and-dti-templates inia19-primate--brain-atlas insight-toolkit instar-(infant-joint-segmentation-and-registration) inter-group-registration-toolbox international-imaging-genetics-conference internet-brain-volume-database-(ibvd) intervar intra--and-inter-scanner-reliability-of-rs-fmri-bold-and-asl-with-eyes-closed-vs.-eyes-open intrinsic-unscented-kalman-filter-(iukf)-tractography-software-v1.0 in-vivo-memri-based-mouse-brain-atlas in-vivo-memri-based-rat-brain-atlas invizian isc-toolbox:-inter-subject-correlation-analysis-for-fmri-in-matlab iso2mesh iterative-dual-regression-with-sparse-prior itk-snap itools-resourceome iview-x-mri-lr---eye-tracking-for-fmri ixi-dataset jhu-proj.-in-applied-medical-imaging jim jip-fmri-analysis-toolkit jist:-java-image-science-toolkit jist-resources-for-algorithm-development joint-anisotropic-lmmse-filter-for-stationary-rician-noise-removal-in-dwi journals k-space-domain-image-fusion kurtosis-imaging-network---kin kwscene:-mrml-based-atlas-and-scene-builder/reader/writer kymata-atlas laboratory-of-neuro-imaging-(loni) landman-neuroimaging-tools laplace-beltrami-filter-on-quadedge-meshes lddmm lead-connectome lead-dbs learning-based-image-segmentation-toolbox level-set-segmentation-for-slicer3 lfpy libeep libminc---medical-image-netcdf-library libsbml licensing lightweight-data-pipeline limo-eeg linear-fascicle-evaluation lipsia l-measure l-neuron local-binary-pattern-analysis-tools-for-mr-brain-images local-label-learning-for-image-segmentation local-label-learning-(lll)-segmentation local-manifold-learning-for-multi-atlas-segmentation longitudinal-multiple-sclerosis-lesion-imaging-archive longitudinal-neuroimaging-hippocampal-markers-for-diagnosing-alzheimer's-disease longitudinal-rhesus-monkey-brain-templates loni-debabeler loni-de-identification-debablet loni-inspector loni-pipeline-environment loni-provenance-editor lop-dwi loris low-resolution-brain-electromagnetic-tomography lumina-lp--400-response-system lupus-patients-with-white-matter-lesions m3 mabmis-for-slicer-4:-multi-atlas-based-multi-image-segmentation mabmis:-multi-atlas-based-segmentation mach mach2dat mach2qtl mach-admix mag-design-and-engineering magnetic-resonance-angiography-atlas-dataset magpro-magnetic-stimulator-(tms) mango manually-labeled-mri-brain-scan-database manualwmh-imagej-macros map3d mapping-population-based-structural-connectomes maps4mipav-(exploratory-jist) marsbar-region-of-interest-toolbox mars-(multi-atlas-robust-segmentation) masi_hardi_reproducibility masi-label-fusion masimatlab masked-ica-(mica)-toolbox mask_explorer massively-expedited-genome-wide-heritability-analysis-(megha) matlab-tutorial-on-diffusion-tensor-mri mayo-clinic-adult-lifespan-template-and-atlases mazesuite mcic--schizophrenic-and-matched-control-data mcml-&-conv mdr-multifactor-dimensionality-reduction measure-projection-toolbox-(mpt) measuring-eye-states-in-functional-mri medical-image-processing-and-visualization-in-virtual-environments medinria megsim meg-tools menga:-multimodal-environment-for-neuroimaging-and-genomic-analysis mesh-based-monte-carlo-(mmc) meshgen---surface-meshes-from-volumetric-segmentation-data meshvalmet:-validation-metric-for-meshes meshview-for-brain-atlases metasearch meth mga---multimodal-glioma-analysis mgdm:-multi-geometric-deformable-model mgh-neonatal/pediatric-adc-atlases mias-registration-toolkit microdraw micro-manager microstructural-correlation-toolbox mid-space-independent-deformable-image-registration minc-example-files minc-toolkit minc-toolkit-testsuite minc-toolkit-v2 mindboggle-101-manually-labeled-brains mindboggle:-open-source-software-for-analyzing-the-shapes-of-brain-structures-from-human-mri-data mindseer mipav mircen-macaca-fascicularis-brain-mri-segmentation-dataset miriad-(minimal-interval-resonance-imaging-in-alzheimer's-disease) misst---microstructure-imaging-sequence-simulation-toolbox misteri mitk-diffusion miview mixed-effect-model-of-genetic-set-and-environment-interaction mmps-docker mne---minimum-norm-current-estimates mni-macaque-atlases moca modelgui monstr-multi-contrast-brain-stripping monte-carlo-extreme-(mcx) monte-carlo-simulation-software:-tmcimg monthly-morphometry-report morphometry-birn motrak:-head-motion-tracking-system mouse-birn mousebirn-atlasing-toolkit-(mbat) mouse-connectome-project-(mcp) mousetracker movement_info mpi-leipzig-mind-brain-body-dataset mpscope mrbrains-evaluation-framework-/-challenge mr-connectome-automated-pipeline-(mrcap) mri-brain-template-and-atlas-of-the-mouse-lemur-primate-microcebus-murinus mricloud mricrogl mricron mricros mri_cvpr mri-dataset-for-hippocampus-segmentation mri-defacer mri-denoising mri-neuroanatomy-labeling-services mri-predict mri-simulator mri-studio mriwatcher mrtool mrtrix multi-atlas-skull-stripping-(mass) multicomponent-t2-estimation-with-stimulated-echo-correction multi-contrast-and-submillimetric-3-tesla-hippocampal-subfield-segmentation-protocol-and-dataset-(mni-hisub25) multi-echo-independent-component-regression-(me-icr)-group-level-connectivity-(groupincorr)-dataset multi-fiber-reconstruction-from-dw-mri multi-modality-visualization-tool multi-modal-mri-reproducibility-resource multiple-correlation-function-tool multiscale-weighted-principal-component-regression multitracer-version-2 multivariate-general-linear-models-(mglm)-on-riemannian-manifolds multixplore:-visual-exploration-platform-for-multimodal-neuroimaging-data n2a n3---minc-b0-nonuniformity-correction ncaa-dod-grand-alliance-care-consortium ncanda:-data-integration-component nc-igt-fast-imaging-library nemo:-neuron-morphological-analysis-tool neo neoseg neosegpipeline---multisegpipeline netpet:-covariance-statistics-and-network-analysis-of-brain-pet-imaging-studies net-station network-based-statistic-(nbs) network-modification-(nemo)-tool-lite neural-electromagnetic-ontologies neuritetracer neuroconstruct neurodebian neurodevelopmental-mri-database neurodocker neuroelf neuroimage-field-of-view-normalization-tool neuro-image-management-system-(nims) neuroimaging-analysis-kit-(niak) neuroimaging-data-access-group neuroimaging-made-easy-blog neuroinformatics---the-journal neurolens neurolucida neuroml neuromorpho.org neuron neuron-c neuronj neuropub-visualizer neuroscience-gateway-(nsg) neuroscope neurosharelibrary neurostars neurosynth neurovault neuroviisas neuroweb---neuroimaging-database nft:-neuroelectromag-forward-modeling niag-addiction-data niagads---nia-genetics-of-alzheimer's-disease-data-storage-site nice-sign nidb---neuroinformatics-database nifti niftyrec niftyreg nifty-sim nighres nih-cidi-lung-segmentation-tool nih-cidi-segmentation-of-pet-images-based-on-affinity-propagation-clustering nih-marmoset-brain-atlas nih-pediatric-mri-data-repository niistat nilearn nimh-data-archive-/-national-database-for-autism-research nipy-community nipy-diffusion-imaging-analysis nipy-file-io nipy-pipeline-and-interfaces nipy-structural-and-functional-analysis niral-utilities nirep nirfast nirs-spm nirx2nirs:-a-nirx-to-.nirs-data-converter nirx-nirs-neuroimaging nitime nitrc-computational-environment-(nitrc-ce) nitrc-gforge-extensions nitrc-image-repository-(nitrc-ir) noddi-matlab-toolbox nonfractal non-redundant-connectivity-(nrc)-maps non-rigid-groupwise-registration-method nordicneurolab normative-independent-component-analysis nornir northwestern-university-schizophrenia-data-and-software-tool-(nusdast) notion-researchpacs npbayes-fmri nptk nrm2018-pet-grand-challenge-dataset ntu-dsi-122:-a-diffusion-spectrum-imaging-template-with-high-anatomical-matching-to-the-icbm-152-space numerical-fibre-generator-(nfg) nutil---neuroimaging-utilities nutmeg nvm nyu-csc-testretest oasis odin oei-fmri-compatible-olfactometer offline-processing-mri ogles---opengl/oiv-stereotactic-tool online-brain-atlas-reconciliation-tool online-eeg-artifact-removal-by-adaptive-spatial-filtering onprc-infant-macaque-brain-atlas open-connectome-project openelectrophy openfmri openmeeg open-neuroimaging-laboratory open-science-framework open-vanderbilt-archive-of-the-temporal-lobe openvibe openwalnut optil.io optseq---fmri-event-scheduler organization-for-human-brain-mapping orion-i ors-visual-si p-[(123)i]iodo-l-phenylalanine-(ipa)-spect-template-in-mni-coordinates palm---permutation-analysis-of-linear-models panda:-a-pipeline-tool-for-diffusion-mri papaya paradigm parallel-graph-theoretical-analysis-toolkit parallel-stochastic-ion-channel-simulator paraview parkinson's-disease-discovery-database pattern-recognition-based-on-joint-gam-sparsity-regression-model pbrain pdbp:-parkinson's-disease-biomarkers-program pd-rs-fmri-meta-and-validation peak_nii penn-chop-age-specific-neonate-atlas penncnv penn-hippocampus-atlas personode pestica-&-slomoco:-physio-and-motion-correction-tools phenolyer philadelphia-neurodevelopmental-cohort-(pnc) philips-users-community phoneme-syllable-tsinghua-university-cs&t-2011 phycaa+:-adaptive-physiological-noise-correction-for-bold-fmri physio-toolbox picasso-skull-stripping-tool picsl-multi-atlas-segmentation-tool pipeline-neuroimaging-virtualenvironment pipeline-system-for-octave-and-matlab plink pliviewer polgui--matlab-gui-for-polhemus-fastrak population-averaged-diffusion-tensor-imaging-atlas-of-the-sprague-dawley-rat-brain pos2spm ppmi predict predicting-alzheimer’s-conversion-in-mild-cognitive-impairment-patients-using-longitudinal-neuroimaging-and-clinical-markers prediction-and-diagnosis-for-depression-and-schizophrenia preprocessed-connectomes-project presentation primatologist principal-components-analysis-of-scalar,-vector,-and-mesh-vertex-data probabilisticbiascorrection probability-associated-community-estimation proctseg profound-hearing-loss-database program-for-optimal-design-of-blocked-fmri-experiments progression-score-model-toolkit pronto-–-pattern-recognition-for-neuroimaging-toolbox propixx-dlp-led-projector prospective-acquisition-correction-(pace)-resting-state-epi-data psychopy pubbrain-database pvs_enhance pydicom pymvpa pynifti pynn pysurfer pythagorean-displacement-and-motion-regressors pyxnat qcqp qmodeling quantification-of-structural-brain-connectivity-via-a-conductance-model quantitative-diffusion-tools quicknii---serial-section-aligner-to-volumetric-atlases random-forest-for-prediction-of-ci_wrs randomized-denoising-autoencoders-for-neuroimaging rapidart rapid-permutation-testing-toolbox rapidtide rda2nifti recorder recview registry-builder-data-harmonization-and-aggregation-tool repronim:-a-center-for-reproducible-neuroimaging-computation repronim-simple-reexecutable-publication resource-ontology-discussion-group rest:-a-toolkit-for-resting-state-fmri resting-state-fmri-from-healthy-young-subjects resting-state-fmri-of-patients-with-pre-eclampsia-and-healthy-pregnant-women resting-state-hemodynamic-response-function-retrieval-and-deconvolution-(rs-hrf) resting-state-meg-data-used-in-marek-et-al.,-2018,-plos-biology resting-state-pediatric-imaging-template rex rft_fdr rncut:-functional-brain-networks-parcellation robust-biological-parametric-mapping robust-brain-extraction-(robex) rockland-download-link-script rodent-brain-extraction-tool rodent-cortical-thickness-analysis rosetta-bit r-package-for-adaptive-dwi-analysis r-package-for-adaptive-fmri-analysis rt_image ruby-nifti s3dl-sparse-dictionary-learning-based-mr-image-and-lesion-segmentation scalable-brain-atlas schizconnect:-large-scale-schizophrenia-neuroimaging-data-mediation-&-federation scikit-learn scirun score scp_taskfmri scralyze scribe seecat seed-based-d-mapping-(sdm,-formerly-signed-differential-mapping) seg3d segediting segmentation-based-multimodal-rigid-image-registration segmentation-validation-engine self-threat-and-self-concept-data serotonin-transporter-highresolution-pet-template sfmproject shapeanalysismancova---spharm-tools shapecomplexatlas shapecomplexityindex shapepopulationviewer shapetools shapeviewer shapeworks signalml sigviewer simple-neurite-tracer simple-medical-imaging-library-interface-(smili) simulated-diffusion-weighted-datasets simulated-dw-mri-brain-data-sets-for-quantitative-evaluation-of-estimated-fiber-orientations single-index-varying-coefficient-model-for-functional-responses single-subject-resting-state-fmri-reproducibility-resource sinomo-(singular-node-motifs) sinoplan sivic skullstrippingtoolkit slep:-sparse-learning-with-efficient-projections slic:-a-whole-brain-parcellation-toolbox slice:drop slicer3-example-modules slicer3-module-rician-noise-filter slicersalt smac:-spatial-multi-category-angle-based-classifier small-effect-size-leads-to-reproducibility-failure-in-resting-state-fmri-studies snpm sock solar-eclipse-imaging-genetics-tools source-information-flow-toolbox spanish-resting-state-network sparse-sampling-of-silence-database spatial-analysis-3d spatially-constrained-parcellation spatial-statistical-parametric-mapping spd-rsfmri-neu-zs spharm-mat spharm-pdm-toolbox spicodyn spikecor:-fmri-tool-for-automated-correction-of-head-motion-spikes spinal-cord-toolbox spinesegmentation-module-for-3dslicer spm spm-for-fnirs-toolbox spm_ss---fmri-functional-localizers spm-workshop-hannover spot3d:-spatial-positioning-toolbox-for-head-markers-using-3d-scans s-rep-fitting,-statistics,-and-segmentation sri24-atlas:-normal-adult-brain-anatomy sr-research-eyelink-eye-trackers stamp-atlases-for-brain-abnormality-detection standard-amygdalar-fmri-probe-tasks staple stark-cross-sectional-aging startyourrecovery.org statistical-methods-for-heterogeneous-neuroimaging-data statistical-methods-for-manifold-valued-data statistics-online-computational-resource stdpc stereo-investigator stereoscopic-atlas-of-intrinsic-brain-networks-(saibn) stochastic-tractography-system structural-asymmetries studyforrest subject-library subject-order-independent-group-ica sumsdb-database surface-based-processing-and-analysis-of-mri surf-ice surfstat svv synchronized-histological-image-view-arc talairach-daemon tapas---translational-algorithms-for-psychiatry-advancing-science tapir tarquin-mrs-analysis-package task-based-fmri-atlases task-independent-fluctuations-discussion template-image-processing-library templates-for-in-vivo-mouse-brain templates-for-long-evans-rat-brain temporal-lobe.com temporal-lobe-epilpsy-multiple-modality-data tetrametrix textpresso---literature-search-engine the-brain-catalogue the-brain-coactivation-map the-cancer-imaging-archive-(tcia) the-cognitive-atlas the-de-identification-toolbox---a-data-sharing-tool-for-neuroimaging-studies the-ebds-infant-dti-fiber-atlases the-fmri-data-center the-grid-code-analysis-toolbox-(gridcat) the-medical-image-computing-and-computer-assisted-intervention-society the-multiscale-object-oriented-simulation-environment the-neuro-bureau the-neurophysiological-biomarker-toolbox the-role-of-v6-in-the-interaction-between-dorsal-and-ventral-streams-in-near-and-far-space-processing the-striatal-subregional-voimap the-virtual-brain-(tvb) time-encoding-and-decoding-toolbox tissue-thickness-estimation-via-minimum-line-integrals tns-data-repository tns-lymphatics-dataset toads-cruise-brain-segmentation-tools toolconnect topographica topographic-tract-filtering tortoise tractography-based-on-riemannian-geodesics-and-tensor-adjugates tractor:-tractography-with-r trafic---tracts-fibers-classification-using-deep-learning trees-toolbox trentool triangle-biosystems tumorsim turbo-brainvoyager turbofire turtleseg ucla-multimodal-connectivity-database uk-biobank---multimodal-imaging ultra-high-field-atlas-for-dbs-planning umanitoba-jhu-functionally-defined-human-white-matter-atlas umeå-brain-arteries unbiased-atlas-construction-via-population-(group-wise)-dramms-registration unbiased,-deformable-spatiotemporal-atlas-of-the-fetal-brain unc-0-1-2-infant-atlases unc-cedars-infant-atlas unc-detail-preserved-longitudinal-0-3-6-9-12-months-old-atlas unc-human-brain-atlas unc-neonate-and-infant-cortical-surface-atlases unc-primate-brain-atlas unc/utah-namic-dti-fiber-analysis-framework unc-wisconsin-neurodevelopment-rhesus-mri-database unfold-toolbox-for-regression-based-eeg-analysis university-of-illinois-cognitive-neuroimaging-laboratory-optical-brain-imaging-software user-friendly-functional-connectivity---uf²c using-make-for-neuroimaging-workflow:-manual-and-examples vaa3d vaa3d-and-vaa3d-neuron validate29-squirrel-monkey-brain-atlas valmap:-simple-statistical-mapping-tool vamca-cortical-meta-analysis-toolbox vandpire vartbx-–-variability-toolbox vectorvaluedhistogramnormalizer vervet-probabilistic-atlas viewpixx viewpixx-/3d viking-family-of-viewing/annotation/analysis-tools-for-connectomics vimeapy viola virtual-brain-generator virtual-digital-brain virtual-electrode-recording-tool-for-extracellular-potentials-(vertex) virtual-reality-brain-network-viewer visapy vismotionqcdb vist/e visualization-toolkit visually-evoked-potential-eeg vis:-visualized-statistics-toolbox-(spm5/8/12) vmtk-in-3d-slicer volbrain:-online-mri-segmentation voxbo vutools wannovar wavelet-based-image-fusion waxholm-space-atlas-of-the-c57bl/6j-mouse-brain waxholm-space-atlas-of-the-sprague-dawley-rat-brain web-game-for-collaborative-labeling web-interfaces-for-multiscale-images wfu_bpm wfu_mmnet---multivariate-modeling-of-brain-networks-toolbox wfu_pickatlas wfu_pipeline whitetext---annotated-neuroscience-text wisconsin-cortical-thickness-analysis-(cta)-toolbox wisconsin-rhesus-brain-parcellation wisconsin-white-matter-hyperintensities-segmentation-toolbox working-memory-trainer xcede-schema xfsl:-an-fsl-toolbox xjview,-a-viewing-tool-for-spm xnat xnat-extras xnbc ymdti:-diffusion-tensor-images-of-healthy-young-males zebrafish-heart-segmentation">Desktop</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="vaa3d">VR</input></li>

                                                        </ul>
                                                    </details>
                                                </li>


                                                <li class="cat-item cat-item-3">
                                                    <details>
                                                        <summary>
                                                            <label>HBP</label>
                                                        </summary>
                                                        <ul class="children">

                                                            <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="1000-functional-connectomes-project [123i]-fp-cit-spect-brain-template-in-mni-space 18f-dopa-pet-and-123i-fp-cit-(ioflupane---datscan)-spect-templates-for-spm-normalisation. 2008-miccai-ms-lesion-segmentation-challenge 3d-brain-atlas-reconstructor 3-d-connective-field-modeling-toolbox 3-dimensional-diffusion-tensor-imaging-(dti)-atlas-of-the-rat-brain-in-postnatal-day-5,-14,-and-adulthood 3dmeshmetric 3d-slicer 3dslicerlupuslesionmodule 3dsvm 3t-dwi-test-retest-reliability-dataset 3t-mri-data-set-on-structural-brain-behavior-correlations 4d-atlas-construction 4d-parser-(pathological-anatomy-regression-via-segmentation-and-registration) 4d-(spatio-temporal)-dramms-registration 7t-probabilistic-atlas-of-subthalamic-nucleus-for-use-with-3t-mri 7t-structural-mri-scans-atag a-3t-probabilistic-locus-coeruleus-atlas abc-(atlas-based-classification) abeat absorb-for-groupwise-registration adaptive-neuro-fuzzy-inference-system-granger-causality-(anfisgc) adjust---eeg-automatic-artifact-removal adni adolescent-brain-cognitive-development-(abcd)-study advanced-connectivity-analysis-(aca):-a-large-scale-functional-connectivity-data-mining-environment advanced-normalization-tools advanced-segmentation-tools-(asets) afni age-ility age-related-atrophy-dataset ahead a-joint-sparse-partial-correlation-method-for-estimating-group-functional-networks a-label-fusion-using-crf allen-brain-atlas:-data-portal altered-states-database alvin---lateral-ventricle-segmentation amilab analyze analyzer-2 anatomist anima anima-scripts annovar:-functional-annotation-of-genetic-variants-from-high-throughput-sequencing-data a-novel-group-fused-sparse-partial-correlation-method-for-simultaneous-estimation-of-functional-networks-in-group-comparisons a-novel-method-for-extracting-hierarchical-functional-subnetworks-based-on-a-multi-subject-spectral-clustering-approach anticorrelated-component-analysis antsr aperture a-probabilistic-subcortical-atlas-for-pd arctic a-robust-method-for-correcting-partial-volume-effect-in-asl-data arterial-territories-of-the-neonatal-brain:-automated-classification artifact-detection-tools-(art) artrepair-for-robust-fmri asa---advanced-source-analysis a-shimadzu-to-.nirs-data-converter ashs:-automatic-segmentation-of-hippocampal-subfields asl-data-processing-tool-box asl_spm8 atlas3d atlasconstructiontoolkit atlases-of-amygdala-and-hippocampus-for-pediatric-populations atlasing-of-the-basal-ganglia atpp:-automatic-tractography-based-parcellation-pipeline augmented-reality-mirror aura-tools-:-automated-retinal-analysis-tools autism-tissue-program---atp automated-image-registration-(air) automatic-extra-axial-cerebrospinal-fluid-(auto-eacsf) automatic-registration-toolbox automatic-segmentation-tool-adapter autoseg autotract axonal-beading-and-microtubule-quantification b0-and-eddy-current-correction-for-dti bash4rfmri bayesian-connectivity-analysis-software-(bca) bayesian-longitudinal-low-rank-regression bci2000 bcilab beast-segmentation-library besa best-practices-for-software-development beta-series-correlation bic-mni-models big-gaba bioera biofilmquant bioimage-suite biomedical-informatics-research-network biomesh3d biosig bisque bluepyopt books brainance-dti-module brainbay brainbox brainbrowser braincolor:-human-brain-anatomical-labeling-protocols-and-colormaps brain-connectivity-toolbox braincsi braindecodertoolbox brain-entropy-in-space-and-time-(best) brainfx brain-genomics-superstruct-project-(gsp)-open-access-data-release braingraph-editor-1.0-beta braingraph---graph-theory-analysis-of-brain-mri-data-in-r brainhack2014 brain-hierarchical-atlas:-a-brain-atlas-where-the-regions-of-interest-are-relevant-for-both-structure-and-function brainlife.io brainliner.jp brainmagix-spm-viewer brainmap-database brainmaps.org brainmask-volume-processing-tool brainnetome-atlas-viewer brainnetome-diffusionkit brainnetome-fmri-toolkit brainnet-viewer brainnetworkconstructionanalysisplatform brainparser brain-registration-evaluations-(2008-2011) brainscape brainsconstellationdetector brain-segmentation-testing-protocol brainsfit brainsight brainsimulator brainsmush brainsolution brainspell brainsroiauto brainssurfacestats brainstem-connectome-atlas brainstools brainstorm brainsuite brainvisa brainvisa-morphologist-extensions brainvoyager brainvoyager-brain-tutor brainvoyager-viewer brainwaver bratumia-(brain-tumor-image-analysis) brava bravissima:-human-brain-artery-nifti-atlas brede-toolbox brede-wiki brian-simulator broccoli bsmac:-bayesian-spatial-model-for-brain-activation-and-connectivity btk-:-baby-brain-toolkit bxh/xcede-tools c8:-corpus-callosum-computations cagwas calatk---atlas-building-tk calcfd---calculate-the-fractal-dimensionality-of-a-3d-structure camba camino camino-trackvis candi-neuroimaging-access-point candi-share:-schizophrenia-bulletin-2008 caret carlsim:-a-gpu-accelerated-snn-simulator cavernous-hemangioma:-epilepsy-&-add caworks cbfbirn cbica:-brain-anatomy-simulator cbica:-brain-tumor-modeling---coupled-solver cbica:-brain-tumor-viewer-(btv) cbica:-build-system-and-software-implementation-standard-(basis) cbica:-cancer-imaging-phenomics-toolkit-(captk) cbica:-chimera cbica:-compare cbica:-confetti cbica:-datasets cbica:-dtidroid cbica:-geodesic-graph-based-segmentation-with-shape-priors-(grasp) cbica:-glioma-image-segmentation-and-registration-(glistr) cbica:-gondola cbica:-identification-of-sparse-connectivity-patterns-in-rsfmri-(scplearn) cbica:-laboratory-for-individualized-breast-radiodensity-assessment-(libra) cbica:-moe cbica:-multi-atlas-region-segmentation-utilizing-ensembles-of-registration-algorithms-and-parameters-(muse) cbica:-multi-atlas-skull-stripping-(mass) cbica:-odvba cbica:-phi-estimator cbica:-portr cbica:-predict cbica:-tutorials-(image-processing-and-machine-learning)-using-cpp,-itk,-etc. cbica:-whitestripe cbica:-wmls-(white-matter-lesion-segmentation) cbinifti:-matlab/octave-nifti-library cbrain cbs-high-res-brain-processing-tools ccseg---corpus-callosum-segmentation cdf-hc-pointsetreg celeritas:-fiber-optic-response-system center-for-computational-biology-(ccb) center-for-functional-neuroimaging-tech cerebellar-white-matter-atlas cerebral-blood-flow-database-and-analysis-pipeline-(cbfdap) challenge-competitions-collection chiasm---combined-histology-mri-integrated-atlas-of-the-squirrel-monkey chinese-pediatric-atlas-(chn-pd-atlas) chronos:-a-multifunctional-response-and-stimulus-device cifti-connectivity-file-format cigal civility-:-cloub-based-interactive-visualization-of-tractography-brain-connectome cleanline cleave:-large-data-set-anova clinical-toolbox-for-spm cloud-reproducibility-challenge cluster-extent-correction cluster-reporter cmfreg c-mind-database cmind_py cmrep cnn-for-3d-brain-mri-classification cocomac-brain-connectivity-database cocomac-paxinos3d-viewer cognision-(tm)-system cognitive-paradigm-ontology coins colin-3t/7t-high-resolution-atlas combining-a-patch-based-approach-with-a-non-rigid-registration-based-label-fusion-method-for-the-hippocampal-segmentation computational-anatomy-toolbox---cat computational-morphometry-toolkit-(cmtk) conferences,-workshops-and-meetings connectir connectome-file-format-(cff) connectome-viewer connectome-workbench connectopy-toolkits conn-:-functional-connectivity-toolbox convert-mni-coordinates-to-or-from-xyz conwhat---a-library-for-connectome-based-white-matter-atlas-analysis corpus-callosum-and-brain-segmentations-from-abide-database corpus-callosum-thickness-profile-analysis-pipeline cortex cost c-pac cppi-toolbox-for-fmri create-dwi-atlas cuda-accelerated-spherical-model-m/eeg curry-7 cutnii---custom-angle-slicer-for-brain-atlas-volumes cwl-eeg-fmri cwl-eeg-fmri-data-set cwx-optical-imaging-system cytoseg daikon dalan:-disconnectivity-atlas-for-lesion-analysis-of-networks databrary data-format-tools datalad datapixx dbgap dbgap_cleaner dbsnp dcm2nii dcmtk---dicom-toolkit dcp：diffusion-connectome-pipeline deformable-template-model-for-brain-analysis demon-deep-multi-contrast-brain-extraction,-application-to-human-&-animal-imaging denoising-of-redundant-mr-series density-spectral-array-of-bis-vista-monitoring-system-in-a-functional-hemispherectomy densparcorr-:-dens-based-method-for-partial-correlation-estimation-in-large-scale-brain-networks dentaltools developmental-resting-state-fmri-and-structrual-sample-10-26-yo dfbidb diamond diatrack-particle-tracking-software dicccol-predictor-(v0.1) dicombrowser dicomconvert dicom_uploadgui diffusional-kurtosis-estimator diffusion-mri-@-dkfz-heidelberg diffusion-mri---in-vivo-and-phantom-data diffusion-mri-orientation-distribution-function-in-constant-solid-angle-(csa-odf)-and-hough-transform-tractography diffusion-mri-tool diffusion-toolkit-/-trackvis diffusion-tractography-with-kalman-filter diffusion-warp dinifti disease-state-prediction displacement-field-viewer distributome done:-detection-of-outlier-neurons dots-wm-tract-segmentation dparsf dramms-deformable-image-registration-toolbox draw3d-and-meshinator dsi-studio dt_bold_calc dti-atlas-builder dtibrainscope dti-denoising dti-fiber-tract-statistics dtiprocess-toolkit dti-reg dti-template-rhesus-macaques dti-tk dual-tree-complex-wavelet-combined-with-non-local-means-for-asl-fmri-denoising dwi/dti-quality-control-tool:-dtiprep dwmri-phantom-drift dw-mri-random-walk-simulator dw-mri-registration-in-fsl dynamicbc e39-macaque-histology-and-mri-data econnectome eegbase eeg-fnirs-hybrid-smr-bci-data eeg-human-categorization-data eeglab eeg-study-data-of-rapid-serial-visual-presentation-(rsvp)-in-ess-format eeg-study-schema-(ess) eeg-toolbox eegvis eeprobe efficient-longitudinal-upload-of-depression-in-the-elderly-(elude) efficient-permutation-testing elastic-net-based-parcellation-(enpa) elastix electrophysiology-analysis-toolkit-(elephant) emory-unc-developmental-macaque-structural-and-dti-atlases emse-suite enigma-dti-pipeline enigma:-enhancing-neuro-imaging-genetics-through-meta-analysis enigma-resting-state-fmri-processing-pipeline enigma-viewer:-visualizing-effect-sizes-in-meta-analysis enumerated-auditory-response-(ear) eons epi-example-dicom epilab epilepcy-neuroimaging-data e-prime e-prime-extensions-for-fmri e-prime-extensions-for-smi e-prime-extensions-for-tobii epsilon-radial-networks erplab erp-pca-toolkit erpwavelab estimating-functional-brain-network-based-on-modularity-prior european-epilepsy-database-epilepsiae evaluation-of-automated-brain-extraction-methods-on-multi-site-multi-disorder-data ew_dmgwas:-edge-weighted-dense-module-search-for-genome-wide-association-studies-and-gene-expression-profiles exposition-packages eyeballgui eye-eeg-(combined-eye-tracking-&-eeg) faceted-search-based-ontology-visualizer factory_t1_dti fadttster false-discovery-rate-weighted fandtasia-java-applet:-dt-mri-processing fasst-–-fmri-artefact-rejection-and-sleep-scoring-toolbox fast-fourier-transform-(fft)-javalibrary fast-lesion-extraction-using-convolutional-neural-networks fast-nonlocal-means-for-mri-denoising fast-t2-relaxation-data-analysis-with-stimulated-echo-correction-and-non-local-spatial-regularisation fbirn-image-processing-scripts-(fips) fiber-tracking-based-on-finsler-distance fiber-tracking-tool fiberviewerlight fieldtrip filamenttracer fine-relational-spatial-topology finsler-tractography-module-for-slicer fitbir:-federal-interagency-traumatic-brain-injury-research fitting-oscillations-&-one-over-f fiv fmri-bold-signal-simulator fmri-classification-in-r fmri-cpca fmri-grocer fmripower fmriprep fmri-results-comparison-library fmristat fnir-devices fnirs-data-analysis-environment fnirs-during-stroop-before-and-after-acute-bout-of-aerobic-exercise fnirs-research-database fnirs-toolbox forward:-accurate-finite-element-electromagnetic-head-models framewise-integrated-real-time-mri-monitoring framework-for-open-programmatic-access freesurfer freesurfer-fsaverage freesurfr fsl fsl-extensions functional-analysis-of-diffusion-tensor functional-by-strutural-hierarchical-mapping-(fsh-mapping) functional-connectivity-analysis-tool-for-near-infrared-spectroscopy-data functional-connectivity-community functional-imaging-birn functional-mixed-processes-models functional-organization-of-the-mouse-lemur-primate-microcebus-murinus-:-from-multilevel-validation-to-comparison-with-humans functional-real-time-interactive-endogenous-neuromodulation-and---decoding-(friend) functional-regression-analysis-of-dti functional-roi-atlas fusion-ica-toolbox-(fit) fvgwas:-fast-voxelwise-genome-wide-association-analysis g-3-/-neurospaces-simulator gaain-toolset gambit gamma gannet---batch-analysis-of-edited-magnetic-resonance-spectroscopy-(mrs)-data gat-:-graph-analysis-toolbox gazereader g.bsanalyze gcca generalized-covariance-analysis-(gcva) generalized-ppi-toolbox generation-r-pediatric-mri-resources genesis-neural-simulator gengen gestr giant:-genome-scale-integrated-analysis-of-gene-networks-in-tissues gibbsconnectome gifti gimias gimme-(group-iterative-multiple-model-estimation) glirt glmdenoise:-a-fast,-automated-technique-for-denoising-task-based-fmri-data gmac:-a-matlab-toolbox-for-spectral-granger-causality-analysis-of-fmri-data gpu-based-affine-registration gradient-non-linearity-unwarping-tool grantees-meeting-for-nitrc graph-theory-glm-(gtg)-matlab-toolbox graphtools graphvar:-a-user-friendly-toolbox-for-comprehensive-graph-analyses-of-functional-brain-connectivity gretna group-ica-toolbox-(gift-and-eegift) group-information-guided-ica group-level-imputation-of-statistic-maps group-sparse-canonical-correlation-analysis groupwise-image-registration gsa-snp gtract gwas3d gwasdb gwasrap hadoop-for-data-colocation haiko89:-mri-in-vivo-baboon-brain-templates-collection-(papio-anubis) hammer-and-wml-modules-for-3d-slicer hammer:-deformable-registration hammer-suite hbm-hackathon hcp-harvard/mgh-ucla hcp-wu-minn-consortium hdbig hd-neuro-informatics hermes hfh_dti_data hfh_dti_tle_tools hierarchical-event-descriptor-(hed)-tags hierarchical-functional-networks-in-resting-state-fmri high-dynamic-range-for-local-contrast-enhancement-of-mri high-quality-diffusion-weighted-imaging-of-parkinson's-disease hint:-hierarchical-independent-component-analysis-toolbox hippocampalunfolding_manualinitialization hippocampome.org hippocampus-subregional-masks-based-on-gray-matter-volume-covariance hi-speed-software-packets histolozee---3d-histology-reconstruction-and-co-registration-with-mri hitachi2nirs hitachi-optical-topography-system homer2 hsnc human-connectome-project-(hcp) human-imaging-database-(hid) hyperion:-mri-digital-projection-system iatr ibeat ibsr ica-components-from-kessler,-et-al,-jama-psychiatry-2016 icn_atlas idea-lab-brain-image-processing-suite ielectrodes iit-human-brain-atlas-(v.4.1) imagecite imagej image-segmentation-based-on-the-local-center-of-mass-computation image-synthesis imagevis3d imcalc:-batch-image-calulations-and-transformations-(spm5/8/12) imeka-tractography-service imod---3d-reconstruction-and-analysis incf incf-neuroimaging-data-sharing individual-morphological-connectivity-using-wavelet-transform infant-macaque-mri-and-dti-templates inia19-primate--brain-atlas insight-toolkit instar-(infant-joint-segmentation-and-registration) inter-group-registration-toolbox international-imaging-genetics-conference internet-brain-volume-database-(ibvd) intervar intra--and-inter-scanner-reliability-of-rs-fmri-bold-and-asl-with-eyes-closed-vs.-eyes-open intrinsic-unscented-kalman-filter-(iukf)-tractography-software-v1.0 in-vivo-memri-based-mouse-brain-atlas in-vivo-memri-based-rat-brain-atlas invizian isc-toolbox:-inter-subject-correlation-analysis-for-fmri-in-matlab iso2mesh iterative-dual-regression-with-sparse-prior itk-snap itools-resourceome iview-x-mri-lr---eye-tracking-for-fmri ixi-dataset jhu-proj.-in-applied-medical-imaging jim jip-fmri-analysis-toolkit jist:-java-image-science-toolkit jist-resources-for-algorithm-development joint-anisotropic-lmmse-filter-for-stationary-rician-noise-removal-in-dwi journals k-space-domain-image-fusion kurtosis-imaging-network---kin kwscene:-mrml-based-atlas-and-scene-builder/reader/writer kymata-atlas laboratory-of-neuro-imaging-(loni) landman-neuroimaging-tools laplace-beltrami-filter-on-quadedge-meshes lddmm lead-connectome lead-dbs learning-based-image-segmentation-toolbox level-set-segmentation-for-slicer3 lfpy libeep libminc---medical-image-netcdf-library libsbml licensing lightweight-data-pipeline limo-eeg linear-fascicle-evaluation lipsia l-measure l-neuron local-binary-pattern-analysis-tools-for-mr-brain-images local-label-learning-for-image-segmentation local-label-learning-(lll)-segmentation local-manifold-learning-for-multi-atlas-segmentation longitudinal-multiple-sclerosis-lesion-imaging-archive longitudinal-neuroimaging-hippocampal-markers-for-diagnosing-alzheimer's-disease longitudinal-rhesus-monkey-brain-templates loni-debabeler loni-de-identification-debablet loni-inspector loni-pipeline-environment loni-provenance-editor lop-dwi loris low-resolution-brain-electromagnetic-tomography lumina-lp--400-response-system lupus-patients-with-white-matter-lesions m3 mabmis-for-slicer-4:-multi-atlas-based-multi-image-segmentation mabmis:-multi-atlas-based-segmentation mach mach2dat mach2qtl mach-admix mag-design-and-engineering magnetic-resonance-angiography-atlas-dataset magpro-magnetic-stimulator-(tms) mango manually-labeled-mri-brain-scan-database manualwmh-imagej-macros map3d mapping-population-based-structural-connectomes maps4mipav-(exploratory-jist) marsbar-region-of-interest-toolbox mars-(multi-atlas-robust-segmentation) masi_hardi_reproducibility masi-label-fusion masimatlab masked-ica-(mica)-toolbox mask_explorer massively-expedited-genome-wide-heritability-analysis-(megha) matlab-tutorial-on-diffusion-tensor-mri mayo-clinic-adult-lifespan-template-and-atlases mazesuite mcic--schizophrenic-and-matched-control-data mcml-&-conv mdr-multifactor-dimensionality-reduction measure-projection-toolbox-(mpt) measuring-eye-states-in-functional-mri medical-image-processing-and-visualization-in-virtual-environments medinria megsim meg-tools menga:-multimodal-environment-for-neuroimaging-and-genomic-analysis mesh-based-monte-carlo-(mmc) meshgen---surface-meshes-from-volumetric-segmentation-data meshvalmet:-validation-metric-for-meshes meshview-for-brain-atlases metasearch meth mga---multimodal-glioma-analysis mgdm:-multi-geometric-deformable-model mgh-neonatal/pediatric-adc-atlases mias-registration-toolkit microdraw micro-manager microstructural-correlation-toolbox mid-space-independent-deformable-image-registration minc-example-files minc-toolkit minc-toolkit-testsuite minc-toolkit-v2 mindboggle-101-manually-labeled-brains mindboggle:-open-source-software-for-analyzing-the-shapes-of-brain-structures-from-human-mri-data mindseer mipav mircen-macaca-fascicularis-brain-mri-segmentation-dataset miriad-(minimal-interval-resonance-imaging-in-alzheimer's-disease) misst---microstructure-imaging-sequence-simulation-toolbox misteri mitk-diffusion miview mixed-effect-model-of-genetic-set-and-environment-interaction mmps-docker mne---minimum-norm-current-estimates mni-macaque-atlases moca modelgui monstr-multi-contrast-brain-stripping monte-carlo-extreme-(mcx) monte-carlo-simulation-software:-tmcimg monthly-morphometry-report morphometry-birn motrak:-head-motion-tracking-system mouse-birn mousebirn-atlasing-toolkit-(mbat) mouse-connectome-project-(mcp) mousetracker movement_info mpi-leipzig-mind-brain-body-dataset mpscope mrbrains-evaluation-framework-/-challenge mr-connectome-automated-pipeline-(mrcap) mri-brain-template-and-atlas-of-the-mouse-lemur-primate-microcebus-murinus mricloud mricrogl mricron mricros mri_cvpr mri-dataset-for-hippocampus-segmentation mri-defacer mri-denoising mri-neuroanatomy-labeling-services mri-predict mri-simulator mri-studio mriwatcher mrtool mrtrix multi-atlas-skull-stripping-(mass) multicomponent-t2-estimation-with-stimulated-echo-correction multi-contrast-and-submillimetric-3-tesla-hippocampal-subfield-segmentation-protocol-and-dataset-(mni-hisub25) multi-echo-independent-component-regression-(me-icr)-group-level-connectivity-(groupincorr)-dataset multi-fiber-reconstruction-from-dw-mri multi-modality-visualization-tool multi-modal-mri-reproducibility-resource multiple-correlation-function-tool multiscale-weighted-principal-component-regression multitracer-version-2 multivariate-general-linear-models-(mglm)-on-riemannian-manifolds multixplore:-visual-exploration-platform-for-multimodal-neuroimaging-data n2a n3---minc-b0-nonuniformity-correction ncaa-dod-grand-alliance-care-consortium ncanda:-data-integration-component nc-igt-fast-imaging-library nemo:-neuron-morphological-analysis-tool neo neoseg neosegpipeline---multisegpipeline netpet:-covariance-statistics-and-network-analysis-of-brain-pet-imaging-studies net-station network-based-statistic-(nbs) network-modification-(nemo)-tool-lite neural-electromagnetic-ontologies neuritetracer neuroconstruct neurodebian neurodevelopmental-mri-database neurodocker neuroelf neuroimage-field-of-view-normalization-tool neuro-image-management-system-(nims) neuroimaging-analysis-kit-(niak) neuroimaging-data-access-group neuroimaging-made-easy-blog neuroinformatics---the-journal neurolens neurolucida neuroml neuromorpho.org neuron neuron-c neuronj neuropub-visualizer neuroscience-gateway-(nsg) neuroscope neurosharelibrary neurostars neurosynth neurovault neuroviisas neuroweb---neuroimaging-database nft:-neuroelectromag-forward-modeling niag-addiction-data niagads---nia-genetics-of-alzheimer's-disease-data-storage-site nice-sign nidb---neuroinformatics-database nifti niftyrec niftyreg nifty-sim nighres nih-cidi-lung-segmentation-tool nih-cidi-segmentation-of-pet-images-based-on-affinity-propagation-clustering nih-marmoset-brain-atlas nih-pediatric-mri-data-repository niistat nilearn nimh-data-archive-/-national-database-for-autism-research nipy-community nipy-diffusion-imaging-analysis nipy-file-io nipy-pipeline-and-interfaces nipy-structural-and-functional-analysis niral-utilities nirep nirfast nirs-spm nirx2nirs:-a-nirx-to-.nirs-data-converter nirx-nirs-neuroimaging nitime nitrc-computational-environment-(nitrc-ce) nitrc-gforge-extensions nitrc-image-repository-(nitrc-ir) noddi-matlab-toolbox nonfractal non-redundant-connectivity-(nrc)-maps non-rigid-groupwise-registration-method nordicneurolab normative-independent-component-analysis nornir northwestern-university-schizophrenia-data-and-software-tool-(nusdast) notion-researchpacs npbayes-fmri nptk nrm2018-pet-grand-challenge-dataset ntu-dsi-122:-a-diffusion-spectrum-imaging-template-with-high-anatomical-matching-to-the-icbm-152-space numerical-fibre-generator-(nfg) nutil---neuroimaging-utilities nutmeg nvm nyu-csc-testretest oasis odin oei-fmri-compatible-olfactometer offline-processing-mri ogles---opengl/oiv-stereotactic-tool online-brain-atlas-reconciliation-tool online-eeg-artifact-removal-by-adaptive-spatial-filtering onprc-infant-macaque-brain-atlas open-connectome-project openelectrophy openfmri openmeeg open-neuroimaging-laboratory open-science-framework open-vanderbilt-archive-of-the-temporal-lobe openvibe openwalnut optil.io optseq---fmri-event-scheduler organization-for-human-brain-mapping orion-i ors-visual-si p-[(123)i]iodo-l-phenylalanine-(ipa)-spect-template-in-mni-coordinates palm---permutation-analysis-of-linear-models panda:-a-pipeline-tool-for-diffusion-mri papaya paradigm parallel-graph-theoretical-analysis-toolkit parallel-stochastic-ion-channel-simulator paraview parkinson's-disease-discovery-database pattern-recognition-based-on-joint-gam-sparsity-regression-model pbrain pdbp:-parkinson's-disease-biomarkers-program pd-rs-fmri-meta-and-validation peak_nii penn-chop-age-specific-neonate-atlas penncnv penn-hippocampus-atlas personode pestica-&-slomoco:-physio-and-motion-correction-tools phenolyer philadelphia-neurodevelopmental-cohort-(pnc) philips-users-community phoneme-syllable-tsinghua-university-cs&t-2011 phycaa+:-adaptive-physiological-noise-correction-for-bold-fmri physio-toolbox picasso-skull-stripping-tool picsl-multi-atlas-segmentation-tool pipeline-neuroimaging-virtualenvironment pipeline-system-for-octave-and-matlab plink polgui--matlab-gui-for-polhemus-fastrak population-averaged-diffusion-tensor-imaging-atlas-of-the-sprague-dawley-rat-brain pos2spm ppmi predict predicting-alzheimer’s-conversion-in-mild-cognitive-impairment-patients-using-longitudinal-neuroimaging-and-clinical-markers prediction-and-diagnosis-for-depression-and-schizophrenia preprocessed-connectomes-project presentation primatologist principal-components-analysis-of-scalar,-vector,-and-mesh-vertex-data probabilisticbiascorrection probability-associated-community-estimation proctseg profound-hearing-loss-database program-for-optimal-design-of-blocked-fmri-experiments progression-score-model-toolkit pronto-–-pattern-recognition-for-neuroimaging-toolbox propixx-dlp-led-projector prospective-acquisition-correction-(pace)-resting-state-epi-data psychopy pubbrain-database pvs_enhance pydicom pymvpa pynifti pynn pysurfer pythagorean-displacement-and-motion-regressors pyxnat qcqp qmodeling quantification-of-structural-brain-connectivity-via-a-conductance-model quantitative-diffusion-tools quicknii---serial-section-aligner-to-volumetric-atlases random-forest-for-prediction-of-ci_wrs randomized-denoising-autoencoders-for-neuroimaging rapidart rapid-permutation-testing-toolbox rapidtide rda2nifti recorder recview registry-builder-data-harmonization-and-aggregation-tool repronim:-a-center-for-reproducible-neuroimaging-computation repronim-simple-reexecutable-publication resource-ontology-discussion-group rest:-a-toolkit-for-resting-state-fmri resting-state-fmri-from-healthy-young-subjects resting-state-fmri-of-patients-with-pre-eclampsia-and-healthy-pregnant-women resting-state-hemodynamic-response-function-retrieval-and-deconvolution-(rs-hrf) resting-state-meg-data-used-in-marek-et-al.,-2018,-plos-biology resting-state-pediatric-imaging-template rex rft_fdr rncut:-functional-brain-networks-parcellation robust-biological-parametric-mapping robust-brain-extraction-(robex) rockland-download-link-script rodent-brain-extraction-tool rodent-cortical-thickness-analysis rosetta-bit r-package-for-adaptive-dwi-analysis r-package-for-adaptive-fmri-analysis rt_image ruby-nifti s3dl-sparse-dictionary-learning-based-mr-image-and-lesion-segmentation scalable-brain-atlas schizconnect:-large-scale-schizophrenia-neuroimaging-data-mediation-&-federation scikit-learn scirun score scp_taskfmri scralyze scribe seecat seed-based-d-mapping-(sdm,-formerly-signed-differential-mapping) seg3d segediting segmentation-based-multimodal-rigid-image-registration segmentation-validation-engine self-threat-and-self-concept-data serotonin-transporter-highresolution-pet-template sfmproject shapeanalysismancova---spharm-tools shapecomplexatlas shapecomplexityindex shapepopulationviewer shapetools shapeviewer shapeworks signalml sigviewer simple-neurite-tracer simple-medical-imaging-library-interface-(smili) simulated-diffusion-weighted-datasets simulated-dw-mri-brain-data-sets-for-quantitative-evaluation-of-estimated-fiber-orientations single-index-varying-coefficient-model-for-functional-responses single-subject-resting-state-fmri-reproducibility-resource sinomo-(singular-node-motifs) sinoplan sivic skullstrippingtoolkit slep:-sparse-learning-with-efficient-projections slic:-a-whole-brain-parcellation-toolbox slice:drop slicer3-example-modules slicer3-module-rician-noise-filter slicersalt smac:-spatial-multi-category-angle-based-classifier small-effect-size-leads-to-reproducibility-failure-in-resting-state-fmri-studies snpm sock solar-eclipse-imaging-genetics-tools source-information-flow-toolbox spanish-resting-state-network sparse-sampling-of-silence-database spatial-analysis-3d spatially-constrained-parcellation spatial-statistical-parametric-mapping spd-rsfmri-neu-zs spharm-mat spharm-pdm-toolbox spicodyn spikecor:-fmri-tool-for-automated-correction-of-head-motion-spikes spinal-cord-toolbox spinesegmentation-module-for-3dslicer spm spm-for-fnirs-toolbox spm_ss---fmri-functional-localizers spm-workshop-hannover spot3d:-spatial-positioning-toolbox-for-head-markers-using-3d-scans s-rep-fitting,-statistics,-and-segmentation sri24-atlas:-normal-adult-brain-anatomy sr-research-eyelink-eye-trackers stamp-atlases-for-brain-abnormality-detection standard-amygdalar-fmri-probe-tasks staple stark-cross-sectional-aging startyourrecovery.org statistical-methods-for-heterogeneous-neuroimaging-data statistical-methods-for-manifold-valued-data statistics-online-computational-resource stdpc stereo-investigator stereoscopic-atlas-of-intrinsic-brain-networks-(saibn) stochastic-tractography-system structural-asymmetries studyforrest subject-library subject-order-independent-group-ica sumsdb-database surface-based-processing-and-analysis-of-mri surf-ice surfstat svv synchronized-histological-image-view-arc talairach-daemon tapas---translational-algorithms-for-psychiatry-advancing-science tapir tarquin-mrs-analysis-package task-based-fmri-atlases task-independent-fluctuations-discussion template-image-processing-library templates-for-in-vivo-mouse-brain templates-for-long-evans-rat-brain temporal-lobe.com temporal-lobe-epilpsy-multiple-modality-data tetrametrix textpresso---literature-search-engine the-brain-catalogue the-brain-coactivation-map the-cancer-imaging-archive-(tcia) the-cognitive-atlas the-de-identification-toolbox---a-data-sharing-tool-for-neuroimaging-studies the-ebds-infant-dti-fiber-atlases the-fmri-data-center the-grid-code-analysis-toolbox-(gridcat) the-medical-image-computing-and-computer-assisted-intervention-society the-multiscale-object-oriented-simulation-environment the-neuro-bureau the-neurophysiological-biomarker-toolbox the-role-of-v6-in-the-interaction-between-dorsal-and-ventral-streams-in-near-and-far-space-processing the-striatal-subregional-voimap the-virtual-brain-(tvb) time-encoding-and-decoding-toolbox tissue-thickness-estimation-via-minimum-line-integrals tns-data-repository tns-lymphatics-dataset toads-cruise-brain-segmentation-tools toolconnect topographica topographic-tract-filtering tortoise tractography-based-on-riemannian-geodesics-and-tensor-adjugates tractor:-tractography-with-r trafic---tracts-fibers-classification-using-deep-learning trees-toolbox trentool triangle-biosystems tumorsim turbo-brainvoyager turbofire turtleseg ucla-multimodal-connectivity-database uk-biobank---multimodal-imaging ultra-high-field-atlas-for-dbs-planning umanitoba-jhu-functionally-defined-human-white-matter-atlas umeå-brain-arteries unbiased-atlas-construction-via-population-(group-wise)-dramms-registration unbiased,-deformable-spatiotemporal-atlas-of-the-fetal-brain unc-0-1-2-infant-atlases unc-cedars-infant-atlas unc-detail-preserved-longitudinal-0-3-6-9-12-months-old-atlas unc-human-brain-atlas unc-neonate-and-infant-cortical-surface-atlases unc-primate-brain-atlas unc/utah-namic-dti-fiber-analysis-framework unc-wisconsin-neurodevelopment-rhesus-mri-database unfold-toolbox-for-regression-based-eeg-analysis university-of-illinois-cognitive-neuroimaging-laboratory-optical-brain-imaging-software user-friendly-functional-connectivity---uf²c using-make-for-neuroimaging-workflow:-manual-and-examples vaa3d vaa3d-and-vaa3d-neuron validate29-squirrel-monkey-brain-atlas valmap:-simple-statistical-mapping-tool vamca-cortical-meta-analysis-toolbox vandpire vartbx-–-variability-toolbox vectorvaluedhistogramnormalizer vervet-probabilistic-atlas viewpixx viewpixx-/3d viking-family-of-viewing/annotation/analysis-tools-for-connectomics vimeapy viola virtual-brain-generator virtual-digital-brain virtual-electrode-recording-tool-for-extracellular-potentials-(vertex) virtual-reality-brain-network-viewer visapy vismotionqcdb vist/e visualization-toolkit visually-evoked-potential-eeg vis:-visualized-statistics-toolbox-(spm5/8/12) vmtk-in-3d-slicer volbrain:-online-mri-segmentation voxbo vutools wannovar wavelet-based-image-fusion waxholm-space-atlas-of-the-c57bl/6j-mouse-brain waxholm-space-atlas-of-the-sprague-dawley-rat-brain web-game-for-collaborative-labeling web-interfaces-for-multiscale-images wfu_bpm wfu_mmnet---multivariate-modeling-of-brain-networks-toolbox wfu_pickatlas wfu_pipeline whitetext---annotated-neuroscience-text wisconsin-cortical-thickness-analysis-(cta)-toolbox wisconsin-rhesus-brain-parcellation wisconsin-white-matter-hyperintensities-segmentation-toolbox working-memory-trainer xcede-schema xfsl:-an-fsl-toolbox xjview,-a-viewing-tool-for-spm xnat xnat-extras xnbc ymdti:-diffusion-tensor-images-of-healthy-young-males zebrafish-heart-segmentation">External</input></li> <li class="cat-item cat-item-3"><input name="filterCheckbox" type="checkbox" onclick="handleClick();" data-software="pliviewer">Internal</input></li>

                                                        </ul>
                                                    </details>
                                                </li>


                                            </ul>
                                        </li>
                                    </ul>
                                </div>
                            </form>
                        </div>
                    </aside>
                </div>
            </div>

            <div id="primary" class="content-area">
                <div id="content" class="site-content" role="main" style="margin-left:150px;">

                    <!--<header class="archive-header">
                <h1 class="archive-title">Archives</h1>

            </header>--><!-- .archive-header -->

                    <header class="archive-header">
                        <h1 class="archive-title">Visualization Catalogue</h1>

                    </header>

                    <button onclick="topFunction()" id="myBtn" title="Go to top"
                            style="position: fixed; bottom: 20px; right: 30px">Top
                    </button>

                    <p>The visualization of data is key element in data analysis workflows in neuroscience, computing, and brain-related medicine.<br>
This catalogue of visualization tools provides information and navigation techniques to find the right tool for a designated analysis task.<br>
It covers tools developed in and outside of HBP.</p>
                                        <p>For questions and comments, please contact Benjamin Weyers (<a href="mailto:weyers@uni-trier.de">weyers@uni-trier.de</a>).</p>
<p>The filters presented on the right enable interactive selection of visualization tools by means of the following fields:</p>
                    <table style="margin: 0px;">
<tbody>
<tr>
<td><b>Usage:</b></td>
<td>Indicates the designated type of usage the tool is developed for, such as interactive analysis or presentation.</td>
</tr>
<tr>
<td><b>Data Type:</b></td>
<td>Indicates the type of data, which can be visualized by the tools, such as abstract, 2D or 3D spatial data.</td>
</tr>
<tr>
<td><b>Data Format:</b></td>
<td>Indicates the data format the tool supports, such as DICOM or NifTi.</td>
</tr>
<tr>
<td><b>Deployment:</b></td>
<td>Indicates, on which type of device the software tool can be deployed, such as HPC or mobile systems.</td>
</tr>
<tr>
<td><b>Display Device:</b></td>
<td>Indicates, on which type of device the resulting visualization can be displayed, such as desktop or VR.</td>
</tr>
<tr>
<td><b>HBP:</b></td>
<td>Indicates whether the software tool has been developed in HPB or is an external software.</td>
</tr>
</tbody>
</table>
                    <p>&nbsp;</p>


                    <h4><b>Active Filter:</b></h4>
                    <div id="filterDIV">&nbsp;</div>
                    <br/>
                    <details>
                        <summary>
                            <b style="font-size: 15px;">Filtered Software:</b>
                        </summary>
                        <br/>
                        <ul id="softwareDIV"></ul>

                    </details>
                    <hr>
                    <article id="1000-functional-connectomes-project" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="1000 Functional Connectomes Project">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=1000-functional-connectomes-project" rel="bookmark" name="1000-functional-connectomes-project">1000 Functional Connectomes Project</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ATTENTION: The 1000 Functional Connectomes Project has a new home page at NITRC. Please visit us at:<br />
<a href="http://fcon_1000.projects.nitrc.org" target="_new">http://fcon_1000.projects.nitrc.org</a><br />
<br />
This is the parent project for ABIDE, ADHD-200 (ADHD200), INDI, CORR, NKI-Rockland (NKI), Healthy Brain Network (HBN) and other projects. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=296&screenshot_id=211"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=296&screenshot_id=211" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 24, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://fcon_1000.projects.nitrc.org">http://fcon_1000.projects.nitrc.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="[123i]-fp-cit-spect-brain-template-in-mni-space" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="[123I] FP-CIT SPECT brain template in MNI space">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=[123i]-fp-cit-spect-brain-template-in-mni-space" rel="bookmark" name="[123i]-fp-cit-spect-brain-template-in-mni-space">[123I] FP-CIT SPECT brain template in MNI space</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The FP-CIT SPECT brain template has been created using a fully automatic procedure involving posterization of the source image to three levels: background, brain and striatum. <br />
<br />
We performed a spatial affine registration of these 40 posterized source images to a posterized reference image in the MNI space. The intensity values of the transformed images is normalized linearly, assuming that the histogram of the intensity values follows an alpha-stable distribution. Lastly, we built the [123I]FP-CIT SPECT template by the mean of the transformed and normalized images.<br />
<br />
More info: <br />
<br />
1) Salas-Gonzalez et al. Building a FP-CIT SPECT brain template using a posterization approach. Accepted in Neuroinformatics.<br />
<br />
2) Salas-Gonzalez et al. Linear intensity normalization of FP-CIT SPECT brain images using the alpha-stable distribution. NeuroImage, Volume 65, 2013, pp. 449-455.<br />
<a href="http://dx.doi.org/10.1016/j.neuroimage.2012.10.005" target="_new">http://dx.doi.org/10.1016/j.neuroimage.2...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 20, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial Share Alike</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/fp_cit_atlas/">http://www.nitrc.org/projects/fp_cit_atlas/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Granada, Spain. <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="18f-dopa-pet-and-123i-fp-cit-(ioflupane---datscan)-spect-templates-for-spm-normalisation." class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="18F-DOPA PET and 123I-FP-CIT (Ioflupane - DaTSCAN) SPECT templates for SPM normalisation.">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=18f-dopa-pet-and-123i-fp-cit-(ioflupane---datscan)-spect-templates-for-spm-normalisation." rel="bookmark" name="18f-dopa-pet-and-123i-fp-cit-(ioflupane---datscan)-spect-templates-for-spm-normalisation.">18F-DOPA PET and 123I-FP-CIT (Ioflupane - DaTSCAN) SPECT templates for SPM normalisation.</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The 18F-DOPA and 123I-FP-CIT (Ioflupane/DaTSCAN) templates are intended for use as templates for SPM automated normalisation. Created for SPM8 but SPM12 compatible. Available in NifTI file format. <br />
<br />
The updated templates are symmetrical and adjusted to MNI-space templates from 12 (18F-DOPA) and 30 controls without evidence of nigrostriatal degeneration (123I-FP-CIT). <br />
<br />
Authors: F.J García-Gómez, I Huertas, JA Lojo, D García-Solís.<br />
Contact: <a href="mailto:javier191185@gmail.com" target="_new">javier191185@gmail.com</a><br />
<br />
Terms of use:<br />
These images are intended for free use by the neuroimaging community. However, we do ask that you please cite the following references in any publications which make use of these images:<br />
- García-Gómez FJ, et al. Elaboration of the SPM template for the standardization of SPECT images with 123I-Ioflupane. Rev Esp Med Nucl Imagen Mol. 2013;32:350-6.<br />
- García-Gómez FJ, et al. Elaboración de una plantilla de SPM para la normalización de imágenes de PET con 18F-DOPA. Imagen Diagn. 2018; 9(1):23−25. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=797&screenshot_id=723"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=797&screenshot_id=723" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 14, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/spmtemplates/">http://www.nitrc.org/projects/spmtemplates/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="2008-miccai-ms-lesion-segmentation-challenge" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="2008 MICCAI MS Lesion Segmentation Challenge">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=2008-miccai-ms-lesion-segmentation-challenge" rel="bookmark" name="2008-miccai-ms-lesion-segmentation-challenge">2008 MICCAI MS Lesion Segmentation Challenge</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Welcome to the MS lesion segmentation challenge 2008 download site at NITRC. The goal of this competition is to compare different algorithms to segment the MS lesions from brain MRI scans.<br />
<br />
If you want to join the competition, you can download data set from links here (with the password in the README that can be downloaded here), and submit your segmentation results at <a href="http://www.ia.unc.edu/MSseg" target="_new">http://www.ia.unc.edu/MSseg</a> after registering your team. We require team name, password, and email address for future contact. You can submit the segmentation data in a zip file format. Please refer submission page for uploading data format.<br />
<br />
The data has a special license:<br />
- the data sets and associated segmentation data, or any data derived from these data sets, must not be given nor redistributed to persons not belonging to the registered team.<br />
- you need to upload the results of your algorithm and allow us to make the quantitative evaluation results publicly available on the MS lesion segmentation challenge site. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=745&screenshot_id=675"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=745&screenshot_id=675" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 22, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Other/Proprietary License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/msseg/">http://www.nitrc.org/projects/msseg/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UNC Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="3d-brain-atlas-reconstructor" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="3d Brain Atlas Reconstructor">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=3d-brain-atlas-reconstructor" rel="bookmark" name="3d-brain-atlas-reconstructor">3d Brain Atlas Reconstructor</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=three-dimensional-display" rel="tag">Three Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=web-service" rel="tag">Web Service</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> 3d Brain Atlas Reconstructor (3dBAR, <a href="http://www.3dbar.org" target="_new">http://www.3dbar.org</a>) is a software package for reconstructing three-dimensional models of brain structures from 2-D delineations using a customizable and reproducible workflow.<br />
<br />
3dBAR also works as an on-line service (<a href="http://service.3dbar.org" target="_new">http://service.3dbar.org</a>) offering a variety of functions for the hosted datasets: <br />
<br />
 - downloading reconstructions of desired brain structures in predefined quality levels in various supported formats as well as created using customizable settings,<br />
 - previewing models as bitmap thumbnails and (for webGL enabled browsers) interactive manipulation (zooming, rotating, etc.) of the structures,<br />
 - downloading slides from available datasets as SVG drawings.<br />
<br />
3dBAR service can also be used by other websites or applications to enhance their functionality.<br />
<br />
Check <a href="http://www.3dbar.org" target="_new">http://www.3dbar.org</a> for detailed description of the software and the latest releases. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 29, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://3dbar.org">http://3dbar.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Nencki Institute of Experimental Biology <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1, Other Format, VRML</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1, Other Format, VRML</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="3-d-connective-field-modeling-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="3-D Connective Field Modeling Toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=3-d-connective-field-modeling-toolbox" rel="bookmark" name="3-d-connective-field-modeling-toolbox">3-D Connective Field Modeling Toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> A framework for both fitting and describing connectivity patterns between regions that is a simple extension from the current population receptive field models in the visual neuroscience literature. This provides a description of the specific nature of connectivity from one region to another. The connectivity from each voxel in a designated seed region to a mapping region is modeled as a 3-dimensional Gaussian, providing location parameters and spread parameters. This allows the direct description of the relative mapping from one region to another. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 30, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/r2r_prf/">http://www.nitrc.org/projects/r2r_prf/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Stony Brook University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="3-dimensional-diffusion-tensor-imaging-(dti)-atlas-of-the-rat-brain-in-postnatal-day-5,-14,-and-adulthood" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="3-Dimensional Diffusion Tensor Imaging (DTI) Atlas of the Rat Brain In Postnatal Day 5, 14, and Adulthood">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=3-dimensional-diffusion-tensor-imaging-(dti)-atlas-of-the-rat-brain-in-postnatal-day-5,-14,-and-adulthood" rel="bookmark" name="3-dimensional-diffusion-tensor-imaging-(dti)-atlas-of-the-rat-brain-in-postnatal-day-5,-14,-and-adulthood">3-Dimensional Diffusion Tensor Imaging (DTI) Atlas of the Rat Brain In Postnatal Day 5, 14, and Adulthood</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> 3D DTI anatomical rat brain atlases have been created by the UNC- Chapel Hill Department of Psychiatry and the CAMID research collaboration. There are three age groups, postnatal day 5, postnatal day 14, and postnatal day 72. <br />
The subjects were Sprague-Dawley rats that were controls in a study on cocaine abuse and development. The P5 and P14 templates were made from scans of twenty rats each (ten female, ten male); the P72, from six females. The individual cases have been resampled to isotropic resolution, manually skull-stripped, and deformably registered via an unbiased atlas building method to create a template for each age group. Each template was then manually segmented using itk-SNAP software. <br />
Each atlas is made up of 3 files, a template image, a segmentation, and a label file. This work was funded by the UNC Neurodevelopment Disorders Research Center HD 03110, the NIH STTR grant R41 NS059095, and the NIH Program Project IP01DA022446-02. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=669&screenshot_id=594"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=669&screenshot_id=594" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 14, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dti_rat_atlas/">http://www.nitrc.org/projects/dti_rat_atlas/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UNC-Chapel Hill Department of Psychiatry <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="3dmeshmetric" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="3DMeshMetric">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=3dmeshmetric" rel="bookmark" name="3dmeshmetric">3DMeshMetric</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=overlap-metrics" rel="tag">Overlap Metrics</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=rendering" rel="tag">Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=shape-analysis" rel="tag">Shape Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> 3DMeshMetric is a visualization tool based on the VTK library.<br />
Its main feature is to compute and display surface-to-surface distance between two triangle meshes using user-specified uniform sampling (based on the source code of MeshValmet: <a href="http://www.nitrc.org/projects/meshvalmet/" target="_new">http://www.nitrc.org/projects/meshvalmet...</a>).<br />
3DMeshMetric also offers all the basic tools to visualize meshes such as color, opacity, smoothing, down sampling or type of representation.<br />
3DMeshMetric also contains an additional tool called ModelToModelDistance that computes distances between two triangle meshes (using the VTK library). This tool can be used either as a command line tool or integrated in 3D Slicer (<a href="http://www.slicer.org" target="_new">http://www.slicer.org</a>) as a CLI module. It is available in 3D Slicer as an extension to facilitate the download and installation process. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=754&screenshot_id=696"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=754&screenshot_id=696" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 4, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/meshmetric3d/">http://www.nitrc.org/projects/meshmetric3d/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina at Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Microsoft</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="3d-slicer" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="3D Slicer">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=3d-slicer" rel="bookmark" name="3d-slicer">3D Slicer</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-data" rel="tag">Atlas Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-to-image" rel="tag">Image-To-Image</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=morphology" rel="tag">Morphology</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=platform-or-development-environment" rel="tag">Platform Or Development Environment</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=resampling" rel="tag">Resampling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tensor-metric" rel="tag">Tensor Metric</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tractography" rel="tag">Tractography</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volume-measurement" rel="tag">Volume Measurement</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volumetric-analysis" rel="tag">Volumetric Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=warping" rel="tag">Warping</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> An extensible, cross-platform, totally open research platform for image computing. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=50&screenshot_id=218"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=50&screenshot_id=218" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 17, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>3D Slicer License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://slicer.org">http://slicer.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>The Slicer Community <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, MGH/MGZ, NIfTI-1, Nrrd, Other Format, bshort/bfloat</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, MGH/MGZ, NIfTI-1, Nrrd, Other Format, bshort/bfloat</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="3dslicerlupuslesionmodule" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="3DSlicerLupusLesionModule">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=3dslicerlupuslesionmodule" rel="bookmark" name="3dslicerlupuslesionmodule">3DSlicerLupusLesionModule</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The purpose of this Slicer3 module is to provide a capability for performing white matter lesion classification and summary. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 12, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>3D Slicer License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/lupuslesion/">http://www.nitrc.org/projects/lupuslesion/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="3dsvm" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="3dsvm">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=3dsvm" rel="bookmark" name="3dsvm">3dsvm</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> 3dsvm is a command-line program and plugin for AFNI (<a href="http://www.nitrc.org/projects/afni/" target="_new">http://www.nitrc.org/projects/afni/</a>), built around SVM-Light (<a href="http://svmlight.joachims.org/" target="_new">http://svmlight.joachims.org/</a>). It performs support vector machine (SVM) analysis on fMRI data and runs on Unix+X11+Motif systems, including SGI, Solaris, Linux, and Mac OS X. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 12, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GPL-Style Open Source</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/afni_3dsvm/">http://www.nitrc.org/projects/afni_3dsvm/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Virginia Tech <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>POSIX/UNIX-like</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>AFNI BRIK, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>AFNI BRIK, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="3t-dwi-test-retest-reliability-dataset" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="3T DWI test-retest reliability dataset">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=3t-dwi-test-retest-reliability-dataset" rel="bookmark" name="3t-dwi-test-retest-reliability-dataset">3T DWI test-retest reliability dataset</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This data-set contains all 3T MRI data used in Boekel et al., (submitted), in which we aimed to investigate the invivo test-retest reliability of several DTI derived measures.<br />
<br />
34 healthy young subjects were scanned twice on the same day with a 3T MRI scanner in which the following data was acquired:<br />
- T1 weighted whole brain scan (1mm isotropic). <br />
- B0 weighted whole brain scan (2mm isotropic)<br />
- Diffusion weighted whole brain scan (2mm isotropic, 32 directions, b-value=1000, 4 repetitions)<br />
<br />
A subset of 15 subjects were also scanned a third session on the same day, as well as in a two-week follow-up session with the same MRI protocol.<br />
<br />
The full reference of the article is: <br />
Boekel, W., Forstmann, B. U., Keuken, M.C. (In press) A test-retest reliability analysis of diffusion measures of white matter tracts relevant for cognitive control. Psychophysiology. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=990&screenshot_id=959"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=990&screenshot_id=959" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 20, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dwi_test-retest/">http://www.nitrc.org/projects/dwi_test-retest/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Amsterdam <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="3t-mri-data-set-on-structural-brain-behavior-correlations" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="3T MRI data set on structural brain-behavior correlations">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=3t-mri-data-set-on-structural-brain-behavior-correlations" rel="bookmark" name="3t-mri-data-set-on-structural-brain-behavior-correlations">3T MRI data set on structural brain-behavior correlations</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This data-set contains all 3T MRI and behavioural data used in Boekel, W., Wagenmakers, E. J., Belay, L., Verhagen, A.J., Brown, S. D., Forstmann, B. U. (2015) A purely confirmatory replication study of structural brain-behavior correlations. Cortex. and consists of:<br />
<br />
- 36 T1 and DWI images acquired on a 3T MRI scanner<br />
- Behavioral data for 2 computerized tasks, and 3 questionnaires. <br />
<br />
The behavioral data occasionally contains a subset of the 36 subjects, due to questionnaires/computer-tasks not having been completed fully or properly. Subject numbers (e.g. 1244) are matched to MRI subject numbers (e.g. sc1244).<br />
<br />
Note! <br />
A mistake was discovered in the post-processing pipeline of the DWI data. See corrigendum for a description of the error and corrected results:<br />
<br />
Keuken, M.C., Ly, A., Boekel W.E., Wagenmakers, E-J., Belay, L., Verhagen, J.A., Brown, S.D., Forstmann, B.U. (2017). Corrigendum A purely confirmatory replication study of structural brain-behavior correlations. Cortex. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 5, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/confrep2014/">http://www.nitrc.org/projects/confrep2014/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Amsterdam <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="4d-atlas-construction" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="4D Atlas Construction">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=4d-atlas-construction" rel="bookmark" name="4d-atlas-construction">4D Atlas Construction</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> 4DAtlas (4D Atlas Construction Toolbox) provides solutions for constructing longitudinal atlases, which are the necessary steps for many brain-related applications.<br />
<br />
This software package was developed in the IDEA group at UNC-Chapel Hill ( <a href="http://bric.unc.edu/ideagroup" target="_new">http://bric.unc.edu/ideagroup</a> ). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 25, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/atlas4d/">http://www.nitrc.org/projects/atlas4d/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina at Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="4d-parser-(pathological-anatomy-regression-via-segmentation-and-registration)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="4D-PARSeR (Pathological Anatomy Regression via Segmentation and Registration)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=4d-parser-(pathological-anatomy-regression-via-segmentation-and-registration)" rel="bookmark" name="4d-parser-(pathological-anatomy-regression-via-segmentation-and-registration)">4D-PARSeR (Pathological Anatomy Regression via Segmentation and Registration)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> 4D-PARSeR (Pathological Anatomy Regression via Segmentation and Registration) is a tool for analyzing 4D images with pathology. Originally developed for processing longitudinal images of patients with traumatic brain injury. The tool contains new image analysis algorithms that combine registration and segmentation in a coherent framework, accounting for extreme changes due to extensive tissue damage. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=724&screenshot_id=654"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=724&screenshot_id=654" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 3, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>3D Slicer License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/parser_4d/">http://www.nitrc.org/projects/parser_4d/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Utah <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="4d-(spatio-temporal)-dramms-registration" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="4D (Spatio-Temporal) DRAMMS Registration">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=4d-(spatio-temporal)-dramms-registration" rel="bookmark" name="4d-(spatio-temporal)-dramms-registration">4D (Spatio-Temporal) DRAMMS Registration</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> 4D (3D+time, or spatial-temporal) consistent deformable registration build on top of 3D pair-wise DRAMMS registration.<br />
<br />
- to quantify tumor changes in multiple visits;<br />
- to quantify voxel-wise neuro-development or structural growth. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 8, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dramms4d/">http://www.nitrc.org/projects/dramms4d/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Massachusetts General Hospital <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="7t-probabilistic-atlas-of-subthalamic-nucleus-for-use-with-3t-mri" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="7T Probabilistic atlas of Subthalamic Nucleus for Use with 3T MRI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=7t-probabilistic-atlas-of-subthalamic-nucleus-for-use-with-3t-mri" rel="bookmark" name="7t-probabilistic-atlas-of-subthalamic-nucleus-for-use-with-3t-mri">7T Probabilistic atlas of Subthalamic Nucleus for Use with 3T MRI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> 7T Probabilistic Atlas of STN for use with clinical 3T MRI </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 18, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/stn7t3t/">http://www.nitrc.org/projects/stn7t3t/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Washington University in St Louis <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="7t-structural-mri-scans-atag" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="7T Structural MRI scans ATAG">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=7t-structural-mri-scans-atag" rel="bookmark" name="7t-structural-mri-scans-atag">7T Structural MRI scans ATAG</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Structural brain data is key for the understanding of brain function and brain networks, i.e., connectomics. Here we present data sets available from the ‘atlasing of the basal ganglia (ATAG)’ consortium, which provides ultra-high resolution 7Tesla (T) magnetic resonance imaging (MRI) scans from young, middle-aged, and elderly participants. The ATAG data sets include whole-brain and reduced field-of-view MP2RAGE and T2* scans with ultra-high resolution at a sub millimeter scale. The data can be used to develop new algorithms that help building new high-resolution atlases both in the basic and clinical neurosciences. Also these atlases can be used to inform the exact positioning of deep-brain electrodes relevant in patients with Parkinson’s disease and neuropsychiatric diseases. Our results indicate that ATAG data sets allow direct visualization of smallest structures in the subcortex as well as of the brain stem. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=820&screenshot_id=801"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=820&screenshot_id=801" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 26, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/atag_mri_scans/">http://www.nitrc.org/projects/atag_mri_scans/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Amsterdam; MPI CBS <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="a-3t-probabilistic-locus-coeruleus-atlas" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="A 3T probabilistic Locus Coeruleus atlas">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=a-3t-probabilistic-locus-coeruleus-atlas" rel="bookmark" name="a-3t-probabilistic-locus-coeruleus-atlas">A 3T probabilistic Locus Coeruleus atlas</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This atlas takes advantage of in vivo high resolution TSE 3T structural MRI. The atlas includes a probability map of the Locus Coeruleus based on healthy young subjects. The subjects were scanned twice to test the test-retest reliability of the identification of the Locus Coeruleus.<br />
<br />
A description of the MRI scan parameters, segmentation protocol, and registration protocol to MNI space can be found in:<br />
Tona, K-D., Keuken, M.C., de Rover, M.,Lakke, E., Forstmann, B.U., Nieuwenhuis, S., van Osch, M.J.P. (accepted). In vivo visualization of the locus coeruleus in humans: Quantifying the test-retest reliability. Brain, Structure and Function </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 22, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial Share Alike</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/prob_lc_3t/">http://www.nitrc.org/projects/prob_lc_3t/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Amsterdam, University of Leiden <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="abc-(atlas-based-classification)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ABC (Atlas Based Classification)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=abc-(atlas-based-classification)" rel="bookmark" name="abc-(atlas-based-classification)">ABC (Atlas Based Classification)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ABC (Atlas Based Classification) is a comprehensive processing pipeline developed and used at University of North Carolina and University of Utah for brain MRIs. The processing pipeline includes image registration, filtering, segmentation and inhomogeneity correction. The tool is cross-platform and can be run within 3D Slicer or as a stand-alone program.<br />
<br />
The image segmentation algorithm is based on the EMS software developed by Koen van Leemput. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=297&screenshot_id=573"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=297&screenshot_id=573" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 3, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>3D Slicer License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/abc/">http://www.nitrc.org/projects/abc/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="abeat" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="aBEAT">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=abeat" rel="bookmark" name="abeat">aBEAT</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> aBEAT is a 4D Adult Brain Extraction and Analysis Toolbox with graphical user interfaces to consistently analyze 4D adult brain MR images. Single-time-point images can also be analyzed. Main functions of the software include image preprocessing, 4D brain extraction, 4D tissue segmentation, 4D brain labeling, ROI analysis. The software is developed by the IDEA group at the University of North Carolina at Chapel Hill, directed by Dr. Dinggang Shen (<a href="mailto:dinggang_shen@med.unc.edu" target="_new">dinggang_shen@med.unc.edu</a>). <br />
<br />
Linux operating system (64 bit) is required. A computer with 8G memory (or more) is recommended for processing many images simutaneously. The graphical user interfaces and overall framework of the software are implemented in MATLAB. The image processing functions are implemented with the combination of C/C++, MATLAB, Perl and Shell languages. Parallelization technologies are used in the software to speed up image processing. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=640&screenshot_id=563"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=640&screenshot_id=563" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 19, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>aBEAT</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/abeat/">http://www.nitrc.org/projects/abeat/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina at Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="absorb-for-groupwise-registration" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ABSORB for groupwise registration">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=absorb-for-groupwise-registration" rel="bookmark" name="absorb-for-groupwise-registration">ABSORB for groupwise registration</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=affine-warp" rel="tag">Affine Warp</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-to-template" rel="tag">Image-To-Template</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=intermodal" rel="tag">Intermodal</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=intersubject" rel="tag">Intersubject</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=intrasubject" rel="tag">Intrasubject</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=nonlinear-warp" rel="tag">Nonlinear Warp</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This software package implements ABSORB: Atlas Building by Self-Organized Registration and Bundling – an algorithm for effective groupwise registration, which has been published as:<br />
<br />
Hongjun Jia, Guorong Wu, Qian Wang, Dinggang Shen, &quot;ABSORB: Atlas Building by Self-Organized Registration and Bundling&quot;, NeuroImage, Vol. 51, No. 3, 1 July 2010, pp. 1057-1070.<br />
<br />
The package is available free to the public for the academic research purpose at <a href="http://bric.unc.edu/ideagroup/free-softwares/" target="_new">http://bric.unc.edu/ideagroup/free-softw...</a> . <br />
<br />
The required input is a set of 3D MR intensity images (in Analyze format with paired .hdr and .img files) with a text file (.txt) listing all header file (.hdr) names. The output is the set of registered images together with the corresponding dense deformation fields. This software has been tested on Windows XP (32-bit) and Linux (64-bit, kernel version 2.6.18-194.el5).<br />
<br />
<br />
This software was developed in IDEA group in UNC-Chapel Hill. For more information, please visit our lab at <a href="http://bric.unc.edu/ideagroup" target="_new">http://bric.unc.edu/ideagroup</a> . 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=397&screenshot_id=281"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=397&screenshot_id=281" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 27, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://bric.unc.edu/ideagroup/free-softwares/ABSORB/">http://bric.unc.edu/ideagroup/free-softwares/ABSORB/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>Windows 95/98/2000<br>Windows NT/2000<br>Windows Vista<br>Windows XP</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, Microsoft Word Document, PDF - Portable Document Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, Microsoft Word Document, PDF - Portable Document Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="adaptive-neuro-fuzzy-inference-system-granger-causality-(anfisgc)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Adaptive Neuro-Fuzzy Inference System Granger Causality (ANFISGC)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=adaptive-neuro-fuzzy-inference-system-granger-causality-(anfisgc)" rel="bookmark" name="adaptive-neuro-fuzzy-inference-system-granger-causality-(anfisgc)">Adaptive Neuro-Fuzzy Inference System Granger Causality (ANFISGC)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The codes belong to the following article:<br />
Farokhzadi, M., Hossein-Zadeh, G.A., Soltanian-Zadeh, H., 2018. Nonlinear Effective Connectivity Measure Based on Adaptive Neuro Fuzzy Inference System and Granger Causality. J Neuroimage 181, 382–394. <a href="https://doi.org/10.1016/j.neuroimage.2018.07.024" target="_new">https://doi.org/10.1016/j.neuroimage.201...</a><br />
The shared MATLAB codes contain: <br />
1) Generating three time series with 4000 samples (n=1,…,4000) based on the following equations:<br />
x_1 (n)=3.4x_1 (n-1)(1-x_1^2 (n-1)) e^(-x_1^2 (n-1) )+ε_1 (n)<br />
x_2 (n)=3.4x_2 (n-1)(1-x_2^2 (n-1)) e^(-x_2^2 (n-1) )+cx_2 (n-1) x_1 (n-1)+ε_2 (n)<br />
x_3 (n)=3.4x_3 (n-1)(1-x_3^2 (n-1)) e^(-x_3^2 (n-1) )+0.3x_2 (n-1)+0.5x_1^2 (n-1)+ε_3 (n)<br />
The signal-to-noise ratio (SNR) is 0 dB.<br />
The parameter c, which indicates the coupling strength of (x1→x2) connection, is set to 0.5.<br />
2) Implementation of ANFISGC criterion to detect both linear and nonlinear causal connections among brain regions.<br />
In the Matlab functions, the inputs and outputs are described. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 31, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>ANFISGC_License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/anfisgc_measure/">http://www.nitrc.org/projects/anfisgc_measure/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="adjust---eeg-automatic-artifact-removal" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ADJUST - EEG Automatic Artifact Removal">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=adjust---eeg-automatic-artifact-removal" rel="bookmark" name="adjust---eeg-automatic-artifact-removal">ADJUST - EEG Automatic Artifact Removal</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=independent-component-analysis" rel="tag">Independent Component Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ADJUST is a completely automatic algorithm for artifact identification and removal in EEG data. ADJUST is based on Independent Component Analysis (ICA), a successful but unsupervised method for isolating artifacts from EEG recordings. ADJUST identifies artifacted ICA components by combining stereotyped artifact-specific spatial and temporal features. Features are optimised to capture blinks, eye movements and generic discontinuities. Once artifacted IC are identified, they can be simply removed from the data while leaving the activity due to neural sources almost unaffected.<br />
<br />
ADJUST runs in Matlab as a plugin of the EEGLAB software, the most popular software for analysis of electrophysiological data. No Matlab toolboxes are required.<br />
<br />
For more details, please read ADJUST reference paper and ADJUST tutorial (&quot;Docs&quot; tab on the left of the page). 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=739&screenshot_id=672"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=739&screenshot_id=672" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 1, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/adjust/">http://www.nitrc.org/projects/adjust/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Trento, Italy <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="adni" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ADNI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=adni" rel="bookmark" name="adni">ADNI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Data collected as part of the ADNI study are freely available to authorized investigators, through the Image Data Archive (IDA). Information about obtaining access to ADNI data may be found in the How to Apply for Data section (<a href="http://adni.loni.usc.edu/data-samples/" target="_new">http://adni.loni.usc.edu/data-samples/</a>).<br />
<br />
For more detailed information about accessing data from the IDA, please review the IDA User Manual.<br />
<br />
The following ADNI data types are available: Clinical: Demographics, Clinical Assessments, Cognitive Assessments); Imaging: MRI: Raw, pre- and post- processed image files; PET: Raw, pre- and post- processed image files; fMRI: (ADNI GO); DTI: (ADNI GO); Chemical Biomarker: Laboratory Results; Genetic: Illumina SNP genotyping; Image Analysis Results: Numeric results derived from image analyses.<br />
<br />
The ADNI database is managed and housed by the Laboratory of Neuro Imaging (<a href="http://www.LONI.usc.edu/" target="_new">http://www.LONI.usc.edu/</a>) at USC. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=470&screenshot_id=344"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=470&screenshot_id=344" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 28, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>ADNI License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://adni.loni.usc.edu/">http://adni.loni.usc.edu/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Southern California <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, NIfTI-1, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, NIfTI-1, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="adolescent-brain-cognitive-development-(abcd)-study" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Adolescent Brain Cognitive Development (ABCD) Study">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=adolescent-brain-cognitive-development-(abcd)-study" rel="bookmark" name="adolescent-brain-cognitive-development-(abcd)-study">Adolescent Brain Cognitive Development (ABCD) Study</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data-resource" rel="tag">Data Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Adolescent Brain Cognitive Development (ABCD) Study is the largest long-term study of brain development and child health in the United States. The ABCD Research Consortium consists of a Coordinating Center, a Data Analysis and Informatics Center, and 21 research sites across the country, which will invite approximately 10,000 children ages 9-10 to join the study. Researchers will track their biological and behavioral development through adolescence into young adulthood.<br />
<br />
This study will determine how childhood experiences (such as sports, videogames, social media, unhealthy sleep patterns, and smoking) interact with each other and with a child’s changing biology to affect brain development and social, behavioral, academic, health, and other outcomes.<br />
<br />
The results of the ABCD Study will provide families; school superintendents, principals, and teachers; health professionals; and policymakers with practical information to promote the health, well-being, and success of children. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1256&screenshot_id=1165"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1256&screenshot_id=1165" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 26, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://abcdstudy.org">https://abcdstudy.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>DICOM, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>DICOM, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="advanced-connectivity-analysis-(aca):-a-large-scale-functional-connectivity-data-mining-environment" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Advanced Connectivity Analysis (ACA): a large scale functional connectivity data mining environment">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=advanced-connectivity-analysis-(aca):-a-large-scale-functional-connectivity-data-mining-environment" rel="bookmark" name="advanced-connectivity-analysis-(aca):-a-large-scale-functional-connectivity-data-mining-environment">Advanced Connectivity Analysis (ACA): a large scale functional connectivity data mining environment</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Using resting-state functional magnetic resonance imaging (rs-fMRI) to study functional connectivity is of great importance to understand normal development and function as well as a host of neurological and psychiatric disorders. Seed-based analysis is one of the most widely used rs-fMRI analysis methods. Here we describe a freely available large scale functional connectivity data mining software package called Advanced Connectivity Analysis (ACA). ACA enables large-scale seed-based analysis and brain-behavior analysis. It can seamlessly examine a large number of seed regions with minimal user input. ACA has a brain-behavior analysis component to delineate associations among imaging biomarkers and one or more behavioral variables. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 22, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU GPL v2</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/aca_rc/">http://www.nitrc.org/projects/aca_rc/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="advanced-normalization-tools" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Advanced Normalization Tools">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=advanced-normalization-tools" rel="bookmark" name="advanced-normalization-tools">Advanced Normalization Tools</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=morphology" rel="tag">Morphology</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=registration" rel="tag">Registration</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=temporal-transformation" rel="tag">Temporal Transformation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Advanced Normalization Tools (ANTS) : Image registration with variable transformation models (elastic, diffeomorphic, unbiased) and similarity metrics (landmarks, cross-correlation, mutual information, optical flow). Designed for neuroscience and medical imaging researchers and users. Special capabilities include symmetric diffeomorphic normalization, optimal template creation and user-guided normalization. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 27, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.picsl.upenn.edu/ANTS">http://www.picsl.upenn.edu/ANTS</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, NIfTI-1, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, NIfTI-1, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="advanced-segmentation-tools-(asets)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Advanced Segmentation Tools (ASeTs)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=advanced-segmentation-tools-(asets)" rel="bookmark" name="advanced-segmentation-tools-(asets)">Advanced Segmentation Tools (ASeTs)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Advanced Segmentation Tools (ASETS) - Fast Continuous Max Flow variants in CUDA/C/Matlab<br />
<br />
Attention: This is a beta version. We appreciate any bug reports and feedback on how to improve usability. For installation and usage please refer to the readme.md. <br />
<br />
<br />
<br />
- Fast parallel continuous max flow solvers in 2D/3D<br />
-- Binary max flow<br />
-- Multi-region (Potts model, Ishikawa model, Hierarchical Max Flow)<br />
-- In two different implementations (full flow and pseudo flow solvers)<br />
<br />
- Implemented in multiple languages<br />
-- Matlab/mex/C<br />
-- Matlab/CUDA<br />
<br />
- Application examples for (medical) image segmentation:<br />
-- Interactive max flow graph cuts<br />
-- Regularization of probabilistic label maps as in atlas-based segmentation<br />
-- High-performance multi-phase levelsets<br />
-- Post-processing of flawed manual segmentations with contrast sensitive regularization<br />
-- L1 intensity segmentation<br />
<br />
- Resources:<br />
<a href="http://www.advancedsegmentationtools.org" target="_new">http://www.advancedsegmentationtools.org</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=923&screenshot_id=922"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=923&screenshot_id=922" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 23, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://asets.github.io/asetsMatlabMaxFlow/">http://asets.github.io/asetsMatlabMaxFlow/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Imperial College London, UK & Robarts Research Institute, CAN <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="afni" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="AFNI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=afni" rel="bookmark" name="afni">AFNI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=haemodynamic-response" rel="tag">Haemodynamic Response</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-display" rel="tag">Image Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=nonlinear" rel="tag">Nonlinear</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=region-of-interest" rel="tag">Region Of Interest</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=registration" rel="tag">Registration</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=rendering" rel="tag">Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=resampling" rel="tag">Resampling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=response-monitoring" rel="tag">Response Monitoring</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-analysis" rel="tag">Surface Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=temporal-convolution---deconvolution" rel="tag">Temporal Convolution - Deconvolution</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tensor-metric" rel="tag">Tensor Metric</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volumetric-analysis" rel="tag">Volumetric Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> AFNI is a set of C programs for processing, analyzing, and displaying FMRI data. It runs on Unix+X11+Motif systems, including SGI, Solaris, Linux, and Mac OS X. It is available free for research purposes. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=23&screenshot_id=98"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=23&screenshot_id=98" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 7, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://afni.nimh.nih.gov/afni/">http://afni.nimh.nih.gov/afni/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>AFNI BRIK, ANALYZE, DICOM, GIfTI, MINC, NIfTI-1, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>AFNI BRIK, ANALYZE, DICOM, GIfTI, MINC, NIfTI-1, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="age-ility" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="age-ility">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=age-ility" rel="bookmark" name="age-ility">age-ility</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This data set consists of 136 subjects of which the following neuroimage data is acquired:<br />
MRI (3T):<br />
- T1 anatomical scan<br />
- DWI scan (two different types of scan paramaters)<br />
- Resting state fMRI<br />
<br />
EEG (64 channels):<br />
- Resting state EEG<br />
<br />
All data have been quality checked. See for more information the following publication:<br />
Karayanidis, F., Keuken, M.C., Wong, S.A., Rennie, J.L., de Hollander, G., Cooper, P.S., Fulham, W.R., Lenroot, R., Parsons, M.W., Philips, N., Michie, P.T., Forstmann, B.U. (2015). The Age-ility Project (Phase 1): Structural and functional imaging and electrophysiological data repository. Neuroimage. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 22, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial Share Alike</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/age-ility/">http://www.nitrc.org/projects/age-ility/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Newcastle <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="age-related-atrophy-dataset" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Age Related Atrophy Dataset">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=age-related-atrophy-dataset" rel="bookmark" name="age-related-atrophy-dataset">Age Related Atrophy Dataset</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Age Related Atrophy dataset contains structural MR images of 70 subjects collected during 2008-2010 across a wide range of ages. The dataset also contains resting state fMRI for most subjects. The structural images are T1 weighted, T2 weighted-FLAIR, 25 direction DTI, and the T1 mapping DESPOT [1] sequence. Reconstructed T1 maps for each subject are also available.<br />
<br />
The aquisition protocol was designed to study structural differences between young and older adults including both shape and intensity changes. Anonymized DICOM image sessions and processed images for each subject are available. The data is licensed under the Creative Commons Attribution License. It may be used freely for commercial, academic, or other use, as long as the original source is properly cited.<br />
<br />
<a href="http://www.bsl.ece.vt.edu/index.php?page=ara-dataset" target="_new">http://www.bsl.ece.vt.edu/index.php?page...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 13, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.bsl.ece.vt.edu/index.php?page=ara-dataset">http://www.bsl.ece.vt.edu/index.php?page=ara-dataset</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Virginia Polytechnic Institute and State University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="ahead" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="AHEAD">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=ahead" rel="bookmark" name="ahead">AHEAD</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=anatomic" rel="tag">Anatomic</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volume-measurement" rel="tag">Volume Measurement</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Automatic Hippocampal Estimator using Atlas-based Delineation (AHEAD) is an open-source turnkey software for automatic hippocampus segmentation. Its primary use is for delineating hippocampus in T1-weighted MRI images. AHEAD is developed by Jung W. Suh, Hongzhi Wang, Sandhitsu Das, Brian Avants, Philip Cook, John Pluta and Paul Yushkevich, and colleagues at the Penn Image Computing and Science Laboratory (PICSL) at the University of Pennsylvania. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 24, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/ahead/">http://www.nitrc.org/projects/ahead/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Cygwin<br>Linux</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="a-joint-sparse-partial-correlation-method-for-estimating-group-functional-networks" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="A joint sparse partial correlation method for estimating group functional networks">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=a-joint-sparse-partial-correlation-method-for-estimating-group-functional-networks" rel="bookmark" name="a-joint-sparse-partial-correlation-method-for-estimating-group-functional-networks">A joint sparse partial correlation method for estimating group functional networks</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Joint graphical models combined with stability selection (JGMSS)is a software toolbox that can be employed to robustly estimate both individual- and group-level sparse networks. The software:<br />
 (1) Reads time series from a group of subjects;<br />
 (2) Subsample data 100 times to estimate stability (i.e. stability <br />
 selection);<br />
 (3) Group graphical-lasso constraints are applied, including 2 <br />
 regularization parameters;<br />
 (4) Ranges of regularization parameters are chosen to implement stability <br />
 selection;<br />
 (5) The alternating direction method of multipliers (ADMM) approach is <br />
 employed to solve the problem.<br />
<br />
The Method is described in the following paper:<br />
Xiaoyun Liang, Alan Connelly, Fernando Calamante. A novel joint sparse partial correlation method for estimating group functional networks. Human Brain Mapping 12/2015; DOI:10.1002/hbm.23092. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=995&screenshot_id=965"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=995&screenshot_id=965" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 21, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/jgmss/">http://www.nitrc.org/projects/jgmss/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Florey Institute of Neuroscience and Mental Health <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="a-label-fusion-using-crf" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="A label fusion using CRF">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=a-label-fusion-using-crf" rel="bookmark" name="a-label-fusion-using-crf">A label fusion using CRF</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> We present a label fusion method based on minimizing an energy function by using graph-cut techniques. We use a conditional random field (CRF) model that allow us to incorporate shape, appearance and context information efficiently. This model is characterized by a pseudo Boolean function defined on unary, pairwise and higher order potentials. To evaluate the performance and the robustness of the proposed label fusion method, we employ two available database of T1-weighted (T1W) magnetic resonance (MR) images of human brains. We compare our approach with other label fusion methods in the hippocampal automatic segmentation from T1W-MR images. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 8, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/lf_crf/">http://www.nitrc.org/projects/lf_crf/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Applied Bioengineering Group <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="allen-brain-atlas:-data-portal" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Allen Brain Atlas: Data Portal">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=allen-brain-atlas:-data-portal" rel="bookmark" name="allen-brain-atlas:-data-portal">Allen Brain Atlas: Data Portal</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-data" rel="tag">Atlas Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=web-service" rel="tag">Web Service</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Allen Institute aims to answer some of the biggest questions in neuroscience and accelerate research worldwide through public releases of new data, knowledge and tools. Beginning with our Allen Mouse Brain Atlas—a molecular map showing where all genes are expressed in all regions of the mouse brain—the Allen Institute has produced a collection of open science resources that give users a powerful way to explore gene expression data, neural connections, single cell characterization and neuroanatomy. All of our resources are openly accessible via the Allen Brain Atlas data portal at brain-map.org.<br />
<br />
Resources include: ALLEN BRAIN OBSERVATORY, CELL TYPES DATABASE, MOUSE BRAIN CONNECTIVITY ATLAS, HUMAN BRAIN ATLAS, BRAINSPAN ATLAS OF THE DEVELOPING HUMAN BRAIN, MOUSE BRAIN ATLAS, DEVELOPING MOUSE BRAIN ATLAS, NIH BLUEPRINT NON-HUMAN PRIMATE (NHP) ATLAS, AGING, DEMENTIA AND TBI STUDY, IVY GLIOBLASTOMA ATLAS PROJECT, SPINAL CORD ATLAS, BRAIN ATLAS API and SOFTWARE DEVELOPMENT KIT. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1240&screenshot_id=1153"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1240&screenshot_id=1153" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 8, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.brain-map.org/">http://www.brain-map.org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Allen Institute for Brain Science <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="altered-states-database" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Altered States Database">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=altered-states-database" rel="bookmark" name="altered-states-database">Altered States Database</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data-model" rel="tag">Data Model</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=journal" rel="tag">Journal</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=ontology" rel="tag">Ontology</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other" rel="tag">Other</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=schema" rel="tag">Schema</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> A variety of pharmacological and non-pharmacological methods were reported to induce consciousness alterations in humans. Within psychological experiments, the subjective experiences of ASCs are typically quantified with retrospective questionnaires. Here, we introduce a database, termed the Altered States Database (ASDB), comprised of questionnaire data extracted from original research articles. The database contains data from articles published in MEDLINE-listed journals from experimentally induced altered states that were assessed with a specified set of standardized questionnaires. The dataset at hand will allow direct comparisons of the psychological effects of different induction methods as well as meta-analyses to establish induction method specific dose-response relationships. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 20, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.asdb.info">http://www.asdb.info</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="alvin---lateral-ventricle-segmentation" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ALVIN - Lateral Ventricle Segmentation">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=alvin---lateral-ventricle-segmentation" rel="bookmark" name="alvin---lateral-ventricle-segmentation">ALVIN - Lateral Ventricle Segmentation</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=region-of-interest" rel="tag">Region Of Interest</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volume-measurement" rel="tag">Volume Measurement</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volumetric-analysis" rel="tag">Volumetric Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ALVIN - Automatic Lateral Ventricle delIneatioN is a fully automated algorithm which works within SPM8 to segment the lateral ventricles from structural MRI images. The algorithm has been validated in infants, adults and patients with Alzheimer's disease (ICC&gt;0.95). ALVIN is insensitive to different scanner sequences (ICC&gt;0.99, 8 different sequences 1.5T and 3T) and sensitive to changes in ventricular volume. Processing time is approx 10mins per subject. Download from here or visit our main website <a href="http://sites.google.com/site/mrilateralventricle/" target="_new">http://sites.google.com/site/mrilateralv...</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=547&screenshot_id=457"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=547&screenshot_id=457" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 14, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://sites.google.com/site/mrilateralventricle/">http://sites.google.com/site/mrilateralventricle/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="amilab" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="AMILab">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=amilab" rel="bookmark" name="amilab">AMILab</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=active-contour" rel="tag">Active Contour</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=animation" rel="tag">Animation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=artifact-removal" rel="tag">Artifact Removal</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=filtering" rel="tag">Filtering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=format-conversion" rel="tag">Format Conversion</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=fourier-transform" rel="tag">Fourier Transform</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=intensity-contour" rel="tag">Intensity Contour</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=medial-axis" rel="tag">Medial Axis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=motion-analysis" rel="tag">Motion Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quality-metrics" rel="tag">Quality Metrics</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=spline-interpolation" rel="tag">Spline Interpolation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-rendering" rel="tag">Surface Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=two-dimensional-display" rel="tag">Two Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volume-rendering" rel="tag">Volume Rendering</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> AMILab is an opensource software for image analysis, processing and visualization. It provides convenient visualization tools for 2D and 3D images and it is highly extensible through its own scripting language. At visualization level, AMILab includes a 2D/3D image viewer, a 3D polygon viewer based on OpenGL, a 2D Curve viewer to visualize 2D curves, histograms and color/opacity transfer functions, and a GPU-enabled raycasting script for Volume Rendering based on VTK. The software includes an automatic C++ wrapping system which permits fast development of new visualization tools and image processing algorithms. This wrapping system currently wraps about 200 classes from wxwidgets library and about 100 classes from VTK. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=543&screenshot_id=455"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=543&screenshot_id=455" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 6, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Lesser General Public License (LGPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://amilab.org">http://amilab.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Las Palmas de Gran Canaria <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, MINC, Nrrd, Other Format, bshort/bfloat</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, MINC, Nrrd, Other Format, bshort/bfloat</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="analyze" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Analyze">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=analyze" rel="bookmark" name="analyze">Analyze</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=manual" rel="tag">Manual</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quantification" rel="tag">Quantification</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=region-growing" rel="tag">Region Growing</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=region-of-interest" rel="tag">Region Of Interest</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=registration" rel="tag">Registration</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=resampling" rel="tag">Resampling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=spectral-analysis" rel="tag">Spectral Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-rendering" rel="tag">Surface Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=three-dimensional-display" rel="tag">Three Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tractography" rel="tag">Tractography</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=two-dimensional-display" rel="tag">Two Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volume-rendering" rel="tag">Volume Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volumetric-analysis" rel="tag">Volumetric Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Analyze provides an integrated, comprehensive set of tools for the visualization, segmentation, registration and analysis of multidimensional biomedical imaging data. With Analyze, both anatomic structure and associated function can be studied and fused together for synergistic display and measurement of important structure-to-function relationships. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=291&screenshot_id=566"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=291&screenshot_id=566" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 6, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Other/Proprietary License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.analyzedirect.com/">http://www.analyzedirect.com/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>AnalyzeDirect <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>SunOS/Solaris<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, Interfile, NIfTI-1, Other Format, VRML</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, Interfile, NIfTI-1, Other Format, VRML</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="analyzer-2" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Analyzer 2">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=analyzer-2" rel="bookmark" name="analyzer-2">Analyzer 2</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=artifact-removal" rel="tag">Artifact Removal</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=connectivity-analysis" rel="tag">Connectivity Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=event-related-potential" rel="tag">Event Related Potential</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=independent-component-analysis" rel="tag">Independent Component Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=source-separation-analysis" rel="tag">Source Separation Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=temporal-transformation" rel="tag">Temporal Transformation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=three-dimensional-display" rel="tag">Three Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=two-dimensional-display" rel="tag">Two Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=wavelet-transformation" rel="tag">Wavelet Transformation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Since 1997, BrainVision Analyzer has made its way into thousands of research labs, helping scientists to manage their daily work of analyzing various neurophysiological data. <br />
<br />
Thanks to its ease of use, the handling of an amazing number of powerful features is just a snap. The editable History Tree® lets you organize, explore and trace evaluation steps. <br />
<br />
Automated analysis with drag &amp; drop functions speeds up recurring tasks. <br />
<br />
Various data format readers recognize files from different EEG manufacturers automatically and guarantee compatibility &amp; exchangeability with other research labs. <br />
<br />
Analyzer includes all necessary pre-processing functions, enhanced time-frequency analysis options, ICA, LORETA, MRI correction and a direct interface to MATLAB®. <br />
<br />
Various views create unlimited output possibilities to publish your results. <br />
<br />
PLEASE NOTE: Running Analyzer requires a dongle that has to be purchased through Brain Products or one of its distributors (<a href="http://www.brainproducts.com/distributors.php" target="_new">http://www.brainproducts.com/distributor...</a>). 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=570&screenshot_id=1199"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=570&screenshot_id=1199" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 6, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Commercial License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://www.brainproducts.com">https://www.brainproducts.com</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Brain Products GmbH <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ASCII, Brain Vision Analyzer Data Exchange Format, CNT, EDF/BDF/GDF, EGI, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ASCII, Brain Vision Analyzer Data Exchange Format, CNT, EDF/BDF/GDF, EGI, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="anatomist" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Anatomist">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=anatomist" rel="bookmark" name="anatomist">Anatomist</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Anatomist is a software for interactive visualization of multimodal data and for manipulation of structured 3D objects. It allows to build scenes that merge or combine images, meshes, regions of interest, fibers, textures, color palettes, referential changes, etc. A user can interact in 3D and in real time with the objects of an Anatomist scene: change point of view, select objects, add/supress objects, change colors, draw regions of interests, do manual registration, etc. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 6, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>CeCill v2</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://brainvisa.info/">http://brainvisa.info/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, MINC, NIfTI-1, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, MINC, NIfTI-1, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="anima" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ANIMA">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=anima" rel="bookmark" name="anima">ANIMA</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=web-service" rel="tag">Web Service</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=websites" rel="tag">Websites</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ANIMA (the Archive of Neuroimaging Meta-analyses) is an online repository of meta-analytic neuroimaging results, using methods such as activation likelihood estimation and multi-kernel density analysis. The ANIMA platform consists of an intuitive online interface for querying, downloading, and contributing data from published meta-analytic studies. Additionally, to aid the process of organizing, visualizing, and working with these data, we present an open-source desktop application called Volume Viewer. Volume Viewer allows users to easily arrange imaging data into composite stacks, and save these sessions as individual files, which can also be uploaded to the ANIMA database. The application also allows users to perform basic functions, such as computing conjunctions between images, or extracting regions-of-interest or peak coordinates for further analysis. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=985&screenshot_id=955"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=985&screenshot_id=955" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 28, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>ANIMA Data Usage and Contributor Agreement</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://anima.fz-juelich.de">http://anima.fz-juelich.de</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Institute for Neuroscience and Medicine (INM-1) <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="anima-scripts" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Anima scripts">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=anima-scripts" rel="bookmark" name="anima-scripts">Anima scripts</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Anima scripts is a set of scripts that use the Anima software to perform complex preprocessing and core processing tasks on medical images such as diffusion imaging, atlasing, brain extraction. This repository is multi-platform for most scripts (as long as python is available for your system). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 4, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Affero General Public License v3</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://github.com/Inria-Visages/Anima-Scripts-Public">https://github.com/Inria-Visages/Anima-Scripts-Public</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Inria <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="annovar:-functional-annotation-of-genetic-variants-from-high-throughput-sequencing-data" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ANNOVAR: Functional annotation of genetic variants from high-throughput sequencing data">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=annovar:-functional-annotation-of-genetic-variants-from-high-throughput-sequencing-data" rel="bookmark" name="annovar:-functional-annotation-of-genetic-variants-from-high-throughput-sequencing-data">ANNOVAR: Functional annotation of genetic variants from high-throughput sequencing data</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=next-generation-sequencing" rel="tag">Next Generation Sequencing</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=snp's-&-genes" rel="tag">Snp'S & Genes</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ANNOVAR is an efficient software tool to utilize update-to-date information to functionally annotate genetic variants detected from diverse genomes (including human genome hg18, hg19, as well as mouse, worm, fly, yeast and many others). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 1, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://annovar.openbioinformatics.org/">http://annovar.openbioinformatics.org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Southern California <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="a-novel-group-fused-sparse-partial-correlation-method-for-simultaneous-estimation-of-functional-networks-in-group-comparisons" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="A novel group-fused sparse partial correlation method for simultaneous estimation of functional networks in group comparisons">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=a-novel-group-fused-sparse-partial-correlation-method-for-simultaneous-estimation-of-functional-networks-in-group-comparisons" rel="bookmark" name="a-novel-group-fused-sparse-partial-correlation-method-for-simultaneous-estimation-of-functional-networks-in-group-comparisons">A novel group-fused sparse partial correlation method for simultaneous estimation of functional networks in group comparisons</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Group-fused multiple-graphical lasso cobined with stability selection (GMGLASS) is a software toolbox that can be employed to simultaneously estimate both individual- and group-level functional networks from 2 groups.<br />
The software:<br />
 (1) Reads time series from 2 groups of subjects;<br />
 (2) Randomly subsamples data 100 times to estimate stability with stability <br />
 selection;<br />
 (3) Group-fused multiple-graphical lasso is applied with two hyper-parameters: alpha and beta;<br />
 (4) Appropriate ranges of hyper-parameters are chosen for achieving stability selection;<br />
 (5) Both individual- and group-level functional networks can be estimated.<br />
<br />
 The method is described in the following paper:<br />
 Xiaoyun Liang, David N. Vaughan, Alan Connelly, Fernando Calamante. A novel group-fused sparse partial correlation method for simultaneous estimation of functional networks.in group comparison studies. Brain Topography, 12/2017; DOI:10.1007/s10548-017-0615-6. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 8, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/gmglass/">http://www.nitrc.org/projects/gmglass/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Florey Institute of Neuroscience and Mental Health <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="a-novel-method-for-extracting-hierarchical-functional-subnetworks-based-on-a-multi-subject-spectral-clustering-approach" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="A novel method for extracting hierarchical functional subnetworks based on a multi-subject spectral clustering approach">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=a-novel-method-for-extracting-hierarchical-functional-subnetworks-based-on-a-multi-subject-spectral-clustering-approach" rel="bookmark" name="a-novel-method-for-extracting-hierarchical-functional-subnetworks-based-on-a-multi-subject-spectral-clustering-approach">A novel method for extracting hierarchical functional subnetworks based on a multi-subject spectral clustering approach</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Group-level Network Hierarchical Clustering (GNetHiClus) is a software toolbox that can be used to extract hierarchical brain network modules at group level. The software: (1) reads connectivity matrices from a group of subjects; (2) subsamples data to obtain subgroups; (3) extracts hierarchical brain subnetworks; (4) uses bootstrap method to estimate the most reliable subnetworks.<br />
The method is described in the following paper:<br />
Xiaoyun Liang, Chun-Hung Yeh, Alan Connelly, Fernando Calamante. A novel method for extracting hierarchical functional subnetworks based on a multi-subject spectral clustering approach. Brain Connectivity, 03/2019; DOI: 10.1089/brain.2019.0668. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1362&screenshot_id=1261"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1362&screenshot_id=1261" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 9</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/gnethiclus/">http://www.nitrc.org/projects/gnethiclus/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Florey Institute of Neuroscience and Mental Health <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="anticorrelated-component-analysis" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Anticorrelated Component Analysis">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=anticorrelated-component-analysis" rel="bookmark" name="anticorrelated-component-analysis">Anticorrelated Component Analysis</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MATLAB code to compute the two most anticorrelated patterns in a multivariate dataset. <br />
<br />
This technique was first applied to identify competing neuronal networks from the spontaneous activity of brain neural circuits in:<br />
<br />
Nathan X. Kodama, Tianyi Feng, James Ullett, Hillel J. Chiel, Siddharth S. Sivakumar, and Roberto F. Galán (2018). Anti-correlated cortical networks arise from spontaneous neuronal dynamics at slow timescales. Scientific Reports, 8:666.<br />
<br />
<a href="https://www.nature.com/articles/s41598-017-18097-0" target="_new">https://www.nature.com/articles/s41598-0...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 13, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://www.mathworks.com/matlabcentral/fileexchange/64555-anticorrelated-component-analysis-aca">https://www.mathworks.com/matlabcentral/fileexchange/64555-anticorrelated-component-analysis-aca</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="antsr" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ANTsR">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=antsr" rel="bookmark" name="antsr">ANTsR</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ANTsR is an R extension to ANTs that performs multivariate statistical parametric mapping of DTI, T1 and other datatypes for the purpose of both performing clinical studies and for tracking the performance of ANTs (and other) image processing methodologies. ANTsR depends upon the R statistical language, bash scripts and the ANTs toolkit. Some branches of ANTsR will also depend upon pipedream and specific datasets. Some of these datasets will be open access and, in that case, ANTsR will provide a 100% reproducible neuroimaging study on that data. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 27, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/antsr/">http://www.nitrc.org/projects/antsr/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="aperture" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="APERTURE">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=aperture" rel="bookmark" name="aperture">APERTURE</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=event-related-potential" rel="tag">Event Related Potential</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-display" rel="tag">Image Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=multivariate-analysis" rel="tag">Multivariate Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=spectral-analysis" rel="tag">Spectral Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> APERTURE (Analysis of Patterns in Electrophysiological Recordings Toolbox with Utilities for REsearch) is a MATLAB-based toolbox for analysis of EEG, MEG, and ECoG data. It is designed to organize entire analysis pipelines with large datasets, from preprocessing to hypothesis testing and figure generation. APERTURE allows flexible multivariate analysis of ERPs and oscillatory activity. It also supports mass-univariate analysis with advanced statistical tests. Computations are accelerated using parallel computing supported through the MATLAB distributed computing toolbox. Examination of large, high-dimensional datasets is made simple through data visualization tools, including advanced plotting routines and generation of PDF reports with many figures. APERTURE is designed to be flexible and highly extensible, to allow the user to easily implement new analysis protocols. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 20, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Lesser General Public License (LGPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/aperture/">http://www.nitrc.org/projects/aperture/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Vanderbilt University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS<br>POSIX/UNIX-like<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="a-probabilistic-subcortical-atlas-for-pd" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="A probabilistic subcortical atlas for PD">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=a-probabilistic-subcortical-atlas-for-pd" rel="bookmark" name="a-probabilistic-subcortical-atlas-for-pd">A probabilistic subcortical atlas for PD</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This atlas takes advantage of ultra-high resolution 7T MRI to provide unprecedented levels of detail on structures of the basal ganglia in-vivo. The atlas includes a disease-specific probability map of the subthalamic Nucleus based on Parkinson's Disease patients.<br />
<br />
When using these masks please cite: Anneke Alkemade, Gilles de Hollander, Max C Keuken, Andreas Schäfer, Derek VM Ott, Johannes Schwarz, David Weise, Sonja A Kotz, Birte U Forstmann (2017). Comparison of T2* and QSM contrasts in Parkinson’s disease to visualize the STN with MRI. PLOS one </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 17, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial Share Alike</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/atag_pd/">http://www.nitrc.org/projects/atag_pd/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UvA / MPI CBS <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="arctic" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ARCTIC">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=arctic" rel="bookmark" name="arctic">ARCTIC</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ARCTIC (Automatic Regional Cortical ThICkness) is an end-to-end application developped at UNC-Chapel Hill allowing individual regional analysis of cortical thickness. This cross-platform tool can be run within Slicer3 as an external module, or directly as a command line. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=186&screenshot_id=178"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=186&screenshot_id=178" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 10, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/arctic/">http://www.nitrc.org/projects/arctic/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina at Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, Nrrd, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, Nrrd, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="a-robust-method-for-correcting-partial-volume-effect-in-asl-data" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="A robust method for correcting partial volume effect in ASL data">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=a-robust-method-for-correcting-partial-volume-effect-in-asl-data" rel="bookmark" name="a-robust-method-for-correcting-partial-volume-effect-in-asl-data">A robust method for correcting partial volume effect in ASL data</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Partial volume correction using modified least trimmed squares (PVC-mLTS) is a software toolbox that can be employed to correct partial volume effect for single inversion-time ASL data.<br />
<br />
The software:<br />
(1) Reading and writing images in nifti format;<br />
(2) Implementing Modified least trimmed squares method (mLTS)<br />
(3) Applying mLTS to ASL data for correcting partial volume effect <br />
(3) Computing CBF maps for both GM and WM.<br />
<br />
The method is described in the following paper:<br />
Xiaoyun Liang, Alan Connelly, Fernando Calamante. Improved partial volume correction for single inversion time arterial spin labeling data. Magn Reson Med. 2013 Feb;69(2):531-7. doi: 10.1002/mrm.24279. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1044&screenshot_id=984"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1044&screenshot_id=984" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 25, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/pvc_mlts/">http://www.nitrc.org/projects/pvc_mlts/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Florey Institute of Neuroscience and Mental Health <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="arterial-territories-of-the-neonatal-brain:-automated-classification" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Arterial Territories of the Neonatal Brain: automated classification">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=arterial-territories-of-the-neonatal-brain:-automated-classification" rel="bookmark" name="arterial-territories-of-the-neonatal-brain:-automated-classification">Arterial Territories of the Neonatal Brain: automated classification</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This automated classification program is intended to facilitate a quick and precise classification of neonatal arterial ischemic strokes (NAIS) according to the arterial territories of the neonatal brain (ATNB). Precise stroke location, as well as the percentage of each arterial territory involved, are the main calculations performed by this script. An anatomical image of the subject(s), along with their lesion segmentation, are required. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 29, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/atnb_classif/">http://www.nitrc.org/projects/atnb_classif/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="artifact-detection-tools-(art)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Artifact Detection Tools (ART)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=artifact-detection-tools-(art)" rel="bookmark" name="artifact-detection-tools-(art)">Artifact Detection Tools (ART)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=artifact-removal" rel="tag">Artifact Removal</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=motion-analysis" rel="tag">Motion Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quality-metrics" rel="tag">Quality Metrics</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=registration" rel="tag">Registration</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Toolbox for post-processing fMRI data. Includes software for comprehensive analysis of sources of artifacts in timeseries data including spiking and motion. Most compatible with SPM processing, but adaptable for FSL as well. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=104&screenshot_id=362"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=104&screenshot_id=362" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 30, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://web.mit.edu/swg/software.htm">http://web.mit.edu/swg/software.htm</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>MIT <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="artrepair-for-robust-fmri" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ArtRepair for robust fMRI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=artrepair-for-robust-fmri" rel="bookmark" name="artrepair-for-robust-fmri">ArtRepair for robust fMRI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=artifact-removal" rel="tag">Artifact Removal</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-display" rel="tag">Image Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=motion-analysis" rel="tag">Motion Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quality-metrics" rel="tag">Quality Metrics</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ArtRepair is a toolbox for SPM to improve fMRI analysis of high motion pediatric and clinical subjects. The toolbox includes special algorithms for motion adjustment, data repair, and noise filtering, and methods to find outlier subjects in group studies. Visualization tools are included for quality checking the data, including a movie format for viewing all data and all contrast estimates on every voxel of every subject. Methods are included to quantify results into percent signal change. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 15, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://cibsr.stanford.edu/tools/human-brain-project/artrepair-software.html">http://cibsr.stanford.edu/tools/human-brain-project/artrepair-software.html</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Stanford University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="asa---advanced-source-analysis" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ASA - Advanced Source Analysis">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=asa---advanced-source-analysis" rel="bookmark" name="asa---advanced-source-analysis">ASA - Advanced Source Analysis</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=artifact-removal" rel="tag">Artifact Removal</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=eeg-modeling" rel="tag">Eeg Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=event-related-potential" rel="tag">Event Related Potential</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=fourier-transform" rel="tag">Fourier Transform</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=meg-modeling" rel="tag">Meg Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=wavelet-transformation" rel="tag">Wavelet Transformation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ASA is a highly flexible EEG/ERP and MEG analysis package with a variety of source reconstruction, signal analysis and MRI processing features. ASA combines functional brain imaging with the visualization and incorporation of morphological information obtained from MRI or CT. ASA is a highly interactive and flexible software tool that can be applied to neuro-physiological and clinical brain research.<br />
<br />
ASA gives a realistic impression of your experimental configuration together with topographical mapping of EEG and MEG and the results of your analysis. ASA is developed for and by people dedicated to brain research. The concept of flexibility and openness covers even most complex analysis demands. The ASA environment is particularly attractive for those that wish to develop their own methods in third party packages like Matlab and use ASA for pre-processing and visualization purposes. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=554&screenshot_id=636"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=554&screenshot_id=636" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 23, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Commercial License: ASA Software</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/asa/">http://www.nitrc.org/projects/asa/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>ANT Neuro B.V. <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Microsoft</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, ANT, ASCII, Brain Vision Analyzer Data Exchange Format, CNT, CTF, DICOM, EDF/BDF/GDF, EGI, FIF, M4D</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, ANT, ASCII, Brain Vision Analyzer Data Exchange Format, CNT, CTF, DICOM, EDF/BDF/GDF, EGI, FIF, M4D</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="a-shimadzu-to-.nirs-data-converter" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="A Shimadzu to .nirs data converter">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=a-shimadzu-to-.nirs-data-converter" rel="bookmark" name="a-shimadzu-to-.nirs-data-converter">A Shimadzu to .nirs data converter</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Shimadzu2nirs is a Matlab script which takes near-infrared spectroscopy data recorded by Shimadzu system(s) and converts it to a .nirs file format for use with the HOMER2 NIRS processing package. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 14, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/shimadzu2nirs/">http://www.nitrc.org/projects/shimadzu2nirs/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Massachusetts General Hospital <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="ashs:-automatic-segmentation-of-hippocampal-subfields" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ASHS: Automatic Segmentation of Hippocampal Subfields">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=ashs:-automatic-segmentation-of-hippocampal-subfields" rel="bookmark" name="ashs:-automatic-segmentation-of-hippocampal-subfields">ASHS: Automatic Segmentation of Hippocampal Subfields</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ASHS is a software package for automatic segmentation of hippocampal subfields in MRI scans. ASHS requires a T1-weighted scan and an oblique coronal T2-weighted scan with high in-plane resolution. ASHS will automatically label main hippocampal subfields and medial temporal lobe subregions. ASHS is easy to train for other brain segmentation applications as well.<br />
<br />
For details, see the ASHS documentation: <a href="https://sites.google.com/site/hipposubfields" target="_new">https://sites.google.com/site/hipposubfi...</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=370&screenshot_id=256"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=370&screenshot_id=256" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 5, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://sites.google.com/site/hipposubfields/">http://sites.google.com/site/hipposubfields/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pennsylvania Perelman School of Medicine <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="asl-data-processing-tool-box" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ASL data processing tool box">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=asl-data-processing-tool-box" rel="bookmark" name="asl-data-processing-tool-box">ASL data processing tool box</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quantification" rel="tag">Quantification</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=statistical-operation" rel="tag">Statistical Operation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This tool box is for arterial spin labeled perfusion MRI data processing. It is based on SPM (copyright by Wellcome Department, London) and Matlab (MathWorks Inc). More detailed documentation can be found in the manual. The current version support 3D or 4D Analyze or Nifiti format and support PASL, CASL, and PCASL data. It contains the code for calculating CBF and a set of SPM batch scripts for preprocessing and statistical analysis. Sample data are also available from the webpage: <a href="http://cfn.upenn.edu/~zewang/ASLtbx.php." target="_new">http://cfn.upenn.edu/~zewang/ASLtbx.php.</a> Please subscribe to the maillist here: <a href="http://www.nitrc.org/mailman/listinfo/asltbx-maillist." target="_new">http://www.nitrc.org/mailman/listinfo/as...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 14, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/asltbx/">http://www.nitrc.org/projects/asltbx/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="asl_spm8" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ASL_spm8">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=asl_spm8" rel="bookmark" name="asl_spm8">ASL_spm8</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Quick ASL Wrapper for preprocessing ASL Data and computing blood flow measurements using UPenn ASL toolbox. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 12, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/asl_spm8/">http://www.nitrc.org/projects/asl_spm8/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>MGH/HMS <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="atlas3d" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Atlas3D">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=atlas3d" rel="bookmark" name="atlas3d">Atlas3D</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Multiplatform 3D visualization tool which allows import, linear transformation, cutting and export of various data formats in 3D atlas space. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 17, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/atlas3d/">http://www.nitrc.org/projects/atlas3d/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Oslo <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="atlasconstructiontoolkit" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="AtlasConstructionToolkit">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=atlasconstructiontoolkit" rel="bookmark" name="atlasconstructiontoolkit">AtlasConstructionToolkit</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Atlas Construction Toolkit Using Sparse Representation<br />
<br />
Please refer to below papers for details:<br />
<br />
&quot;Infant Brain Atlases from Neonates to 1- and 2-year-olds&quot;, PLoS ONE, 6(4): e18746, Apr. 2011. [Feng Shi, Pew-Thian Yap, Guorong Wu, Hongjun Jia, John H. Gilmore, WeiliLin, Dinggang Shen]<br />
<br />
&quot;Neonatal Atlas Construction Using Sparse Representation”, Human Brain Mapping 35(9):4663–4677, 2014. [Feng Shi, Li Wang, Guorong Wu, Gang Li, John H. Gilmore, Weili Lin, Dinggang Shen] </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 23, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>iBEAT</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/atlastoolkit/">http://www.nitrc.org/projects/atlastoolkit/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina at Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="atlases-of-amygdala-and-hippocampus-for-pediatric-populations" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Atlases of amygdala and hippocampus for pediatric populations">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=atlases-of-amygdala-and-hippocampus-for-pediatric-populations" rel="bookmark" name="atlases-of-amygdala-and-hippocampus-for-pediatric-populations">Atlases of amygdala and hippocampus for pediatric populations</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The long term goal of Computational Anatomy (CA) is to create algorithmic<br />
tools that aid basic and clinical neuroscientists in the analysis of<br />
variability in anatomical structures at different scales. <br />
The aim is to provide capability for integrating 3D Slicer application and ITK software library with the statistical shape analysis<br />
pipeline and thus enable the wider neuroimaging community to efficiently<br />
analyze anatomical variations in disease. We have constructed anatomical atlases needed for analysis of shape vectors. These atlases were generated from<br />
segmented hippocampal and amygdala structures in acquired populations of<br />
children, adolescents and young adults in neuroimaging studies of major<br />
depression disorder (MDD) at Washington University at St Louis. As a major<br />
public health burden, MDD provides the biological testbed for the pipeline<br />
from which probabilistic atlases will be generated. Software and datasets for statistical shape analysis of anatomical structures are thus provided. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 14, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BIRN Data License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/jhucis_pedatlas/">http://www.nitrc.org/projects/jhucis_pedatlas/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Johns Hopkins University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="atlasing-of-the-basal-ganglia" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Atlasing of the basal ganglia">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=atlasing-of-the-basal-ganglia" rel="bookmark" name="atlasing-of-the-basal-ganglia">Atlasing of the basal ganglia</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This atlas takes advantage of ultra-high resolution 7T MRI to provide unprecedented levels of detail on structures of the basal ganglia in-vivo. The ATAG atlas includes probability maps of the striatum, GPe, GPi, red nucleus, substantia nigra, subthalamic Nucleus(STh) and the PAG. The atlas has been created on 30 young (M:24.2), 14 middle-aged (M: 52.5), and 10 elderly (M: 69.6) participants.<br />
<br />
Separately a STh atlas was created based on 13 young (M:24.38), 8 middle-aged (M:50.67), and 9 elderly participants (M:72.33).<br />
<br />
You can find more details about the creation of these maps in the following papers:<br />
<br />
ATAG young atlas: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24650599" target="_new">http://www.ncbi.nlm.nih.gov/pubmed/24650...</a><br />
<br />
ATAG middle aged, elderly: <a href="http://www.ncbi.nlm.nih.gov/pubmed/28168364" target="_new">http://www.ncbi.nlm.nih.gov/pubmed/28168...</a><br />
<br />
STh young: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22227131" target="_new">http://www.ncbi.nlm.nih.gov/pubmed/22227...</a><br />
<br />
STh Middle-aged &amp; Elderly: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23486960" target="_new">http://www.ncbi.nlm.nih.gov/pubmed/23486...</a><br />
<br />
Participating institutions are the Max Planck Institute for Human Cognitive and Brain Sciences, Germany, and the University of Amsterdam, the Netherlands. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=653&screenshot_id=741"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=653&screenshot_id=741" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 2, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/atag/">http://www.nitrc.org/projects/atag/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Amsterdam <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="atpp:-automatic-tractography-based-parcellation-pipeline" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ATPP: Automatic Tractography-based Parcellation Pipeline">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=atpp:-automatic-tractography-based-parcellation-pipeline" rel="bookmark" name="atpp:-automatic-tractography-based-parcellation-pipeline">ATPP: Automatic Tractography-based Parcellation Pipeline</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ATPP (Automatic Tractography-based Parcellation Pipeline) is an integrated pipeline named ATPP realizing tractography-based brain parcellation with automatic processing and massive parallel computing. <br />
<br />
ATPP offers a powerful CLI version (<a href="https://github.com/haililihai/ATPP_CLI" target="_new">https://github.com/haililihai/ATPP_CLI</a>) for parcellating multiple brain regions and a user-friendly GUI version (<a href="https://github.com/haililihai/ATPP_GUI" target="_new">https://github.com/haililihai/ATPP_GUI</a>) for parcellating a specific brain region. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1099&screenshot_id=1057"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1099&screenshot_id=1057" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 9, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License v3</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/atpp/">http://www.nitrc.org/projects/atpp/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Institute of Automation, Chinese Academy of Sciences <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="augmented-reality-mirror" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Augmented Reality Mirror">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=augmented-reality-mirror" rel="bookmark" name="augmented-reality-mirror">Augmented Reality Mirror</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Augmented Reality Mirror is an open source rendering software to provide users with a virtual reality experience. With the provisions of a projection display, a laptop with multi-core GPU, and XBOX kinect, users can stand in front of the mirror and representative anatomical images will overlay the body. The images implemented are adapted to approximate the actual anatomy. The open source software will have an application programming interface (API) to allow users to develop additional content within the Augmented Reality Mirror. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 21, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/ar_mirror/">http://www.nitrc.org/projects/ar_mirror/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>MASI <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="aura-tools-:-automated-retinal-analysis-tools" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="AURA tools : AUtomated Retinal Analysis tools">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=aura-tools-:-automated-retinal-analysis-tools" rel="bookmark" name="aura-tools-:-automated-retinal-analysis-tools">AURA tools : AUtomated Retinal Analysis tools</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The dissemination of software for the analysis of optical coherence tomography scans of the retina, including the macula. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 18, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU GPL v2</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/aura_tools/">http://www.nitrc.org/projects/aura_tools/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Image Analysis and Communications Laboratory <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="autism-tissue-program---atp" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Autism Tissue Program - ATP">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=autism-tissue-program---atp" rel="bookmark" name="autism-tissue-program---atp">Autism Tissue Program - ATP</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Autism Tissue Program (ATP) is a research program organized to provide post-mortem brain tissue to qualified neuroscientists all over the world. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 23, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/atp/">http://www.nitrc.org/projects/atp/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Autism Tissue Program <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="automated-image-registration-(air)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Automated Image Registration (AIR)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=automated-image-registration-(air)" rel="bookmark" name="automated-image-registration-(air)">Automated Image Registration (AIR)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Automated Image Registration (AIR) tool is used for alignment of 3D and 2D images within and across subjects and across imaging modalities. The AIR library can easily incorporate automated image registration into site specific programs adapted to your particular needs.<br />
<br />
AIR is distributed by the Laboratory of Neuro Imaging (<a href="http://loni.usc.edu/Software/AIR" target="_new">http://loni.usc.edu/Software/AIR</a>) at USC. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=301&screenshot_id=289"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=301&screenshot_id=289" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 8, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/air/">http://www.nitrc.org/projects/air/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Southern California <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="automatic-extra-axial-cerebrospinal-fluid-(auto-eacsf)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Automatic Extra-Axial Cerebrospinal Fluid (Auto EACSF)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=automatic-extra-axial-cerebrospinal-fluid-(auto-eacsf)" rel="bookmark" name="automatic-extra-axial-cerebrospinal-fluid-(auto-eacsf)">Automatic Extra-Axial Cerebrospinal Fluid (Auto EACSF)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Automatic Extra-Axial Cerebrospinal Fluid (Auto EACSF) is an open-source, interactive tool for automatic computation of brain extra-axial cerebrospinal fluid (EA-CSF) in magnetic resonance image (MRI) scans of infants. Auto EACSF aims to automatically calculate the volume of EA-CSF. Auto EACSF is a user-friendly application that generates a Qt application to calculate the volume of EA-CSF. The application is run through a GUI, but also provides an advanced use mode that allows execution of different steps by themselves via Python and XML scripts. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 2, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/auto_eacsf/">http://www.nitrc.org/projects/auto_eacsf/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>NIRAL, UNC Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="automatic-registration-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Automatic Registration Toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=automatic-registration-toolbox" rel="bookmark" name="automatic-registration-toolbox">Automatic Registration Toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=nonlinear-warp" rel="tag">Nonlinear Warp</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=registration" rel="tag">Registration</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ATRA software for fully automated, fast, reliable, unbiased and inverse-consistent registration of within-subject multiple T1-weighted structural MRI brain volumes.<br />
<br />
ART `acpcdetect' program for automatic detection of the AC and PC landmarks and the mid-sagittal plane on 3D structural MRI scans.<br />
<br />
ART `brainwash' program for automatic multi-atlas skull-stripping of 3D structural MRI scans.<br />
<br />
ART `3dwarper' program of non-linear inter-subject registration of 3D structural MRI scans.<br />
<br />
Software (art2) for linear rigid-body intra-subject inter-modality (MRI-PET) image registration.<br />
<br />
Data resource: The ART projects makes available corpus callosum segmentations of 316 normal subjects from the OASIS cross-sectional database<br />
<br />
ART `yuki' program for fast, robust, and fully automatic segmentation of the corpus callosum on 3D structural MRI scans.<br />
<br />
ART 'kaiba' program: estimates the hippocampal parenchymal fraction (HPF) from 3D T1W MRI. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=90&screenshot_id=398"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=90&screenshot_id=398" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 6, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/art/">http://www.nitrc.org/projects/art/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="automatic-segmentation-tool-adapter" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Automatic Segmentation Tool Adapter">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=automatic-segmentation-tool-adapter" rel="bookmark" name="automatic-segmentation-tool-adapter">Automatic Segmentation Tool Adapter</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> More and more automatic segmentation tools are publicly available to today's researchers. However, when applied by their end-users, these segmentation tools usually can not achieve the performance that the tool developer reported. Discrepancies between the tool developer and its users in manual segmentation protocols and imaging modalities are the main reasons for such inconsistency. <br />
<br />
In this project, we will provide an open source learning-based software that automatically learns how to transfer the output of a host segmentation tool closer to the user's manual segmentation using the image data and manual segmentation provided by the user. The motivation of this project is to bridge the gap between the segmentation tool developer and the tool users such that the existing segmentation tools can more effectively serve the community. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 12, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/segadapter/">http://www.nitrc.org/projects/segadapter/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>POSIX/UNIX-like</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="autoseg" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="AutoSeg">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=autoseg" rel="bookmark" name="autoseg">AutoSeg</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=expectation-minimization" rel="tag">Expectation Minimization</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-to-image" rel="tag">Image-To-Image</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-to-template" rel="tag">Image-To-Template</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=labeling" rel="tag">Labeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=nonlinear-warp" rel="tag">Nonlinear Warp</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=spline-interpolation" rel="tag">Spline Interpolation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volumetric-analysis" rel="tag">Volumetric Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> AutoSeg is a novel C++ based application developped at UNC-Chapel Hill that performs automatic brain tissue classification and structural segmentation. <br />
<br />
AutoSeg is designed for use with human and non-human primate pediatric, adolescent and adult data.<br />
<br />
AutoSeg uses a BatchMake pipeline script that includes the main steps of the framework entailing N4 bias field correction, rigid registration to a common coordinate image, tissue segmentation, skull-stripping, intensity rescaling, atlas-based registration, subcortical segmentation and lobar parcellation, regional cortical thickness and intensity statistics. AutoSeg allows efficient batch processing and grid computing to process large datasets and provides quality control visualizations via Slicer3 MRML scenes. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=421&screenshot_id=320"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=421&screenshot_id=320" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 22, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/autoseg/">http://www.nitrc.org/projects/autoseg/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina at Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, Nrrd, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, Nrrd, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="autotract" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="AutoTract">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=autotract" rel="bookmark" name="autotract">AutoTract</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=diffusion-mr-fiber-tracking" rel="tag">Diffusion Mr Fiber Tracking</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> AutoTract is an automatic tractography tool featuring advanced processing tools to clean tracts after the initial tractography. It uses the Slicer TractographyLabelMapSeeding, with tracts from a reference DTI atlas as label maps. <br />
<br />
The 3 main steps of the tool are:<br />
-Registration between the DTI image that we want to tract and the reference DTI atlas.<br />
-Tractography of the DTI image using the TractographyLabelMapSeeding Slicer module.<br />
-Process of each tract individually to clean and remove any unwanted fiber.<br />
<br />
The processing includes (for each tract): <br />
-cutting every fiber ends that are in the gray matter (cutting them until the fiber ends is in the white matter), based on a white matter mask created by the tool.<br />
-ensuring every fiber is not in the CSF (removes them from the tract otherwise)<br />
-several image comparisons with the reference tracts (such as fiber length matching, use of a distance map to ensure that the tract is close to the reference tract, and potentially to be added later on, shape comparison). 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=948&screenshot_id=935"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=948&screenshot_id=935" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 16, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/autotract/">http://www.nitrc.org/projects/autotract/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina at Chapel Hill (UNC-CH) <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Nrrd</td></tr>
			<tr class="even"><td>Output Formats</td><td>Nrrd</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="axonal-beading-and-microtubule-quantification" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Axonal Beading and Microtubule Quantification">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=axonal-beading-and-microtubule-quantification" rel="bookmark" name="axonal-beading-and-microtubule-quantification">Axonal Beading and Microtubule Quantification</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quantitative-shape-analysis" rel="tag">Quantitative Shape Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Interactive image analysis software for quantifying axonal beading morphology and focal microtubule loss using phase and fluorescent images of a neuron.<br />
<br />
Purpose: Focal axonal beading and focal disruption of microtubule structure are characteristic to traumatic axonal injury. We reproduced these morphological and structural changes in our in vitro model system [Kilinc, Gallo, Barbee, 2008. Exp. Neurol. 212:422–430]. In order to measure bead formation objectively, an observer-independent quantification of beading was necessary. In addition, a quantitative measure for the extent of co-localization of axonal beads and microtubule disruptions was required to establish a causal relationship between focal cytoskeletal damage and bead formation. In this paper we describe Matlab-based, interactive image analysis programs for axonal beading quantification and co-localization analysis. Injury-induced increases in the axonal beading could be successfully detected using the bead analysis program. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1052&screenshot_id=998"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1052&screenshot_id=998" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 30, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/axonal_beading/">http://www.nitrc.org/projects/axonal_beading/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Drexel University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="b0-and-eddy-current-correction-for-dti" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="B0 and eddy current correction for DTI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=b0-and-eddy-current-correction-for-dti" rel="bookmark" name="b0-and-eddy-current-correction-for-dti">B0 and eddy current correction for DTI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-software-resource" rel="tag">Other Software Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Software tool (excecutable and source code in C and C++) to correct distortions in diffusion MR images that are generated by main magnetic field inhomogeneities and eddy current induced fields generated from the direction-dependent diffusion encoding </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 23, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://nbirn.net/tools/bo_eddy/index.shtm">http://nbirn.net/tools/bo_eddy/index.shtm</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Microsoft</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="bash4rfmri" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BASH4RfMRI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=bash4rfmri" rel="bookmark" name="bash4rfmri">BASH4RfMRI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=correlation" rel="tag">Correlation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=information-theory" rel="tag">Information Theory</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=multivariate-analysis" rel="tag">Multivariate Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=platform-or-development-environment" rel="tag">Platform Or Development Environment</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=spectral-analysis" rel="tag">Spectral Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> BASH Scripts for a resting-state functional MRI study. The functions include seed-based correlation analysis, amplitude analysis and independent component analysis. Note: this tool is just a plug-in for FSL, AFNI and FreeSurfer. Thus, you need have them before you use BASH4RfMRI. Of note: most functions have been integrated in 1000 Functional Connectomes Project (www.nitrc.org/projects/fcon_1000). I do not update this package, thus please visit 1000 Functional Connectomes Project site and download relevant bash scripts. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 26, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/bash4rfmri/">http://www.nitrc.org/projects/bash4rfmri/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>POSIX/UNIX-like</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="bayesian-connectivity-analysis-software-(bca)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Bayesian Connectivity Analysis software (BCA)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=bayesian-connectivity-analysis-software-(bca)" rel="bookmark" name="bayesian-connectivity-analysis-software-(bca)">Bayesian Connectivity Analysis software (BCA)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=software" rel="tag">Software</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Bayesian Connectivity analysis software (BCA) is a pipeline for connectivity matrix analysis. A general user is a researcher who is interested in brain network analysis and has basic knowledge of Linux/Unix, R, and Matlab. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 9, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GPL-Style Open Source</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/bca/">http://www.nitrc.org/projects/bca/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Maryland Baltimore <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="bayesian-longitudinal-low-rank-regression" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Bayesian longitudinal low-rank regression">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=bayesian-longitudinal-low-rank-regression" rel="bookmark" name="bayesian-longitudinal-low-rank-regression">Bayesian longitudinal low-rank regression</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> To perform a joint analysis of multivariate neuroimaging phenotypes and candidate genetic markers obtained from longitudinal studies, we develop a Bayesian longitudinal low-rank regression (L2R2) model. The L2R2 model integrates three key methodologies: a low-rank matrix for approximating the high-dimensional regression coefficient matrices corresponding to the genetic main effects and their interactions with time, penalized splines for characterizing the overall time effect, and a sparse factor analysis model coupled with random effects for capturing within-subject spatio-temporal correlations of longitudinal phenotypes. Posterior computation proceeds via an efficient Markov chain Monte Carlo algorithm. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 28, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Affero General Public License v3</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/l2r2/">http://www.nitrc.org/projects/l2r2/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>MD Anderson Cancer Center <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="bci2000" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BCI2000">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=bci2000" rel="bookmark" name="bci2000">BCI2000</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=animation" rel="tag">Animation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=artifact-removal" rel="tag">Artifact Removal</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=event-related-potential" rel="tag">Event Related Potential</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=fourier-time-domain-analysis" rel="tag">Fourier Time-Domain Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=spatial-convolution---deconvolution" rel="tag">Spatial Convolution - Deconvolution</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=spectral-analysis" rel="tag">Spectral Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> BCI2000 is a general-purpose system for brain-computer interface (BCI) research. It can also be used for data acquisition, stimulus presentation, and brain monitoring applications. The mission of the BCI2000 project is to facilitate research and applications in these areas. Our vision is that BCI2000 will become a widely used software tool for diverse areas of real-time biosignal processing. The BCI2000 system is available for free for non-profit research and educational purposes. We have provided it to over 3000 users around the world.<br />
BCI2000 development has been sponsored by a NIH (NIBIB/NINDS) Bioengineering Research Partnership, a NIH (NIBIB) R01 grant, and a NIH (NIBIB) Center grant. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=888&screenshot_id=872"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=888&screenshot_id=872" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 7, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU GPL v2</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/bci2000/">http://www.nitrc.org/projects/bci2000/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Wadsworth Center <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="bcilab" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BCILAB">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=bcilab" rel="bookmark" name="bcilab">BCILAB</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=discriminant-analysis" rel="tag">Discriminant Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=temporal-integration" rel="tag">Temporal Integration</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> BCILAB is a MATLAB toolbox for Brain-Computer Interface (BCI) research. It facilitates the design and development of new methods for cognitive state estimation and their use in both offline data analysis and real-time applications. BCILAB includes an easily extensible collection of currently over 100 methods from the literature (covering signal processing, machine learning and BCI-specific methods). Aside from supporting advanced BCI research, a special aim of BCILAB is to facilitate the adoption of machine learning and advanced statistical modeling for functional neuroimaging purposes in tandem with the EEGLAB platform. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=559&screenshot_id=561"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=559&screenshot_id=561" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 6, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/bcilab/">http://www.nitrc.org/projects/bcilab/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UCSD <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="beast-segmentation-library" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BEaST segmentation library">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=beast-segmentation-library" rel="bookmark" name="beast-segmentation-library">BEaST segmentation library</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Library of anatomical priors for BEaST <a href="http://www.bic.mni.mcgill.ca/ServicesSoftwareAdvancedImageProcessingTools/BEaST" target="_new">http://www.bic.mni.mcgill.ca/ServicesSof...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 13, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BIC-MNI Louis Collins</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/beast-library/">http://www.nitrc.org/projects/beast-library/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Montreal Neurological Institute <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="besa" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BESA">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=besa" rel="bookmark" name="besa">BESA</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=connectivity-analysis" rel="tag">Connectivity Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=eeg-modeling" rel="tag">Eeg Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=event-related-potential" rel="tag">Event Related Potential</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=finite-element-analysis" rel="tag">Finite Element Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=fourier-time-domain-analysis" rel="tag">Fourier Time-Domain Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=independent-component-analysis" rel="tag">Independent Component Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=meg-modeling" rel="tag">Meg Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=principal-component-analysis" rel="tag">Principal Component Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=source-separation-analysis" rel="tag">Source Separation Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=spectral-analysis" rel="tag">Spectral Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surrogate-data-analysis" rel="tag">Surrogate Data Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> BESA is the one of the most widely used software for source analysis and dipole localization in EEG and MEG research. BESA Research has been developed on the basis of 20 years experience in human brain research by Michael Scherg, University of Heidelberg, and Patrick Berg, University of Konstanz.<br />
<br />
BESA Research is a highly versatile and user-friendly Windows® program with optimized tools and scripts to preprocess raw or averaged data for source analysis. All important aspects of source analysis are displayed in one window for immediate selection of a wide range of tools. BESA Research provides a variety of source analysis algorithms, a standardized realistic head model (FEM), and allows for fast and easy hypothesis testing and integration with MRI and fMRI. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=643&screenshot_id=559"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=643&screenshot_id=559" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 20, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Commercial License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://besa.de">http://besa.de</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="best-practices-for-software-development" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Best Practices for Software Development">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=best-practices-for-software-development" rel="bookmark" name="best-practices-for-software-development">Best Practices for Software Development</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=community" rel="tag">Community</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=websites" rel="tag">Websites</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> &quot;Best Practices for Software Development in the Imaging Community: Users and Developers Unite&quot; is a tutorial that will take place as part of MICCAI 2010 is the 13th International Conference on Medical Image Computing and Computer Assisted Intervention, will be held from September 20-24, 2010 in Beijing, China. See <a href="http://www.miccai2010.org/" target="_new">http://www.miccai2010.org/</a><br />
<br />
This project will support community outreach and dialog between presenters and audience, both before and after the tutorial session, and is intended to engage the broader community in the deliberations on this important topic. <br />
<br />
&quot;Best Practices&quot; will cover software engineering practices as well end-user installation and support practices. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=360&screenshot_id=283"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=360&screenshot_id=283" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 30, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/best_practices/">http://www.nitrc.org/projects/best_practices/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="beta-series-correlation" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BetA-Series COrrelation">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=beta-series-correlation" rel="bookmark" name="beta-series-correlation">BetA-Series COrrelation</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=connectivity-analysis" rel="tag">Connectivity Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> BASCO (BetA-Series COrrelation) is a software tool (with GUI) for investigating inter-regional functional connectivity in event-related fMRI data and allows you to assess the modulation of functional connectivity by an experimental condition. This tool is inspired by an approach introduced by Rissman and colleagues (2004). The method is based on a general linear model (GLM) where the evoked activity in each trial is modeled by a separate covariate. This renders a series of beta-values for each voxel which is related to a given experimental condition. The functional connectivity between brain regions is derived from correlating voxel beta-series. BASCO offers the following analysis approaches: i) Seed-based functional connectivity analysis. ii) ROI-based network analysis. Given a parcellation of the brain the mean beta-series are extracted for each ROI and a network matrix is calculated correlating all ROI beta-series. iii) Voxel-based whole brain network analysis. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 20, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/basco/">http://www.nitrc.org/projects/basco/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Luebeck <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="bic-mni-models" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="bic-mni-models">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=bic-mni-models" rel="bookmark" name="bic-mni-models">bic-mni-models</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> bic-mni-models anatomical brain template library, includes models from ICBM 2009 template ( <a href="http://www.bic.mni.mcgill.ca/ServicesAtlases/ICBM152NLin2009" target="_new">http://www.bic.mni.mcgill.ca/ServicesAtl...</a> ) </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 13, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BIC-MNI Louis Collins</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/bic-mni-models/">http://www.nitrc.org/projects/bic-mni-models/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Montreal Neurological Institute <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="big-gaba" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Big GABA">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=big-gaba" rel="bookmark" name="big-gaba">Big GABA</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This repository contains a subset of a large-scale multi-vendor, multi-site collection of single-voxel MRS and structural MRI datasets that were acquired across 26 international research sites on 3T MRI scanners from the three major vendors (GE, Philips, Siemens). Each site acquired up to 12 datasets.<br />
<br />
Currently available data include: (1) standard GABA+-edited MEGA-PRESS data; (2) macromolecule-suppressed GABA-edited MEGA-PRESS data<br />
<br />
If using these data, please cite:<br />
Mikkelsen M et al. Big GABA: Edited MR spectroscopy at 24 research sites. Neuroimage 2017;159:32–45. doi: 10.1016/j.neuroimage.2017.07.021<br />
Mikkelsen M et al. Big GABA II: Water-referenced edited MR spectroscopy at 25 research sites. Neuroimage 2017. doi: 10.1016/j.neuroimage.2019.02.059<br />
and acknowledge: NIH grant R01 EB016089 </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 29, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/biggaba/">http://www.nitrc.org/projects/biggaba/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>The Johns Hopkins University School of Medicine <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="bioera" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BioEra">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=bioera" rel="bookmark" name="bioera">BioEra</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> BioEra provides environment and tools to create various types of processing tasks. It can be used for research, games, self exploration, entrainment, sound processing and many others. Each task is contained in a design. To create a design no programming skill is required, only understanding of the process and its requirements. BioEra is used to create a design visually. A design represents data flow from input (e.g. biofeedback device) to output (e.g. visual or sound feedback). The flow can be customized with hundreds built-in objects (elements). For example an amplitude of alpha brainwaves can be filtered from input EEG signal, it can then trigger a MIDI, video or a computer task at a certain threshold level. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=925&screenshot_id=1135"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=925&screenshot_id=1135" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 1, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Commercial License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/bioera/">http://www.nitrc.org/projects/bioera/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="biofilmquant" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BiofilmQuant">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=biofilmquant" rel="bookmark" name="biofilmquant">BiofilmQuant</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volumetric-analysis" rel="tag">Volumetric Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> A semi-automated software tool for dental plaque biofilm quantification in quantitative light-induced fluorescence (QLF) images. Please cite our paper:<br />
<br />
Mansoor, A.; Patsekin, V.; Scherl, D.; Robinson, J.; Rajwa, B., &quot;A Statistical Modeling Approach to Computer-Aided Quantification of Dental Biofilm,&quot; Biomedical and Health Informatics, IEEE Journal of<br />
URL: <a href="http://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&amp;arnumber=6758338&amp;isnumber=6363502" target="_new">http://ieeexplore.ieee.org/stamp/stamp.j...</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=809&screenshot_id=742"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=809&screenshot_id=742" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 3, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/biofilmquant/">http://www.nitrc.org/projects/biofilmquant/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Purdue University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="bioimage-suite" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BioImage Suite">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=bioimage-suite" rel="bookmark" name="bioimage-suite">BioImage Suite</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=active-contour" rel="tag">Active Contour</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=animation" rel="tag">Animation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=expectation-minimization" rel="tag">Expectation Minimization</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=fiber-tracking" rel="tag">Fiber Tracking</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-to-template" rel="tag">Image-To-Template</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=intermodal" rel="tag">Intermodal</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=intersubject" rel="tag">Intersubject</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=morphology" rel="tag">Morphology</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=three-dimensional-display" rel="tag">Three Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volume-rendering" rel="tag">Volume Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volumetric-analysis" rel="tag">Volumetric Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=web-resource" rel="tag">Web Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> BioImage Suite is a web-based medical image analysis software package with image processing , image registration and visualization capabiliies.<br />
<br />
See <a href="https://bioimagesuiteweb.github.io/webapp/" target="_new">https://bioimagesuiteweb.github.io/webap...</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=51&screenshot_id=1254"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=51&screenshot_id=1254" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 27, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://bioimagesuite.org">http://bioimagesuite.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Yale School of Medicine <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>BSD<br>IRIX<br>Linux<br>MacOS<br>SunOS/Solaris<br>Windows NT/2000</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, MINC2, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, MINC2, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="biomedical-informatics-research-network" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Biomedical Informatics Research Network">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=biomedical-informatics-research-network" rel="bookmark" name="biomedical-informatics-research-network">Biomedical Informatics Research Network</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data-resource" rel="tag">Data Resource</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=knowledge-environment" rel="tag">Knowledge Environment</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=web-resource" rel="tag">Web Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> BIRN is a geographically distributed virtual community of shared resources to advance the diagnosis and treatment of disease. It's emphasis is on enabling the sharing of biomedical data through a suite of software capabilities. It currently consists of expertise in Data Movement, Data Security, Mediation Across Data Sources, Knowledge Engineering, Workflows, and Genetics. The main BIRN site is at the Information Sciences Institute at USC with collborators at the University of Chicago, the Massachusetts General Hospital, University of California, Irvine and University of California, Los Angeles. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 30, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://en.wikipedia.org/wiki/Biomedical_Informatics_Research_Network">https://en.wikipedia.org/wiki/Biomedical_Informatics_Research_Network</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="biomesh3d" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BioMesh3D">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=biomesh3d" rel="bookmark" name="biomesh3d">BioMesh3D</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> BioMesh3D is a free, easy to use program for generating quality meshes for use in biological simulations. It is included when downloading SCIRun v4.6. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 29, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MIT/X Consortium License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/biomesh3d/">http://www.nitrc.org/projects/biomesh3d/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="biosig" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BioSig">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=biosig" rel="bookmark" name="biosig">BioSig</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=autocorrelation" rel="tag">Autocorrelation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=cross-correlation" rel="tag">Cross-Correlation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=directed-transfer-analysis" rel="tag">Directed Transfer Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=discriminant-analysis" rel="tag">Discriminant Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=event-related-potential" rel="tag">Event Related Potential</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=format-conversion" rel="tag">Format Conversion</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=granger-causality" rel="tag">Granger Causality</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=information-theory" rel="tag">Information Theory</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=multivariate-analysis" rel="tag">Multivariate Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=partial-directed-coherence" rel="tag">Partial Directed Coherence</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=regression" rel="tag">Regression</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=spectral-analysis" rel="tag">Spectral Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Biosig provides tools for processing of electroencephalogram (EEG) and other biomedical signals like ECG, EOG, EMG, etc. Biosig contains tools for quality control, artifact processing, time series analysis, feature extraction, classification and machine learning, and tools for statistical analysis. <br />
<br />
Many tools are able to handle data with missing values (statistics, time series analysis, machine learning). Another feature is that more then 40 different data formats are supported, and a number of converters for EEG,, ECG and polysomnography are provided. <br />
<br />
Biosig has been widely used for scientific research on EEG-based BraiN-Computer Interfaces (BCI), sleep research, and ECG and HRV analysis. <br />
<br />
It provides software interfaces several programming languages (C, C++, Matlab/Octave, Python), and it provides also an interactive viewing and scoring software for adding, and editing of annotations, markers and events. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=557&screenshot_id=635"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=557&screenshot_id=635" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 27, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GPL-Style Open Source</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/biosig/">http://www.nitrc.org/projects/biosig/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>BSD<br>Linux<br>Other UNIX-like<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="bisque" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Bisque">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=bisque" rel="bookmark" name="bisque">Bisque</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Bisque (Bio-Image Semantic Query User Environment) is a scalable web-based system for biological image analysis, management and exploration. The Bisque system incorporates many features useful to imaging researchers from image capture to extensible image analysis and querying. At the core, bisque maintains a flexible database of images and experimental metadata. Image analyses can be incorporated into the system and deployed on clusters and desktops. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 14, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/bisque/">http://www.nitrc.org/projects/bisque/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="bluepyopt" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BluePyOpt">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=bluepyopt" rel="bookmark" name="bluepyopt">BluePyOpt</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Blue Brain Python Optimisation Library (BluePyOpt) is an extensible framework for data-driven model parameter optimisation that wraps and standardises several existing open-source tools.<br />
<br />
It simplifies the task of creating and sharing these optimisations, and the associated techniques and knowledge. This is achieved by abstracting the optimisation and evaluation tasks into various reusable and flexible discrete elements according to established best-practices.<br />
<br />
Further, BluePyOpt provides methods for setting up both small- and large-scale optimisations on a variety of platforms, ranging from laptops to Linux clusters and cloud-based compute infrastructures.<br />
<br />
The main website of BluePyOpt is: <a href="https://github.com/BlueBrain/BluePyOpt" target="_new">https://github.com/BlueBrain/BluePyOpt</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1072&screenshot_id=1018"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1072&screenshot_id=1018" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 3, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Lesser General Public License (LGPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/bluepyopt/">http://www.nitrc.org/projects/bluepyopt/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Blue Brain Project <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>POSIX/UNIX-like</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="books" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Books">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=books" rel="bookmark" name="books">Books</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=books" rel="tag">Books</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> If you are interested in finding books related to neuroscience, you have come to the right place! Books is a bibliography of books addressing the topic of functional and structural neuroimaging. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 24, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/books/">http://www.nitrc.org/projects/books/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainance-dti-module" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Brainance DTI Module">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainance-dti-module" rel="bookmark" name="brainance-dti-module">Brainance DTI Module</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=diffusion-mr-fiber-tracking" rel="tag">Diffusion Mr Fiber Tracking</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Brainance DTI Module is a web-based highly accurate and user friendly fiber tracking software created by Advantis Medical Imaging. The fiber tracking methodology developed and implemented achieves extremely high accuracy rates (90%) in the final 3D reconstruction of the nerve fibers, compared to the existing deterministic methodologies which do not exceed 65%. Create an account through our website (<a href="http://advantis.io/" target="_new">http://advantis.io/</a>) and start using our one-month free trial! 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1056&screenshot_id=1003"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1056&screenshot_id=1003" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 29, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Commercial: Advantis Medical Imaging</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/brainance/">http://www.nitrc.org/projects/brainance/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Advantis Medical Imaging <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>DICOM, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>DICOM, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainbay" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BrainBay">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainbay" rel="bookmark" name="brainbay">BrainBay</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=animation" rel="tag">Animation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Brainbay supports Human-Computer-Interface functions and diverse Biosignal- and EEG amplifier hardware - including the OpenBCI Cyton and Ganglion devices - for live recording and processing of bioelectric signals. <br />
BrainBay is a part of the OpenEEG project and provides graphical + musical feedback functions and feedback-games. <br />
<br />
The project is hosted on Github:<br />
<a href="https://github.com/ChrisVeigl/BrainBay" target="_new">https://github.com/ChrisVeigl/BrainBay</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=924&screenshot_id=1134"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=924&screenshot_id=1134" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 1, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.shifz.org/brainbay/">http://www.shifz.org/brainbay/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>SHIFZ <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BrainBox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainbox" rel="bookmark" name="brainbox">BrainBox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=web-service" rel="tag">Web Service</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=websites" rel="tag">Websites</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> BrainBox is a web application developed by the Group of Applied and Theoretical Neuroanatomy – NAAT – for the Open Neuroimaging Laboratory Project. BrainBox allows you to visualise and segment collaboratively any brain MRI dataset available online. Segmentations are automatically saved and can be downloaded as Nifti files or triangular meshes. Point BrainBox to your own Nifti data, or try data catalogues created by the community. To access BrainBox, go to: <a href="http://brainbox.pasteur.fr/" target="_new">http://brainbox.pasteur.fr/</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1075&screenshot_id=1026"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1075&screenshot_id=1026" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 16, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://brainbox.pasteur.fr/">http://brainbox.pasteur.fr/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Instut Pasteur <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>MGH/MGZ, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>MGH/MGZ, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainbrowser" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BrainBrowser">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainbrowser" rel="bookmark" name="brainbrowser">BrainBrowser</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-rendering" rel="tag">Surface Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=three-dimensional-display" rel="tag">Three Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=two-dimensional-display" rel="tag">Two Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volume-rendering" rel="tag">Volume Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=web-resource" rel="tag">Web Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> BrainBrowser ( <a href="https://brainbrowser.cbrain.mcgill.ca" target="_new">https://brainbrowser.cbrain.mcgill.ca</a> ) is a web-enabled brain surface viewer that allows the user to explore in real time a 3D brain map expressed on a base surface. <br />
<br />
BrainBrowser has two modes of operation, exploring either a pre-calculated database of structural correlation maps or working with user-defined data. In this mode, the user may choose to explore the correlation structure for cortical thickness, cortical area or cortical volume, or any other pre-calculated metric.<br />
<br />
In the second mode, the user is prompted for the local filenames of the statistical map and the base surface. BrainBrowser can also be used to manipulate 3D fibre pathways derived from DTI, using the same simple file format (.obj) as for surface data. <br />
<br />
BrainBrowser on Youtube: <a href="http://www.youtube.com/watch?v=HlRTUYUf1Ew" target="_new">http://www.youtube.com/watch?v=HlRTUYUf1...</a><br />
<br />
NOTE: BrainBrowser requires a WebGL-enabled browser such as Google Chrome to support its 3D graphics capability. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=467&screenshot_id=359"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=467&screenshot_id=359" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 24, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Web Application - Free to use</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://brainbrowser.cbrain.mcgill.ca">http://brainbrowser.cbrain.mcgill.ca</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>MINC2, NIfTI-1, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>MINC2, NIfTI-1, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="braincolor:-human-brain-anatomical-labeling-protocols-and-colormaps" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="brainCOLOR: human brain anatomical labeling protocols and colormaps">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=braincolor:-human-brain-anatomical-labeling-protocols-and-colormaps" rel="bookmark" name="braincolor:-human-brain-anatomical-labeling-protocols-and-colormaps">brainCOLOR: human brain anatomical labeling protocols and colormaps</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=knowledge-environment" rel="tag">Knowledge Environment</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This resource was created to host descriptions of protocols, definitions, and &quot;rules of thumb&quot; for the reliable identification and localization of human brain anatomy and discussions of best practices in brain labeling. The projects related to this resource are on <a href="http://binarybottle.github.io/braincolor" target="_new">http://binarybottle.github.io/braincolor</a> and <a href="https://www.synapse.org/Synapse:syn3207269/wiki/160639" target="_new">https://www.synapse.org/Synapse:syn32072...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 14, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://binarybottle.github.io/braincolor/">http://binarybottle.github.io/braincolor/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brain-connectivity-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Brain Connectivity Toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brain-connectivity-toolbox" rel="bookmark" name="brain-connectivity-toolbox">Brain Connectivity Toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=statistical-operation" rel="tag">Statistical Operation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Brain Connectivity Toolbox (brain-connectivity-toolbox.net) is a MATLAB toolbox for complex-network (graph) analysis of structural and functional brain-connectivity data sets. Several people have contributed to the toolbox and users are welcome to contribute new functions with due acknowledgement.<br />
<br />
<br />
All efforts have been made to avoid errors, but users are strongly urged to independently verify the accuracy and suitability of individual toolbox functions. Please report bugs or substantial improvements to Olaf Sporns or to Mika Rubinov. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 30, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.brain-connectivity-toolbox.net/">http://www.brain-connectivity-toolbox.net/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="braincsi" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BrainCSI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=braincsi" rel="bookmark" name="braincsi">BrainCSI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=application" rel="tag">Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> BrainCSI is a tool for analysis of Magnetic Resonance Spectroscopy (MRS) data by registering it to anatomical images. BrainCSI imports LCModel results to calculate absolute metabolite concentrations using tissue water. Corrections to LCModel metabolite concentrations for partial volume of tissues are accomplished by tissue classification of the anatomical images. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 6, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/braincsi/">http://www.nitrc.org/projects/braincsi/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS<br>POSIX/UNIX-like<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>DICOM, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>DICOM, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="braindecodertoolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BrainDecoderToolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=braindecodertoolbox" rel="bookmark" name="braindecodertoolbox">BrainDecoderToolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=application" rel="tag">Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Brain Decoder Toolbox performs “decoding” of brain activity, by learning the difference between brain activity patterns among conditions and then classifying the brain activity based on the learning results.<br />
<br />
BDTB is a set of Matlab functions.<br />
BDTB is OS-independent. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 31, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.cns.atr.jp/dni/en/downloads/brain-decoder-toolbox/">http://www.cns.atr.jp/dni/en/downloads/brain-decoder-toolbox/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Computational Neuroscience Labs <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brain-entropy-in-space-and-time-(best)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Brain Entropy in space and time (BEst)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brain-entropy-in-space-and-time-(best)" rel="bookmark" name="brain-entropy-in-space-and-time-(best)">Brain Entropy in space and time (BEst)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=eeg" rel="tag">Eeg</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=eeg-modeling" rel="tag">Eeg Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=meg" rel="tag">Meg</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=meg-modeling" rel="tag">Meg Modeling</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Are you interested in estimating the spatial extent of EEG/MEG sources?<br />
<br />
Are you interested in localizing oscillatory patterns?<br />
<br />
Are you interested in localizing synchronous cortical sources? <br />
<br />
We introduce here the toolbox BEst – &quot;Brain Entropy in space and time&quot; that implements several EEG/MEG source localization techniques within the “Maximum Entropy on the Mean (MEM)” framework. These methods are particularly dedicated to estimate accurately the source of EEG/MEG generators together with their spatial extent along the cortical surface. Assessing the spatial extent of the sources might be very important in some application context, and notably when localizing spontaneous epileptic discharges. We also proposed two other extensions of the MEM framework within the time frequency domain dedicated to localize oscillatory patterns in specific frequency bands and synchronous sources. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=849&screenshot_id=826"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=849&screenshot_id=826" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 30, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/best/">http://www.nitrc.org/projects/best/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Ecole de Technologie Supérieure, McGill University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainfx" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BrainFX">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainfx" rel="bookmark" name="brainfx">BrainFX</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=platform-or-development-environment" rel="tag">Platform Or Development Environment</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> A developer tool to provide batch processing capability for pipelines. Users input data into a input table and run analysis with it. It is used to power CamBA's and Brainwaver's User interface. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 18, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.brainfx.org/">http://www.brainfx.org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brain-genomics-superstruct-project-(gsp)-open-access-data-release" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Brain Genomics Superstruct Project (GSP) Open Access Data Release">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brain-genomics-superstruct-project-(gsp)-open-access-data-release" rel="bookmark" name="brain-genomics-superstruct-project-(gsp)-open-access-data-release">Brain Genomics Superstruct Project (GSP) Open Access Data Release</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Instructions for accessing the GSP Open Access Data may be found at <br />
<br />
<a href="http://www.neuroinfo.org/gsp/" target="_new">http://www.neuroinfo.org/gsp/</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=906&screenshot_id=902"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=906&screenshot_id=902" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 19, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GSP Open Access Data Use Terms</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.neuroinfo.org/gsp/">http://www.neuroinfo.org/gsp/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Harvard University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="braingraph-editor-1.0-beta" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BrainGraph Editor 1.0 Beta">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=braingraph-editor-1.0-beta" rel="bookmark" name="braingraph-editor-1.0-beta">BrainGraph Editor 1.0 Beta</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=ontology" rel="tag">Ontology</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-information-resource" rel="tag">Other Information Resource</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The BrainGraph Editor 1.0 Beta is a JAVA application designed to create taxonomies or hierarchies in order to classify and organize information. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 22, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BIRN License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.loni.ucla.edu/Software/Software_Detail.jsp?software_id=60">http://www.loni.ucla.edu/Software/Software_Detail.jsp?software_id=60</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="braingraph---graph-theory-analysis-of-brain-mri-data-in-r" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="brainGraph - Graph Theory Analysis of Brain MRI Data in R">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=braingraph---graph-theory-analysis-of-brain-mri-data-in-r" rel="bookmark" name="braingraph---graph-theory-analysis-of-brain-mri-data-in-r">brainGraph - Graph Theory Analysis of Brain MRI Data in R</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=connectivity-analysis" rel="tag">Connectivity Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> brainGraph is an R package containing tools for graph theory (network) analysis of brain MRI data. It was initially developed with a focus on Freesurfer data (i.e., cortical thickness, surface area, volume, etc.) but also works with DTI tractography data (particularly from FSL's &quot;probtrackx2&quot;), and should work with resting-state fMRI data (given correlation matrices).<br />
<br />
In addition to general network operations (available through the R package &quot;igraph&quot;), there is code to perform: bootstrapping, permutation tests, random graph generation, small-worldness/efficiency, robustness/targeted attack/random failure analysis, rich-club analysis, and more. Additionally, there is a GUI for quick data viewing and exploration.<br />
<br />
For more information, please visit the GitHub page at: <a href="https://github.com/cwatson/brainGraph" target="_new">https://github.com/cwatson/brainGraph</a><br />
From there, you will find a link to the User Guide, which is also accessible through the &quot;Docs&quot; link here on the NITRC page. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 12, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GPL-Style Open Source</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://github.com/cwatson/braingraph/">https://github.com/cwatson/braingraph/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainhack2014" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="brainhack2014">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainhack2014" rel="bookmark" name="brainhack2014">brainhack2014</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other" rel="tag">Other</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> brainhack.org seeks to provide a forum for collaborative projects in the field of brain science.<br />
<br />
We invite everyone to submit projects, either existing ones that you want to see featured or new ones that you want launch and work on in a collaborative, open way.<br />
<br />
brainhack.org is an effort of the Neuro Bureau, neuro-collaboration in action. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 5, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://brainhack.org">http://brainhack.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brain-hierarchical-atlas:-a-brain-atlas-where-the-regions-of-interest-are-relevant-for-both-structure-and-function" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Brain Hierarchical Atlas: A brain atlas where the regions of interest are relevant for both structure and function">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brain-hierarchical-atlas:-a-brain-atlas-where-the-regions-of-interest-are-relevant-for-both-structure-and-function" rel="bookmark" name="brain-hierarchical-atlas:-a-brain-atlas-where-the-regions-of-interest-are-relevant-for-both-structure-and-function">Brain Hierarchical Atlas: A brain atlas where the regions of interest are relevant for both structure and function</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This atlas results from a hierarchical clustering approach applied to a combination of functional (resting fMRI) and structural (DTI) datasets. The novelty of the atlas is based on the fact that ROIs are functionally coherent (i.e., the dynamics of voxels within regions have high similarity) and at the same time they are structurally wired (the voxels within regions are highly integrated by white-matter fibers).<br />
<br />
This Project contains the following files:<br />
<br />
-average_networks.mat: The population (N=12) functional and structural matrices, each one with dimensions 2514x2514<br />
<br />
-functional.zip/structural.zip: Zip files containing the functional/structural atlas for all the stages in the hierarchical tree, from M=1 to 2514 ROIs<br />
<br />
-test_crosmodularity.m: Code example on how to compute crosmodularity. It needs crossmodularity.m and modularity_index.m<br />
<br />
<br />
For further information see:<br />
<br />
Diez, P. et al, A novel brain partition highlights the modular skeleton shared by structure and function, Sci Rep 5: 10532, 2015 </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 24, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GPL-Style Open Source</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/biocr_hcatlas/">http://www.nitrc.org/projects/biocr_hcatlas/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Biocruces Heatlh Research Institute. Barakaldo, Spain <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainlife.io" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="brainlife.io">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainlife.io" rel="bookmark" name="brainlife.io">brainlife.io</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=connectivity-analysis" rel="tag">Connectivity Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=platform-or-development-environment" rel="tag">Platform Or Development Environment</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=shape-analysis" rel="tag">Shape Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-analysis" rel="tag">Surface Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tractography" rel="tag">Tractography</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> brainlife.io is a platform that allows publishing reproducible code and datasets. The platform provides seamless access to national supercomputers, private clouds, and institutional high-performance computer systems. The goals of the platform are to promote data sharing in exchange for high-throughput computing and to contribute upcycling the long tail of neuroimaging data. To do so, brainlife.io makes available data and analyses software to pursuit gathering diverse neuroimaging datasets and redistribute them across multiple research communities such as psychology, engineering, computer science, and neuroscience. Brainlife.io combines modern web visualization, and database technologies as well as network neuroscience and image processing algorithms into a unique online platform. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1317&screenshot_id=1233"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1317&screenshot_id=1233" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 18, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MIT</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://brainlife.io">https://brainlife.io</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Indiana University, Bloomington <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent<br>POSIX/UNIX-like</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainliner.jp" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BrainLiner.jp">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainliner.jp" rel="bookmark" name="brainliner.jp">BrainLiner.jp</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description><a href="http://brainliner.jp/" target="_new">http://brainliner.jp/</a><br />
<br />
A web portal for sharing neurophysiological and behavioral data. Users can search for existing data or login with their Google, Facebook, or Twitter account and upload new data.<br />
<br />
Our main focus is on supporting brain-machine interface research, so we encourage users to not just provide recordings of brain activity data, but also information about stimuli, etc., so that statistical relationships can be found between stimuli and/or subject behavior and brain activity.<br />
<br />
Data on the site is freely available using mostly either CC-BY or CC-0 Creative Commons licenses, though some custom licenses also exist.<br />
<br />
We also have a page at <a href="https://www.facebook.com/brainliner" target="_new">https://www.facebook.com/brainliner</a> where you can contact us 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=594&screenshot_id=499"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=594&screenshot_id=499" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 3, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/brainliner/">http://www.nitrc.org/projects/brainliner/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>ATR <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainmagix-spm-viewer" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BrainMagix SPM Viewer">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainmagix-spm-viewer" rel="bookmark" name="brainmagix-spm-viewer">BrainMagix SPM Viewer</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> BrainMagix SPM Viewer is a free, professional viewer for SPM fMRI results.<br />
<br />
SPM (Statistical Parametric Mapping, UCL, London) is a powerful fMRI analysis software but its visualization capabilities are sometimes a limitation for the researchers. That's why Imagilys has decided to offer the neuroimaging community a free version of its commercial &quot;BrainMagix&quot; neuroimaging software, called &quot;BrainMagix SPM viewer&quot;.<br />
<br />
BrainMagix SPM Viewer's Features<br />
<br />
- Professional viewer for your SPM-based fMRI activations<br />
- JAVA-programmed, cross-platform (Windows, MAC, Linux), without Matlab license, making it possible to share your results with colleagues who do not have SPM installed<br />
- Reads SPM.mat files and NIfTI images in an user-friendly way<br />
- Overlay the blobs with an atlas or any anatomical image<br />
- On the fly adjustment of threshold and cluster size<br />
- Localize your activations in an atlas<br />
- BOLD signal curves in ROIs (future feature)<br />
- Export your results as PNG images 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=738&screenshot_id=668"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=738&screenshot_id=668" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 29, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BrainMagix SPM Viewer: License & Disclaimer</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.imagilys.com/brainmagix-spm-viewer/">http://www.imagilys.com/brainmagix-spm-viewer/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Imagilys <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainmap-database" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BrainMap Database">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainmap-database" rel="bookmark" name="brainmap-database">BrainMap Database</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=modeling" rel="tag">Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=ontology" rel="tag">Ontology</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=websites" rel="tag">Websites</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> BrainMap is a community database of published functional neuroimaging studies (mainly PET and fMRI). BrainMap contains both metadata descriptions of experimental design and activation locations in the form of stereotactic coordinates. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=53&screenshot_id=268"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=53&screenshot_id=268" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 31, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BrainMap License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://brainmap.org">http://brainmap.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainmaps.org" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BrainMaps.org">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainmaps.org" rel="bookmark" name="brainmaps.org">BrainMaps.org</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> BrainMaps.org is an interactive multiresolution next-generation brain atlas that is based on over 20 million megapixels of sub-micron resolution, annotated, scanned images of serial sections of both primate and non-primate brains. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 11, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/brainmaps/">http://www.nitrc.org/projects/brainmaps/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainmask-volume-processing-tool" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BrainMask Volume Processing Tool">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainmask-volume-processing-tool" rel="bookmark" name="brainmask-volume-processing-tool">BrainMask Volume Processing Tool</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=constrained-region-growing" rel="tag">Constrained Region Growing</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-display" rel="tag">Image Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=region-of-interest" rel="tag">Region Of Interest</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volume-measurement" rel="tag">Volume Measurement</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volumetric-analysis" rel="tag">Volumetric Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Segmentation of the brain from three-dimensional MR images is a crucial pre-processing step in morphological and volumetric brain studies. BrainMask implements a fully automatic brain segmentation algorithm that uses advanced &quot;thresholding with morphology&quot; and 3D edge detection algorithms. BrainMask demonstrates high segmentation accuracy. For a representative 26 datasets, the segmentation error averaged 3.4% ± 1.3% (Mikheev A et al. J Magn Reson Imag 27(6):1235-41;2008).<br />
<br />
BrainMask includes NNN - a tool based on the algorithm developed by John Sled for correcting the intensity non-uniformity in MR data (Sled JG et al. IEEE Trans Med Imag 17(1):87-97;1998).<br />
<br />
BrainMask also includes a versatile DICOM wiewer and allows to selectively load and organize DICOM images into 3D and 4D datasets. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 6, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>NYUMC FireVoxel-BrainMask Software License Agreement</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/brainmask/">http://www.nitrc.org/projects/brainmask/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Windows XP</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, Other Format, bshort/bfloat</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, Other Format, bshort/bfloat</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainnetome-atlas-viewer" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Brainnetome Atlas Viewer">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainnetome-atlas-viewer" rel="bookmark" name="brainnetome-atlas-viewer">Brainnetome Atlas Viewer</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Brainnetome Atlas Viewer(v1.0) shows the anatomical connectivity-based parcellation results, including the maximum probabilistic maps,probabilistic maps and both the anatomical and functional connectivity patterns, which have been developed in Brainnetome Center(<a href="http://www.brainnetome.org/" target="_new">http://www.brainnetome.org/</a>), CASIA. The atlas is based on the analysis of connectional architecture with in vivo multi-modal MRI data during the last 6 years. Please see more details in our homepage: <a href="http://atlas.brainnetome.org/" target="_new">http://atlas.brainnetome.org/</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=843&screenshot_id=821"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=843&screenshot_id=821" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 9, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Brainnetome Atlas</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/bn_atlas/">http://www.nitrc.org/projects/bn_atlas/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Institute of Automation.Chinese Academy of Sciences <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainnetome-diffusionkit" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Brainnetome DiffusionKit">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainnetome-diffusionkit" rel="bookmark" name="brainnetome-diffusionkit">Brainnetome DiffusionKit</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=diffusion-mr-fiber-tracking" rel="tag">Diffusion Mr Fiber Tracking</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-rendering" rel="tag">Surface Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=three-dimensional-display" rel="tag">Three Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=two-dimensional-display" rel="tag">Two Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volume-rendering" rel="tag">Volume Rendering</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This page is no longer maintained and please visit <a href="http://diffusion.brainnetome.org" target="_new">http://diffusion.brainnetome.org</a> . 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1040&screenshot_id=982"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1040&screenshot_id=982" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 17, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial Share Alike</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://diffusion.brainnetome.org">http://diffusion.brainnetome.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>CASIA <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>Windows XP</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainnetome-fmri-toolkit" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Brainnetome fMRI toolkit">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainnetome-fmri-toolkit" rel="bookmark" name="brainnetome-fmri-toolkit">Brainnetome fMRI toolkit</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> We have update this toolkit to Brainnetome fMRI toolkit. Detail of this toolkit can be found @ <a href="http://brant.brainnetome.org" target="_new">http://brant.brainnetome.org</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=819&screenshot_id=773"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=819&screenshot_id=773" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 25, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/brant/">http://www.nitrc.org/projects/brant/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Brainnetome Center <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainnet-viewer" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BrainNet Viewer">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainnet-viewer" rel="bookmark" name="brainnet-viewer">BrainNet Viewer</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> BrainNet Viewer is a brain network visualization tool, which can help researchers to visualize structural and functional connectivity patterns from different levels in a quick, easy, and flexible way. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=504&screenshot_id=710"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=504&screenshot_id=710" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 7, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/bnv/">http://www.nitrc.org/projects/bnv/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Beijing Normal University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainnetworkconstructionanalysisplatform" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BrainNetworkConstructionAnalysisPlatform">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainnetworkconstructionanalysisplatform" rel="bookmark" name="brainnetworkconstructionanalysisplatform">BrainNetworkConstructionAnalysisPlatform</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Construct and analyse brain network is a brain network visualization tool, which can help researchers to visualize construct and analyse resting state functional brain networks from different levels in a quick, easy and flexible way. Entrance parameter of construct and analyse brain network is export parameters of dparsf software.It would be greatly appreciated if you have any suggestions about the package or manual. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 28, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cabn/">http://www.nitrc.org/projects/cabn/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainparser" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BrainParser">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainparser" rel="bookmark" name="brainparser">BrainParser</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=anatomic" rel="tag">Anatomic</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Brain Parser software uses a novel statistical-learning technique to segment brain regions of interest (ROIs) based on a training set of data and generates 3D MRI volumes. The software comes pre-trained on a provided data set but can be retrained to work with your desired regions of interest. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 8, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/brainparser/">http://www.nitrc.org/projects/brainparser/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of California, San Diego <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>Windows NT/2000<br>Windows Vista</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brain-registration-evaluations-(2008-2011)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Brain registration evaluations (2008-2011)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brain-registration-evaluations-(2008-2011)" rel="bookmark" name="brain-registration-evaluations-(2008-2011)">Brain registration evaluations (2008-2011)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=information-resource" rel="tag">Information Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This project contains four human brain MR image registration evaluation projects from 2008 to 2011, including the largest ever conducted. They are accessible via Synapse: <a href="https://www.synapse.org/#!Synapse:syn3207152/wiki/160605" target="_new">https://www.synapse.org/#!Synapse:syn320...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 2, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/reg_evals/">http://www.nitrc.org/projects/reg_evals/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainscape" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Brainscape">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainscape" rel="bookmark" name="brainscape">Brainscape</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=correlation" rel="tag">Correlation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=statistical-operation" rel="tag">Statistical Operation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Brainscape is a database for resting state functional connectivity studies. Functional connectivity has shown tremendous promise in mapping the intrinsic functional topography of the brain, evaluating neuroanatomical models, and investigating neurological and psychiatric disease. Brainscape includes a repository of public and private data and an analysis engine for exploring the correlation structure of spontaneous fluctuations in the fMRI BOLD signal. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=107&screenshot_id=154"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=107&screenshot_id=154" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 13, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.brainscape.org/">http://www.brainscape.org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>DICOM</td></tr>
			<tr class="even"><td>Output Formats</td><td>DICOM</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainsconstellationdetector" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BRAINSConstellationDetector">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainsconstellationdetector" rel="bookmark" name="brainsconstellationdetector">BRAINSConstellationDetector</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=fiducial" rel="tag">Fiducial</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NOTE: All active development of BRAINSConstellationDetector is now being stored in a Git repository at github.com. Please go to<br />
<br />
<a href="https://github.com/BRAINSia/BRAINSTools" target="_new">https://github.com/BRAINSia/BRAINSTools</a><br />
<br />
This program will find the mid-sagittal plane, the AC, PC, and mpj points in an image, and create an AC/PC aligned data set with the AC point at the center of the voxel lattice (la<br />
beled at the origin of the image physical space.) <br />
 <br />
This work is an extention of the algorithms originally described by Dr. Babak A. Ardekani, Alvin H. Bachman, Model-based automatic detection of the anterior and posterior commissures on MRI scans, N<br />
euroImage, Volume 46, Issue 3, 1 July 2009, Pages 677-682, ISSN 1053-8119, DOI: 10.1016/j.neuroimage.2009.02.030.<br />
 (<a href="http://www.sciencedirect.com/science/article/B6WNP-4VRP25C-4/2/8207b962a38aa83c822c6379bc43fe4c" target="_new">http://www.sciencedirect.com/science/art...</a>) </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 1, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/brainscdetector/">http://www.nitrc.org/projects/brainscdetector/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, NIfTI-1, Nrrd</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, NIfTI-1, Nrrd</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brain-segmentation-testing-protocol" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Brain Segmentation Testing Protocol">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brain-segmentation-testing-protocol" rel="bookmark" name="brain-segmentation-testing-protocol">Brain Segmentation Testing Protocol</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=test-data" rel="tag">Test Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Brain Segmentation Testing Protocol (Neuroimage 2011, <a href="http://dx.doi.org/10.1016/j.neuroimage.2011.06.080" target="_new">http://dx.doi.org/10.1016/j.neuroimage.2...</a>) is a freely available collection of MRI images for testing segmentation algorithms. The 312 MRI datasets may be downloaded to assess the accuracy, reproducibility and sensitivity of MRI segmentation software. The accuracy/validation dataset includes images from infants, adults and patients with Alzheimer’s disease and the reproducibility dataset includes 9 subjects who have been scanned with 8 different sequences at 1.0T and 3.0T. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=548&screenshot_id=460"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=548&screenshot_id=460" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 15, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/bstp/">http://www.nitrc.org/projects/bstp/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainsfit" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BRAINSFit">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainsfit" rel="bookmark" name="brainsfit">BRAINSFit</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=affine-warp" rel="tag">Affine Warp</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=intermodal" rel="tag">Intermodal</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=intersubject" rel="tag">Intersubject</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=linear-warp" rel="tag">Linear Warp</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NOTE: All active development of BRAINSFit is now being stored in a Git repository at github.com. Please go to<br />
<br />
<a href="https://github.com/BRAINSia/BRAINSTools" target="_new">https://github.com/BRAINSia/BRAINSTools</a><br />
<br />
A program for registering images with with mutual information based metric. Several registration options are given for 3,6, 9,12,16 parameter (i.e. translate, rigid, scale, scale/skew, full affine) based constraints for the registration. The program uses the Slicer3 execution model framework to define the command line arguments and can be fully integrated with Slicer3 using the module discovery capabilities of Slicer3 </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 30, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.github.com/BRAINSia/BRAINSTools">http://www.github.com/BRAINSia/BRAINSTools</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Iowa <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS<br>OS Independent<br>POSIX/UNIX-like<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, NIfTI-1, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, NIfTI-1, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainsight" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Brainsight">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainsight" rel="bookmark" name="brainsight">Brainsight</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=eeg" rel="tag">Eeg</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=nirs/fnir" rel="tag">Nirs/Fnir</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-rendering" rel="tag">Surface Rendering</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Neuronavigation system for use in human cognitive neuroscience (TMS, EEG, NIRS) and for non-human neurosurgical applications. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=499&screenshot_id=374"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=499&screenshot_id=374" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 30, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Comercial Licence: Rogue Research Inc.</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.rogue-research.com">http://www.rogue-research.com</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Rogue Research Inc. <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>DICOM, MINC, MINC2, NIfTI-1, Philips PAR/REC</td></tr>
			<tr class="even"><td>Output Formats</td><td>DICOM, MINC, MINC2, NIfTI-1, Philips PAR/REC</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainsimulator" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="brainSimulator">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainsimulator" rel="bookmark" name="brainsimulator">brainSimulator</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> brainSimulator is a brain image synthesis toolkit written in python, intended to generate a new image set that share characteristics with an original one. The system focuses on nuclear imaging modalities such as PET or SPECT brain images. It analyses the dataset by applying PCA to the original dataset, and then model the distribution of samples in the projected eigenbrain space using a Probability Density Function (PDF) estimator. Once the model has been built, anyone can generate new coordinates on the eigenbrain space belonging to the same class, which can be then projected back to the image space. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 21, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License v3</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/brainsimulator/">http://www.nitrc.org/projects/brainsimulator/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>SIPBA Research Group, University of Granada <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainsmush" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BRAINSMush">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainsmush" rel="bookmark" name="brainsmush">BRAINSMush</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NOTE: All active development of BRAINSMush is now being stored in a Git repository at github.com. Please go to<br />
<br />
<a href="https://github.com/BRAINSia/BRAINSTools" target="_new">https://github.com/BRAINSia/BRAINSTools</a><br />
<br />
Tool to generate brain volume mask from input of T1 and T2-weighted images alongside a region of interest brain mask. This volume mask omits dura, skull, eyes, etc. The program is built upon ITK and uses the Slicer3 execution model framework to define the command line arguments and can be fully integrated with Slicer3 using the module discovery capabilities of Slicer3. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 26, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/brainsmush/">http://www.nitrc.org/projects/brainsmush/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, NIfTI-1, Nrrd</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, NIfTI-1, Nrrd</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainsolution" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BrainSolution">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainsolution" rel="bookmark" name="brainsolution">BrainSolution</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=labeling" rel="tag">Labeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> BrainSolution is a collection of tools for MRI T1 brain image segmentation in the Windows® environment. It helps construct a complete pipeline with necessary preprocessing and postprocessing procedures besides brainparser, the core program of our fast brain segmentation. The execution of the whole pipeline can be completed in 2 hours with good segmentation results. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 17, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>LONI Software License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/brainsolution/">http://www.nitrc.org/projects/brainsolution/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>Windows NT/2000<br>Windows Vista</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainspell" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="brainspell">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainspell" rel="bookmark" name="brainspell">brainspell</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=bibliographic-resource" rel="tag">Bibliographic Resource</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> brainspell is the first open, human-curated, collaborative classification of the neuroimaging literature. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 16, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://brainspell.org/">http://brainspell.org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Instut Pasteur <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainsroiauto" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BRAINSROIAuto">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainsroiauto" rel="bookmark" name="brainsroiauto">BRAINSROIAuto</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NOTE: All new development is being managed in a github repository. Please visit<br />
<br />
<a href="https://github.com/BRAINSia/BRAINSTools" target="_new">https://github.com/BRAINSia/BRAINSTools</a><br />
<br />
Automatically creates a mask based on the 'foreground' of an anatomical scan volume. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 30, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BRAINS License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/brainsroiauto/">http://www.nitrc.org/projects/brainsroiauto/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainssurfacestats" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BRAINSSurfaceStats">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainssurfacestats" rel="bookmark" name="brainssurfacestats">BRAINSSurfaceStats</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This program provides a tool for performing a per vertex statistical analysis across a population. The underlying statistical framework uses the R language. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 21, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/surfacestat/">http://www.nitrc.org/projects/surfacestat/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainstem-connectome-atlas" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Brainstem Connectome Atlas">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainstem-connectome-atlas" rel="bookmark" name="brainstem-connectome-atlas">Brainstem Connectome Atlas</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This is a probabilistic atlas of 23 brainstem bundles using high-quality connectome imaging data and advanced analysis techniques. We performed rigorous quality control on connectome imaging data from the Human Connectome Project (HCP) and only accepted high-quality imaging data with minimal residual distortions for atlas construction. A systematic protocol is then developed to manually delineate 1300 ROIs on 20 HCP subjects (10 males; 10 females) for the reconstruction of brainstem fiber bundles and the probabilistic atlases in the MNI152 space. The 23 brainstem bundles can be divided into three groups: 1) Major motor tracts: the corticospinal tract (CST), the fronto-pontine tract (FPT), and the parieto-occipito-temporo-pontine tract (POTPT); 2) Major sensory tracts: the medial lemniscus (ML), the spinothalamic tract (STT), and the lateral lemniscus (LL); 3) Cerebellar peduncles: superior cerebellar peduncle (SCP), the middle cerebellar peduncle (MCP), and the inferior cerebellar peduncle (ICP). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 3, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Brainstem Connectome Atlas License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/brainstem_atlas/">http://www.nitrc.org/projects/brainstem_atlas/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Southern California <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainstools" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BRAINSTools">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainstools" rel="bookmark" name="brainstools">BRAINSTools</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=diffusion-mr-fiber-tracking" rel="tag">Diffusion Mr Fiber Tracking</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-display" rel="tag">Image Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=intermodal" rel="tag">Intermodal</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=intersubject" rel="tag">Intersubject</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=kmeans" rel="tag">Kmeans</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=mesh-generation" rel="tag">Mesh Generation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=model-generation" rel="tag">Model Generation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=neural-network" rel="tag">Neural Network</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=resampling" rel="tag">Resampling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tensor-metric" rel="tag">Tensor Metric</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> DO NOT USE THIS PAGE -- GOTO --&gt; <a href="https://github.com/BRAINSia/BRAINSTools" target="_new">https://github.com/BRAINSia/BRAINSTools</a><br />
DO NOT USE THIS PAGE<br />
DO NOT USE THIS PAGE<br />
<br />
NOTE: All active development of BRAINS is now being stored in a Git repository at github.com. Please go to<br />
<br />
<a href="https://github.com/BRAINSia/BRAINSTools" target="_new">https://github.com/BRAINSia/BRAINSTools</a><br />
<br />
The BRAINS (Brain Research: Analysis of Images, Networks, and Systems) image analysis software has been developed to study the brain thus providing a better understand of psychiatric and neurological disorders. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 1, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BRAINS License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://github.com/BRAINSia">https://github.com/BRAINSia</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Iowa <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, MGH/MGZ, NIfTI-1, Nrrd, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, MGH/MGZ, NIfTI-1, Nrrd, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainstorm" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Brainstorm">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainstorm" rel="bookmark" name="brainstorm">Brainstorm</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=connectivity-analysis" rel="tag">Connectivity Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database-application" rel="tag">Database Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=eeg-modeling" rel="tag">Eeg Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=format-conversion" rel="tag">Format Conversion</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=meg-modeling" rel="tag">Meg Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=mesh-generation" rel="tag">Mesh Generation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=spatial-transformation" rel="tag">Spatial Transformation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=statistical-operation" rel="tag">Statistical Operation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=temporal-transformation" rel="tag">Temporal Transformation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=time-domain-analysis" rel="tag">Time Domain Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Brainstorm is a free Matlab application dedicated to MEG and EEG data visualization, processing and cortical source estimation.<br />
Its rich and intuitive graphic interface does not require any programming knowledge or experience with Matlab.<br />
<br />
MEG/EEG visualization and processing:<br />
- Support for most of the common file formats for MEG, EEG and MRI<br />
- Pre-processing: frequency filtering, resampling, artifact detection and correction, epoching, averaging<br />
- Visualization of recordings: time series, 2D/3D topographies<br />
- MRI/EEG/MEG co-registration<br />
- Powerful database explorer<br />
<br />
Source estimation:<br />
- Forward modeling: spherical models, overlapping spheres, BEM<br />
- Inverse modeling: minimum norm, dSPM, sLORETA, beamformers<br />
- Visualization of sources: cortex surface, MRI volume, normalization to atlases<br />
<br />
Post-processing:<br />
- Regions of interests at cortical level<br />
- Time-frequency analysis: Morlet wavelets, Hilbert transform, FFT, PSD<br />
- Group analysis: Registration of individual brains on a template, t-test, permutations 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=282&screenshot_id=523"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=282&screenshot_id=523" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 23, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://neuroimage.usc.edu/brainstorm">http://neuroimage.usc.edu/brainstorm</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Southern California <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, MGH/MGZ, NIfTI-1, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, MGH/MGZ, NIfTI-1, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainsuite" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BrainSuite">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainsuite" rel="bookmark" name="brainsuite">BrainSuite</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=clipping" rel="tag">Clipping</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=mesh-generation" rel="tag">Mesh Generation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-rendering" rel="tag">Surface Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=three-dimensional-display" rel="tag">Three Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=two-dimensional-display" rel="tag">Two Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volumetric-analysis" rel="tag">Volumetric Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> BrainSuite is a collection of open source software tools that enable largely automated processing of magnetic resonance images (MRI) of the human brain. The latest version of the BrainSuite software (v.16a1) is available for download from the BrainSuite website (<a href="http://brainsuite.org/" target="_new">http://brainsuite.org/</a>). This release includes a GUI for processing and visualization, tools to extract and parameterize cortex surface meshes, modules for automatic registration and labeling of brain volumes and surfaces, and distortion correction and coregistration of diffusion data with structural MRIs (available via the command line), for Windows, Mac OS X, and Linux platforms. BrainSuite source code is available under a GPLv2 license.<br />
<br />
BrainSuite is produced as a collaborative project between David Shattuck's research group at the UCLA Brain Mapping Center (<a href="http://shattuck.bmap.ucla.edu" target="_new">http://shattuck.bmap.ucla.edu</a>) and Richard Leahy's Biomedical Imaging Group at USC (<a href="http://neuroimage.usc.edu" target="_new">http://neuroimage.usc.edu</a>). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 21, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://brainsuite.org/">http://brainsuite.org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UCLA <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows NT/2000<br>Windows Vista<br>Windows XP</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainvisa" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BrainVISA">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainvisa" rel="bookmark" name="brainvisa">BrainVISA</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=diffusion-mr-fiber-tracking" rel="tag">Diffusion Mr Fiber Tracking</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=morphology" rel="tag">Morphology</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=platform-or-development-environment" rel="tag">Platform Or Development Environment</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quantitative-shape-analysis" rel="tag">Quantitative Shape Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=spatial-transformation" rel="tag">Spatial Transformation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-analysis" rel="tag">Surface Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tensor-metric" rel="tag">Tensor Metric</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> BrainVISA is a modular an customizable software platform built to host heterogeneous tools dedicated to neuroimaging research. Many toolboxes have already been developped for BrainVISA (T1 MRI, sulcal identification and morphometry, cortical surface analysis, diffusion imaging and tractography, fMRI, nuclear imaging, EEG and MEG, TMS, histology and autoradiography, etc.). <br />
<br />
BrainVISA main features are:<br />
- Harmonization of communications between different software. For instance, BrainVISA toolboxes are using home-made software but also third-party software such as FreeSurfer, FSL, SPM, nipy, R-project, Matlab, etc.<br />
- Ontology-based data organization allowing database sharing and automation of mass of data analysis.<br />
- Fusion and interactive visualization of multimodal data (using Anatomist software).<br />
- Automatic generation of graphical user interfaces.<br />
- Workflow monitoring and data quality checking.<br />
- Full customization possible.<br />
- Runs on Linux, Mac and Windows. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 6, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>CeCill v2</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://brainvisa.info">http://brainvisa.info</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, GIfTI, MINC, NIfTI-1, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, GIfTI, MINC, NIfTI-1, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainvisa-morphologist-extensions" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BrainVisa Morphologist extensions">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainvisa-morphologist-extensions" rel="bookmark" name="brainvisa-morphologist-extensions">BrainVisa Morphologist extensions</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-display" rel="tag">Image Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=morphology" rel="tag">Morphology</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=shape-analysis" rel="tag">Shape Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> BrainVisa Morphologist is collaborative BrainVisa extension library that provides computational tools for performing Object Based Morphology measurements to assess groupwise differences and track morphological changes to study aging, development, neurological and neurodegenerative disorders and genetic factors that affect them. This library includes source codes and methods for computation of regional GM thickness (multi-threaded and single thread), 3D gyrification index, sulcal lenght and depth and sulcal span and gyral white matter span.<br />
These tools are distributed as plugins for a popular analysis package BrainVisa 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=127&screenshot_id=135"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=127&screenshot_id=135" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 25, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Artistic License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/brainvisa_ext/">http://www.nitrc.org/projects/brainvisa_ext/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainvoyager" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BrainVoyager">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainvoyager" rel="bookmark" name="brainvoyager">BrainVoyager</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=eeg-modeling" rel="tag">Eeg Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=fiber-tracking" rel="tag">Fiber Tracking</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=granger-causality" rel="tag">Granger Causality</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=independent-component-analysis" rel="tag">Independent Component Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=meg-modeling" rel="tag">Meg Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=mesh-generation" rel="tag">Mesh Generation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=multivariate-analysis" rel="tag">Multivariate Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=platform-or-development-environment" rel="tag">Platform Or Development Environment</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=source-separation-analysis" rel="tag">Source Separation Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=spatial-transformation" rel="tag">Spatial Transformation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=thickness" rel="tag">Thickness</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=time-domain-analysis" rel="tag">Time Domain Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volume-measurement" rel="tag">Volume Measurement</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> BrainVoyager is a powerful neuroimaging software package. It started as a tool for the analysis of anatomical and functional MRI data sets, but has evolved over the years into a multi-modal analysis software suite for fMRI, DTI, TMS, EEG and MEG data. The software is highly optimized and user friendly running on all major computer platforms including Microsoft Windows (7, 8, 8.1 and 10), Linux (e.g. Ubuntu 14.04 (64 bit), recent versions of Red Hat Enterprise Linux (RHEL), Fedora and most other distributions with a 3.x Linux kernel) and Mac OS X (10.10 or 10.11; it should also run fine on 10.9 and 10.8). 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=309&screenshot_id=969"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=309&screenshot_id=969" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 9, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Commercial license from Brain Innovation B.V., Maastricht, The Netherlands</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.brainvoyager.com/">http://www.brainvoyager.com/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, EEG Formats, GIfTI, NIfTI-1, Philips PAR/REC</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, EEG Formats, GIfTI, NIfTI-1, Philips PAR/REC</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainvoyager-brain-tutor" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BrainVoyager Brain Tutor">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainvoyager-brain-tutor" rel="bookmark" name="brainvoyager-brain-tutor">BrainVoyager Brain Tutor</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=application" rel="tag">Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=information-resource" rel="tag">Information Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The free BrainVoyager Brain Tutor teaches you knowledge about the human brain through interactive exploration of rotatable 3D models. The models have been computed with BrainVoyager using original data from magnetic resonance imaging (MRI) scans. Besides having fun with the rotatable 3D models, the program contains information about the major lobes, gyri, sulci and Brodmann areas of the cerebral cortex. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=173&screenshot_id=89"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=173&screenshot_id=89" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 28, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/bvbraintutor/">http://www.nitrc.org/projects/bvbraintutor/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainvoyager-viewer" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BrainVoyager Viewer">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainvoyager-viewer" rel="bookmark" name="brainvoyager-viewer">BrainVoyager Viewer</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> BrainVoyager Brain Viewer allows to browse and inspect essential BrainVoyager data files. In addition, the program supports viewing the header and content of DICOM files. The Viewer also supports standard image files (JPEG, GIF, PNG, TIFF, BMP) allowing to inspect snapshots, figures or photos. The program offers an elegant user interface with fluid navigation abilities inspired by Apple's &quot;Cover Flow&quot; and the iPhone interface. The program can be downloaded and distributed freely. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=172&screenshot_id=88"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=172&screenshot_id=88" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 28, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/bvviewer/">http://www.nitrc.org/projects/bvviewer/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Windows XP</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>DICOM, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>DICOM, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brainwaver" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Brainwaver">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brainwaver" rel="bookmark" name="brainwaver">Brainwaver</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=application" rel="tag">Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Characterisation of small-world networks constructed from wavelet analysis of resting fMRI. This package is currently available as an R library. Futrure development will take place within the CamBA software repository </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 18, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.wbic.cam.ac.uk/~sa428/brainwaver/">http://www.wbic.cam.ac.uk/~sa428/brainwaver/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Other UNIX-like</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="bratumia-(brain-tumor-image-analysis)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BraTumIA (Brain Tumor Image Analysis)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=bratumia-(brain-tumor-image-analysis)" rel="bookmark" name="bratumia-(brain-tumor-image-analysis)">BraTumIA (Brain Tumor Image Analysis)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> BraTumIA, for Brain Tumor Image Analysis, is a software dedicated to multimodal image analysis of brain tumor studies. <br />
<br />
<br />
It performs volumetric segmentation of healthy and tumor tissues by employing multispectral MRI sequences (currently T1, T1-contrast, T2-contrast, and FLAIR). Segmented tissues include Gray Matter (GM), White Matter (WM), Cerebrospinal Fluid (CSF), necrotic core, edema, non-enhancing tumor and enhancing tumor.<br />
<br />
The current version of BraTumIA is designed to work on pre-operative images. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=817&screenshot_id=1196"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=817&screenshot_id=1196" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 20, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://mia-software.artorg.unibe.ch/registration/bratumia/requirement.php">http://mia-software.artorg.unibe.ch/registration/bratumia/requirement.php</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Bern, University Hospital Bern <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brava" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BraVa">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brava" rel="bookmark" name="brava">BraVa</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The BraVa database contains digital reconstructions of the human brain arterial arborizations from 61 healthy adult subjects along with extracted morphological measurements.<br />
<br />
The arterial arborizations include the six major trees stemming from the circle of Willis, namely: the left and right Anterior Cerebral Arteries (ACAs), Middle Cerebral Arteries (MCAs), and Posterior Cerebral Arteries (PCAs). 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=725&screenshot_id=650"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=725&screenshot_id=650" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 4, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://cng.gmu.edu/brava">http://cng.gmu.edu/brava</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>George Mason University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="bravissima:-human-brain-artery-nifti-atlas" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Bravissima: Human Brain Artery NIfTI Atlas">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=bravissima:-human-brain-artery-nifti-atlas" rel="bookmark" name="bravissima:-human-brain-artery-nifti-atlas">Bravissima: Human Brain Artery NIfTI Atlas</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-data" rel="tag">Atlas Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Bravissima is a translation of the BraVa arterial vasculature database (nitrc.org/projects/breva) into the more useful NIFTI MRI file format that can be applied to stroke studies, fMRI resting state imaging studies and other clinical neuroscience studies. We have successfully converted BraVa artery locations in both hemispheres for 43 healthy subjects into standard NIFTI MRI format files, including normalizing artery locations into MNI152 stereotaxic space. We have also included subject NIFTI files of the distance each arterial location is from the Circle of Willis as well as group maps of the locations of major arterial branch junctions; frequent locations of arterial occlusion. Group artery region labels and arterial density maps are provided as well. Matlab scripts used to produce all files are included. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1266&screenshot_id=1184"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1266&screenshot_id=1184" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 18, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/bravissima/">http://www.nitrc.org/projects/bravissima/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>US Department of Veterans Affairs <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brede-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Brede Toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brede-toolbox" rel="bookmark" name="brede-toolbox">Brede Toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database-application" rel="tag">Database Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=independent-component-analysis" rel="tag">Independent Component Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=ontology" rel="tag">Ontology</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=principal-component-analysis" rel="tag">Principal Component Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=regression" rel="tag">Regression</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Brede Toolbox is a package for neuroinformatics and neuroimaging analysis mostly programmed in Matlab with a few additional programs in Python and Perl. It allows coordinate-based meta-analysis and visualization, neuroimaging analysis of voxel or regional data. Among the algorithms implemented are kernel density estimation (for coordinate-based meta-analysis), independent component analysis, non-negative matrix factorization, k-means clustering, singular value decomposition, partial correlation analysis with permutation testing and partial canonical correlation analysis. Visualization of coordinate, surfaces and volumes are possible in 2D and 3D. <br />
 Generation of HTML for results are possible and algorithms can be accessed from the command line or via a flexible graphical interface. With the Brede Toolbox comes the Brede Database with a small coordinate database from published neuroimaging studies, and ontologies for, e.g., brain function and brain regions. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 20, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Brede Toolbox License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://neuro.compute.dtu.dk/software/brede/">http://neuro.compute.dtu.dk/software/brede/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Technical University of Denmark <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brede-wiki" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Brede Wiki">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brede-wiki" rel="bookmark" name="brede-wiki">Brede Wiki</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=knowledge-environment" rel="tag">Knowledge Environment</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=websites" rel="tag">Websites</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Brede Wiki is a semantic wiki with structured information from neuroscience. It contains listing of results from neuroimaging studies such as Talairach coordinates and brain volume measurements as well as items for researchers, software packages and brain regions. SQL dumps of the structured information in the wiki is available so complex queries can be formed.<br />
Search on Talairach coordinates in the database is available from an external specialized search engine. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 11, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://neuro.imm.dtu.dk/wiki/Main_Page">http://neuro.imm.dtu.dk/wiki/Main_Page</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="brian-simulator" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Brian simulator">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=brian-simulator" rel="bookmark" name="brian-simulator">Brian simulator</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Brian is a simulator for spiking neural networks available on almost all platforms. The motivation for this project is that a simulator should not only save the time of processors, but also the time of scientists.<br />
<br />
Brian is easy to learn and use, highly flexible and easily extensible. The Brian package itself and simulations using it are all written in the Python programming language, which is an easy, concise and highly developed language with many advanced features and development tools, excellent documentation and a large community of users providing support and extension packages. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=853&screenshot_id=831"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=853&screenshot_id=831" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 7, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>CeCill v2</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.briansimulator.org">http://www.briansimulator.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="broccoli" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BROCCOLI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=broccoli" rel="bookmark" name="broccoli">BROCCOLI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Analysis of functional magnetic resonance imaging (fMRI) data is becoming ever more computationally demanding as temporal and spatial resolutions improve, and large, publicly available data sets proliferate. Moreover, methodological improvements in the neuroimaging pipeline, such as non-linear spatial normalization, non-parametric permutation tests and Bayesian Markov Chain Monte Carlo approaches, can dramatically increase the computational burden. Here, we therefore present BROCCOLI, a free software package written in OpenCL (Open Computing Language) that can be used for parallel analysis of fMRI data on a large variety of hardware configurations. BROCCOLI (running on a GPU) can perform non-linear spatial normalization to a 1 mm brain template in 4–6 s, and run a second level permutation test with 10,000 permutations in about a minute. The new software is freely available under GNU GPL3 and can be downloaded from github (<a href="https://github.com/wanderine/BROCCOLI/" target="_new">https://github.com/wanderine/BROCCOLI/</a>). 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=828&screenshot_id=778"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=828&screenshot_id=778" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 9, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/broccoli/">http://www.nitrc.org/projects/broccoli/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>POSIX/UNIX-like<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="bsmac:-bayesian-spatial-model-for-brain-activation-and-connectivity" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BSMac: Bayesian Spatial Model for Brain Activation and Connectivity">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=bsmac:-bayesian-spatial-model-for-brain-activation-and-connectivity" rel="bookmark" name="bsmac:-bayesian-spatial-model-for-brain-activation-and-connectivity">BSMac: Bayesian Spatial Model for Brain Activation and Connectivity</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> We provide a statistical and graphical visualization MATLAB toolbox for the analysis of fMRI data, called the Bayesian Spatial Model for activation and connectivity (BSMac). BSMac simultaneously performs whole-brain activation analyses at the voxel and region of interest levels as well as task-related functional connectivity (FC) analyses using a flexible Bayesian modeling framework (Bowman et al., 2008). BSMac allows for inputting data in either Analyze or Nifti file formats. The user provides information pertaining to subgroup memberships, scanning sessions, and experimental tasks (stimuli), from which the design matrix is constructed. BSMac then performs parameter estimation based on MCMC methods and generates plots for activation and FC, such as interactive 2D maps of voxel and region-level task-related changes in neural activity and animated 3D graphics of the FC results. <br />
<br />
The toolbox can be downloaded: <a href="http://web1.sph.emory.edu/bios/CBIS/links.html" target="_new">http://web1.sph.emory.edu/bios/CBIS/link...</a> (<a href="http://www.ncbi.nlm.nih.gov/pubmed/22101143" target="_new">http://www.ncbi.nlm.nih.gov/pubmed/22101...</a>). 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=654&screenshot_id=569"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=654&screenshot_id=569" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 2, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License, GNU Lesser General Public License and GNU General Public License, v</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://web1.sph.emory.edu/bios/CBIS/download_page.php">http://web1.sph.emory.edu/bios/CBIS/download_page.php</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Emory University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="btk-:-baby-brain-toolkit" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BTK : Baby Brain Toolkit">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=btk-:-baby-brain-toolkit" rel="bookmark" name="btk-:-baby-brain-toolkit">BTK : Baby Brain Toolkit</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Fbrain is a ERC funded project. The purpose of this project is to develop image processing tools for a better understanding of the fetal brain development.<br />
<br />
BTK (baby brain toolkit), which is the toolkit developed for the fbrain project, consists of several image processing tools: image reconstruction, image denoising, image segmentation, tractography etc. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=683&screenshot_id=624"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=683&screenshot_id=624" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 11, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>CeCill v2</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/btk/">http://www.nitrc.org/projects/btk/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>CNRS <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="bxh/xcede-tools" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="BXH/XCEDE Tools">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=bxh/xcede-tools" rel="bookmark" name="bxh/xcede-tools">BXH/XCEDE Tools</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=format-conversion" rel="tag">Format Conversion</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quantification" rel="tag">Quantification</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=statistical-operation" rel="tag">Statistical Operation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> A collection of data processing and image analysis tools for data in BXH or XCEDE format. This includes data format encapsulation/conversion, event-related analysis, QA tools, and more. These tools form the basis of the fBIRN QA procedures and are also distributed as part of the fBIRN Data Upload Scripts. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 13, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BIRN License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/bxh_xcede_tools/">http://www.nitrc.org/projects/bxh_xcede_tools/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Other UNIX-like<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, MGH/MGZ, MINC, NIfTI-1, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, MGH/MGZ, MINC, NIfTI-1, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="c8:-corpus-callosum-computations" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="C8: Corpus Callosum Computations">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=c8:-corpus-callosum-computations" rel="bookmark" name="c8:-corpus-callosum-computations">C8: Corpus Callosum Computations</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> C8 is a small, stand-alone MatLab toolbox that measures sagittal cross-section thickness and area of the human corpus callosum from high-resolution T1 in vivo MR images. C8 takes as input affine normalized white matter segmentations derived from high-resolution (in-plane) T1 images and outputs both regional callosal thicknesses in three different formats and geometrically-defined regional areas in three different configurations. It is a small package that is easily configurable and modifiable and it measures callosa at the rate of several per minute. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=484&screenshot_id=355"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=484&screenshot_id=355" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 3, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/c8c8/">http://www.nitrc.org/projects/c8c8/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cagwas" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="caGWAS">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cagwas" rel="bookmark" name="cagwas">caGWAS</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=genetic-association" rel="tag">Genetic Association</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Cancer Genome-Wide Association Studies (caGWAS) allows researchers to integrate, query, report, and analyze significant associations between genetic variations and disease, drug response or other clinical outcomes. SNP array technologies make it possible to genotype hundreds of thousands of single nucleotide polymorphisms (SNPs) simultaneously, enabling whole genome association studies. Within the Clinical Genomic Object Model (CGOM), the caIntegrator team created a domain model for Whole Genome Association Study Analysis. CGOM-caGWAS is a semantically annotated domain model that captures associations between Study, Study Participant, Disease, SNP Association Analysis, SNP Population Frequency and SNP annotations. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=709&screenshot_id=643"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=709&screenshot_id=643" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 25, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://wiki.nci.nih.gov/display/caGWAS/caGWAS">http://wiki.nci.nih.gov/display/caGWAS/caGWAS</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>National Cancer Informatics Program <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="calatk---atlas-building-tk" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CalaTK - Atlas Building TK">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=calatk---atlas-building-tk" rel="bookmark" name="calatk---atlas-building-tk">CalaTK - Atlas Building TK</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-to-image" rel="tag">Image-To-Image</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> CalaTK is an open-source toolkit for cross-sectional and longitudinal atlas building.<br />
<br />
The CalaTK project develops innovative methods and tools for longitudinal atlases with a focus on neurodevelopment. The computational toolbox is developed with the objective to analyze the neural developmental patterns observed in human and non-human primate structural and diffusion tensor magnetic resonance (MR) images. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=630&screenshot_id=551"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=630&screenshot_id=551" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 14, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.calatk.org/">http://www.calatk.org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina at Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="calcfd---calculate-the-fractal-dimensionality-of-a-3d-structure" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="calcFD - Calculate the fractal dimensionality of a 3D structure">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=calcfd---calculate-the-fractal-dimensionality-of-a-3d-structure" rel="bookmark" name="calcfd---calculate-the-fractal-dimensionality-of-a-3d-structure">calcFD - Calculate the fractal dimensionality of a 3D structure</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> A toolbox for MATLAB for calculating the fractal dimensionality of a 3D structure, designed to work with intermediate files from FreeSurfer analysis pipeline, but can also use other volumes. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1179&screenshot_id=1098"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1179&screenshot_id=1098" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 3, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License v3</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://github.com/cMadan/calcFD">https://github.com/cMadan/calcFD</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="camba" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CamBA">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=camba" rel="bookmark" name="camba">CamBA</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=affine-warp" rel="tag">Affine Warp</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-to-template" rel="tag">Image-To-Template</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=regression" rel="tag">Regression</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=spectral-analysis" rel="tag">Spectral Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=time-domain-analysis" rel="tag">Time Domain Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=wavelet-transformation" rel="tag">Wavelet Transformation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Software respository containing pipelines of chained modules and GUI for batch processing. Current pipelines include fMRI analysis, statistical testing based on randomisation methods and fractal spectral analysis. Pipelines are continually being added. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 18, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www-bmu.psychiatry.cam.ac.uk/software">http://www-bmu.psychiatry.cam.ac.uk/software</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="camino" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Camino">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=camino" rel="bookmark" name="camino">Camino</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=diffusion-mr-fiber-tracking" rel="tag">Diffusion Mr Fiber Tracking</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=q-ball" rel="tag">Q-Ball</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tensor-metric" rel="tag">Tensor Metric</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Camino is a free, open-source, object-oriented software package for analysis and reconstruction of Diffusion MRI data, tractography and connectivity mapping.<br />
<br />
Please see the Camino home page (camino.org.uk) for documentation and tutorials. If you need support, join the Camino mailing list on this site. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=253&screenshot_id=508"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=253&screenshot_id=508" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 30, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Artistic License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://camino.cs.ucl.ac.uk">http://camino.cs.ucl.ac.uk</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS<br>POSIX/UNIX-like</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="camino-trackvis" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CAMINO-TRACKVIS">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=camino-trackvis" rel="bookmark" name="camino-trackvis">CAMINO-TRACKVIS</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> With increasing efforts on brain connectivity analyses it becomes important to have tools that can allow increased interoperability among different tractography tools. This package allows interoperability between CAMINO and TRACKVIS. CAMINO is a leading software package in DTI processing. The package is from University of College London. TRACKVIS is a tract visualizing utility with capability of visualizing up to and over a million white matter tracts seamlessly. The package is from Massachusetts General Hospital.<br />
<br />
The tools in this package allow conversion of tracts from one format to another in a very effective way with ability to handle over a million tracts.<br />
<br />
Please use the sourceforge link below to download source code, instructions example scripts and other miscellaneous tools.<br />
<a href="http://sourceforge.net/projects/camino-trackvis/" target="_new">http://sourceforge.net/projects/camino-t...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 19, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MIT/X Consortium License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/camino-trackvis/">http://www.nitrc.org/projects/camino-trackvis/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="candi-neuroimaging-access-point" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CANDI Neuroimaging Access Point">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=candi-neuroimaging-access-point" rel="bookmark" name="candi-neuroimaging-access-point">CANDI Neuroimaging Access Point</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-data" rel="tag">Atlas Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=knowledge-environment" rel="tag">Knowledge Environment</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volumetric-analysis" rel="tag">Volumetric Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=web-resource" rel="tag">Web Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Child and Adolescent NeuroDevelopment Initiative (CANDI) at UMass Medical School is making available a series of structural brain images, as well as their anatomic segmentations, demographic and behavioral data and a set of related morphometric resources (static and dynamic atlases).<br />
<br />
Schiz Bull 2008 data is now available on NITRC-IR. Please register for access: <a href="http://www.nitrc.org/project/request.php?group_id=377" target="_new">http://www.nitrc.org/project/request.php...</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=377&screenshot_id=456"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=377&screenshot_id=456" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 7, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://labs.umassmed.edu/candi/">http://labs.umassmed.edu/candi/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Massachusetts Medical School <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="candi-share:-schizophrenia-bulletin-2008" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CANDI Share: Schizophrenia Bulletin 2008">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=candi-share:-schizophrenia-bulletin-2008" rel="bookmark" name="candi-share:-schizophrenia-bulletin-2008">CANDI Share: Schizophrenia Bulletin 2008</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This project hosts data for CANDI Share Schizophrenia Bulletin 2008 (reference below) as part of the CANDI Neuroimaging Access Point. This set includes preprocessed MRI images and segmentation results of all 4 diagnostic groups (Healthy Controls, N=29; Schizophrenia Spectrum, N=20; Bipolar Disorder with Psychosis, N=19; and Bipolar Disorder without Psychosis, N=35).<br />
<br />
Frazier JA, Hodge SM, Breeze JL, Giuliano AJ, Terry JE, Moore CM, Kennedy DN, Lopez-Larson MP, Caviness VS, Seidman LJ, Zablotsky B, Makris N. Diagnostic and sex effects on limbic volumes in early-onset bipolar disorder and schizophrenia. Schizophr Bull. 2008 Jan;34(1):37-46. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 9, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cs_schizbull08/">http://www.nitrc.org/projects/cs_schizbull08/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Massachusetts Medical School <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="caret" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Caret">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=caret" rel="bookmark" name="caret">Caret</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=animation" rel="tag">Animation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=gaussian-curvature" rel="tag">Gaussian Curvature</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-to-template" rel="tag">Image-To-Template</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=intersubject" rel="tag">Intersubject</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=mean-curvature" rel="tag">Mean Curvature</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=mesh-generation" rel="tag">Mesh Generation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quantitative-shape-analysis" rel="tag">Quantitative Shape Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=shape-analysis" rel="tag">Shape Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-rendering" rel="tag">Surface Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=three-dimensional-display" rel="tag">Three Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=two-dimensional-display" rel="tag">Two Dimensional Display</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Caret is free, open source software used to visualize and analyze the structural and functional characteristics of cerebral and cerebellar cortex in humans, nonhuman primates, and rodents. Visit the Tool/Resource Home Page for more information. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 26, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://brainvis.wustl.edu/wiki/index.php/Caret:About">http://brainvis.wustl.edu/wiki/index.php/Caret:About</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Washington University in St. Louis <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>AFNI BRIK, ANALYZE, MINC, NIfTI-1, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>AFNI BRIK, ANALYZE, MINC, NIfTI-1, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="carlsim:-a-gpu-accelerated-snn-simulator" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CARLsim: a GPU-accelerated SNN Simulator">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=carlsim:-a-gpu-accelerated-snn-simulator" rel="bookmark" name="carlsim:-a-gpu-accelerated-snn-simulator">CARLsim: a GPU-accelerated SNN Simulator</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Installation (see <a href="http://www.socsci.uci.edu/~jkrichma/CARLsim/index.html" target="_new">http://www.socsci.uci.edu/~jkrichma/CARL...</a>)<br />
<br />
Supported platforms: Linux / Windows<br />
<br />
Dependencies: NVIDIA CUDA SDK<br />
The code has been predominately developed under Linux and the build instructions reflect that. While the code should function under MS Windows these instructions are not applicable. We will hopefully release instructions in the future on how to setup the code base under Visual Studio on Windows based systems.<br />
Detailed installation instructions can be found in the file INSTALL in the code package. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 6, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/carlsim/">http://www.nitrc.org/projects/carlsim/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of California, Irvine <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cavernous-hemangioma:-epilepsy-&-add" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Cavernous Hemangioma: Epilepsy & ADD">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cavernous-hemangioma:-epilepsy-&-add" rel="bookmark" name="cavernous-hemangioma:-epilepsy-&-add">Cavernous Hemangioma: Epilepsy & ADD</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This resource is populated with information related to a single subject with multiple cavernous hemangiomas with a diagnosis of epilepsy and ADD. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 22, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cavernoush/">http://www.nitrc.org/projects/cavernoush/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="caworks" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CAWorks">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=caworks" rel="bookmark" name="caworks">CAWorks</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=three-dimensional-display" rel="tag">Three Dimensional Display</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Current enhancements to CAWorks have focused on interactive support for landmarking of subcortical structures. Specific plugins are available for landmark placement of the hippocampus, amygdala and entorhinal cortex regions. After landmarking is completed, CAWorks facilitates submission for automated segmentation processing. CAWorks is being used by the Biomarkers for Older Controls At Risk for Dementia (BIOCARD), Resource for Quantitative Functional MRI: TRD4, Validation of Structural/Functional MRI Localization and Biomedical Informatics Research Network (BIRN) projects.<br />
<br />
CAWorks has been further enhanced with a browser plugin module for the Extensible Neuroimaging Archive Toolkit (XNAT). The XNAT software facilitates the storage and management of neuroimaging and associated data. The XNAT browser enables the retrieval of medical image data from XNAT for analysis in CAWorks and the storage of CAWorks analysis results in XNAT. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 21, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BIRN License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.cis.jhu.edu/software/caworks/">http://www.cis.jhu.edu/software/caworks/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Johns Hopkins University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cbfbirn" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CBFBIRN">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cbfbirn" rel="bookmark" name="cbfbirn">CBFBIRN</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Cerebral Blood Flow Biomedical Informatics Research Network (CBFBIRN) (<a href="https://cbfbirn.ucsd.edu" target="_new">https://cbfbirn.ucsd.edu</a>) provides a web based central repository for individual and group analysis of Arterial Spin Labeling (ASL) data sets and ASL pulse sequences developed at CMFRI UCSD for MRI researchers. <br />
<br />
This resource currently hosts more 1300 ASL data sets from 22 projects and consists of mainly two main tools<br />
1) The Cerebral Blood Flow Database and Analysis Pipeline (CBFDAP) is a web enabled data and workflow management system extended from the HID codebase on NITRC specialized for Arterial Spin Labeling data management and analysis (including group analysis) in a centralized manner.<br />
2) Pulse Sequence Distribution System (PSDS) for managing dissamination of ASL pulse sequences developed at the UCSD CFMRI.<br />
<br />
This resource also includes web and video tutorials for end users. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 26, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BIRN License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://cbfbirn.ucsd.edu/">http://cbfbirn.ucsd.edu/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UCSD <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cbica:-brain-anatomy-simulator" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CBICA: Brain Anatomy Simulator">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cbica:-brain-anatomy-simulator" rel="bookmark" name="cbica:-brain-anatomy-simulator">CBICA: Brain Anatomy Simulator</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This package estimates the statistical properties of high-dimensional deformation fields[6], which are produced by deformable registration packages like HAMMER above, and then uses the estimates statistics to simulate brain images with very high degree of realism. The statistical properties of a family of deformations are estimated via a combination of wavelet packet decomposition on the deformations and their Jacobians, and PCA. This package can be used to better represent the statistical priors of deformation fields in deformable registration methods that use priors, such as active shape models. Moreover, SSD can be used to generate very realistic and rich deformations, which can be used for generation of gold standard against which different registration methods can be compared. Finally, a method for generation of simulated tissue atrophy is supplied as a separate package [7], which further enables simulation and validation studies. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 21, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>SBIA License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/brainanatsim/">http://www.nitrc.org/projects/brainanatsim/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pennsylvania <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cbica:-brain-tumor-modeling---coupled-solver" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CBICA: Brain Tumor Modeling - Coupled Solver">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cbica:-brain-tumor-modeling---coupled-solver" rel="bookmark" name="cbica:-brain-tumor-modeling---coupled-solver">CBICA: Brain Tumor Modeling - Coupled Solver</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This is software package for tumor growth modeling employs a DiffusionSolver approach. It is a one-step further compared to the previous purely mechanical, pressure-based approach employed in ElasticSolver. In addition to the previous elasticity-based approach to simulate brain tissue deformations caused by growing tumors (mass-effect), a nonlinear reaction-advection-diffusion equation to describe the tumor spatio-temporal evolution is added. This equation has a two-way coupling with the underlying tissue elastic deformation. In this approach, the forces exerted by the tumor growth and infiltration onto the underlying brain parenchyma are local ones, proportional to local tumor density gradients. Curretnly BTMCS is used in GLioma Image SegmenTation and Registration (GLISTR) and Pre-Operative and post-Recurrence brain Tumor Registration (PORTR). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 21, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>SBIA License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/btmcs/">http://www.nitrc.org/projects/btmcs/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pennsylvania <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cbica:-brain-tumor-viewer-(btv)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CBICA: Brain Tumor Viewer (BTV)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cbica:-brain-tumor-viewer-(btv)" rel="bookmark" name="cbica:-brain-tumor-viewer-(btv)">CBICA: Brain Tumor Viewer (BTV)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> BrainTumorViewer has been superceded by the Cancer Imaging Pheonmics Toolkit (CaPTk) Console: <a href="https://www.nitrc.org/projects/captk/" target="_new">https://www.nitrc.org/projects/captk/</a><br />
<br />
-----------------------------<br />
<br />
BrainTumorViewer (BTV) is a lightweight viewer, built for fast and simple interaction with MRI image volumes.<br />
<br />
BTV is primarily designed for visualizing multi-modal MRI brain volumes and initializing seed-points for the following software packages:<br />
<br />
i) GLioma Image SegmenTation and Registration (GLISTR)<br />
ii) Pre-Operative and post-Recurrence brain Tumor Registration (PORTR)<br />
<br />
After initializing these seed-points, a user can generate execution scripts through BTV's graphical user interface for various platforms and custom input parameters. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 22, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>SBIA License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cbica_btv/">http://www.nitrc.org/projects/cbica_btv/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pennsylvania <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cbica:-build-system-and-software-implementation-standard-(basis)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CBICA: Build system And Software Implementation Standard (BASIS)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cbica:-build-system-and-software-implementation-standard-(basis)" rel="bookmark" name="cbica:-build-system-and-software-implementation-standard-(basis)">CBICA: Build system And Software Implementation Standard (BASIS)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The CMake Build system And Software Implementation Standard (BASIS) makes it easy to create sharable software and libraries that work together. This is accomplished by combining and documenting some of the best practices, utilities, and open source projects available. More importantly, BASIS supplies a fully integrated suite of functionality to make the whole process seamless!<br />
<br />
NOTE: This tool has been deprecated and is no longer actively maintained. Details at <a href="https://cmake-basis.github.io/index.html" target="_new">https://cmake-basis.github.io/index.html</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 6, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>SBIA License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/basis/">http://www.nitrc.org/projects/basis/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pennsylvania <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cbica:-cancer-imaging-phenomics-toolkit-(captk)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CBICA: Cancer Imaging Phenomics Toolkit (CaPTk)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cbica:-cancer-imaging-phenomics-toolkit-(captk)" rel="bookmark" name="cbica:-cancer-imaging-phenomics-toolkit-(captk)">CBICA: Cancer Imaging Phenomics Toolkit (CaPTk)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=application" rel="tag">Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Cancer Imaging Phenomics Toolkit (CaPTk) is a comprehensive imaging analytics suite providing algorithms that produce extensive panels of quantitative imaging features. Applications in CaPTk use those imaging features in diagnostic and predictive models to support optimized, personalized treatment planning.<br />
<br />
The software has been designed for research purposes only and has not been reviewed or approved by the Food and Drug Administration or by any other agency. It is not intended or recommended for clinical applications.<br />
<br />
DEPRECATED: Our NITRC page is only used to distribute binaries; for issues and other communication, please see our GitHub page at github.com/cbica/captk. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 22, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>SBIA License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://www.cbica.upenn.edu/captk">https://www.cbica.upenn.edu/captk</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pennsylvania <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>DICOM, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>DICOM, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cbica:-chimera" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CBICA: Chimera">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cbica:-chimera" rel="bookmark" name="cbica:-chimera">CBICA: Chimera</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> CHIMERA is a probabilistic clustering approach that models the pathological process by a combination of multiple regularized transformations from normal control population to the patient population, while controlling the similarity in covariates (e.g. age, gender, height). Therefore, it seeks to identify multiple imaging patterns that relate to disease effects and to better characterize disease heterogeneity. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 17, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>SBIA License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cbica_chimera/">http://www.nitrc.org/projects/cbica_chimera/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pennsylvania <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cbica:-compare" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CBICA: COMPARE">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cbica:-compare" rel="bookmark" name="cbica:-compare">CBICA: COMPARE</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> COMPARE :Generic classification tool for 3D images </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 27, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>SBIA License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/compare/">http://www.nitrc.org/projects/compare/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cbica:-confetti" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CBICA: Confetti">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cbica:-confetti" rel="bookmark" name="cbica:-confetti">CBICA: Confetti</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=connectivity-analysis" rel="tag">Connectivity Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Confetti is a method for automated identification of white matter tracts of interest in a consistent and comparable manner over a large group of subjects without drawing the inclusion and exclusion regions of interest (ROI), facilitating an easy correspondence between different subjects, as well as providing a representation that is robust to edema, mass effect, and tract infiltration.<br />
Confetti includes three main steps: (1) Connectivity signature generation for fibers, (2) Clustering of fibers using a mixture of multinomials (MNs) clustering method and Expectation-Maximization (EM) optimization framework, (3) Extraction of predefined white matter tracts. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 25, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>CBICA License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cbica_confetti/">http://www.nitrc.org/projects/cbica_confetti/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pennsylvania <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cbica:-datasets" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CBICA: Datasets">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cbica:-datasets" rel="bookmark" name="cbica:-datasets">CBICA: Datasets</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Meta-project, providing datasets associated with various CBICA projects: Mouse Brain, Facial Expression/Schizophrenia, Prostate, etc. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 14, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>SBIA License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cbica_datasets/">http://www.nitrc.org/projects/cbica_datasets/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pennsylvania <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cbica:-dtidroid" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CBICA: DTIDROID">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cbica:-dtidroid" rel="bookmark" name="cbica:-dtidroid">CBICA: DTIDROID</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This software implements a method for deformable registration of diffusion tensor (DT) images (DROID) that integrates geometry and orientation features into a hierarchical matching framework. The geometric feature is derived from the structural geometry of diffusion and characterizes the shape of the tensor in terms of prolateness, oblateness, and sphericity of the tensor. Local spatial distributions of the prolate, oblate, and spherical geometry are used to create an attribute vector of geometric feature for matching. The orientation feature improves the matching of the WM fiber tracts by taking into account the statistical information of underlying fiber orientations. These features are incorporated into a hierarchical deformable registration framework to develop a diffusion tensor image registration algorithm. Extensive experiments on simulated and real brain DT data establish the superiority of this algorithm for deformable matching of diffusion tensors, thereby aiding in atlas creation. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 16, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>SBIA License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dtidroid/">http://www.nitrc.org/projects/dtidroid/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pennsylvania <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cbica:-geodesic-graph-based-segmentation-with-shape-priors-(grasp)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CBICA: Geodesic graph-based segmentation with shape priors (GraSP)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cbica:-geodesic-graph-based-segmentation-with-shape-priors-(grasp)" rel="bookmark" name="cbica:-geodesic-graph-based-segmentation-with-shape-priors-(grasp)">CBICA: Geodesic graph-based segmentation with shape priors (GraSP)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> GraSP is a graph-based parcellation software.<br />
GraSP was initially developed for parcellating the cortex into functionally coherent regions, based on their Pearson correlation, that presents different cortical parcellations of the left hemisphere. However, the software can handle any graph, and 3D parcellations can be obtained easily. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 22, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>SBIA License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cbica_grasp/">http://www.nitrc.org/projects/cbica_grasp/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pennsylvania <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cbica:-glioma-image-segmentation-and-registration-(glistr)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CBICA: GLioma Image SegmenTation and Registration (GLISTR)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cbica:-glioma-image-segmentation-and-registration-(glistr)" rel="bookmark" name="cbica:-glioma-image-segmentation-and-registration-(glistr)">CBICA: GLioma Image SegmenTation and Registration (GLISTR)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> GLioma Image SegmenTation and Registration (GLISTR) is a software package designed for simultaneously segmenting brain scans of glioma patients and registering these scans to a normal atlas.<br />
<br />
Some typical applications of GLISTR include,<br />
<br />
 Labeling entire brain regions of glioma patients;<br />
 Mapping gliomas into the healthy atlas space;<br />
 Estimating parameters of the tumor growth model.<br />
<br />
GLISTR is implemented as a command-line tool. It is semi-automatic and requires minimal user initializations. Users could use the visual interface called BrainTumorViewer to easily make initializations and a script for the execution. As a results, GLISTR will output a label map, a mapping between atlas and input, tumor parameters, etc. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 22, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>SBIA License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.med.upenn.edu/sbia/glistr.html">http://www.med.upenn.edu/sbia/glistr.html</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pennsylvania <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cbica:-gondola" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CBICA: GONDOLA">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cbica:-gondola" rel="bookmark" name="cbica:-gondola">CBICA: GONDOLA</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This software implements Generative-Discriminative Basis Learning (GONDOLA). Theoretical ideas are explained in the related papers, but as a brief explanation, GONDOLA provides a generative method to reduce the dimensionality of medical images while using class labels. It produces basis vectors that are useful for classification and also clinically interpretable. When provided with two sets of labeled images as input, the software outputs features in Weka Format (.arff files) and a MATLAB data file (.mat file). The program can also save basis vectors as NIfTI-1 images. Scripts are provided to find and build an optimal classifier using Weka. The software can also be used for semi-supervised cases in which a number of subjects do not have class labels. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 17, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>SBIA License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/gondola/">http://www.nitrc.org/projects/gondola/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pennsylvania <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cbica:-identification-of-sparse-connectivity-patterns-in-rsfmri-(scplearn)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CBICA: Identification of Sparse Connectivity Patterns in rsfMRI (SCPLearn)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cbica:-identification-of-sparse-connectivity-patterns-in-rsfmri-(scplearn)" rel="bookmark" name="cbica:-identification-of-sparse-connectivity-patterns-in-rsfmri-(scplearn)">CBICA: Identification of Sparse Connectivity Patterns in rsfMRI (SCPLearn)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This software is used to calculate Sparse Connectivity Patterns (SCPs) from resting state fMRI connectivity data. SCPs consist of those regions whose between-region connectivity co-varies across subjects. This algorithm was developed as a complementary approach to existing network identification methods.<br />
<br />
SCPLearn has the following advantages:<br />
<br />
 Does not require thresholding of correlation matrices<br />
 Allows for both positive and negative correlations<br />
 Does not constrain the SCPs to have spatial/temporal orthogonality/independence<br />
 Provides group-common SCPs and subject-specific measures of average correlation within each SCP<br />
 Can be run within a hierarchical framework to get &quot;primary&quot; (large spatial extent) and &quot;secondary&quot; level (small spatial extent) SCPs<br />
<br />
Subject-level coefficients can be used for subsequent group-level analysis. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 22, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>SBIA License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cbica_scplearn/">http://www.nitrc.org/projects/cbica_scplearn/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pennsylvania <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cbica:-laboratory-for-individualized-breast-radiodensity-assessment-(libra)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CBICA: Laboratory for Individualized Breast Radiodensity Assessment (LIBRA)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cbica:-laboratory-for-individualized-breast-radiodensity-assessment-(libra)" rel="bookmark" name="cbica:-laboratory-for-individualized-breast-radiodensity-assessment-(libra)">CBICA: Laboratory for Individualized Breast Radiodensity Assessment (LIBRA)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Laboratory for Individualized Breast Radiodensity Assessment (LIBRA) is a fully-automatic breast density estimation solution based on a published algorithm that works on either raw (i.e., “FOR PROCESSING”) or vendor post-processed (i.e., “FOR PRESENTATION”) digital mammography images from two vendors: GE Healthcare and Hologic. LIBRA has been applied to over 70,000 screening exams and is being increasingly utilized in larger studies.<br />
<br />
LIBRA has two modes of operations:<br />
<br />
1. An easy-to-use interactive mode with Graphical-User-Interface where the user is prompted to select either a single DICOM image or a folder of DICOM images, an output folder for the results, and whether they wish to save intermediate files.<br />
2. A command-line interface amenable to batch processing and scripting, where the user can explicitly define the input and output paths.<br />
<br />
For more info click Homepage in the left panel. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1011&screenshot_id=1017"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1011&screenshot_id=1017" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 22, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>SBIA License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://www.nitrc.org/projects/cbica_libra/">https://www.nitrc.org/projects/cbica_libra/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pennsylvania <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cbica:-moe" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CBICA: MOE">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cbica:-moe" rel="bookmark" name="cbica:-moe">CBICA: MOE</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This software can be used for datasets where the reference group (&quot;controls&quot;) and the affected group (&quot;patients&quot;) cannot be separated by a single line (&quot;hyperplane&quot;). MOE combines multiple hyperplanes along with a clustering objective, to split the affected group into multiple sub-groups. This is done in such a manner that each of the resulting sub-group is separable from the reference group by a single line. <br />
<br />
See Figure in test/All_test_results.png, where the reference group is denoted by circles, and the affected group by triangles. In Case1, the affected group is not heterogeneous, as it is seperable from the reference group with a single line. Cases 2, 3, 4 require multiple lines to separate the affected group from the reference group.<br />
<br />
This software works with data saved in comma-separated value format. Input features can be based on any type of data - volumes, density, connectivity, diffusion, clinical scores, cognitive data etc. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 16, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>SBIA License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/moe/">http://www.nitrc.org/projects/moe/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pennsylvania <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cbica:-multi-atlas-region-segmentation-utilizing-ensembles-of-registration-algorithms-and-parameters-(muse)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CBICA: MUlti-atlas region Segmentation utilizing Ensembles of registration algorithms and parameters (MUSE)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cbica:-multi-atlas-region-segmentation-utilizing-ensembles-of-registration-algorithms-and-parameters-(muse)" rel="bookmark" name="cbica:-multi-atlas-region-segmentation-utilizing-ensembles-of-registration-algorithms-and-parameters-(muse)">CBICA: MUlti-atlas region Segmentation utilizing Ensembles of registration algorithms and parameters (MUSE)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MUSE: MUlti-atlas region Segmentation utilizing Ensembles of registration algorithms and parameters, and locally optimal atlas selection<br />
<br />
MUSE generates a large ensemble of candidate labels in the target image space using multiple atlases, registration algorithms and smoothness values for these algorithms. The ensemble is then fused into a final segmentation. <br />
<br />
MUSE is a Linux command-line tool, with support for several HPC schedulers or the ability to run as multiple processes on a single workstation. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 22, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>SBIA License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cbica_muse/">http://www.nitrc.org/projects/cbica_muse/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pennsylvania <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cbica:-multi-atlas-skull-stripping-(mass)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CBICA: Multi Atlas Skull Stripping (MASS)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cbica:-multi-atlas-skull-stripping-(mass)" rel="bookmark" name="cbica:-multi-atlas-skull-stripping-(mass)">CBICA: Multi Atlas Skull Stripping (MASS)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Multi Atlas Skull Stripping (MASS) [ARAD2013], is a software package designed for robust and accurate brain extraction, applicable for both individual as well as large population studies.<br />
<br />
MASS is implemented as a Unix command-line tool. It is fully automatic and easy to use — users input an image, and MASS will output the extracted brain and the associated brain mask. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 22, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>SBIA License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cbica_mass/">http://www.nitrc.org/projects/cbica_mass/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pennsylvania <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cbica:-odvba" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CBICA: ODVBA">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cbica:-odvba" rel="bookmark" name="cbica:-odvba">CBICA: ODVBA</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ODVBA provides a mathematically rigorous framework for determining the optimal spatial smoothing of structural and functional images, prior to applying voxel-based group analysis. In order to determine the optimal smoothing kernel, a local discriminative analysis, restricted by appropriate nonnegativity constraints, is applied to a spatial neighborhood around each voxel, aiming to find the direction best highlights the difference between two groups in that neighborhood. Since each voxel belongs to a large number of such neighborhoods, each centered on one of its neighboring voxels, the group difference at each voxel is determined by a composition of all these optimal smoothing directions. Permutation tests are used to obtain the statistical significance of the resulting Optimally-Discriminative VBM (ODVBA) maps. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 16, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>SBIA License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/odvba/">http://www.nitrc.org/projects/odvba/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pennsylvania <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cbica:-phi-estimator" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CBICA: PHI Estimator">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cbica:-phi-estimator" rel="bookmark" name="cbica:-phi-estimator">CBICA: PHI Estimator</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Peritumoral Heterogeneity Index (PHI) Estimator is a lightweight tool built towards the following goals:<br />
Perform quantitative pattern analysis of the spatial heterogeneity of peritumoral perfusion imaging dynamics, retrieved from Dynamic Susceptibility Contrast Magnetic Resonance Imaging (DSC-MRI) data. <br />
Evaluate the imaging biomarker of the Epidermal Growth Factor Receptor variant III (EGFRvIII) mutation status, in individual patients diagnosed with Glioblastoma. After v1.0.0, this will get updated in the CaPTk project as &quot;EGFRvIII Surrogate Index&quot;. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 21, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>SBIA License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/phiestimator/">http://www.nitrc.org/projects/phiestimator/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pennsylvania <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cbica:-portr" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CBICA: PORTR">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cbica:-portr" rel="bookmark" name="cbica:-portr">CBICA: PORTR</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Pre-Operative and post-Recurrence brain Tumor Registration (PORTR) is a software package designed for determining the optimal deformation between pre-operative and post-recurrence scans by finding the minimum of an energy function, which is based on the concept of symmetric registration.<br />
<br />
Some typical applications of PORTR include:<br />
 - Mapping post-recurrence scans to pre-operative scans<br />
 - Labeling entire brain regions of each scan.<br />
<br />
PORTR is implemented as a command-line tool. It is semi-automatic and requires minimal user initializations. Users could use the visual interface called CaPTk to easily make initializations and a script for the execution. As a result, PORTR will output a label map of each scan, a mapping between two scans, etc. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 16, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>SBIA License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/portr/">http://www.nitrc.org/projects/portr/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pennsylvania <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cbica:-predict" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CBICA: PREDICT">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cbica:-predict" rel="bookmark" name="cbica:-predict">CBICA: PREDICT</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> In our approach to simulate brain tissue atrophy, we adapt the strategy delineated by the well-known Occam''s razor: solution to problems must be as simple as possible, but not simpler. In the case of brain tissue atrophy, in the absence of a priori knowledge of precisely how a specific medical condition causes tissue loss and in which pattern, the only information available is that the apparent volume of the tissue is reduced. We therefore proceed to find a deformation that corresponds to a reduction of tissue volume in a specified region of the brain. Specifically, given a prescribed level of tissue volume after atrophy over an image, we solve for a warping deformation that produces the desired volumetric changes using an energy minimization approach. The advantages of the method is that it is automated, easily generalizable to other instances of spatially varying volumetric change, and can readily be modified to incorporate a priori knowledge in the form of a statistical atlas of tissue loss. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 21, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>SBIA License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/atrophysim/">http://www.nitrc.org/projects/atrophysim/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pennsylvania <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cbica:-tutorials-(image-processing-and-machine-learning)-using-cpp,-itk,-etc." class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CBICA: Tutorials (Image Processing and Machine Learning) using CPP, ITK, etc.">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cbica:-tutorials-(image-processing-and-machine-learning)-using-cpp,-itk,-etc." rel="bookmark" name="cbica:-tutorials-(image-processing-and-machine-learning)-using-cpp,-itk,-etc.">CBICA: Tutorials (Image Processing and Machine Learning) using CPP, ITK, etc.</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=community" rel="tag">Community</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Welcome to CBICA’s C++ learning resource.<br />
<br />
Here, we will be showcasing our seminar series “CPP for Image Processing and Machine Learning” including presentations and code examples.<br />
<br />
There are image processing and machine learning libraries out there which use C++ as a base and have become industry standards (ITK for medical imaging, OpenCV for computer vision and machine learning, Eigen for linear algebra, Shogun for machine learning). The documentation provided with these packages, though extensive, assume a certain level of experience with C++. Our tutorials are intended for those people who have basic understanding of medical image processing and machine learning but who are just starting to get their toes wet with C++ (and possibly have prior experience with Python or MATLAB).<br />
<br />
Here we will be focusing on how someone with a good theoretical background in image processing and machine learning can quickly prototype algorithms using CPP and extend them to create meaningful software packages. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 4, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>SBIA License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cbica_tutorials/">http://www.nitrc.org/projects/cbica_tutorials/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pennsylvania <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cbica:-whitestripe" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CBICA: WhiteStripe">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cbica:-whitestripe" rel="bookmark" name="cbica:-whitestripe">CBICA: WhiteStripe</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=statistical-operation" rel="tag">Statistical Operation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> WhiteStripe is a user-friendly implementation of an algorithm to normalize conventional magnetic resonance images by detecting a latent subdistribution of normal tissue and linearly scaling the histogram of the images. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 25, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>CBICA License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cbica_whitestri/">http://www.nitrc.org/projects/cbica_whitestri/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pennsylvania <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cbica:-wmls-(white-matter-lesion-segmentation)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CBICA: WMLS (White Matter Lesion Segmentation)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cbica:-wmls-(white-matter-lesion-segmentation)" rel="bookmark" name="cbica:-wmls-(white-matter-lesion-segmentation)">CBICA: WMLS (White Matter Lesion Segmentation)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Brain lesions appear in different brain diseases as well as in normal aging; they can be due to small vessel disease, infarcts, MS, or other reasons. MRI is used as surrogate imaging marker, as MRI signal changes reﬂect certain aspects of brain pathology.Computer algorithms have started to complement expert-readings of MRI as they may improve throughput and consistency, in addition to providing more accurate quantitative measures of lesion type and volume. Computerized segmentation methods can also offer more precise measurements of longitudinal change of a lesion with disease progression or treatment response. <br />
 The brain lesion segmentation tool uses image analysis and machine learning techniques (Support Vector Machines). Image intensities from multiple MR acquisition protocols, after coregistration, are used to form a voxel-wise attribute vector which is used to perform the segmentation.<br />
 Primary Contact: Christos Davatzikos<br />
 Contributors: Zhiqiang Lao ,Dinggang Shen ,Eva Zacharaki,Nick Bryan </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 29, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>SBIA License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/wmls/">http://www.nitrc.org/projects/wmls/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cbinifti:-matlab/octave-nifti-library" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="cbiNifti: Matlab/Octave Nifti library">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cbinifti:-matlab/octave-nifti-library" rel="bookmark" name="cbinifti:-matlab/octave-nifti-library">cbiNifti: Matlab/Octave Nifti library</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> cbiNifti: An I/O library for Matlab/Octave<br />
<br />
Matlab and Octave library for reading and writing Nifti-1 files. <br />
<br />
cbiNifti is intended to be a small, self-contained library that makes minimal assumptions about what Nifti files should look like and allow users easy access to the raw data. <br />
<br />
cbiNifti handles compressed file formats for reading and writing, using Unix pipes for compression and decompression.<br />
<br />
More information and code examples at:<br />
<br />
<a href="http://www.pc.rhul.ac.uk/staff/J.Larsson/software.html" target="_new">http://www.pc.rhul.ac.uk/staff/J.Larsson...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 15, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cbinifti/">http://www.nitrc.org/projects/cbinifti/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cbrain" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CBRAIN">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cbrain" rel="bookmark" name="cbrain">CBRAIN</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> CBRAIN is a flexible software platform with a small footprint and minimal requirements. CBRAIN was created to allow the integration the often extremely heterogeneous research HPC facilities across Canada and the World. Moreover, the platform's goal was not limited to deploying national and international distributed data and compute grids for neuroimaging sites, CBRAIN is an online collaborative web platform from which users transparently control their data, compute and results, including various forms of 2D and 3D data visualisation, regardless of where these resources and data are. <br />
<br />
CBRAIN is currently deployed on 6 Compute Canada HPC clusters, one German HPC cluster and 3 clusters local to McGill University Campus, totaling more than 80,000 potential CPU cores.<br />
<br />
Visit us at: <a href="http://cbrain.mcgill.ca" target="_new">http://cbrain.mcgill.ca</a><br />
CBRAIN code is now available on GuitHub: <a href="https://github.com/aces/cbrain" target="_new">https://github.com/aces/cbrain</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=486&screenshot_id=356"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=486&screenshot_id=356" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 16, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Web Application - Free to use</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://cbrain.mcgill.ca/">http://cbrain.mcgill.ca/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>McGill University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cbs-high-res-brain-processing-tools" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CBS High-Res Brain Processing Tools">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cbs-high-res-brain-processing-tools" rel="bookmark" name="cbs-high-res-brain-processing-tools">CBS High-Res Brain Processing Tools</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> With 7 Tesla MRI, neuroscientists can investigate the structure of the human brain with enhanced resolution, currently up to 350µm isotropic in vivo. Quantitative MR contrasts and increased level of detail within tissues reveal many fine anatomical structures, but require efficient computational methods that handle various contrasts and scale well with data complexity.<br />
<br />
The CBS High-Res Brain Processing Tools provide a fully automated processing pipeline for cortical analysis of structural MR images at a resolution of up to 400µm, including skull stripping, whole brain segmentation, cortical extraction, surface inflation and mapping, as well as dedicated tools for profile estimation across the cortical thickness.<br />
<br />
The tools are released as a set of plug-ins for the MIPAV software package and the JIST pipeline environment. They are therefore cross-platform and compatible with a wide variety of file formats. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=606&screenshot_id=505"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=606&screenshot_id=505" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 24, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cbs-tools/">http://www.nitrc.org/projects/cbs-tools/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Max Planck Institute for Human Cognitive and Brain Sciences <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="ccseg---corpus-callosum-segmentation" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CCSeg - Corpus Callosum Segmentation">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=ccseg---corpus-callosum-segmentation" rel="bookmark" name="ccseg---corpus-callosum-segmentation">CCSeg - Corpus Callosum Segmentation</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=shape-analysis" rel="tag">Shape Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Corpus Callosum Segmentation Tool<br />
<br />
CCSeg is an open-source C++-based application developed at UNC-Chapel Hill that allows automatic as well as user-interactive segmentation of the Corpus Callosum. Via a Qt-based graphical user interface, CCSeg also performs semi-automatic segmentation.<br />
<br />
Descriptions of methods:<br />
<br />
Media 1997: G. Szekely, A. Kelemen, C. Brechbühler, and G. Gerig, “Segmentation of 2-D and 3-D objects from MRI volume data using constrained elastic deformations of flexible Fourier contour and surface models.,” Med Image Anal, vol. 1, no. 1, pp. 19–34, Mar. 1996.<br />
<br />
recent SPIE article: <a href="http://www.nitrc.org/docman/view.php/466/1131/SPIE_2012_Vachet_CCSeg.pdf" target="_new">http://www.nitrc.org/docman/view.php/466...</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=466&screenshot_id=345"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=466&screenshot_id=345" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 23, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/ccseg/">http://www.nitrc.org/projects/ccseg/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina at Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cdf-hc-pointsetreg" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CDF-HC PointSetReg">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cdf-hc-pointsetreg" rel="bookmark" name="cdf-hc-pointsetreg">CDF-HC PointSetReg</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This is a Matlab demo for group wise point set registration using a novel CDF-based Havrda-Charvat Divergence, which is based on the paper: <br />
Ting Chen, Baba C. Vemuri, Anand Rangarajan and Stephan J. Eisenschenk, Group-wise Point-set registration using a novel CDF-based Havrda-Charvat Divergence. In IJCV : International Journal of Computer Vision, 86(1):111-124, January, 2010. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 8, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.cise.ufl.edu/~tichen/cdfHC.zip">http://www.cise.ufl.edu/~tichen/cdfHC.zip</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Florida <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="celeritas:-fiber-optic-response-system" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Celeritas: Fiber Optic Response System">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=celeritas:-fiber-optic-response-system" rel="bookmark" name="celeritas:-fiber-optic-response-system">Celeritas: Fiber Optic Response System</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=response-monitoring" rel="tag">Response Monitoring</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Celeritas Fiber Optic Response System can be customized to fit your research needs by selecting the combination of Button Response Units (BRUs) and Joysticks. The BRUs and Joysticks comfortably fit small to large hand sizes and are constructed entirely of non-metallic components - completely eliminating all metal inside the magnet room. The units communicate button presses through fiber optic cabling which connects to an Interface Console located in the control room through an available wave guide. The Interface Console provides real-time feedback of participant responses via LED indicators and includes a set of buttons to make responses for the participant as needed. The system seamlessly integrates with E-Prime® through a USB connection. The new Interface Console comes with an Optical Connector and a BNC Connector. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=277&screenshot_id=1085"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=277&screenshot_id=1085" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 21, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://pstnet.com/products/celeritas/">https://pstnet.com/products/celeritas/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="center-for-computational-biology-(ccb)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Center for Computational Biology (CCB)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=center-for-computational-biology-(ccb)" rel="bookmark" name="center-for-computational-biology-(ccb)">Center for Computational Biology (CCB)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Center for Computational Biology (<a href="http://ccb.loni.usc.edu" target="_new">http://ccb.loni.usc.edu</a>) provides 3 types of resources: (1) Stand-alone computational software tools (image and volume processing, analysis, visualization, graphical workflow environments). (2) Infrastructure Resources (Databases, computational Grid, services). (3) Web-services (web-accessible resources for processing, validation and exploration of multimodal/multichannel data including clinical data, imaging data, genetics data and phenotypic data).<br />
<br />
CCB is headquartered at the Laboratory of Neuro Imaging (<a href="http://CCB.LONI.usc.edu/" target="_new">http://CCB.LONI.usc.edu/</a>) at USC. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=298&screenshot_id=202"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=298&screenshot_id=202" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 4, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.CCB.usc.edu">http://www.CCB.usc.edu</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Southern California <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="center-for-functional-neuroimaging-tech" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Center for Functional Neuroimaging Tech">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=center-for-functional-neuroimaging-tech" rel="bookmark" name="center-for-functional-neuroimaging-tech">Center for Functional Neuroimaging Tech</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=knowledge-environment" rel="tag">Knowledge Environment</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Center for Functional Neuroimaging Technologies (CFNT) is a Regional Resource located at the Athinoula A. Martinos Center for Biomedical Imaging at the Massachusetts General Hospital. The primary mission of the Center is to expand understanding of the human brain in health and disease through the development and dissemination of innovative multimodal Magnetic Resonance (MR)-based neuroimaging techniques and technologies. The Resource is sponsored by the National Center for Research Resources of the NIH. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=165&screenshot_id=87"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=165&screenshot_id=87" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 18, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MGH CSRL License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nmr.mgh.harvard.edu/martinos/ncrr/index">http://www.nmr.mgh.harvard.edu/martinos/ncrr/index</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cerebellar-white-matter-atlas" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Cerebellar White Matter Atlas">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cerebellar-white-matter-atlas" rel="bookmark" name="cerebellar-white-matter-atlas">Cerebellar White Matter Atlas</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=diffusion-mr-fiber-tracking" rel="tag">Diffusion Mr Fiber Tracking</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Cerebellar White Matter Atlas is based on probabilistic tractography of high resolution, high quality diffusion MR data of 90 healthy subjects from the Human Connectome Project database. The atlas comes as a 3D probabilistic map, as well as binary parcellations at different probability thresholds. MNI - as well as SUIT (Spatially Unbiased Infratentorial Template) - compatible versions of the atlas are provided. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=967&screenshot_id=953"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=967&screenshot_id=953" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 21, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Cerebellar White Matter Atlas</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nccn.nl/nccn-en/research/cerebellar_mutism">http://www.nccn.nl/nccn-en/research/cerebellar_mutism</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Radboud University Medical Centre Nijmegen, The Netherlands <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cerebral-blood-flow-database-and-analysis-pipeline-(cbfdap)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Cerebral Blood Flow Database and Analysis Pipeline (CBFDAP)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cerebral-blood-flow-database-and-analysis-pipeline-(cbfdap)" rel="bookmark" name="cerebral-blood-flow-database-and-analysis-pipeline-(cbfdap)">Cerebral Blood Flow Database and Analysis Pipeline (CBFDAP)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Cerebral Blood Flow Database and Analysis Pipeline (CBFDAP) is a web enabled data and workflow management system extended from the HID codebase on NITRC specialized for Arterial Spin Labeling data management and analysis (including group analysis) in a centralized manner. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 6, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BIRN License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cbfdap/">http://www.nitrc.org/projects/cbfdap/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UCSD <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="challenge-competitions-collection" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Challenge Competitions Collection">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=challenge-competitions-collection" rel="bookmark" name="challenge-competitions-collection">Challenge Competitions Collection</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This resource provides a guide to the many platforms available to conduct and compete in grand challenges, all in the name of improving science!<br />
<br />
Challenges allow researchers to put their problems before other subject matter experts to be solved. The community then competes to craft the best algorithms and developers have an opportunity to evaluate novel algorithms by testing them against other competitors using real-world datasets. Everybody wins.<br />
<br />
You can see a curated list of challenges by navigating to the Challenge Competitions Collection MediaWiki. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1209&screenshot_id=1231"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1209&screenshot_id=1231" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 28, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/challenges/">http://www.nitrc.org/projects/challenges/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>NeuroImaging Tools and Resources Collaboratory <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="chiasm---combined-histology-mri-integrated-atlas-of-the-squirrel-monkey" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CHIASM - Combined Histology-MRI Integrated Atlas of the Squirrel Monkey">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=chiasm---combined-histology-mri-integrated-atlas-of-the-squirrel-monkey" rel="bookmark" name="chiasm---combined-histology-mri-integrated-atlas-of-the-squirrel-monkey">CHIASM - Combined Histology-MRI Integrated Atlas of the Squirrel Monkey</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> A web-based combined MRI-histology digital atlas of the squirrel monkey brain </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 30, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/smatlas/">http://www.nitrc.org/projects/smatlas/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Vanderbilt University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="chinese-pediatric-atlas-(chn-pd-atlas)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Chinese Pediatric Atlas (CHN-PD Atlas)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=chinese-pediatric-atlas-(chn-pd-atlas)" rel="bookmark" name="chinese-pediatric-atlas-(chn-pd-atlas)">Chinese Pediatric Atlas (CHN-PD Atlas)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-data" rel="tag">Atlas Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> We use an unbiased template construction algorithm to create a set of age-specific Chinese pediatric (CHN-PD) atlases based on high-quality T1- and T2-weighted MR images from 328 cognitively normal Chinese children aged 6 to 12 years. The CHN-PD brain atlases include asymmetric and symmetric templates, sex-specific templates and tissue probability templates, and contain multiple age-specific templates at one-year intervals. We expect this set of age-specific Chinese brain atlases is useful for the image registration and spatial statistics in Chinese pediatric studies. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1291&screenshot_id=1208"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1291&screenshot_id=1208" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 13, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License v3</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/chn-pd/">http://www.nitrc.org/projects/chn-pd/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="chronos:-a-multifunctional-response-and-stimulus-device" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Chronos: A Multifunctional Response and Stimulus Device">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=chronos:-a-multifunctional-response-and-stimulus-device" rel="bookmark" name="chronos:-a-multifunctional-response-and-stimulus-device">Chronos: A Multifunctional Response and Stimulus Device</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Chronos allows the accurate collection and verification of tactile, auditory, visual, and analog responses along with the precise source of audio and generic analog output timing. Chronos features millisecond accuracy and consistent sound output latencies across machines. Chronos includes 16 digital inputs and 16 digital outputs, eliminating the need for a parallel port. The Photo Sensor accessory (included) can be used on CRT, LCD, and projection displays to detect stimulus onset events, refreshes, and measure rise and fall times. All responses collected are synchronous to the E-Prime® time domain. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1171&screenshot_id=1079"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1171&screenshot_id=1079" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 22, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://pstnet.com/products/chronos/">https://pstnet.com/products/chronos/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Psychology Software Tools, Inc. <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cifti-connectivity-file-format" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CIFTI Connectivity File Format">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cifti-connectivity-file-format" rel="bookmark" name="cifti-connectivity-file-format">CIFTI Connectivity File Format</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=file-format" rel="tag">File Format</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> CIFTI (Connectivity Informatics Technology Initiative) standardizes file formats for the storage of connectivity data. These formats are developed by the Human Connectome Project and other interested parties.<br />
<br />
The CIFTI-2 specification is in pdf files attached to forum posts, see: <br />
<br />
<a href="http://www.nitrc.org/forum/forum.php?thread_id=4380&amp;forum_id=1955" target="_new">http://www.nitrc.org/forum/forum.php?thr...</a><br />
<br />
<a href="http://www.nitrc.org/forum/forum.php?thread_id=4381&amp;forum_id=1955" target="_new">http://www.nitrc.org/forum/forum.php?thr...</a><br />
<br />
The specification of CIFTI-1 is on the MediaWiki link to the left.<br />
<br />
Access the CIFTI discussion forum using the Forums entry in the menu on the left. Subscribe to the discussion forum and you will be informed about issues involving the CIFTI file formats via email. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 3, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cifti/">http://www.nitrc.org/projects/cifti/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>CIFTI</td></tr>
			<tr class="even"><td>Output Formats</td><td>CIFTI</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cigal" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CIGAL">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cigal" rel="bookmark" name="cigal">CIGAL</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> CIGAL is a program that provides accurate real-time stimulus control, behavioral and physiological recording, and synchronization with external devices. It can also provide continuous real-time feedback of task performance and physiological responses. Task programming typically involves a simple text file specifying basic parameter settings (e.g. screen color) and a list of stimulus events, which can include images, animated movies, sound files, text stimuli, video graphics, or commands that communicate with external hardware devices. Multiple video and auditory stimuli can be presented simultaneously. Multi-channel response recording and real-time feedback features require no user programming. Advanced users can add customized stimulus events using CIGAL’s real-time programming capabilities. Output files can be automatically created in a variety of output formats (e.g. FSL 3-column files, XML Events files, CSV trial tables). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 18, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>CIGAL License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://wiki.biac.duke.edu/jvs:cigal">https://wiki.biac.duke.edu/jvs:cigal</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Duke University Medical Center <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="civility-:-cloub-based-interactive-visualization-of-tractography-brain-connectome" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CIVILITY : Cloub based Interactive Visualization of Tractography Brain Connectome">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=civility-:-cloub-based-interactive-visualization-of-tractography-brain-connectome" rel="bookmark" name="civility-:-cloub-based-interactive-visualization-of-tractography-brain-connectome">CIVILITY : Cloub based Interactive Visualization of Tractography Brain Connectome</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Cloud Based Interactive Visualization of Tractography Brain Connectome (CIVILITY) is an interactive visualization tool of brain connectome in the web.<br />
CIVILITY is a web application and has mainly 2 components.<br />
<br />
-CIVILITY-visualization ; front end of the application. This is a circle plot of the brain connectivity using the method of visualization : Hierarchical Edge Bundling. The graphic visualization of the brain connectivity is generated using Data Driven Documents (D3.js).<br />
<br />
-CIVILITY-tractography ; analysis pipeline. The analysis of the brain connectome is computed with a probabilistic tractography method (FSL tools : bepostx and probtrackx2) using surfaces as seeds.<br />
<br />
CIVILITY performs the brain connectivity analysis in remote computing grids where the CIVILITY-tractography pipeline is deployed. CIVILITY uses clusterpost (<a href="https://github.com/NIRALUser/clusterpost" target="_new">https://github.com/NIRALUser/clusterpost</a>) to submit the jobs to the computing grid. The front end of CIVILITY submits tasks to clusterpost and retrieves results when they are finished. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1051&screenshot_id=1000"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1051&screenshot_id=1000" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 29, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/civility/">http://www.nitrc.org/projects/civility/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cleanline" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CleanLine">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cleanline" rel="bookmark" name="cleanline">CleanLine</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=spectral-analysis" rel="tag">Spectral Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Sinusoidal noise can be a prominent artifact in recorded electrophysiological data. This can stem from AC power line fluctuations (e.g. 50/60 Hz line noise + harmonics), power suppliers (e.g. in medical equipment), fluorescent lights, etc. Notch filtering is generally undesirable due to creation of band-holes, and significant distortion of frequencies around the notch frequency (as well as phase distortion at other frequencies and Gibbs rippling in the time-domain). CleanLine is an EEGLAB plugin which adaptively estimates and removes sinusoidal artifacts from ICA components or scalp channels using a frequency-domain (multi-taper) regression technique with a Thompson F-statistic for identifying significant sinusoidal artifacts. This approach has been advocated by Partha Mitra and Hemant Bokil (Observed Brain Dynamics, Chapter 7.3.4., 2007) and CleanLine utilizes modified routines from the Mitra Lab's Chronux Toolbox (www.chronux.org). 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=572&screenshot_id=477"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=572&screenshot_id=477" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 8, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cleanline/">http://www.nitrc.org/projects/cleanline/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UCSD <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cleave:-large-data-set-anova" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CLEAVE: Large Data Set ANOVA">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cleave:-large-data-set-anova" rel="bookmark" name="cleave:-large-data-set-anova">CLEAVE: Large Data Set ANOVA</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=statistical-operation" rel="tag">Statistical Operation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> CLEAVE is a UNIX-style command-line program which quickly computes multifactorial ANOVAs for very large data sets with minimal memory use (without loading all of the data into memory). It has been used for fMRI analysis, e.g.<br />
<br />
CLEAVE adds the following to the standard ANOVA analyses:<br />
<br />
0) Unlimited numbers of factors can be analyzed.<br />
1) Factor Correlation and Unequal Variance Corrections<br />
2) Treatment Magnitudes: omega^2, partial eta^2, and R^2<br />
3) A convenient Ranking of Factors based upon treatment magnitudes and significance levels.<br />
4) Post-Hoc Significance Tests<br />
5) Post-Hoc Power Table to gauge how many subjects will be needed to achieve significance.<br />
6) Allows the use of Random Factors.<br />
7) A Configuration File to make the program more tunable<br />
8) A Histogram and Cell Line Diagrams: which help the user to detect outliers.<br />
9) Associated MATLAB functions: port CLEAVE-style data sets in or out of MATLAB.<br />
<br />
Software is at <a href="http://www.ebire.org/hcnlab/software/cleave.html" target="_new">http://www.ebire.org/hcnlab/software/cle...</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=405&screenshot_id=278"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=405&screenshot_id=278" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 24, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.ebire.org/hcnlab/software/cleave.html">http://www.ebire.org/hcnlab/software/cleave.html</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS<br>Microsoft<br>POSIX/UNIX-like</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="clinical-toolbox-for-spm" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Clinical Toolbox for SPM">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=clinical-toolbox-for-spm" rel="bookmark" name="clinical-toolbox-for-spm">Clinical Toolbox for SPM</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The clinical toolbox is useful for normalizing data from individuals with brain injury and/or modalities popular in the clinical environment (CT), as described by Rorden et al. (2012, PMID: 22440645). The latest version supports either enantiomorphic (PMID: 18023365) or lesion-masked (PMID: 11467921) normalization. It can be either scripted or used with SPM's simple graphical interface. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=881&screenshot_id=867"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=881&screenshot_id=867" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 24, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/clinicaltbx/">http://www.nitrc.org/projects/clinicaltbx/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of South Carolina <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cloud-reproducibility-challenge" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Cloud Reproducibility Challenge">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cloud-reproducibility-challenge" rel="bookmark" name="cloud-reproducibility-challenge">Cloud Reproducibility Challenge</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The use of standardizable computing environments offers a promise for enhancing the reproducibility of neuroimaging computational workflows. In this hackathon project we seek to develop a reproducibility assessment framework for testing and validating similar workflows run under different conditions. For a given workflow, which can be run on a local computer system, as a virtual machine, or in a cloud computing environment, we want to assess the potential variances introduced in workflow results as a function of the operating environment. This framework can then be extended to assessment of variations due to additional factors, such as software versioning, workflow parameters, and input data. The end result of this project would be a stand alone application that details the reproducibility of workflow/result pair in the context of the various execution variables.<br />
<br />
This project hopes to leverage the NITRC-CE, nipype, testkraut, and existing shared data/shared results. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=946&screenshot_id=931"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=946&screenshot_id=931" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 12, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cloudrepro/">http://www.nitrc.org/projects/cloudrepro/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>OHBM 2015 Hackathon Project <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cluster-extent-correction" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Cluster Extent Correction">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cluster-extent-correction" rel="bookmark" name="cluster-extent-correction">Cluster Extent Correction</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This script will take any .img file and correct it based on a cluster extent, cluster definition and voxelwise threshold. The threshold entered will be applied to positive and negative values separately, and separate pos and neg corrected images will be output. This script requires a license for the matlab image processing toolbox. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 7, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cluster_correct/">http://www.nitrc.org/projects/cluster_correct/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cluster-reporter" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Cluster reporter">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cluster-reporter" rel="bookmark" name="cluster-reporter">Cluster reporter</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This matlab script and associated files will take resultant statistical images and essentially output everything you could ever want to know. It can work off of images that were previously corrected for multiple comparisons, but it can actually do the correction itself. This is because the cluster_correct script is incorporated within. It will iterate through atlases (borrowed from other software) to tell you the location of significant results. It outputs an extremely detailed report as well as a summary table for quick investigation. In addition, it will output statistics for each surviving cluster, and the image as a whole.Feedback would be much appreciated. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 8, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cluster_report/">http://www.nitrc.org/projects/cluster_report/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cmfreg" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CMFreg">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cmfreg" rel="bookmark" name="cmfreg">CMFreg</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> We have developed a sequence of fully automated voxel-wise rigid registration that utilizes stable structures of reference for assessment of craniofacial changes overtime.The major strengths of this method are that registration does not depend on the precision of the 3D surface models and that a “stable structure of reference” can be used without the simple “best fit” of all surfaces. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 29, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>3D Slicer License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cmfreg/">http://www.nitrc.org/projects/cmfreg/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina at Chapel Hill - University of Michigan <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="c-mind-database" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="C-MIND Database">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=c-mind-database" rel="bookmark" name="c-mind-database">C-MIND Database</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The CMIND database contains brain imaging data collected on 3T MRI scanners from over 200 normally developing healthy children from birth to 18 years. There is also longitudinal data collected from a sub-group of children ages 0-3 and 7-9 years who returned scans for three consecutive years. The imaging data stored in the C-MIND database are DTI, HARDI, 3DT1W, 3DT2W, concurrent ASL-BOLD scans during two language tasks (Stories &amp; Sentence-Picture Matching), Resting State fMRI and Baseline ASL scans. The database also hosts extensive age-appropriate behavioral measures for all the subjects. The data and their behavioral measures can be downloaded from a public database (<a href="https://cmind.research.cchmc.org/" target="_new">https://cmind.research.cchmc.org/</a>). Registration to this website is available and the information can be found in the opening page (<a href="https://cmind.research.cchmc.org/auth/consent" target="_new">https://cmind.research.cchmc.org/auth/co...</a>). A tool using python was developed to process the data and templates of various age ranges are available at bitbucket ((<a href="https://bitbucket.org/grlee77/cmind-py" target="_new">https://bitbucket.org/grlee77/cmind-py</a>). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 11, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cmind_2014/">http://www.nitrc.org/projects/cmind_2014/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Cincinnati Children's Hospital Medical Center <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cmind_py" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CMIND_PY">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cmind_py" rel="bookmark" name="cmind_py">CMIND_PY</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Cmind_Py was written entirely in Python due to its relative ease of use and the availability of a wide number of high quality scientific (<a href="http://scipy.org" target="_new">http://scipy.org</a>) and neuroimaging software modules (<a href="http://nipy.org" target="_new">http://nipy.org</a>) . The software relies on calls to a number of widely tested algorithms from the FMRIB software library (FSL) and the advanced normalization tools (ANTS). Linux and Max OSX are supported. Both LONI and Nipype interfaces are supported. A unique feature provided by the processing pipelines is analysis of simultaneously acquired ASL/BOLD fMRI data. The tool and its code are available at bitbucket (<a href="https://bitbucket.org/grlee77/cmind-py" target="_new">https://bitbucket.org/grlee77/cmind-py</a>). Pediatric templates with 2mm resolution were created for the following age ranges: 0-6 months, 0.5 – 2 years, 2-4 years and 4-18 years using ANTS. These can be found at (<a href="https://bitbucket.org/grlee77/cmind-py/downloads" target="_new">https://bitbucket.org/grlee77/cmind-py/d...</a>). The data used and their behavioral measures can be downloaded from a public database (<a href="https://cmind.research.cchmc.org/" target="_new">https://cmind.research.cchmc.org/</a>). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 11, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cmind_py_2014/">http://www.nitrc.org/projects/cmind_py_2014/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Cincinnati Children's Hospital Medical Center <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cmrep" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="cmrep">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cmrep" rel="bookmark" name="cmrep">cmrep</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The continuous medial representation (cm-rep) is a set of deformable modeling algorithms for shape analysis and structure-specific normalization. Applications of cm-reps include structure-specific fMRI analysis, DTI analysis, and structural brain mor </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 6, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cmrep/">http://www.nitrc.org/projects/cmrep/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cnn-for-3d-brain-mri-classification" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CNN for 3D brain MRI classification">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cnn-for-3d-brain-mri-classification" rel="bookmark" name="cnn-for-3d-brain-mri-classification">CNN for 3D brain MRI classification</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> CNN for 3D brain MRI classification </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 15, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cnn_brain_image/">http://www.nitrc.org/projects/cnn_brain_image/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pisa <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cocomac-brain-connectivity-database" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CoCoMac Brain Connectivity Database">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cocomac-brain-connectivity-database" rel="bookmark" name="cocomac-brain-connectivity-database">CoCoMac Brain Connectivity Database</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database-application" rel="tag">Database Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=ontology" rel="tag">Ontology</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tractography" rel="tag">Tractography</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> CoCoMac.org gives online access (html or xml) to structural connectivity ('wiring') data on the Macaque brain. The database has become by far the largest of its kind, with data extracted from more than four hundred published tracing studies. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=148&screenshot_id=923"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=148&screenshot_id=923" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 11, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://cocomac.org">http://cocomac.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Forschungszentrum Jülich, Radboud Univ. Nijmegen, G-node <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cocomac-paxinos3d-viewer" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CoCoMac-Paxinos3D viewer">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cocomac-paxinos3d-viewer" rel="bookmark" name="cocomac-paxinos3d-viewer">CoCoMac-Paxinos3D viewer</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> CoCoMac-Paxinos3D viewer represents an interactive interface of macaque stereotaxic atlas with a connectivity database, allowing integrated data analysis and mapping between 3D structures and database vocabularies. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 15, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cp3d/">http://www.nitrc.org/projects/cp3d/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cognision-(tm)-system" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="COGNISION (TM) System">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cognision-(tm)-system" rel="bookmark" name="cognision-(tm)-system">COGNISION (TM) System</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=event-related-potentials" rel="tag">Event-Related Potentials</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-software-resource" rel="tag">Other Software Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> COGNISION TM with auditory or visual event-related potential (ERP) technology, provides a direct physiologic measure of patients' cognitive processing (i.e., a &quot;cognitive biomarker&quot;). COGNISION is a portable, highly integrated, internet-enabled, hardware/software platform and patient management system, which includes an online Patient Manager module which complies with the HIPAA Final Security Rule, an ERP Viewer module to view and analyze raw and average ERP waves, and a Protocol Editor module to simplify the choice and administration of selected ERP protocols. It is also easy to train and administer with non-specialized personnel, and is designed to be used in an out-patient setting. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=591&screenshot_id=493"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=591&screenshot_id=493" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 2, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Commercial License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://neuronetrix.com/">http://neuronetrix.com/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Neuronetrix <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cognitive-paradigm-ontology" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Cognitive Paradigm Ontology">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cognitive-paradigm-ontology" rel="bookmark" name="cognitive-paradigm-ontology">Cognitive Paradigm Ontology</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Cognitive Paradigm Ontology (CogPO) is a domain ontology of cognitive paradigms for application and use in the functional neuroimaging community. CogPO has been developed through the integration of the Functional Imaging Biomedical Informatics Research Network (FBIRN) Human Imaging Database (HID) and the BrainMap Database. The design of CogPO concentrates on what can be observed directly: categorization of each paradigm in terms of (1) the stimulus presented to the subjects, (2) the requested instructions, and (3) the returned response. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=740&screenshot_id=747"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=740&screenshot_id=747" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 6, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://cogpo.org">http://cogpo.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Florida International University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="coins" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="COINS">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=coins" rel="bookmark" name="coins">COINS</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database-application" rel="tag">Database Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> As neuroimaging research continues to grow, dynamic neuroinformatics systems are necessary to store, retrieve, mine and share the massive amounts of data. The Collaborative Informatics and Neuroimaging Suite (COINS) has been created to facilitate communication and cultivate a data community. This web-based neuroimaging and neuropsychology software suite offers versatile, automatable data upload/import/entry options, rapid and secure sharing of data among PIs, querying and export all data, real-time reporting, and HIPAA and IRB compliant study-management tools suitable to large institutions as well as smaller scale neuroscience and neuropsychology researchers.<br />
 <br />
COINS manages over 180 studies and their data at the Mind Research Network, the Nathan Kline Institute, University of Colorado – Boulder, the Olin Neuropsychiatry Research Center @ Hartford Hospital, and other sites. Please click on COINS website on the left menu to visit our promotional site with a Live Demo! 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=595&screenshot_id=985"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=595&screenshot_id=985" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 9, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/coins/">http://www.nitrc.org/projects/coins/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>The Mind Research Network <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="colin-3t/7t-high-resolution-atlas" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Colin 3T/7T High-resolution Atlas">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=colin-3t/7t-high-resolution-atlas" rel="bookmark" name="colin-3t/7t-high-resolution-atlas">Colin 3T/7T High-resolution Atlas</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-data" rel="tag">Atlas Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> High-field extension of the Colin27 single-subject atlas with additional high-resolution, quantitative, averaged scans at both 3T and 7T. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=825&screenshot_id=774"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=825&screenshot_id=774" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 5, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/colin3t7t/">http://www.nitrc.org/projects/colin3t7t/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Western University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="combining-a-patch-based-approach-with-a-non-rigid-registration-based-label-fusion-method-for-the-hippocampal-segmentation" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Combining a patch-based approach with a non-rigid registration-based label fusion method for the hippocampal segmentation">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=combining-a-patch-based-approach-with-a-non-rigid-registration-based-label-fusion-method-for-the-hippocampal-segmentation" rel="bookmark" name="combining-a-patch-based-approach-with-a-non-rigid-registration-based-label-fusion-method-for-the-hippocampal-segmentation">Combining a patch-based approach with a non-rigid registration-based label fusion method for the hippocampal segmentation</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> We propose a patch-based labeling method, which cooperates<br />
with a label fusion using non-rigid registrations. We present<br />
a novel patch-based label fusion method, whose selected patches and their weights are calculated from a combination of similarity measures between patches using intensity-based distances and labeling-based distances, where a previous labeling of the target image is inferred by a label fusion method using non-rigid registrations. We compare several label fusion methods on publicly available MR images of human brains for segmenting the hippocampus. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 1, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/lf_patches/">http://www.nitrc.org/projects/lf_patches/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Technical University of Madrid (Spain) <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="computational-anatomy-toolbox---cat" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Computational Anatomy Toolbox - CAT">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=computational-anatomy-toolbox---cat" rel="bookmark" name="computational-anatomy-toolbox---cat">Computational Anatomy Toolbox - CAT</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=expectation-minimization" rel="tag">Expectation Minimization</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=mean-curvature" rel="tag">Mean Curvature</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quality-metrics" rel="tag">Quality Metrics</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=spherical-harmonics" rel="tag">Spherical Harmonics</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-rendering" rel="tag">Surface Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=thickness" rel="tag">Thickness</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volume-measurement" rel="tag">Volume Measurement</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volumetric-analysis" rel="tag">Volumetric Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> CAT is a an extension to SPM12 (Wellcome Department of Cognitive Neurology) to provide computational anatomy. This covers diverse morphometric methods such as voxel-based morphometry (VBM), surface-based morphometry (SBM), deformation-based morphometry (DBM), and region- or label-based morphometry (RBM). 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1027&screenshot_id=974"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1027&screenshot_id=974" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 21, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.neuro.uni-jena.de/cat/">http://www.neuro.uni-jena.de/cat/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Jena University Hospital <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS<br>Microsoft<br>POSIX/UNIX-like</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>GIfTI, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>GIfTI, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="computational-morphometry-toolkit-(cmtk)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Computational Morphometry Toolkit (CMTK)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=computational-morphometry-toolkit-(cmtk)" rel="bookmark" name="computational-morphometry-toolkit-(cmtk)">Computational Morphometry Toolkit (CMTK)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=anatomic" rel="tag">Anatomic</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=artifact-removal" rel="tag">Artifact Removal</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=format-conversion" rel="tag">Format Conversion</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-reconstruction" rel="tag">Image Reconstruction</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-to-image" rel="tag">Image-To-Image</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=linear" rel="tag">Linear</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=principal-component-analysis" rel="tag">Principal Component Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=resampling" rel="tag">Resampling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=two-dimensional-display" rel="tag">Two Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=warping" rel="tag">Warping</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> A software toolkit for computational morphometry of biomedical images, CMTK comprises a set of command line tools and a back-end general-purpose library for processing and I/O.<br />
<br />
The command line tools primarily provide the following functionality: registration (affine and nonrigid; single and multi-channel; pairwise and groupwise), image correction (MR bias field estimation; interleaved image artifact correction; EPI unwarping), processing (filters; combination of segmentations via voting and STAPLE; shape-based averaging), statistics (t-tests; general linear model).<br />
<br />
CMTK is implemented in C++ with parallel processing using POSIX Threads (SMP), OpenMP (SMP), Grand Central Dispatch (SMP), and CUDA (GPU). <br />
<br />
Supported file formats include Analyze (r/w), NIFTI (r/w), Nrrd (r/w), DICOM (read), BioRad (read). Data exchange with other toolkits, such as ITK, FSL, AFNI, SPM, etc. is thus easily accomplished. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=212&screenshot_id=212"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=212&screenshot_id=212" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 27, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cmtk/">http://www.nitrc.org/projects/cmtk/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Cygwin<br>Linux<br>MacOS<br>Other OS<br>SunOS/Solaris<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, NIfTI-1, Nrrd, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, NIfTI-1, Nrrd, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="conferences,-workshops-and-meetings" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Conferences, Workshops and Meetings">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=conferences,-workshops-and-meetings" rel="bookmark" name="conferences,-workshops-and-meetings">Conferences, Workshops and Meetings</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=knowledge-environment" rel="tag">Knowledge Environment</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This project assists the community in the support of information about upcoming Conferences, Workshops and Meetings. Such support may be documents, news, files, etc. <br />
<br />
To see a listing of upcoming Events, please use the NITRC Community Events Page at <a href="http://www.nitrc.org/incf/event_list.php." target="_new">http://www.nitrc.org/incf/event_list.php...</a><br />
<br />
All users are encouraged to check this site for upcoming meetings, and promote future meetings here. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=146&screenshot_id=223"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=146&screenshot_id=223" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 2, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/meetings/">http://www.nitrc.org/projects/meetings/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="connectir" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="connectir">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=connectir" rel="bookmark" name="connectir">connectir</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=software" rel="tag">Software</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Connectir is an R-based package to conduct brain connectivity analyses with a focus on a novel approach to conducting Connectome-Wide Association Studies (CWAS) using functional connectivity.<br />
<br />
Please see our web page for more details: <a href="http://czarrar.github.io/connectir" target="_new">http://czarrar.github.io/connectir</a> or click the Home Page link on the right below. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 3, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://czarrar.github.io/connectir">http://czarrar.github.io/connectir</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Yale University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="connectome-file-format-(cff)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Connectome File Format (CFF)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=connectome-file-format-(cff)" rel="bookmark" name="connectome-file-format-(cff)">Connectome File Format (CFF)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=file-format" rel="tag">File Format</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Connectome File Format (CFF) is a container format for multi-modal neuroimaging data. It comprises connectome objects of type: CMetadata, CNetwork, CVolume, CSurface, CTrack, CScript, CData, CTimeseries, CImagestack. The Python library cfflib provides read/write functionality. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 4, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.connectomics.org/cfflib/">http://www.connectomics.org/cfflib/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>GIfTI, NIfTI-1, Other Format, PDF - Portable Document Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>GIfTI, NIfTI-1, Other Format, PDF - Portable Document Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="connectome-viewer" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Connectome Viewer">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=connectome-viewer" rel="bookmark" name="connectome-viewer">Connectome Viewer</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=animation" rel="tag">Animation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=format-conversion" rel="tag">Format Conversion</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-rendering" rel="tag">Surface Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=three-dimensional-display" rel="tag">Three Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tractography" rel="tag">Tractography</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=two-dimensional-display" rel="tag">Two Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volume-rendering" rel="tag">Volume Rendering</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Connectome Viewer is free, open source, cross-platform Python-based software application for visualization and analysis in connectome research.<br />
<br />
Features of the software include: <br />
* Connectome File Format including metadata, networks, surfaces, volumes, track files<br />
* Complex network analysis toolboxes<br />
* Modular plugin architecture for extensibility<br />
* Mayavi2 for 3D Scientific Visualization and Plotting<br />
* Interactive data manipulation and scripting capabilities<br />
* Neuroimaging and Diffusion in Python libraries 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=355&screenshot_id=225"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=355&screenshot_id=225" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 18, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://connectomics.org/viewer/">http://connectomics.org/viewer/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>GIfTI, NIfTI-1, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>GIfTI, NIfTI-1, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="connectome-workbench" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Connectome Workbench">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=connectome-workbench" rel="bookmark" name="connectome-workbench">Connectome Workbench</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=clipping" rel="tag">Clipping</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=gaussian-curvature" rel="tag">Gaussian Curvature</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=mean-curvature" rel="tag">Mean Curvature</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=region-of-interest" rel="tag">Region Of Interest</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=resampling" rel="tag">Resampling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-rendering" rel="tag">Surface Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=three-dimensional-display" rel="tag">Three Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=two-dimensional-display" rel="tag">Two Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volume-rendering" rel="tag">Volume Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=warping" rel="tag">Warping</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Connectome Workbench is a brain visualization, analysis and discovery tool for fMRI and dMRI brain imaging data, including functional and structural connectivity data generated by the Human Connectome Project. The software package includes wb_view, a GUI-based visualization platform and wb_command, a command-line program for performing a variety of analytical tasks for volume, surface, and CIFTI grayordinates data.<br />
<br />
Downloads and other related resources are on this site: <a href="http://www.humanconnectome.org/software/get-connectome-workbench.html" target="_new">http://www.humanconnectome.org/software/...</a><br />
<br />
Source code is available here: <a href="https://github.com/Washington-University/workbench" target="_new">https://github.com/Washington-University...</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=765&screenshot_id=689"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=765&screenshot_id=689" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 4, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.humanconnectome.org/software/get-connectome-workbench.html">http://www.humanconnectome.org/software/get-connectome-workbench.html</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>HCP WU-Minn Consortium <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>CIFTI, GIfTI, NIfTI-1, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>CIFTI, GIfTI, NIfTI-1, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="connectopy-toolkits" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Connectopy Toolkits">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=connectopy-toolkits" rel="bookmark" name="connectopy-toolkits">Connectopy Toolkits</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tractography" rel="tag">Tractography</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> While the topographic organization of brain pathways, i.e., connectopy, as a general principle has been known for a long time, it has yet to be quantitatively studied with in vivo neuroimaging. Using connectome imaging data, we have developed several novel algorithms to quantitively model the connectopy in human brains. These algorithms can be used to reconstruct topography-preserving pathways in visual, auditory, somatosensory, cortical-striatal, and fiber pathways in other brain networks. They can also improve the robustness of functional independent component analysis with regularization from topography-preserving fiber tracts. In this ConnectopyToolkits, we will publicly release the software tools for connectopy modeling. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 24, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MOCA License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/connectopytool/">http://www.nitrc.org/projects/connectopytool/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Southern California <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="conn-:-functional-connectivity-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CONN : functional connectivity toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=conn-:-functional-connectivity-toolbox" rel="bookmark" name="conn-:-functional-connectivity-toolbox">CONN : functional connectivity toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=connectivity-analysis" rel="tag">Connectivity Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=correlation" rel="tag">Correlation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=modeling" rel="tag">Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=multivariate-analysis" rel="tag">Multivariate Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=principal-component-analysis" rel="tag">Principal Component Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=regression" rel="tag">Regression</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> CONN is a Matlab-based cross-platform software for the computation, display, and analysis of functional connectivity in fMRI (fcMRI).<br />
<br />
CONN includes a rich set of connectivity analyses (seed-based correlations, ROI-to-ROI graph analyses, group ICA, masked ICA, generalized PPI, ALFF, ICC, GCOR, LCOR, etc.) in a simple-to-use and powerful software package<br />
<br />
CONN is available for resting state data (rsfMRI) as well as task-related designs. It covers the entire pipeline from raw fMRI data to hypothesis testing.<br />
<br />
Highlights: <br />
 Comprehensive quality assurance methods/measures/displays (scrubbing/ART, CompCor/ICA/GSR denoising, GCOR/FD QC covariates)<br />
 Connectome-wide analyses (ICA, MVPA)<br />
 Dynamic connectivity analyses (dyn-ICA)<br />
 FWE-control of connectivity matrices (NBS)<br />
 Non-parametric cluster-level statistics (permutation tests)<br />
 ANOVA, regression, longitudinal, and mixed designs<br />
<br />
Parallelization options (SGE/Grid Engine, PBS/Torque, LSF, Slurm)<br />
Mac/Windows/Linux/HPC<br />
SPM8/SPM12<br />
<br />
<a href="http://www.conn-toolbox.org" target="_new">http://www.conn-toolbox.org</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=279&screenshot_id=1022"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=279&screenshot_id=1022" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 1, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MIT/X Consortium License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.conn-toolbox.org">http://www.conn-toolbox.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>MIT <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, MGH/MGZ, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, MGH/MGZ, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="convert-mni-coordinates-to-or-from-xyz" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Convert MNI coordinates to or from XYZ">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=convert-mni-coordinates-to-or-from-xyz" rel="bookmark" name="convert-mni-coordinates-to-or-from-xyz">Convert MNI coordinates to or from XYZ</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Input either normalized &quot;MNI&quot; coordinates from a 3D image, or input &quot;real world&quot; XYZ matrix coordinates, and this code will convert coordinates of one type to the other. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 7, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mni2orfromxyz/">http://www.nitrc.org/projects/mni2orfromxyz/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="conwhat---a-library-for-connectome-based-white-matter-atlas-analysis" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ConWhAt - A library for Connectome-Based White Matter Atlas analysis">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=conwhat---a-library-for-connectome-based-white-matter-atlas-analysis" rel="bookmark" name="conwhat---a-library-for-connectome-based-white-matter-atlas-analysis">ConWhAt - A library for Connectome-Based White Matter Atlas analysis</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ConWhAt - a library for connectome-based white matter atlas analysis.<br />
<br />
Conventional approaches to atlasing white matter structures follow a tract-based ontology: they assign locations in stereotaxic space to a relatively small number of gross white matter tracts from the classical neuroanatomy literature.<br />
<br />
Problem is, they aren't particularly well-suited to network-based descriptions of brain organization.<br />
<br />
Connectome-based white matter atlases take a different approach: they follow a connectivity-based ontology. The idea here is rather than following the classical anatomical tract nomenclature, to label white matter voxels according to the grey matter regions that their constituent fibers interconnect.<br />
<br />
The benefit of this approach is that a scientist/clinician/citizen can take a set of (standard space) coordinates, or a nifti-format ROI mask such as a binary lesion map, and straightforwardly query which grey matter region pairs (i.e. connectome-edges) have fibers passing through those locations. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 10, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GPL-Style Open Source</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/conwhat/">http://www.nitrc.org/projects/conwhat/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Rotman Research Institute <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="corpus-callosum-and-brain-segmentations-from-abide-database" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Corpus callosum and brain segmentations from ABIDE database">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=corpus-callosum-and-brain-segmentations-from-abide-database" rel="bookmark" name="corpus-callosum-and-brain-segmentations-from-abide-database">Corpus callosum and brain segmentations from ABIDE database</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> - NIFTI format corpus callosum segmentation data<br />
- NIFTI format brain segmentations<br />
<br />
Both derived from ABIDE database </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 8, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://sites.google.com/site/hpardoe/cc_abide">http://sites.google.com/site/hpardoe/cc_abide</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>New York University School of Medicine <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="corpus-callosum-thickness-profile-analysis-pipeline" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Corpus Callosum Thickness Profile Analysis Pipeline">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=corpus-callosum-thickness-profile-analysis-pipeline" rel="bookmark" name="corpus-callosum-thickness-profile-analysis-pipeline">Corpus Callosum Thickness Profile Analysis Pipeline</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-software-resource" rel="tag">Other Software Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> An end-to-end pipeline for corpus callosum processing that provides automated midsagittal alignment, CC segmentation with a quality control tool, and thickness profile generation. Groupwise analysis is facilitated by permutation testing with FWER and FDR multiple comparison correction. Results display is facilitated by a display script that shows p-values on a 3D pipe representation of a CC. This pipeline is implemented in Python. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 21, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/ccsegthickness/">http://www.nitrc.org/projects/ccsegthickness/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Murdoch Childrens Research Institute <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cortex" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="cortex">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cortex" rel="bookmark" name="cortex">cortex</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=thickness" rel="tag">Thickness</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This software package contains functions that will help researchers plan how many subjects per group need to be included in an MRI-based cortical thickness study to ensure a thickness difference is detected. The package requires cortical thickness mapping and co-registration to be carried out using Freesurfer. The power analyses are implemented in the R software package. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 12, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://brain.org.au/software/cortex/power">http://brain.org.au/software/cortex/power</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Florey Neuroscience Institutes <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>MGH/MGZ</td></tr>
			<tr class="even"><td>Output Formats</td><td>MGH/MGZ</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cost" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="COST">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cost" rel="bookmark" name="cost">COST</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Connectivity Orientation Spherical distribution Tool (COST) - This tool implements the method from the paper &quot;Efficient, Graph-based White Matter Connectivity from Orientation Distribution Functions via Multi-directional Graph Propagation,&quot; Proc. of SPIE 2011; 7962(79620S):1-8. However, with it we also seek to modify the method into a minimum cost-based approach and eventually test it on applications. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 12, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cost_unc/">http://www.nitrc.org/projects/cost_unc/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UNC-Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="c-pac" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="C-PAC">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=c-pac" rel="bookmark" name="c-pac">C-PAC</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=connectivity-analysis" rel="tag">Connectivity Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Configurable Pipeline for the Analysis of Connectomes (C-PAC) is a configurable, open-source, Nipype-based, automated processing pipeline for resting state functional MRI (R-fMRI) data, for use by both novice and expert users. C-PAC was designed to bring the power, flexibility and elegance of the Nipype platform to users in a plug and play fashion—without requiring the ability to program. Using an easy to read, text-editable configuration file, C-PAC can rapidly orchestrate automated R-fMRI processing procedures, including:<br />
- quality assurance measurements<br />
- image preprocessing based upon user specified preferences<br />
- generation of functional connectivity maps (e.g., correlation analyses)<br />
- customizable extraction of time-series data<br />
- generation of local R-fMRI metrics (e.g., regional homogeneity, voxel-matched homotopic connectivity, fALFF/ALFF)<br />
<br />
C-PAC makes it possible to use a single configuration file to launch a factorial number of pipelines differing with respect to specific processing steps. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 14, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MIT/X Consortium License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://fcp-indi.github.io">http://fcp-indi.github.io</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Child Mind Institute, Nathan Kline Institute, NYU Langone Medical Center <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS<br>POSIX/UNIX-like</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cppi-toolbox-for-fmri" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="cPPI Toolbox for fMRI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cppi-toolbox-for-fmri" rel="bookmark" name="cppi-toolbox-for-fmri">cPPI Toolbox for fMRI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=connectivity-analysis" rel="tag">Connectivity Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This is a Matlab toolbox that allows computation of task-related functional connectivity between multiple pairs of regions. <br />
<br />
Task-related functional connectivity is computed using the correlational psychophysiological interaction (cPPI) methodology described in Fornito et al. (2012) PNAS, 109: 12788-12793.<br />
<br />
The toolbox assumes that first-level design matrices have been specified and estimated using SPM5 or later.<br />
<br />
It takes as input these design matrices as well as user-extracted regional time courses and returns a matrix of pair-wise, task-related functional connectivity for each participant. The method is scalable to large networks comprising hundreds of regions and is well-suited to graph theoretic analyses and functional connectomics.<br />
<br />
One modifiable script, cPPI_master.m, can be used to run the analysis for an entire sample of participants. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 25, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cppi_toolbox/">http://www.nitrc.org/projects/cppi_toolbox/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Monash Clinical and Imaging Neuroscience <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="create-dwi-atlas" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Create DWI Atlas">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=create-dwi-atlas" rel="bookmark" name="create-dwi-atlas">Create DWI Atlas</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This package is a set of three slicer modules which together are able to compute a DWI Atlas given a set of DWI's. The modules included are:<br />
<br />
1) A Groupwise Registration module -&gt; compute's a deformation field for each DWI, using, for example, FA maps as input<br />
2) Warp DWI module -&gt; used to warp each DWI using the deformation from (1)<br />
3) DWI Averager -&gt; used to average the set of warped DWI's producing the final DWI Atlas </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 25, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>3D Slicer License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/createdwiatlas/">http://www.nitrc.org/projects/createdwiatlas/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Psychiatry Neuroimaging Laboratory <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cuda-accelerated-spherical-model-m/eeg" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CUDA accelerated spherical model M/EEG">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cuda-accelerated-spherical-model-m/eeg" rel="bookmark" name="cuda-accelerated-spherical-model-m/eeg">CUDA accelerated spherical model M/EEG</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=eeg-modeling" rel="tag">Eeg Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=meg-modeling" rel="tag">Meg Modeling</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> CUDA-SPHERE-FWD-MEEG is a CUDA C based toolkit which provides a GPU based implementation of the spherical model forward solution for the 306 channel Elekta Neuromag MEG system and the EEG. The 1-Sphere forward solution for the MEG and the 4-Sphere forward solution for the EEG is implemented in CUDA C and an accelerated solution is obtained using the NVIDIA GPU when the solution is calculated for a large number of dipoles (on the order of 15000 and above) and sensor location. Speedup by a factor of 22 and 32 is obtained for the EEG and MEG solution respectively when compared to the fastest CPU implementation available in the public domain. The complete source code and pre-compiled binaries are also made available via an open source license (GPL Version 3). A CUDA enabled NVIDIA graphics card is required to use the software. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 10, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cuda_sphere_fwd/">http://www.nitrc.org/projects/cuda_sphere_fwd/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="curry-7" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Curry 7">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=curry-7" rel="bookmark" name="curry-7">Curry 7</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=connectivity-analysis" rel="tag">Connectivity Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database-application" rel="tag">Database Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=eeg-modeling" rel="tag">Eeg Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=finite-element-analysis" rel="tag">Finite Element Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=format-conversion" rel="tag">Format Conversion</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-reconstruction" rel="tag">Image Reconstruction</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=meg-modeling" rel="tag">Meg Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=mesh-generation" rel="tag">Mesh Generation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=model-generation" rel="tag">Model Generation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=position" rel="tag">Position</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=simulation" rel="tag">Simulation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=spatial-transformation" rel="tag">Spatial Transformation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volumetric-analysis" rel="tag">Volumetric Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Throughout the world, Curry has the well deserved reputation as the most advanced and comprehensive tool for Multimodal Neuroimaging. Curry’s strength has always been centered on combining functional data such as EEG and MEG with imaging data from MRI and CT to optimize source reconstruction. Now we are combining Curry’s strength with the acquisition and signal processing features of the SCAN software for a comprehensive EEG acquisition, data analysis, source localization and source imaging package. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=578&screenshot_id=472"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=578&screenshot_id=472" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 20, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Commercial License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.neuroscan.com/curry.cfm">http://www.neuroscan.com/curry.cfm</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Compumedics <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cutnii---custom-angle-slicer-for-brain-atlas-volumes" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CutNII - Custom-angle slicer for brain atlas volumes">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cutnii---custom-angle-slicer-for-brain-atlas-volumes" rel="bookmark" name="cutnii---custom-angle-slicer-for-brain-atlas-volumes">CutNII - Custom-angle slicer for brain atlas volumes</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> CutNII provides visualization and slicing of 3D isotropic image data (e.g. MRI) and atlases. The software features orthogonal views of the three standard planes (coronal, sagittal, horizontal) and a custom-angle slice cut through the volume. Slicing generates an arbitrarily positioned, oriented and sized rectangular sample from the dataset.<br />
<br />
The main use of CutNII is to produce custom-angle atlas slices that match histological sections with non-standard cutting angles. In addition, custom slices allow inspection of anatomical features from non-conventional angles.<br />
<br />
Multi-modality atlasing datasets to be provided with CutNII:<br />
<br />
- Waxholm Space Atlas of the Sprague Dawley rat brain (T2* MRI, DTI, and delineations)<br />
- Allen Mouse Brain Atlas reference atlas (grayscale Nissl volume reconstruction and delineations) </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 13, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cutnii/">http://www.nitrc.org/projects/cutnii/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Oslo <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Microsoft</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cwl-eeg-fmri" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CWL-EEG-fMRI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cwl-eeg-fmri" rel="bookmark" name="cwl-eeg-fmri">CWL-EEG-fMRI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This is a plugin for EEGlab that does Carbon-Wire Loops regression for getting rid of Ballistocardiac artifacts and other motion-related artifacts such as the Helium pump artifact (for the paper of its performance on the He pump artifact, see &quot;Carbon-wire loop based artifact correction outperforms post-processing EEG/fMRI corrections-A validation of a real-time simultaneous EEG/fMRI correction method.&quot;, NeuroImage, 2016 Jan 15;125:880-94.<br />
<br />
This EEGLAB plugin also comes with a (very) small example dataset for testing out the regression: this dataset has got 30 EEG channels, 1 EOG, 1 ECG and 6 CWL signals (the latter of which are used for the regression of the EEG and the EOG channel). <br />
<br />
A full CWL-EEG/fMRI 8-subject eyes open/eyes closed data set (EEG and MR data ~ 25 Gig) is available at the NITRC website (see: <a href="https://www.nitrc.org/search/?type_of_search=group&amp;q=cwl+eeg+fmri&amp;sa.x=0&amp;sa.y=0" target="_new">https://www.nitrc.org/search/?type_of_se...</a> ) See also the following NITRC Project for more information: <a href="https://www.nitrc.org/projects/cwleegfmri_data/" target="_new">https://www.nitrc.org/projects/cwleegfmr...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 22, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MIT/X Consortium License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cwl_eeg_fmri/">http://www.nitrc.org/projects/cwl_eeg_fmri/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Max Planck Institute (Leipzig) <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cwl-eeg-fmri-data-set" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CWL EEG fMRI Data Set">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cwl-eeg-fmri-data-set" rel="bookmark" name="cwl-eeg-fmri-data-set">CWL EEG fMRI Data Set</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> EEG/fMRI Data from 8 subject doing a simple eyes open/eyes closed task is provided on this webpage.<br />
<br />
The EEG/fMRI data are six files for each subject, with two basic factors: recording during Helium pump On and Helium pump Off, and recording during MRI scanning and without MRI scanning. In addition 'outside' EEG data is provided, before as well as after the MRI session.<br />
<br />
There are 30 EEG channels, 1 EOG channel, 1 ECG channel, as well as 6 CWL signals. A tool to use the CWL signals for regression of artifacts can be found here: <a href="https://www.nitrc.org/projects/cwl_eeg_fmri/" target="_new">https://www.nitrc.org/projects/cwl_eeg_f...</a><br />
<br />
<br />
See also the “Eyes Open – Eyes Closed” EEG/fMRI data set including dedicated “Carbon Wire Loop” motion detection channels paper linked here: <a href="http://www.sciencedirect.com/science/article/pii/S2352340916301056" target="_new">http://www.sciencedirect.com/science/art...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 31, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MIT</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cwleegfmri_data/">http://www.nitrc.org/projects/cwleegfmri_data/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>CANLAB <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cwx-optical-imaging-system" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="CWx Optical Imaging System">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cwx-optical-imaging-system" rel="bookmark" name="cwx-optical-imaging-system">CWx Optical Imaging System</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> CWx Optical Imaging System is a real-time data acquisition system used widely for functional NIRS measurements. Data from the CWx Systems is specific to fNIRS and directly compatible with Homer2 fNIRS analysis software. <br />
<br />
The Current CW6 system offers up to 32 lasers and 32 detectors in any combination for maximum channel count with an adjustable output data sample rate. Typical sample rate is 50Hz for most configurations.<br />
<br />
The CWx Software includes SDGui, used to create compatible source-detector probe geometry files designed by the user. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 29, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/techen_cwx/">http://www.nitrc.org/projects/techen_cwx/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="cytoseg" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Cytoseg">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cytoseg" rel="bookmark" name="cytoseg">Cytoseg</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The goal of the Cytoseg project is to produce tools for automatic segmentation of of 3D electron microscopy, specifically neuropil. The project is written in Python and uses the pythonxy platform (which includes scipy and ITK image processing tools). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 11, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Lesser General Public License (LGPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://cytoseg.googlecode.com">http://cytoseg.googlecode.com</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="daikon" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Daikon">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=daikon" rel="bookmark" name="daikon">Daikon</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Daikon is a pure JavaScript DICOM reader:<br />
- Works in the browser and Node.js environments.<br />
- Parses DICOM headers and reads image data.<br />
- Supports compressed DICOM data.<br />
- Orders and concatenates multi-file image data.<br />
- Supports RGB and Palette data.<br />
- Supports Siemens &quot;Mosaic&quot; image data.<br />
- Parses Siemens CSA header.<br />
<br />
Supported Transfer Syntax:<br />
1.2.840.10008.1.2 (Implicit VR Little Endian)<br />
1.2.840.10008.1.2.1 (Explicit VR Little Endian)<br />
1.2.840.10008.1.2.2 (Explicit VR Big Endian)<br />
1.2.840.10008.1.2.1.99 (Deflated Explicit VR Little Endian)<br />
1.2.840.10008.1.2.4.50 (JPEG Baseline (Process 1) Lossy JPEG 8-bit)<br />
1.2.840.10008.1.2.4.51 (JPEG Baseline (Processes 2 &amp; 4) Lossy JPEG 12-bit)<br />
1.2.840.10008.1.2.4.57 (JPEG Lossless, Nonhierarchical (Processes 14))<br />
1.2.840.10008.1.2.4.70 (JPEG Lossless, Nonhierarchical (Processes 14 [Selection 1]))<br />
1.2.840.10008.1.2.4.90 (JPEG 2000 (Lossless Only))<br />
1.2.840.10008.1.2.4.91 (JPEG 2000)<br />
1.2.840.10008.1.2.5 (RLE Lossless)<br />
<br />
<a href="https://github.com/rii-mango/Daikon" target="_new">https://github.com/rii-mango/Daikon</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 12, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD/MIT-Style Open Source License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/daikon/">http://www.nitrc.org/projects/daikon/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UTHSCSA <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>DICOM</td></tr>
			<tr class="even"><td>Output Formats</td><td>DICOM</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="dalan:-disconnectivity-atlas-for-lesion-analysis-of-networks" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="DALAN: Disconnectivity Atlas for Lesion Analysis of Networks">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=dalan:-disconnectivity-atlas-for-lesion-analysis-of-networks" rel="bookmark" name="dalan:-disconnectivity-atlas-for-lesion-analysis-of-networks">DALAN: Disconnectivity Atlas for Lesion Analysis of Networks</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> DALAN is a HARDI diffusion streamline atlas of 41 older, healthy subjects to be used with lesion masks from clinical images to provide projected medium to long distance connectivity deficits implied by WM stroke lesion damage. The atlas uses the popular Harvard-Oxford (H-O) cortical and subcortical atlas as endpoint targets so that WM connectivity deficits can be analyzed in conjunction with corresponding GM deficits as well. Octave/MatLab functions are provided to illustrate how to use the streamline atlas. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1270&screenshot_id=1191"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1270&screenshot_id=1191" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 27, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dalan/">http://www.nitrc.org/projects/dalan/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>US Department of Veterans Affairs <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="databrary" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Databrary">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=databrary" rel="bookmark" name="databrary">Databrary</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> A video-based data library for behavioral science. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 13, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.databrary.org">http://www.databrary.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="data-format-tools" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Data Format Tools">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=data-format-tools" rel="bookmark" name="data-format-tools">Data Format Tools</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=format-conversion" rel="tag">Format Conversion</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-software-resource" rel="tag">Other Software Resource</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> DFT is a loose collection of programs and configuration options that intend to make working with data more transparent to formats.<br />
<br />
Currently available is a basic specification for NIfTI-1 for the UNIX file command and proof of concept code for the concept of treating data as an abstract concept and instantiating physical instances on demand. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 30, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dft/">http://www.nitrc.org/projects/dft/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS<br>POSIX/UNIX-like</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>DICOM, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>DICOM, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="datalad" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="DataLad">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=datalad" rel="bookmark" name="datalad">DataLad</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data-model" rel="tag">Data Model</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data-resource" rel="tag">Data Resource</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database-application" rel="tag">Database Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> DataLad makes data management and data distribution more accessible. To do that it stands on the shoulders of Git and Git-annex to deliver a decentralized system for data exchange. This includes automated ingestion of data from online portals, and exposing it in readily usable form as Git(-annex) repositories, so-called datasets. The actual data storage and permission management, however, remains with the original data providers.<br />
At the moment, DataLad provides access to over 11TB of data in a variety of datasets from different resources (see <a href="http://datasets.datalad.org" target="_new">http://datasets.datalad.org</a>). It allows for efficient search, dataset(s) installation, data modifications and their tracking within Git version control, publication of datasets to http websites, S3, figshare, etc; reproducible computation, and other features. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1246&screenshot_id=1158"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1246&screenshot_id=1158" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 10, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MIT</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://datalad.org">http://datalad.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Dartmouth College (USA), University of Magdeburg (Germany) <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="datapixx" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="DATAPixx">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=datapixx" rel="bookmark" name="datapixx">DATAPixx</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=eeg" rel="tag">Eeg</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=response-monitoring" rel="tag">Response Monitoring</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=stimulus-presentation" rel="tag">Stimulus Presentation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The DATAPixx is a complete multi-function data and video processing USB peripheral for vision research. In addition to a dual-display video processor, the DATAPixx includes an array of peripherals which often need to be synchronized to video during an experiment, including a stereo audio stimulator, a button box port for precise reaction-time measurement, triggers for electrophysiology equipment, and even a complete analog I/O subsystem. Because we implemented the video controller and peripheral control on the same circuit board, you can now successfully synchronize all of your subject I/O to video refresh with microsecond precision. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=496&screenshot_id=371"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=496&screenshot_id=371" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 30, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Commercial License: VPixx</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://vpixx.com/our-products/video-i-o-hub/">http://vpixx.com/our-products/video-i-o-hub/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>VPixx Technologies <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="dbgap" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="dbGaP">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=dbgap" rel="bookmark" name="dbgap">dbGaP</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=snp's-&-genes" rel="tag">Snp'S & Genes</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The database of Genotypes and Phenotypes (dbGaP) was developed to archive and distribute the results of studies that have investigated the interaction of genotype and phenotype. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 7, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dbgap/">http://www.nitrc.org/projects/dbgap/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>NIH <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="dbgap_cleaner" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="DbGaP_Cleaner">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=dbgap_cleaner" rel="bookmark" name="dbgap_cleaner">DbGaP_Cleaner</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This tool will help assist site staff with curation of data dictionary, data item, and subject item files for preparation to uploading and sharing data with DbGaP resource. The intent is to help the user save time and effort in finding and fixing data artifacts prior to submission, since processing times can take some time and effort. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 29, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Lesser General Public License (LGPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dbgapcleaner/">http://www.nitrc.org/projects/dbgapcleaner/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Advanced Biomedical Informatics Group, LLC <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="dbsnp" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="dbSNP">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=dbsnp" rel="bookmark" name="dbsnp">dbSNP</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-web-resource" rel="tag">Other Web Resource</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=snp's-&-genes" rel="tag">Snp'S & Genes</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Single Nucleotide Polymorphism Database (dbSNP) is a free public archive for genetic variation within and across different species developed and hosted by the National Center for Biotechnology Information (NCBI) in collaboration with the National Human Genome Research Institute (NHGRI). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 7, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dbsnp/">http://www.nitrc.org/projects/dbsnp/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>NIH <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="dcm2nii" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="dcm2nii">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=dcm2nii" rel="bookmark" name="dcm2nii">dcm2nii</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> dcm2nii is a popular tool for converting images from the complicated formats used by scanner manufacturers (DICOM, PAR/REC) to the simple NIfTI format used by many scientific tools. dcm2nii works for all modalities (CT, MRI, PET, SPECT) and sequence types. Note: dcm2nii has traditionally been included with MRIcron downloads. This project describes the next generation version which we refer to as dcm2niix. Future releases will supersede the (legacy) dcm2nii distributed with MRIcron. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 24, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dcm2nii/">http://www.nitrc.org/projects/dcm2nii/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of South Carolina <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>OS Independent<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>DICOM, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>DICOM, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="dcmtk---dicom-toolkit" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="DCMTK - DICOM Toolkit">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=dcmtk---dicom-toolkit" rel="bookmark" name="dcmtk---dicom-toolkit">DCMTK - DICOM Toolkit</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=file-format" rel="tag">File Format</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=format-conversion" rel="tag">Format Conversion</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> DCMTK is a collection of libraries and applications implementing large parts the DICOM standard. It includes software for examining, constructing and converting DICOM image files, handling offline media, sending and receiving images over a network connection, as well as demonstrative image storage and worklist servers. DCMTK is is written in a mixture of ANSI C and C++. It comes in complete source code and is made available as &quot;open source&quot; software.<br />
<br />
DCMTK has been used at numerous DICOM demonstrations to provide central, vendor-independent image storage and worklist servers (CTNs - Central Test Nodes). It is used by hospitals and companies all over the world for a wide variety of purposes ranging from being a tool for product testing to being a building block for research projects, prototypes and commercial products.<br />
<br />
The DCMTK software can be compiled under Windows and a wide range of Unix operating systems. All necessary configuration scripts and project makefiles are supplied. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 27, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>DCMTK Copyright</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://dicom.offis.de/dcmtk.php.en">https://dicom.offis.de/dcmtk.php.en</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>OFFIS Computer Science Institute <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>SunOS/Solaris<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>DICOM</td></tr>
			<tr class="even"><td>Output Formats</td><td>DICOM</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="dcp：diffusion-connectome-pipeline" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="DCP：Diffusion Connectome Pipeline">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=dcp：diffusion-connectome-pipeline" rel="bookmark" name="dcp：diffusion-connectome-pipeline">DCP：Diffusion Connectome Pipeline</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The diffusion connectome pipeline is used for for building DTI networks.Firstly, it provides a friendly graphical user interface, and the users only need to use the mouse to set the parameters which are used in the whole process and click the button with label ‘RUN’. The DCP will process every participant’s data automatically, and generate DTI network of every participant. And the default parameters we provide are currently recognized as classic processing parameters. When the program is running, the user can accurately get which step the program is running through the monitor window. After the program is finished, a folder of quality control is generated, and the registration results of each subject are saved for quality checking. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 5, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>DTI connectome pipeline</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dcp/">http://www.nitrc.org/projects/dcp/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Beijing Normal University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="deformable-template-model-for-brain-analysis" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Deformable Template Model for Brain Analysis">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=deformable-template-model-for-brain-analysis" rel="bookmark" name="deformable-template-model-for-brain-analysis">Deformable Template Model for Brain Analysis</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The objective of the project is to develop computational shape models of brain structures for shape analysis. We develop a deformable template model framework based on “progressive surface deformation” method (Kim et al, IEEE-TMI: 34(6), 2015) to reconstruct target shapes with point-wise correspondence to template models robustly by minimizing the distortion of template models. Our toolbox contains a set of tools for constructing mean images, reconstructing individual surface models, computing average surface and finally computing local deformity. <br />
<br />
This toolbox was developed by KAIST and University of Edinburgh (<a href="http://cgv.kaist.ac.kr/brain" target="_new">http://cgv.kaist.ac.kr/brain</a>) </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 1, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dtmframework/">http://www.nitrc.org/projects/dtmframework/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>KAIST, University of Edinburgh <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="demon-deep-multi-contrast-brain-extraction,-application-to-human-&-animal-imaging" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="DEMON Deep Multi-Contrast Brain Extraction, Application to Human & Animal Imaging">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=demon-deep-multi-contrast-brain-extraction,-application-to-human-&-animal-imaging" rel="bookmark" name="demon-deep-multi-contrast-brain-extraction,-application-to-human-&-animal-imaging">DEMON Deep Multi-Contrast Brain Extraction, Application to Human & Animal Imaging</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=manual" rel="tag">Manual</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=neural-network" rel="tag">Neural Network</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> DEMON (DEep Multi-cONtrast skullstripping) is a deep convolutional neural network (CNN) based software tool to generate brainmasks (or skullstrip) from multi-contrast MR brain images, such as T1, T2, PD, or FLAIR. Unlike most other skullstripping methods, DEMON can take multi-contrast inputs, and based on multiple atlases, it generates a brainmask of a given subject image. It is especially useful for images with pathology such as traumatic brain injury (TBI), where most of the traditional skullstripping algorithms perform poorly. DEMON has been shown to be robust on both normal and pathological human brains, as well as rodent (mice and rats) brains.<br />
<br />
In addition to the software, we also provide 2 sets of atlas images, one human brains with TBI, and one normal mice . Each atlas contains multi-contrast MRI and their manually delineated brainmasks. If you use DEMON in your research, please cite the following paper,<br />
<a href="https://ieeexplore.ieee.org/abstract/document/8363667/" target="_new">https://ieeexplore.ieee.org/abstract/doc...</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1288&screenshot_id=1197"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1288&screenshot_id=1197" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 6, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Academic Free License ("AFL")</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/demon/">http://www.nitrc.org/projects/demon/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Henry Jackson Foundation <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>Other UNIX-like</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="denoising-of-redundant-mr-series" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Denoising of redundant MR series">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=denoising-of-redundant-mr-series" rel="bookmark" name="denoising-of-redundant-mr-series">Denoising of redundant MR series</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Denoising of MR series (e.g. diffusion MRI, functional MRI, relaxometry, ...) by exploiting data redundancy in the PCA domain using the prior knowledge that the eigenspectrum of random covariance matrices is described by the universal Marchenko Pastur distribution. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 19, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MPPCA</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mppca/">http://www.nitrc.org/projects/mppca/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>NYU School of Medicine <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="density-spectral-array-of-bis-vista-monitoring-system-in-a-functional-hemispherectomy" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Density Spectral Array of BIS VISTA Monitoring System in a functional hemispherectomy">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=density-spectral-array-of-bis-vista-monitoring-system-in-a-functional-hemispherectomy" rel="bookmark" name="density-spectral-array-of-bis-vista-monitoring-system-in-a-functional-hemispherectomy">Density Spectral Array of BIS VISTA Monitoring System in a functional hemispherectomy</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Bilateral Bispectral index (BIS) was designed to allow the user to record and display four channels of EEG; two from each side of the brain. This monitor was used to detect changes in EEG, and in the power spectrum distribution using the density spectral array (DSA) in both hemispheres1. Functional hemispherectomy (FH) is a surgical technique used in pediatric population to treat drug-resistant epilepsy caused by extensive hemispheric lesions. There are few cases in the literature about FH in adults but there aren’t about the use of DSA in this surgery. We present the case of a patient scheduled for a right FH to treat epilepsy where the DSA provides information about the depth of anaesthesia and sedation, and it was helpful for monitoring the effectiveness of the surgery. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 19, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Academic Free License ("AFL")</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dsa-fh/">http://www.nitrc.org/projects/dsa-fh/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Hospital del Mar <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="densparcorr-:-dens-based-method-for-partial-correlation-estimation-in-large-scale-brain-networks" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="DensParcorr : Dens-Based Method for Partial Correlation Estimation in Large Scale Brain Networks">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=densparcorr-:-dens-based-method-for-partial-correlation-estimation-in-large-scale-brain-networks" rel="bookmark" name="densparcorr-:-dens-based-method-for-partial-correlation-estimation-in-large-scale-brain-networks">DensParcorr : Dens-Based Method for Partial Correlation Estimation in Large Scale Brain Networks</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> DensParcorr is a R package for estimating the direct functional connectivity in large scale brain networks based on partial correlation. Specifically, this package implements the statistical method proposed in Wang et al (2016) which utilized the constrained L1-minimization approach (CLIME) to estimate precision matrix and then applied the Dens-based tuning parameter selection method to help select an appropriate tuning parameter for sparsity control in the network estimation. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 31, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/densparcorr_1/">http://www.nitrc.org/projects/densparcorr_1/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Emory University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="dentaltools" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="DentalTools">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=dentaltools" rel="bookmark" name="dentaltools">DentalTools</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This package provides 3Dimaging resources such as multimodal imaging, volume and mesh processing or segmentation for Dental Research. DentalTools package intends to contribute to the free exchange of information and methods in the dentistry research community. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 18, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>3D Slicer License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dentaltools/">http://www.nitrc.org/projects/dentaltools/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina at Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="developmental-resting-state-fmri-and-structrual-sample-10-26-yo" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Developmental Resting State fMRI and Structrual Sample 10-26 yo">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=developmental-resting-state-fmri-and-structrual-sample-10-26-yo" rel="bookmark" name="developmental-resting-state-fmri-and-structrual-sample-10-26-yo">Developmental Resting State fMRI and Structrual Sample 10-26 yo</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Eyes closed 5 min rest and structural 1.5 TR 3T Siemens Trio in developmenetal sample 10-26 years of age. 192 subjects. Data used in PLOS Bio paper, Contribution of Network Organization and Integration to the development of Cognitive Control<br />
Download: <a href="http://devrsfmri_2015.projects.nitrc.org/devrsfmri_2015.tar.bz2" target="_new">http://devrsfmri_2015.projects.nitrc.org...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 23, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Open Data Commons Open Database License (ODbL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/devrsfmri_2015/">http://www.nitrc.org/projects/devrsfmri_2015/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pittsburgh <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="dfbidb" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="DFBIdb">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=dfbidb" rel="bookmark" name="dfbidb">DFBIdb</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> DFBIdb is a suite of tools for efficient management of neuroimaging<br />
project data. Specifically, DFBIdb was designed to allow users to quickly<br />
perform routine management tasks of sorting, archiving, exploring, exporting<br />
and organising raw data. DFBIdb was implemented as a collection of Python<br />
scripts that maintain a project-based, centralised database that is based on the<br />
XCEDE 2 data model. Project data is imported from a filesystem hierarchy of<br />
raw files, which is an often-used convention of imaging devices, using a single<br />
script that catalogues meta-data into a modified XCEDE 2 data model. During<br />
the import process data are reversibly anonymised, archived and compressed.<br />
The import script was designed to support multiple file formats and features an<br />
extensible framework that can be adapted to novel file formats. Graphical user interfaces are provided for data exploration. DFBIdb includes facilities to export, convert and organise customisable subsets of project data according to user-specified criteria. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 31, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dfbidb/">http://www.nitrc.org/projects/dfbidb/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="diamond" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="DIAMOND">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=diamond" rel="bookmark" name="diamond">DIAMOND</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database-application" rel="tag">Database Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=format-conversion" rel="tag">Format Conversion</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-display" rel="tag">Image Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-reconstruction" rel="tag">Image Reconstruction</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> UC Davis IDeA Lab Applications for Management Of Neuroimaging Data -<br />
View dicom files and assemble them into 3D volumes.<br />
View and convert between Analyze, Nifti, and Interfile.<br />
Classify and organize dicoms and 3D volumes using metadata.<br />
Search and report on a collection of scans. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 17, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/diamond/">http://www.nitrc.org/projects/diamond/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="diatrack-particle-tracking-software" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Diatrack particle tracking software">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=diatrack-particle-tracking-software" rel="bookmark" name="diatrack-particle-tracking-software">Diatrack particle tracking software</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Diatrack, powerful particle tracking (win64)<br />
<br />
Benchmark object tracking software for 2D and 3D applications, such as biological imaging, rheology, or single molecule imaging. Reliably follow and characterize hundreds of objects simultaneously with a precision that reaches 0.01 pixels (depending on data). DiaTrack provides a compelling set of tools to extract quantitative information from image sequences.<br />
<br />
Diatrack can be downloaded under <a href="http://www.diatrack.org/contact.html" target="_new">http://www.diatrack.org/contact.html</a><br />
zip file is about 30MB; additional Mathworks libraries are automatically downloaded and installed by Setup. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 5, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/diatrack/">http://www.nitrc.org/projects/diatrack/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>www.diatrack.org <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="dicccol-predictor-(v0.1)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="DICCCOL predictor (v0.1)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=dicccol-predictor-(v0.1)" rel="bookmark" name="dicccol-predictor-(v0.1)">DICCCOL predictor (v0.1)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> DICCCOL predictor (v0.1) is a toolbox to predict 358 DICCCOL landmarks on a new brain given b0, brain surface data and DTI derived fiber data (vtk format). DICCCOL is the abbreviation of Dense Individualized and Common Connectivity-based Cortical landmarks (<a href="http://dicccol.cs.uga.edu" target="_new">http://dicccol.cs.uga.edu</a>) and developed by CAID (caid.cs.uga.edu). Each DICCCOL landmark is defined by group-wise consistent white-matter fiber connection patterns derived from diffusion tensor imaging (DTI) data. DICCCOL aims to provide large-scale cortical landmarks with finer granularity, better functional homogeneity, more accurate functional localization, and automatically-established cross-subjects correspondence. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 8, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GPL-Style Open Source</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://caid.cs.uga.edu">http://caid.cs.uga.edu</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>CAID lab, the University of Georgia <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="dicombrowser" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="DicomBrowser">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=dicombrowser" rel="bookmark" name="dicombrowser">DicomBrowser</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-software-resource" rel="tag">Other Software Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> DicomBrowser is a platform-independent desktop tool for inspecting DICOM header fields, editing DICOM header fields, viewing DICOM images, and transferring DICOM files to a DICOM receiver. DicomBrowser includes scriptable header editing to support various de-identification protocols. Dicomrowser is written in Java and uses ImageJ for image viewing and the dcm4che toolkit for much of its DICOM implementation. For more information: <a href="http://nrg.wustl.edu/software/dicom-browser" target="_new">http://nrg.wustl.edu/software/dicom-brow...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 20, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dicom_browser/">http://www.nitrc.org/projects/dicom_browser/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Washington University School of Medicine <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>DICOM</td></tr>
			<tr class="even"><td>Output Formats</td><td>DICOM</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="dicomconvert" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="DICOMConvert">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=dicomconvert" rel="bookmark" name="dicomconvert">DICOMConvert</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=software" rel="tag">Software</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Dicom Converteris based on the ITK IO mechanism for reading and writing images. The formats currently supported by the converter are DICOM to: <br />
 Analyze (*.hdr)<br />
 MetaImage (*.mhd)<br />
 Nrrd (*.nhdr, *.nrrd)<br />
In addition the application supports drop directory support. Please check for updates and leave feedback at:<br />
<br />
Dr Awais Mansoor (<a href="mailto:awais.mansoor@gmail.com" target="_new">awais.mansoor@gmail.com</a>) and Dr. Ulas Bagci (Ulasbagci@gmail.com).<br />
Center for Infectious Disease Imaging (CIDI)<br />
Department of Radiology and Imaging Sciences<br />
National Institutes of Health (NIH), Bethesda, MD. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=837&screenshot_id=786"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=837&screenshot_id=786" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 19, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dicomconvert/">http://www.nitrc.org/projects/dicomconvert/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>National Institutes of Health <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, NIfTI-1, Nrrd</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, NIfTI-1, Nrrd</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="dicom_uploadgui" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="DICOM_UploadGUI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=dicom_uploadgui" rel="bookmark" name="dicom_uploadgui">DICOM_UploadGUI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> UploadGUI is a Java tool that takes an unorganized collection of DICOM scans, sorts and categorizes them according to user-customizable rules, gathers metadata about the scans, and saves out this information to help facilitate data uploads. Batch pr </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 3, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dicomuploadgui/">http://www.nitrc.org/projects/dicomuploadgui/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="diffusional-kurtosis-estimator" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Diffusional Kurtosis Estimator">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=diffusional-kurtosis-estimator" rel="bookmark" name="diffusional-kurtosis-estimator">Diffusional Kurtosis Estimator</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quantification" rel="tag">Quantification</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=super-tensor" rel="tag">Super Tensor</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Diffusional Kurtosis Estimator (DKE) is a software tool for post-processing diffusional kurtosis imaging (DKI) datasets that includes a suite of command-line programs along with a graphical user interface (GUI). DKE supports 32-, 64-bit Windows and linux. DKE generates a set of kurtosis (axial, mean, radial, KFA, MKT) parametric maps with a given set of diffusion weighted images acquired from a valid DKI protocol. Diffusivity (axial, mean, radial) and fractional anisotropy maps using either DKI or diffusion tensor imaging signal models are also calculated in the processing. DKE features include: DICOM, NIfTI and Bruker format support, interactive (GUI) as well as batch mode (command-line) processing, and rigid-body motion correction. DKE implements the methods described in the following paper: Tabesh A, Jensen JH, Ardekani BA, and Helpern JA. Estimation of tensors and tensor-derived measures in diffusional kurtosis imaging. Mag Reson Med. 2011 Mar;65(3):823-36. <a href="http://www.ncbi.nlm.nih.gov/pubmed/21337412" target="_new">http://www.ncbi.nlm.nih.gov/pubmed/21337...</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=652&screenshot_id=568"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=652&screenshot_id=568" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 27, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>DKE License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://musc.edu/cbi/dki">http://musc.edu/cbi/dki</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Medical University of South Carolina <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Windows Vista<br>Windows XP</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>DICOM, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>DICOM, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="diffusion-mri-@-dkfz-heidelberg" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Diffusion MRI @ DKFZ Heidelberg">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=diffusion-mri-@-dkfz-heidelberg" rel="bookmark" name="diffusion-mri-@-dkfz-heidelberg">Diffusion MRI @ DKFZ Heidelberg</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This central project points to all open-source and open-data initiatives provide by the German Cancer Research Center in the field of diffusion MRI. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 19, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD/MIT-Style Open Source License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dkfz-diffusion/">http://www.nitrc.org/projects/dkfz-diffusion/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>German Cancer Research Center <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="diffusion-mri---in-vivo-and-phantom-data" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Diffusion MRI - In-vivo and Phantom Data">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=diffusion-mri---in-vivo-and-phantom-data" rel="bookmark" name="diffusion-mri---in-vivo-and-phantom-data">Diffusion MRI - In-vivo and Phantom Data</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This projects is an open-data initiative for the distributation of common datasets for the evaluation and validation of diffusion MRI processing methods. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=627&screenshot_id=571"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=627&screenshot_id=571" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 19, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD/MIT-Style Open Source License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.dkfz.de/en/medphysrad/projectgroups/dwi/DTI_projects.html#inhalt3">http://www.dkfz.de/en/medphysrad/projectgroups/dwi/DTI_projects.html#inhalt3</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>German Cancer Research Center <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1, Nrrd</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1, Nrrd</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="diffusion-mri-orientation-distribution-function-in-constant-solid-angle-(csa-odf)-and-hough-transform-tractography" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Diffusion MRI Orientation Distribution Function in Constant Solid Angle (CSA-ODF) and Hough-Transform Tractography">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=diffusion-mri-orientation-distribution-function-in-constant-solid-angle-(csa-odf)-and-hough-transform-tractography" rel="bookmark" name="diffusion-mri-orientation-distribution-function-in-constant-solid-angle-(csa-odf)-and-hough-transform-tractography">Diffusion MRI Orientation Distribution Function in Constant Solid Angle (CSA-ODF) and Hough-Transform Tractography</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=connectivity-analysis" rel="tag">Connectivity Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=diffusion-application" rel="tag">Diffusion Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=diffusion-mr-fiber-tracking" rel="tag">Diffusion Mr Fiber Tracking</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=q-ball" rel="tag">Q-Ball</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This is a diffusion-weighted MRI processing Matlab toolbox (including binaries), which can be used to:<br />
•Compute the Q-Ball Imaging Orientation Distribution Function in Constant Solid Angle (CSA-ODF) for Diffusion-Weighted MRI, introduced in (Aganj et al, MRM 2010).<br />
•Perform Hough-transform tractography, as described in (Aganj et al, MedIA 2011).<br />
•Visualize the ODFs and tracts, and export them for further analysis.<br />
•Verify the correctness of the diffusion gradient table, following (Aganj, Sci Rep 2018).<br />
<br />
For short tutorials, see EXAMPLE.m and EXAMPLE_CLI.m. If available, GPUs can be used for speed-up. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1148&screenshot_id=1077"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1148&screenshot_id=1077" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 10, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/csaodf-hough">http://www.nitrc.org/projects/csaodf-hough</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Minnesota, and Harvard Medical School / MGH <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="diffusion-mri-tool" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Diffusion MRI Tool">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=diffusion-mri-tool" rel="bookmark" name="diffusion-mri-tool">Diffusion MRI Tool</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-reconstruction" rel="tag">Image Reconstruction</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=modeling" rel="tag">Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> DMRITool is a free and open source toolbox for diffusion MRI data processing. It is written in C++ with Matlab interface. You can also use the released mex executables in matlab.<br />
<br />
DMRITool website: <br />
<a href="https://diffusionmritool.github.io/" target="_new">https://diffusionmritool.github.io/</a><br />
<br />
Latest source code: <br />
<a href="https://github.com/DiffusionMRITool/dmritool" target="_new">https://github.com/DiffusionMRITool/dmri...</a><br />
<br />
Mailing list:<br />
<a href="http://www.nitrc.org/mailman/listinfo/dmritool-discussion" target="_new">http://www.nitrc.org/mailman/listinfo/dm...</a><br />
<br />
DMRITool is currently supported in SQITS at NIH<br />
<a href="https://science.nichd.nih.gov/confluence/display/sqits/Home" target="_new">https://science.nichd.nih.gov/confluence...</a><br />
<br />
With DMRITool, you can: <br />
<br />
* perform reconstruction/estimation of diffusion data, including diffusion weighted signal, ensemble average propagator (EAP), diffusion orientation distribution function (dODF), and some meaningful scalar maps, etc.<br />
* generate spherically uniform sampling schemes for single or multiple shells.<br />
* perform diffusion MRI data simulation. <br />
* visualize spherical function field (e.g. dODF field, EAP profile field) </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 8, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dmritool/">http://www.nitrc.org/projects/dmritool/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>NIH <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS<br>POSIX/UNIX-like</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1, Nrrd</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1, Nrrd</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="diffusion-toolkit-/-trackvis" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Diffusion Toolkit / TrackVis">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=diffusion-toolkit-/-trackvis" rel="bookmark" name="diffusion-toolkit-/-trackvis">Diffusion Toolkit / TrackVis</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=diffusion-mr-fiber-tracking" rel="tag">Diffusion Mr Fiber Tracking</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-reconstruction" rel="tag">Image Reconstruction</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Diffusion Toolkit (with TrackVis) is a cross-platform software package that does reconstruction, fiber tracking, visualization and analysis on various diffusion imaging data. Features of the software include:<br />
<br />
• Handles DTI, DSI, Q-Ball and HARDI imaging techniques.<br />
• Works on Windows, Mac OS X and Linux with native look and feel. Native 64-bit support on Mac OS X and Linux.<br />
• Fast streamlined data processing.<br />
• Intuitive GUI front-end with command-line driven back-end. Thus, allows advanced user to write their own scripts for automated multiple dataset processing.<br />
• Real-time fiber track visualization and analysis. Parameter adjustment applied to 3D render on the fly. Various tracking selection methods (filters) allows locating specific track bundle with ease. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=112&screenshot_id=138"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=112&screenshot_id=138" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 17, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MGH CSRL License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.trackvis.org">http://www.trackvis.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="diffusion-tractography-with-kalman-filter" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Diffusion Tractography with Kalman Filter">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=diffusion-tractography-with-kalman-filter" rel="bookmark" name="diffusion-tractography-with-kalman-filter">Diffusion Tractography with Kalman Filter</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=diffusion-mr-fiber-tracking" rel="tag">Diffusion Mr Fiber Tracking</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> We present a framework which uses an unscented Kalman filter for performing tractography. At each point on the fiber the most consistent direction is found as a mixture of previous estimates and of the local model.<br />
<br />
It is very easy to expand the framework and to implement new fiber representations for it. Currently it is possible to tract fibers using two different 1-, 2-, or 3-tensor methods. Both methods use a mixture of Gaussian tensors. One limits the diffusion ellipsoids to a cylindrical shape (the second and third eigenvalue are assumed to be identical) and the other one uses a full tensor representation.<br />
<br />
The project is written in C++. It could be used both as a Slicer3 module and as a standalone commandline application. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 28, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Lesser General Public License (LGPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/ukftractography/">http://www.nitrc.org/projects/ukftractography/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Harvard Medical School <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>POSIX/UNIX-like<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Nrrd, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>Nrrd, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="diffusion-warp" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Diffusion Warp">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=diffusion-warp" rel="bookmark" name="diffusion-warp">Diffusion Warp</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This project contains tools appropriate for the registration of diffusion tensor images to an average coordinate system. The tools include image registration methods and algorithms for the correct alignment of the diffusion tensor when applying the resulting transformation. The program uses the Slicer3 execution model framework to define the command line arguments, and can be fully integrated using the module discovery capabilities of Slicer3. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 28, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/diffusionwarp/">http://www.nitrc.org/projects/diffusionwarp/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="dinifti" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="dinifti">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=dinifti" rel="bookmark" name="dinifti">dinifti</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=format-conversion" rel="tag">Format Conversion</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Convert DICOM images to NIfTI format. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 23, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://as.nyu.edu/cbi/resources/Software/DINIfTI.html">http://as.nyu.edu/cbi/resources/Software/DINIfTI.html</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>FreeBSD<br>Linux</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="disease-state-prediction" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Disease State Prediction">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=disease-state-prediction" rel="bookmark" name="disease-state-prediction">Disease State Prediction</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> These are the scripts used for the analyses reported in: Craddock RC, Holtzheimer PE, 3rd, Hu XP, Mayberg HS. (2009): Disease state prediction from resting state functional connectivity. Magn Reson Med 62(6):1619-28.<br />
<br />
Specifically included are scripts for performing t-test filter, reliability filter, recursive feature elimination, and reliability recursive feature elimination feature selection methods. These make use of wrappers that perform .632 bootstrap and k-fold cross validation strategies. The scripts are written in matlab and require the Bioinformatics toolbox. If you do not have the bioinformatics toolbox, the scripts can be easily modified to run with other matlab SVM toolboxes (i.e., libsvm, svmlight, shogun, etc.). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 12, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/diseasestate/">http://www.nitrc.org/projects/diseasestate/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="displacement-field-viewer" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Displacement Field Viewer">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=displacement-field-viewer" rel="bookmark" name="displacement-field-viewer">Displacement Field Viewer</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Displacement field viewer or DFViewer is a simple tool for visualizing displacement fields estimated in association with image registration. Based on the displacement vector field, a mesh is generated for visualization. The mesh can be color mapped with the jacobian determinant at each point for better localization of regions that undergo compression or expansion.<br />
<br />
Key features: <br />
1) Jacobian determinant color mapping. <br />
2) View synchronization. <br />
3) Save/Load view configurations. <br />
4) Conversion from deformation fields. <br />
5) Conversion from HAMMER displacement fields. <br />
6) Drag-and-drop mechanism for opening file.<br />
7) Adjustable mesh resolution.<br />
8) Option to save mesh in VTK format.<br />
9) Option to view mesh in different image planes.<br />
10) Glyph (arrow) view.<br />
11) Accepts multiple image formats, including NIfTI (.nii.gz or .nii), NRRD (.nrrd or .nhdr), and MetaImage (.mha or .mhd). 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=782&screenshot_id=709"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=782&screenshot_id=709" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 19, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dfviewer/">http://www.nitrc.org/projects/dfviewer/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1, Nrrd, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1, Nrrd, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="distributome" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Distributome">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=distributome" rel="bookmark" name="distributome">Distributome</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Distributome Project is an open-source, open content-development project for exploring, discovering, navigating, learning, and computational utilization of diverse probability distributions. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=409&screenshot_id=681"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=409&screenshot_id=681" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 14, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Lesser General Public License (LGPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/distributome/">http://www.nitrc.org/projects/distributome/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Michigan <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="done:-detection-of-outlier-neurons" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="DONE: Detection of Outlier NEurons">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=done:-detection-of-outlier-neurons" rel="bookmark" name="done:-detection-of-outlier-neurons">DONE: Detection of Outlier NEurons</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This tool was used by Zawadzki et al. (2012), who reported on a morphology-based approach for the automatic identification of outlier neurons and its application to the NeuroMorpho database. For the analysis, each neuron is represented by a feature vector composed of 20 measurements, which are projected into lower dimensional space with PCA. Bivariate kernel density estimation is then used to obtain a probability distribution for cells. Cells with high probabilities are understood as archetypes, while those with the small probabilities are classified as outliers. Further details about the method and its application in other domains can be found in Costa et al. (2009) and Echtermeyer et al. (2011).<br />
<br />
References:<br />
* Costa, Rodrigues, Hilgetag, and Kaiser. Europhysics Letters, 87, 1 (2009)<br />
* Echtermeyer, Costa, Rodrigues, Kaiser. PLoS ONE 6, 9 (2011)<br />
* Zawadzki, Feenders, Viana, Kaiser, and Costa. Neuroinformatics (2012) </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 2, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/done/">http://www.nitrc.org/projects/done/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="dots-wm-tract-segmentation" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="DOTS WM tract segmentation">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=dots-wm-tract-segmentation" rel="bookmark" name="dots-wm-tract-segmentation">DOTS WM tract segmentation</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Diffusion-Oriented Tract Segmentation (DOTS) is a fast, scalable tool developed at the Johns Hopkins University to automatically segment the major anatomical fiber tracts within the human brain from clinical quality diffusion tensor MR imaging. With an atlas-based Markov Random Field representation, DOTS directly estimates the tract probabilities, bypassing tractography and associated issues. Overlapping and crossing fibers are modeled and DOTS can also handle white matter lesions. <br />
<br />
DOTS is released as a plug-in for the MIPAV software package and as a module for the JIST pipeline environment. They are therefore cross-platform and compatible with a wide variety of file formats. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=439&screenshot_id=318"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=439&screenshot_id=318" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 18, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dots/">http://www.nitrc.org/projects/dots/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Johns Hopkins University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="dparsf" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="DPARSF">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=dparsf" rel="bookmark" name="dparsf">DPARSF</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Data Processing Assistant for Resting-State fMRI (DPARSF) is a convenient plug-in software based on SPM and REST. You just need to arrange your DICOM files, and click a few buttons to set parameters, DPARSF will then give all the preprocessed (slice timing, realign, normalize, smooth) data, FC, ReHo, ALFF and fALFF results. DPARSF can also create a report for excluding subjects with excessive head motion and generate a set of pictures for easily checking the effect of normalization. You can use DPARSF to extract AAL or ROI time courses (or extract Gray Matter Volume of AAL regions, command line only) efficiently if you want to perform small-world analysis.<br />
<br />
Please download it and find more information from <a href="http://rfmri.org/DPARSF" target="_new">http://rfmri.org/DPARSF</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 25, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dparsf/">http://www.nitrc.org/projects/dparsf/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="dramms-deformable-image-registration-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="DRAMMS Deformable Image Registration Toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=dramms-deformable-image-registration-toolbox" rel="bookmark" name="dramms-deformable-image-registration-toolbox">DRAMMS Deformable Image Registration Toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=motion-analysis" rel="tag">Motion Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=registration" rel="tag">Registration</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=shape-analysis" rel="tag">Shape Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=warping" rel="tag">Warping</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> DRAMMS is for 2D, 3D, 3.5D, 4D and population-wise deformable image registration. Typical applications include,<br />
<br />
-- Cross-subject registration;<br />
<br />
-- Single- or Multi-site data;<br />
<br />
-- Single- or Multi-modal registration; <br />
<br />
-- Longitudinal registration for quantifying changes;<br />
<br />
-- Registration under missing correspondences.<br />
<br />
-- Group-wise registration for unbiased atlas construction.<br />
<br />
DRAMMS runs in command line in UNIX/Mac OS, It accepts Nifti/ANALYZE/MetaImage image formats. It is fully-automatic --- takes two input images, and generates a registered image and (optionally) the deformation field; no user initialization or interaction is needed.<br />
<br />
More information (installation, tutorial, manual, demonstration, FAQ, etc) can be found at <a href="http://www.rad.upenn.edu/sbia/software/dramms/" target="_new">http://www.rad.upenn.edu/sbia/software/d...</a> .<br />
<br />
4d-dramms: <a href="https://www.nitrc.org/projects/dramms4d/" target="_new">https://www.nitrc.org/projects/dramms4d/</a><br />
group-wise dramms: <a href="https://www.nitrc.org/projects/popdramms/" target="_new">https://www.nitrc.org/projects/popdramms...</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=647&screenshot_id=686"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=647&screenshot_id=686" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 25, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.cbica.upenn.edu/sbia/software/dramms">http://www.cbica.upenn.edu/sbia/software/dramms</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pennsylvania <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>BSD<br>Cygwin<br>IRIX<br>Linux<br>MacOS<br>Other UNIX-like<br>SunOS/Solaris</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="draw3d-and-meshinator" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Draw3D and Meshinator">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=draw3d-and-meshinator" rel="bookmark" name="draw3d-and-meshinator">Draw3D and Meshinator</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Draw3D is a 3D rendering tool written entirely in VTK-TCL script. It is intended for fast command line rendering and visual inspection of datasets commonly found in medical imaging. It also allows the generation of images for reports or videos. As it is based on pure VTK, it can render whatever VTK can render, and runs wherever VTK can run. <br />
<br />
Meshinator is a simpler tool that uses VTKs isosurface functions to generate meshes from volumetric data.<br />
<br />
Please see the wiki for documentation 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=469&screenshot_id=341"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=469&screenshot_id=341" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 27, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/draw3d/">http://www.nitrc.org/projects/draw3d/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="dsi-studio" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="DSI Studio">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=dsi-studio" rel="bookmark" name="dsi-studio">DSI Studio</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=diffusion-mr-fiber-tracking" rel="tag">Diffusion Mr Fiber Tracking</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=diffusion-spectrum" rel="tag">Diffusion Spectrum</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-reconstruction" rel="tag">Image Reconstruction</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=q-ball" rel="tag">Q-Ball</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quantification" rel="tag">Quantification</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tensor-metric" rel="tag">Tensor Metric</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> DSI Studio is a software for diffusion MR images analysis. The provided functions include reconstruction (DTI, QBI, DSI, and GQI), deterministic fiber tracking, and 3D visualization. It has a window-based interface and operates on Microsoft Windows system. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=160&screenshot_id=642"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=160&screenshot_id=642" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 25, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://dsi-studio.labsolver.org">http://dsi-studio.labsolver.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, NIfTI-1, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, NIfTI-1, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="dt_bold_calc" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="DT_BOLD_calc">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=dt_bold_calc" rel="bookmark" name="dt_bold_calc">DT_BOLD_calc</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> matlab script to calculate the sigmoidal delay function of Henson<br />
dx.doi.org/10.1006/nimg.2001.0940<br />
for the SPM.mat and condition specified in the input. Output as images of dt </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 26, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Academic Free License ("AFL")</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/sigmoidal_dt/">http://www.nitrc.org/projects/sigmoidal_dt/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Newcastle University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="dti-atlas-builder" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="DTI Atlas Builder">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=dti-atlas-builder" rel="bookmark" name="dti-atlas-builder">DTI Atlas Builder</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=application" rel="tag">Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This tool creates an Atlas image as an average of several DTI images that will be registered. The registration will be done in two steps :<br />
- Affine Registration with BRAINSFit<br />
- Non Linear Registration with GreedyAtlas<br />
A final step will apply the transformations to the original DTI images so that the final average can be computed.<br />
<br />
The main function writes a python script that will be executed to compute the Atlas.<br />
<br />
By running DTIAtlasBuilder, you will need to fill in informations in a Graphical User Interface, and then compute the Atlas. You can also run the tool in command line (no GUI).<br />
Using the GUI, you will be able to save or load a dataset file or a parameter file.<br />
<br />
The tool needs these other tools to work, so be sure to have these installed on your computer:<br />
- ImageMath<br />
- ResampleDTIlogEuclidean<br />
- CropDTI<br />
- dtiprocess<br />
- BRAINSFit<br />
- GreedyAtlas<br />
- dtiaverage<br />
- DTI-Reg<br />
- unu<br />
- MriWatcher (requires glut library)<br />
<br />
This package is also available as an extension of 3D Slicer (<a href="http://www.slicer.org" target="_new">http://www.slicer.org</a>) 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=636&screenshot_id=607"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=636&screenshot_id=607" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 12, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dtiatlasbuilder/">http://www.nitrc.org/projects/dtiatlasbuilder/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina at Chapel Hill (UNC-CH) <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="dtibrainscope" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="dtiBrainScope">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=dtibrainscope" rel="bookmark" name="dtibrainscope">dtiBrainScope</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This is a software package for processing diffusion tensor imaging data. The following functions are included: <br />
1.Converting imaging data in DICOME format to ANALYZE format<br />
2.Extracting binary brain mask for quick scalp-removing<br />
3.Correcting eddy-current induced distortion<br />
4.Optimized tensor estimation based on noisy diffusion-weighted imaging (DWI) data<br />
5.Scalp removal using a brain mask image<br />
6.Corregistering imaging data and generating deformation field for mapping images from individual spaces to a template or target space<br />
7.Spatial Normalization and Warping DTI <br />
8.Fiber tracking<br />
9.Clustering fiber tracts<br />
10.Identifying brain ventricles and generating binary masks for the baseline and DW imaging data<br />
11.Deriving diffusion anisotropy indices (DAIs) and principal directions (PD) and the corresponding color-coded PD-map. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 30, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>dtiBrainScope</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.columbia.edu/~dx2103/brainimagescope.html">http://www.columbia.edu/~dx2103/brainimagescope.html</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="dti-denoising" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="DTI denoising">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=dti-denoising" rel="bookmark" name="dti-denoising">DTI denoising</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The proposed Matlab package DWIdenoisingPackage contains six denoising filters and a noise estimation method for 4D DWI. <br />
The package includes nonlocal means, local PCA and Oracle DCT methods.<br />
Based on image redundancy and/or sparsity, the proposed filters provide efficient denoising while preserving fine structures.<br />
The package is compiled for Linux (64 bits), Windows (64 bits) and MAC (64bits). The proposed GUI enables the processing of several .nii files in 2-clicks to make easier the processing of large database. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=805&screenshot_id=737"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=805&screenshot_id=737" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 18, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dti-denoising/">http://www.nitrc.org/projects/dti-denoising/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Bordeaux University, Universidad Politécnica de Valencia <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="dti-fiber-tract-statistics" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="DTI Fiber Tract Statistics">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=dti-fiber-tract-statistics" rel="bookmark" name="dti-fiber-tract-statistics">DTI Fiber Tract Statistics</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=regression" rel="tag">Regression</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tractography" rel="tag">Tractography</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This is a software which allows the user to study the behavior of water diffusion (using DTI data) along the length of the white matter fiber-tracts.<br />
Various tract-oriented scalar diffusion measures obtained from DTI brain images, are treated as a continuous function of white matter fibers' arc-length. To analyze the trend along a given fiber tract, a command line tool performs kernel regression on this data. The idea is to try out different noise models and maximum likelihood estimates within kernel windows (along the tract), such that they best represent the data and are robust to noise and Partial Volume effect.<br />
<br />
The package contains several command line based modules and an GUI based tool called DTIAtlasFiberAnalyzer to access most functions.<br />
<br />
This package is also available as a 3D Slicer extension (<a href="http://www.slicer.org" target="_new">http://www.slicer.org</a>) called DTIAtlasFiberAnalyzer.<br />
<br />
Some documentation about the features available in this package are available here: <a href="http://www.na-mic.org/Wiki/index.php/Projects:dtistatisticsfibers" target="_new">http://www.na-mic.org/Wiki/index.php/Pro...</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=403&screenshot_id=768"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=403&screenshot_id=768" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 19, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dti_tract_stat/">http://www.nitrc.org/projects/dti_tract_stat/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Utah - UNC Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="dtiprocess-toolkit" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="DTIProcess ToolKit">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=dtiprocess-toolkit" rel="bookmark" name="dtiprocess-toolkit">DTIProcess ToolKit</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=resampling" rel="tag">Resampling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tensor-metric" rel="tag">Tensor Metric</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tractography" rel="tag">Tractography</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> DTIProcess is a DTI processing and analysis toolkit developed in UNC and University of Utah. Tools in this toolkit include dtiestim, dtiprocess, dtiaverage, fibertrack, fiberprocess, et al..<br />
Most of the tools included in this package are available in 3D Slicer (<a href="http://www.slicer.org" target="_new">http://www.slicer.org</a>) in the DTIProcess extension. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=312&screenshot_id=769"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=312&screenshot_id=769" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 15, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dtiprocess/">http://www.nitrc.org/projects/dtiprocess/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS<br>POSIX/UNIX-like<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Nrrd</td></tr>
			<tr class="even"><td>Output Formats</td><td>Nrrd</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="dti-reg" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="DTI-Reg">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=dti-reg" rel="bookmark" name="dti-reg">DTI-Reg</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=registration" rel="tag">Registration</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=resampling" rel="tag">Resampling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> DTI-Reg is an open-source C++ application that performs pair-wise DTI registration, using scalar FA map to drive the registration. <br />
<br />
Individual steps of the pair-wise registration pipeline are performed via external applications - some of them being 3D Slicer modules. Starting with two input DTI images, scalar FA maps are generated via dtiprocess. Registration is then performed between these FA maps, via BRAINSFit/BRAINSDemonWarp or ANTS -Advanced Normalization Tools-, which provide different registration schemes: rigid, affine, BSpline, diffeomorphic, logDemons. The final deformation is then applied to the source DTI image via ResampleDTIlogEuclidean.<br />
<br />
DTI-Reg is available in a 3D Slicer (<a href="http://www.slicer.org" target="_new">http://www.slicer.org</a>) extension called DTIAtlasBuilder </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 2, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dtireg/">http://www.nitrc.org/projects/dtireg/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina at Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, Nrrd, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, Nrrd, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="dti-template-rhesus-macaques" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="DTI-TEMPLATE-RHESUS-MACAQUES">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=dti-template-rhesus-macaques" rel="bookmark" name="dti-template-rhesus-macaques">DTI-TEMPLATE-RHESUS-MACAQUES</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Diffusion Tensor Imaging (DTI) studies of non-human primates (NHP) are becoming increasingly common. Recently, many DTI analysis methodologies have been developed for human brain studies; however, few are directly applicable to NHP. A prerequisite for most statistical DTI analyses to localize WM differences across populations is to spatially normalize the individual scans to a representative template. Here, we report the development of a population-specific DTI template for young adolescent Rhesus Macaque (Macaca mulatta) monkeys using 271 high-quality scans. Using such a large number of animals in generating a template allows it to account for variability in the species. Our DTI template is based on the largest number of animals ever used in generating a computational brain template. It is anticipated that our DTI template will help facilitate voxel-based and tract specific WM analyses in non-human primate species, which in turn may increase our understanding of brain function, development, and evolution. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=429&screenshot_id=337"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=429&screenshot_id=337" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 4, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>DTI-TK license</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/rmdtitemplate/">http://www.nitrc.org/projects/rmdtitemplate/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Wisconsin-Madison <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="dti-tk" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="DTI-TK">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=dti-tk" rel="bookmark" name="dti-tk">DTI-TK</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-to-template" rel="tag">Image-To-Template</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=intersubject" rel="tag">Intersubject</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> DTI-TK is a spatial normalization &amp; atlas construction toolkit, designed from ground up to support the manipulation of diffusion-tensor images (DTI) with special cares taken to respect the tensorial nature of the data. It implements a state-of-the-art registration algorithm that drives the alignment of white matter (WM) tracts by matching the orientation of the underlying fiber bundle at each voxel. The algorithm has been shown to both improve WM tract alignment and to enhance the power of statistical inference in clinical settings.<br />
<br />
The key features include:<br />
<br />
- NIfTI support for scalar, vector and DTI volumes<br />
<br />
- tool chains for manipulating DTI volumes: resampling, smoothing, warping, registration &amp; visualization<br />
<br />
- pipelines for WM morphometry: spatial normalization &amp; atlas construction for population-based studies<br />
<br />
- built-in cluster-computing support<br />
<br />
- Interoperability with other major DTI tools: AFNI, Camino, DTIStudio &amp; FSL 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=207&screenshot_id=107"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=207&screenshot_id=107" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 18, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://dti-tk.sourceforge.net">http://dti-tk.sourceforge.net</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University College London <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="dual-tree-complex-wavelet-combined-with-non-local-means-for-asl-fmri-denoising" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Dual-tree complex wavelet combined with non-local means for ASL fMRI denoising">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=dual-tree-complex-wavelet-combined-with-non-local-means-for-asl-fmri-denoising" rel="bookmark" name="dual-tree-complex-wavelet-combined-with-non-local-means-for-asl-fmri-denoising">Dual-tree complex wavelet combined with non-local means for ASL fMRI denoising</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Dual-tree complex wavelet combined with non-local means ASL fMRI denoising is a software toolbox that can denoise MR images, especially ASL fMRI images. The software:<br />
<br />
(1) Reads and writes images in analyze format;<br />
(2) Pre-processes ASL fMRI images and provides denoised images for further analysis;<br />
(3) Achieves variable denoising outcomes by changing certain parameters;<br />
(4) Can be employed for denoising other MR images, such as diffusion MRI,etc. <br />
<br />
The method is described in the following paper:<br />
Voxel-wise functional connectomics using arterial spin labeling fMRI: the role of denoising.<br />
Xiaoyun Liang, Alan Connelly, Fernando Calamante, 2015. Brain connectivity. DOI: 10.1089/brain.2014.0290. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=936&screenshot_id=928"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=936&screenshot_id=928" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 27, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dt-cwt-nlm/">http://www.nitrc.org/projects/dt-cwt-nlm/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Florey Institute of Neuroscience and Mental Health <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="dwi/dti-quality-control-tool:-dtiprep" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="DWI/DTI Quality Control Tool: DTIPrep">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=dwi/dti-quality-control-tool:-dtiprep" rel="bookmark" name="dwi/dti-quality-control-tool:-dtiprep">DWI/DTI Quality Control Tool: DTIPrep</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=application" rel="tag">Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> DTIPrep performs a &quot;Study-specific Protocol&quot; based automatic pipeline for DWI/DTI quality control and preparation. This is both a GUI and command line tool. The configurable pipeline includes image/diffusion information check, padding/Cropping of data, slice-wise, interlace-wise and gradient-wise intensity and motion check, head motion and Eddy current artifact correction, and DTI computing.<br />
<br />
DTIPrep is also available as an extension of 3D Slicer (<a href="http://www.slicer.org" target="_new">http://www.slicer.org</a>) 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=283&screenshot_id=216"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=283&screenshot_id=216" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 14, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dtiprep/">http://www.nitrc.org/projects/dtiprep/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows XP</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Nrrd</td></tr>
			<tr class="even"><td>Output Formats</td><td>Nrrd</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="dwmri-phantom-drift" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="DWMRI Phantom Drift">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=dwmri-phantom-drift" rel="bookmark" name="dwmri-phantom-drift">DWMRI Phantom Drift</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This is a set of DWMRI acquisitions of two different diffusion phantoms with interspersed images with a b-value of zero. These were used to analyzed the affects of temporal scanner drift. There are 5 unique corrections (and associated corrected DWMRI data) that was applied to this data to correct for temporal/spatial drift over the course of each scan. The corrections were implemented in MATLAB. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 13, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MASI</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/phantom_drift/">http://www.nitrc.org/projects/phantom_drift/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Vanderbilt University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="dw-mri-random-walk-simulator" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="DW-MRI Random Walk Simulator">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=dw-mri-random-walk-simulator" rel="bookmark" name="dw-mri-random-walk-simulator">DW-MRI Random Walk Simulator</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=monte-carlo" rel="tag">Monte Carlo</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-software-resource" rel="tag">Other Software Resource</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=random-walk" rel="tag">Random Walk</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=simulation" rel="tag">Simulation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The DW-MRI Random Walk Simulator provides a simple interface to simulate Brownian motion in arbitrary, complex environments. The analysis routines enable visualization of these models with DTI, q-space, and higher order diffusion weighted MRI.. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 8, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Lesser General Public License (LGPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/randomwalks/">http://www.nitrc.org/projects/randomwalks/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS<br>Microsoft<br>POSIX/UNIX-like</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="dw-mri-registration-in-fsl" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="DW-MRI registration in FSL">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=dw-mri-registration-in-fsl" rel="bookmark" name="dw-mri-registration-in-fsl">DW-MRI registration in FSL</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This code registers linearly and non-linearly Diffusion Weighted Magnetic Resonance Images (DW-MRIs) by extending FLIRT (linear registration of 3D scalar volumes) and FNIRT (non-linear registration of 3D scalar volumes) in the FMRIB Software Library (FSL) to work with 4D volumes. The basis for registering DW-MRIs is the concept of Angular Interpolation (Tao, X., Miller, J. V., 2006. A method forregistering diffusion weighted magnetic resonance images. In: MICCAI. Vol. 9. pp. 594–602), which is implemented and extended to non-linear registration, based on the FLIRT and FNIRT models in FSL. See <a href="http://www.frontiersin.org/Brain_Imaging_Methods/10.3389/fnins.2013.00041/abstract." target="_new">http://www.frontiersin.org/Brain_Imaging...</a><br />
<br />
The code does not overwrite FLIRT, FNIRT or any of the FSL C++ code. It is added as FLIRT4D, FNIRT4D and supporting cost functions. The makefiles will however be overwritten to compile the new code, without affecting any version of FSL. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 1, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>FSL License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dwiregistration/">http://www.nitrc.org/projects/dwiregistration/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Minnesota <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="dynamicbc" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="DynamicBC">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=dynamicbc" rel="bookmark" name="dynamicbc">DynamicBC</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Dynamic brain connectome (DynamicBC) analysis toolbox is a Matlab toolbox to calculate Dynamic Functional Connectivity (d-FC) and Dynamic Effective Connectivity (d-EC). Sliding window analysis (Bivariate Pearson correlation and Granger causality) and time varying parameter regression method (Flexible Least Squares) are two dynamic analysis strategies for time-variant connectivity analysis in the DynamicBC. Granger causality density/strength (GCD/GCS) and functional connectivity density/strength (FCD/FCS) analysis would be performed in this toolbox. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1296&screenshot_id=1256"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1296&screenshot_id=1256" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 22, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License v3</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://guorongwu.github.io/DynamicBC/">https://guorongwu.github.io/DynamicBC/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="e39-macaque-histology-and-mri-data" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="E39 Macaque Histology and MRI data">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=e39-macaque-histology-and-mri-data" rel="bookmark" name="e39-macaque-histology-and-mri-data">E39 Macaque Histology and MRI data</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Macaque diffusion MRI dataset and corresponding histology for validating diffusion microstructure and tractography measures. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 7, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/e39macaque/">http://www.nitrc.org/projects/e39macaque/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Vanderbilt University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="econnectome" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="eConnectome">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=econnectome" rel="bookmark" name="econnectome">eConnectome</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=forward---inverse" rel="tag">Forward - Inverse</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=multivariate-analysis" rel="tag">Multivariate Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=region-of-interest" rel="tag">Region Of Interest</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> eConnectome (Electrophysiological Connectome) is an open-source MATLAB software package for imaging brain functional connectivity from electrophysiological signals. It provides interactive graphical interfaces for EEG/ECoG/MEG preprocessing, source estimation, connectivity analysis and visualization. Connectivity from EEG/ECoG/MEG can be mapped over sensor and source domains.<br />
<br />
This package is designed for use by researchers in neuroscience, psychology, cognitive science, clinical neurophysiology, neurology and other disciplines. The graphical interface-based platform requires little programming knowledge or experience with MATLAB.<br />
<br />
eConnectome is developed by the Biomedical Functional Imaging and Neuroengineering Laboratory at the University of Minnesota, directed by Dr. Bin He. The visualization module is jointly developed with Drs. Fabio Babiloni and Laura Astolfi at the University of Rome &quot;La Sapienza&quot;. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=353&screenshot_id=938"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=353&screenshot_id=938" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 12, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://econnectome.umn.edu/">http://econnectome.umn.edu/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Windows XP</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="eegbase" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="EEGBase">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=eegbase" rel="bookmark" name="eegbase">EEGBase</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> EEGBase portal enables community researchers to store, update, download and search data and metadata from EEG/ERP experiments </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 5, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/eegbase/">http://www.nitrc.org/projects/eegbase/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of West Bohemia <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="eeg-fnirs-hybrid-smr-bci-data" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="EEG-FNIRS hybrid SMR BCI data">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=eeg-fnirs-hybrid-smr-bci-data" rel="bookmark" name="eeg-fnirs-hybrid-smr-bci-data">EEG-FNIRS hybrid SMR BCI data</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> To be filled after registering </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 22, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/eegfnirshbci/">http://www.nitrc.org/projects/eegfnirshbci/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="eeg-human-categorization-data" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="EEG human categorization data">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=eeg-human-categorization-data" rel="bookmark" name="eeg-human-categorization-data">EEG human categorization data</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This ressource is a collection of 32-channel data from 14 subjects (7 males, 7 females) acquired using the Neuroscan software. Subjects are performing a go-nogo categorization task and a go-no recognition task on natural photographs presented very briefly (20 ms). Each subject responded to a total of 2500 trials. Data is CZ referenced and is sampled at 1000 Hz (total data size is 4Gb; more details are given later). 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=644&screenshot_id=560"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=644&screenshot_id=560" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 21, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/eegdatacateg/">http://www.nitrc.org/projects/eegdatacateg/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Paul Sabatier University, Toulouse <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="eeglab" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="EEGLAB">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=eeglab" rel="bookmark" name="eeglab">EEGLAB</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=anova" rel="tag">Anova</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=eeg-modeling" rel="tag">Eeg Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=event-related-potential" rel="tag">Event Related Potential</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=format-conversion" rel="tag">Format Conversion</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=fourier-time-domain-analysis" rel="tag">Fourier Time-Domain Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=independent-component-analysis" rel="tag">Independent Component Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=meg-modeling" rel="tag">Meg Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=source-separation-analysis" rel="tag">Source Separation Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=spectral-analysis" rel="tag">Spectral Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=temporal-wavelet-analysis" rel="tag">Temporal Wavelet Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=three-dimensional-display" rel="tag">Three Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=two-dimensional-display" rel="tag">Two Dimensional Display</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> EEGLAB is to date the most popular EEG/MEG/ECoG software with about 100,000 download worldwide since 2003. EEGLAB provides an interactive graphic user interface (GUI) allowing users to flexibly and interactively process their high-density EEG and other dynamic brain data using independent component analysis (ICA) and/or spectral time/frequency and coherence analysis, as well as standard methods including event-related potentials (ERP). EEGLAB also incorporates extensive tutorial and help windows, plus a command history function that eases users' transition from GUI-based data exploration to building and running batch or custom data analysis scripts. EEGLAB offers a wealth of methods for visualizing and modeling event-related brain dynamics, both at the level of individual EEGLAB 'datasets' and/or across a collection of datasets. For experienced Matlab users, EEGLAB offers a structured programming environment for storing, accessing, measuring, manipulating and visualizing event-related EEG data. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=536&screenshot_id=465"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=536&screenshot_id=465" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 4, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://sccn.ucsd.edu/eeglab/index.php#">https://sccn.ucsd.edu/eeglab/index.php#</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UCSD <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS<br>Microsoft<br>OS Independent<br>POSIX/UNIX-like</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANT, ASCII, Brain Vision Analyzer Data Exchange Format, CNT, EDF/BDF/GDF, EGI</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANT, ASCII, Brain Vision Analyzer Data Exchange Format, CNT, EDF/BDF/GDF, EGI</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="eeg-study-data-of-rapid-serial-visual-presentation-(rsvp)-in-ess-format" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="EEG Study data of Rapid Serial Visual Presentation (RSVP) in ESS format">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=eeg-study-data-of-rapid-serial-visual-presentation-(rsvp)-in-ess-format" rel="bookmark" name="eeg-study-data-of-rapid-serial-visual-presentation-(rsvp)-in-ess-format">EEG Study data of Rapid Serial Visual Presentation (RSVP) in ESS format</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=ontology" rel="tag">Ontology</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Rapid Serial Visual Presentation (12Hz) Target (airplane) detection without immediate response.<br />
<br />
The purpose of this study was to explore the neural basis of target detection in human brain and to compare the performance of brain-computer interface (BCI) methods in classification of target vs. non-target images. The data was acquired during a rapid serial visual presentation task which was composed of a sequential presentation of image clips in rapid succession (12/s) in 4.1-s bursts, to which subjects were to indicate whether or not the satellite image clips of London presented included a small target airplane image by making one of two button presses.<br />
<br />
This study was collected at UCSD/Swartz Center and placed into ESS version 2.02, levels 1&amp;2 by Syntrogi Inc. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=957&screenshot_id=1133"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=957&screenshot_id=1133" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 21, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/eeg_rsvp/">http://www.nitrc.org/projects/eeg_rsvp/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Syntrogi <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="eeg-study-schema-(ess)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="EEG Study Schema (ESS)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=eeg-study-schema-(ess)" rel="bookmark" name="eeg-study-schema-(ess)">EEG Study Schema (ESS)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=ontology" rel="tag">Ontology</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> EEG Study Schema (ESS) makes it easier for researchers in the field of EEG/BCI to package, share and automatize the analysis workflow of their study data. You can think of ESS as a &quot;shipping container&quot; for your EEG study data.<br />
<br />
Using other people’s EEG data could be painful as often one has to do detective work to find out:<br />
<br />
What happened in the experiment?<br />
Which files are for which subjects/sessions?<br />
What do these event codes mean?<br />
To remove ‘EEG data sharing pain' we have created a set of standards (HED and ESS) and tools<br />
<br />
ESS is designed from a user-centered viewpoint that emphasizes simplicity and ease of use. It is created to contain all the information a researcher unfamiliar with a particular EEG (or MEG) Study needs to further analyze the data. It is An XML-based specification<br />
Holds all the information necessary to analyze an EEG study.<br />
<br />
Please visit <a href="http://www.eegstudy.org" target="_new">http://www.eegstudy.org</a> for more information. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=956&screenshot_id=943"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=956&screenshot_id=943" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 21, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MIT</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/ess/">http://www.nitrc.org/projects/ess/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Syntrogi <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="eeg-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="EEG Toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=eeg-toolbox" rel="bookmark" name="eeg-toolbox">EEG Toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=event-related-potential" rel="tag">Event Related Potential</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This toolbox has been developed to facilitate quick and easy import, visualisation and measurement for ERP data. The toolbox can open and visualise ERP averaged data (Neuroscan, ascii formats), 2D/3D electrode coordinates and 3D cerebral tissue tesselations (meshes). All the features can be explored quickly and easily using the example data provided in the toolbox. The GUI interface is simple and intuitive.<br />
<br />
For more details, see: <a href="http://eeg.sourceforge.net/" target="_new">http://eeg.sourceforge.net/</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 30, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/eeg/">http://www.nitrc.org/projects/eeg/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="eegvis" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="EEGVIS">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=eegvis" rel="bookmark" name="eegvis">EEGVIS</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=statistical-operation" rel="tag">Statistical Operation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> EEGVIS is a MATLAB toolbox for exploration of multi-channel EEG and other large array-based data sets using multi-scale drill-down techniques. The toolbox can be used directly in MATLAB at any stage in a user's processing pipeline, as a plug in for EEGLAB, or as a standalone precompiled application without MATLAB running. EEGVIS and its supporting packages are freely available under the GNU general public license. The toolbox also supplies a number of extensible base classes for users who wish to develop their own visualizations. Additional information is provided at <a href="http://visual.cs.utsa.edu/eegvis" target="_new">http://visual.cs.utsa.edu/eegvis</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 11, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://visual.cs.utsa.edu/eegvis">http://visual.cs.utsa.edu/eegvis</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Texas at San Antonio <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="eeprobe" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="EEProbe">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=eeprobe" rel="bookmark" name="eeprobe">EEProbe</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=event-related-potential" rel="tag">Event Related Potential</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> EEProbe is a complete software package for the study of event-related brain activity with high-resolution EEG/MEG. This package has been designed to suit the high standards of neuroscience research. The software has been developed originally at the Max Planck Institute for Cognitive Neuroscience in Leipzig, Germany, and is available for other institutions through ANT Neuro B.V., The Netherlands, enhanced with the EEProbe Databrowser.<br />
<br />
ERP investigations, both in psychophysiology research and clinical applications require a multitude of processing steps. Analysis of large data sets is made efficient through advanced scripting possibilities. <br />
<br />
All different aspects of data handling are efficiently available in the EEProbe Databrowser. Alternatively, external data can be imported from a multitude of formats.<br />
<br />
Processing in EEProbe makes use of open file formats (see LIBEEP) and is designed to integrate with ASA for advanced source analysis.<br />
<br />
EEProbe is available for Linux and Mac OS X. <a href="http://www.ant-neuro.com" target="_new">http://www.ant-neuro.com</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=555&screenshot_id=632"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=555&screenshot_id=632" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 23, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Commercial License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/eeprobe/">http://www.nitrc.org/projects/eeprobe/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>ANT Neuro B.V. <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="efficient-longitudinal-upload-of-depression-in-the-elderly-(elude)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Efficient Longitudinal Upload of Depression in the Elderly (ELUDE)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=efficient-longitudinal-upload-of-depression-in-the-elderly-(elude)" rel="bookmark" name="efficient-longitudinal-upload-of-depression-in-the-elderly-(elude)">Efficient Longitudinal Upload of Depression in the Elderly (ELUDE)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The ELUDE dataset is a longitudinal study of late-life depression at Duke University. There are 281 depressed subjects and 154 controls included. An MR scan of each subject was obtained every 2 years for up to 8 years (total of 1093 scans). Clinical assessments occurred more frequently and consists of a battery of psychiatric tests including several depression-specific tests. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 27, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BIRN Data License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/elude/">http://www.nitrc.org/projects/elude/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Duke University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="efficient-permutation-testing" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Efficient Permutation Testing">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=efficient-permutation-testing" rel="bookmark" name="efficient-permutation-testing">Efficient Permutation Testing</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Matlab implementation for efficient permutation testing (<a href="http://pages.cs.wisc.edu/~vamsi/pt_fast.html" target="_new">http://pages.cs.wisc.edu/~vamsi/pt_fast....</a>).<br />
<br />
Permutation testing is a non parametric procedure to estimate the distribution of max null statistic. However it computationally very expensive, particularly in neuroimaging and biostatistics studies where dimensionality of data is on the order of millions. We proposed an efficient procedure to estimate this max null using matrix completion. A speed up of atleast 50 times is achieved while estimating the max null upto a high degree of accuracy. For complete details please refer to the project website (link above).<br />
<br />
Please cite the following paper upon usage of the tool. <br />
C. Hinrichs, V. K. Ithapu, Q. Sun, V. Singh, S. C. Johnson, Speeding up Permutation Testing in Neuroimaging, Neural Information Processing Systems (NIPS), 2013 </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 16, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://pages.cs.wisc.edu/~vamsi/pt_fast.html">http://pages.cs.wisc.edu/~vamsi/pt_fast.html</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Wisconsin Madison <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="elastic-net-based-parcellation-(enpa)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Elastic Net-based Parcellation (ENPA)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=elastic-net-based-parcellation-(enpa)" rel="bookmark" name="elastic-net-based-parcellation-(enpa)">Elastic Net-based Parcellation (ENPA)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Parcellating brain regions into functionally homogeneous subdivisions is critical for understanding normal and abnormal brain functions. Hence, functional parcellation techniques have recently gained further research momentum. <br />
The toolbox is for parcellation of brain regions with fMRI data using regularized sparse representation method, and consists of parcellation modules at group and individual levels. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1062&screenshot_id=1006"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1062&screenshot_id=1006" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 2, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Affero General Public License v3</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/enpa_ninet/">http://www.nitrc.org/projects/enpa_ninet/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Non-Invasive Neurostimulation Therapies Laboratory <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="elastix" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="elastix">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=elastix" rel="bookmark" name="elastix">elastix</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=registration" rel="tag">Registration</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=resampling" rel="tag">Resampling</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Welcome to elastix: a toolbox for rigid and nonrigid registration of images.<br />
<br />
elastix is open source software, based on the well-known Insight Segmentation and Registration Toolkit (ITK). The software consists of a collection of algorithms that are commonly used to solve (medical) image registration problems. The modular design of elastix allows the user to quickly configure, test, and compare different registration methods for a specific application. A command-line interface enables automated processing of large numbers of data sets, by means of scripting.<br />
<br />
The website is found at <a href="http://elastix.isi.uu.nl/." target="_new">http://elastix.isi.uu.nl/.</a><br />
<br />
A paper describing elastix contains more details: S. Klein, M. Staring, K. Murphy, M.A. Viergever, J.P.W. Pluim, &quot;elastix: a toolbox for intensity based medical image registration,&quot; IEEE Transactions on Medical Imaging, vol. 29, no. 1, pp. 196 - 205, January 2010. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=671&screenshot_id=592"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=671&screenshot_id=592" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 19, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/elastix/">http://www.nitrc.org/projects/elastix/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1, Nrrd, Other Format, Philips PAR/REC</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1, Nrrd, Other Format, Philips PAR/REC</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="electrophysiology-analysis-toolkit-(elephant)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Electrophysiology Analysis Toolkit (Elephant)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=electrophysiology-analysis-toolkit-(elephant)" rel="bookmark" name="electrophysiology-analysis-toolkit-(elephant)">Electrophysiology Analysis Toolkit (Elephant)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Electrophysiology Analysis Toolkit (Elephant) is an open-source library for the analysis of electrophysiological data in the Python programming language. The focus of Elephant is on generic analysis functions for spike train data and time series recordings from electrodes, such as local field potentials (LFP) or intracellular voltages. In addition to providing a common platform for analysis code from different laboratories, the Elephant project aims to provide a consistent and homogeneous analysis framework built on a modular foundation. Elephant is built on the Neo data object model for electrophysiological data. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1237&screenshot_id=1152"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1237&screenshot_id=1152" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 21, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.python-elephant.org/">http://www.python-elephant.org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>NeuralEnsemble community <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="emory-unc-developmental-macaque-structural-and-dti-atlases" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Emory-UNC Developmental Macaque Structural and DTI Atlases">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=emory-unc-developmental-macaque-structural-and-dti-atlases" rel="bookmark" name="emory-unc-developmental-macaque-structural-and-dti-atlases">Emory-UNC Developmental Macaque Structural and DTI Atlases</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This is a set of cross-sectional structural atlases as well as cross-sectional and longitudinal DTI atlases of a cohort of 40 rhesus macaques scanned longitudinally at ages 2 weeks, 3 months, 6 months, 12 months, and 18 months. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 17, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Share Alike</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/macaque_atlas/">http://www.nitrc.org/projects/macaque_atlas/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina at Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="emse-suite" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="EMSE Suite">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=emse-suite" rel="bookmark" name="emse-suite">EMSE Suite</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=eeg-modeling" rel="tag">Eeg Modeling</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> EMSE Suite is a professionally supported modular platform for space-time-frequency analyses of EEG/MEG/ECoG integrated (optionally) with structural MRI and functional hemodynamic measures (fMRI and NIRS). The Locator module digitizes 3D sensor coordinates. Data Editor provides live and offline pipelines of spatial and temporal filters, and easy-to-use event pipelines for conditional binning of time, frequency, and time-frequency data across participants, with group results. Coherence, phase synchronication, and quasi-causal information assess connectivity. Source Estimator enables modeling of discrete overdetermined and distributed underdetermined sources, and spatial filtering for 3D brain regions of interest. Statistical nonparametric mapping (SnPM) may be performed for all measures. MR Viewer and Image Processor comprise tools for BEM and FEM volume conductor models, using cortical source space models. See <a href="http://www.cortechsolutions.com/EMSE/" target="_new">http://www.cortechsolutions.com/EMSE/</a> for details and a supported free trial. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=657&screenshot_id=576"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=657&screenshot_id=576" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 20, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Commercial License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.cortechsolutions.com/EMSE/">http://www.cortechsolutions.com/EMSE/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Cortech Solutions, Inc. <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="enigma-dti-pipeline" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ENIGMA-DTI pipeline">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=enigma-dti-pipeline" rel="bookmark" name="enigma-dti-pipeline">ENIGMA-DTI pipeline</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The ENIGMA DTI pipeline provides tools to extract whole-brain average and regional measurements from DTI images including FA, AD, RD and MD. The pipeline has been used successfully across sites worldwide and shown to extract reliable and heritable measures from diffusion imaging. We provide a FA-template created from multiple high resolution datasets to better accommodate registrations to a standard atlas than current existing tools. The ENIGMA-DTI analysis team is continuously working to bridge together pipelines to analyze DTI in reliable ways across multiple datasets and reduce inter-site inhomogeneities and inconsistencies between findings, so stay tuned for more updates and tools. Successful global studies of genetic or disease effects underlying variability in white matter neuroanatomy have been published through ENIGMA and many more are underway. To get involved in any ENIGMA group send us a message -- <a href="mailto:enigma@ini.usc.edu" target="_new">enigma@ini.usc.edu</a> or support.enigmaDTI@ini.usc.edu. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=981&screenshot_id=948"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=981&screenshot_id=948" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 12, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Public Domain</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/enigma_dti/">http://www.nitrc.org/projects/enigma_dti/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>ENIGMA <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="enigma:-enhancing-neuro-imaging-genetics-through-meta-analysis" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ENIGMA: Enhancing Neuro Imaging Genetics through Meta-Analysis">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=enigma:-enhancing-neuro-imaging-genetics-through-meta-analysis" rel="bookmark" name="enigma:-enhancing-neuro-imaging-genetics-through-meta-analysis">ENIGMA: Enhancing Neuro Imaging Genetics through Meta-Analysis</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=genetic-association" rel="tag">Genetic Association</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=imputation" rel="tag">Imputation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The ENIGMA Network brings together researchers in imaging genomics, to understand brain structure and function, based on MRI, DTI, fMRI and genomewide association scan (GWAS) data. The ENIGMA Network has several goals:<br />
<br />
 * To create a network of like-minded individuals, interested in pushing forward the field of imaging genetics.<br />
 * To ensure promising findings are replicated via member collaborations, in order to satisfy the mandates of most journals.<br />
 * To share ideas, algorithms, data, and information on research studies and methods.<br />
 * To facilitate training, including workshops and conferences on key methods and emerging directions in imaging genetics. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=693&screenshot_id=628"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=693&screenshot_id=628" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 4, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://enigma.ini.usc.edu/">http://enigma.ini.usc.edu/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>USC <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="enigma-resting-state-fmri-processing-pipeline" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ENIGMA resting state fMRI processing pipeline">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=enigma-resting-state-fmri-processing-pipeline" rel="bookmark" name="enigma-resting-state-fmri-processing-pipeline">ENIGMA resting state fMRI processing pipeline</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The ENIGMA rsFMRI pipeline provides tools for pre-processing of fMRI data and measurements of connectivity for stable functional networks, including default mode network. This is yet experimental pipeline that requires only EPI fMRI data and uses a population based template and tissue priors for regression of global connectivity signal. The pipeline is based on AFNI software and includes PCA-based denoising and group-based regularization approach for imputation of censored frames and removal of outliers. The pipeline can also be used for pre-processing of the task-based functional MRI studies in ENIGMA workgroups.The pipeline is distributed as a set of standard unix scripts and software and as a virtual machine's container for unix, mac and windows platforms. <br />
To get involved in any ENIGMA group send us a message -- <a href="mailto:enigma@ini.usc.edu" target="_new">enigma@ini.usc.edu</a> or support.enigmaDTI@ini.usc.edu. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 26, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/enigma-fmri/">http://www.nitrc.org/projects/enigma-fmri/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>ENIGMA center for imaging genetics <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="enigma-viewer:-visualizing-effect-sizes-in-meta-analysis" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ENIGMA-Viewer: Visualizing Effect Sizes in Meta-Analysis">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=enigma-viewer:-visualizing-effect-sizes-in-meta-analysis" rel="bookmark" name="enigma-viewer:-visualizing-effect-sizes-in-meta-analysis">ENIGMA-Viewer: Visualizing Effect Sizes in Meta-Analysis</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-rendering" rel="tag">Surface Rendering</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ENIGMA-Viewer is an interactive visualization tool for neuroscientists to examine DTI data and compare effective sizes and associated genetics information in meta-analysis. The program supports the visualization of sub-cortical and cortical regions. The user may upload your own data for comparison. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1048&screenshot_id=999"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1048&screenshot_id=999" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 18, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MIT</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://enigma-viewer.org">http://enigma-viewer.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UMBC; UMB; USC <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="enumerated-auditory-response-(ear)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Enumerated Auditory Response (EAR)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=enumerated-auditory-response-(ear)" rel="bookmark" name="enumerated-auditory-response-(ear)">Enumerated Auditory Response (EAR)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> EAR is an automated auditory fMRI post processing method that provides consistent radiological interpretations for evaluating auditory response in TBI, carbon monoxide poisoning, and other neurological disease processes that affect auditory response. The tool was developed by an expert in auditory fMRI interpretation. Accuracy has been demonstrated with very high correlation to expert reads and other quantitative methods. A similar approach for fMRI analysis might be appropriate for both future research projects with constrained resources, as well as possible routine clinical use. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 22</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/audquant/">http://www.nitrc.org/projects/audquant/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Imgen <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="eons" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="EONS">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=eons" rel="bookmark" name="eons">EONS</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> EONS (Elementary objects of the Nervous System) is the perfect modeling platform to study the dynamic interactions between synaptic elements in a friendly manner. There are no complicated equations to write: all the elementary models are predefined using state of the art models! <br />
EONS consists of a graphical modeling platform containing the major elements that comprise a glutamatergic synapse (both pre- and post-synaptically). These elements parameters as well as the underlying synaptic geometry can be modified. <br />
<br />
EONS is a parametric model of a generic glutamatergic synapse that takes into account pre-synaptic mechanisms, such as calcium buffering and diffusion, neurotransmitter release, diffusion and uptake in the cleft, and postsynaptic elements, such as ionotropic AMPA and NMDA receptors, their distribution and synaptic geometry, as well as metabotropic glutamate receptors. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 23, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://synapticmodeling.com">http://synapticmodeling.com</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>USC <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS<br>Other OS<br>Windows XP</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="epi-example-dicom" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="epi-example-dicom">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=epi-example-dicom" rel="bookmark" name="epi-example-dicom">epi-example-dicom</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Contains an example echoplanar session, in Dicom format, and a python script (extract_timings.py) to extract the times of slice acquisition. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 27, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Academic Free License ("AFL")</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/epi-dicom/">http://www.nitrc.org/projects/epi-dicom/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>CEA <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="epilab" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="EPILAB">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=epilab" rel="bookmark" name="epilab">EPILAB</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> A Matlab®-based software package, EPILAB, was developed for supporting researchers in performing studies on the prediction of epileptic seizures. It provides an intuitive and convenient graphical user interface.<br />
Fundamental concepts that are crucial for epileptic seizure prediction studies were implemented.This includes, for example, the development and statistical validation of prediction methodologies in long-term continuous recordings. Seizure prediction is usually based on electroencephalography (EEG) and electrocardiography (ECG) signals. EPILAB is able to process both EEG and ECG data stored in different formats. More than 35 time and frequency domain measures (features) can be extracted based on univariate and multivariate data analysis. These features can be post-processed and used for prediction purposes. The predictions may be conducted based on optimized thresholds or by applying classifications methods such as artificial neural networks, cellular neuronal networks, and support vector machines. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=737&screenshot_id=714"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=737&screenshot_id=714" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 28, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.epilepsiae.eu/project_outputs/epilab_software">http://www.epilepsiae.eu/project_outputs/epilab_software</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Epilepsiae FP7 Consortium <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="epilepcy-neuroimaging-data" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Epilepcy Neuroimaging Data">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=epilepcy-neuroimaging-data" rel="bookmark" name="epilepcy-neuroimaging-data">Epilepcy Neuroimaging Data</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The shared image modalities and their relevant clinical information are for 12 TLE patients.<br />
<br />
Please cite the following article if you use the uncertainty Analysis MATLAB code and its relevant image modalities (T1, FLAIR, and DTI):<br />
<br />
Mohammad-Reza Nazem-Zadeh, Kost V Elisevich, Jason M Schwalb, Hassan Bagher-Ebadian, Fariborz Mahmoudi, Hamid Soltanian-Zadeh. Lateralization of Temporal Lobe Epilepsy by Multimodal Multinomial Hippocampal Response-Driven Models. Journal of the neurological sciences, 347 (1), 107-118, 2014. <br />
<br />
<br />
Please cite the following article if you use T1, FLAIR, and SPECT modalities:<br />
<br />
Mohammad-Reza Nazem-Zadeh, Jason M Schwalb, Kost V Elisevich, Hassan Bagher-Ebadian, Hajar Hamidian, Ali-Reza Akhondi-Asl, Kourosh Jafari-Khouzani, Hamid Soltanian-Zadeh. Lateralization of temporal lobe epilepsy using a novel uncertainty analysis of MR diffusion in hippocampus, cingulum, and fornix, and hippocampal volume and FLAIR intensity. Journal of the neurological sciences, 342 (1), 152-161, 2014. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 18, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Epilepsy Research - Henry Ford Hospital License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/hfhs-tle-proj/">http://www.nitrc.org/projects/hfhs-tle-proj/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Henry Ford Hospital <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="e-prime" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="E-Prime">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=e-prime" rel="bookmark" name="e-prime">E-Prime</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=experimental-control" rel="tag">Experimental Control</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> E-Prime® is a suite of applications to fulfill all of your computerized experiment needs. With more than 100,000 users in research institutions and laboratories in over 60 countries, E-Prime® provides a truly easy-to-use environment for computerized experiment design, data collection, and analysis. E-Prime® provides millisecond precision timing to ensure the accuracy of your data. E-Prime’s flexibility to create simple to complex experiments is ideal for both novice and advanced users. <br />
<br />
The E-Prime® suite of applications includes:<br />
•E-Studio – Drag and drop graphical interface for experiment design<br />
•E-Basic – Underlying scripting language of E-Prime<br />
•E-Run – Once the experiment is generated with a single click, E-Run affords you the millisecond precision of stimulus presentation, synchronizations, and data collection.<br />
•E-Merge – Merges your single session data files for group analysis<br />
•E-DataAid – Data management utility<br />
•E-Recovery – Recovers data files 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=219&screenshot_id=1045"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=219&screenshot_id=1045" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 23, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>E-Prime License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.pstnet.com/eprime.cfm">http://www.pstnet.com/eprime.cfm</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Windows Vista<br>Windows XP</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="e-prime-extensions-for-fmri" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="E-Prime Extensions for fMRI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=e-prime-extensions-for-fmri" rel="bookmark" name="e-prime-extensions-for-fmri">E-Prime Extensions for fMRI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=experimental-control" rel="tag">Experimental Control</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> E-Prime® Extensions for fMRI (EEfMRI) software is designed to optimize E-Prime® experiments for fMRI research. EEfMRI allows you to synchronize the start of your experiment with the first scanner trigger pulse along with several valuable features to enhance the control you have over your experiment. Implementing EEfMRI into your current experiments is achieved by simply dragging and dropping the correct EEfMRI package calls into the E-Prime® experiment in the appropriate places. EEfMRI is designed to integrate with other PST hardware and software to increase usability for researchers while maintaining the millisecond accuracy of E-Prime®. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=245&screenshot_id=1090"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=245&screenshot_id=1090" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 7, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>E-Prime Extensions for fMRI</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://pstnet.com/products/e-prime-extensions-for-fmri/">https://pstnet.com/products/e-prime-extensions-for-fmri/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Windows Vista<br>Windows XP</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="e-prime-extensions-for-smi" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="E-Prime Extensions for SMI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=e-prime-extensions-for-smi" rel="bookmark" name="e-prime-extensions-for-smi">E-Prime Extensions for SMI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> E-Prime® Extensions for SMI adds eye gaze and eye movement capabilities to existing and novel E-Prime experiments. Create new paradigms with E-Studio's graphical design interface and connect with flexible Remote Eye Tracking technologies by SensoMotoric Instruments.<br />
 <br />
Create interactive or passive eye tracking paradigms<br />
Create paradigms which change based on participant eye gaze data, AOI hit tests, fixation and user interaction<br />
Combine eye gaze data with E-Prime condition data for powerful analysis<br />
Train participants to fixate and control eye movements<br />
Give feedback on vigilance or attentive behaviors<br />
Supports diagnostic and interactive studies:<br />
Face perception and recognition<br />
Infant studies<br />
Reading (sliding window)<br />
Attention patterns and behavior<br />
Visual search<br />
Scene recognition 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1174&screenshot_id=1086"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1174&screenshot_id=1086" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 23, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://pstnet.com/products/e-prime-extensions-for-smi/">https://pstnet.com/products/e-prime-extensions-for-smi/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Psychology Software Tools, Inc. <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="e-prime-extensions-for-tobii" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="E-Prime Extensions for Tobii">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=e-prime-extensions-for-tobii" rel="bookmark" name="e-prime-extensions-for-tobii">E-Prime Extensions for Tobii</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> E-Prime® Extensions for Tobii combines the power of E-Prime with easy to use Tobii eye trackers. The E-Prime® Extensions for Tobii includes the script necessary to integrate E-Prime with Tobii eye tracking technology. E-Studio’s graphical design interface allows users to drag and drop eye tracking functionality into existing E-Prime® experiments or to easily create new E-Prime® eye tracking experiments.<br />
Supports multiple types of studies: Diagnostic and Interactive<br />
Face perception and recognition<br />
Infant studies<br />
Reading (sliding window)<br />
Attention patterns and behavior<br />
Visual search<br />
Scene recognition<br />
No head restraint necessary<br />
Seamless E-Prime® integration 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1173&screenshot_id=1088"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1173&screenshot_id=1088" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 23, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://pstnet.com/products/e-prime-extensions-for-tobii/">https://pstnet.com/products/e-prime-extensions-for-tobii/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Psychology Software Tools, Inc. <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="epsilon-radial-networks" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Epsilon Radial Networks">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=epsilon-radial-networks" rel="bookmark" name="epsilon-radial-networks">Epsilon Radial Networks</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Currently there is no agreed-upon method for constructing the brain anatomical connectivity graphs out of large number of white matter tracts. In this paper, we present an efficient framework for building and analyzing graphs called epsilon radial networks (ERNs) using tractography data in a normalized space.<br />
<br />
The key challenge in defining brain networks is node delineation and our method defines nodes in the graph using tract-end points clustered in a sphere of a given radius (epsilon). Using a kd-tree based search algorithm we can identify the nodes computationally efficiently and in a fully automatic way.<br />
<br />
These networks can be used not only to analyze topo-physical properties of the structural brain networks but also to perform classical region-of-interest (ROI) analyses in a very efficient way. Thus ERNs can be used as a novel image processing lens for statistical and machine learning based analyses. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 17, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>DTI-TK license</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/ern/">http://www.nitrc.org/projects/ern/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Wisconsin-Madison <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="erplab" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ERPLAB">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=erplab" rel="bookmark" name="erplab">ERPLAB</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=anova" rel="tag">Anova</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=event-related-potential" rel="tag">Event Related Potential</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ERPLAB Toolbox is a set of open source, freely available Matlab routines for analyzing ERP data. It is tightly integrated with the EEGLAB Toolbox. ERPLAB routines can be accessed from the Matlab command window and from Matlab scripts in addition to being accessed from the EEGLAB GUI. Consequently, ERPLAB provides the ease of learning of a GUI-based system but also provides the power and flexibility of a scripted system.The development of ERPLAB Toolbox is being coordinated by Steve Luck and Javier Lopez-Calderon at the UC-Davis Center for Mind &amp; Brain, with financial support from NIMH. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=541&screenshot_id=631"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=541&screenshot_id=631" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 2, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/erplab/">http://www.nitrc.org/projects/erplab/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of California, Davis <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="erp-pca-toolkit" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ERP PCA Toolkit">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=erp-pca-toolkit" rel="bookmark" name="erp-pca-toolkit">ERP PCA Toolkit</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=event-related-potential" rel="tag">Event Related Potential</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=principal-component-analysis" rel="tag">Principal Component Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This Matlab toolkit is a general purpose tool for editing, visualizing, and analyzing EEG data (both ERP and spectral) whose most recent version has been downloaded over 1000 times. Its three chief highlights are: 1) an optimized automatic artifact correction function that includes ICA correction for eye blinks and saccades. 2) Extensive support for easily conducting PCA and ICA through all stages of the procedure, including inspection of reconstituted waveforms and batch ANOVAs. 3) Implementation of robust ANOVAs, including McCarthy-Wood vector test. It has a graphical user interface for point and click usage and comes with an extensive illustrated tutorial. A description of the toolkit was published in Dien (2010) in Journal of Neuroscience Methods. It relies on both internal functions as well as borrowed functions from both EEGlab and FieldTrip. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=616&screenshot_id=908"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=616&screenshot_id=908" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 8, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://sourceforge.net/projects/erppcatoolkit/files/">http://sourceforge.net/projects/erppcatoolkit/files/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Maryland <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="erpwavelab" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ERPwavelab">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=erpwavelab" rel="bookmark" name="erpwavelab">ERPwavelab</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=event-related-potential" rel="tag">Event Related Potential</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=fourier-time-domain-analysis" rel="tag">Fourier Time-Domain Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=temporal-wavelet-analysis" rel="tag">Temporal Wavelet Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The open source toolbox 'ERPWAVELAB' is developed for multi-channel time- frequency analysis of event related activity of EEG and MEG data. The toolbox provides tools for data analysis and visualization of the most commonly used mea- sures of time-frequency transformed event related data as well as data decomposition through non-negative matrix and multi-way (tensor) factorization. The decomposi- tions provided can accommodate additional dimensions like subjects, conditions or repeats and as such they are perfected for group analysis. Furthermore, the toolbox enables tracking of phase locked activity from one channel-time-frequency instance to another as well as tools for artifact rejection in the time-frequency domain. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=878&screenshot_id=848"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=878&screenshot_id=848" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 23, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.erpwavelab.org/">http://www.erpwavelab.org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Technical University of Denmark <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="estimating-functional-brain-network-based-on-modularity-prior" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Estimating Functional Brain Network based on Modularity Prior">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=estimating-functional-brain-network-based-on-modularity-prior" rel="bookmark" name="estimating-functional-brain-network-based-on-modularity-prior">Estimating Functional Brain Network based on Modularity Prior</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This is the source code of a new method for estimating functional brain networks. The method is based on sparse and low-rank representation for incorporating modularity prior into brain network construction. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 12, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/modularbrain/">http://www.nitrc.org/projects/modularbrain/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UNC@Chapel hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="european-epilepsy-database-epilepsiae" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="European Epilepsy Database EPILEPSIAE">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=european-epilepsy-database-epilepsiae" rel="bookmark" name="european-epilepsy-database-epilepsiae">European Epilepsy Database EPILEPSIAE</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Many technological applications in the field of neurology and neuroscience depend on evaluations based on electroencephalographic data (EEG). So far, public resources for EEG recordings are limited. We present here the European database developed in the EU-founded project “EPILEPSIAE” (Grant 211713).<br />
<br />
The EPILEPSIE database is by far the largest and most comprehensive database for human surface and intracranial eeg data. It is suitable for a broad range of applications e.g. of time series analyses of brain activity. The database is accessible for users who apply to the research groups in charge (<a href="http://epilepsy-database.eu" target="_new">http://epilepsy-database.eu</a>).<br />
<br />
Currently, the EU database contains annotated EEG datasets from more than 200 patients with epilepsy, 50 of them with intracranial recordings with up to 122 channels. Each dataset provides EEG data for a continuous recording time of at least 96 hours (4 days) at a sample rate of up to 2500 Hz. Clinical patient information and MR imaging data supplement the EEG data. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=790&screenshot_id=719"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=790&screenshot_id=719" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 21, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Comercial License: EPILEPSIAE Database</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://epilepsy-database.eu/">http://epilepsy-database.eu/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University Medical Center Freiburg <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="evaluation-of-automated-brain-extraction-methods-on-multi-site-multi-disorder-data" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Evaluation of automated brain extraction methods on multi-site multi-disorder data">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=evaluation-of-automated-brain-extraction-methods-on-multi-site-multi-disorder-data" rel="bookmark" name="evaluation-of-automated-brain-extraction-methods-on-multi-site-multi-disorder-data">Evaluation of automated brain extraction methods on multi-site multi-disorder data</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=information-resource" rel="tag">Information Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This project was the largest evaluation ever conducted of automated brain extraction methods. The code, data, and results are accessible via Synapse: <a href="https://www.synapse.org/#!Synapse:syn3207674/wiki/160612" target="_new">https://www.synapse.org/#!Synapse:syn320...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 2, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/brain_extracts/">http://www.nitrc.org/projects/brain_extracts/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="ew_dmgwas:-edge-weighted-dense-module-search-for-genome-wide-association-studies-and-gene-expression-profiles" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="EW_dmGWAS: Edge-weighted dense module search for genome-wide association studies and gene expression profiles">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=ew_dmgwas:-edge-weighted-dense-module-search-for-genome-wide-association-studies-and-gene-expression-profiles" rel="bookmark" name="ew_dmgwas:-edge-weighted-dense-module-search-for-genome-wide-association-studies-and-gene-expression-profiles">EW_dmGWAS: Edge-weighted dense module search for genome-wide association studies and gene expression profiles</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> dmGWAS is designed to identify significant protein-protein interaction (PPI) modules and, from which, the candidate genes for complex diseases by an integrative analysis of GWAS dataset(s) and PPI network. The new algorithm, EW_dmGWAS, combines node weight, which are computed based on GWAS signals, and edge weight, which are computed based on gene expression data. Allow tissue-specific gene expression profile. For example, for breast cancer GWAS analysis, gene expression data from breast cancer patients can be used; for schizophrenia, gene expression data in brain tissues can be used.<br />
All old versions are here: <a href="https://bioinfo.uth.edu/dmGWAS/dmGWAS_old.html." target="_new">https://bioinfo.uth.edu/dmGWAS/dmGWAS_ol...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 24, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dmgwas/">http://www.nitrc.org/projects/dmgwas/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Texas Health Science Center at Houston <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="exposition-packages" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ExPosition Packages">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=exposition-packages" rel="bookmark" name="exposition-packages">ExPosition Packages</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=discriminant-analysis" rel="tag">Discriminant Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-omics" rel="tag">Other Omics</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=principal-component-analysis" rel="tag">Principal Component Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Exposition is defined as a comprehensive explanation of an idea. In the world of high dimensional data analysis and multivariate analysis one technique stands out because of its versatility and ubiquity: The singular<br />
value decomposition (SVD). With the ExPosition packages for R, a comprehensive explanation of your data will be provided with minimal effort. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 4, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/exposition/">http://www.nitrc.org/projects/exposition/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="eyeballgui" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="EyeBallGUI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=eyeballgui" rel="bookmark" name="eyeballgui">EyeBallGUI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> EyeBallGUI is a tool for interactively viewing &amp; marking multi-channel biosignals<br />
<br />
Please cite using:<br />
<a href="https://doi.org/10.1101/129437" target="_new">https://doi.org/10.1101/129437</a><br />
<br />
[1] K. Mohr, B. Nasseroleslami, P. M. Iyer, O. Hardiman, and E. C. Lalor, ‘EyeBallGUI: A Tool For Visual Inspection And Binary Marking of Multi-Channel Bio-Signals’, bioRxiv, p. 129437, May 2017. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=992&screenshot_id=1115"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=992&screenshot_id=1115" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 5, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://sourceforge.net/projects/eyeballgui/">https://sourceforge.net/projects/eyeballgui/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Trinity College Dublin, the University of Dubin <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="eye-eeg-(combined-eye-tracking-&-eeg)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="EYE-EEG (combined eye-tracking & EEG)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=eye-eeg-(combined-eye-tracking-&-eeg)" rel="bookmark" name="eye-eeg-(combined-eye-tracking-&-eeg)">EYE-EEG (combined eye-tracking & EEG)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=software" rel="tag">Software</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> EYE-EEG is a plugin for the open-source MATLAB toolbox EEGLAB developed with the goal to facilitate integrated analyses of electrophysiological and oculomotor data. The plugin parses, imports, and synchronizes simultaneously recorded eye tracking data and adds it as extra channels to the EEG.<br />
<br />
Saccades and fixations can be imported from the eye tracking raw data or detected with an adaptive velocity-based algorithm. Eye movements are then added as new time-locking events to EEGLAB's event structure, allowing easy saccade- and fixation-related EEG analysis (e.g., fixation-related potentials, FRPs). Alternatively, EEG data can be aligned to stimulus onsets and analyzed according to oculomotor behavior (e.g. pupil size, microsaccades) in a given trial. Saccade-related ICA components can be objectively identified based on their covariance with the electrically independent eye tracker.<br />
<br />
All functions can be accessed via EEGLAB's GUI or called from the command line. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=744&screenshot_id=674"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=744&screenshot_id=674" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 18, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www2.hu-berlin.de/eyetracking-eeg">http://www2.hu-berlin.de/eyetracking-eeg</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Humboldt University at Berlin <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="faceted-search-based-ontology-visualizer" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Faceted Search Based Ontology Visualizer">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=faceted-search-based-ontology-visualizer" rel="bookmark" name="faceted-search-based-ontology-visualizer">Faceted Search Based Ontology Visualizer</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Allows user to do faceted search on an ontology and enables visualization of the search results on the 3D digital atlas. Currently supports faceted search of functional neuroanatomy. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 25, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>3D Slicer License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/ontologyviz/">http://www.nitrc.org/projects/ontologyviz/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="factory_t1_dti" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="factory_t1_dti">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=factory_t1_dti" rel="bookmark" name="factory_t1_dti">factory_t1_dti</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Tools to make easier on using spm, pipedream, dti-tk, and other softwares to analyze t1 or dti images. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 7, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/factory4t1ndti/">http://www.nitrc.org/projects/factory4t1ndti/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="fadttster" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="FADTTSter">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=fadttster" rel="bookmark" name="fadttster">FADTTSter</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> FADTTSter is a user-friendly version of FADTTS directed to users without advanced coding skills. FADTTS is Matlab (MathWorks Inc, MA, USA) based and needs Matlab coding knowledge to operate. FADTTSter was created to overcome this issue and make the statistical analysis accessible to any non-technical researcher. FADTTSter is a command line based module as well as an GUI based tool. Not only is FADTTSter practical but it enables any investigator to perform DTI analysis efficiently. <br />
<br />
In addition, FADTTSter performs fiber profile QC as well as customized plotting of the FADTTS output<br />
<br />
This tool can be divided in two main parts, each one working independently: a) Profile QC and Matlab script generation and matlab run for FADTTS, b) Statistical data plotting for FADTTS output. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1068&screenshot_id=1011"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1068&screenshot_id=1011" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 13, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>3D Slicer License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/fadttster/">http://www.nitrc.org/projects/fadttster/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina at Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="false-discovery-rate-weighted" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="False Discovery Rate Weighted">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=false-discovery-rate-weighted" rel="bookmark" name="false-discovery-rate-weighted">False Discovery Rate Weighted</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Simple and efficient, this application performs the Weighted False Discovery Rate procedure of Benjamini and Hochberg (1997) to correct for multiple testing. The good think is that you can test virtually any number of p-values (even millions) obtained with any test-statistics for any data set. The bonus is that you can assign a-priori weights to give a better chance to those variables that you deem important. In practice, this procedure is powerful only with a relatively small number of p-values. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 21, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/fdrw/">http://www.nitrc.org/projects/fdrw/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>GIPSA-lab, Grenoble, FRANCE <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="fandtasia-java-applet:-dt-mri-processing" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="fanDTasia Java Applet: DT-MRI Processing">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=fandtasia-java-applet:-dt-mri-processing" rel="bookmark" name="fandtasia-java-applet:-dt-mri-processing">fanDTasia Java Applet: DT-MRI Processing</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> FanDTasia is a Java applet tool for DT-MRI processing. It opens dw-mri datasets from user's computer and performs very efficient tensor field estimation using parallel threaded processing on user's browser. No installation is required. It runs on any operating system that supports Java (Windows, Mac, Linux,...). The estimated tensor field is guaranteed to be positive definite second order or higher order and is saved in user's local disc. MATLAB functions are also provided to open the tensor fields for your convenience in case you need to perform further processing. The fanDTasia Java applet provides also vector field visualization for 2nd and 4th-order tensors, as well as calculation of various anisotropic maps. Another useful feature is 3D fiber tracking (DTI-based) which is also shown using 3d graphics on the user's browser. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 5, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Non-Commercial Software License Agreement</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/fandtasia/">http://www.nitrc.org/projects/fandtasia/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="fasst-–-fmri-artefact-rejection-and-sleep-scoring-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="FASST – fMRI Artefact rejection and Sleep Scoring Toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=fasst-–-fmri-artefact-rejection-and-sleep-scoring-toolbox" rel="bookmark" name="fasst-–-fmri-artefact-rejection-and-sleep-scoring-toolbox">FASST – fMRI Artefact rejection and Sleep Scoring Toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> FASST is an EEG toolbox developed to help users with 3 specific types of data and problems: simulatenous EEG-fMRI recording, continuous EEG scoring (e.g. sleep) and handling (visualisation, cutting, power spectrum, etc.) multi-channel recording of spontaneous EEG.<br />
The toolbox is written in Matlab and is specifically compatible with the BrainAmp family of EEG recorders (from BrainProducts GmbH) Three other data formats are now also supported: the edf 'European Data Format', exported raw-EGI data (from Electrical Geodesics, Inc.) and the BCI2000 format.The results are directly compatible with SPM8 and are saved with SPM8 EEG data format. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=648&screenshot_id=564"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=648&screenshot_id=564" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 26, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.montefiore.ulg.ac.be/~phillips/FASST.html">http://www.montefiore.ulg.ac.be/~phillips/FASST.html</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Liège <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="fast-fourier-transform-(fft)-javalibrary" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Fast Fourier Transform (FFT) JavaLibrary">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=fast-fourier-transform-(fft)-javalibrary" rel="bookmark" name="fast-fourier-transform-(fft)-javalibrary">Fast Fourier Transform (FFT) JavaLibrary</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The FFT Java library (<a href="http://www.loni.usc.edu/Software/FFT" target="_new">http://www.loni.usc.edu/Software/FFT</a>) is used for the execution of discrete Fourier transforms in 1-D, 2-D and 3-D through the implementation of Fast Fourier Transform (FFT) algorithms.<br />
<br />
FFT toolbox is distributed by the Laboratory of Neuro Imaging (<a href="http://www.loni.usc.edu/Software/FFT" target="_new">http://www.loni.usc.edu/Software/FFT</a>) at USC. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=304&screenshot_id=409"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=304&screenshot_id=409" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 8, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>LONI Software License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/fft/">http://www.nitrc.org/projects/fft/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Southern California <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="fast-lesion-extraction-using-convolutional-neural-networks" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Fast Lesion Extraction using Convolutional Neural Networks">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=fast-lesion-extraction-using-convolutional-neural-networks" rel="bookmark" name="fast-lesion-extraction-using-convolutional-neural-networks">Fast Lesion Extraction using Convolutional Neural Networks</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> FLEXCONN (Fast Lesion Extraction using Convolutional Neural Networks) is a toolbox for segmenting white matter lesions from multi-contrast MR images. Using T1-w and FLAIR images, a fully convolutional neural network (CNN) is trained using manually labeled training data. The trained CNN model can be applied to pre-processed pair of T1 and FLAIR images to generate a lesion membership as well as a hard segmentation. The algorithm is described in the following paper:<br />
<a href="https://arxiv.org/abs/1803.09172" target="_new">https://arxiv.org/abs/1803.09172</a><br />
<br />
Python codes for training and prediction are provided. Trained models using manually labeled atlases from ISBI-2015 challenge (<a href="https://www.nitrc.org/projects/longitudinal_ms" target="_new">https://www.nitrc.org/projects/longitudi...</a>) are also provided. The CNN is implemented in Tensorflow and Keras (<a href="https://github.com/fchollet/keras" target="_new">https://github.com/fchollet/keras</a>).<br />
<br />
This work was developed with funding support from the National Multiple Sclerosis Society (RG-1507-05243) and from the Center for Neuroscience and Regenerative Medicine in the Department of Defense. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1214&screenshot_id=1138"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1214&screenshot_id=1138" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 23, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial Share Alike</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/flexconn/">http://www.nitrc.org/projects/flexconn/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Henry Jackson Foundation <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="fast-nonlocal-means-for-mri-denoising" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Fast Nonlocal Means for MRI denoising">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=fast-nonlocal-means-for-mri-denoising" rel="bookmark" name="fast-nonlocal-means-for-mri-denoising">Fast Nonlocal Means for MRI denoising</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=artifact-removal" rel="tag">Artifact Removal</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=filtering" rel="tag">Filtering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=intensity-non-uniformity-correction" rel="tag">Intensity Non-Uniformity Correction</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quality-metrics" rel="tag">Quality Metrics</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This is a fast and robust implementation of the popular Nonlocal Means for MRI-Rician denoising. It works by computing the non-local weights based on distances in a features space comprising the local mean value and gradients of the image.<br />
<br />
It can reach an acceleration factor of 20x over the original implementation, with an improved performance for medium-low SNR images.<br />
<br />
We use a bias correction step for Rician noise based on the well-known Conventional Approach.<br />
<br />
This software can be compiled either as a Slicer module or a stand-alone:<br />
<br />
<a href="http://www.nitrc.org/snapshots.php?group_id=518" target="_new">http://www.nitrc.org/snapshots.php?group...</a><br />
<br />
For diffusion MRI volumes, see also: <a href="http://www.nitrc.org/projects/jalmmse_dwi" target="_new">http://www.nitrc.org/projects/jalmmse_dw...</a><br />
<br />
Key words: nonlocal (non-local) means, NLM, C++, ITK 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=518&screenshot_id=400"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=518&screenshot_id=400" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 26, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>3D Slicer License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/unlmeans/">http://www.nitrc.org/projects/unlmeans/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Brigham and Women's Hospital <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Microsoft<br>Other UNIX-like</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, MINC, NIfTI-1, Nrrd, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, MINC, NIfTI-1, Nrrd, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="fast-t2-relaxation-data-analysis-with-stimulated-echo-correction-and-non-local-spatial-regularisation" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Fast T2 relaxation data analysis with stimulated echo correction and non-local spatial regularisation">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=fast-t2-relaxation-data-analysis-with-stimulated-echo-correction-and-non-local-spatial-regularisation" rel="bookmark" name="fast-t2-relaxation-data-analysis-with-stimulated-echo-correction-and-non-local-spatial-regularisation">Fast T2 relaxation data analysis with stimulated echo correction and non-local spatial regularisation</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This tool is developed to offer a fast algorithm for computing myelin maps from multiecho T2 relaxation data using parallel computation with multicore CPUs and graphics processing units (GPUs). The tool also provides non-local spatial regularization to produce more accurate and reliable myelin maps for noisy T2 relaxation data.<br />
<br />
The details of the method are given the following papers:<br />
<br />
Yoo, Youngjin et al., &quot;Fast computation of myelin maps from MRI T2 relaxation data using multicore CPU and graphics card parallelization.&quot; Journal of Magnetic Resonance Imaging (2014).<br />
<br />
Yoo, Youngjin, and Roger Tam. &quot;Non-local spatial regularization of MRI T2 relaxation images for myelin water quantification.&quot; In Medical Image Computing and Computer-Assisted Intervention–MICCAI 2013, pp. 614-621. Springer Berlin Heidelberg, 2013. <br />
<br />
The JMRI paper stated that we intented to contribute to Gadgetron, but we found that NITRC seems to be a better fit so we have decided to share our code through NITRC. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 25, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/nlsrnnls/">http://www.nitrc.org/projects/nlsrnnls/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of British Columbia <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="fbirn-image-processing-scripts-(fips)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="FBIRN Image Processing Scripts (FIPS)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=fbirn-image-processing-scripts-(fips)" rel="bookmark" name="fbirn-image-processing-scripts-(fips)">FBIRN Image Processing Scripts (FIPS)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database-application" rel="tag">Database Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=statistical-operation" rel="tag">Statistical Operation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> FBIRN Imaging Processing Scripts (FIPS) is a FSL package for the comprehensive management of large-scale multi-site fMRI projects, including data storage, retrieval, calibration, analysis, multi-modal integration, and quality control. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 19, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/fips/">http://www.nitrc.org/projects/fips/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="fiber-tracking-based-on-finsler-distance" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Fiber-tracking based on Finsler distance">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=fiber-tracking-based-on-finsler-distance" rel="bookmark" name="fiber-tracking-based-on-finsler-distance">Fiber-tracking based on Finsler distance</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=diffusion-mr-fiber-tracking" rel="tag">Diffusion Mr Fiber Tracking</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This software is currently provided as a sub-project in the Finsler-tractography module: <a href="http://www.nitrc.org/projects/finslertract" target="_new">http://www.nitrc.org/projects/finslertra...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 23, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>3D Slicer License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/finslerbacktr/">http://www.nitrc.org/projects/finslerbacktr/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Brigham and Woman's Hospital <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="fiber-tracking-tool" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Fiber Tracking Tool">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=fiber-tracking-tool" rel="bookmark" name="fiber-tracking-tool">Fiber Tracking Tool</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Used to analyze the fibers produced by ukf tractography </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 12, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/fiber-sig/">http://www.nitrc.org/projects/fiber-sig/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UNC CH <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="fiberviewerlight" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="FiberViewerLight">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=fiberviewerlight" rel="bookmark" name="fiberviewerlight">FiberViewerLight</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=diffusion-mr-fiber-tracking" rel="tag">Diffusion Mr Fiber Tracking</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Fiber ViewerLight is an open-source C++ application to analyze fiber bundles. It provides:<br />
- Several methods for clustering: Length, Gravity, Hausdorff, and Mean methods but also a Normalized Cut algorithm.<br />
- 3D fibers visualization<br />
- 3D plane selection<br />
<br />
FiberViewerLight is now available as a 3D Slicer extension (<a href="http://www.slicer.org" target="_new">http://www.slicer.org</a>) 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=534&screenshot_id=767"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=534&screenshot_id=767" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 28, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/fvlight/">http://www.nitrc.org/projects/fvlight/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolin at Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Microsoft</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="fieldtrip" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="FieldTrip">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=fieldtrip" rel="bookmark" name="fieldtrip">FieldTrip</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=eeg-modeling" rel="tag">Eeg Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=fourier-time-domain-analysis" rel="tag">Fourier Time-Domain Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=independent-component-analysis" rel="tag">Independent Component Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=meg-modeling" rel="tag">Meg Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=multivariate-analysis" rel="tag">Multivariate Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=registration" rel="tag">Registration</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=spectral-analysis" rel="tag">Spectral Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> FieldTrip is the Matlab software toolbox for MEG and EEG analysis that is being developed at the Donders Institute for Brain, Cognition and Behaviour at the Radboud University Nijmegen, the Netherlands together with collaborating institutes.<br />
<br />
The toolbox includes algorithms for simple and advanced analysis of MEG, EEG, and invasive electrophysiological data, such as time-frequency analysis, source reconstruction using dipoles, distributed sources and beamformers and non-parametric statistical testing. It supports the data formats of all major MEG systems (CTF, Neuromag, BTi) and of the most popular EEG systems, and new formats can be added easily. FieldTrip contains high-level functions that you can use to construct your own analysis protocols in Matlab. Furthermore, it easily allows developers to incorporate low-level algorithms for new EEG/MEG analysis methods. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=243&screenshot_id=735"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=243&screenshot_id=735" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 1, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.fieldtriptoolbox.org">http://www.fieldtriptoolbox.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Radboud University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>AFNI BRIK, ANALYZE, DICOM, MINC, MINC2, NIfTI-1, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>AFNI BRIK, ANALYZE, DICOM, MINC, MINC2, NIfTI-1, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="filamenttracer" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="FilamentTracer">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=filamenttracer" rel="bookmark" name="filamenttracer">FilamentTracer</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?tag=image-processing" rel="tag">Image Processing</a>, <a href="https://hbp-hpc-platform.fz-juelich.de/?tag=neurons" rel="tag">Neurons</a>, <a href="https://hbp-hpc-platform.fz-juelich.de/?tag=tracing" rel="tag">Tracing</a></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=module" rel="tag">Module</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description><b><a href="http://www.bitplane.com/imaris/filamenttracer">FilamentTracer</a></b> is the most advanced software product for the automatic detection
      of neurons (dendritic trees, axons and spines), microtubules, and other filament-like structures in 2D, 3D and 4D. When combined with ImarisTrack,
      detection of temporal changes in length and volume of developing spines and dendrites will help researchers understand alterations caused by
      developmental and environmental changes.
      <p/>
      <ul>
        <li>Versatile tracing methods</li>
        <li>Obtain branch length and diameter statistics</li>
        <li>Manually draw segments with automatic centering</li>
        <li>Utilize multiple branch selection methods</li>
        <li>Visualize filaments along with non-filamentous structures</li>
      </ul>
    </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td></td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Proprietary commercial software license</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.bitplane.com/imaris/filamenttracer">http://www.bitplane.com/imaris/filamenttracer</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>
      Bitplane AG
      
     <a href="mailto:(
        eusupport@bitplane.com
      )">(
        eusupport@bitplane.com
      )</a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td>Imaris</td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>CSV</td></tr>
			<tr class="even"><td>Output Formats</td><td>CSV</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="fine-relational-spatial-topology" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Fine Relational Spatial Topology">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=fine-relational-spatial-topology" rel="bookmark" name="fine-relational-spatial-topology">Fine Relational Spatial Topology</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This tool takes resting state data and computes whole brain gradient maps describing the preferred connectivity across the spatial domains of your region of interest. This is a Matlab toolbox that depends on SPM, but primarily runs off of it's own GUI. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1225&screenshot_id=1147"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1225&screenshot_id=1147" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 18, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/first/">http://www.nitrc.org/projects/first/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="finsler-tractography-module-for-slicer" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Finsler tractography module for Slicer">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=finsler-tractography-module-for-slicer" rel="bookmark" name="finsler-tractography-module-for-slicer">Finsler tractography module for Slicer</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=connectivity-analysis" rel="tag">Connectivity Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=diffusion-mr-fiber-tracking" rel="tag">Diffusion Mr Fiber Tracking</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=diffusion-spectrum" rel="tag">Diffusion Spectrum</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=direction" rel="tag">Direction</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=q-ball" rel="tag">Q-Ball</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This module implements the Finsler tractography method with HARDI data described by J. Melonakos et al. From a set of seeding and target points, the paths are estimated as the shortest path taking into account a local, directional dependent cost.<br />
<br />
The output provided is the connectivity map from each voxel in the volume to the seeding points, plus a vector volume with the directions tangent to the fiber bundles at each point. If the Backtracing module within is built, these directions can be traced back to actually compute the fiber bundles (VTK required).<br />
<br />
The software can be built as either a stand-alone or a CLI plugin for 3D Slicer. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=464&screenshot_id=401"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=464&screenshot_id=401" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 15, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>3D Slicer License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/finslertract/">http://www.nitrc.org/projects/finslertract/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Microsoft<br>Other UNIX-like</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Nrrd</td></tr>
			<tr class="even"><td>Output Formats</td><td>Nrrd</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="fitbir:-federal-interagency-traumatic-brain-injury-research" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="FITBIR: Federal Interagency Traumatic Brain Injury Research">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=fitbir:-federal-interagency-traumatic-brain-injury-research" rel="bookmark" name="fitbir:-federal-interagency-traumatic-brain-injury-research">FITBIR: Federal Interagency Traumatic Brain Injury Research</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=knowledge-environment" rel="tag">Knowledge Environment</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=web-resource" rel="tag">Web Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Federal Interagency Traumatic Brain Injury Research (FITBIR) informatics system was developed to share data across the entire TBI research field and to facilitate collaboration between laboratories, as well as interconnectivity with other informatics platforms. Sharing data, methodologies, and associated tools, rather than summaries or interpretations of this information, can accelerate research progress by allowing re-analysis of data, as well as re-aggregation, integration, and rigorous comparison with other data, tools, and methods. This community-wide sharing requires common data definitions and standards, as well as comprehensive and coherent informatics approaches. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=674&screenshot_id=602"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=674&screenshot_id=602" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 8, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>NIH Data Access Policy</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://fitbir.nih.gov/">http://fitbir.nih.gov/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>National Institutes of Health <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="fitting-oscillations-&-one-over-f" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Fitting Oscillations & One-Over F">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=fitting-oscillations-&-one-over-f" rel="bookmark" name="fitting-oscillations-&-one-over-f">Fitting Oscillations & One-Over F</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> FOOOF is a fast, efficient, and physiologically-informed tool to parameterize neural power spectra. FOOOF conceives of a model of the power spectrum as a combination of two distinct functional processes - a) an aperiodic 'background' component, reflecting 1/f like characteristics, modeled with an exponential fit, with b) a variable number of periodic components, that exhibit as band-limited peaks rising above this background, reflecting putative oscillations, modeled as Gaussians. For putative oscillations, the benefit of the FOOOF approach is that these peaks are characterized in terms of their specific center frequency, amplitude and bandwidth without requiring predefining specific bands of interest. It separates these peaks from a dynamic and independently interesting aperiodic background. This tool offers a way to parameterize both the aperiodic and periodic components together, explicitly dealing with the fact that they overlap in ways that makes estimating one or other in isolation difficult. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 20, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://github.com/voytekresearch/fooof">https://github.com/voytekresearch/fooof</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>VoytekLab <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="fiv" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="FIV">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=fiv" rel="bookmark" name="fiv">FIV</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-display" rel="tag">Image Display</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Functional Image Viewer, also known as FIV, is a tool for visualizing functional and anatomic MRI data. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 20, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://nrg.wustl.edu/projects/fiv">http://nrg.wustl.edu/projects/fiv</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Microsoft<br>Other OS<br>SunOS/Solaris</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, MINC, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, MINC, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="fmri-bold-signal-simulator" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="fMRI BOLD Signal Simulator">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=fmri-bold-signal-simulator" rel="bookmark" name="fmri-bold-signal-simulator">fMRI BOLD Signal Simulator</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This library provides simulation tools for highly realistic resting state fMRI BOLD signal based on the scientific findings in a PhD dissertation. This library is intended to be used in scientific research, primarily in Neuroscience. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 17, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Academic Free License ("AFL")</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/fmriboldsim2016/">http://www.nitrc.org/projects/fmriboldsim2016/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Arkansas at Little Rock <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="fmri-classification-in-r" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="fMRI Classification in R">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=fmri-classification-in-r" rel="bookmark" name="fmri-classification-in-r">fMRI Classification in R</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> We demonstrate and provide R code that can classify between groups of fMRI scans based on functional network connectivity differences, requiring only 4 lines of code to be altered. In addition, we include a detailed article explaining the methods behind and motivations of this tool. This code can also be altered to perform connectivity analysis and classification using ROI based methods by reading in distance arrays previously created. We run ICA on fMRI data to establish functional networks, measure the functional connectivity between these networks using the temporal cross-correlations between independent component to create a distance matrix and indicating the networking. Connectivity properties are used as a feature matrix for an SVM classifier. Collectively, this project provides and explains both methods and code to perform functional network connectivity and fMRI SVM classiﬁcation. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 9, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/fmriclassify/">http://www.nitrc.org/projects/fmriclassify/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="fmri-cpca" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="fMRI-CPCA">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=fmri-cpca" rel="bookmark" name="fmri-cpca">fMRI-CPCA</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=principal-component-analysis" rel="tag">Principal Component Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=regression" rel="tag">Regression</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Constrained Principal Component Analysis (CPCA) combines regression analysis and principal component analysis into a unified framework. This method derives images of functional neural networks from singular-value decomposition of BOLD signal time series, and allows derivation of images when the analyzed BOLD signal is constrained to the scans occurring in peristimulus time, using all other scans as baseline. <br />
<br />
CPCA provides allows (1) determination of multiple functional networks involved in a task, (2) estimation of the pattern of BOLD changes associated with each functional network over peristimulus time points, (3) quantification of the degree of interaction between these multiple functional networks, and (4) a statistical test of the degree to which experimental manipulations affect each functional network. <br />
<br />
fMRI CPCA provides all results in matlab.mat file format, as well as writing images in analyze format for all components, rotated and unrotated. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=203&screenshot_id=332"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=203&screenshot_id=332" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 19, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/fmricpca/">http://www.nitrc.org/projects/fmricpca/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="fmri-grocer" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="fMRI Grocer">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=fmri-grocer" rel="bookmark" name="fmri-grocer">fMRI Grocer</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This toolbox contains many kinds of kits that you may be interested in during fMRI data analysis.<br />
<br />
This toolbox is a homebrew kits built during practical ASL(arterial spin labeling) based CBF data analysis. Meanwhile, this toolbox is also compatible with BOLD data analysis. Everyone would find something useful for their own data analysis!<br />
<br />
This toolbox is run and tested on SPM8 with MATLAB 7.6.0(R2008a) under the Linux OS.<br />
<br />
Theoretically, most of the functions (except the menu1&amp;2 which are specially designed for the Batch Editor of SPM8) of this toolbox should be compatible with SPM5 and should also work smoothly under the Windows OS.<br />
<br />
Feel free to give feedback to authors if you encounter any bugs or problems. <br />
<br />
Senhua Zhu<br />
Center for functional Neuroimaging, University of Pennsylvania<br />
3 W.Gates Bldg, 3400, Philadelphia, PA (19104), United States<br />
<br />
Email: <a href="mailto:zshtom@gmail.com" target="_new">zshtom@gmail.com</a> ; <a href="mailto:senhua@mail.med.upenn.edu" target="_new">senhua@mail.med.upenn.edu</a><br />
<br />
QQ group number (QQ群): 60524357<br />
<br />
Google group: <a href="https://groups.google.com/d/forum/fmri-grocer" target="_new">https://groups.google.com/d/forum/fmri-g...</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=700&screenshot_id=630"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=700&screenshot_id=630" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 19, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>fMRI Grocer License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/fmri_grocer/">http://www.nitrc.org/projects/fmri_grocer/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pennsylvania <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="fmripower" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="FMRIpower">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=fmripower" rel="bookmark" name="fmripower">FMRIpower</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=linear" rel="tag">Linear</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-software-resource" rel="tag">Other Software Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> A common question investigators ask statisticians when working on grants is how to calculate power. FMRIpower is an easy to use matlab-based graphical user interface that calculates power for future studies based on older analyses or pilot data. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 29, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Public Domain</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://fmripower.org">http://fmripower.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="fmriprep" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="fMRIPrep">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=fmriprep" rel="bookmark" name="fmriprep">fMRIPrep</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> fMRIPrep is a functional magnetic resonance imaging (fMRI) data preprocessing pipeline that is designed to provide an easily accessible, state-of-the-art interface that is robust to variations in scan acquisition protocols and that requires minimal user input while providing easily interpretable and comprehensive error and output reporting. It performs basic processing steps (coregistration, normalization, unwarping, noise component extraction, segmentation, skull-stripping etc.) providing outputs that can be easily submitted to a variety of group-level analyses, including task-based or resting-state fMRI, graph theory measures, surface or volume-based statistics, etc. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 21, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD/MIT-Style Open Source License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://fmriprep.org">http://fmriprep.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Stanford University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>BIDS (Brain Imaging Data Structure)</td></tr>
			<tr class="even"><td>Output Formats</td><td>BIDS (Brain Imaging Data Structure)</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="fmri-results-comparison-library" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="fMRI Results Comparison Library">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=fmri-results-comparison-library" rel="bookmark" name="fmri-results-comparison-library">fMRI Results Comparison Library</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Exported results from 28 variants of an fMRI analysis, conducted in AFNI, FSL, and SPM, carried out on the OpenfMRI BIDS-compliant ds000011 dataset. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 13, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Public Domain</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/frcl/">http://www.nitrc.org/projects/frcl/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Warwick <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="fmristat" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="fMRIstat">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=fmristat" rel="bookmark" name="fmristat">fMRIstat</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=linear" rel="tag">Linear</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=modeling" rel="tag">Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=principal-component-analysis" rel="tag">Principal Component Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> A Matlab toolbox for the statistical analysis of fMRI data. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 23, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.math.mcgill.ca/keith/fmristat">http://www.math.mcgill.ca/keith/fmristat</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>AFNI BRIK, ANALYZE, MINC, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>AFNI BRIK, ANALYZE, MINC, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="fnir-devices" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="fNIR Devices">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=fnir-devices" rel="bookmark" name="fnir-devices">fNIR Devices</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=nirs/fnir" rel="tag">Nirs/Fnir</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> fNIR Imager 1100 is a new generation portable functional near-infrared (fNIR) imaging research tool capable of monitoring brain’s hemodynamics and thereby the cognitive state of the subject in natural environments. <br />
<br />
Neuroimaging Solution for Natural Environments:<br />
•fNIR is the only stand-alone and field-deployable technology able to determine localized brain activity. <br />
•fNIR can be readily integrated with other physiological and neurobehavioral measures that assess human brain activity, including eye tracking, pupil reflex, respiration and electrodermal activity. fNIR can also complement other techniques.<br />
•Studies have shown a positive correlation between a participant's performance and fNIR responses as a function of task load.<br />
•It has also been shown that fNIR can effectively monitor attention and working memory in real-life situations. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=492&screenshot_id=370"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=492&screenshot_id=370" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 29, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>FNIR COBI Suite</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.fnirdevices.com">http://www.fnirdevices.com</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>fNIR Devices <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="fnirs-data-analysis-environment" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="fNIRS Data Analysis Environment">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=fnirs-data-analysis-environment" rel="bookmark" name="fnirs-data-analysis-environment">fNIRS Data Analysis Environment</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Our resource is a data analysis environment for diffuse optical tomography (DOT) functional neuroimaging data. Developed to process data from steady-state time-series measurements, it allows for maximal flexibility in the number and positions of optodes. The central component is an application called NAVI. Features include:<br />
1. An electronic ledger (records metadata for all data transformations).<br />
2. Data conditioning (e.g., frequency-filtering, selection of data on the basis of signal-to-noise ratio.)<br />
3. 2D or 3D image formation and display.<br />
4. Interpretation: atlas-based mapping; automated anatomical labeling; GLM; data-driven methods (e.g., PCA, ICA); model-based (e.g., dynamic causal modeling) and data-driven (e.g., correlation) connectivity analysis.<br />
 Another important component is the Brain Model Generator, which includes FEM meshes for all parts of the head accessible to DOT measurements. The user can input the numbers of optodes, and manually specify their locations or input tracking-system data. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 27, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/fnirs_downstate/">http://www.nitrc.org/projects/fnirs_downstate/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>SUNY Downstate Medical Center <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="fnirs-during-stroop-before-and-after-acute-bout-of-aerobic-exercise" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="fNIRS during Stroop before and after acute bout of aerobic exercise">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=fnirs-during-stroop-before-and-after-acute-bout-of-aerobic-exercise" rel="bookmark" name="fnirs-during-stroop-before-and-after-acute-bout-of-aerobic-exercise">fNIRS during Stroop before and after acute bout of aerobic exercise</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> fNIRS data collected from prefrontal and motor cortices while human subjects perform Stroop task before and after thirty minutes of cardiovascular exercise. Associated preprint: <a href="https://www.biorxiv.org/content/early/2018/06/29/359307" target="_new">https://www.biorxiv.org/content/early/20...</a><br />
<br />
Associated Dissertation: <br />
<a href="https://digital.library.unt.edu/ark:/67531/metadc1157584/" target="_new">https://digital.library.unt.edu/ark:/675...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 8, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Academic Free License ("AFL")</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/exercisefnirs/">http://www.nitrc.org/projects/exercisefnirs/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UNT Denton <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="fnirs-research-database" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="fNIRS Research Database">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=fnirs-research-database" rel="bookmark" name="fnirs-research-database">fNIRS Research Database</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> In the DOC tab, you can find 7 fNIRS datasets as well as the article that corresponds to them. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1071&screenshot_id=1118"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1071&screenshot_id=1118" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 2, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/fnirsdata/">http://www.nitrc.org/projects/fnirsdata/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>MGH <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="fnirs-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="fNIRS-toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=fnirs-toolbox" rel="bookmark" name="fnirs-toolbox">fNIRS-toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This toolbox is a set of Matlab based tools for the analysis of functional near-infrared spectroscopy (fNIRS) developed by T. Huppert at the University of Pittsburgh </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 13, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/nirs-toolbox/">http://www.nitrc.org/projects/nirs-toolbox/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pittsburgh <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="forward:-accurate-finite-element-electromagnetic-head-models" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Forward: Accurate finite element electromagnetic head models">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=forward:-accurate-finite-element-electromagnetic-head-models" rel="bookmark" name="forward:-accurate-finite-element-electromagnetic-head-models">Forward: Accurate finite element electromagnetic head models</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This project aims to simplify the preparation of accurate electromagnetic head models for EEG forward modeling.<br />
<br />
It builds off of the seminal SimNIBS tool for electromagnetic field modelling of transcranial magnetic stimulation (TMS) and transcranial direct current stimulation. Human skin, skull, cerebrospinal fluid, and brain meshing pipelines have been rewritten with Nipype to ease access parallel processing and to allow users to start/stop the workflows. Conductivity tensor mapping from diffusion-weighted imaging is also included. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 5, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU GPL v2</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/forward/">http://www.nitrc.org/projects/forward/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Liège <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="framewise-integrated-real-time-mri-monitoring" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Framewise Integrated Real-time MRI Monitoring">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=framewise-integrated-real-time-mri-monitoring" rel="bookmark" name="framewise-integrated-real-time-mri-monitoring">Framewise Integrated Real-time MRI Monitoring</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> [For Investigational Use Only]<br />
<br />
Easy to set up and use, the FIRMM software suite provides MRI scanner operators with data quality metrics in real time. Using FIRMM to identify the ideal scan time for each person can reduce total brain MRI scan times and associated costs by 50% or more.<br />
<br />
Features:<br />
<br />
Accurate FD calculations as you scan. FIRMM’s framewise displacement (FD) calculations are not only fast, but also accurate, when compared to a standard, commonly utilized offline, post-hoc processing stream.<br />
<br />
Prediction of remaining scan time needed. We built an algorithm that accurately predicts the required scan time until you have captured as much quality data as you need.<br />
<br />
User-friendly, real-time feedback. FIRMM users have been able to share the percentage of quality data frames with participants or display the FIRMM GUI on the participant’s screen in the scanner room for feedback and training purposes. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1144&screenshot_id=1107"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1144&screenshot_id=1107" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 23, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>FIRMM license</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://firmm.io/">http://firmm.io/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Oregon Health and Science University and Washington University in St. Louis <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="framework-for-open-programmatic-access" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Framework for Open Programmatic Access">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=framework-for-open-programmatic-access" rel="bookmark" name="framework-for-open-programmatic-access">Framework for Open Programmatic Access</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Framework for Open Programmatic Access (FOPA) aims to provide a standardized framework for communication and data exchange between medical imaging applications, with particular focus on neuroimaging technologies. FOPA is an attempt to design and implement a common protocol for network and command line communication with either file-system or imbedded data structures. Initial reference implementations will support interoperability between ITK, VTK, and Java platforms. Contributions are welcome from other neuroimaging development communities. <br />
<br />
This project is an open community effort under the BSD license. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 24, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/fopa/">http://www.nitrc.org/projects/fopa/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="freesurfer" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="FreeSurfer">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=freesurfer" rel="bookmark" name="freesurfer">FreeSurfer</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=anatomic" rel="tag">Anatomic</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=gaussian-curvature" rel="tag">Gaussian Curvature</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-display" rel="tag">Image Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-reconstruction" rel="tag">Image Reconstruction</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=mean-curvature" rel="tag">Mean Curvature</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=spatial-transformation" rel="tag">Spatial Transformation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-rendering" rel="tag">Surface Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=thickness" rel="tag">Thickness</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> FreeSurfer is a set of automated tools for reconstruction of the brain’s cortical surface from structural MRI data, and overlay of functional MRI data onto the reconstructed surface. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=62&screenshot_id=7"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=62&screenshot_id=7" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 13, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>FreeSurfer License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://surfer.nmr.mgh.harvard.edu/">http://surfer.nmr.mgh.harvard.edu/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>MGH/MGZ</td></tr>
			<tr class="even"><td>Output Formats</td><td>MGH/MGZ</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="freesurfer-fsaverage" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Freesurfer fsaverage">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=freesurfer-fsaverage" rel="bookmark" name="freesurfer-fsaverage">Freesurfer fsaverage</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Freesurfer fsaverage mesh files in gifti format </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 29, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/fsaverage_gifti/">http://www.nitrc.org/projects/fsaverage_gifti/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Harvard University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="freesurfr" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="freesurfR">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=freesurfr" rel="bookmark" name="freesurfr">freesurfR</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=software" rel="tag">Software</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This package contains tools for doing group analysis of FreeSurfer (<a href="https://surfer.nmr.mgh.harvard.edu/" target="_new">https://surfer.nmr.mgh.harvard.edu/</a>) surface data using the general linear model in R (lm).<br />
Results can be rendered in FreeSurfer's freeview or AFNI's SUMA. Plots for selected vertices can be rendered in R with ggplot2. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 18, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU GPL v2</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/freesurf_r/">http://www.nitrc.org/projects/freesurf_r/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>POSIX/UNIX-like</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>MGH/MGZ</td></tr>
			<tr class="even"><td>Output Formats</td><td>MGH/MGZ</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="fsl" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="FSL">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=fsl" rel="bookmark" name="fsl">FSL</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=affine-warp" rel="tag">Affine Warp</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-display" rel="tag">Image Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=independent-component-analysis" rel="tag">Independent Component Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=registration" rel="tag">Registration</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=temporal-transformation" rel="tag">Temporal Transformation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tensor-metric" rel="tag">Tensor Metric</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tractography" rel="tag">Tractography</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volumetric-analysis" rel="tag">Volumetric Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> FSL is a comprehensive library of image analysis and statistical tools for FMRI, MRI and DTI brain imaging data. FSL is written mainly by members of the Analysis Group, FMRIB, Oxford, UK. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=25&screenshot_id=658"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=25&screenshot_id=658" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 7, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>FSL License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.fmrib.ox.ac.uk/fsl/">http://www.fmrib.ox.ac.uk/fsl/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>SunOS/Solaris<br>Windows NT/2000</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="fsl-extensions" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="FSL extensions">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=fsl-extensions" rel="bookmark" name="fsl-extensions">FSL extensions</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> FSL extensions is a reference for modifications, extensions, and utilities for the FMRIB Software Library (FSL). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 2, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/fsl_extensions/">http://www.nitrc.org/projects/fsl_extensions/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="functional-analysis-of-diffusion-tensor" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Functional Analysis of Diffusion Tensor">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=functional-analysis-of-diffusion-tensor" rel="bookmark" name="functional-analysis-of-diffusion-tensor">Functional Analysis of Diffusion Tensor</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=genomic-analysis" rel="tag">Genomic Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> A functional analysis of diffusion tensor tract statistics (FADTTS) pipeline was developed for delineating the association between multiple diffusion properties along major white matter fiber bundles with a set of covariates of interest, such as age, diagnostic status, genetic marker (e.g., SNP), and gender, and the structure of the variability of these white matter tract properties in various diffusion tensor imaging studies. FADTTS can be used to facilitate understanding of normal brain development, the neural bases of neuropsychiatric disorders, and the joint effects of environmental and genetic factors on white matter fiber bundles. The advantages of FADTTS compared with the other existing approaches are that they are capable of modelling the structured inter-subject variability, testing the joint effects, and constructing their simultaneous confidence bands. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 22, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Academic Free License ("AFL")</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/fadtts/">http://www.nitrc.org/projects/fadtts/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="functional-by-strutural-hierarchical-mapping-(fsh-mapping)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Functional-by-Strutural Hierarchical Mapping (FSH mapping)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=functional-by-strutural-hierarchical-mapping-(fsh-mapping)" rel="bookmark" name="functional-by-strutural-hierarchical-mapping-(fsh-mapping)">Functional-by-Strutural Hierarchical Mapping (FSH mapping)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Not every edge in the diffusion-MRI derived structural network is utilized in each specific brain status. Therefore, this tool, which we name functional by structural hierarchical (FSH) mapping, is to predict functional MRI (fMRI) correlation matrix using diffusion MRI-derived structural connectome. The output is a binary utilization matrix, which indicates the utilization of the structural network edges. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 2, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Academic Free License ("AFL")</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/fsh_mapping/">http://www.nitrc.org/projects/fsh_mapping/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="functional-connectivity-analysis-tool-for-near-infrared-spectroscopy-data" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Functional Connectivity Analysis Tool for near-infrared spectroscopy data">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=functional-connectivity-analysis-tool-for-near-infrared-spectroscopy-data" rel="bookmark" name="functional-connectivity-analysis-tool-for-near-infrared-spectroscopy-data">Functional Connectivity Analysis Tool for near-infrared spectroscopy data</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> FC-NIRS is a Functional Connectivity Analysis Tool for near-infrared spectroscopy dataThe package’s functions include preprocessing, quality control, FC calculation and network analysis. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 25, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/fcnirs/">http://www.nitrc.org/projects/fcnirs/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Beijing Normal University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="functional-connectivity-community" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Functional Connectivity Community">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=functional-connectivity-community" rel="bookmark" name="functional-connectivity-community">Functional Connectivity Community</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=community" rel="tag">Community</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The purpose of this project is to provide a community for the discussion of functional connectivity and all related topics. This includes discussion of related tools, data sets, methodological discussion, related websites and publications, etc. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 20, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/func_connect/">http://www.nitrc.org/projects/func_connect/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="functional-imaging-birn" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Functional Imaging BIRN">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=functional-imaging-birn" rel="bookmark" name="functional-imaging-birn">Functional Imaging BIRN</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database-application" rel="tag">Database Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=ontology" rel="tag">Ontology</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-data-resource" rel="tag">Other Data Resource</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quality-metrics" rel="tag">Quality Metrics</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=statistical-operation" rel="tag">Statistical Operation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=web-service" rel="tag">Web Service</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The FBIRN Federated Informatics Research Environment (FIRE) includes tools and methods for multi-site functional neuroimaging. This includes resources for data collection, storage, sharing and management, tracking, and analysis of large fMRI datasets. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 18, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nbirn.net/research/function">http://www.nbirn.net/research/function</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="functional-mixed-processes-models" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Functional Mixed Processes Models">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=functional-mixed-processes-models" rel="bookmark" name="functional-mixed-processes-models">Functional Mixed Processes Models</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The aim of this tool is to implement a functional analysis pipeline, for the joint analysis of longitudinally measured functional data and clinical data, for example age, gender and disease status. FMPM consists of a functional mixed effects model for characterizing the association of functional response with covariates of interest by incorporating complex spatial–temporal correlation structure, an efficient method for spatially smoothing varying coefficient functions, an estimation method for estimating the spatial– temporal correlation structure, a test procedure with local and global test statistics for testing hypotheses of interest associated with functional response, and a simultaneous confidence band for quantifying the uncertainty in the estimated coefficient functions. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 3, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>NIH Data Access Policy</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/fmpm/">http://www.nitrc.org/projects/fmpm/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UNC-CH <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="functional-organization-of-the-mouse-lemur-primate-microcebus-murinus-:-from-multilevel-validation-to-comparison-with-humans" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Functional organization of the mouse lemur Primate Microcebus murinus : from multilevel validation to comparison with humans">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=functional-organization-of-the-mouse-lemur-primate-microcebus-murinus-:-from-multilevel-validation-to-comparison-with-humans" rel="bookmark" name="functional-organization-of-the-mouse-lemur-primate-microcebus-murinus-:-from-multilevel-validation-to-comparison-with-humans">Functional organization of the mouse lemur Primate Microcebus murinus : from multilevel validation to comparison with humans</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Brain network organization in the mouse lemur (Microcebus murinus) Primate.<br />
Archives contain:<br />
<br />
- Raw anatomical and resting-state functional MR images recorded from 14 mouse lemurs at 11.7 Tesla (2 time point per animal).<br />
<br />
- A functional atlas of the mouse lemur brain (48 cerebral regions) issued from resting fMRI.<br />
<br />
- A functional atlas of human brains (56 cerebral regions) issued from resting state fMRI data recorded from 42 healthy humans ranging from 41 to 60 years old at 3.0 Tesla.<br />
<br />
- Dictionary learning analysis in mouse lemurs and humans showing 6 cerebral networks identified in these two species. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 5, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>CeCill v2</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/fmri_mouselemur/">http://www.nitrc.org/projects/fmri_mouselemur/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Commissariat à l'énergie atomique et aux énergies alternatives <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="functional-real-time-interactive-endogenous-neuromodulation-and---decoding-(friend)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Functional Real-time Interactive Endogenous Neuromodulation and   Decoding (FRIEND)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=functional-real-time-interactive-endogenous-neuromodulation-and---decoding-(friend)" rel="bookmark" name="functional-real-time-interactive-endogenous-neuromodulation-and---decoding-(friend)">Functional Real-time Interactive Endogenous Neuromodulation and   Decoding (FRIEND)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=application" rel="tag">Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> FRIEND is a GUI-based user-friendly software for real-time fMRI processing, multivoxel pattern decoding and neurofeedback. The package integrates routines for image preprocessing in real-time, ROI-based feedback (single-ROI percent signal change and functional connectivity) and brain decoding-based feedback using the FSL and libSVM libraries. Users can create or employ pre-specified visual stimuli for neurofeedback experiments.<br />
The FRIEND for Windows standalone version runs embedded FSL and libSVM functions and provides a simple, straightforward solution. The FRIEND Engine is a multiplatform (Mac/Linux) toolbox that incorporates the same processing capabilities, but in which the front-end is separated from the processing module allowing users to develop their own front-end GUIs. Advanced users can develop plugins to extend the original pipeline provided by the engine (e.g., Matlab functions for data preprocessing or stimulus presentation features from standard packages such as E-Prime). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 6, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>FRIEND License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://github.com/InstitutoDOr/FriendENGINE/blob/master/README.md">https://github.com/InstitutoDOr/FriendENGINE/blob/master/README.md</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>D'Or Institute for Research and Education (IDOR) <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="functional-regression-analysis-of-dti" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Functional Regression Analysis of DTI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=functional-regression-analysis-of-dti" rel="bookmark" name="functional-regression-analysis-of-dti">Functional Regression Analysis of DTI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Functional Regression Analysis of DTI Tract Statistics (FRATS), for the analysis of multiple diffusion properties along fiber bundle as functions in an infinite dimensional space and their association with a set of covariates of interest, such as age, diagnostic status and gender, in real applications. The resulting analysis pipeline can be used for understanding normal brain development, the neural bases of neuropsychiatric disorders, and the joint effects of environmental and genetic factors on white matter fiber bundles. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 22, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Academic Free License ("AFL")</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/frats/">http://www.nitrc.org/projects/frats/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="functional-roi-atlas" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Functional ROI Atlas">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=functional-roi-atlas" rel="bookmark" name="functional-roi-atlas">Functional ROI Atlas</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Functional ROI Atlas is an effort to provide a set of quasi-probabilistic atlases for established &quot;functional ROIs&quot; in the human neuroimaging literature. Many atlases exist for various anatomical parcellation schemes, such as the Brodmann areas, the structural atlases, tissue segmentation atlases, etc. To date, however, there is no atlas for so-called &quot;functional ROIs&quot;. Such fROIs are typically associated with an anatomical label of some kind (e.g. the _fusiform_ face area), but these labels are only approximate and can be misleading inasmuch as fROIs are not constrained by anatomical landmarks, whether cytoarchitectonic or based on sulcal and gyral landmarks.<br />
<br />
The goal of this project is to provide quasi-probabilistic atlases for fROIs that are based on published coordinates in the neuroimaging literature. This is an open-ended enterprise and the atlas can grow as needed. Members of the neuroscience and neuroimaging community interested in contributing to the project are encouraged to do so. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 16, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/froi_atlas/">http://www.nitrc.org/projects/froi_atlas/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Rotman Research Institute <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="fusion-ica-toolbox-(fit)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Fusion ICA Toolbox (FIT)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=fusion-ica-toolbox-(fit)" rel="bookmark" name="fusion-ica-toolbox-(fit)">Fusion ICA Toolbox (FIT)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=independent-component-analysis" rel="tag">Independent Component Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=principal-component-analysis" rel="tag">Principal Component Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> FIT is a MATLAB toolbox which uses Independent Component Analysis (ICA) to extract the shared information across modalities like fMRI, EEG, sMRI and SNP data. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 3, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://icatb.sourceforge.net/fusion/fusion_startup.php">http://icatb.sourceforge.net/fusion/fusion_startup.php</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>OS Independent<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="fvgwas:-fast-voxelwise-genome-wide-association-analysis" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="FVGWAS: Fast Voxelwise Genome Wide Association Analysis">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=fvgwas:-fast-voxelwise-genome-wide-association-analysis" rel="bookmark" name="fvgwas:-fast-voxelwise-genome-wide-association-analysis">FVGWAS: Fast Voxelwise Genome Wide Association Analysis</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Fast Voxelwise Genome Wide Association analysiS (FVGWAS) framework to efficiently carry out whole-genome analyses of whole-brain data. FVGWAS consists of three components including a heteroscedastic linear model, a global sure independence screening (GSIS) procedure, and a detection procedure based on wild bootstrap methods. Specically, for standard linear association, the computational complexity is O(n*N_V*N_C) for voxelwise genome wide association analysis (VGWAS) method compared with<br />
O((N_C+N_V)*n^2) for FVGWAS. Our FVGWAS may be a valuable statistical toolbox for large-scale imaging genetic analysis as the field is rapidly advancing with ultra-high-resolution imaging and whole-genome sequencing. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 10, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BIAS License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/fvgwas/">http://www.nitrc.org/projects/fvgwas/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="g-3-/-neurospaces-simulator" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="G-3 / Neurospaces Simulator">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=g-3-/-neurospaces-simulator" rel="bookmark" name="g-3-/-neurospaces-simulator">G-3 / Neurospaces Simulator</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Neurospaces toolchain is being developed in the context of the GENESIS 3 project. Neurospaces software components serve the purpose of user-friendly computational model construction and simulation, morphology and topology analysis. A webinterface allows to define research projects in a database and browse models, simulation results and free text and other content related to a project. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 2, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU GPL v2</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/neurospaces/">http://www.nitrc.org/projects/neurospaces/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Neurospaces Development <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="gaain-toolset" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="GAAIN toolset">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=gaain-toolset" rel="bookmark" name="gaain-toolset">GAAIN toolset</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-data-resource" rel="tag">Other Data Resource</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-information-resource" rel="tag">Other Information Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Alzheimer's Association created The Global Alzheimer's Association Interactive Network (GAAIN) to connect Alzheimer’s and dementia researchers and speed discovery by connecting potential collaborators to those sharing data (GAAIN Data Partners). GAAIN's powerful interactive tools allow users to explore data and create cohorts across multiple data sources and run immediate preliminary analysis while upholding the data control, security, and privacy policies of the data owners. These tools are freely accessible to all at <a href="http://www.gaaindata.org." target="_new">http://www.gaaindata.org.</a><br />
<br />
A wide and vastly varying range of metadata from nearly 500,000 subjects can be explored using these tools. Attributes include demographics, assessments, health factors, biologicals, imaging and genetics. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1264&screenshot_id=1179"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1264&screenshot_id=1179" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 26, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/gaain/">http://www.nitrc.org/projects/gaain/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>GAAIN, Global Alzheimer's Association Interactive Network <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="gambit" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="GAMBIT">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=gambit" rel="bookmark" name="gambit">GAMBIT</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=mesh-generation" rel="tag">Mesh Generation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=thickness" rel="tag">Thickness</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> GAMBIT is an end-to-end application developped at UNC-Chapel Hill allowing Group-wise Automatic Mesh-Based analysis of cortIcal Thickness as well as other surface area measurements. This cross-platform tool can be run within 3D Slicer as an external module, or directly as a command line. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=384&screenshot_id=311"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=384&screenshot_id=311" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 10, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/gambit/">http://www.nitrc.org/projects/gambit/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina at Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, Nrrd, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, Nrrd, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="gamma" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="GAMMA">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=gamma" rel="bookmark" name="gamma">GAMMA</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=modeling" rel="tag">Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=statistical-operation" rel="tag">Statistical Operation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volumetric-analysis" rel="tag">Volumetric Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> GAMMA suite is an open-source cross-platform data mining software package designed to analyze neuroimaging data. A neuroimaging study often focuses on biomarker detection and classification. We designed and implemented a Bayesian, multivariate, nonparametric suite of algorithms for analyzing neuroimaging data. The GAMMA suite can be used for brain morphometric analysis, lesion-deficit analysis, and functional MR data analysis. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 29, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/gamma_suite/">http://www.nitrc.org/projects/gamma_suite/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Cygwin<br>HP-UX<br>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="gannet---batch-analysis-of-edited-magnetic-resonance-spectroscopy-(mrs)-data" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Gannet - Batch Analysis of Edited Magnetic Resonance Spectroscopy (MRS) Data">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=gannet---batch-analysis-of-edited-magnetic-resonance-spectroscopy-(mrs)-data" rel="bookmark" name="gannet---batch-analysis-of-edited-magnetic-resonance-spectroscopy-(mrs)-data">Gannet - Batch Analysis of Edited Magnetic Resonance Spectroscopy (MRS) Data</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=modeling" rel="tag">Modeling</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Gannet is a software package designed for the batch analysis of edited magnetic resonance spectroscopy (MRS) data. <br />
<br />
Gannet runs in Matlab, and is distributed as code rather than executables, empowering users to make local changes. <br />
<br />
Gannet is designed to run without user intervention, to remove operator variance from the quantification of edited MRS data. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1252&screenshot_id=1163"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1252&screenshot_id=1163" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 5, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.gabamrs.com/about">http://www.gabamrs.com/about</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>The Johns Hopkins University School of Medicine <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Other Format, Philips PAR/REC</td></tr>
			<tr class="even"><td>Output Formats</td><td>Other Format, Philips PAR/REC</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="gat-:-graph-analysis-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="GAT : Graph Analysis Toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=gat-:-graph-analysis-toolbox" rel="bookmark" name="gat-:-graph-analysis-toolbox">GAT : Graph Analysis Toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=connectivity-analysis" rel="tag">Connectivity Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=multivariate-analysis" rel="tag">Multivariate Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> GAT is a Matlab-based software that provides a GUI framework for conducting graph analysis with MR data. It integrates the Brain Connectivity Toolbox, REX toolbox, BrainNet Viewer, with original code for additional graph analyses and comparing networks between groups. <br />
 <br />
Using GAT, you can analyze structural/functional networks constructed from VBM, Free Surfer data, resting state/task fMRI, DTI, MRS, and behavioral data. GAT provides non-parametric statistics for comparing small-world parameters, regional topology, hubs, network resilience, and modularity between groups. <br />
<br />
Please cite the following paper if you are using GAT: <br />
Hosseini et al (2012). GAT: a graph-theoretical analysis toolbox for analyzing between-group differences in large-scale structural and functional brain networks. PLoS ONE 7(7), e40709. <br />
<br />
Please subscribe to GAT mailing list at <br />
<a href="https://mailman.stanford.edu/mailman/listinfo/gat_user_forum" target="_new">https://mailman.stanford.edu/mailman/lis...</a> to get updates and share questions. <br />
<br />
Contact: Hadi Hosseini (hosseiny (at) stanford.edu) </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 14, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/gat/">http://www.nitrc.org/projects/gat/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Stanford University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="gazereader" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="GazeReader">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=gazereader" rel="bookmark" name="gazereader">GazeReader</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=eye-tracking" rel="tag">Eye Tracking</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> GazeReader is a toolbox for a point-process derived GLM analysis of eye tracking data in Matlab (Mathworks, Nattick, MA). It was developed with the following goals in mind:<br />
<br />
1. Flexible yet well ordered integration of the steps involved in linear model specification, fitting, selection and review.<br />
2. Ease of use and efficiency in specifying and comparing numerous alternate models through both a graphical interface and Matlab scripting.<br />
3. Simple error checking and review of model fitting and procedures.<br />
4. The ability to display model data side by side with raw eye-tracking data along with any appropriate stimulus related background images in order to facilitate interpretation.<br />
5.Efficiency of numerical model fitting. <br />
<br />
 Data loading, model specification, fitting and review are organized into a sequence of events, each of which is handled by a separate module in the toolbox. The graphical interface was created using the Matlab graphical user interface development environment (GUIDE). 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=885&screenshot_id=850"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=885&screenshot_id=850" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 30, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/gazereader/">http://www.nitrc.org/projects/gazereader/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Iowa <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="g.bsanalyze" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="g.BSanalyze">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=g.bsanalyze" rel="bookmark" name="g.bsanalyze">g.BSanalyze</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=event-related-potential" rel="tag">Event Related Potential</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=spectral-analysis" rel="tag">Spectral Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> g.BSanalyze - gtec's Biosignal Analysis Software, is an interactive environment for multimodal biosignal data processing and analysis in the fields of clinical research and life sciences. It is the most comprehensive package to analyze non-invasive and invasive brain-, heart- and muscle-functions and dysfunctions. It includes many functions such as support vector machines, event-related ECG, support for P300 and SSVEP/SSSEP BCIs, zero class detection for BCIs, compressed spectral array, minimum energy, and more!<br />
<br />
g.BSanalyze consists of a base version for data import, visualization, transformation and pre-processing and has several dedicated toolboxes. <br />
<br />
The package comes with many sample biosignal data-sets, including P300, SSVEP, motor imagery, CSP BCIs, Tilt-Table, EPs, multi-unit activity, CFM, and ERD/ERS. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=743&screenshot_id=748"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=743&screenshot_id=748" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 14, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>g.tec Licence</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.gtec.at/Products/Software/g.BSanalyze-Specs-Features">http://www.gtec.at/Products/Software/g.BSanalyze-Specs-Features</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>g.tec medical engineering GmbH <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ASCII, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ASCII, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="gcca" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="gCCA">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=gcca" rel="bookmark" name="gcca">gCCA</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=multivariate-analysis" rel="tag">Multivariate Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> gCCA is a multivariate method for fMRI data analysis based on generalized canonical correlation analysis (gCCA) to maximize SPM reproducibility without adopting any model for the hemodynamic response or other temporal brain responses. The underlying assumption is that there are multiple subjects that share an unknown spatial response (or spatial map) to the common experimental manipulation but may show different temporal responses to external stimulation. For each subject, our gCCA explores a broad range of temporal responses in fMRI time-series space while maximizing the mean of correlation coefficients between the pair-wise spatial maps of the subjects.<br />
<br />
[1] Afshin-Pour, B., et. al (2012). Enhancing reproducibility of fMRI statistical maps using generalized canonical correlation analysis in NPAIRS framework. NeuroImage, 60(4), 1970-1981.<br />
<br />
[2] Afshin-Pour, et. al (2014). Evaluation of spatio-temporal decomposition techniques for group analysis of fMRI resting state data sets. NeuroImage, 87, 363-382. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 14, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://babakafshinpour.com/wp-content/uploads/2014/05/gCCA-14-05-2014.rar">http://babakafshinpour.com/wp-content/uploads/2014/05/gCCA-14-05-2014.rar</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Rotman research institute <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="generalized-covariance-analysis-(gcva)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Generalized Covariance Analysis (gCVA)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=generalized-covariance-analysis-(gcva)" rel="bookmark" name="generalized-covariance-analysis-(gcva)">Generalized Covariance Analysis (gCVA)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Generalized Covariance Analysis: a platform for any PCA-based analysis on functional neuroimaging data (PET and fMRI). <br />
<br />
Includes:<br />
<br />
- Ordinal Trend Canonical Variance Analysis for parametric designs<br />
(C. Habeck et al. A New Approach to Spatial Covariance Modeling of Functional Brain Imaging Data: Ordinal Trend Analysis. Neural Computation 2005; 17: 1602-1645)<br />
<br />
- Partial Least Squares for any design matrix<br />
<br />
- Subprofile Scaling Model for cross-sectional designs<br />
(JR. Moeller, Strother SC. A regional covariance approach to the analysis of functional patterns in positron emission tomographic data.J Cereb Blood Flow Metab. 1991 Mar;11(2):A121-35.) </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 24, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/gcva_pca/">http://www.nitrc.org/projects/gcva_pca/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="generalized-ppi-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Generalized PPI Toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=generalized-ppi-toolbox" rel="bookmark" name="generalized-ppi-toolbox">Generalized PPI Toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> An automated toolbox for a generalized form of psychophysiological interactions for SPM and FSFAST.<br />
<br />
The automated toolbox can do the following:<br />
(a1) produce identical results to the current implementation in SPM<br />
(a2) use the current implementation of PPI in SPM but using the regional mean instead of the eigenvariate<br />
(a3) uses a generalized form that allows a PPI for each task to be in the same model using either the regional mean of eigenvariate<br />
(b) creates the model using the output of one of the (a) options and the first level design<br />
(c) estimates the model (/results directory)<br />
(d) computes the contrasts specified<br />
<br />
When using the toolbox, please cite the following paper:<br />
McLaren, DG, Ries, ML, Xu, G, Johnson, SC. A Generalized Form of Context-Dependent Psychophysiological Interactions (gPPI): A Comparison to Standard Approaches. NeuroImage (in press). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 7, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Personal License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/gppi/">http://www.nitrc.org/projects/gppi/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Wisconsin Madison <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="generation-r-pediatric-mri-resources" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Generation R Pediatric MRI Resources">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=generation-r-pediatric-mri-resources" rel="bookmark" name="generation-r-pediatric-mri-resources">Generation R Pediatric MRI Resources</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Generation R (GenR) Pediatric MRI Resources: Pediatric MRI experiences unique challenges with respect to spatial normalization/registration. An average, age-appropriate T1-weighted image, constructed from 130 typically developing children ages 6-to-10, is available.<br />
<br />
Further, a set of 25 resting-state ICA components are made available. The components were generated from 536 typically developing children, ages 6-to-10 years old. The components are the result of a meta-ICA which combined the output of 500 different ICA analyses where 50 subjects were randomly sub-sampled.<br />
<br />
Both of these resources are described in detail the following publication:<br />
<br />
<a href="http://www.ncbi.nlm.nih.gov/pubmed/27417416" target="_new">http://www.ncbi.nlm.nih.gov/pubmed/27417...</a><br />
<br />
<br />
Contents:<br />
<br />
GenR130 T1-weighted template. 1mm and 2mm resolution brain images and brain masks.<br />
<br />
GenR25_metaICA resting state components. 25 Resting state components in 3mm and 2mm resolution. These are available in the GenR130 space (space in which original ICAs and metaICA was conducted), and also in MNI152 space. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 25, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/genr/">http://www.nitrc.org/projects/genr/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Erasmus MC - Sophia Children's Hospital <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="genesis-neural-simulator" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="GENESIS neural simulator">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=genesis-neural-simulator" rel="bookmark" name="genesis-neural-simulator">GENESIS neural simulator</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> GENESIS (the GEneral NEural SImulation System) is a software platform for the simulation of neural systems ranging from subcellular components and biochemical reactions to complex models of single neurons, large networks, and systems-level processes.<br />
<br />
Features:<br />
<br />
 Object oriented modeling paradigm facilitates:<br />
 - modification, reuse, and exchange of models or model components<br />
 - extension of simulator functionality by adding new classes or commands<br />
 &quot;hsolve&quot; solver object allows fast implicit matrix methods and delivery of spike events, while preserving the illusion of separate objects<br />
 From the outset GENESIS was designed for network modeling and parallelism<br />
 Powerful scripting language allows custom-scripted simulation GUIs and specification of large network models with only a few lines of code.<br />
<br />
Website:<br />
<br />
<a href="http://genesis-sim.org/GENESIS/" target="_new">http://genesis-sim.org/GENESIS/</a> - the stable, &quot;classic&quot; GENESIS 2 simulator<br />
<br />
<a href="http://genesis-sim.org" target="_new">http://genesis-sim.org</a> - GENESIS 3 (G-3), a 21st century neural simulator under development 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=859&screenshot_id=840"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=859&screenshot_id=840" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 14, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Lesser General Public License (LGPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://genesis-sim.org">http://genesis-sim.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>GENESIS Developers Group <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="gengen" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="GenGen">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=gengen" rel="bookmark" name="gengen">GenGen</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=pathways-&-networks" rel="tag">Pathways & Networks</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=snp's-&-genes" rel="tag">Snp'S & Genes</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> GenGen is a suite of free software tools to facilitate the analysis of high-throughput genomics data sets. The package is currently a work-in-progress and infrequently updated. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 1, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://github.com/WGLab/GenGen">https://github.com/WGLab/GenGen</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Southern California <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="gestr" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="GesTr">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=gestr" rel="bookmark" name="gestr">GesTr</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> GesTr is a cross platform, open source gesture tracking program. You launch it from the web, and use it to streamline the way you communicate with the computer. It allows for a more natural method of issuing commands than with keyboard shortcuts or GUI buttons.<br />
<br />
GesTr supports simple XML files to customize recognized gestures and their corresponding actions.<br />
<br />
GesTr also has experimental support for the Wii Remote used with an infrared pen as an alternative input device. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 6, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Lesser General Public License (LGPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/gestr/">http://www.nitrc.org/projects/gestr/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="giant:-genome-scale-integrated-analysis-of-gene-networks-in-tissues" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="GIANT: Genome-scale Integrated Analysis of gene Networks in Tissues">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=giant:-genome-scale-integrated-analysis-of-gene-networks-in-tissues" rel="bookmark" name="giant:-genome-scale-integrated-analysis-of-gene-networks-in-tissues">GIANT: Genome-scale Integrated Analysis of gene Networks in Tissues</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> GIANT leverages a tissue-specific gold standard to automatically up-weight datasets relevant to a tissue from a large data compendium of diverse tissues and cell-types. The resulting functional networks accurately capture tissue-specific functional interactions. Beyond questions pertaining to the role of single genes in single tissues, GIANT also enables examination of changes in gene function across tissues on a broad scale. Users can compare a gene's functional interaction in different tissues by selecting the relevant tissues in the dropdown menu. GIANT can effectively reprioritize functional associations from a genome-wide association study (GWAS) and potentially identify additional disease-associated genes. The approach, named NetWAS, can be applied to any GWAS study, and does not require that the phenotype or disease have any known associated genes. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 6, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/giant/">http://www.nitrc.org/projects/giant/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="gibbsconnectome" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Gibbsconnectome">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=gibbsconnectome" rel="bookmark" name="gibbsconnectome">Gibbsconnectome</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Gibbs' connectome is a set of ready-processed high-dimensional structural and functional connectivity matrices and group fibertract information of 169 subjects. The data is based on the openly available NKI enhanced Rockland sample (<a href="http://fcon_1000.projects.nitrc.org/indi/enhanced/" target="_new">http://fcon_1000.projects.nitrc.org/indi...</a>) which represents a state-of-the-art community sample. To estimate a group connectome of fiber tracts, the Gibbs' tracking algorithm has been applied to each subjects dMRI data. Fiber tracts have been normalized into standard space using DARTEL.<br />
Explicit methodology of the data set is described here:<br />
<a href="http://www.ncbi.nlm.nih.gov/pubmed/26327244" target="_new">http://www.ncbi.nlm.nih.gov/pubmed/26327...</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=987&screenshot_id=958"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=987&screenshot_id=958" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 8, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU GPL v2</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/gibbsconnectome/">http://www.nitrc.org/projects/gibbsconnectome/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Charité University Medicine <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="gifti" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="GIFTI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=gifti" rel="bookmark" name="gifti">GIFTI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-information-resource" rel="tag">Other Information Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Geometry format under the Neuroimaging Informatics Technology Initiative (NIfTI). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 14, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/gifti/">http://www.nitrc.org/projects/gifti/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>GIfTI</td></tr>
			<tr class="even"><td>Output Formats</td><td>GIfTI</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="gimias" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="GIMIAS">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=gimias" rel="bookmark" name="gimias">GIMIAS</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=application" rel="tag">Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=platform-or-development-environment" rel="tag">Platform Or Development Environment</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> GIMIAS is a workflow-oriented environment focused on biomedical image computing and simulation. The open source framework is extensible through plug-ins and is focused on building research and clinical software prototypes. Gimias has been used to develop clinical prototypes in the fields of cardiac imaging and simulation, angiography imaging and simulation, and neurology. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=410&screenshot_id=285"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=410&screenshot_id=285" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 19, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.gimias.org/">http://www.gimias.org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>POSIX/UNIX-like<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, NIfTI-1, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, NIfTI-1, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="gimme-(group-iterative-multiple-model-estimation)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="GIMME (Group Iterative Multiple Model Estimation)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=gimme-(group-iterative-multiple-model-estimation)" rel="bookmark" name="gimme-(group-iterative-multiple-model-estimation)">GIMME (Group Iterative Multiple Model Estimation)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=connectivity-analysis" rel="tag">Connectivity Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=modeling" rel="tag">Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=multivariate-analysis" rel="tag">Multivariate Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=time-domain-analysis" rel="tag">Time Domain Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> **** We have transitioned GIMME completely into R. The R package is maintained. **** <br />
Group Iterative Multiple Model Estimation (GIMME) is a free Matlab toolbox for directed functional connectivity analysis of the fMRI BOLD signal from predefined regions of interest. GIMME reliably recovers the true structure of connections and estimates the weights attributed to each connection (equivalently termed “path” or “adjacency”). Importantly, patterns are obtained at both the group and individual levels. GIMME is one of the few approaches that accommodates heterogeneity in brain connectivity patterns across individuals within a sample. Req: Matlab, Lisrel, preproccesing toolbox/software. Gates, K. M. &amp; Molenaar, P. C. M. (2012). Group search algorithm recovers effective connectivity maps for individuals in homogeneous and heterogeneous samples, NeuroImage, Volume 63, Issue 1(15), 310-319, <a href="http://dx.doi.org/10.1016/j.neuroimage.2012.06.026" target="_new">http://dx.doi.org/10.1016/j.neuroimage.2...</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=792&screenshot_id=720"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=792&screenshot_id=720" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 28, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/gimme/">http://www.nitrc.org/projects/gimme/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina; The Pennsylvania State University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="glirt" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="GLIRT">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=glirt" rel="bookmark" name="glirt">GLIRT</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=intersubject" rel="tag">Intersubject</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=motion-analysis" rel="tag">Motion Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> GLIRT (Groupwise and Longitudinal Image Registration Toolbox) provides solutions for both groupwise registration and longitudinal registration, which are the necessary steps for many brain-related applications.<br />
<br />
Specifically, groupwise registration is important for unbiased analysis of a large set of MR brain images. Therefore, in this software package, we have included two of our recently-developed groupwise registration algorithms: 1) Improved unbiased groupwise registration guided with the sharp group-mean image, and 2) Hierarchical feature-based groupwise registration with implicit template (Groupwise-HAMMER for short). <br />
<br />
On the other hand, we also included our recently-developed groupwise longitudinal registration algorithm that aligns not only the longitudinal image sequence for each subject, but also align all longitudinal image sequences of all subjects to the common space simultaneously. <br />
<br />
This software package was developed in the IDEA group at UNC-Chapel Hill ( <a href="http://bric.unc.edu/ideagroup" target="_new">http://bric.unc.edu/ideagroup</a> ). 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=474&screenshot_id=351"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=474&screenshot_id=351" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 2, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/glirt/">http://www.nitrc.org/projects/glirt/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="glmdenoise:-a-fast,-automated-technique-for-denoising-task-based-fmri-data" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="GLMdenoise: a fast, automated technique for denoising task-based fMRI data">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=glmdenoise:-a-fast,-automated-technique-for-denoising-task-based-fmri-data" rel="bookmark" name="glmdenoise:-a-fast,-automated-technique-for-denoising-task-based-fmri-data">GLMdenoise: a fast, automated technique for denoising task-based fMRI data</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> GLMdenoise is a MATLAB toolbox for denoising task-based fMRI data. The basic idea is to derive noise regressors from voxels unrelated to the experimental paradigm and to use these regressors in a general linear model (GLM) analysis of the data. The technique is fast and automated and general: it requires only a design matrix indicating the experimental design and an fMRI dataset. In the published paper (Kay et al., Frontiers in Neuroscience, 2013), it is shown that GLMdenoise outperforms other popular denoising methods, such as using motion parameters as noise regressors, ICA-based denoising, and RETROICOR/RVHRCOR. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 10, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://kendrickkay.net/GLMdenoise/">http://kendrickkay.net/GLMdenoise/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="gmac:-a-matlab-toolbox-for-spectral-granger-causality-analysis-of-fmri-data" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="GMAC: A Matlab toolbox for spectral Granger causality analysis of fMRI data">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=gmac:-a-matlab-toolbox-for-spectral-granger-causality-analysis-of-fmri-data" rel="bookmark" name="gmac:-a-matlab-toolbox-for-spectral-granger-causality-analysis-of-fmri-data">GMAC: A Matlab toolbox for spectral Granger causality analysis of fMRI data</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=connectivity-analysis" rel="tag">Connectivity Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The open-source software toolbox GMAC (Granger Multivariate Autoregressive Connectivity) implemented multivariate spectral Granger Causality Analysis for studying brain connectivity using fMRI data. Available features are: fMRI data importing, network nodes definition, time series preprocessing, multivariate autoregressive modeling, spectral Granger causality indexes estimation, statistical significance assessment using surrogate data, network analysis and visualization of connectivity results. All functions are integrated into a graphical user interface developed in Matlab environment. <br />
Dependencies: Matlab, BIOSIG, SPM, MarsBar.<br />
<br />
GMAC documentation: Tana MG, Sclocco R, Bianchi AM (2012) GMAC: a Matlab toolbox for spectral Granger causality analysis of fMRI data. Computers in Biology and Medicine 42: 943-956 (<a href="http://dx.doi.org/10.1016/j.compbiomed.2012.07.003" target="_new">http://dx.doi.org/10.1016/j.compbiomed.2...</a>) 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=660&screenshot_id=578"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=660&screenshot_id=578" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 7, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/gmac_2012/">http://www.nitrc.org/projects/gmac_2012/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Politecnico di Milano <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="gpu-based-affine-registration" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="GPU based affine registration">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=gpu-based-affine-registration" rel="bookmark" name="gpu-based-affine-registration">GPU based affine registration</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This tool can be used as a command line module with 3D Slicer (version 3 and above) for the affine registration of image volumes. The registration toolbox has 2 options: 1) a Mutual Information based registration, 2) a Sum-of-Square differences registration method. The final output is in the same space as the fixed image. You do require to have CUDA v2.2 or greater installed on your system with atleast 256MB Nvidia GPU memmory card. All operating systems are supported, but take a look at the CMakeLists.txt file for how to compile for you system. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 9, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GPL-Style Open Source</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/gpu-areg/">http://www.nitrc.org/projects/gpu-areg/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="gradient-non-linearity-unwarping-tool" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Gradient Non-linearity Unwarping Tool">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=gradient-non-linearity-unwarping-tool" rel="bookmark" name="gradient-non-linearity-unwarping-tool">Gradient Non-linearity Unwarping Tool</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=artifact-removal" rel="tag">Artifact Removal</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Brain Morphometry testbed of the BIRN provides a tool to correct gradient non-linearity distortions in MR structural images. This correction improves test-retest reproducibility crucial for multi-site studies. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 21, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BIRN License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://nbirn.net/tools/gradient_non_linearity/index.shtm">http://nbirn.net/tools/gradient_non_linearity/index.shtm</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>DICOM, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>DICOM, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="grantees-meeting-for-nitrc" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Grantees Meeting for NITRC">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=grantees-meeting-for-nitrc" rel="bookmark" name="grantees-meeting-for-nitrc">Grantees Meeting for NITRC</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-ke" rel="tag">Other Ke</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This project is meant for planning the NITRC Grantee meetings. The meetings introduce NITRC participants to one another, promote discussion of common interests, and identify opportunities for collaboration and interoperability. <br />
<br />
Plans for the 2009 Grantee Meeting are underway. Click on the MediaWiki link (in gray menu on left) to learn more. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 7, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/nihgrantees/">http://www.nitrc.org/projects/nihgrantees/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="graph-theory-glm-(gtg)-matlab-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Graph Theory GLM (GTG) MATLAB Toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=graph-theory-glm-(gtg)-matlab-toolbox" rel="bookmark" name="graph-theory-glm-(gtg)-matlab-toolbox">Graph Theory GLM (GTG) MATLAB Toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=connectivity-analysis" rel="tag">Connectivity Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This MATLAB toolbox calculates &amp; runs a GLM on graph theory properties derived from brain networks. The GLM accepts continuous &amp; categorical between-participant predictors &amp; categorical within-participant predictors. Significance is determined via non-parametric permutation tests, including correction for multiple comparisons. Both fully connected &amp; thresholded networks are tested.<br />
<br />
The toolbox also provides a data processing path for resting state &amp; task fMRI data. Options for partialing nuisance signals include: local &amp; total white matter signal (Jo et al., 2013), PCA of white matter/ventricular signal (Muschelli et al., 2014), Saad et al. (2013)'s GCOR, &amp; Chen et al. (2012)’s GNI. In addition, Power et al. (2014)'s motion scrubbing method &amp; Patel et al. (2014)'s WaveletDespike are available.<br />
<br />
Please sign up for the mailing list, as this is where we announce new releases. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=802&screenshot_id=732"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=802&screenshot_id=732" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 12, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://sites.bu.edu/metalab/tools/">http://sites.bu.edu/metalab/tools/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="graphtools" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Graphtools">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=graphtools" rel="bookmark" name="graphtools">Graphtools</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Graphtools is a set of MATLAB scripts for analysis of networks derived from neuroimaging data. Some of these scripts are entirely original, while some are adapted (or just copied) from the Brain Connectivity Toolbox (<a href="https://sites.google.com/a/brain-connectivity-toolbox.net/bct" target="_new">https://sites.google.com/a/brain-connect...</a>)<br />
<br />
The source code is available via git:<br />
<br />
git clone <a href="ssh://{user}@www.nitrc.org/home/groups/graphtools/git" target="_new">ssh://{user}@www.nitrc.org/home/groups/g...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 30, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/graphtools/">http://www.nitrc.org/projects/graphtools/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Washington University School of Medicine <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="graphvar:-a-user-friendly-toolbox-for-comprehensive-graph-analyses-of-functional-brain-connectivity" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="GraphVar: A user-friendly toolbox for comprehensive graph analyses of functional brain connectivity">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=graphvar:-a-user-friendly-toolbox-for-comprehensive-graph-analyses-of-functional-brain-connectivity" rel="bookmark" name="graphvar:-a-user-friendly-toolbox-for-comprehensive-graph-analyses-of-functional-brain-connectivity">GraphVar: A user-friendly toolbox for comprehensive graph analyses of functional brain connectivity</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=connectivity-analysis" rel="tag">Connectivity Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> &quot;Journal of Neuroscience Methods&quot; (2015, 2018). <br />
<br />
“GraphVar” is a user-friendly graphical-user-interface based toolbox (MATLAB) for comprehensive graph-theoretical analyses of brain connectivity, including network construction and characterization, statistical analysis (GLM and Machine Learning) on network topological measures, and interactive exploration of results. By combining together features across multiple current toolboxes, such as the Brain Connectivity Toolbox, the Graph Analysis Toolbox, and the Network Based Statistic Toolbox (BCT, Rubinov and Sporns 2010; GAT, Hosseini et al., 2012; NBS, Zalesky et al., 2010), GraphVar represents a comprehensive collection of graph analysis routines for functional neuroimaging researchers. GraphVar offers an interactive viewer that allows intuitive exploration of statistical results. Results can easily be exported and reloaded. The program entails a detailed manual that includes usage instructions and a description of all the implemented functions. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=844&screenshot_id=804"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=844&screenshot_id=804" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 10, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/graphvar/">http://www.nitrc.org/projects/graphvar/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Charité Universitätsmedizin, Berlin <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="gretna" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="GRETNA">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=gretna" rel="bookmark" name="gretna">GRETNA</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> GRETNA is a graph theoretical network analysis toolbox which allows researchers to perform comprehensive analysis on the topology of brain connectome by integrating the most of network measures studied in current neuroscience field. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 12, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/gretna/">http://www.nitrc.org/projects/gretna/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Beijing Normal University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="group-ica-toolbox-(gift-and-eegift)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Group ICA Toolbox (GIFT and EEGIFT)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=group-ica-toolbox-(gift-and-eegift)" rel="bookmark" name="group-ica-toolbox-(gift-and-eegift)">Group ICA Toolbox (GIFT and EEGIFT)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=artifact-removal" rel="tag">Artifact Removal</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=independent-component-analysis" rel="tag">Independent Component Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=linear" rel="tag">Linear</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=principal-component-analysis" rel="tag">Principal Component Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Group ICA Toolbox is a MATLAB toolbox which implements multiple algorithms for independent component analysis of group magneto resonance imaging data (GIFT) and electro encephalogram data (EEGIFT) to group studies. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 4, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://icatb.sourceforge.net/groupica.htm">http://icatb.sourceforge.net/groupica.htm</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>SunOS/Solaris<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="group-information-guided-ica" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Group Information Guided ICA">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=group-information-guided-ica" rel="bookmark" name="group-information-guided-ica">Group Information Guided ICA</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The toolbox is for group-information guided ICA (GIG-ICA). In GIG-ICA, group information captured by standard Independent Component Analysis (ICA) on the group level is used as guidance to compute individual subject specific Independent Components (ICs) using a multi-objective optimization strategy. For computing subject specific ICs, GIG-ICA is applicable to subjects that are involved or not involved in the computation of the group information. Besides the group ICs, group information captured from other imaging modalities and meta analysis could be used as the guidance in GIG-ICA too. <br />
<br />
References:<br />
Y. Du, Y. Fan. Group information guided ICA for fMRI data analysis. Neuroimage. 2013 Apr 1;69:157-97. doi: 10.1016/j.neuroimage.2012.11.008. Epub 2012 Nov 27.<br />
Y. Du, Y. Fan. Group information guided ICA for analysis of multi-subject fMRI data. 2011, 17th Annual Meeting of the Organization for Human Brain Mapping, Quebec City, Canada. Trainee Abstract Travel Awards, Interactive poster. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 3, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>PAMI License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/gig-ica/">http://www.nitrc.org/projects/gig-ica/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Institute of Automation, Chinese Academy of Sciences <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="group-level-imputation-of-statistic-maps" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Group Level Imputation of Statistic Maps">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=group-level-imputation-of-statistic-maps" rel="bookmark" name="group-level-imputation-of-statistic-maps">Group Level Imputation of Statistic Maps</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Group Level Imputation of Statistic Maps (version 1.0) is a toolkit that performs multiple imputation for group level, single sample t-tests. Whole brain group level statistic maps from fMRI rarely cover the entire brain as a result of missing data. Missingness between subjects in fMRI datasets can result from susceptibility artifacts, bounding box (acquisition parameters), and small differences in post-normalized morphology. The toolkit consists of several interactive command line scripts that guide the user to map the spatial distribution of missing data across contrast images, calculate spatial neighborhood averages that help impute values, perform conventional and multiple imputed t-statistics, save the results to brain maps, and create result tables. The toolkit contains an instruction manual (pdf), two Matlab scripts and one R-Statistics script, which depend on functions defined in the popular SPM toolbox and functions defined in the MICE package for [R]. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=585&screenshot_id=492"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=585&screenshot_id=492" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 13, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/multimpute/">http://www.nitrc.org/projects/multimpute/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Medical University of South Carolina <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="group-sparse-canonical-correlation-analysis" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Group Sparse Canonical Correlation Analysis">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=group-sparse-canonical-correlation-analysis" rel="bookmark" name="group-sparse-canonical-correlation-analysis">Group Sparse Canonical Correlation Analysis</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Group sparse Canonical Correlation Analysis is a method designed to study the mutual relationship between two different types of data. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 13, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BioMed Central Ltd</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/gscca_2013/">http://www.nitrc.org/projects/gscca_2013/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>NSF and NIH <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="groupwise-image-registration" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Groupwise Image Registration">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=groupwise-image-registration" rel="bookmark" name="groupwise-image-registration">Groupwise Image Registration</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Groupwise Image Registration Toolbox<br />
<br />
1. A method for group-wise image registration by pairwisely registering similar images identified using graph theoretic techniques. Particularly, we use sparse coding to estimate image similarity measures among images to be registered, yielding asymmetric, group-wise image similarity measures for each image to others in the group. Based on the asymmetric group-wise image similarity measures among different images, a directed graph is built for learning a manifold of images and identifying a group center image so that all other images can be registered to the center image following the shortest directed paths, each of them decomposing a large deformation into a series of small and anatomically meaningful deformations. <br />
<br />
2. A method for group-wise registration of functional MRI images based on their local functional connectivity patterns. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 14, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>PAMI License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/girt/">http://www.nitrc.org/projects/girt/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Institute of Automation, Chinese Academy of Sciences <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="gsa-snp" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="GSA-SNP">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=gsa-snp" rel="bookmark" name="gsa-snp">GSA-SNP</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> A tool for the gene-set (or pathway) analysis of a genome-wide association study result. It accepts a genome-wide list of SNPs and their association P-values. It summarizes the SNP P-values into nearby genes. The gene-by-gene summary results are then further summarized by gene-sets such as Gene Ontology, KEGG pathways, or user-created gene-sets. Various standardization and statistical tests can be performed and the resulting gene-sets that pass a significance level after multiple-testing correction are reported. The tool is written in Java and is available as a standalone version. It has been published as &quot;Nam, D., Kim, J., Kim, S. Y. &amp; Kim, S. GSA-SNP: a general approach for gene set analysis of polymorphisms. Nucleic Acids Res. 38 (Suppl. 2), W749–W754 (2010). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 28, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://sourceforge.net/projects/gsa-snp/">http://sourceforge.net/projects/gsa-snp/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Soongsil University, Seoul, Korea <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="gtract" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="GTRACT">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=gtract" rel="bookmark" name="gtract">GTRACT</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=morphology" rel="tag">Morphology</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tensor-metric" rel="tag">Tensor Metric</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tractography" rel="tag">Tractography</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NOTE: All new development is being managed in a github repository. Please visit<br />
<br />
<a href="https://github.com/BRAINSia/BRAINSTools" target="_new">https://github.com/BRAINSia/BRAINSTools</a><br />
<br />
GTRACT is a Diffusion Tensor fiber tracking suite that includes streamline tracking tools. The fiber tracking includes a guided tracking tool that integrates apriori information into a streamlines algorithm. This suite of programs is built using the NA-MIC toolkit and uses the Slicer3 execution model framework to define the command line arguments. These tools can be fully integrated with Slicer3 using the module discovery capabilities of Slicer3. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 29, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/vmagnotta/">http://www.nitrc.org/projects/vmagnotta/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS<br>POSIX/UNIX-like<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>DICOM, NIfTI-1, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>DICOM, NIfTI-1, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="gwas3d" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="GWAS3D">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=gwas3d" rel="bookmark" name="gwas3d">GWAS3D</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Main Functions<br />
Identify the most probable functional variants which affect transcriptional regulation;<br />
<br />
Prioritize the leading variants when given a full list of GWAS result;<br />
<br />
Evaluate the deleteriousness of genetic variants affecting the gene regulation when given a list of variants;<br />
<br />
Annotate genetic variant from regulatory perspective. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 4, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License v3</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/gwas3d/">http://www.nitrc.org/projects/gwas3d/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>The University of Hong Kong <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="gwasdb" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="GWASdb">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=gwasdb" rel="bookmark" name="gwasdb">GWASdb</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The database provides following functions: <br />
(i) In addition to all the TASs attained genome-wide significance (P-value &lt; 5 x 10-8), we manually curated the TASs that are marginally significant (P-value &lt; 10-3) by looking into the supplementary materials of each original publication; <br />
(ii) Extensive functional annotations and predictions for those TASs across multiple domains; <br />
(iii) Furthermore, we have manually mapped those TASs by phenotype according to Disease Ontology (DO), Human Phenotype Ontology (HPO) and Medical Subject Headings (MeSH) for easy access. <br />
(iv) Besides the SNP-trait association, we also collect SNP-Drug Response data. <br />
(v) Comprehensive and interactive user interface to facilitate the GWAS research. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 1, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License v3</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/gwasdb/">http://www.nitrc.org/projects/gwasdb/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>HKU <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="gwasrap" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="GWASrap">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=gwasrap" rel="bookmark" name="gwasrap">GWASrap</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> GWASrap uses a one-stop solution, users can quickly fetch very comprehensive annotation when they are browsing the GWAS result in highly interactive Circos-style graph or dynamic Manhattan panel. System can perform independent variant prioritization based on additive effect principle by combining the original statistical value and variant prioritization score. The server also hosts a well-structured and up-to-date repositories which store all of significant TASs about hot cases frequently investigated by current GWAS to satisfy the increasing number of GWA studies. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 1, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License v3</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/gwasrap/">http://www.nitrc.org/projects/gwasrap/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>HKU <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="hadoop-for-data-colocation" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Hadoop for data colocation">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=hadoop-for-data-colocation" rel="bookmark" name="hadoop-for-data-colocation">Hadoop for data colocation</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Data colocation via Hadoop and Hbase custom region split policy. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 19, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/hadoop_2016/">http://www.nitrc.org/projects/hadoop_2016/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Vanderbilt University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="haiko89:-mri-in-vivo-baboon-brain-templates-collection-(papio-anubis)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Haiko89: MRI in vivo Baboon Brain Templates Collection (Papio anubis)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=haiko89:-mri-in-vivo-baboon-brain-templates-collection-(papio-anubis)" rel="bookmark" name="haiko89:-mri-in-vivo-baboon-brain-templates-collection-(papio-anubis)">Haiko89: MRI in vivo Baboon Brain Templates Collection (Papio anubis)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> A collection of open access MRI in vivo brain template images from average baboon brains (Papio anubis):<br />
 <br />
- Symmetric and Asymmetric population-average T1 brain templates from N=89 subjects (Haiko89 &amp; Haiko89sym)<br />
<br />
- Tissue (Grey matter, White matter &amp; CSF) probability maps from symmetric and asymmetric Haiko89 brain templates<br />
<br />
- Sex balanced Brain template created with an equal number of N=31 male and N=31 female images (Haiko62MF)<br />
<br />
- Adult Brain Template created from N=67 adult baboons' (over 7 years old) images (Haiko67A)<br />
<br />
When using these images please cite:<br />
Love, S.A., Marie, D., Roth, M., Lacoste, R., Nazarian, B., Bertello, A., Coulon, O., Anton, J.-L., Meguerditchian, A., 2016. The average baboon brain: MRI templates and tissue probability maps from 89 individuals. NeuroImage. doi:10.1016/j.neuroimage.2016.03.018 </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 2, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/haiko89/">http://www.nitrc.org/projects/haiko89/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Centre National de la Recherche Scientifique (CNRS) / Aix-Marseille University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS<br>Microsoft<br>OS Independent<br>OS/2<br>Other OS<br>POSIX/UNIX-like</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="hammer-and-wml-modules-for-3d-slicer" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Hammer And WML Modules for 3D Slicer">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=hammer-and-wml-modules-for-3d-slicer" rel="bookmark" name="hammer-and-wml-modules-for-3d-slicer">Hammer And WML Modules for 3D Slicer</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> HAMMER is an acronym for Hierarchical Attribute Matching Mechanism for Elastic Registration (Dinggang Shen, Christos Davatzikos, HAMMER: Hierarchical Attribute Matching Mechanism for Elastic Registration, IEEE Trans. on Medical Imaging, 21(11):1421-1439, Nov 2002) - an elastic registration algorithm for medical images, matching morphological signatures of images in a hierarchical multi-scale regime. <br />
<br />
White matter lesion (WML) segmentation is a novel multi-spectral WML segmentation protocol via incorporating information from T1-w, T2-w, PD-w and FLAIR MR brain images. (Zhiqiang Lao, Dinggang Shen, Dengfeng Liu, Abbas F Jawad, Elias R Melhem, Lenore J Launer, Nick R Bryan, Christos Davatzikos, Computer-Assisted Segmentation of White Matter Lesions in 3D MR images, Using Pattern Recognition, Academic Radiology, 15(3):300-313, March 2008). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 23, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>3D Slicer License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/hammerwml/">http://www.nitrc.org/projects/hammerwml/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="hammer:-deformable-registration" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="HAMMER: Deformable Registration">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=hammer:-deformable-registration" rel="bookmark" name="hammer:-deformable-registration">HAMMER: Deformable Registration</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The HAMMER package performs high-dimensional warping of brain images. Standard voxel-based analysis can be applied to these tissue density maps, in order to examine regional volumetrics, effects of disease, or correlations with clinical measurements. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 4, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>SBIA License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://www.med.unc.edu/bric/ideagroup/free-softwares/fast-hammer">https://www.med.unc.edu/bric/ideagroup/free-softwares/fast-hammer</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="hammer-suite" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="HAMMER Suite">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=hammer-suite" rel="bookmark" name="hammer-suite">HAMMER Suite</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The tool is a GUI for a complete processing pipeline of brain MR images. It provides functions on skull-stripping, cerebellum removal, tissue segmentation, and HAMMER registration. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 1, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/hammer_suite/">http://www.nitrc.org/projects/hammer_suite/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="hbm-hackathon" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="HBM Hackathon">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=hbm-hackathon" rel="bookmark" name="hbm-hackathon">HBM Hackathon</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=community" rel="tag">Community</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=websites" rel="tag">Websites</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Aloha! Registration is now open for the 3rd Annual OHBM Hackathon taking place at the Hawaii Convention Center in Hawaii on the beautiful island of Oahu from June 12-14, 2015.<br />
<br />
During this three day event, the first two days will focus on collaborative and open neuroscience projects in data analysis and methods development. See brainhack.org for examples of previous projects and to submit your project ideas!<br />
<br />
The third and final day of the event will feature a FREE morning course on “Brain Hacking 101” where participants will be introduced to brain imaging as a data science. The afternoon will feature a session freely available to all OHBM attendees on the hackathon project outcomes.<br />
<br />
The spirit of the Hackathon will continue into the OHBM Annual Meeting, June 14-18 in Honolulu through an on-site collaboration space available throughout the conference.<br />
<br />
Dates: June 12-14, 2015<br />
Location: Hawaii Convention Center, Honolulu, HI<br />
OHBM 2015 website<br />
Project website: <a href="http://www.brainhack.org" target="_new">http://www.brainhack.org</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=715&screenshot_id=919"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=715&screenshot_id=919" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 6, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://brainhack.org/hackathon-2015/">http://brainhack.org/hackathon-2015/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>OHBM <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="hcp-harvard/mgh-ucla" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="HCP Harvard/MGH-UCLA">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=hcp-harvard/mgh-ucla" rel="bookmark" name="hcp-harvard/mgh-ucla">HCP Harvard/MGH-UCLA</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=diffusion-spectrum" rel="tag">Diffusion Spectrum</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=eeg" rel="tag">Eeg</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=fiber-tracking" rel="tag">Fiber Tracking</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=meg" rel="tag">Meg</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=q-ball" rel="tag">Q-Ball</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quantification" rel="tag">Quantification</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=scanner" rel="tag">Scanner</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=shape-analysis" rel="tag">Shape Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=spatial-transformation" rel="tag">Spatial Transformation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tensor-metric" rel="tag">Tensor Metric</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=web-resource" rel="tag">Web Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ***THE HUMAN CONNECTOME PROJECT Harvard/MGH-UCLA***<br />
<br />
The HCP is a multi-center project comprising two distinct consortia (Mass. Gen. Hosp. and UCLA; and Wash. U. and the U. of Minn.) seeking to map white matter fiber pathways in the human brain using leading edge neuroimaging methods, genomics, architectonics, mathematical approaches, informatics, and interactive visualization.<br />
<br />
This NITRC project site is dedicated to linking all listed connectivity analysis tools under one common heading: HCP-related as well as independently created software packages. Please use this site to explore the variety of connectomics software tools available via NITRC as well well as valuable data resources, forthcoming scientific meetings, etc. Scroll down to view the list of useful tools. If you have such tools yourself, please list them with NITRC, and request to link with us!<br />
<br />
See <a href="http://www.humanconnectomeproject.org" target="_new">http://www.humanconnectomeproject.org</a> or <a href="http://www.humanconnectome.org" target="_new">http://www.humanconnectome.org</a> for details. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=418&screenshot_id=296"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=418&screenshot_id=296" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 11, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD/MIT-Style Open Source License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/hcp_mgh-ucla/">http://www.nitrc.org/projects/hcp_mgh-ucla/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows XP</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, NIfTI-1, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, NIfTI-1, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="hcp-wu-minn-consortium" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="HCP WU-Minn Consortium">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=hcp-wu-minn-consortium" rel="bookmark" name="hcp-wu-minn-consortium">HCP WU-Minn Consortium</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=animation" rel="tag">Animation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-data" rel="tag">Atlas Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=diffusion-mr-fiber-tracking" rel="tag">Diffusion Mr Fiber Tracking</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=independent-component-analysis" rel="tag">Independent Component Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=meg" rel="tag">Meg</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=registration" rel="tag">Registration</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=scanner" rel="tag">Scanner</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-rendering" rel="tag">Surface Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=three-dimensional-display" rel="tag">Three Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=time-domain-analysis" rel="tag">Time Domain Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=two-dimensional-display" rel="tag">Two Dimensional Display</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> *THE HUMAN CONNECTOME PROJECT WU-Minn Consortium* is acquiring data and developing analysis pipelines for several modalities of neuroimaging data (rfMRI, tfMRI, dMRI, MEG) plus behavioral and genetic data from 1200 healthy adults (300 twin pairs and siblings). The project will yield invaluable information about brain connectivity, its relationship to behavior, and contributions of genetic and environmental factors to individual differences in brain circuitry. Data is being shared with the scientific community via user-friendly platforms for data mining, analysis, and visualization. Extensive refinements were made for data acquisition and analysis methods in the first phase of the project (2010-2012). Now in production phase (2012-2015), HCP is collecting and releasing data from 400 subjects/year. <a href="http://www.humanconnectome.org" target="_new">http://www.humanconnectome.org</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=456&screenshot_id=461"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=456&screenshot_id=461" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 4, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/hcp_wuminn/">http://www.nitrc.org/projects/hcp_wuminn/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Washington University in St. Louis, University of Minnesota, Oxford University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>CIFTI, DICOM, GIfTI, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>CIFTI, DICOM, GIfTI, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="hdbig" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="HDBIG">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=hdbig" rel="bookmark" name="hdbig">HDBIG</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=genetic-association" rel="tag">Genetic Association</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Recent advances in brain imaging and high throughput genotyping and sequencing techniques enable new approaches to study the influence of genetic variation on brain structure and function. HDBIG is a collection of software tools for high dimensional brain imaging genomics. These tools are designed to perform comprehensive joint analysis of heterogeneous imaging genomics data. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 22, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/hdbig/">http://www.nitrc.org/projects/hdbig/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Indiana University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="hd-neuro-informatics" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="HD Neuro-Informatics">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=hd-neuro-informatics" rel="bookmark" name="hd-neuro-informatics">HD Neuro-Informatics</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Huntington Disease Nueroimaging Initiative (HDNI) is an international effort to establish resources necessary to study the application of neuroimaging measures as (surrogate) biomarkers in HD. The primary aims are to develop and apply software tools, imaging protocols, quality control procedures, data archiving, data distribution, and participation guidelines that will accelerate existing and prospective imaging studies. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 19, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/hdni/">http://www.nitrc.org/projects/hdni/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="hermes" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="HERMES">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=hermes" rel="bookmark" name="hermes">HERMES</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=directed-transfer-analysis" rel="tag">Directed Transfer Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=granger-causality" rel="tag">Granger Causality</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=information-theory" rel="tag">Information Theory</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=partial-directed-coherence" rel="tag">Partial Directed Coherence</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=transfer-entropy-analysis" rel="tag">Transfer Entropy Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Here we present HERMES (<a href="http://hermes.ctb.upm.es" target="_new">http://hermes.ctb.upm.es</a>), a toolbox for the Matlab® environment, which is designed to study functional and effective brain connectivity from neurophysiological data such as multivariate EEG and/or MEG records. It includes also visualization tools and statistical methods to address the problem of multiple comparisons. We believe that this toolbox will be very helpful to all the researchers working in the emerging field of brain connectivity analysis. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=722&screenshot_id=649"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=722&screenshot_id=649" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 28, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://hermes.ctb.upm.es/">http://hermes.ctb.upm.es/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Technical University of Madrid (UPM) <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="hfh_dti_data" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="HFH_DTI_DATA">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=hfh_dti_data" rel="bookmark" name="hfh_dti_data">HFH_DTI_DATA</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The shared dataset consists of T1, FLAIR, and DTI images for 24 Temporal Lobe Epilepsy (TLE) patients along with their clinical description as described in the following:<br />
<br />
1- T1<br />
2- FLAIR<br />
3- Diffusion Tensor Imaging (DTI)<br />
4- Apparent Diffusion Coefficient (ADC)<br />
<br />
The T1 folder includes the following image sets:<br />
1- Skull-Stripped T1 images<br />
2- Manually segmented hippocampi <br />
3- Brain segmented regions by FreeSurfer<br />
<br />
The FLAIR folder includes:<br />
1- Skull-Stripped FLAIR <br />
2- Skull-Stripped T1 co-registered to the FLAIR <br />
3- Manually segmented hippocampi co-registered to the FLAIR <br />
<br />
The DTI folder includes the following image sets: <br />
1- Diffusion Weighted Images (DWIs)<br />
2- Fractional Anisotropy (FA) <br />
3- Apparent Diffusion Coefficient (ADC)<br />
4- Axial and radial diffusivity maps (AD &amp; RD)<br />
5- Geometry Shapes and the Eigen Value images of the tensor<br />
<br />
The ADC folder includes:<br />
1- ADC map images <br />
2- Skull-Stripped T1 co-registered to the ADC map.<br />
3- Manually segmented hippocampi co-registered to the ADC map. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 9, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>HFH_DTI_License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/hfh_dti_unix/">http://www.nitrc.org/projects/hfh_dti_unix/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Henry Ford Health System <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="hfh_dti_tle_tools" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="HFH_DTI_TLE_TOOLS">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=hfh_dti_tle_tools" rel="bookmark" name="hfh_dti_tle_tools">HFH_DTI_TLE_TOOLS</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The shared MATLAB codes contains: <br />
1-Preprocessing Step including: Extraction of DWIs from Dicom images, calculation of tensor and diffusion maps, seed insertion or ROI depiction utility, MATLAB toolbox and instruction for segmentation of brain structures.<br />
2-Uncertainty Analysis of hippocampal volume, FLAIR intensity, and MD; and cingulate and forniceal FA, for Lateralization of temporal lobe epilepsy.<br />
3-DTI-based Response-Driven Modeling of mTLE Laterality using FA in Cingulum, Fornix, and Corpus Callosum.<br />
Please cite the related articles for using the segmentation of Cingulum, Fornix, and Corpus Callosum, uncertainty analysis or DTI-based TLE modeling MATLAB code and its relevant image modalities (DTI, T1, and FLAIR). 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1034&screenshot_id=980"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1034&screenshot_id=980" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 9, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>HFH_DTI_DATA_TOOLS_License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/HFH_DTI_TLE_TOOLS/">http://www.nitrc.org/projects/HFH_DTI_TLE_TOOLS/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Henry Ford Health System <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="hierarchical-event-descriptor-(hed)-tags" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Hierarchical Event Descriptor (HED) Tags">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=hierarchical-event-descriptor-(hed)-tags" rel="bookmark" name="hierarchical-event-descriptor-(hed)-tags">Hierarchical Event Descriptor (HED) Tags</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> HED tags are assigned to event codes (also known as triggers or event numbers) in EEG recordings and allow humans and computers to better understand what these codes represent (e.g. code #5 -&gt; Target detection in an RSVP paradigm).<br />
<br />
Why tags?<br />
<br />
In the same way that we tag a picture on Flicker, or a video clip on Youtube (e.g. cat, cute, funny), we can tag EEG experimental event types used in event-related EEG research. Hierarchical Event Descriptors (HED) is a set of descriptor tags partially adopted from BrainMap/NeuroLex ontologies and organized hierarchically. HED tags can be used to describe many types of EEG experiment events in a uniform, extensible, and machine-readable manner.<br />
<br />
Visit <a href="http://www.hedtags.org" target="_new">http://www.hedtags.org</a> for more information. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=955&screenshot_id=941"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=955&screenshot_id=941" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 16, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MIT</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/hed/">http://www.nitrc.org/projects/hed/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Syntrogi <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="hierarchical-functional-networks-in-resting-state-fmri" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Hierarchical Functional Networks in Resting State fMRI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=hierarchical-functional-networks-in-resting-state-fmri" rel="bookmark" name="hierarchical-functional-networks-in-resting-state-fmri">Hierarchical Functional Networks in Resting State fMRI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> We proposed a fully automatic, iterative reclustering framework in which a small number of spatially large, heterogeneous networks are initially extracted to maximize spatial reproducibility. <br />
Then, the large networks of the brain are iteratively subdivided to create spatially reproducible subnetworks until the overall within-network homogeneity does not increase substantially. <br />
Here, you may download the experimental results generated using the proposed clustering method. The results include a meaningful pattern for spatially hierarchical structure of the brain.<br />
Shams et al., &quot;Automated Iterative Reclustering Framework for Determining Hierarchical Functional Networks in Resting State fMRI&quot;. Human Brain Mapping, Accepted. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=897&screenshot_id=879"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=897&screenshot_id=879" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 2, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU GPL v2</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/iterative_clust/">http://www.nitrc.org/projects/iterative_clust/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Tehran <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="high-dynamic-range-for-local-contrast-enhancement-of-mri" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="High Dynamic Range for Local Contrast Enhancement of MRI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=high-dynamic-range-for-local-contrast-enhancement-of-mri" rel="bookmark" name="high-dynamic-range-for-local-contrast-enhancement-of-mri">High Dynamic Range for Local Contrast Enhancement of MRI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=artifact-removal" rel="tag">Artifact Removal</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=intensity-operations" rel="tag">Intensity Operations</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=three-dimensional-display" rel="tag">Three Dimensional Display</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> An implementation of the high dynamic range (HDR) algorithm for MRI published in Magnetic Resonance in Medicine (MRM) as<br />
<br />
Local Contrast Enhanced MR Images via High Dynamic Range Processing<br />
Chandra SS, Engstrom C, Fripp J, Walker D, Rose S, Ho C, Crozier S. <br />
Magnetic Resonance in Medicine, vol. 80, no. 3, pp. 1206–1218, 2018.<br />
<br />
The HDR MRI algorithm allows for fusing the scans together to form a single omnibus image with enhanced definition of thin, complex anatomical structures, especially in the presence of variable or hyperintense signals. The algorithm takes as input coregistered MR images (preferrably of different contrasts), non-linearly combines them and outputs a single (contrast enhanced) HDR MR image. This library provides a single ITK class, a commandline executable and a GUI application built on the open source [SMILI Biomedical Imaging Framework](<a href="https://smili-project.sourceforge.io/" target="_new">https://smili-project.sourceforge.io/</a>). 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1332&screenshot_id=1242"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1332&screenshot_id=1242" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 13, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/shdr/">http://www.nitrc.org/projects/shdr/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Queenland <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="high-quality-diffusion-weighted-imaging-of-parkinson's-disease" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="High-quality diffusion-weighted imaging of Parkinson's disease">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=high-quality-diffusion-weighted-imaging-of-parkinson's-disease" rel="bookmark" name="high-quality-diffusion-weighted-imaging-of-parkinson's-disease">High-quality diffusion-weighted imaging of Parkinson's disease</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This project contains data and analysis pipelines for a set of 53 subjects in a cross-sectional Parkinson's disease (PD) study. The dataset contains diffusion-weighted images (DWI) of 27 PD patients and 26 age, sex, and education-matched control subjects. The DWIs were acquired with 120 unique gradient directions, b=1000 and b=2500 s/mm2, and isotropic 2.4 mm3 voxels. The acquisition used a twice-refocused spin echo sequence in order to avoid distortions induced by eddy currents.<br />
<br />
Processing scripts for the paper can be found on Github: <a href="https://github.com/CyclotronResearchCentre/parktdi_scripts" target="_new">https://github.com/CyclotronResearchCent...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 17, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Share Alike</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/parktdi/">http://www.nitrc.org/projects/parktdi/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Liège <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="hint:-hierarchical-independent-component-analysis-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="HINT: Hierarchical Independent Component Analysis Toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=hint:-hierarchical-independent-component-analysis-toolbox" rel="bookmark" name="hint:-hierarchical-independent-component-analysis-toolbox">HINT: Hierarchical Independent Component Analysis Toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=connectivity-analysis" rel="tag">Connectivity Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=independent-component-analysis" rel="tag">Independent Component Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=modeling" rel="tag">Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=multivariate-analysis" rel="tag">Multivariate Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=regression" rel="tag">Regression</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Hierarchical Independent Component Analysis Toolbox (HINT) is a Matlab toolbox serving as a user-friendly platform for conducting analyses under the hierarchical ICA framework. At this time, the toolbox implements the hc-ICA technique of Shi and Guo, 2016, with further extensions of this approach, including a longitudinal ICA method, in active development. <br />
<br />
Highlights: <br />
Model based estimation and hypothesis testing of covariate effects<br />
Visualization windows to examine covariate effects, contrasts, and to compare sub-populations<br />
<br />
Shi, R., &amp; Guo, Y. (2016). Investigating differences in brain functional networks using hierarchical covariate-adjusted independent component analysis. The Annals of Applied Statistics, 10(4), 1930–1957. <a href="http://doi.org/10.1214/16-AOAS946" target="_new">http://doi.org/10.1214/16-AOAS946</a><br />
<br />
Mac/Windows/Linux/HPC 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1331&screenshot_id=1247"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1331&screenshot_id=1247" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 12, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MIT</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://github.com/Emory-CBIS/HINT">https://github.com/Emory-CBIS/HINT</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Center for Biomedical Imaging Statistics (CBIS) <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="hippocampalunfolding_manualinitialization" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="HippocampalUnfolding_ManualInitialization">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=hippocampalunfolding_manualinitialization" rel="bookmark" name="hippocampalunfolding_manualinitialization">HippocampalUnfolding_ManualInitialization</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Data here includes manually segmented hippocampal grey matter and dark band, as well as labels used in computational unfolding of grey matter using Laplace equation (1).<br />
<br />
The goal of this work is to improve subfield segmentation and provide a standardized indexing system for hippocampal tissue that accounts for variability in ontological folding of this structure (e.g. digitation and curvature of the uncus in the anterior hippocampus).<br />
<br />
Current atlas is v2.0. <br />
Updates include addition and extension of labels surrounding the hippocampus to ensure easier integration with other neocortical and subcortical atlases, and some changes to labelling of hippocampal grey matter and dark band (mainly in subject V075 who originally had some tracing errors). <br />
<br />
1. DeKraker, J., Ferko, K. M., Lau, J. C., Köhler, S., &amp; Khan, A. R. (2017). Unfolding the hippocampus: An intrinsic coordinate system for subfield segmentations and quantitative mapping. bioRxiv, 146878.<br />
code at <a href="https://github.com/jordandekraker/HippUnfolding" target="_new">https://github.com/jordandekraker/HippUn...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 21, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/hippunfolding/">http://www.nitrc.org/projects/hippunfolding/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Western Ontario <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="hippocampome.org" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Hippocampome.org">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=hippocampome.org" rel="bookmark" name="hippocampome.org">Hippocampome.org</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=knowledge-environment" rel="tag">Knowledge Environment</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Hippocampome is a curated knowledge base of the circuitry of the hippocampus of normal adult, or adolescent, rodents at the mesoscopic level of neuronal types. Knowledge concerning dentate gyrus, CA3, CA2, CA1, subiculum, and entorhinal cortex is distilled from published evidence and is continuously updated as new information becomes available. Each reported neuronal property is documented with a pointer to, and excerpt from, relevant published evidence, such as citation quotes or illustrations. <br />
<br />
The goal of the Hippocampome is dense coverage of available data characterizing neuronal types. The Hippocampome is a public and free resource for the neuroscience community, and the knowledge is presented for user-friendly browsing and searching and for machine-readable downloading. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=872&screenshot_id=843"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=872&screenshot_id=843" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 14, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial Share Alike</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/hippocampome/">http://www.nitrc.org/projects/hippocampome/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>George Mason University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="hippocampus-subregional-masks-based-on-gray-matter-volume-covariance" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="hippocampus subregional masks based on gray matter volume covariance">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=hippocampus-subregional-masks-based-on-gray-matter-volume-covariance" rel="bookmark" name="hippocampus-subregional-masks-based-on-gray-matter-volume-covariance">hippocampus subregional masks based on gray matter volume covariance</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The masks were generated by parcellating the human hippocampus with two independent datasets, each of which consisted of 198 T1-weighted structural magnetic resonance imaging images of healthy young subjects. The method was based on gray matter volume (GMV) covariance, which was quantified by a bivariate voxel-to-voxel linear correlation approach, as well as a multivariate masked independent component analysis approach. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 26</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License v3</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/hippo_gmvc/">http://www.nitrc.org/projects/hippo_gmvc/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of British Columbia (UBC) <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="hi-speed-software-packets" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="HI-SPEED Software Packets">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=hi-speed-software-packets" rel="bookmark" name="hi-speed-software-packets">HI-SPEED Software Packets</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-reconstruction" rel="tag">Image Reconstruction</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=statistical-operation" rel="tag">Statistical Operation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tensor-metric" rel="tag">Tensor Metric</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> HI-SPEED Software Packets contain <br />
(1) unconstrained and constrained nonlinear least squares diffusion tensor estimation techniques, <br />
(2) 2-dimensional and 3-dimensional analytical (Shepp-Logan) magnetic resonance imaging phantoms in both the Fourier and image domains, <br />
(3) techniques for reporting the underlying signal-to-noise ratio in magnetic resonance (MR) images, <br />
(4) Probabilistic Identification and EStimation of NOise (PIESNO)---a technique for identifying noise-only pixels and estimating the underlying noise standard deviation in MR images, and <br />
(5) a signal-transformational technique for breaking the noise floor in MR images. <br />
<br />
Many more computational tools will be shared with users and developers as they become available.<br />
<br />
Download info: <a href="http://sites.google.com/site/hispeedpackets/" target="_new">http://sites.google.com/site/hispeedpack...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 16, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>HI-SPEED Software Packets License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/hispeed/">http://www.nitrc.org/projects/hispeed/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="histolozee---3d-histology-reconstruction-and-co-registration-with-mri" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="HistoloZee - 3D histology reconstruction and co-registration with MRI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=histolozee---3d-histology-reconstruction-and-co-registration-with-mri" rel="bookmark" name="histolozee---3d-histology-reconstruction-and-co-registration-with-mri">HistoloZee - 3D histology reconstruction and co-registration with MRI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=intermodal" rel="tag">Intermodal</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=manual" rel="tag">Manual</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=three-dimensional-display" rel="tag">Three Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=two-dimensional-display" rel="tag">Two Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> HistoloZee is a tool that integrates histology reconstruction, MRI co-registration, and manual segmentation tools in an easy-to-use and intuitive interface. HistoloZee permits real-time interaction with complex and large (multi-GB) histology datasets during the co-registration steps of histology reconstruction. Please see our website (<a href="http://picsl.upenn.edu/software/histolozee/" target="_new">http://picsl.upenn.edu/software/histoloz...</a>) and the series of tutorial videos on YouTube: <a href="https://www.youtube.com/playlist?list=PL68v8FP_IVlg2tCgJXrsO3UHve1q9wLjI" target="_new">https://www.youtube.com/playlist?list=PL...</a><br />
<br />
HistoloZee was developed by Daniel Adler and Paul Yushkevich. The current release of HistoloZee (v.03) runs only under Mac OS X, version 10.7 or later. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1028&screenshot_id=977"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1028&screenshot_id=977" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 23, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>End-User License Agreement for HistoloZee</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://picsl.upenn.edu/software/histolozee/">http://picsl.upenn.edu/software/histolozee/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pennsylvania <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="hitachi2nirs" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Hitachi2nirs">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=hitachi2nirs" rel="bookmark" name="hitachi2nirs">Hitachi2nirs</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=nirs/fnir" rel="tag">Nirs/Fnir</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> A Matlab script to convert the raw .csv Hitachi ETG4000 output file into a .nirs file for use with Homer2. The script also requires a .pos file. This is the output of the polhemus 3D digitiser that we use to record where the optodes are located spatially. I realise that not everyone uses a 3D digitiser so I have included three example .pos files - one for each of the possible optode arrangements of the Hitachi system (either two 3x3 arrays, one 3x5 array or one 4x4 array). If you use a different arrangement or have more probes than us, feel free to get in touch and I may be able to advise on how to create a model .pos file.<br />
<br />
There are two versions of the conversion script:<br />
1. single - this will read in ONE .csv file and ONE .pos file and create ONE .nirs file <br />
2. multi - this will read in a user-specified number of .csv files and ONE .pos file. It will then create one .nirs file for each .csv file that was read in and deposit it in the same directory as that .csv file. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=665&screenshot_id=589"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=665&screenshot_id=589" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 3, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/hitachi2nirs/">http://www.nitrc.org/projects/hitachi2nirs/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Nottingham <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="hitachi-optical-topography-system" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Hitachi Optical Topography System">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=hitachi-optical-topography-system" rel="bookmark" name="hitachi-optical-topography-system">Hitachi Optical Topography System</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=nirs/fnir" rel="tag">Nirs/Fnir</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=scanner" rel="tag">Scanner</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Hitachi Medical Corporation is a leader in fNIRS neuroimaging with more than 10 years of experience. We provide researchers and clinicians with sophisticated All-in-One solutions in the field of neuroscience. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=495&screenshot_id=378"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=495&screenshot_id=378" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 30, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Commercial License: Hitachi</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.hitachi-medical.co.jp/english/">http://www.hitachi-medical.co.jp/english/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Hitachi Medical Corporation <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="homer2" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Homer2">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=homer2" rel="bookmark" name="homer2">Homer2</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> HOMER2 is a set of matlab scripts used for analyzing fNIRS data to obtain estimates and maps of brain activation. This set of scripts has evolved since the early 1990s, first as the Photon Migration Imaging toolbox, then HOMER, and now HOMER2.<br />
<br />
HOMER2 has a familiar GUI interface, but now more easily supports group analyses and re-configuration of the processing stream. Further, it enables users to integrate their own algorithms into the processing stream. Underneath the GUI, all of the processing functions can be accessed at the script level, adding additional flexibility.<br />
<br />
HOMER2 also includes the head and probe registration and brain imaging tool, AtlasViewer.<br />
<br />
Users are encouraged to contribute their own scripts for public dissemination within HOMER2.<br />
<br />
For Documentation and Installation Instructions visit <a href="https://www.nitrc.org/plugins/mwiki/index.php/homer2:MainPage" target="_new">https://www.nitrc.org/plugins/mwiki/inde...</a> . </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 9, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MGH CSRL License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/homer2/">http://www.nitrc.org/projects/homer2/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Boston University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="hsnc" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="HSNC">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=hsnc" rel="bookmark" name="hsnc">HSNC</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> 89 accurate masks of skull-stripped MR images </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 17, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>XNAT License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/hsnc/">http://www.nitrc.org/projects/hsnc/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>fudan university <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="human-connectome-project-(hcp)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Human Connectome Project (HCP)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=human-connectome-project-(hcp)" rel="bookmark" name="human-connectome-project-(hcp)">Human Connectome Project (HCP)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=community" rel="tag">Community</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data-resource" rel="tag">Data Resource</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=federal-funding-resource" rel="tag">Federal Funding Resource</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=hardware" rel="tag">Hardware</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=software" rel="tag">Software</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The NIH Human Connectome Project is an effort to map the neural pathways that underlie human brain function. The purpose of the Project is to acquire and share data about the structural and functional connectivity of the human brain. It will advance the capabilities for imaging and analyzing brain connections, resulting in improved sensitivity, resolution, and utility.<br />
<br />
The NIH Blueprint for Neuroscience (<a href="http://www.neuroscienceblueprint.nih.gov/" target="_new">http://www.neuroscienceblueprint.nih.gov...</a>) has funded two grants that have taken complementary approaches:<br />
<br />
* A consortium led by Washington University in St. Louis and the University of Minnesota is using advanced rfMRI, tfMRI, dMRI and MEG methods to chart brain circuitry in 1200 healthy subjects. <a href="http://www.humanconnectome.org/" target="_new">http://www.humanconnectome.org/</a><br />
<br />
* A consortium led by Massachusetts General Hospital and the University of California at Los Angeles has enabled building a next-generation 3T MR scanner that improves the quality and spatial resolution of brain connectivity data. <a href="http://www.humanconnectomeproject.org/" target="_new">http://www.humanconnectomeproject.org/</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 11, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://www.humanconnectome.org">https://www.humanconnectome.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="human-imaging-database-(hid)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Human Imaging Database (HID)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=human-imaging-database-(hid)" rel="bookmark" name="human-imaging-database-(hid)">Human Imaging Database (HID)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database-application" rel="tag">Database Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other" rel="tag">Other</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-software-resource" rel="tag">Other Software Resource</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-web-resource" rel="tag">Other Web Resource</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Human Imaging Database (HID) is an extensible database management system developed to handle the increasingly large and diverse datasets collected as part of the MBIRN and FBIRN collaboratories and throughout clinical imaging communities at large. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 8, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://nbirn.net/research/function/hid.shtm">http://nbirn.net/research/function/hid.shtm</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="hyperion:-mri-digital-projection-system" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Hyperion: MRI Digital Projection System">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=hyperion:-mri-digital-projection-system" rel="bookmark" name="hyperion:-mri-digital-projection-system">Hyperion: MRI Digital Projection System</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=stimulus-presentation" rel="tag">Stimulus Presentation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Visual presentation is central to almost all fMRI experiments. Previous fMRI display technologies involve compromises on image quality and presentation speed. LCD technology is slow, with low contrast images and imprecise analog VGA. These may be acceptable for tenth of a second resolution displays where image quality and repeatability can be overlooked, but we are pleased to announce these compromises are no longer necessary. The Hyperion MRI Digital Projection System features a native resolution of 1920 x 1080 (1080p) and synchronized image frame rate with continuous monitoring for optimum presentation and timing of fMRI experiments. The system uses Digital Light Processing® (DLP) technology that provides microsecond pixel rise times, outstanding contrast with all-digital fiber optic control that allows you to project crystal clear, sharp images. The Hyperion MRI Digital Projection System has been validated for use in 1.5T, 3T, and 7T MRI environments. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=307&screenshot_id=206"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=307&screenshot_id=206" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 9, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://pstnet.com/products/hyperion/">https://pstnet.com/products/hyperion/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Psychology Software Tools, Inc. <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="iatr" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="IATR">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=iatr" rel="bookmark" name="iatr">IATR</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=knowledge-environment" rel="tag">Knowledge Environment</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=software" rel="tag">Software</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=web-service" rel="tag">Web Service</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=websites" rel="tag">Websites</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> IATR is the Image Analysis Tools Registry. The goal for this site is to provide a centrally available listing of all image analysis tools that are available to the neuroscience community. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 16, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Public Domain</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://iatr.virtualbrain.org">http://iatr.virtualbrain.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="ibeat" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="iBEAT">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=ibeat" rel="bookmark" name="ibeat">iBEAT</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-data" rel="tag">Atlas Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-display" rel="tag">Image Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=registration" rel="tag">Registration</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=warping" rel="tag">Warping</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> iBEAT (Infant Brain Extraction and Analysis Toolbox, previously called LIBRA) is a toolbox with graphical user interfaces for processing infant brain MR images. Longitudinal (or single-time-point) multimodality (including T1, T2, and FA) (or single-modality) data can be processed using the toolbox. Main functions of the software (step by step) include image preprocessing, brain extraction, tissue segmentation and brain labeling. The software is developed by the IDEA group at the University of North Carolina at Chapel Hill, directed by Dr. Dinggang Shen(<a href="mailto:dinggang_shen@med.unc.edu" target="_new">dinggang_shen@med.unc.edu</a>).<br />
<br />
Linux operating system (64 bit) is required. A workstation or server with memory &gt; 8G is recommended for processing many images simultaneously. The graphical user interfaces and overall framework of the software are implemented in MATLAB. The image processing functions are implemented with the combination of C/C++, MATLAB, Perl and Shell languages. Parallelization technologies are used in the software to speed up image processing. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=574&screenshot_id=471"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=574&screenshot_id=471" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 9, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>iBEAT</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/ibeat/">http://www.nitrc.org/projects/ibeat/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina - Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="ibsr" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="IBSR">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=ibsr" rel="bookmark" name="ibsr">IBSR</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=knowledge-environment" rel="tag">Knowledge Environment</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quality-metrics" rel="tag">Quality Metrics</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=test-data" rel="tag">Test Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=web-service" rel="tag">Web Service</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=websites" rel="tag">Websites</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Internet Brain Segmentation Repository (IBSR) provides manually-guided expert segmentation results along with magnetic resonance brain image data. Its purpose is to encourage the evaluation and development of segmentation methods. Please see the MediaWiki for more information. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=48&screenshot_id=662"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=48&screenshot_id=662" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 16, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://www.nitrc.org/projects/ibsr/">https://www.nitrc.org/projects/ibsr/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="ica-components-from-kessler,-et-al,-jama-psychiatry-2016" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ICA Components from Kessler, et al, JAMA Psychiatry 2016">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=ica-components-from-kessler,-et-al,-jama-psychiatry-2016" rel="bookmark" name="ica-components-from-kessler,-et-al,-jama-psychiatry-2016">ICA Components from Kessler, et al, JAMA Psychiatry 2016</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This dataset includes 15 unthresholded ICA connectomic components and the grid parcellation from Kessler et al “Growth Charting Brain Connectivity Networks and the Identification of Attention Impairment in Youth”. This release can be used to extract component expression scores for new datasets (see manuscript for methodological details). These expression scores can then be age-corrected and used to predict attention-related phenotypes. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 15, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Share Alike</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/kessler_jama16/">http://www.nitrc.org/projects/kessler_jama16/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Michigan <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="icn_atlas" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ICN_Atlas">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=icn_atlas" rel="bookmark" name="icn_atlas">ICN_Atlas</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-data" rel="tag">Atlas Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=overlap-metrics" rel="tag">Overlap Metrics</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=statistical-operation" rel="tag">Statistical Operation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volumetric-analysis" rel="tag">Volumetric Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ICN_Atlas is an SPM compatible toolbox that is aimed at facilitating the interpretation of neuroimaging data in the context of intrinsic connectivity networks (ICNs, a.k.a. resting-state networks or RSNs) by describing fMRI maps or other statistical maps in function-oriented, objective and quantitative way using a set of 15 metrics conceived to quantify the degree of ‘engagement’ of ICNs for any given statistical map of interest. Besides the functionally-derived atlasing capabilities (based on the BrainMap ICN data) the toolbox provides anatomy-based atlasing (based on the JHU and Brainnetome Atlas data), as well. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1241&screenshot_id=1156"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1241&screenshot_id=1156" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 14, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial Share Alike</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/icn_atlas/">http://www.nitrc.org/projects/icn_atlas/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Semmelweis University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="idea-lab-brain-image-processing-suite" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="IDeA Lab brain image processing suite">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=idea-lab-brain-image-processing-suite" rel="bookmark" name="idea-lab-brain-image-processing-suite">IDeA Lab brain image processing suite</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-to-template" rel="tag">Image-To-Template</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=intermodal" rel="tag">Intermodal</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=intersubject" rel="tag">Intersubject</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=nonlinear-warp" rel="tag">Nonlinear Warp</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=spline-interpolation" rel="tag">Spline Interpolation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tri-linear" rel="tag">Tri-Linear</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=two-dimensional-display" rel="tag">Two Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volume-measurement" rel="tag">Volume Measurement</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Suite of tools for brain image analysis. Image manipulation, 2D visualization, linear alignment, BBSI, template-based bias correction, skullstrip. GUI Image analysis tools. Now modified to read/write single file nifti (.nii) format. Other packages to be added. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 20, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/idea_lab/">http://www.nitrc.org/projects/idea_lab/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="ielectrodes" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="iElectrodes">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=ielectrodes" rel="bookmark" name="ielectrodes">iElectrodes</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=position" rel="tag">Position</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=simulation" rel="tag">Simulation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-rendering" rel="tag">Surface Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=three-dimensional-display" rel="tag">Three Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=two-dimensional-display" rel="tag">Two Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> iElectrodes is an open source toolbox to semiautomatically localize and label intracranial electrodes in 2-3 minutes. The toolbox precisely localizes subdural grids (ECoG) and depth (SEEG) electrodes using MRI and CT images. <br />
<br />
2D and 3D visualization simplify the comprehension of anatomical locations.<br />
<br />
Also, it is possible to add anatomical information (brain parcellation images) and surfaces from Freesurfer. Data can be exported in multiple formats, including EEGLAB and Fieldtrip.<br />
<br />
Additionally, grid electrodes can be corrected for brain shift or compression effects, and projected to brain surface.<br />
<br />
Planning of depth electrode implantation is fast and easy.<br />
<br />
More information in our paper Blenkmann et al., Front. Neuroinformatics, 2017 <a href="http://journal.frontiersin.org/article/10.3389/fninf.2017.00014/full" target="_new">http://journal.frontiersin.org/article/1...</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1178&screenshot_id=1091"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1178&screenshot_id=1091" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 29, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License v3</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://sourceforge.net/projects/ielectrodes/">https://sourceforge.net/projects/ielectrodes/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Oslo - CONICET Argentina <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="iit-human-brain-atlas-(v.4.1)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="IIT Human Brain Atlas (v.4.1)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=iit-human-brain-atlas-(v.4.1)" rel="bookmark" name="iit-human-brain-atlas-(v.4.1)">IIT Human Brain Atlas (v.4.1)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-data" rel="tag">Atlas Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The IIT Human Brain Atlas (v.4.1) contains anatomical, DTI, HARDI templates, probabilistic gray matter labels, probabilistic connectivity-based white matter labels, and major fiber-bundles of the adult human brain in ICBM-152 space. Artifact-free MRI data from 72 subjects were used in the development of the atlas. All diffusion MRI data collection was conducted using Turboprop, and spatial normalization was accomplished in a population-based fashion. If you use components of the atlas, please reference: Zhang S, Arfanakis K. Evaluation of standardized and study-specific diffusion tensor imaging templates of the adult human brain: Template characteristics, spatial normalization accuracy, and detection of small inter-group FA differences. Neuroimage 2018;172:40-50.<br />
<br />
- A description of the contents of the atlas can be found in &quot;Downloads&quot;.<br />
- Information on HOW TO use the atlas can be found in &quot;Docs&quot;.<br />
- Post questions or experiences with the atlas on the &quot;Forums&quot;.<br />
<br />
Version 4.1 was uploaded on 10/15/15. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=432&screenshot_id=639"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=432&screenshot_id=639" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 20, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>IIT Human Brain Template License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.iit.edu/~mri/Home.html">http://www.iit.edu/~mri/Home.html</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Illinois Institute of Technology <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="imagecite" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ImageCite">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=imagecite" rel="bookmark" name="imagecite">ImageCite</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The ImageCite Consortium is a group of image data host providers banded together with the objective to plan and implement an imaging unique identification (DOI) system across a broad set of imaging data sharing host providers. The consortium will review a proposed scheme of identification and attribution (<a href="https://doi.org/10.3389/fninf.2016.00034" target="_new">https://doi.org/10.3389/fninf.2016.00034</a>) in terms of its suitability and applicability to the needs of the community and feasibility of implementation in the various image hosting systems. Outcomes will be a conceptual plan of what can be accomplished by the repositories in the short term under current, existing development cycles (Phase 0); identification of additional future objectives that can be achieved with additional, potentially coordinated, support; and a ‘white paper’ document that documents this identifier vision that can be circulated to additional resource providers in order to attempt to gain even wider adoption. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 9, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/imagecite/">http://www.nitrc.org/projects/imagecite/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>The ImageCite Consortium <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="imagej" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ImageJ">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=imagej" rel="bookmark" name="imagej">ImageJ</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?tag=image-processing" rel="tag">Image Processing</a>, <a href="https://hbp-hpc-platform.fz-juelich.de/?tag=medical-imaging" rel="tag">Medical Imaging</a></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=application" rel="tag">Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description>
      <a href="http://imagej.net/">ImageJ</a> is an open source image processing program designed for scientific multidimensional images.
<p/><p/>
      ImageJ is highly extensible, with thousands of plugins and scripts for performing a wide variety of tasks, and a large user community.
    <p/>
      <ul>
        <li><b>Open source:</b>	ImageJ is a tool for the scientific community. To maintain transparency, the ImageJ application and its source code will always be freely available.</li>
<li><b>Reproducible:</b>	Powerful tools such as the Script Editor and personal update sites help you develop and share reproducible analysis workflows.</li>
<li><b>Interoperable:</b>	ImageJ is not an island. Use the best tool for the job, including KNIME, ITK, MATLAB, and a multitude of scripting languages.</li>
      </ul>
    </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dezember 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Public Domain</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td>imagej-2.0.0-rc-71</td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://imagej.net/">http://imagej.net/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href="https://github.com/imagej/imagej">https://github.com/imagej/imagej</a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Curtis Rueden
    
     <a href="mailto:(ctrueden@wisc.edu)">(ctrueden@wisc.edu)</a><br>Wayne Rasband
     <a href="mailto:(rasbandw@mail.nih.gov)">(rasbandw@mail.nih.gov)</a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td>Java >= 1.8</td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>BMP, DICOM, FITS, GIF, JPEG, PGM, PNG, TIFF</td></tr>
			<tr class="even"><td>Output Formats</td><td>AVI, BMP, FITS, GIF, JPEG, LUT, PNG, Raw Data, TIFF</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="image-segmentation-based-on-the-local-center-of-mass-computation" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Image Segmentation Based on the Local Center of Mass Computation">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=image-segmentation-based-on-the-local-center-of-mass-computation" rel="bookmark" name="image-segmentation-based-on-the-local-center-of-mass-computation">Image Segmentation Based on the Local Center of Mass Computation</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=center-of-mass" rel="tag">Center Of Mass</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This is a Matlab toolbox for medical image segmentation, which uses an approach based on the local center of mass, as introduced by Aganj et al (Sci Rep, 2018). A short tutorial is included in EXAMPLE.m. If available, a GPU can be used to speed up the segmentation. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1239&screenshot_id=1221"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1239&screenshot_id=1221" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 6, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://www.nitrc.org/projects/seg">https://www.nitrc.org/projects/seg</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Harvard Medical School / Massachusetts General Hospital <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="image-synthesis" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Image Synthesis">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=image-synthesis" rel="bookmark" name="image-synthesis">Image Synthesis</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Image Synthesis Tools are a collection of software tools developed for medical image synthesis of typically magnetic resonance (MR) brain images, however, the approaches have been used to create computed tomography (CT) images from MR input. The goal of image synthesis is to recover MR images with a desired optimal contrast for further processing by either registration or segmentation. Image synthesis can be thought of as a new image restoration technique that recovers images with both the desired tissue contrast and a normalized intensity profile. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 12, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU GPL v2</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/image_synthesis/">http://www.nitrc.org/projects/image_synthesis/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Johns Hopkins University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="imagevis3d" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ImageVis3D">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=imagevis3d" rel="bookmark" name="imagevis3d">ImageVis3D</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Note ImageVis3D development is hosted elsewhere -- see <a href="http://www.imagevis3d.org/" target="_new">http://www.imagevis3d.org/</a><br />
<br />
ImageVis3D is a new volume rendering program developed by the NIH/NCRR Center for Integrative Biomedical Computing (CIBC). The main design goals of ImageVis3D are: simplicity, scalability, and interactivity. Simplicity is achieved with a new user interface that gives an unprecedented level of flexibility (as shown in the images). Scalability and interactivity for ImageVis3D mean that both on a notebook computer as well as on a high end graphics workstation, the user can interactively explore terabyte sized data sets. Finally, the open source nature as well as the strict component-by-component design allow developers not only to extend ImageVis3D itself but also reuse parts of it, such as the rendering core. This rendering core, for instance, is planned to replace the volume rendering subsystems in many applications at the SCI Institute and with our collaborators. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 28, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MIT/X Consortium License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.imagevis3d.org/">http://www.imagevis3d.org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="imcalc:-batch-image-calulations-and-transformations-(spm5/8/12)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="imcalc: Batch image calulations and transformations (SPM5/8/12)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=imcalc:-batch-image-calulations-and-transformations-(spm5/8/12)" rel="bookmark" name="imcalc:-batch-image-calulations-and-transformations-(spm5/8/12)">imcalc: Batch image calulations and transformations (SPM5/8/12)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=statistical-operation" rel="tag">Statistical Operation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ** Updated Nov 2017 ** Image maths and transformations with batch functionality for SPM5, SPM8, or SPM12; features include:<br />
<br />
 (1) Binarize: non-zero voxels, or to specific thresholds<br />
 (2) Pairwise operations: add, subtract, mean, min, max ...<br />
 (3) Create or split cluster images <br />
 (4) Create inter-subject agreement maps<br />
 (5) Statistical transformations: T-to-Z, Z-score calculation<br />
 (6) Create masks based on an image<br />
 (7) Replace zeros / NaNs<br />
 (8) Pad a bounding box with extra voxels<br />
<br />
For **CURRENT VERSION** download and documentation, visit <a href="http://tools.robjellis.net/" target="_new">http://tools.robjellis.net/</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 31, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://tools.robjellis.net/">http://tools.robjellis.net/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>BIDMC / Harvard Medical School <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="imeka-tractography-service" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Imeka Tractography Service">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=imeka-tractography-service" rel="bookmark" name="imeka-tractography-service">Imeka Tractography Service</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tractography" rel="tag">Tractography</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Imeka Tractography is a diffusion MRI service that handles the processing of diffusion data from raw data to structural connectivity, including Quality Assurance. We work with neuroscience researchers who want quality diffusion data reconstruction. These researchers do not necessarily have the knowledge to perform diffusion analysis or don't have the manpower. <br />
<br />
Our service is not a black box solution: we work with the researchers and make sure they can work with the tractography results and are aware of the pitfalls of the techniques by providing training sessions. We have a system to work at a distance as if we were in your lab. We can do high angular resolution (HARDI) reconstruction from DTI data, with as little as 20 gradient directions acquisitions.<br />
<br />
State-of-the-art diffusion MRI is just one click away. Contact us right now to submit your project and have access to structural Connectome research today.<br />
<br />
Visit us at <a href="http://www.imeka.ca" target="_new">http://www.imeka.ca</a> for more details </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 31, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.imeka.ca">http://www.imeka.ca</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Imeka Solutions <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="imod---3d-reconstruction-and-analysis" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="IMOD - 3D Reconstruction and Analysis">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=imod---3d-reconstruction-and-analysis" rel="bookmark" name="imod---3d-reconstruction-and-analysis">IMOD - 3D Reconstruction and Analysis</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> IMOD is a free, cross - platform set of image processing, modeling and display programs used for tomographic reconstruction, reconstruction and segmentation of 3D volumes. IMOD has tools for assembling and aligning data within multiple types and sizes of image stacks, viewing 3D data from any orientation, and modeling and displaying the image files. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=426&screenshot_id=301"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=426&screenshot_id=301" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 29, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/imod/">http://www.nitrc.org/projects/imod/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="incf" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="INCF">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=incf" rel="bookmark" name="incf">INCF</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=community" rel="tag">Community</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=web-service" rel="tag">Web Service</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=websites" rel="tag">Websites</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The INCF is a professional organization devoted to advancing the field of neuroinformatics. One of its aims is to develop an international neuroinformatics infrastructure, which promotes the sharing of data and computing resources to the international research community. A second objective of INCF is to help develop scalable, portable, and extensible applications that can be used by neuroscience laboratories worldwide. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 6, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://www.incf.org/">https://www.incf.org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="incf-neuroimaging-data-sharing" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="INCF Neuroimaging Data Sharing">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=incf-neuroimaging-data-sharing" rel="bookmark" name="incf-neuroimaging-data-sharing">INCF Neuroimaging Data Sharing</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=community" rel="tag">Community</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Neuroscience data, particularly those in neuroinformatics related areas such as neuroimaging and electrophysiology, are associated with a rich set of descriptive information often called metadata. For data archive, storage, sharing and re-use, metadata are of equal importance to primary data, as they define the methods and conditions of data acquisition (such as device characteristics, study/experiment protocol and parameters, behavioral paradigms, and subject/patient information), and statistical procedures. A further challenge for datasharing is the rapidly evolving nature of investigative methods and scientific applications.<br />
<br />
The overall scope of this program is to develop generic standards and tools to facilitate the recording, sharing, and reporting of metadata. It is expected that these efforts will greatly improve upon current practices for archiving and sharing neuroscience data. This group is often refered to as NI-DASH or NIDASH. It has generated the neuroimaging data model called NI-DM. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 25, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://incf.org/programs/datasharing/neuroimaging-task-force">http://incf.org/programs/datasharing/neuroimaging-task-force</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="individual-morphological-connectivity-using-wavelet-transform" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Individual morphological connectivity using wavelet transform">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=individual-morphological-connectivity-using-wavelet-transform" rel="bookmark" name="individual-morphological-connectivity-using-wavelet-transform">Individual morphological connectivity using wavelet transform</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The goal of this research is to build novel morphological connectivity in the single subject level based on anatomical MRI. To this end, we developed a package to compute individual morphological connectivity using wavelet transform. The packge included preprocessing of anatomical MRI, VBM, wavelet transform of 3D MRI datasets, and morphological connectivity computation. In addition, we also planned to release the prepocessed features of individual morphological connectivity. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 24, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Public Domain</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mcwt/">http://www.nitrc.org/projects/mcwt/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Hangzhou Dianzi University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="infant-macaque-mri-and-dti-templates" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Infant Macaque MRI And DTI Templates">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=infant-macaque-mri-and-dti-templates" rel="bookmark" name="infant-macaque-mri-and-dti-templates">Infant Macaque MRI And DTI Templates</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This is a multi-mode (T1w and DTI) brain template of monkeys during late infancy based on longitudinal data (from 6 months to 16 months), which will help spatial normalization, voxel-based and tract-based analyses in non-human primates. <br />
<br />
Lots efforts were made to ensure the quality:<br />
1) we manually skull-striped brains for all T1w&amp;DTI data (120+ brains). The time-consuming procedure guaranteed more accurate brain extraction than automatic methods (3dSkullStrip or BET).<br />
2) We had tested different methods, and the best ones were chosen to build the T1w (by ANTS) and DTI (by DTI-TK) templates.<br />
3) We also made high quality template-related files, including parcellation maps, tissue probability maps, inferior cingulum mask for TBSS, and templates for each time point.<br />
<br />
Detailed information about monkeys and methods of building templates can be found in:<br />
Liu,C, et al. &quot;Rhesus monkey brain development during late infancy and the effect of phencyclidine: a longitudinal MRI and DTI study.&quot; NeuroImage (2015). 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=903&screenshot_id=895"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=903&screenshot_id=895" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 14, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/monkey_development/">http://www.nitrc.org/projects/monkey_development/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Laboratory of the Primate Model for Brain Diseases <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="inia19-primate--brain-atlas" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="INIA19 Primate  Brain Atlas">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=inia19-primate--brain-atlas" rel="bookmark" name="inia19-primate--brain-atlas">INIA19 Primate  Brain Atlas</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-data" rel="tag">Atlas Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The INIA19 primate brain atlas was created from over 100 structural MR scans of 19 rhesus macaque animals. The atlas currently comprises high-resolution T1-weighted average MR images with and without skull stripping, tissue probability maps, and a detailed parcellation map based on the NeuroMaps atlas.<br />
<br />
INIA19 was created by researchers of the NIH-funded Integrative Neuroscience Initiative on Alcoholism (INIA). 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=561&screenshot_id=584"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=561&screenshot_id=584" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 17, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/inia19/">http://www.nitrc.org/projects/inia19/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>SRI International <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="insight-toolkit" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Insight Toolkit">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=insight-toolkit" rel="bookmark" name="insight-toolkit">Insight Toolkit</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ITK is an open-source, cross-platform system that provides developers with an extensive suite of software tools for image analysis. The Insight Toolkit is based on generic programming, with a data pipeline architecture, supports for multi-threads, and it is tested nightly on multiple platforms including Linux, Windows, and Mac. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=145&screenshot_id=190"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=145&screenshot_id=190" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 30, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://itk.org/">http://itk.org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, MINC2, NIfTI-1, Nrrd, Philips PAR/REC, bshort/bfloat</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, MINC2, NIfTI-1, Nrrd, Philips PAR/REC, bshort/bfloat</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="instar-(infant-joint-segmentation-and-registration)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="INSTAR (Infant Joint Segmentation and Registration)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=instar-(infant-joint-segmentation-and-registration)" rel="bookmark" name="instar-(infant-joint-segmentation-and-registration)">INSTAR (Infant Joint Segmentation and Registration)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=intersubject" rel="tag">Intersubject</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> INSTAR (Infant Joint Segmentation and Registration) provides a solution of accurate segmentation and registration for infant brain images, especially from birth to 1 year old, which plays an important role in many early brain development studies. <br />
Specifically, infant brain image segmentation or registration often encounters much more challenges than segmentation/registration of adult brain images due to the dynamic appearance change with rapid brain development. Therefore, in this software package, we included out latest joint segmentation and registration framework [1] to tackle the challenges.<br />
This software package was developed in the IDEA group at UNC-Chapel Hill (<a href="http://bric.unc.edu/ideagroup" target="_new">http://bric.unc.edu/ideagroup</a>).<br />
[1] P. Dong, L. Wang, W. Lin, D. Shen, and G. Wu, &quot;Scalable joint segmentation and registration framework for infant brain images,&quot; Neurocomputing, In Press, 2016 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1089&screenshot_id=1053"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1089&screenshot_id=1053" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 22, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/instar/">http://www.nitrc.org/projects/instar/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UNC <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="inter-group-registration-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Inter-Group Registration Toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=inter-group-registration-toolbox" rel="bookmark" name="inter-group-registration-toolbox">Inter-Group Registration Toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> InterGroupReg (Inter-Group Registration Toolbox) provides solutions for registering two groups of images, which are the necessary steps for many brain-related applications.<br />
<br />
This software package was developed in the IDEA group at UNC-Chapel Hill ( <a href="http://bric.unc.edu/ideagroup" target="_new">http://bric.unc.edu/ideagroup</a> ). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 25, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/intergroupreg/">http://www.nitrc.org/projects/intergroupreg/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina at Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="international-imaging-genetics-conference" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="International Imaging Genetics Conference">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=international-imaging-genetics-conference" rel="bookmark" name="international-imaging-genetics-conference">International Imaging Genetics Conference</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=community" rel="tag">Community</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=websites" rel="tag">Websites</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This international symposium was held initially to assess the new technology and innovation in the various established fields of genetics and imaging, and to facilitate the transdisciplinary fusion needed to optimize the development of the emerging field of Imaging Genetics. Over the past decade, the conference has helped to define and document progress in the field and to create a dialogue for the challenges and opportunities it faces. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 21, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/iigc/">http://www.nitrc.org/projects/iigc/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UC Irvine <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="internet-brain-volume-database-(ibvd)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Internet Brain Volume Database (IBVD)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=internet-brain-volume-database-(ibvd)" rel="bookmark" name="internet-brain-volume-database-(ibvd)">Internet Brain Volume Database (IBVD)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volumetric-analysis" rel="tag">Volumetric Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=web-service" rel="tag">Web Service</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The IBVD provides a web-based searchable database of brain neuroanatomic volumetric observations. This is designed to access both group volumetric results as well as volume observations in individual cases. A major thrust effort is to enable electronic access to the results that exist in the published literature. Currently, there is quite limited electronic or searchable methods for the data observations that are contained in publications. This effort will facilitate the dissemination of volumetric observations by making a more complete corpus of volumetric observations findable to the neuroscience researcher. This also enhances the ability to perform comparative and integrative studies, as well as meta-analysis. Extensions that permit pre-published, non-published and other representation are planned, again to facilitate comparative analyses. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=226&screenshot_id=130"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=226&screenshot_id=130" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 12, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://ibvd.virtualbrain.org/">http://ibvd.virtualbrain.org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="intervar" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="InterVar">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=intervar" rel="bookmark" name="intervar">InterVar</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=genomic-analysis" rel="tag">Genomic Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> InterVar is a bioinformatics software tool for clinical interpretation of genetic variants by the ACMG/AMP 2015 guideline. The input to InterVar is an annotated file generated from ANNOVAR, while the output of InterVar is the classification of variants into 'Benign', 'Likely benign', 'Uncertain significance', 'Likely pathogenic' and 'Pathogenic', together with detailed evidence code. InterVar is a command-line-driven software written in Python and can be used as a standalone application on a variety of operating systems—includingWindows, Linux, and MacOS—where Python is installed. wInterVar is a web server, which offers a graphical user interface for InterVar. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 13, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/intervar/">http://www.nitrc.org/projects/intervar/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Children's Hospital of Philadelphia <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="intra--and-inter-scanner-reliability-of-rs-fmri-bold-and-asl-with-eyes-closed-vs.-eyes-open" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Intra- and inter-scanner reliability of RS-fMRI BOLD and ASL with eyes closed vs. eyes open">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=intra--and-inter-scanner-reliability-of-rs-fmri-bold-and-asl-with-eyes-closed-vs.-eyes-open" rel="bookmark" name="intra--and-inter-scanner-reliability-of-rs-fmri-bold-and-asl-with-eyes-closed-vs.-eyes-open">Intra- and inter-scanner reliability of RS-fMRI BOLD and ASL with eyes closed vs. eyes open</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Data information:<br />
21 subjects were scanned 3 times (V1, V2, V3), with V1 and V2 on a scanner, V3 on another scanner in another site. Resting-state blood-oxygenation-level-dependent functional MRI (BOLD-fMRI), pseudo-continuous arterial spin labeling (pCASL), and high-resolution 3D T1 imaging were performed under eyes open (EO) and eyes closed (EC) conditions. Please see below for the original paper (Yuan et al., 2018) and another paper (Zhao et al. 2018).<br />
Yuan, L.-X. et al. (2018) ‘Intra- and Inter-scanner Reliability of Scaled Subprofile Model of Principal Component Analysis on ALFF in Resting-State fMRI Under Eyes Open and Closed Conditions’, Frontiers in Neuroscience, 12, p. 311. doi: 10.3389/fnins.2018.00311<br />
Zhao, N. et al. (2018) ‘Intra- and Inter-Scanner Reliability of Voxel-Wise Whole-Brain Analytic Metrics for Resting State fMRI’, Frontiers in Neuroinformatics, 12, p. 54. doi: 10.3389/fninf.2018.00054 </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 6</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/reliability/">http://www.nitrc.org/projects/reliability/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Hangzhou Normal University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="intrinsic-unscented-kalman-filter-(iukf)-tractography-software-v1.0" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Intrinsic Unscented Kalman Filter (IUKF) Tractography Software v1.0">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=intrinsic-unscented-kalman-filter-(iukf)-tractography-software-v1.0" rel="bookmark" name="intrinsic-unscented-kalman-filter-(iukf)-tractography-software-v1.0">Intrinsic Unscented Kalman Filter (IUKF) Tractography Software v1.0</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Intrinsic Unscented Kalman Filter (IUKF) is a recently introduced tractography algorithm for HARDI. IUKF provides a relatively accurate and efficient fiber tracking mechanism, compared to most existing tractography methods. Following are the key features of IUKF:<br />
• Given HARDI data sets, it is capable of tracking in the presence of complex local geometries such as crossing and kissing fibers.<br />
• Unlike many existing techniques, IUKF does not require one to reconstruct the multi-tensor field a priori, because reconstruction is only performed at the voxels along estimated fibers. This significantly increases the tracking speed.<br />
• IUKF reconstructs a bi-tensor model for underlying signal and exploits intrinsic operations on the space of diffusion tensors (symmetric positive definite matrices). This will provide more accurate and smooth estimates of the underlying signal and the fiber orientations compared to its non-intrinsic counterparts. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=785&screenshot_id=711"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=785&screenshot_id=711" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 1, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Non-Commercial Software License Agreement</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/iukf_2013/">http://www.nitrc.org/projects/iukf_2013/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Florida <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="in-vivo-memri-based-mouse-brain-atlas" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="In vivo MEMRI-based Mouse Brain Atlas">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=in-vivo-memri-based-mouse-brain-atlas" rel="bookmark" name="in-vivo-memri-based-mouse-brain-atlas">In vivo MEMRI-based Mouse Brain Atlas</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> In vivo MEMRI-based Mouse Brain Atlas provides a reference MEMRI data of NSG mouse brain and corresponding 41 structure labels. Using the reference MRI and the atlas, data users can perform regional specific MRI parametric and volumetric analysis. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 3, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/memribrainatlas/">http://www.nitrc.org/projects/memribrainatlas/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Nebraska Medical Center, Omaha <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="in-vivo-memri-based-rat-brain-atlas" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="In vivo MEMRI-based Rat Brain Atlas">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=in-vivo-memri-based-rat-brain-atlas" rel="bookmark" name="in-vivo-memri-based-rat-brain-atlas">In vivo MEMRI-based Rat Brain Atlas</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> In vivo MEMRI-based Rat Brain Atlas provides a reference MEMRI data of Wistar rat brain and corresponding 40 structure labels. Using the reference MRI and the atlas data users can perform regional specific MRI parametric and volumetric analysis. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 29, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/memriratbrains/">http://www.nitrc.org/projects/memriratbrains/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Nebraska Medical Center <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="invizian" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="INVIZIAN">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=invizian" rel="bookmark" name="invizian">INVIZIAN</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Fly through and interact with hundreds of human brains to compare structural differences or carefully inspect individual specimens. Introducing the “Informatics Visualization in Neuroimaging (INVIZIAN©)” Project (<a href="http://invizian.loni.usc.edu" target="_new">http://invizian.loni.usc.edu</a>), a 21st century visualization environment that enables you, via your computer, to display and interact with hundreds of neuroimaging data sets at once —bringing together brain image data from some of the world’s best neuroscience research teams. INVIZIAN© empowers both researchers and students of neuroscience to explore and understand the human brain using a simple yet powerful user interface for neuroimaging data exploration and discovery. See a beautiful example of a cloud of individual brains tumbling around in the INVIZIAN© interface in Vimeo (<a href="http://vimeo.com/67984681" target="_new">http://vimeo.com/67984681</a>). Visit often to see how we are making continuing progress to make Invizian© even more amazing. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=567&screenshot_id=464"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=567&screenshot_id=464" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 28, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>LONI Software License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://invizian.loni.usc.edu">http://invizian.loni.usc.edu</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Southern California <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="isc-toolbox:-inter-subject-correlation-analysis-for-fmri-in-matlab" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="isc-toolbox: Inter-subject correlation analysis for fMRI in Matlab">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=isc-toolbox:-inter-subject-correlation-analysis-for-fmri-in-matlab" rel="bookmark" name="isc-toolbox:-inter-subject-correlation-analysis-for-fmri-in-matlab">isc-toolbox: Inter-subject correlation analysis for fMRI in Matlab</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This Matlab toolbox performs inter-subject correlation (ISC) based analysis of fMRI data and includes a GUI for the visualization of the resulting thresholded statistical maps. <br />
<br />
The toolbox contains functions to carry out various ISC based analysis such as mean, frequency band, time window and phase synchronization ISC analysis. All analyses can be run from a GUI. Isc-toolbox includes also automated support for Slurm/SGE based grid environments. <br />
<br />
A set of visualization tools - particularly designed for ISC analyses - are integrated to the GUI. Visualization tools contain functionalities to which standard functional neuroimaging packages are difficult to adapt.<br />
<br />
Reference: J.-P. Kauppi, J. Pajula J and J. Tohka (2014). A Versatile Software Package for Inter-subject Correlation Based Analyses of fMRI. Frontiers in Neuroinformatics, 8:2.<br />
<br />
Project in Research Gate: <a href="https://www.researchgate.net/project/Inter-subject-correlation-based-analysis-of-fMRI" target="_new">https://www.researchgate.net/project/Int...</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=947&screenshot_id=1258"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=947&screenshot_id=1258" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 15, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MIT/X Consortium License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/isc-toolbox/">http://www.nitrc.org/projects/isc-toolbox/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Tampere University of Technology <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="iso2mesh" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="iso2mesh">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=iso2mesh" rel="bookmark" name="iso2mesh">iso2mesh</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=mesh-generation" rel="tag">Mesh Generation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> &quot;Iso2mesh&quot; is a Matlab/Octave-based mesh generation toolbox designed for easy creation of high quality surface and tetrahedral meshes from 3D volumetric images. It contains a rich set of mesh processing scripts/programs, functioning independently or interfacing with external free meshing utilities. Iso2mesh toolbox can operate directly on 3D binary, segmented or gray-scale images, such as those from MRI or CT scans, making it particularly suitable for multi-modality medical imaging data analysis or multi-physics modeling.<br />
<br />
URL: <a href="http://iso2mesh.sf.net" target="_new">http://iso2mesh.sf.net</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=392&screenshot_id=262"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=392&screenshot_id=262" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 24, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://iso2mesh.sourceforge.net/">http://iso2mesh.sourceforge.net/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="iterative-dual-regression-with-sparse-prior" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Iterative dual-regression with sparse prior">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=iterative-dual-regression-with-sparse-prior" rel="bookmark" name="iterative-dual-regression-with-sparse-prior">Iterative dual-regression with sparse prior</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=application" rel="tag">Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Iterative Dual-Regression with Sparse Prior (IDRwSP) is aimed to better estimate an individual's neuronal activation using the results of an independent component analysis (ICA) method applied to a temporally concatenated group of functional magnetic resonance imaging (fMRI) data (i.e., Tc-GICA method). <br />
 An ordinary DR approach estimates the spatial patterns (SPs) of neuronal activation and corresponding time courses (TCs) specific to each individual's fMRI data with two steps involving least-squares (LS) solutions. The proposed approach employs iterative LS solutions to refine both the individual SPs and TCs with an additional a priori assumption of sparseness in the SPs (i.e., minimally overlapping SPs) based on L(1)-norm minimization.<br />
<br />
See the reference paper.<br />
<br />
Kim YH, Kim J, Lee JH., Iterative approach of dual regression with a sparse prior enhances the performance of independent component analysis for group functional magnetic resonance imaging (fMRI) data., Neuroimage. 2012. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 22, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/iterdrwsp/">http://www.nitrc.org/projects/iterdrwsp/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Brain Signal Processing Lab. <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="itk-snap" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ITK-SNAP">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=itk-snap" rel="bookmark" name="itk-snap">ITK-SNAP</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=active-contour" rel="tag">Active Contour</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=anatomic" rel="tag">Anatomic</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=manual" rel="tag">Manual</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volumetric-analysis" rel="tag">Volumetric Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ITK-SNAP is an open-source software application for medical image segmentation. Its primary use is for delineating anatomical structures and regions in MRI, CT and other 3D biomedical imaging data. ITK-SNAP is actively developed by Paul Yushkevich, Hui Zhang, and colleagues at the Penn Image Computing and Science Laboratory at the University of Pennsylvania. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=110&screenshot_id=8"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=110&screenshot_id=8" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 30, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.itksnap.org">http://www.itksnap.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, NIfTI-1, Nrrd, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, NIfTI-1, Nrrd, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="itools-resourceome" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ITools Resourceome">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=itools-resourceome" rel="bookmark" name="itools-resourceome">ITools Resourceome</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data-model" rel="tag">Data Model</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data-resource" rel="tag">Data Resource</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=experiment-control" rel="tag">Experiment Control</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=ontology" rel="tag">Ontology</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-information-resource" rel="tag">Other Information Resource</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=web-resource" rel="tag">Web Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> iTools is an infrastructure for managing of diverse computational biology resources - data, software tools and web-services. The iTools design, implementation and meta-data content reflect the broad NCBC needs and expertise (www.NCBCs.org).<br />
<br />
iTools is distributed by the Laboratory of Neuro Imaging (<a href="http://www.loni.usc.edu/Software/iTools" target="_new">http://www.loni.usc.edu/Software/iTools</a>) at USC. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=126&screenshot_id=410"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=126&screenshot_id=410" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 24, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Lesser General Public License (LGPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://cms.loni.usc.edu/iTools/">http://cms.loni.usc.edu/iTools/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Southern California <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="iview-x-mri-lr---eye-tracking-for-fmri" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="iView X MRI-LR - Eye Tracking for fMRI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=iview-x-mri-lr---eye-tracking-for-fmri" rel="bookmark" name="iview-x-mri-lr---eye-tracking-for-fmri">iView X MRI-LR - Eye Tracking for fMRI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The iView X MRI-LR is a non-invasive, long-range eye tracking system for use in the fMRI environment. Some features of the system include:<br />
<br />
- Elaborate faraday shielding and fiber optics to avoid noise in high-field magnets.<br />
<br />
- Includes stimulus presentation software “Experiment Center” and is compatible with 3rd party products such as “Presentation” by NeuroBS.<br />
<br />
- Utilizes mirror box customized for large field of view.<br />
<br />
- Includes powerful analysis software “BeGaze2” for graphical and statistical analysis of eye movements.<br />
<br />
- Includes fixation, saccade and blink detection, and area-of-interest based statistics<br />
<br />
- Real-time data available via digital or analog output </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 29, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Commerical Software</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/iviewx_mri-lr/">http://www.nitrc.org/projects/iviewx_mri-lr/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="ixi-dataset" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="IXI Dataset">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=ixi-dataset" rel="bookmark" name="ixi-dataset">IXI Dataset</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The IXI Dataset is a collection of nearly 600 MR images from normal, healthy subjects. The MR image acquisition protocol for each subject includes T1, T2 and PD-weighted images, MRA images, and diffusion-weighted images (15 directions). The data was collected at three different hospitals in London using 1.5T and 3T scanners. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 23, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://biomedic.doc.ic.ac.uk/brain-development/index.php?n=Main.Datasets">http://biomedic.doc.ic.ac.uk/brain-development/index.php?n=Main.Datasets</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Imperial College London <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="jhu-proj.-in-applied-medical-imaging" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="JHU Proj. in Applied Medical Imaging">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=jhu-proj.-in-applied-medical-imaging" rel="bookmark" name="jhu-proj.-in-applied-medical-imaging">JHU Proj. in Applied Medical Imaging</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This project is used for students enrolled in courses using the JIST framework. Content in this CVS is freely available, but it is not intended for any specific purpose. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 19, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/iaclmedic/">http://www.nitrc.org/projects/iaclmedic/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="jim" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Jim">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=jim" rel="bookmark" name="jim">Jim</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=application" rel="tag">Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Jim is a medical image display package that allows easy viewing and analysis of Magnetic Resonance, x-ray CT and other types of medical image. Jim is an up-to-the-minute design with a familiar user-interface. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=154&screenshot_id=920"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=154&screenshot_id=920" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 1, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Available by Purchase Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/jim/">http://www.nitrc.org/projects/jim/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Xinapse Systems Ltd <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Microsoft<br>SunOS/Solaris</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, NIfTI-1, Other Format, VRML</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, NIfTI-1, Other Format, VRML</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="jip-fmri-analysis-toolkit" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="JIP fMRI Analysis Toolkit">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=jip-fmri-analysis-toolkit" rel="bookmark" name="jip-fmri-analysis-toolkit">JIP fMRI Analysis Toolkit</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=affine-warp" rel="tag">Affine Warp</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-data" rel="tag">Atlas Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-display" rel="tag">Image Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-to-image" rel="tag">Image-To-Image</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=nonlinear-warp" rel="tag">Nonlinear Warp</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=resampling" rel="tag">Resampling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=time-domain-analysis" rel="tag">Time Domain Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Refer to <a href="http://www.nmr.mgh.harvard.edu/~jbm/jip" target="_new">http://www.nmr.mgh.harvard.edu/~jbm/jip</a> for web documentation (in progress). The JIP Toolkit was developed primarily for analysis of rodent and non-human primate fMRI data. The toolkit consists of binary executables, highly portable open-source c code, and image resources that enable 1) Automated registration based upon mutual information (affine, non-linear warps), with flexible control and visualization of each step; 2) visualization of 4-dimensional data using either mosaic or tri-planar display of the z/slice dimension, and integration of a general linear model for graphical display of time series analysis; 3) A simple and flexible 1st-order GLM for fMRI time series analysis, a 1st-order GLM analysis for PET data within the SRTM framework, plus a 2nd-order GLM analysis following the Worsley 2002 scheme, and 4) MRI templates to place your rodent and non-human primate data into standardized spaces. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=310&screenshot_id=315"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=310&screenshot_id=315" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 11, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MGH CSRL License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/jip/">http://www.nitrc.org/projects/jip/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="jist:-java-image-science-toolkit" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="JIST: Java Image Science Toolkit">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=jist:-java-image-science-toolkit" rel="bookmark" name="jist:-java-image-science-toolkit">JIST: Java Image Science Toolkit</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=experimental-control" rel="tag">Experimental Control</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=modeling" rel="tag">Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quantification" rel="tag">Quantification</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=shape-analysis" rel="tag">Shape Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=spatial-transformation" rel="tag">Spatial Transformation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Java Image Science Toolkit (JIST) provides a native Java-based imaging processing environment similar to the ITK/VTK paradigm. Initially developed as an extension to MIPAV (CIT, NIH, Bethesda, MD), the JIST processing infrastructure provides automated GUI generation for application plug-ins, graphical layout tools, and command line interfaces. <br />
<br />
This repository maintains the current multi-institutional JIST development tree and is recommended for public use and extension. To participate in this consortium or seek assistance in integrating JIST modules in your application, please contact us at <a href="mailto:jist-admin@www.nitrc.org" target="_new">jist-admin@www.nitrc.org</a>.<br />
<br />
JIST was originally developed at IACL and MedIC (Johns Hopkins University) and is now also supported by MASI (Vanderbilt University).<br />
<br />
Please use the download nightly build link below to retrieve a currently supported version of JIST. The previous versions are available for archival access. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=228&screenshot_id=173"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=228&screenshot_id=173" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 2, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Lesser General Public License (LGPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/jist/">http://www.nitrc.org/projects/jist/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>BSD<br>Linux<br>MacOS<br>OS Independent<br>SunOS/Solaris<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>AFNI BRIK, ANALYZE, COR, DICOM, GIfTI, MGH/MGZ, MINC, MINC2, NIfTI-1, Nrrd, Other Format, Philips PAR/REC</td></tr>
			<tr class="even"><td>Output Formats</td><td>AFNI BRIK, ANALYZE, COR, DICOM, GIfTI, MGH/MGZ, MINC, MINC2, NIfTI-1, Nrrd, Other Format, Philips PAR/REC</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="jist-resources-for-algorithm-development" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="JIST Resources for Algorithm Development">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=jist-resources-for-algorithm-development" rel="bookmark" name="jist-resources-for-algorithm-development">JIST Resources for Algorithm Development</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=application" rel="tag">Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-software-resource" rel="tag">Other Software Resource</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=platform-or-development-environment" rel="tag">Platform Or Development Environment</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This repository stores plugins, tutorial code, and examples demonstrating MRI manipulation within the MIPAV plugin environment.<br />
<br />
This project is separate from JIST so that we can provide WRITE access to any interested party without overly exposing the infrastructure to unplanned modification. Please contact the administrators if you would like to join this project - open use is encouraged. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=143&screenshot_id=182"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=143&screenshot_id=182" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 29, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Lesser General Public License (LGPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/jhumipavplugins/">http://www.nitrc.org/projects/jhumipavplugins/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS<br>Microsoft<br>POSIX/UNIX-like</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>AFNI BRIK, ANALYZE, COR, DICOM, GIfTI, MGH/MGZ, MINC, MINC2, NIfTI-1, Nrrd, Other Format, bshort/bfloat</td></tr>
			<tr class="even"><td>Output Formats</td><td>AFNI BRIK, ANALYZE, COR, DICOM, GIfTI, MGH/MGZ, MINC, MINC2, NIfTI-1, Nrrd, Other Format, bshort/bfloat</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="joint-anisotropic-lmmse-filter-for-stationary-rician-noise-removal-in-dwi" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Joint Anisotropic LMMSE Filter for Stationary Rician noise removal in DWI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=joint-anisotropic-lmmse-filter-for-stationary-rician-noise-removal-in-dwi" rel="bookmark" name="joint-anisotropic-lmmse-filter-for-stationary-rician-noise-removal-in-dwi">Joint Anisotropic LMMSE Filter for Stationary Rician noise removal in DWI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=artifact-removal" rel="tag">Artifact Removal</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=band-pass" rel="tag">Band Pass</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=high-pass" rel="tag">High Pass</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=intensity-non-uniformity-correction" rel="tag">Intensity Non-Uniformity Correction</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=low-pass" rel="tag">Low Pass</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quality-metrics" rel="tag">Quality Metrics</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=spatial-convolution---deconvolution" rel="tag">Spatial Convolution - Deconvolution</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This module reduces Rician noise on DWI. It filters the image in the mean squared error sense using a Rician noise model. The N closest gradient directions to the direction being processed are filtered together to improve the results: the noise-free signal is seen as an n-dimensional vector which has to be estimated with the LMMSE method from a set of corrupted measurements. The covariance matrix of the noise-free vector and the cross covariance between this signal and the noise have to be estimated, which is done taking into account the image formation process.<br />
<br />
All these estimations are performed as sample estimates in a 'shaped neighborhood' defined by the weights extracted from the structural similarity of the voxels following the same idea as in the Non-Local Means filter.<br />
<br />
This C++ code can be compiled as either a standalone (ITK required. Versions 3 and 4 are supported) or as a CLI module for 3-D Slicer (versions 3 and 4). Only &quot;nhdr/nrrd&quot; DWIs are supported. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=680&screenshot_id=617"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=680&screenshot_id=617" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 3, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>3D Slicer License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/jalmmse_dwi/">http://www.nitrc.org/projects/jalmmse_dwi/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Valladolid <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS<br>Microsoft<br>POSIX/UNIX-like</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Nrrd</td></tr>
			<tr class="even"><td>Output Formats</td><td>Nrrd</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="journals" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Journals">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=journals" rel="bookmark" name="journals">Journals</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=journal" rel="tag">Journal</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> If you are seeking journals addressing functional and structural neuroimaging topics, this is the right place to start. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 24, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/journals/">http://www.nitrc.org/projects/journals/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="k-space-domain-image-fusion" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="K-space Domain Image Fusion">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=k-space-domain-image-fusion" rel="bookmark" name="k-space-domain-image-fusion">K-space Domain Image Fusion</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Matlab code that implements a K-space magnetic resonance image fusion technique. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 9, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/kspace_fusion/">http://www.nitrc.org/projects/kspace_fusion/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Computational Radiology Laboratory <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="kurtosis-imaging-network---kin" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Kurtosis Imaging Network - KIN">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=kurtosis-imaging-network---kin" rel="bookmark" name="kurtosis-imaging-network---kin">Kurtosis Imaging Network - KIN</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Sorry, but images are no longer available for download from KIN.<br />
<br />
--------------------------------------------------------------------------------------------<br />
<br />
Kurtosis Imaging Network (KIN) was an open source database for normal healthy controls as well as various pathologies in an attempt to establish a standard range of kurtosis values within each population. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=904&screenshot_id=911"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=904&screenshot_id=911" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 18, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/kin/">http://www.nitrc.org/projects/kin/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>MUSC <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="kwscene:-mrml-based-atlas-and-scene-builder/reader/writer" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="KWScene: MRML-based Atlas and Scene Builder/Reader/Writer">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=kwscene:-mrml-based-atlas-and-scene-builder/reader/writer" rel="bookmark" name="kwscene:-mrml-based-atlas-and-scene-builder/reader/writer">KWScene: MRML-based Atlas and Scene Builder/Reader/Writer</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ITK-based processing and 3D Slicer scene management in ParaView. We believe this will broaden the use of ParaView for high performance computing and visualization in the medical imaging research community. The effort is focused on developing ParaView plug-ins for managing VTK structures from 3D Slicer MRML scenes and encapsulating ITK filters for deployment in ParaView. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 4, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mrml-paraview/">http://www.nitrc.org/projects/mrml-paraview/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Johns Hopkins University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="kymata-atlas" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Kymata Atlas">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=kymata-atlas" rel="bookmark" name="kymata-atlas">Kymata Atlas</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-data" rel="tag">Atlas Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data-model" rel="tag">Data Model</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database-application" rel="tag">Database Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-rendering" rel="tag">Surface Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=three-dimensional-display" rel="tag">Three Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=web-service" rel="tag">Web Service</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=websites" rel="tag">Websites</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Kymata Atlas is a freely available, online atlas of neural functions, developed by Cambridge University and the MRC-CBSU. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=830&screenshot_id=781"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=830&screenshot_id=781" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 11, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://kymata-atlas.org">http://kymata-atlas.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Cambridge University, MRC CBSU <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="laboratory-of-neuro-imaging-(loni)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Laboratory of Neuro Imaging (LONI)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=laboratory-of-neuro-imaging-(loni)" rel="bookmark" name="laboratory-of-neuro-imaging-(loni)">Laboratory of Neuro Imaging (LONI)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Laboratory of Neuro Imaging strives to improve our understanding of the brain in health and disease. LONI develops advanced computational algorithms and scientific approaches for the comprehensive and quantitative mapping of brain structure and function. LONI aims to encourage communication between users and LONI software engineers in order to improve the effectiveness of computational brain mapping software and to promote its use by researchers worldwide. LONI software website (<a href="http://www.loni.usc.edu/Software/" target="_new">http://www.loni.usc.edu/Software/</a>) includes downloadable and web-accessible tools, training and support. Be sure to visit LONI Forums and discuss our tools with other end-users and LONI investigators.<br />
<br />
<br />
<a href="http://www.loni.usc.edu" target="_new">http://www.loni.usc.edu</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=525&screenshot_id=422"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=525&screenshot_id=422" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 16, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>LONI Software License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/loni/">http://www.nitrc.org/projects/loni/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Southern California <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="landman-neuroimaging-tools" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Landman NeuroImaging Tools">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=landman-neuroimaging-tools" rel="bookmark" name="landman-neuroimaging-tools">Landman NeuroImaging Tools</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=statistical-operation" rel="tag">Statistical Operation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Project to provide long-term hosting and release for small tools related to medical image analysis. Source repository contains highly experimental code intended for collaborative development. However, any interested parties are welcome to browse/reuse code. Stable/evolved projects will be moved to independent projects. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 6, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Lesser General Public License (LGPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/landman/">http://www.nitrc.org/projects/landman/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="laplace-beltrami-filter-on-quadedge-meshes" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Laplace Beltrami Filter on QuadEdge Meshes">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=laplace-beltrami-filter-on-quadedge-meshes" rel="bookmark" name="laplace-beltrami-filter-on-quadedge-meshes">Laplace Beltrami Filter on QuadEdge Meshes</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The purpose of this filter is to use the Laplace-Beltrami operator to determine surface harmonics in terms<br />
of PointData at each vertex. In the same way that a sound signal can be approximately characterized by a<br />
combination of its most significant frequency components, so can a surface be expressed as a combination<br />
of its surface harmonics. This filter determines the requested N most significant harmonics. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 4, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/laplacebeltrami/">http://www.nitrc.org/projects/laplacebeltrami/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Johns Hopkins University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="lddmm" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="LDDMM">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=lddmm" rel="bookmark" name="lddmm">LDDMM</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=shape-analysis" rel="tag">Shape Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Large Deformation Diffeomorphic Metric Mapping (LDDMM) tool is an application which aims to assign metric distances on the space of anatomical images in Computational Anatomy thereby allowing for the direct comparison and quantization of morphometric changes in shapes. As part of these efforts the Center for Imaging Science at Johns Hopkins University developed techniques to not only compare images, but also to visualize the changes and differences.<br />
<br />
For additional information please refer to:<br />
<br />
 Faisal Beg, Michael Miller, Alain Trouve, and Laurent Younes. Computing Large Deformation Metric Mappings via Geodesic Flows of Diffeomorphisms. International Journal of Computer Vision, Volume 61, Issue 2; February 2005.<br />
<br />
 M.I. Miller and A. Trouve and L. Younes, On the Metrics and Euler-Lagrange Equations of Computational Anatomy, Annual Review of biomedical Engineering, 4:375-405, 2002. <br />
<br />
Software developed with support from National Institutes of Health NCRR grant P41 RR15241. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 5, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://cis.jhu.edu/software">http://cis.jhu.edu/software</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="lead-connectome" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Lead-Connectome">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=lead-connectome" rel="bookmark" name="lead-connectome">Lead-Connectome</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=connectivity-analysis" rel="tag">Connectivity Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=fiber-tracking" rel="tag">Fiber Tracking</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Lead-Connectome is a fully equipped structural-functional connectome processing toolbox. It features several modern fiber-tracking algorithms that support single- and multishell (~DSI) data and state-of-the art nonlinear deformation algorithms. It operates in the Matlab development environment and is part of the Lead Neuroimaging Suite. As such, it seamlessly integrates with the Lead-DBS toolbox. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1058&screenshot_id=1004"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1058&screenshot_id=1004" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 11, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/lead-connectome/">http://www.nitrc.org/projects/lead-connectome/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Charité University Medicine <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>OS Independent<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="lead-dbs" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="LEAD-DBS">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=lead-dbs" rel="bookmark" name="lead-dbs">LEAD-DBS</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=diffusion-mr-fiber-tracking" rel="tag">Diffusion Mr Fiber Tracking</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=simulation" rel="tag">Simulation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-rendering" rel="tag">Surface Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=three-dimensional-display" rel="tag">Three Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=two-dimensional-display" rel="tag">Two Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volume-rendering" rel="tag">Volume Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> LEAD-DBS is a MATLAB-toolbox facilitating the:<br />
- reconstruction of deep-brain-stimulation (DBS) electrodes in the human brain on basis of postoperative MRI and/or CT imaging<br />
- visualization of localization results in 2D/3D<br />
- group-analysis of DBS-electrode placement results and their effects on clinical results<br />
- simulation of DBS stimulations (calculation of volume of activated tissue – VAT)<br />
- diffusion tensor imaging (DTI) based connectivity estimates and fiber-tracking from the VAT to other brain regions (connectomic surgery)<br />
<br />
LEAD-DBS builds on SPM8/12, especially regarding warping and segmentation procedures. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=896&screenshot_id=881"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=896&screenshot_id=881" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 30, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.lead-dbs.org">http://www.lead-dbs.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Charité University Medicine <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS<br>OS Independent<br>POSIX/UNIX-like<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="learning-based-image-segmentation-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Learning based Image Segmentation Toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=learning-based-image-segmentation-toolbox" rel="bookmark" name="learning-based-image-segmentation-toolbox">Learning based Image Segmentation Toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Local Label Learning for Multi-atlas based Image Segmentation Toolbox </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 14, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>PAMI License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/list/">http://www.nitrc.org/projects/list/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UPENN <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="level-set-segmentation-for-slicer3" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Level-set Segmentation for Slicer3">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=level-set-segmentation-for-slicer3" rel="bookmark" name="level-set-segmentation-for-slicer3">Level-set Segmentation for Slicer3</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The modules in the framework support different tasks in the segmentation realization in 3DSlicer.<br />
<br />
A module called Level-set label map evolver was developed, which takes an initial label image and a feature image as input and performs a Geodesic Active Contours evolution on the label image according to the feature image and to a different terms in the level-set equation. The evolution takes place for a customizable number of iterations. <br />
<br />
The output is a label image that can be used to produce a model.<br />
<br />
Other modules were developed to accompany the main module as can be seen in <a href="http://www.slicer.org/slicerWiki/index.php/Slicer3:Module:Level-Set_Segmentation_Framework-Documentation" target="_new">http://www.slicer.org/slicerWiki/index.p...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 21, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>3D Slicer License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/levelsetslicer3/">http://www.nitrc.org/projects/levelsetslicer3/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="lfpy" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="LFPy">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=lfpy" rel="bookmark" name="lfpy">LFPy</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> LFPy is a Python package for calculation of extracellular potentials from multicompartment neuron models. It relies on the Python interface provided by the NEURON simulator. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 5, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/lfpy/">http://www.nitrc.org/projects/lfpy/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="libeep" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="LIBEEP">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=libeep" rel="bookmark" name="libeep">LIBEEP</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=format-conversion" rel="tag">Format Conversion</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The LIBEEP library deals with reading and writing RIFF-format CNT/AVR-files.<br />
<br />
This file format is also called &quot;EEProbe data format&quot;, and is used in the software packages EEProbe, ASA, ASA-Lab, Cognitrace, eemagine EEG, Visor, by ANT Neuro B.V., The Netherlands.<br />
<br />
The file format provides for storage of EEG/ERP/MEG data as 32-bit values, and includes a very efficient compression algorithm. Encoding/decoding from the compressed data is performed automatically through the LIBEEP interface functions. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=553&screenshot_id=930"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=553&screenshot_id=930" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 23, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Lesser General Public License (plus Addendum)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://libeep.sourceforge.net/">http://libeep.sourceforge.net/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>ANT Neuro B.V. <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="libminc---medical-image-netcdf-library" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="libminc - Medical Image NetCDF library">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=libminc---medical-image-netcdf-library" rel="bookmark" name="libminc---medical-image-netcdf-library">libminc - Medical Image NetCDF library</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=file-format" rel="tag">File Format</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MINC is primarily a file format and a 3 level API for medical image analysis with a particular focus on the needs of research. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 27, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MINC Licence (BSD'ish)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://bic-mni.github.io/">http://bic-mni.github.io/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>McGill University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>MINC, MINC2</td></tr>
			<tr class="even"><td>Output Formats</td><td>MINC, MINC2</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="libsbml" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="libSBML">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=libsbml" rel="bookmark" name="libsbml">libSBML</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> LibSBML is a free, open-source programming library to help you read, write, manipulate, translate, and validate SBML files and data streams. It's not an application itself (though it does come with example programs), but rather a library you embed in your own applications. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=860&screenshot_id=839"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=860&screenshot_id=839" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 15, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Lesser General Public License (LGPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://sbml.org">http://sbml.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>California Institute of Technology <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>AIX<br>BSD<br>Cygwin<br>Linux<br>MacOS<br>Other UNIX-like<br>SunOS/Solaris<br>Windows NT/2000<br>Windows Vista<br>Windows XP</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="licensing" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Licensing">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=licensing" rel="bookmark" name="licensing">Licensing</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Licensing issues in software and data.<br />
<br />
All are invited to participate or join the project and contribute. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 7, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/licensing/">http://www.nitrc.org/projects/licensing/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="lightweight-data-pipeline" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Lightweight Data Pipeline">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=lightweight-data-pipeline" rel="bookmark" name="lightweight-data-pipeline">Lightweight Data Pipeline</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This is a lightweight framework for setting up dependency-driven processing pipelines. The tool is essentially a configurable shell script (sh/bash), which can be included in other scripts and primarily provides a small number of utility functions for dependency checking and NFS-safe file locking for cluster processing.<br />
<br />
While the framework is not very powerful (deliberately so), it is extremely simple and easy to apply. There is no need to provide any command wrappers. An existing processing script (or sequence of shell commands) can be turned into a data-driven, cluster-safe pipeline by adding just a few extra lines of shell code. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 8, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/lwdp/">http://www.nitrc.org/projects/lwdp/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>POSIX/UNIX-like</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="limo-eeg" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="LIMO EEG">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=limo-eeg" rel="bookmark" name="limo-eeg">LIMO EEG</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=time-domain-analysis" rel="tag">Time Domain Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> LIMO EEG is a matlab toolbox (EEGlab compatible) allowing the processing of<br />
MEEG data using single trials and hierarchical linear models. Almost all statistical designs can be analyzed with our tool. Across subject analyses are performed using bootstrap offering robust inferences. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=506&screenshot_id=633"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=506&screenshot_id=633" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 11, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://github.com/LIMO-EEG-Toolbox">https://github.com/LIMO-EEG-Toolbox</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="linear-fascicle-evaluation" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Linear Fascicle Evaluation">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=linear-fascicle-evaluation" rel="bookmark" name="linear-fascicle-evaluation">Linear Fascicle Evaluation</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=diffusion-mr-fiber-tracking" rel="tag">Diffusion Mr Fiber Tracking</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This software implements a framework to encode structural brain connectomes into multidimensional arrays. These arrays are commonly referred to as tensors. Encoding Connectomes provides an agile framework for computing over connectome edges and nodes efficiently. We provide several examples of operations that can be performed using the framework. One major application of the tensor encoding is the implementation of the Linear Fascicle Evaluation method, in short LiFE. The tensor encoding method allows implementing LiFE with a dramatic reduction in storage requirements, up to 40x compression factors. Furthermore, connectome encoding allows performing multiple computational neuroanatomy operations such as tract-dissections, virtual lesions, and connectivity estimates very efficiently using the machine-friendly array operators. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1261&screenshot_id=1176"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1261&screenshot_id=1176" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 23, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://github.com/brain-life/encode">https://github.com/brain-life/encode</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Indiana University, Bloomington <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="lipsia" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Lipsia">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=lipsia" rel="bookmark" name="lipsia">Lipsia</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Lipsia is a software tool for processing functional magnetic resonance imaging (fMRI) data. It was developed over the course of several years at the Max-Planck-Institute for Human Cognitive and Brain Sciences in Leipzig, Germany.<br />
Lipsia contains software tools for all aspects of fMRI data processing:<br />
<br />
 * registration and normalization<br />
 * preprocessing<br />
 * exploratory processing<br />
 * statistical evaluation<br />
 * region of interest analysis<br />
 * timecourse analysis<br />
 * visualization, rendering<br />
 * converters to various data formats<br />
<br />
It was developed in C/C++ on Linux PCs. Lipsia provides extremely fast implementations. A standard analysis sequence from the raw data to a statistical parametric map generally takes less than 10 minutes per test subject. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 16, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/lipsia/">http://www.nitrc.org/projects/lipsia/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="l-measure" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="L-Measure">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=l-measure" rel="bookmark" name="l-measure">L-Measure</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> L-Measure is a robust and web-accessible neuroinformatic tool for quantitative and qualitative analysis of three-dimensional neuromorphological reconstructions. Visit LM home at <a href="http://cng.gmu.edu:8080/Lm/" target="_new">http://cng.gmu.edu:8080/Lm/</a> for downloading the tool. Or, you may also analyze your vector form reconstructions by directly uploading them to the server via web-interfaced application available at the same URL. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 18, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/lmeasure/">http://www.nitrc.org/projects/lmeasure/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>George Mason University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="l-neuron" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="L-Neuron">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=l-neuron" rel="bookmark" name="l-neuron">L-Neuron</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=neuronal-modeling" rel="tag">Neuronal Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=simulation" rel="tag">Simulation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The L-Neuron program creates anatomically realistic virtual neurons using the formalism of the Lyndenmayer systems to implement sets of neuroanatomical rules discovered by several research groups (and in particular, Hillman's, Tamori', and Burke's). These rules are local and recursive. The L-Neuron algorithms read in experimental data to generate virtual structures. The experimental data are in the form of statistical distributions (for example, bifurcation angles in Purkinje cells can be represented with a Gaussian distribution, with a certain average and standard deviation). L-Neuron samples the values of the parameters within these statistical distributions in a stochastic (random) fashion during dendritic growth. Therefore, with the same set of parameter distributions, the program can generate an unlimited number of virtual neurons. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 2, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://krasnow1.gmu.edu/cn3/L-Neuron/home.htm">http://krasnow1.gmu.edu/cn3/L-Neuron/home.htm</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>George Mason University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="local-binary-pattern-analysis-tools-for-mr-brain-images" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Local Binary Pattern Analysis Tools for MR Brain Images">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=local-binary-pattern-analysis-tools-for-mr-brain-images" rel="bookmark" name="local-binary-pattern-analysis-tools-for-mr-brain-images">Local Binary Pattern Analysis Tools for MR Brain Images</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The packaged tools perform Local Binary Pattern on Three Orthogonal Planes(LBP-TOP) analysis on MR brain images. One can use them to extract LBP texture features for machine learning applications or other advance analysis. LBP-TOP mapping programs written by Java are including in this package. The output is the histogram describing the brain morphology. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 19, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mri_lbptop/">http://www.nitrc.org/projects/mri_lbptop/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Department of Electrical Engineering, National Taiwan University, Taipei, Taiwan <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="local-label-learning-for-image-segmentation" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Local Label Learning for Image Segmentation">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=local-label-learning-for-image-segmentation" rel="bookmark" name="local-label-learning-for-image-segmentation">Local Label Learning for Image Segmentation</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Local Label Learning for Multi-atlas based Image Segmentation Toolbox </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 17, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License, GNU Lesser General Public License and GNU General Public License, v</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/locallabel/">http://www.nitrc.org/projects/locallabel/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pennsylvania <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="local-label-learning-(lll)-segmentation" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Local Label Learning (LLL) Segmentation">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=local-label-learning-(lll)-segmentation" rel="bookmark" name="local-label-learning-(lll)-segmentation">Local Label Learning (LLL) Segmentation</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Automatic and reliable segmentation of subcortical structures is an important but difficult task in quantitative brain image analysis. Recently, multi-atlas based segmentation methods have attracted great interest due to their competitive performance. While many label fusion strategies have been developed, most of these methods adopt predefined weighting models which were not necessarily optimal. In this paper, we proposed a novel local label learning strategy to estimate the target image’s segmentation label using statistical machine learning techniques. We used a support vector machine (SVM) with a K nearest neighbor (KNN) based training sample selection strategy to learn a classifier for each of the target image voxel based on a training dataset consisting of its neighboring voxels in the atlases. Validation experiments on hippocampus segmentation of 117 MR images demonstrated that our method can produce segmentation results consistently better than state-of-the-art label fusion methods. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 30, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/lll/">http://www.nitrc.org/projects/lll/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>chinese academy of sciense <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="local-manifold-learning-for-multi-atlas-segmentation" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Local Manifold Learning for Multi-atlas segmentation">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=local-manifold-learning-for-multi-atlas-segmentation" rel="bookmark" name="local-manifold-learning-for-multi-atlas-segmentation">Local Manifold Learning for Multi-atlas segmentation</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> LMLMS(Local Manifold Learning for Multi-atlas segmentation) employs manifold learning to determine spatially local weights for label fusion in a low-dimensional coordinate space. It's based on paper &quot;Local manifold learning for multiatlas segmentation: application to hippocampal segmentation in healthy population and Alzheimer's disease&quot; </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 22, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Lesser General Public License (LGPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mllf/">http://www.nitrc.org/projects/mllf/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Beihang University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="longitudinal-multiple-sclerosis-lesion-imaging-archive" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Longitudinal Multiple Sclerosis Lesion Imaging Archive">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=longitudinal-multiple-sclerosis-lesion-imaging-archive" rel="bookmark" name="longitudinal-multiple-sclerosis-lesion-imaging-archive">Longitudinal Multiple Sclerosis Lesion Imaging Archive</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Longitudinal Multiple Sclerosis (MS) Lesion Imaging Archive provides Training data consisting of longitudinal images from 5 patients and Testing data from 14 patients.<br />
<br />
Each longitudinal dataset includes T1-w, T2-w, PD-w, and T2-w FLAIR MRI with 3-5 time points acquired on a 3T MR scanner. With the multiple time points, this<br />
constitutes 82 individual data sets. To minimize the dependency of the results on registration performance and brain extraction, all images are already rigidly co-registered to the baseline T1-w image with automatically computed skull stripping masks. Unprocessed data is also included. The Training data also contains manual segmentations of the MS lesions from two different raters for each of the time points provided.<br />
<br />
The main page regarding the challenge is <a href="http://www.iacl.jhu.edu/MSChallenge" target="_new">http://www.iacl.jhu.edu/MSChallenge</a> and the data is available from the Smart Stats Website: <a href="https://smart-stats-tools.org/node/26" target="_new">https://smart-stats-tools.org/node/26</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=886&screenshot_id=880"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=886&screenshot_id=880" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 30, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial No Derivatives</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/longitudinal_ms/">http://www.nitrc.org/projects/longitudinal_ms/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Henry M. Jackson Foundation for the Advancement of Military Medicine <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="longitudinal-neuroimaging-hippocampal-markers-for-diagnosing-alzheimer's-disease" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Longitudinal neuroimaging hippocampal markers for diagnosing Alzheimer's disease">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=longitudinal-neuroimaging-hippocampal-markers-for-diagnosing-alzheimer's-disease" rel="bookmark" name="longitudinal-neuroimaging-hippocampal-markers-for-diagnosing-alzheimer's-disease">Longitudinal neuroimaging hippocampal markers for diagnosing Alzheimer's disease</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> We introduce a longitudinal image analysis framework based on robust registration and simultaneous hippocampal segmentation and longitudinal marker classification of brain MRI of an arbitrary number of time points. The framework comprises two innovative parts: a longitudinal segmentation and a longitudinal classification step. We introduce a novel approach to the joint segmentation of the hippocampus across multiple time points; this approach is based on graph cuts of longitudinal MRI scans with constraints on hippocampal atrophy and supported by atlases. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 29, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/longhippsegm/">http://www.nitrc.org/projects/longhippsegm/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UPM <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="longitudinal-rhesus-monkey-brain-templates" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Longitudinal Rhesus Monkey Brain Templates">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=longitudinal-rhesus-monkey-brain-templates" rel="bookmark" name="longitudinal-rhesus-monkey-brain-templates">Longitudinal Rhesus Monkey Brain Templates</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Age-specific rhesus monkey brain templates from T1-weighted structural MRI (3T Scanner). Templates created from 24 rhesus macaques using deformable image registration (ANTS), at 1week, 4weeks, 8weeks, 13weeks, 26weeks, 39weeks, 52weeks, 104weeks, and 260 weeks of age. Tissue segmentations and lobar parcellations are also provided. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 29, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/ama24/">http://www.nitrc.org/projects/ama24/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UC Davis <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="loni-debabeler" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="LONI Debabeler">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=loni-debabeler" rel="bookmark" name="loni-debabeler">LONI Debabeler</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-software-resource" rel="tag">Other Software Resource</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=platform-or-development-environment" rel="tag">Platform Or Development Environment</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The LONI Debabeler manages the conversion of imaging data from one file format and convention to another. It consists of a GUI to visually program the translations, and a data translation engine to read, sort and translate the input files. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 19, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>LONI Software License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.loni.ucla.edu/Software/Software_Detail.jsp?software_id=11">http://www.loni.ucla.edu/Software/Software_Detail.jsp?software_id=11</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, MINC, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, MINC, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="loni-de-identification-debablet" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="LONI De-identification Debablet">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=loni-de-identification-debablet" rel="bookmark" name="loni-de-identification-debablet">LONI De-identification Debablet</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-software-resource" rel="tag">Other Software Resource</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=platform-or-development-environment" rel="tag">Platform Or Development Environment</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The LONI de-identification Debablet is an application for removing patient-identifying information from medical image files. Removing this information is often necessary for enabling investigators to share image files in a HIPAA compliant manner. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 19, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>LONI Software License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.loni.ucla.edu/Software/Software_Detail.jsp?software_id=23">http://www.loni.ucla.edu/Software/Software_Detail.jsp?software_id=23</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, MINC, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, MINC, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="loni-inspector" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="LONI Inspector">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=loni-inspector" rel="bookmark" name="loni-inspector">LONI Inspector</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-software-resource" rel="tag">Other Software Resource</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=platform-or-development-environment" rel="tag">Platform Or Development Environment</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The LONI Inspector is an application for examining medical image files. The Inspector focuses on reading, displaying, searching, comparing, and exporting &quot;metadata&quot; (e.g., patient name, the model of scanner). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 19, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>LONI Software License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.loni.ucla.edu/Software/Software_Detail.jsp?software_id=54">http://www.loni.ucla.edu/Software/Software_Detail.jsp?software_id=54</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, MINC, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, MINC, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="loni-pipeline-environment" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="LONI Pipeline Environment">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=loni-pipeline-environment" rel="bookmark" name="loni-pipeline-environment">LONI Pipeline Environment</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The LONI Pipeline is a free workflow application primarily aimed at neuroimaging researchers, but is useful for many other fields of science. The Pipeline Client runs on your PC/Mac/Linux computer upon which you can create sophisticated processing workflows using a variety of commonly available executable tools (e.g. FSL, AIR, FreeSurfer, AFNI, Diffusion Toolkit, etc). The Distributed Pipeline Server can be installed on your Linux cluster and you can submit processing jobs directly to your own compute systems. Visit <a href="http://pipeline.loni.usc.edu" target="_new">http://pipeline.loni.usc.edu</a> for more info.<br />
<br />
The Pipeline is distributed by the Laboratory of Neuro Imaging (<a href="http://www.loni.usc.edu/Software/Pipeline" target="_new">http://www.loni.usc.edu/Software/Pipelin...</a>). 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=32&screenshot_id=282"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=32&screenshot_id=282" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 25, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>LONI Software License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://pipeline.loni.usc.edu">http://pipeline.loni.usc.edu</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Southern California <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>SunOS/Solaris<br>Windows NT/2000</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>AFNI BRIK, ANALYZE, DICOM, MGH/MGZ, MINC, MINC2, NIfTI-1, Other Format, bshort/bfloat</td></tr>
			<tr class="even"><td>Output Formats</td><td>AFNI BRIK, ANALYZE, DICOM, MGH/MGZ, MINC, MINC2, NIfTI-1, Other Format, bshort/bfloat</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="loni-provenance-editor" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="LONI Provenance Editor">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=loni-provenance-editor" rel="bookmark" name="loni-provenance-editor">LONI Provenance Editor</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The LONI Provenance Editor (<a href="http://www.loni.usc.edu/Software/ProvenanceEditor" target="_new">http://www.loni.usc.edu/Software/Provena...</a>) is a self-contained, platform-independent application that automatically extracts the provenance information from an image header (such as a DICOM image) and generates a data provenance XML file with that information.<br />
<br />
Provenance Editor is distributed by the Laboratory of Neuro Imaging (<a href="http://www.LONI.usc.edu/Software/ProvenanceEditor" target="_new">http://www.LONI.usc.edu/Software/Provena...</a>) at USC. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=305&screenshot_id=412"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=305&screenshot_id=412" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 8, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>LONI Software License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/provenance/">http://www.nitrc.org/projects/provenance/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Southern California <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="lop-dwi" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="lop-DWI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=lop-dwi" rel="bookmark" name="lop-dwi">lop-DWI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> For information about lo-DWI and the data please visit: <br />
<a href="https://sites.google.com/site/lopdwi/home" target="_new">https://sites.google.com/site/lopdwi/hom...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 5, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Data License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/lop-dwi/">http://www.nitrc.org/projects/lop-dwi/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Queensland <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="loris" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="LORIS">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=loris" rel="bookmark" name="loris">LORIS</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database-application" rel="tag">Database Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> LORIS (Longitudinal Online Research and Imaging System) is an open-source database solution platform for neuroimaging and neuroscience research. <br />
<br />
LORIS is a platform designed for seamless data acquisition, curation, processing, and dissemination across large neuroimaging, clinical and genomic datasets. Its secure web-accessible database infrastructure automates the flow of data throughout the lifecycle of multi-site longitudinal studies, and includes extensive Quality Control and Data Validation processes for both scalar and multidimensional data. <br />
<br />
LORIS’ extensible design enables researchers to aggregate, query, and distribute subject data in a powerfully customizable and flexible manner to image processing pipelines and analysis suites frequently used in neuroimaging research studies.<br />
<br />
More info? visit <a href="http://www.LORIS.CA" target="_new">http://www.LORIS.CA</a><br />
<br />
Try our demonstration database at <a href="https://demo.loris.ca" target="_new">https://demo.loris.ca</a><br />
<br />
Find our latest Release and Documentation Wiki on GitHub --<br />
<a href="https://github.com/aces/Loris" target="_new">https://github.com/aces/Loris</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=542&screenshot_id=1002"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=542&screenshot_id=1002" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 5, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://loris.ca">http://loris.ca</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Montreal Neurological Institute <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>MINC, MINC2</td></tr>
			<tr class="even"><td>Output Formats</td><td>MINC, MINC2</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="low-resolution-brain-electromagnetic-tomography" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Low Resolution Brain Electromagnetic Tomography">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=low-resolution-brain-electromagnetic-tomography" rel="bookmark" name="low-resolution-brain-electromagnetic-tomography">Low Resolution Brain Electromagnetic Tomography</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=eeg-modeling" rel="tag">Eeg Modeling</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Non-invasive scalp measurements of electric potentials (EEG) [as well as magnetic fields (MEG)] can be used for estimating the electric neuronal activity distribution (current density vector field) on the cortex. The methods used here are LORETA, standardized LORETA (sLORETA), and exact LORETA (eLORETA).<br />
Time series of cortical electric neuronal activity computed with LORETA can be used for estimating intracortical functional and effective connectivity. A collection of new methods are included here, especially well-suited for signals from these tomographies:<br />
1. Lagged coherence and lagged phase synchronization.<br />
2. Dynamic intracortical connectivity in terms of senders, hubs, and receivers.<br />
3. Partial coherence fields.<br />
4. Isolated effective coherence.<br />
5. Functional ICA (in the sense of functional data analysis) for discovering generalized functional connectivity across space, frequency, and time. <br />
All these methods and much more are programmed and available in the alpha LORETA-KEY software package. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=475&screenshot_id=802"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=475&screenshot_id=802" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 3, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/loreta/">http://www.nitrc.org/projects/loreta/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>The KEY Institute for Brain-Mind Research <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="lumina-lp--400-response-system" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Lumina LP- 400 Response System">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=lumina-lp--400-response-system" rel="bookmark" name="lumina-lp--400-response-system">Lumina LP- 400 Response System</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=response-monitoring" rel="tag">Response Monitoring</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Lumina LP-400 is a reliable patient response system designed specifically for use in an fMRI. Lumina was developed to satisfy the requirements of both the clinical and research fields. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 14, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://cedrus.com/ordering/usa/">http://cedrus.com/ordering/usa/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Cedrus Corporation <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="lupus-patients-with-white-matter-lesions" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Lupus patients with white matter lesions">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=lupus-patients-with-white-matter-lesions" rel="bookmark" name="lupus-patients-with-white-matter-lesions">Lupus patients with white matter lesions</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Data set consisting of 20 Lupus patients (T1w, FLAIR, FLAIR corrected, WML). The correction of the FLAIR images consists of the skull removal and the intensity correction (anisotropic filter and bias correction).<br />
<br />
This data set has been used for experimental purposes evaluating an automated approach for the detection of Lupus WML in MRI images. Lupus WML appear as small focal abnormal tissue observed as hyperintensities in the FLAIR images. The quantification of these WML is a key factor for the stratification of lupus patients and therefore both lesion detection and segmentation play an important role. Therefore, this resource can be used either for reproducibility of the results presented in the work mentioned above, or for testing new approaches with similar purposes. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 27, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/lupus_2016/">http://www.nitrc.org/projects/lupus_2016/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Hospital Clínic, University of Barcelona <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="m3" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="M3">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=m3" rel="bookmark" name="m3">M3</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The M3 (multi-modal imaging and multi-level characteristic with multi-classifier) is a brain imaging classification tool, which can help researchers to discriminate patients from normal controls. The M3 includes three steps: feature selection, maximum uncertainty linear discriminant analysis (MLDA)-based classification and multi-classifier. A leave-one-out cross-validation (LOOCV) is further used to estimate the performance of the M3. Finally, the most discriminative features are identified. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 25, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/pare/">http://www.nitrc.org/projects/pare/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Beijing Normal University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mabmis-for-slicer-4:-multi-atlas-based-multi-image-segmentation" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MABMIS for Slicer 4: Multi-Atlas Based Multi-Image Segmentation">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mabmis-for-slicer-4:-multi-atlas-based-multi-image-segmentation" rel="bookmark" name="mabmis-for-slicer-4:-multi-atlas-based-multi-image-segmentation">MABMIS for Slicer 4: Multi-Atlas Based Multi-Image Segmentation</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MABMIS is a module for Slicer 4 that implements a multi-atlas based multi-image method for group-wise segmentation. (H. Jia, P-T Yap, D. Shen, &quot;Iterative multi-atlas-based multi-image segmentation with tree-based registration&quot;, NeuroImage, 59:422-430, 2012. )<br />
The method utilizes a novel tree-based groupwise registration method for concurrent alignment of both the atlases and the target images, and an iterative groupwise segmentation method for simultaneous consideration of segmentation information propagated from all available images, including the atlases and other newly segmented target images.<br />
<br />
MABMIS contains a training step and a testing step. In the training step, a groupwise registration is performed to generate a tree structure for the atlases. In the testing step, a group of target images are simultaneously and iteratived segmented using the pre-constructed atlas tree. <br />
<br />
Prior to applying MABMIS, the atlas images and target images need be skullstripped, and linearly transformed to a common space. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 2, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>3D Slicer License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mabmis_slicer4/">http://www.nitrc.org/projects/mabmis_slicer4/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UNC/GE <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mabmis:-multi-atlas-based-segmentation" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MABMIS: Multi-Atlas Based Segmentation">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mabmis:-multi-atlas-based-segmentation" rel="bookmark" name="mabmis:-multi-atlas-based-segmentation">MABMIS: Multi-Atlas Based Segmentation</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This software package implements MABMIS: Multi-Atlas-Based Multi-Image Segmentation – an algorithm for accurate and consistent segmentation/labeling on a group of images. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 28, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mabmis/">http://www.nitrc.org/projects/mabmis/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mach" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MaCH">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mach" rel="bookmark" name="mach">MaCH</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=imputation" rel="tag">Imputation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MaCH is a tool for haplotyping, genotype imputation and disease association analysis developed by Goncalo Abecasis and Yun Li. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 13, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mach/">http://www.nitrc.org/projects/mach/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mach2dat" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="mach2dat">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mach2dat" rel="bookmark" name="mach2dat">mach2dat</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=genetic-association" rel="tag">Genetic Association</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> mach2dat performs logistic regression, using imputed SNP dosage data and adjusting for covariates. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 13, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mach2dat/">http://www.nitrc.org/projects/mach2dat/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina at Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mach2qtl" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="mach2qtl">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mach2qtl" rel="bookmark" name="mach2qtl">mach2qtl</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=genetic-association" rel="tag">Genetic Association</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> QTL analysis based on imputed dosages/posterior_probabilities. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 13, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mach2qtl/">http://www.nitrc.org/projects/mach2qtl/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina at Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mach-admix" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MaCH-Admix">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mach-admix" rel="bookmark" name="mach-admix">MaCH-Admix</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=imputation" rel="tag">Imputation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MaCH-Admix is an extension to MaCH for faster and more flexible imputaiton, especially in admixed populations. <br />
<br />
It has incorporated a novel piecewise reference selection method to create reference panels tailored for target individual(s). This reference selection method generates better imputation quality in shorter running time. MaCH-Admix also separates model parameter estimation from imputation. The separation allows users to perform imputation with standard reference panels + pre-calibrated parameters in a data independent fashion. Alternatively, if one works with study-specific reference panels, or isolated target population, one has the option to simultaneously estimate these model parameters while performing imputation. MaCH-Admix has included many other useful options and supports VCF input files. All existing MaCH documentation applies to MaCH-Admix. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 13, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mach-admix/">http://www.nitrc.org/projects/mach-admix/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina at Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mag-design-and-engineering" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Mag Design and Engineering">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mag-design-and-engineering" rel="bookmark" name="mag-design-and-engineering">Mag Design and Engineering</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=optical-imaging" rel="tag">Optical Imaging</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=response-monitoring" rel="tag">Response Monitoring</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=stimulus-presentation" rel="tag">Stimulus Presentation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Mag Design and Engineering sells a variety of MEG- and fMRI-compatible hardware for research use. These items include typical response collection devices such as joysticks, response pads, mice, as well as stimulation devices such as vibrotactile stimulators, olfactometers, and pressure/force generators. The company also offers custom design and production services for many different applications. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=493&screenshot_id=386"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=493&screenshot_id=386" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 29, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Commercial License: MAG</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.magconcept.com">http://www.magconcept.com</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Mag Design and Engineering <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="magnetic-resonance-angiography-atlas-dataset" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Magnetic Resonance Angiography Atlas Dataset">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=magnetic-resonance-angiography-atlas-dataset" rel="bookmark" name="magnetic-resonance-angiography-atlas-dataset">Magnetic Resonance Angiography Atlas Dataset</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Characterization of the complex branching architecture of cerebral arteries across a representative sample of the human population is important for diagnosing, analyzing, and predicting pathological states. Brain arterial vasculature can be visualized by magnetic resonance angiography (MRA). However, most MRA studies are limited to qualitative assessments, partial morphometric analyses, individual (or small numbers of) subjects, proprietary datasets, or combinations of the above limitations. Neuroinformatics tools, developed for neuronal arbor analysis, were used to quantify vascular morphology from 3T time-of-flight MRA high-resolution (620 μm isotropic) images collected in 61 healthy volunteers (36/25 F/M, average age=31.2 ± 10.7, range=19-64 years). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 31, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/icbmmra/">http://www.nitrc.org/projects/icbmmra/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="magpro-magnetic-stimulator-(tms)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MagPro Magnetic Stimulator (TMS)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=magpro-magnetic-stimulator-(tms)" rel="bookmark" name="magpro-magnetic-stimulator-(tms)">MagPro Magnetic Stimulator (TMS)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=response-monitoring" rel="tag">Response Monitoring</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=stimulus-presentation" rel="tag">Stimulus Presentation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MagPro is a complete line of non-invasive magnetic stimulation systems designed for clinical examinations and for research in the areas of neurophysiology, neurology, cognitive neuroscience, rehabilitation and psychiatry. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 13, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MagPro Medical Device License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.magventure.com">http://www.magventure.com</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>MagVenture <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Microsoft Word Document</td></tr>
			<tr class="even"><td>Output Formats</td><td>Microsoft Word Document</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mango" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Mango">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mango" rel="bookmark" name="mango">Mango</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=platform-or-development-environment" rel="tag">Platform Or Development Environment</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=region-of-interest" rel="tag">Region Of Interest</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=registration" rel="tag">Registration</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=rendering" rel="tag">Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=statistical-operation" rel="tag">Statistical Operation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-analysis" rel="tag">Surface Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=temporal-transformation" rel="tag">Temporal Transformation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volumetric-analysis" rel="tag">Volumetric Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=websites" rel="tag">Websites</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Mango is a viewer for medical research images. It provides analysis tools and a user interface to navigate image volumes. There are three versions of Mango, each geared for a different platform:<br />
<br />
* Mango – Desktop – Mac OS X, Windows, and Linux<br />
* webMango – Browser – Safari, Firefox, Chrome, and Internet Explorer<br />
* iMango – Mobile – Apple iPad<br />
<br />
Key Features:<br />
- Built-in support for DICOM, NIFTI, Analyze, and NEMA-DES formats<br />
- Customizable: Create plugins, custom filters, color tables, file formats, and atlases<br />
- ROI Editing: Threshold and component-based tools for painting and tracing ROIs<br />
- Surface Rendering: Interactive surface models supporting cut planes and overlays<br />
- Image Registration: Semi-automatic image coregistration and manual transform editing<br />
- Image Stacking: Threshold and transparency-based image overlay stacking<br />
- Analysis: Histogram, cross-section, time-series analysis, image and ROI statistics<br />
- Processing: Kernel and rank filtering, arithmetic/logic image and ROI calculators 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=100&screenshot_id=213"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=100&screenshot_id=213" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 15, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://ric.uthscsa.edu/mango/">http://ric.uthscsa.edu/mango/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UTHSCSA <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>OS Independent<br>SunOS/Solaris<br>Windows Vista<br>Windows XP</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, GIfTI, NIfTI-1, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, GIfTI, NIfTI-1, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="manually-labeled-mri-brain-scan-database" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Manually Labeled MRI Brain Scan Database">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=manually-labeled-mri-brain-scan-database" rel="bookmark" name="manually-labeled-mri-brain-scan-database">Manually Labeled MRI Brain Scan Database</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-data" rel="tag">Atlas Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data-model" rel="tag">Data Model</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=test-data" rel="tag">Test Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This is a continuously growing and improving database of high-quality neuroanatomically labeled MRI brain scans, created not by an algorithm, but by neuroanatomical experts. All results are checked and corrected. Regions of interest include the usual sub-cortical structures (thalamus, caudate, putamen, hippocampus, etc), along with ventricles, brain stem, cerebellum, and gray and white matter. We also sub-divide the cortex into &quot;parcellation units&quot; that are defined by gyral and sulcal landmarks. There are 157 ROIs now and more to come.<br />
This data is offered as a subscription and while it is not free, it is a tiny fraction of the cost of creating the database. The idea is to spread the cost of adding new labeled scans to the database so we can continue to increase the number of scans, along with the age range and other demographics of the subjects. In 2012, we will have at least 63 comprehensively labeled scans available. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=656&screenshot_id=664"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=656&screenshot_id=664" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 17, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Other/Proprietary License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://neuromorphometrics.com/?page_id=23">http://neuromorphometrics.com/?page_id=23</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Neuromorphometrics, Inc. <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="manualwmh-imagej-macros" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ManualWMH ImageJ Macros">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=manualwmh-imagej-macros" rel="bookmark" name="manualwmh-imagej-macros">ManualWMH ImageJ Macros</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ManualWMH is a set of ImageJ macros that facilitates the efficient manual segmentation of white matter hyperintensities (WMH / WMHI / leukoaraiosis) in T2 FLAIR images. It outputs volumetric 3D WMH masks and descriptive statistics. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1324&screenshot_id=1241"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1324&screenshot_id=1241" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 10, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License v3</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://github.com/stowler/ManualWMH">https://github.com/stowler/ManualWMH</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="map3d" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="map3d">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=map3d" rel="bookmark" name="map3d">map3d</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=forward---inverse" rel="tag">Forward - Inverse</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> map3d supports interactive visualization of surface based time signals such as high resolution EEG, MEG, and ECG. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 16, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.sci.utah.edu/cibc-software/map3d.html">http://www.sci.utah.edu/cibc-software/map3d.html</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows 95/98/2000</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mapping-population-based-structural-connectomes" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Mapping population-based structural connectomes">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mapping-population-based-structural-connectomes" rel="bookmark" name="mapping-population-based-structural-connectomes">Mapping population-based structural connectomes</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Our developed PSC framework is a workflow which can simultaneously characterize a large number of white matter bundles within and across different subjects for group analysis. Given the DWI and T1 images it has three major components: (i) reliable construction of the structural connectome for the whole brain, including a robust tractography algorithm and streamline post-processing techniques, such as dilation of gray matter regions, streamline cutting, and outlier streamline removal are applied to improve the robustness of the extracted structural connectomes; (ii) low-dimensional representation of streamlines in each connection, including a shape analysis framework to separate the variation of streamlines in each cell of the streamlines, and an encoding and decoding procedure to efﬁciently compress the streamlines; (iii) multi-level connectome analysis, including the groupwise connectome analysis at three different levels, the streamline level, the weighted network level; and the binary network level. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 23, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/psc/">http://www.nitrc.org/projects/psc/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UNC, BIAS <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="maps4mipav-(exploratory-jist)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Maps4Mipav (Exploratory JIST)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=maps4mipav-(exploratory-jist)" rel="bookmark" name="maps4mipav-(exploratory-jist)">Maps4Mipav (Exploratory JIST)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Java Image Science Toolkit (JIST) is an extension to the MIPAV (Medical Image Processing, Analysis, and Visualization) plug-in framework that allows the user to design and execute pipelines, which are multi-stage processing tasks.<br />
<br />
JIST was formerly known as the MedIC Automated Pipeline Scheduler (MAPS).<br />
<br />
This is the exploratory development tree. New features and designs are tested here before general release into the JIST project. <br />
<br />
Access to the source code is freely given. However, registration is required so that we can notify users of critical changes. For unrestricted access to the stable release, see the JIST project. <a href="http://www.nitrc.org/projects/jist/" target="_new">http://www.nitrc.org/projects/jist/</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 9, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/maps4mipav/">http://www.nitrc.org/projects/maps4mipav/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="marsbar-region-of-interest-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MarsBaR region of interest toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=marsbar-region-of-interest-toolbox" rel="bookmark" name="marsbar-region-of-interest-toolbox">MarsBaR region of interest toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=linear" rel="tag">Linear</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MarsBaR (MARSeille Boîte À Région d'Intérêt) is a toolbox for SPM which provides routines for region of interest analysis. Features include region of interest definition, combination of regions of interest with simple algebra, extraction of data for regions with and without SPM preprocessing (scaling, filtering), and statistical analyses of ROI data using the SPM statistics machinery. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 1, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://marsbar.sourceforge.net/">http://marsbar.sourceforge.net/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mars-(multi-atlas-robust-segmentation)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MARS (Multi-Atlas Robust Segmentation)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mars-(multi-atlas-robust-segmentation)" rel="bookmark" name="mars-(multi-atlas-robust-segmentation)">MARS (Multi-Atlas Robust Segmentation)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MARS (Multi-Atlas Robust Segmentation) provides the automatic solutions for efficent segmentation/labeling anatomcial structures from medical images.<br />
<br />
Specifically, this software has integrated several state-of-the-art multi-atlas based segmentation methods, such as majority voting, local weighted voting, and non-local patch based segmentation methods. <br />
<br />
More importantly, we also included our recently-developed joint sparse patch based segmentation method in this software. Compared with convention methods, our method has the following advantages: (1) add sparsity constraint to suppress the influence of misleading patches; (2) reduce the joint risk of two patches jointly making the same segmentation errors, and (3) use iterative framework to correct the possible mis-segmentations. <br />
<br />
This software package was developed in the IDEA group at UNC-Chapel Hill ( <a href="http://bric.unc.edu/ideagroup" target="_new">http://bric.unc.edu/ideagroup</a>).<br />
Wu et. al., &quot;A generative probability model of joint label fusion for multi-atlas based brain segmentation&quot;, MIA, 2013. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=758&screenshot_id=684"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=758&screenshot_id=684" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 23, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mars/">http://www.nitrc.org/projects/mars/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina at Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="masi_hardi_reproducibility" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="masi_hardi_reproducibility">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=masi_hardi_reproducibility" rel="bookmark" name="masi_hardi_reproducibility">masi_hardi_reproducibility</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Massive single subject reproducibility data set acquired with HARDI suggested acquisition schemes </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 18, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MASI</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/masi_hardi_repr/">http://www.nitrc.org/projects/masi_hardi_repr/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Vanderbilt <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="masi-label-fusion" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MASI Label Fusion">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=masi-label-fusion" rel="bookmark" name="masi-label-fusion">MASI Label Fusion</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This collection of tools is maintained by the Medical-image Analysis and Statistical Interpretation (MASI) group at Vanderbilt University.<br />
<br />
The purpose of this tool is to provide a unified framework for testing and applying statistical and voting label fusion techniques. The project will include implementations of several different voting techniques including majority vote, weighted voting, and regionally weighted voting. Additionally, multiple statistical fusion methods will be included, notably, STAPLE, Spatial STAPLE, STAPLER and COLLATE.<br />
<br />
In addition to the fusion algorithms, code for running specialized simulations and various tools and utilities to test the efficacy of the algorithms will be provided. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 10, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Lesser General Public License (LGPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/masi-fusion/">http://www.nitrc.org/projects/masi-fusion/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="masimatlab" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MASIMatlab">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=masimatlab" rel="bookmark" name="masimatlab">MASIMatlab</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The MASI research laboratory concentrates on analyzing large-scale cross-sectional and longitudinal neuroimaging data. Specifically, we are interested in population characterization with magnetic resonance imaging (MRI), multi-parametric studies (DTI, sMRI, qMRI), and shape modeling.<br />
<br />
This repository stores and provides opportunities for collaboration through Matlab code, libraries, and configuration information for projects in early stage development. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 11, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/masimatlab/">http://www.nitrc.org/projects/masimatlab/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="masked-ica-(mica)-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="masked ICA (mICA) Toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=masked-ica-(mica)-toolbox" rel="bookmark" name="masked-ica-(mica)-toolbox">masked ICA (mICA) Toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=independent-component-analysis" rel="tag">Independent Component Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> mICA Toolbox is a user-friendly toolbox to perform masked independent component analysis, i.e. ICA within a spatially restricted subregion of the brain. It is based on command line tools from FSL suite to perform the ICA and related analyses (e.g. specific data preprocessing, atlas-based mask generation, mICA-based parcellation, dual-regression and direct back reconstruction). Various options provide flexible control of the analysis.<br />
<br />
mICA Toolbox is a helpful utility for researchers in the field of neuroimaging. It facillitates the study of intrinsic connectivity of brain (sub-)regions and the reconstruction of their extrinsic connectivity (i.e. connectivity with regions outside of the mask).<br />
<br />
The toolbox furthermore offers an easy way to calculate ICA reproducibility over a range of decomposition dimensions, which can be used to overcome the common problem of dimensionality estimation. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=937&screenshot_id=929"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=937&screenshot_id=929" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 29, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mica/">http://www.nitrc.org/projects/mica/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Hannover Medical School <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mask_explorer" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="mask_explorer">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mask_explorer" rel="bookmark" name="mask_explorer">mask_explorer</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Tool mask_explorer is used to explore fMRI datasets and to find inconsistences in fMRI data, it is suitable especially for group analyses.<br />
<br />
Main features of mask_explorer are:<br />
1) Estimation of binary masks<br />
2) Exploration and export of valid subject counts image and binary group mask<br />
3) Estimation of outlier masks in dataset<br />
4) Creating list of suitable/unsuitable data in relation of positions and ROIs (coverage by masks included) and export to Microsoft Excel or text file. <br />
5) Creating and loading list of positions of interest<br />
6) Compare valid subject counts image with NIFTI file<br />
7) Use of user specified background template<br />
<br />
Mask_explorer can compute subject’s masks. Loaded masks are used to create parametric map. Its values show number of datasets with valid information.<br />
<br />
The best citations for this software is:<br />
<br />
Gajdoš M, Mikl M, Mareček R. Mask_explorer: a tool for exploring brain masks in fMRI group analysis. Computer Methods and Programs in Biomedicine. 2016 Jul. DOI: 10.1016/j.cmpb.2016.07.015 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=916&screenshot_id=917"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=916&screenshot_id=917" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 20, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU GPL v2</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mask_explorer/">http://www.nitrc.org/projects/mask_explorer/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Masaryk University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="massively-expedited-genome-wide-heritability-analysis-(megha)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Massively Expedited Genome-wide Heritability Analysis (MEGHA)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=massively-expedited-genome-wide-heritability-analysis-(megha)" rel="bookmark" name="massively-expedited-genome-wide-heritability-analysis-(megha)">Massively Expedited Genome-wide Heritability Analysis (MEGHA)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> A MATLAB toolbox for fast and flexible high-dimensional heritability analysis using genome-wide single nucleotide polymorphism (SNP) data from unrelated individuals, developed in the article &quot;Massively Expedited Genome-wide Heritability Analysis (MEGHA)&quot;. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 3, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://scholar.harvard.edu/tge/software/megha">http://scholar.harvard.edu/tge/software/megha</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Massachusetts General Hospital / Harvard Medical School <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="matlab-tutorial-on-diffusion-tensor-mri" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MATLAB Tutorial on Diffusion Tensor MRI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=matlab-tutorial-on-diffusion-tensor-mri" rel="bookmark" name="matlab-tutorial-on-diffusion-tensor-mri">MATLAB Tutorial on Diffusion Tensor MRI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This is an on-line tutorial on how to use MATLAB for Diffusion-Weighted MRI processing. The following subjects are covered in this tutorial: Generation of Synthetic Diffusion-Weighted MRI datasets, Diffusion Tensor (DTI) Estimation from DW-MRI, DTI Visualization as a field of ellipsoids, Higher-order Diffusion Tensor Estimation from DW-MRI, Computing of Tensor Orientation Distribution Functions (Tensor ODF), Computing of Fiber Orientations, Higher-order Diffusion Tensor Image Visualization as fields of spherical functions, Multi-fiber reconstruction etc.<br />
<br />
The tutorial contains numerous illustrations, figures and Matlab scripts embedded in the text. The reader/user can automatically generate Matlab script for a self-designed DW-MRI experiment by selecting which steps needs to be followed. The code that corresponds to the selected steps is then appropriately merged in the &quot;Matlab Script Generator&quot;, and the user can easily copy and paste the produced code directly to the Matlab command prompt. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 6, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dwmri_in_matlab/">http://www.nitrc.org/projects/dwmri_in_matlab/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mayo-clinic-adult-lifespan-template-and-atlases" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Mayo Clinic Adult Lifespan Template and Atlases">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mayo-clinic-adult-lifespan-template-and-atlases" rel="bookmark" name="mayo-clinic-adult-lifespan-template-and-atlases">Mayo Clinic Adult Lifespan Template and Atlases</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-data" rel="tag">Atlas Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-software-resource" rel="tag">Other Software Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Mayo Clinic Adult Lifespan Template and its associated atlases were constructed and made publicly available to provide a template suitable for the analysis needs of aging and Alzheimer’s Disease population studies. Population-matched templates are known to allow more accurate quantitative MRI analysis, but most MRI standard templates are generated from scans of younger individuals. Unlike these, the MCALT is designed for analysis of MRI of adult subjects age 30+. MCALT was designed for use with SPM12 and integrates easily into its workflows, but is not specific to SPM12 and may be used easily with other segmentation software. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1189&screenshot_id=1109"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1189&screenshot_id=1109" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 28, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Mixed: CC-by-NC-SA and GPL (MCALT)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mcalt/">http://www.nitrc.org/projects/mcalt/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Mayo Clinic <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mazesuite" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MazeSuite">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mazesuite" rel="bookmark" name="mazesuite">MazeSuite</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MazeSuite is a complete set of tools that enables researchers to perform spatial and navigational behavior experiments within interactive, easy to create, and extendable (e.g., multiple rooms) 3D virtual environments. MazeSuite can be used to design/edit adapted 3D environments where subjects’ behavioral performance can be tracked. MazeSuite consists of three main applications; an editing program to create and alter maps (MazeMaker), a visualization/rendering module (MazeWalker), and finally an analysis/mapping tool (MazeAnalyzer). Additionally, MazeSuite has the capabilities of sending signal pulses to physiological recording devices using standard computer ports. MazeSuite, with all 3 applications, is a unique and complete toolset for researchers who want to easily and rapidly deploy interactive 3D environments. For more information and related publications please see <a href="http://www.mazesuite.com" target="_new">http://www.mazesuite.com</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 11, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Non-Commercial Software License Agreement</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mazesuite/">http://www.nitrc.org/projects/mazesuite/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Drexel University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mcic--schizophrenic-and-matched-control-data" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MCIC- schizophrenic and matched control data">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mcic--schizophrenic-and-matched-control-data" rel="bookmark" name="mcic--schizophrenic-and-matched-control-data">MCIC- schizophrenic and matched control data</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Expertly collected, well-curated data sets consisting of comprehensive clinical characterization and raw structural, functional and diffusion-weighted DICOM images in schizophrenia patients and gender and age-matched controls are now accessible to the scientific community through an on-line data repository (coins.mrn.org). This data repository will be useful to 1) educators in the fields of neuroimaging, medical image analysis and medical imaging informatics who need exemplar data sets for courses and workshops; 2) computer scientists and software algorithm developers for testing and validating novel registration, segmentation, and other analysis software; and 3) scientists who can study schizophrenia by further analysis of this cohort and/or by pooling with other data. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 17, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>COINS Data Use Agreement</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mcic/">http://www.nitrc.org/projects/mcic/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Massachusetts General Hospital <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>DICOM</td></tr>
			<tr class="even"><td>Output Formats</td><td>DICOM</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mcml-&-conv" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MCML & CONV">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mcml-&-conv" rel="bookmark" name="mcml-&-conv">MCML & CONV</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MCML is a Monte Carlo simulation program for Multi-layered Turbid<br />
Media with an infinitely narrow photon beam as the light source. The<br />
simulation is specified by an input text file called, for example,<br />
&quot;sample.mci&quot;, which can be modified by any simple text editor. The<br />
output is another text file called, for example, &quot;sample.mco&quot;. (The<br />
names are arbitrary.)<br />
<br />
CONV is a convolution program which uses the MCML output file to<br />
convolve for photon beams of any size in a Gaussian or flat<br />
field shape. CONV can provide a variety of output formats (reflectance,<br />
transmission, iso-fluence contours, etc.), which are compatible with<br />
standard graphics applications. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 9, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Open Software License 3.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mcml/">http://www.nitrc.org/projects/mcml/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Washington University in St. Louis <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mdr-multifactor-dimensionality-reduction" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MDR Multifactor Dimensionality Reduction">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mdr-multifactor-dimensionality-reduction" rel="bookmark" name="mdr-multifactor-dimensionality-reduction">MDR Multifactor Dimensionality Reduction</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=genetic-association" rel="tag">Genetic Association</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Multifactor Dimensionality Reduction (MDR) is a nonparametric and genetic model-free machine learning alternative to logistic regression for detecting and characterizing nonlinear interactions among discrete genetic and environmental attributes. The MDR method combines attribute selection, attribute construction, and classification with cross-validation and permutation testing to provide a comprehensive and powerful approach to detecting epistasis. MDR has been successfully applied to numerous different diseases and is currently being adapted to genome-wide assoication studies (GWAS). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 5, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.multifactordimensionalityreduction.org/">http://www.multifactordimensionalityreduction.org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Geisel Medical School at Dartmouth College <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="measure-projection-toolbox-(mpt)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Measure Projection Toolbox (MPT)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=measure-projection-toolbox-(mpt)" rel="bookmark" name="measure-projection-toolbox-(mpt)">Measure Projection Toolbox (MPT)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surrogate-data-analysis" rel="tag">Surrogate Data Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This toolbox is an EEGLAB plugin for performing Measure Projection Analysis. <br />
<br />
Measure Projection Analysis (MPA) is a novel probabilistic multi-subject inference method that overcomes EEG Independent Component (IC) clustering issues by abandoning the notion of distinct IC clusters. Instead, it searches voxel by voxel for brain regions having event-related IC process dynamics that exhibit statistically significant consistency across subjects and/or sessions as quantified by the values of various EEG measures. <br />
<br />
Local-mean EEG measure values are then assigned to all such locations based on a probabilistic model of IC localization error and inter-subject anatomical and functional differences. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=560&screenshot_id=463"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=560&screenshot_id=463" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 6, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MPT</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://sccn.ucsd.edu/wiki/MPT">http://sccn.ucsd.edu/wiki/MPT</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of California San Diego <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="measuring-eye-states-in-functional-mri" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Measuring eye states in functional MRI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=measuring-eye-states-in-functional-mri" rel="bookmark" name="measuring-eye-states-in-functional-mri">Measuring eye states in functional MRI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> An automized tool to segment eye bulbs and measure movements and eye state. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 20, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/eye_state_fmri/">http://www.nitrc.org/projects/eye_state_fmri/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>FSU Jena / Neurology <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="medical-image-processing-and-visualization-in-virtual-environments" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Medical Image Processing and Visualization in Virtual Environments">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=medical-image-processing-and-visualization-in-virtual-environments" rel="bookmark" name="medical-image-processing-and-visualization-in-virtual-environments">Medical Image Processing and Visualization in Virtual Environments</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This resouce will centralize development of tools for interaction with medical imaging data in immersive virtual environments (based on the Vizard platform). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 10, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Lesser General Public License (LGPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/medvr/">http://www.nitrc.org/projects/medvr/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Vanderbilt Unveriversity <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="medinria" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MedINRIA">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=medinria" rel="bookmark" name="medinria">MedINRIA</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MedINRIA allows to process and analyze a wide range of magnetic resonance (MR) images including anatomical MRI, functional MRI (fMRI), and diffusion tensor MRI (DT-MRI). MedINRIA is intended to be used by anyone curious about medical images! </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 9, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://med.inria.fr/">http://med.inria.fr/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="megsim" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MEGSIM">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=megsim" rel="bookmark" name="megsim">MEGSIM</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data-model" rel="tag">Data Model</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=meg-modeling" rel="tag">Meg Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other" rel="tag">Other</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=test-data" rel="tag">Test Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MEGSIM, <a href="http://cobre.mrn.org/megsim/," target="_new">http://cobre.mrn.org/megsim/,</a> contains realistic simulated MEG datasets ranging from basic sensory to oscillatory sets that mimic functional connectivity; as well as basic visual, auditory, and somatosensory empirical sets. The simulated sets were created for the purpose of testing analysis algorithms across the different MEG systems when the truth is known. MEG baseline recordings were obtained from 5 healthy participants, using three MEG systems: VSM/CTF Omega, Elekta Neuromag Vectorview, 4-D Magnes 3600. Simulated signals were embedded within the CTF and Neuromag 306 baseline recordings (4-D to be added). Participant MRIs are available. Averaged simulation files are available as netcdf files. Neuromag 306 averaged simulations are also available in fif format. Also available: single trials of data where the simulated signal is jittered about a mean value, continuous fif files where the simulated signal is marked by a trigger, and simulations with oscillations added to mimic functional connectivity. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=538&screenshot_id=616"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=538&screenshot_id=616" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 10, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/megsim/">http://www.nitrc.org/projects/megsim/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of New Mexico & The MIND Research Network <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="meg-tools" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MEG Tools">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=meg-tools" rel="bookmark" name="meg-tools">MEG Tools</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-reconstruction" rel="tag">Image Reconstruction</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MEG Localization using FOCUSS and MR-FOCUSS </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 6, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.megimaging.com/">http://www.megimaging.com/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Microsoft</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="menga:-multimodal-environment-for-neuroimaging-and-genomic-analysis" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MENGA: Multimodal Environment for Neuroimaging and Genomic Analysis">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=menga:-multimodal-environment-for-neuroimaging-and-genomic-analysis" rel="bookmark" name="menga:-multimodal-environment-for-neuroimaging-and-genomic-analysis">MENGA: Multimodal Environment for Neuroimaging and Genomic Analysis</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MENGA is a platform for the integration of imaging data and Allen Human Brain Atlas mRNA data. MENGA aims to provide a comprehensive environment to investigate correlation patterns between various imaging modalities and gene expression profiles based on the Allen Brain Atlas.<br />
<br />
The project comes from a collaboration between the Department of Information Engineering (DEI), University of Padova with the Institute of Psychiatry, Psychology and Neuroscience (IoPPN), King’s College, London (UK).<br />
<br />
For any enquiries, please contact Gaia Rizzo, Mattia Veronese or Paul Expert.<br />
<br />
Software: <a href="http://fair.dei.unipd.it/require-software/" target="_new">http://fair.dei.unipd.it/require-softwar...</a><br />
<br />
We have now published the reference paper of MENGA on PlosOne, in which all the aspects of the software and its functionalities are described in details: <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0148744" target="_new">http://journals.plos.org/plosone/article...</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1000&screenshot_id=967"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1000&screenshot_id=967" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 13, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://fair.dei.unipd.it/software/">http://fair.dei.unipd.it/software/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Partnership University of Padova / King’s College London <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mesh-based-monte-carlo-(mmc)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Mesh-based Monte Carlo (MMC)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mesh-based-monte-carlo-(mmc)" rel="bookmark" name="mesh-based-monte-carlo-(mmc)">Mesh-based Monte Carlo (MMC)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=monte-carlo" rel="tag">Monte Carlo</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Mesh-based Monte Carlo, or MMC, is a Monte Carlo (MC) solver for photon migration in 3D turbid media. Different from existing MC software designed for layered (such as MCML) or voxel-based media (such as MMC or tMCimg), MMC can represent a complex domain using a tetrahedral mesh. This not only greatly improves the accuracy of the solutions when modeling objects with smooth/complex boundaries, but also gives an efficient way to sample the problem domain to use less memory. The current version of MMC support multi-threaded programming and can give a almost proportional speed-up when using multiple CPU cores. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=394&screenshot_id=497"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=394&screenshot_id=497" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 24, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://mcx.sf.net/mmc/">http://mcx.sf.net/mmc/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>POSIX/UNIX-like</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="meshgen---surface-meshes-from-volumetric-segmentation-data" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MeshGen - Surface meshes from volumetric segmentation data">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=meshgen---surface-meshes-from-volumetric-segmentation-data" rel="bookmark" name="meshgen---surface-meshes-from-volumetric-segmentation-data">MeshGen - Surface meshes from volumetric segmentation data</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MeshGen generates separate mesh files for each label present in the segmentation volume. The generation employs a variant of Constrained Elastic Surface Nets [CESN], which has been modified in order to avoid intersection between neighboring meshes.<br />
Supported input formats:<br />
- NIfTI for segmentation volume, with integer elements (8/16/32 bits, signed/unsigned)<br />
- ITK label file (ITK-Snap compatible, does not contain hierarchy)<br />
- ILF - Integrated Label File (MBAT-compatible, hierarchical)<br />
- Brain-Map XML file (ontology from brain-map.org, hierarchical)<br />
When label hierarchy is available, MeshGen can follow containment relations e.g. eliminates generation of extra inner contours around contained labels. The tool can optionally eliminate contained labels if their colour matches with the colour of their parent, e.g. when rendering or not rendering the meshes using their original colours would make no visual difference. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 13, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution-NonCommercial-NoDerivatives 4.0 International (CC BY-NC-ND 4.0)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/meshgen/">http://www.nitrc.org/projects/meshgen/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Oslo <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="meshvalmet:-validation-metric-for-meshes" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MeshValmet: Validation Metric for Meshes">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=meshvalmet:-validation-metric-for-meshes" rel="bookmark" name="meshvalmet:-validation-metric-for-meshes">MeshValmet: Validation Metric for Meshes</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MeshValmet is a tool that measures surface to surface distance between two triangle meshes using user-specified uniform sampling. Thus, users can choose finer sampling level to calculate errors to gain more accuracy in the &quot;error space&quot;, or sparser sampling to gain speed and get an approximate feeling of error distribution between boundaries.<br />
<br />
Besides its pleasant visualization using the VTK library, MeshValmet also provides useful histogram and statistical information based on the sample errors, such as mean and median distance, root mean square distance, mean square distance, mean absolute distance, Hausdorff distance, 95 percentile, 68 percentile, etc.<br />
<br />
MeshValmet is based on the work of Nicolas Aspert, etc.: MESH: Measuring Errors between Surfaces using the Hausdorff distance in the proceedings of the IEEE Int. Conf. on Multimedia and Expo 2002 (ICME), vol. I, pp. 705-708.<br />
<br />
The calculation of the Dice's Coefficient is calculated by Joshua Stough using the concept of a Riemannian sum. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 11, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/meshvalmet/">http://www.nitrc.org/projects/meshvalmet/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="meshview-for-brain-atlases" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MeshView for brain atlases">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=meshview-for-brain-atlases" rel="bookmark" name="meshview-for-brain-atlases">MeshView for brain atlases</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MeshView (see MediaWiki page) is a web application for real-time 3D display of surface mesh data representing structural parcellations from volumetric atlases, such as the Waxholm Space Atlas of the Sprague Dawley Rat Brain (<a href="https://www.nitrc.org/projects/whs-sd-atlas" target="_new">https://www.nitrc.org/projects/whs-sd-at...</a>).<br />
<br />
Key features:<br />
- orbiting view with toggleable opaque/transparent/hidden parcellation meshes<br />
- rendering of user defined cut surface as if the meshes were solid objects<br />
- rendering point-clouds (simple type-in, or loading from a JSON format)<br />
- coordinate system is compatible with QuickNII (<a href="https://www.nitrc.org/projects/quicknii" target="_new">https://www.nitrc.org/projects/quicknii</a>)<br />
<br />
Two atlases are provided with MeshView:<br />
- Waxholm Space Atlas of the Sprague Dawley rat brain, parcellation version 2 (generated from 39 μm resolution volume)<br />
- Allen Mouse Brain Atlas reference atlas, CCFv3 parcellation from 2015 (generated from 25 μm resolution volume)<br />
<br />
A converter will be provided for importing STL meshes, like the ones generated by MeshGen (<a href="https://www.nitrc.org/projects/meshgen/" target="_new">https://www.nitrc.org/projects/meshgen/</a>) </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 1</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://www.nitrc.org/plugins/mwiki/index.php/meshview:MainPage">https://www.nitrc.org/plugins/mwiki/index.php/meshview:MainPage</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Oslo <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="metasearch" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MetaSearch">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=metasearch" rel="bookmark" name="metasearch">MetaSearch</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=web-service" rel="tag">Web Service</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=websites" rel="tag">Websites</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MetaSearch (<a href="http://openneu.ro/metasearch/" target="_new">http://openneu.ro/metasearch/</a>) is a search tool intended to help you find MRI data shared publicly on the Web. It is provided by the Open Neuroimaging Laboratory project as a companion to the BrainBox, a brain curation and annotation tool. If you want to see your data on MetaSearch, please let us know at <a href="https://github.com/OpenNeuroLab/metasearch/issues/new." target="_new">https://github.com/OpenNeuroLab/metasear...</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1077&screenshot_id=1144"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1077&screenshot_id=1144" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 16, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://openneu.ro/metasearch">http://openneu.ro/metasearch</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="meth" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="METH">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=meth" rel="bookmark" name="meth">METH</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> METH is a collection of MATLAB functions to conduct data analysis of MEG and EEG data. A detailed documentation is contained in the download. <br />
Its main features are: <br />
•EEG and MEG forward calculation in realistic volume conductors based on expansions of the lead fields in spherical harmonics.<br />
•Inverse Calculations: Dipole fitting for ERFs/ERPs and cross-spectra, MUSIC, RAP-MUSIC, SC-MUSIC, minimum norm source reconstruction, eLoreta, and beamformers.<br />
•Power and coupling measures: cross-spectra coherency, cross-bispectra and bicoherence, 1:1 phase locking, power correlation including corrections for artifacts of volume conduction, lagged coherence, phase lag index, multivariate coupling measures robust to artifacts of volume conduction (MIM, MIC, GIM).<br />
•Decomposition: MOCA for two or more source distributions<br />
•Statistics: false discovery rate and permutation test<br />
•Visualization: topo-plots, head-in-head plots and time-frequency plots, MRIs and surface plots (e.g. cortex) including results. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1257&screenshot_id=1167"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1257&screenshot_id=1167" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 27, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://www.uke.de/english/departments-institutes/institutes/neurophysiology-and-pathophysiology/research/working-groups/">https://www.uke.de/english/departments-institutes/institutes/neurophysiology-and-pathophysiology/research/working-groups/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Dept. of Neurophysiology and Pathophysiology <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mga---multimodal-glioma-analysis" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MGA - Multimodal Glioma Analysis">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mga---multimodal-glioma-analysis" rel="bookmark" name="mga---multimodal-glioma-analysis">MGA - Multimodal Glioma Analysis</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MGA is an MRI preprocessing pipeline built with HOF (Heterogeneous Optimization Framework) methodology. MGA prepares neuro-oncology clinical imaging studies for scientific analysis such as multispectral ROI analysis, in both longitudinal and cross-sectional studies. MGA works on DICOM images from a single MRI study. Includes perfusion (DSC sequence based) analysis and DTI analysis, and spatially co-registers all study images to an atlas template and to a template image within the study. MGA is designed to reasonably minimize user configuration steps and to accept a broad range of MRI submodalities, including support for clinical image quality. The distribution is available as a stand-alone docker.io virtual appliance for 64-bit Linux environments (see detailed instructions in the installation package):<br />
<a href="https://bitbucket.org/mmilch01/mga_docker_install" target="_new">https://bitbucket.org/mmilch01/mga_docke...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 19, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://bitbucket.org/mmilch01/mga_docker_install">http://bitbucket.org/mmilch01/mga_docker_install</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Neuroinformatics Research Group <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mgdm:-multi-geometric-deformable-model" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MGDM: Multi Geometric Deformable Model">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mgdm:-multi-geometric-deformable-model" rel="bookmark" name="mgdm:-multi-geometric-deformable-model">MGDM: Multi Geometric Deformable Model</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MGDM is an efficient level set framework for multi-object segmentation. Its representation inherently prevents overlaps and gaps and it readily preserves object topology and object relationships. MGDM is efficient, storing only a fixed number of functions for any number of objects, and therefore scales well to segmentation problems with many classes and large images. It's representation also avoids some instabilities in other multi-class level set methods. MGDM is cross-platform; MATLAB wrappers, Java source and API are provided, with MIPAV plugins forthcoming. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 20, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mgdm/">http://www.nitrc.org/projects/mgdm/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Johns Hopkins University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mgh-neonatal/pediatric-adc-atlases" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MGH Neonatal/Pediatric ADC atlases">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mgh-neonatal/pediatric-adc-atlases" rel="bookmark" name="mgh-neonatal/pediatric-adc-atlases">MGH Neonatal/Pediatric ADC atlases</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-data" rel="tag">Atlas Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Existing neonatal/pediatric atlases were mainly on structural MRI. We release here a set of Apparent Diffusion Coefficient (ADC) atlases, on 10 age groups: first 2 weeks of life, the rest of the 1st quarter, the 2nd, 3rd, and 4th quarters of life, year 1-2, 2-3, 3-4, 4-5 and 5-6). Atlases represent the mean geometry/anatomy of the age group, and the mean and standard deviation ADC values at the whole brain, regional (61 regions) and voxel levels. <br />
<br />
The age-specific ADC atlases characterize normative ADC development in 0-6 years of life (in neuroscience) and can provide reference to identify abnormalities (in pediatric neuroradiology).<br />
<br />
Refs:<br />
Y Ou, et al, &quot;Using Clinically-Acquired MRI to Construct Age-Specific ADC Atlases: Quantifying Spatiotemporal ADC Changes from Birth to 6 Years Old&quot;, 38(6): 3052-3068, Human Brain Mapping, (2017).<br />
<br />
S Sotardi, et al, &quot;Using Clinically-Acquired MRI to Construct Age-Specific ADC Atlases: Quantifying Spatiotemporal ADC Changes from Birth to 6 Years Old&quot;, under review. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=929&screenshot_id=925"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=929&screenshot_id=925" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 21, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Data License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mgh_adcatlases/">http://www.nitrc.org/projects/mgh_adcatlases/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Massachusetts General Hospital <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mias-registration-toolkit" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MIAS Registration Toolkit">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mias-registration-toolkit" rel="bookmark" name="mias-registration-toolkit">MIAS Registration Toolkit</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MIAS toolkit provides the following libraries and functions on linux platform:<br />
<br />
1. Multi-resolution registration of MR images include T1, multimodality, and DTI images.<br />
<br />
2. The registration model is B-spline, and users can custermize their own image similarity measures by writing a plugin function and recompile the program. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 2, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MIAS license</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mias/">http://www.nitrc.org/projects/mias/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Houston Methodist Research Institute <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="microdraw" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MicroDraw">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=microdraw" rel="bookmark" name="microdraw">MicroDraw</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MicroDraw (<a href="http://microdraw.pasteur.fr/" target="_new">http://microdraw.pasteur.fr/</a>) is a web application by NAAT to visualise and annotate collaboratively high resolution histology data. Annotations are vectorial, and you can use boolean operations to combine, subtract and split regions. Simply point MicroDraw to your own DeepZoom data! </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 16, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Affero General Public License v3</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://microdraw.pasteur.fr/">http://microdraw.pasteur.fr/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Instut Pasteur <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="micro-manager" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Micro-Manager">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=micro-manager" rel="bookmark" name="micro-manager">Micro-Manager</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Micro-Manager (<a href="http://micro-manager.org" target="_new">http://micro-manager.org</a>) operates motorized light microscopes and associated equipment. It runs as a plugin in ImageJ, works with virtually all scientific grade microscope equipment, and has a simple interface towards routine image acquisition strategies such as time-lapse, z- stacks, multi-channel, and multi-position acquisition. In addition, it uses a device abstraction layer available from various programming environments (such a C, Java, Python, Matlab and LabView), facilitating development of novel approaches to image acquisition. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 5, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Mixed: BSD and LGPL</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/micromanager/">http://www.nitrc.org/projects/micromanager/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UCSF <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="microstructural-correlation-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Microstructural correlation toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=microstructural-correlation-toolbox" rel="bookmark" name="microstructural-correlation-toolbox">Microstructural correlation toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Microstructural correlation toolbox (MSC) is a Matlab-based software library to perform independent component analysis on group white matter skeleton generated by FSL TBSS.<br />
The script produces stable estimates of the white matter tract or tract segments that resemble highly correlated variation profiles across a group of subjects. Please refer to the following publication for detailed information:<br />
Li YO, et al., &quot;Independent component analysis of DTI reveals multivariate microstructural correlations of white matter in the human brain&quot;, Hum Brain Mapp.2011. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 7, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>UCSF NCL Software License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/msc_toolbox/">http://www.nitrc.org/projects/msc_toolbox/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of California San Francisco <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mid-space-independent-deformable-image-registration" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Mid-Space-Independent Deformable Image Registration">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mid-space-independent-deformable-image-registration" rel="bookmark" name="mid-space-independent-deformable-image-registration">Mid-Space-Independent Deformable Image Registration</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=intersubject" rel="tag">Intersubject</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=intramodal" rel="tag">Intramodal</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=intrasubject" rel="tag">Intrasubject</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=nonlinear-warp" rel="tag">Nonlinear Warp</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This open-source toolbox contains Matlab codes and binaries for 2D and 3D deformable image registration, using the mid-space-independent approach introduced by Aganj et al (NeuroImage 2017). A short tutorial is included in EXAMPLE.m. If available, a GPU can be used for speed-up. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 14, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/msi-register">http://www.nitrc.org/projects/msi-register</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Harvard Medical School / Massachusetts General Hospital <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="minc-example-files" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MINC Example files">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=minc-example-files" rel="bookmark" name="minc-example-files">MINC Example files</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This is a reference MINC set of files. It currently includes human head images only of standard modalities.<br />
<br />
The goal is to build a well curated collection of files that demonstrate the capabilities of MINC </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 5, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MINC Licence (BSD'ish)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/minc_ex/">http://www.nitrc.org/projects/minc_ex/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Queensland <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>MINC, MINC2</td></tr>
			<tr class="even"><td>Output Formats</td><td>MINC, MINC2</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="minc-toolkit" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="minc-toolkit">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=minc-toolkit" rel="bookmark" name="minc-toolkit">minc-toolkit</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Whole set of MINC-based image processing tools packaged together.<br />
Includes: MINC,N3,BICPL,EBKTS,ANIMAL,INSECT,BEaST,Register,Display,xdisp 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=777&screenshot_id=702"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=777&screenshot_id=702" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 6, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.bic.mni.mcgill.ca/ServicesSoftware/ServicesSoftwareMincToolKit">http://www.bic.mni.mcgill.ca/ServicesSoftware/ServicesSoftwareMincToolKit</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Montreal Neurological Institute <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>MINC, MINC2</td></tr>
			<tr class="even"><td>Output Formats</td><td>MINC, MINC2</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="minc-toolkit-testsuite" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="minc-toolkit-testsuite">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=minc-toolkit-testsuite" rel="bookmark" name="minc-toolkit-testsuite">minc-toolkit-testsuite</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Sample data in MINC format and collection of scripts to verify correct installation of minc-toolkit. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 13, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/minctoolkittest/">http://www.nitrc.org/projects/minctoolkittest/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Montreal Neurological Institute <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="minc-toolkit-v2" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="minc-toolkit-v2">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=minc-toolkit-v2" rel="bookmark" name="minc-toolkit-v2">minc-toolkit-v2</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Whole set of MINC-based image processing tools packaged together and several ITKv4 tools Includes: MINC,N3,BICPL,EBKTS,ANIMAL,INSECT,BEaST,Register,Display,xdisp,ANTS, Elastix, ABC, C3D </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 15, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Artistic License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/minc-toolkit-v2/">http://www.nitrc.org/projects/minc-toolkit-v2/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Montreal Neurological Institute <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mindboggle-101-manually-labeled-brains" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Mindboggle-101 manually labeled brains">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mindboggle-101-manually-labeled-brains" rel="bookmark" name="mindboggle-101-manually-labeled-brains">Mindboggle-101 manually labeled brains</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data-resource" rel="tag">Data Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Mindboggle-101 is the largest and most complete set of free, publicly accessible, manually labeled human brain images. See <a href="http://mindboggle.info/data.html" target="_new">http://mindboggle.info/data.html</a> for data and article: <a href="http://www.frontiersin.org/brain_imaging_methods/10.3389/fnins.2012.00171/full" target="_new">http://www.frontiersin.org/brain_imaging...</a><br />
<br />
All data are accessible on the Open Science Framework: <a href="https://osf.io/nhtur/" target="_new">https://osf.io/nhtur/</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=667&screenshot_id=590"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=667&screenshot_id=590" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 4, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial Share Alike</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.mindboggle.info/data.html">http://www.mindboggle.info/data.html</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>MGH/MGZ, NIfTI-1, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>MGH/MGZ, NIfTI-1, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mindboggle:-open-source-software-for-analyzing-the-shapes-of-brain-structures-from-human-mri-data" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Mindboggle: open source software for analyzing the shapes of brain structures from human MRI data">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mindboggle:-open-source-software-for-analyzing-the-shapes-of-brain-structures-from-human-mri-data" rel="bookmark" name="mindboggle:-open-source-software-for-analyzing-the-shapes-of-brain-structures-from-human-mri-data">Mindboggle: open source software for analyzing the shapes of brain structures from human MRI data</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=shape-analysis" rel="tag">Shape Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-analysis" rel="tag">Surface Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volumetric-analysis" rel="tag">Volumetric Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Mindboggle (<a href="http://mindboggle.info" target="_new">http://mindboggle.info</a> and <a href="https://osf.io/ydyxu/" target="_new">https://osf.io/ydyxu/</a>) is open source software for analyzing the shapes of brain structures from human MRI data. The primary reference documenting and evaluating Mindboggle was published in PLoS Computational Biology:<br />
<br />
Klein A, Ghosh SS, Bao FS, Giard J, Hame Y, Stavsky E, Lee N, Rossa B, Reuter M, Neto EC, Keshavan A. (2017) Mindboggling morphometry of human brains. PLoS Computational Biology 13(3): e1005350. doi:10.1371/journal.pcbi.1005350 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=33&screenshot_id=986"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=33&screenshot_id=986" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 25, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.mindboggle.info">http://www.mindboggle.info</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>MGH/MGZ, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>MGH/MGZ, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mindseer" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MindSeer">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mindseer" rel="bookmark" name="mindseer">MindSeer</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MindSeer is a cross-platform application for 3D brain visualization. It is written in Java/Java3D, and runs in both standalone and client-server mode. Click on the MindSeer Project Page for downloads, tutorials, source code, demos and a publication. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 6, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mindseer/">http://www.nitrc.org/projects/mindseer/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mipav" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MIPAV">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mipav" rel="bookmark" name="mipav">MIPAV</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=animation" rel="tag">Animation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=colocalization" rel="tag">Colocalization</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=diffusion-mr-fiber-tracking" rel="tag">Diffusion Mr Fiber Tracking</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=frequency-domain" rel="tag">Frequency Domain</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-display" rel="tag">Image Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=mesh-generation" rel="tag">Mesh Generation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=registration" rel="tag">Registration</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=resampling" rel="tag">Resampling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=simulation" rel="tag">Simulation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=spatial-convolution---deconvolution" rel="tag">Spatial Convolution - Deconvolution</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-rendering" rel="tag">Surface Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volume-rendering" rel="tag">Volume Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volumetric-analysis" rel="tag">Volumetric Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The MIPAV (Medical Image Processing, Analysis, and Visualization) application enables quantitative analysis and visualization of medical images of numerous modalities such as PET, MRI, CT, or microscopy. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=70&screenshot_id=95"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=70&screenshot_id=95" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 24, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Other/Proprietary License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://mipav.cit.nih.gov/">http://mipav.cit.nih.gov/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Other OS<br>Other UNIX-like<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>AFNI BRIK, ANALYZE, COR, DICOM, MGH/MGZ, MINC, MINC2, NIfTI-1, Nrrd, Other Format, bshort/bfloat</td></tr>
			<tr class="even"><td>Output Formats</td><td>AFNI BRIK, ANALYZE, COR, DICOM, MGH/MGZ, MINC, MINC2, NIfTI-1, Nrrd, Other Format, bshort/bfloat</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mircen-macaca-fascicularis-brain-mri-segmentation-dataset" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MIRCen Macaca fascicularis brain MRI segmentation dataset">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mircen-macaca-fascicularis-brain-mri-segmentation-dataset" rel="bookmark" name="mircen-macaca-fascicularis-brain-mri-segmentation-dataset">MIRCen Macaca fascicularis brain MRI segmentation dataset</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This dataset comprises 10 T2-weighted MRIs acquired in healthy Cynomolgus macaques (Macaca fascicularis), aged 3 to 5 years, on a 7T scanner, along with manual segmentations into 17 anatomical regions of 15 sections per scan, along different incidences.<br />
<br />
A set of python functions is also provided in order to compute F1/Dice scores between complete 3D segmentations and these 2D references.<br />
<br />
It was used to validate the Primatologist segmentation pipeline and is intended to be used by any team or individual wishing to validate its algorithm or compare its results with ours. It is freely available for academic work upon citing the following Data in Brief article.<br />
<br />
A thorough description of the material &amp; methods can be found in:<br />
- Balbastre et al., 2017. &quot;A validation dataset for Macaque brain MRI segmentation.&quot;<br />
<br />
Our results on this dataset can be found in:<br />
- Balbastre et al., 2017. &quot;Primatologist: a modular segmentation pipeline for macaque brain morphometry.&quot; 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1199&screenshot_id=1120"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1199&screenshot_id=1120" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 3, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>CeCill v2</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mircen_macset/">http://www.nitrc.org/projects/mircen_macset/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>CEA <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="miriad-(minimal-interval-resonance-imaging-in-alzheimer's-disease)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MIRIAD (Minimal Interval Resonance Imaging in Alzheimer's Disease)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=miriad-(minimal-interval-resonance-imaging-in-alzheimer's-disease)" rel="bookmark" name="miriad-(minimal-interval-resonance-imaging-in-alzheimer's-disease)">MIRIAD (Minimal Interval Resonance Imaging in Alzheimer's Disease)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The MIRIAD dataset is a database of volumetric MRI brain-scans of 46 Alzheimer's sufferers and 23 healthy elderly people. Many scans were collected of each participant at intervals from 2 weeks to 2 years, the study was designed to investigate the feasibility of using MRI as an outcome measure for clinical trials of Alzheimer's treatments. It includes a total of 708 scans and should be of particular interest for work on longitudinal biomarkers and image analysis. The dataset can be found at <a href="http://miriad.drc.ion.ucl.ac.uk/" target="_new">http://miriad.drc.ion.ucl.ac.uk/</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 8, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MIRIAD Data Use Agreement</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://miriad.drc.ion.ucl.ac.uk/">http://miriad.drc.ion.ucl.ac.uk/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UCL Institute of Neurology, UCL, UK <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="misst---microstructure-imaging-sequence-simulation-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MISST - Microstructure Imaging Sequence Simulation ToolBox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=misst---microstructure-imaging-sequence-simulation-toolbox" rel="bookmark" name="misst---microstructure-imaging-sequence-simulation-toolbox">MISST - Microstructure Imaging Sequence Simulation ToolBox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=simulation" rel="tag">Simulation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Microstructure Imaging Sequence Simulation Toolbox (MISST), is a practical diffusion MRI simulator for development, testing, and optimisation of novel MR pulse sequences for microstructure imaging. MISST is based on a matrix method approach and simulates the signal for a large variety of pulse sequences and tissue models. Its key purpose is to provide a deep understanding of the restricted diffusion MRI signal for a wide range of realistic, fully flexible, scanner acquisition protocols, in practical computational time. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=814&screenshot_id=752"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=814&screenshot_id=752" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 8, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Artistic License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/misst/">http://www.nitrc.org/projects/misst/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University College London <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="misteri" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MisterI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=misteri" rel="bookmark" name="misteri">MisterI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MisterI (Mr. I) is a powerful and modular medical image viewer/editor. It should be particularly useful to Undergraduates, Postdocs and Researchers in Medical Imaging to visualize data and to easily make attractive figures (for papers or presentations). It will also in a near future offer a number of advanced algorithms for medical image processing. Look at the video to get an idea ! <br />
<br />
<a href="http://www.benoitscherrer.com/MisterI/videos.html" target="_new">http://www.benoitscherrer.com/MisterI/vi...</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=714&screenshot_id=703"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=714&screenshot_id=703" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 2, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Cardware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/misteri/">http://www.nitrc.org/projects/misteri/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>MGH/MGZ, NIfTI-1, Nrrd</td></tr>
			<tr class="even"><td>Output Formats</td><td>MGH/MGZ, NIfTI-1, Nrrd</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mitk-diffusion" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MITK Diffusion">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mitk-diffusion" rel="bookmark" name="mitk-diffusion">MITK Diffusion</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=connectivity-analysis" rel="tag">Connectivity Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=fiber-tracking" rel="tag">Fiber Tracking</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-reconstruction" rel="tag">Image Reconstruction</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=modeling" rel="tag">Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quantification" rel="tag">Quantification</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The MITK Diffusion application offers a selection of image analysis algorithms for the processing of diffusion-weighted MR images. It encompasses the research of the Division [https://www.dkfz.de/en/mic/index.php Medical Image Computing] at the German Cancer Research Center (dkfz).<br />
<br />
Features &amp; Highlights <br />
- Support for most established image formats, such as DICOM, NIFTI, and NRRD<br />
- Tensor and Q-ball reconstruction<br />
- Intravoxel Incoherent Motion (IVIM) analysis<br />
- Fiber tractography and fiber processing<br />
-- Global tractography<br />
-- Interactive peak, ODF and tensor streamline tractography<br />
-- Machine learning based tractography<br />
- Brain network statistics and visualization (connectomics)<br />
- Fiberfox: simulation of diffusion-weighted MR images<br />
- Command line tools for most functionalities 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=615&screenshot_id=538"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=615&screenshot_id=538" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 3, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD/MIT-Style Open Source License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://mitk.org/wiki/DiffusionImaging">http://mitk.org/wiki/DiffusionImaging</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Division of Medical Image Computing <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>OS Independent<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>DICOM, NIfTI-1, Nrrd, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>DICOM, NIfTI-1, Nrrd, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="miview" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MIView">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=miview" rel="bookmark" name="miview">MIView</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MIview is an OpenGL based medical image viewer that contains useful tools such as a DICOM anonymizer and format conversion utility. MIView can read DICOM, Analyze/Nifti, and raster images, and can write Analyze/Nifti and raster images. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 25, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/miview/">http://www.nitrc.org/projects/miview/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mixed-effect-model-of-genetic-set-and-environment-interaction" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Mixed Effect Model of Genetic-Set and Environment Interaction">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mixed-effect-model-of-genetic-set-and-environment-interaction" rel="bookmark" name="mixed-effect-model-of-genetic-set-and-environment-interaction">Mixed Effect Model of Genetic-Set and Environment Interaction</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Toolbox: Mixed Effect Model of Genetic-Set and Environment Interaction for Neuroimage applications<br />
<br />
This MATLAB Toolbox provides a mixed effect model for gene-environment interaction (MixGE) on neuroimaging phenotypes, such as structural volumes and tensor-based morphometry (TBM). This model incorporates both fixed and random effects of genetic-set and environment interaction, to investigate homogeneous and heterogeneous contributions of sets of genetic variants and their interaction with environmental risks to phenotypes. To avoid direct parameter estimation in the MixGE model due to small sample size and high computational cost, score statistics were constructed for the terms associated with fixed and random effects of the genetic-set and environment interaction. They were combined into a single significance value. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 26, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://http://www.bioeng.nus.edu.sg/cfa/imaginggenetics.html">http://http://www.bioeng.nus.edu.sg/cfa/imaginggenetics.html</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>National University of Singapore <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mmps-docker" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MMPS Docker">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mmps-docker" rel="bookmark" name="mmps-docker">MMPS Docker</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Docker image of the MMPS pipeline. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 28, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mmps_docker/">http://www.nitrc.org/projects/mmps_docker/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UCSD <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mne---minimum-norm-current-estimates" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MNE - Minimum Norm Current Estimates">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mne---minimum-norm-current-estimates" rel="bookmark" name="mne---minimum-norm-current-estimates">MNE - Minimum Norm Current Estimates</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=forward---inverse" rel="tag">Forward - Inverse</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=temporal-transformation" rel="tag">Temporal Transformation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=time-domain-analysis" rel="tag">Time Domain Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The MNE software has been developed to compute cortically-constrained L2 minimum-norm current estimates and associated dynamic statistical parametric maps from MEG and EEG data, optionally constrained by fMRI. This software includes MEG and EEG preprocessing tools, interactive and batch-mode modules for the forward and inverse calculations, as well as various data conditioning and data conversion utilities.<br />
<br />
This is a subproject of the Center for Functional Neuroimaging Techniques (<a href="http://www.nitrc.org/projects/cfnt/" target="_new">http://www.nitrc.org/projects/cfnt/</a>) 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=168&screenshot_id=614"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=168&screenshot_id=614" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 20, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MGH CSRL License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.martinos.org/mne/">http://www.martinos.org/mne/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>MGH/MGZ, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>MGH/MGZ, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mni-macaque-atlases" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MNI Macaque Atlases">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mni-macaque-atlases" rel="bookmark" name="mni-macaque-atlases">MNI Macaque Atlases</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> We present unbiased standard macaque monkey magnetic resonance imaging template brain volumes that offers a common stereotaxic reference frame to localize anatomical and functional information in an organized and reliable way for comparison across individual nonhuman primates and studies. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=500&screenshot_id=377"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=500&screenshot_id=377" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 30, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.bic.mni.mcgill.ca/ServicesAtlases/HomePage">http://www.bic.mni.mcgill.ca/ServicesAtlases/HomePage</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Montreal Neurological Inst. <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>MINC, MINC2, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>MINC, MINC2, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="moca" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MOCA">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=moca" rel="bookmark" name="moca">MOCA</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=connectivity-analysis" rel="tag">Connectivity Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=mesh-generation" rel="tag">Mesh Generation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=shape-analysis" rel="tag">Shape Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-analysis" rel="tag">Surface Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MOCA stands for &quot;Metric Optimization for Computational Anatomy&quot;. It is a collection of software tools for the computational analysis of brain anatomy with MRI data. It includes automated software tools from surface reconstruction to their mapping via metric optimization in the Laplace-Beltrami embedding space. It is general and can be applied to a wide range of anatomical structures including cortical, sub-cortical, and fiber bundle surfaces. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 21, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MOCA License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/moca_2015/">http://www.nitrc.org/projects/moca_2015/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Southern California <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="modelgui" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ModelGUI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=modelgui" rel="bookmark" name="modelgui">ModelGUI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=clipping" rel="tag">Clipping</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database-application" rel="tag">Database Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=platform-or-development-environment" rel="tag">Platform Or Development Environment</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-rendering" rel="tag">Surface Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=three-dimensional-display" rel="tag">Three Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=two-dimensional-display" rel="tag">Two Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volume-rendering" rel="tag">Volume Rendering</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ModelGUI (or simply mgui) is an open source Java-based project intended to provide a graphic user interface (GUI) for interactions between scientists (or enthusiasts) and their data. In its current (beta) form, mgui offers the following functionality:<br />
<br />
Established:<br />
<br />
Cross-platform functionality (with a Java Runtime installation, runs on Linux, Windows, Mac, or Solaris)<br />
2D rendering of data based upon Java2D, and 3D rendering based upon Java3D<br />
An extensible I/O framework accommodating a variety of standard and non-standard file formats<br />
Database connectivity using JDBC<br />
Graph visualization based upon the JUNG library<br />
An intuitive Swing-based GUI for managing, querying, and visualizing data<br />
<br />
Experimental:<br />
<br />
Various CAD-type tools for editing and creating geometry<br />
The ability to organize complex datasets into intuitive mgui projects<br />
A processing pipeline interface which allows users to process their datasets with any available Java or native software tools<br />
A computational modelling framework 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=726&screenshot_id=692"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=726&screenshot_id=692" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 6, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://mgui.wikidot.com">http://mgui.wikidot.com</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>GIfTI, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>GIfTI, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="monstr-multi-contrast-brain-stripping" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MONSTR Multi Contrast Brain Stripping">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=monstr-multi-contrast-brain-stripping" rel="bookmark" name="monstr-multi-contrast-brain-stripping">MONSTR Multi Contrast Brain Stripping</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MONSTR (Multi-cONtrast brain STRipping) is a software tool to generate brainmasks (or skull-strip) from multi-contrast MR brain images, such as T1, T2, PD, or FLAIR. Compared to other T1-based skullstripping methods, MONSTR can take multi-modal inputs, and based on multi-modal atlases, it generates a brainmask of a given subject image. <br />
<br />
It is a patch-based method. Multiple atlas images are first registered to a subject. Then based on the similarities between multiple atlas patches and a given subject patch, the corresponding label (i.e. brain or non-brain) of the subject patch is estimated.<br />
<br />
In addition to the software, we also provide 3 sets of atlas images, having 17 subjects in total. Each atlas contains T1, T2, (FLAIR for one set), and their manually delineated brainmasks. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1063&screenshot_id=1007"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1063&screenshot_id=1007" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 24, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/monstr/">http://www.nitrc.org/projects/monstr/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Henry Jackson Foundation <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="monte-carlo-extreme-(mcx)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Monte Carlo eXtreme (MCX)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=monte-carlo-extreme-(mcx)" rel="bookmark" name="monte-carlo-extreme-(mcx)">Monte Carlo eXtreme (MCX)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=monte-carlo" rel="tag">Monte Carlo</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Monte Carlo eXtreme, or MCX, is a Monte Carlo simulation software for photon migration in 3D turbid media. It uses Graphics Processing Units (GPU) based massively parallel computing techniques and is extremely fast compared to the traditional single-threaded CPU-based simulations. Using an nVidia 8800GT graphics card (14MP/114Cores), the acceleration is about 300x~400x compared to a single core of Xeon 5120 CPU; this ratio can be as high as 700x with a GTX 280 GPU and 1400x with a GTX 470. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 24, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://mcx.sourceforge.net/">http://mcx.sourceforge.net/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS<br>POSIX/UNIX-like<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="monte-carlo-simulation-software:-tmcimg" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Monte Carlo Simulation Software: tMCimg">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=monte-carlo-simulation-software:-tmcimg" rel="bookmark" name="monte-carlo-simulation-software:-tmcimg">Monte Carlo Simulation Software: tMCimg</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=monte-carlo" rel="tag">Monte Carlo</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> tMCimg uses a Monte Carlo algorithm to model the transport of photons through 3D volumes with spatially varying optical properties. Both highly-scattering tissues (e.g. white matter) and weakly scattering tissues (e.g. cerebral spinal fluid) are supported. Using the anatomical information provided by MRI, X-ray CT, or ultrasound, accurate solutions to the photon migration forward problems are computed in times ranging from minutes to hours, depending on the optical properties and the computing resources available. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 24, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/tmcimg/">http://www.nitrc.org/projects/tmcimg/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS<br>POSIX/UNIX-like<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="monthly-morphometry-report" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Monthly Morphometry Report">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=monthly-morphometry-report" rel="bookmark" name="monthly-morphometry-report">Monthly Morphometry Report</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=knowledge-environment" rel="tag">Knowledge Environment</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=web-resource" rel="tag">Web Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Monthly Morphometry Report (MMR) is designed to be a quick, high-level synopsis of the volumetric literature, compiled on a monthly basis. We seek to represent the corpus of published findings in a suscinct and common format, to ease human interpretation of ths vast set of measures that are being reported.<br />
<br />
To start, monthly PUBMED reports of potential volumetric publications in humans are identified, and human curators then summarise these abstracts into a standard report format. In the future, we hope to add automation to the curation process and index more complete full-text content.<br />
<br />
This project is HIGHLY alpha in its design and should be considered only a 'work in progress'. Enter at your own risk, or, volunteer to help! </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 11, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mmr/">http://www.nitrc.org/projects/mmr/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Massachusetts Medical School <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="morphometry-birn" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Morphometry BIRN">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=morphometry-birn" rel="bookmark" name="morphometry-birn">Morphometry BIRN</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=knowledge-environment" rel="tag">Knowledge Environment</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quantification" rel="tag">Quantification</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Morphometry Testbed (MBIRN) of the Biomedical Informatics Research Network (BIRN) focuses on pooling and analyzing of neuroimaging data acquired at multiple sites. Specific applications include potential relationships between anatomical differences and specific memory dysfunctions, such as Alzheimer’s disease. With the completion of the initial BIRN testbed phase, each of the original BIRN testbeds have now been retired in order to focus on new users in other biomedical domains. The BIRN project has ended. Information on the entire BIRN project can be found at <a href="https://en.wikipedia.org/wiki/Biomedical_Informatics_Research_Network" target="_new">https://en.wikipedia.org/wiki/Biomedical...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 5, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://en.wikipedia.org/wiki/Biomedical_Informatics_Research_Network">https://en.wikipedia.org/wiki/Biomedical_Informatics_Research_Network</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>DICOM, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>DICOM, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="motrak:-head-motion-tracking-system" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MoTrak: Head Motion Tracking System">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=motrak:-head-motion-tracking-system" rel="bookmark" name="motrak:-head-motion-tracking-system">MoTrak: Head Motion Tracking System</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=application" rel="tag">Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=experiment-control" rel="tag">Experiment Control</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=training" rel="tag">Training</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Designed for use in an MRI simulator, MoTrak software uses Ascension Technology’s trakSTAR. The sensor attached to the subject’s head determines the position of the head in space relative to the transmitter. The sensor records angular rotations as well as positional displacements from an initially calibrated position. This information is displayed and logged by the program in real-time, allowing observation of head motion in an MRI simulator. The goal is to monitor how much the participant moves while the scan is in progress. If a participant moves too much, it can potentially result in unusable data and wasted resources.<br />
<br />
In the simulator, the participant can simultaneously be habituated to the MRI environment, while being trained to remain still via feedback from the MoTrak system. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=229&screenshot_id=204"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=229&screenshot_id=204" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 8, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://pstnet.com/products/motrak/">https://pstnet.com/products/motrak/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Windows Vista<br>Windows XP</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mouse-birn" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Mouse BIRN">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mouse-birn" rel="bookmark" name="mouse-birn">Mouse BIRN</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-data" rel="tag">Atlas Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Mouse BIRN has ended. The next phase of this project is the Mouse Connectome Project (<a href="https://www.nitrc.org/projects/mcp/" target="_new">https://www.nitrc.org/projects/mcp/</a>). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 3, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BIRN License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mcp/">http://www.nitrc.org/projects/mcp/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mousebirn-atlasing-toolkit-(mbat)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MouseBIRN Atlasing Toolkit (MBAT)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mousebirn-atlasing-toolkit-(mbat)" rel="bookmark" name="mousebirn-atlasing-toolkit-(mbat)">MouseBIRN Atlasing Toolkit (MBAT)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=information-resource" rel="tag">Information Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MBAT provides a workflow environment bringing together heterogenous, online biological image resources, a user’s image data and biological atlases in a concise, unified and intuitive workspace. The MBAT viewer displays multiple images on a single virtual canvas allowing easy side-by-side comparisons and image compositing. MBAT is written in Java so it is platform independent and is highly extensible through it’s plugin architecture.<br />
<br />
MBAT integrates three distinct workspaces for online search, image alignment (registration) and image display:<br />
<br />
Search Workspace: able to submit a query to multiple databases simultaneously and online literature searches.<br />
<br />
Registration Workspace: performs 2D landmark based registration.<br />
<br />
Viewer Workspace: displays &amp; composites images and image volumes using high performance graphics hardware.<br />
<br />
Atlas Viewer: allows navigation and interrogation of volumetric atlases.<br />
<br />
Hierarchy Editor: create logical groupings of atlas labels. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=86&screenshot_id=245"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=86&screenshot_id=245" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 22, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BIRN License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://mbat.loni.ucla.edu">http://mbat.loni.ucla.edu</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mouse-connectome-project-(mcp)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Mouse Connectome Project (MCP)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mouse-connectome-project-(mcp)" rel="bookmark" name="mouse-connectome-project-(mcp)">Mouse Connectome Project (MCP)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-data" rel="tag">Atlas Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Mouse Connectome Project (MCP,http://www.mouseconnectome.org) aims to create a three-dimensional digital connectome atlas of the C57Black/6J mouse brain. MCP offers a catalog of neural tracer injection cases, which is updated continuously to eventually cover the entire brain. Serial sections of each case are available to view at 10x magnification in the interactive iConnectome browser. Sample data set is available to explore in iConnectome. The data set features 5 layers which may be turned on, off, or adjusted for transparency, and split window features are available for comparing data sets.<br />
<br />
The MCP project is housed at the Laboratory of Neuro Imaging at USC. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=472&screenshot_id=413"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=472&screenshot_id=413" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 28, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>LONI Software License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mcp/">http://www.nitrc.org/projects/mcp/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Southern California <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mousetracker" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MouseTracker">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mousetracker" rel="bookmark" name="mousetracker">MouseTracker</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MouseTracker is a free, user-friendly software package that allows researchers to record and analyze real-time hand movements en route to responses on the screen (via the coordinates of an MR-safe mouse, trackball, or sensor). By looking at how the hand settles into a response alternative--and how it may be partially pulled toward other alternatives--researchers glean information about real-time mental processing. It's like opening up a single reaction time into a continuous stream of rich cognitive output. Experiments can incorporate images, letter strings, and sounds. Once recorded, participants' mouse trajectories can be visualized, averaged, and explored, and measures of curvature, complexity, velocity, acceleration, and angle can be computed. Precise characterizations of mouse trajectories' temporal and spatial dynamics are available, and these can shed light on a variety of important empirical questions across psychology, neuroscience, and beyond. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=430&screenshot_id=310"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=430&screenshot_id=310" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 7, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Non-Commercial Software License Agreement</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://mousetracker.jbfreeman.net">http://mousetracker.jbfreeman.net</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="movement_info" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="movement_info">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=movement_info" rel="bookmark" name="movement_info">movement_info</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quantification" rel="tag">Quantification</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Software movement_info_2 is written to visualize movement parameters estimated during preprocessing of fMRI data. Its development was based on movement parameters estimated with the tool SPM, but it should work in general for all 6 parametric movement estimates (translations and rotations in axis x, y and z). The visualization is implemented in user-friendly GUI.<br />
Main features of movement_info_2 are:<br />
1)Visualization of 6 movement parameters and their differences<br />
2)Framewise displacement (FD) –Computation and visualization<br />
3)Setting threshold for scans based on FD value, visualization <br />
4)Export of suprathreshold scans to Excel or mat file (button “Export suitable data”)<br />
5)Setting threshold of affected scans for exclusion of datasets<br />
6)Visualization of suitable and unsuitable datasets based on abovementioned thresholds. Export to Excel possible (button “Export suitable data”) </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 21, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU GPL v2</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/movement_info/">http://www.nitrc.org/projects/movement_info/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Masaryk University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mpi-leipzig-mind-brain-body-dataset" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MPI-Leipzig Mind-Brain-Body Dataset">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mpi-leipzig-mind-brain-body-dataset" rel="bookmark" name="mpi-leipzig-mind-brain-body-dataset">MPI-Leipzig Mind-Brain-Body Dataset</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The participants included in this dataset participated in one or two protocols. Each of these protocols included structural and resting-state fMRI data acquisition, as well as an extensive battery of behavioural tests. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 22, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mpilmbb/">http://www.nitrc.org/projects/mpilmbb/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Max Planck Institute for Human Cognitive and Brain Sciences <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mpscope" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MPScope">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mpscope" rel="bookmark" name="mpscope">MPScope</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=experiment-control" rel="tag">Experiment Control</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=scanner" rel="tag">Scanner</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MPScope is a comprehensive software suite for custom-built multiphoton microscopes available as freeware for the Wintel platform. The MPScope package features the acquisition software MPScan, the analysis program MPView and several software utilities </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 30, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mpscope/">http://www.nitrc.org/projects/mpscope/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Windows 95/98/2000<br>Windows NT/2000</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Other Format, bshort/bfloat</td></tr>
			<tr class="even"><td>Output Formats</td><td>Other Format, bshort/bfloat</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mrbrains-evaluation-framework-/-challenge" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MRBrainS Evaluation Framework / Challenge">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mrbrains-evaluation-framework-/-challenge" rel="bookmark" name="mrbrains-evaluation-framework-/-challenge">MRBrainS Evaluation Framework / Challenge</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The aim of the MRBrainS online evaluation framework is to compare automatic algorithms that segment gray matter (GM), white matter (WM), and cerebrospinal fluid (CSF) on 3T MRI scans. Participants apply their algorithms to the provided data, after which their results are evaluated and ranked. Full manual segmentations of GM, WM, and CSF are available for all 20 scans and used as the reference standard (trainingset: 5, testset: 15). The evaluated methods are ranked based on their overall performance to segment GM, WM, and CSF and evaluated using three evaluation metrics (Dice, 95th percentile Hausdorff distance, and absolute volume difference). The results are published on the MRBrainS13 website. If you would like to join the MRBrainS challenge, please register a team on the MRBrainS website: <a href="http://mrbrains13.isi.uu.nl/register.php" target="_new">http://mrbrains13.isi.uu.nl/register.php</a> and sign the confidentiality agreement. You will be granted access to the data and can submit your automatic segmentation results for evaluation. Visit <a href="http://mrbrains13.isi.uu.nl" target="_new">http://mrbrains13.isi.uu.nl</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 12, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MRBrainS license</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://mrbrains13.isi.uu.nl/">http://mrbrains13.isi.uu.nl/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University Medical Center Utrecht, the Netherlands <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mr-connectome-automated-pipeline-(mrcap)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MR Connectome Automated Pipeline (MRCAP)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mr-connectome-automated-pipeline-(mrcap)" rel="bookmark" name="mr-connectome-automated-pipeline-(mrcap)">MR Connectome Automated Pipeline (MRCAP)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This is the project page for the MR Connectome Automated Pipeline based on JIST and MIPAV. The pipeline combines structural magnetic resonance data with diffusion tensor imaging to estimate a connectome, which is a comprehensive description of the wiring diagram of the brain. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=482&screenshot_id=367"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=482&screenshot_id=367" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 24, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Lesser General Public License (LGPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mrcap/">http://www.nitrc.org/projects/mrcap/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Johns Hopkins University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mri-brain-template-and-atlas-of-the-mouse-lemur-primate-microcebus-murinus" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MRI Brain Template and Atlas of the Mouse Lemur Primate Microcebus murinus">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mri-brain-template-and-atlas-of-the-mouse-lemur-primate-microcebus-murinus" rel="bookmark" name="mri-brain-template-and-atlas-of-the-mouse-lemur-primate-microcebus-murinus">MRI Brain Template and Atlas of the Mouse Lemur Primate Microcebus murinus</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MRI template and 120-region atlas for the mouse lemur primate Microcebus murinus.<br />
<br />
Generated from 34 animals aged 15-58 months old scanned at 7T using a T2-weighted sequence, resolution 115 × 115 × 230 µm. The code developed to create and manipulate the template has been refined into general procedures for registering small mammal brain MR images, available within a python module sammba-mri (SmAll-maMMals BrAin MRI; <a href="https://sammba-mri.github.io/" target="_new">https://sammba-mri.github.io/</a>). The template was up-sampled to 91 µm isotropic for hand-segmentation of structures, and also used to create probability maps of grey matter, white matter and cerebro-spinal fluid.<br />
<br />
Note that this atlas is from July 2018, and will be regularly improved in future versions. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 23, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>CeCill v2</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mouselemuratlas/">http://www.nitrc.org/projects/mouselemuratlas/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>CEA <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mricloud" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MriCloud">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mricloud" rel="bookmark" name="mricloud">MriCloud</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tensor-metric" rel="tag">Tensor Metric</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MriCloud is a cloud-based platform that provides several advanced features including;<br />
<br />
1) Fully automated cloud service for brain parcellation of MPRAGE images based on Multiple-Atlas Likelihood Fusion (MALF) algorithm (Tang, et al. PLoS ONE, 2013), JHU multi-atlas inventories with 286 defined structures, and our Ontology Level Control (OLC) technology (Djamanakova, et al. NeuroImage, 2014).<br />
<br />
2) Fully automated cloud service for DTI tensor calculation and extensive quantitative reports for quality control (Li, et al. PLoS ONE, 2013). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 12, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MriCloud Agreement</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://mricloud.org/">http://mricloud.org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Johns Hopkins University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mricrogl" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MRIcroGL">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mricrogl" rel="bookmark" name="mricrogl">MRIcroGL</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=three-dimensional-display" rel="tag">Three Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=two-dimensional-display" rel="tag">Two Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volume-rendering" rel="tag">Volume Rendering</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MRIcroGL allows you to view 2D slices and renderings of your brain imaging data. It can display many image formats and includes a graphical interface for dcm2nii to convert DICOM images to NIfTI format. It allows you to draw regions of interest which can aid lesion mapping and fMRI analysis. It provides sophisticated rendering. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=889&screenshot_id=852"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=889&screenshot_id=852" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 7, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.mccauslandcenter.sc.edu/mricrogl/">http://www.mccauslandcenter.sc.edu/mricrogl/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of South Carolina <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>AFNI BRIK, ANALYZE, DICOM, MGH/MGZ, NIfTI-1, Nrrd, Philips PAR/REC</td></tr>
			<tr class="even"><td>Output Formats</td><td>AFNI BRIK, ANALYZE, DICOM, MGH/MGZ, NIfTI-1, Nrrd, Philips PAR/REC</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mricron" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MRIcron">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mricron" rel="bookmark" name="mricron">MRIcron</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=center-of-mass" rel="tag">Center Of Mass</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=clipping" rel="tag">Clipping</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=regression" rel="tag">Regression</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=two-dimensional-display" rel="tag">Two Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volume-rendering" rel="tag">Volume Rendering</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MRIcron is a cross-platform NIfTI format image viewer. It can load multiple layers of images, generate volume renderings and draw volumes of interest. It also provides dcm2nii for converting DICOM images to NIfTI format and NPM for statistics. MRIcron is a mature and useful tool, however you may want to consider the more recent MRIcroGL as an alternative. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=152&screenshot_id=851"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=152&screenshot_id=851" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 19, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.mricro.com">http://www.mricro.com</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of South Carolina <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mricros" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MRIcroS">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mricros" rel="bookmark" name="mricros">MRIcroS</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-rendering" rel="tag">Surface Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=three-dimensional-display" rel="tag">Three Dimensional Display</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> When performing computational neuroscience-based analysis, it is often beneficial to view the data. Also, as Matlab is a common tool for such analysis, it is beneficial to visualize the data without having to open an external application that does not run in the Matlab environment.<br />
<br />
MRIcroS, a Matlab-based tool, provides:<br />
1) surface mesh visualization (PLY, PIAL, NV, STL,VTK, GIFTI formats)<br />
2) Convert NIfTI voxel images to surface meshes (and save as PLY or VTK)<br />
3) track (TRK files) visualization<br />
4) connectome data (BrainNet Viewer .node and .edge files) visualization<br />
5) intuitive GUI<br />
6) all functions available in the GUI are available through scripting. Therefore, automated scripts can be created<br />
7) can export rendered image as bitmap 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=877&screenshot_id=870"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=877&screenshot_id=870" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 22, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://www.nitrc.org/projects/mricros/">https://www.nitrc.org/projects/mricros/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of South Carolina, Medical University Of South Carolina <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, GIfTI, NIfTI-1, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, GIfTI, NIfTI-1, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mri_cvpr" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MRI_CVPR">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mri_cvpr" rel="bookmark" name="mri_cvpr">MRI_CVPR</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Tools processing MRI data with a number of techniques from cvpr conference, including segmentation, matching, features, and classification. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 7, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/medical_cvpr/">http://www.nitrc.org/projects/medical_cvpr/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mri-dataset-for-hippocampus-segmentation" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MRI Dataset for Hippocampus Segmentation">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mri-dataset-for-hippocampus-segmentation" rel="bookmark" name="mri-dataset-for-hippocampus-segmentation">MRI Dataset for Hippocampus Segmentation</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This dataset contains T1-weighted MR images of 50 subjects, 40 of whom are patients with temporal lobe epilepsy and 10 are nonepileptic subjects. Hippocampus labels are provided for 25 subjects for training. The users may submit their segmentation outcomes for the remaining 25 testing images to get a table of segmentation metrics. More information about the dataset may be found from the following reference:<br />
<br />
K. Jafari-Khouzani, K. Elisevich, S. Patel, and H. Soltanian-Zadeh, “Dataset of magnetic resonance images of nonepileptic subjects and temporal lobe epilepsy patients for validation of hippocampal segmentation techniques,” Neuroinformatics, 2011. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 22, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>HFH hippocampus segmentation</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/hippseg_2011/">http://www.nitrc.org/projects/hippseg_2011/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mri-defacer" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MRI Defacer">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mri-defacer" rel="bookmark" name="mri-defacer">MRI Defacer</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=intensity-operations" rel="tag">Intensity Operations</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=resampling" rel="tag">Resampling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-analysis" rel="tag">Surface Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Tool to remove facial features from an MRI structural image for the purpose of de-identification. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=249&screenshot_id=151"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=249&screenshot_id=151" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 22, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>FreeSurfer License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://surfer.nmr.mgh.harvard.edu/fswiki/AutomatedDefacingTools">https://surfer.nmr.mgh.harvard.edu/fswiki/AutomatedDefacingTools</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>MGH/MGZ</td></tr>
			<tr class="even"><td>Output Formats</td><td>MGH/MGZ</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mri-denoising" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MRI denoising">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mri-denoising" rel="bookmark" name="mri-denoising">MRI denoising</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Matlab package MRIdenoisingPackage contains five denoising filters and a noise estimation method for 3D MRI. The package includes nonlocal means, Oracle DCT and locally adaptive NLM methods.<br />
Based on image redundancy and/or sparsity, the proposed filters provide efficient denoising while preserving fine structures. The package is compiled for Linux (64 bits), Windows (64 bits) and MAC (64bits). The proposed GUI enables the processing of several .nii files in 2-clicks to make easier the processing of large database. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=806&screenshot_id=738"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=806&screenshot_id=738" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 18, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mri-denoising/">http://www.nitrc.org/projects/mri-denoising/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Bordeaux University, Universidad Politécnica de Valencia <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mri-neuroanatomy-labeling-services" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MRI Neuroanatomy Labeling Services">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mri-neuroanatomy-labeling-services" rel="bookmark" name="mri-neuroanatomy-labeling-services">MRI Neuroanatomy Labeling Services</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=consulting-services" rel="tag">Consulting Services</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=labeling" rel="tag">Labeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=manual" rel="tag">Manual</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=morphology" rel="tag">Morphology</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volumetric-analysis" rel="tag">Volumetric Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Neuromorphometrics provides brain labeling and measurement services. Given raw MRI brain scans, we make precise quantitative measurements of the volume, shape, and location of specific neuroanatomical structures. The final measurements result from automated analyses that are manually guided, inspected and certified by a neuroanatomical expert. More than two months of training and a great deal of practice are required for this person to be able to identify and delineate the borders of the structures on each slice where they appear. All data from raw images through final results are checked for completeness and integrity. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=597&screenshot_id=663"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=597&screenshot_id=663" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 18, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://Neuromorphometrics.com">http://Neuromorphometrics.com</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Neuromorphometrics, Inc. <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mri-predict" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MRI Predict">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mri-predict" rel="bookmark" name="mri-predict">MRI Predict</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MRI Predict is an SPM toolbox, FSL-flavour lib and R package to easily predict an outcome (e.g. whether the patient will respond to a treatment, or whether the patient has one or another disorder) from sMRI scans. The model may include covariates, and the software conducts both cross-validation of the model and fitting for its use with new VBM data in other dimensions. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 7, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.mripredict.com/">http://www.mripredict.com/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Translational Neuroimaging Group <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mri-simulator" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MRI Simulator">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mri-simulator" rel="bookmark" name="mri-simulator">MRI Simulator</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=experiment-control" rel="tag">Experiment Control</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=training" rel="tag">Training</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Psychology Software Tools MRI Simulator provides a realistic approximation of an actual MRI scanner to allow habituation and training of participants in an environment less daunting than a real scanner. Special populations such as children, the elderly, and psychiatric patients, are often prone to claustrophobia and anxiety in the bore of a magnet, and consequently have a much higher rate of terminating the experiment or scan session before its completion. Some centers that have dealt with these populations estimate a 50%-80% failure rate. With the use of the MRI Simulator this failure rate can often be reduced below 5%, improving cost effectiveness. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=208&screenshot_id=1034"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=208&screenshot_id=1034" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 22, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://pstnet.com/products/mri-simulator/">https://pstnet.com/products/mri-simulator/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mri-studio" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MRI Studio">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mri-studio" rel="bookmark" name="mri-studio">MRI Studio</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=diffusion-mr-fiber-tracking" rel="tag">Diffusion Mr Fiber Tracking</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MRI Studio is an image processing program running under Windows. It is suitable for such tasks as tensor calculation, color mapping, fiber tracking, and 3D visualization. Most of operations can be done with only a few clicks. This tool evolved from DTI Studio. (May also be called mristudio or dtistudio)<br />
<br />
Tools in the program can be grouped in the following way:<br />
<br />
* Image Viewer<br />
* Diffusion Tensor Calculations<br />
* Fiber Tracking and Editing<br />
* 3D Visualization<br />
* Image File Management<br />
* Region of Interesting (ROI) Drawing and Statistics<br />
* Image Registration </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 26, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.mristudio.org/">http://www.mristudio.org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Johns Hopkins University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Microsoft</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mriwatcher" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MriWatcher">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mriwatcher" rel="bookmark" name="mriwatcher">MriWatcher</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This simple visualization tool allows to load several images at the same time. The cursor across all windows are coupled and you can move/zoom on all the images at the same time.<br />
Very useful for quality control, image comparison. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 6, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mriwatcher/">http://www.nitrc.org/projects/mriwatcher/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mrtool" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MRTool">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mrtool" rel="bookmark" name="mrtool">MRTool</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MRTool is a comprehensive collection of analysis tools for MR brain imaging data. MRTool has been developed by Dr. Marco Ganzetti in the context of his doctoral degree in 2016. MRTool runs on Apple computers and PCs, it is very easy to install and only requires SPM12. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1117&screenshot_id=1195"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1117&screenshot_id=1195" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 15, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mrtool/">http://www.nitrc.org/projects/mrtool/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="mrtrix" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MRtrix">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=mrtrix" rel="bookmark" name="mrtrix">MRtrix</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=diffusion-mr-fiber-tracking" rel="tag">Diffusion Mr Fiber Tracking</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-display" rel="tag">Image Display</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MRtrix provides a set of tools to perform diffusion-weighted MR white-matter tractography in a manner robust to crossing fibres, using constrained spherical deconvolution (CSD) and probabilistic streamlines.<br />
<br />
Note: if you are looking for the latest unofficial version of MRtrix, MRtrix3, please visit this site: <a href="https://github.com/MRtrix3/mrtrix3" target="_new">https://github.com/MRtrix3/mrtrix3</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 25, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.brain.org.au/software">http://www.brain.org.au/software</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, NIfTI-1, Other Format, bshort/bfloat</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, NIfTI-1, Other Format, bshort/bfloat</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="multi-atlas-skull-stripping-(mass)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Multi Atlas Skull Stripping (MASS)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=multi-atlas-skull-stripping-(mass)" rel="bookmark" name="multi-atlas-skull-stripping-(mass)">Multi Atlas Skull Stripping (MASS)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MASS [1] is a software package designed for robust and accurate brain extraction, applicable for both individual as well as large population studies. MASS is implemented as a Unix command-line tool. It is fully automatic and easy to use — users input an image, and MASS will output the extracted brain and the associated brain mask. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 25, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>SBIA License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mass/">http://www.nitrc.org/projects/mass/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pennsylvania <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="multicomponent-t2-estimation-with-stimulated-echo-correction" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Multicomponent T2 estimation with stimulated echo correction">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=multicomponent-t2-estimation-with-stimulated-echo-correction" rel="bookmark" name="multicomponent-t2-estimation-with-stimulated-echo-correction">Multicomponent T2 estimation with stimulated echo correction</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This tool is designed to assist users in the estimation of multiple relaxation components from MRI T2 weighted spin-echo data such as that produced by a Carr-Purcell-Meiboom-Gill (CPMG) sequence. This problem is important to study myelin content in white matter diseases such as multiple sclerosis. <br />
<br />
Stimulated echoes arising from non-ideal flip angles are accounted for using the Extended Phase Graph (EPG) algorithm. The distribution is modelled as a small number of discrete components and a Bayesian estimation algorithm is provided to determine the weights and locations of the components as well as the actual flip angle. This algorithm outperforms iterative gradient descent based approaches. <br />
<br />
The details of the algorithm are given the following paper:<br />
Layton, K. J. et al. (2013) Modelling and Estimation of Multicomponent T2 Distributions, IEEE Transactions on Medical Imaging 32:1423-1434 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=734&screenshot_id=667"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=734&screenshot_id=667" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 16, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/multi_t2/">http://www.nitrc.org/projects/multi_t2/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Melbourne <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="multi-contrast-and-submillimetric-3-tesla-hippocampal-subfield-segmentation-protocol-and-dataset-(mni-hisub25)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Multi-contrast and submillimetric 3-Tesla hippocampal subfield segmentation protocol and dataset (MNI-HISUB25)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=multi-contrast-and-submillimetric-3-tesla-hippocampal-subfield-segmentation-protocol-and-dataset-(mni-hisub25)" rel="bookmark" name="multi-contrast-and-submillimetric-3-tesla-hippocampal-subfield-segmentation-protocol-and-dataset-(mni-hisub25)">Multi-contrast and submillimetric 3-Tesla hippocampal subfield segmentation protocol and dataset (MNI-HISUB25)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The MNI-HISUB25 dataset contains manual hippocampal subfield labels and associated high-resolution MRI data (submillimetric T1- and T2-weighted images, detailed sequence information, and stereotaxic probabilistic anatomical maps) based on 25 healthy subjects. Data were acquired on a 3-Tesla MRI system using a 32 phased-array head coil. The protocol divided the hippocampal formation into three subregions: subicular complex, Cornu Ammonis 1, 2 and 3 (CA1-3), and CA4-dentate gyrus (CA4-DG). Segmentation was guided by consistent intensity and morphology characteristics of the densely myelinated molecular layer together with few geometry-based boundaries flexible to overall mesiotemporal anatomy, and achieved excellent intra-/inter-rater reliability (Dice index ≥90/87%). The dataset can inform neuroimaging assessments of the mesiotemporal lobe and help to develop segmentation algorithms relevant for basic and clinical neurosciences. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=969&screenshot_id=946"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=969&screenshot_id=946" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 29, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mni-hisub25/">http://www.nitrc.org/projects/mni-hisub25/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Montreal Neurological Institute (MNI) <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="multi-echo-independent-component-regression-(me-icr)-group-level-connectivity-(groupincorr)-dataset" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Multi-Echo Independent Component Regression (ME-ICR) Group-Level Connectivity (GroupInCorr) Dataset">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=multi-echo-independent-component-regression-(me-icr)-group-level-connectivity-(groupincorr)-dataset" rel="bookmark" name="multi-echo-independent-component-regression-(me-icr)-group-level-connectivity-(groupincorr)-dataset">Multi-Echo Independent Component Regression (ME-ICR) Group-Level Connectivity (GroupInCorr) Dataset</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The data provided is an AFNI GroupInCorr session for multi-echo independent component regression (ME-ICR) for a cohort of 52 subjects. This dataset provides a high-quality atlas of seed-based functional connectivity with strong statistical conditioning. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 26, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Data License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/me-icr/">http://www.nitrc.org/projects/me-icr/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>National Institute of Mental Health <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="multi-fiber-reconstruction-from-dw-mri" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Multi-fiber Reconstruction from DW-MRI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=multi-fiber-reconstruction-from-dw-mri" rel="bookmark" name="multi-fiber-reconstruction-from-dw-mri">Multi-fiber Reconstruction from DW-MRI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=diffusion-mr-fiber-tracking" rel="tag">Diffusion Mr Fiber Tracking</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This program contains Python modules for modeling and reconstruction of diffusion weighted MRI data. It is a subset of the code internally used in the CVGMI lab at the University of Florida. Three different reconstruction methods are currently included in this program, namely, Mixture of Wisharts (MOW), Diffusion Orientation Transform (DOT) and Q-ball Imaging (QBI).<br />
<br />
This program is mainly developed and maintained by Bing Jian, as part of his Ph.D. research, supervised by Prof. Baba Vemuri.<br />
<br />
Please note that the source code of this program is hosted at Google Code, see the &quot;Source Code&quot; link on the left. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 9, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/diffusion-mri/">http://www.nitrc.org/projects/diffusion-mri/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="multi-modality-visualization-tool" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Multi-Modality Visualization Tool">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=multi-modality-visualization-tool" rel="bookmark" name="multi-modality-visualization-tool">Multi-Modality Visualization Tool</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The visualization and exploration of neuroimaging data are important for the analysis of anatomical and functional images and statistical parametric maps. While two-dimensional orthogonal views of neuroimaging data are used to display activity and statistical analysis, real three-dimensional (3d) depictions are helpful for showing the spatial distribution of a functional network, as well as its temporal evolution. For our best knowledge, currently, there is no neuroimaging 3d tool which can visualize both MEG, fMRI and invasive electrodes (ECOG, depth electrodes, DBS, etc.). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 16, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD/MIT-Style Open Source License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mmvt/">http://www.nitrc.org/projects/mmvt/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>MGH <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="multi-modal-mri-reproducibility-resource" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Multi-Modal MRI Reproducibility Resource">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=multi-modal-mri-reproducibility-resource" rel="bookmark" name="multi-modal-mri-reproducibility-resource">Multi-Modal MRI Reproducibility Resource</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> We have acquired scan-rescan imaging sessions on 21 healthy volunteers (no history of neurological disease). Imaging modalities include MPRAGE, FLAIR, DTI, resting state fMRI, B0 and B1 field maps, ASL, VASO, quantitative T1 mapping, quantitative T2 mapping, and magnetization transfer imaging. All data have been converted to NIFTI format. <br />
<br />
This is intended to be a resource for statisticians and imaging scientists to be able to quantify the reproducibility of their imaging methods using data available from a generic &quot;1 hour&quot; session at 3T.<br />
<br />
Please cite: Bennett. A. Landman, Alan J. Huang, Aliya Gifford, Deepti S. Vikram, Issel Anne L. Lim, Jonathan A.D. Farrell, John A. Bogovic, Jun Hua, Min Chen, Samson Jarso, Seth A. Smith, Suresh Joel, Susumu Mori, James J. Pekar, Peter B. Barker, Jerry L. Prince, and Peter C.M. van Zijl. “Multi-Parametric Neuroimaging Reproducibility: A 3T Resource Study”, NeuroImage. (2010) NIHMS/PMC:252138 doi:10.1016/j.neuroimage.2010.11.047 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=313&screenshot_id=253"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=313&screenshot_id=253" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 16, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BIRN Data License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/multimodal/">http://www.nitrc.org/projects/multimodal/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="multiple-correlation-function-tool" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Multiple Correlation Function Tool">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=multiple-correlation-function-tool" rel="bookmark" name="multiple-correlation-function-tool">Multiple Correlation Function Tool</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The intention of this tool is to provide a convenient environment to simulate NMR diffusion in closed pores. It builds on the eigenfunction expansion of the magnetization. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=655&screenshot_id=572"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=655&screenshot_id=572" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 10, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD/MIT-Style Open Source License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mcftool/">http://www.nitrc.org/projects/mcftool/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>German Cancer Research Center <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="multiscale-weighted-principal-component-regression" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Multiscale Weighted Principal Component Regression">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=multiscale-weighted-principal-component-regression" rel="bookmark" name="multiscale-weighted-principal-component-regression">Multiscale Weighted Principal Component Regression</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> In MWPCR, we build an importance score weight matrix for the selection<br />
of individual features at each location and a spatial weight matrix for the incorporation of the spatial pattern of feature vectors. We integrate the importance score weights with the spatial weights in order to recover the low dimensional structure of high dimensional features. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 10, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BIAS License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/mwpcr/">http://www.nitrc.org/projects/mwpcr/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="multitracer-version-2" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MultiTracer version 2">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=multitracer-version-2" rel="bookmark" name="multitracer-version-2">MultiTracer version 2</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=manual" rel="tag">Manual</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=region-growing" rel="tag">Region Growing</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=three-dimensional-display" rel="tag">Three Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=two-dimensional-display" rel="tag">Two Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volumetric-analysis" rel="tag">Volumetric Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MultiTracer version 2 is a compiled Java 1.6 application. It supports NIfTI-1.1 format float, double and signed and unsigned byte, short, and integer formats and provides legacy support for Analyze 7.5 8 and 16 bit images. It provides image display, editing, delineation of structure boundaries, export of traced contours and generation of masked volumes. Images are displayed in 3 orthogonal views. Time series can be displayed as averaged or contrast images and time courses can be visualized graphically. Version 2 provides enhancements to the original MultiTracer feature set described in Woods RP. Multitracer: a Java-based tool for anatomic delineation of grayscale volumetric images. Neuroimage 2003 Aug;19(4):1829-34.<br />
<br />
Documentation: <a href="http://air.bmap.ucla.edu/MultiTracer2/MultiTracer.html" target="_new">http://air.bmap.ucla.edu/MultiTracer2/Mu...</a><br />
Download: <a href="http://www.bmap.ucla.edu/portfolio/software/MultiTracer/" target="_new">http://www.bmap.ucla.edu/portfolio/softw...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 29, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://air.bmap.ucla.edu/MultiTracer2/MultiTracer.html">http://air.bmap.ucla.edu/MultiTracer2/MultiTracer.html</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UCLA <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="multivariate-general-linear-models-(mglm)-on-riemannian-manifolds" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Multivariate General Linear Models (MGLM) on Riemannian Manifolds">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=multivariate-general-linear-models-(mglm)-on-riemannian-manifolds" rel="bookmark" name="multivariate-general-linear-models-(mglm)-on-riemannian-manifolds">Multivariate General Linear Models (MGLM) on Riemannian Manifolds</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Statistical analysis tool for manifold-valued data. The SPD manifold for diffusion tensor images (DTI) and the Hilbert unit sphere for square root representation of orientation distribution functions (ODF) can be used. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 26, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/riem_mglm/">http://www.nitrc.org/projects/riem_mglm/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Wisconsin-Madison <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="multixplore:-visual-exploration-platform-for-multimodal-neuroimaging-data" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="MultiXplore: Visual Exploration Platform for Multimodal Neuroimaging Data">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=multixplore:-visual-exploration-platform-for-multimodal-neuroimaging-data" rel="bookmark" name="multixplore:-visual-exploration-platform-for-multimodal-neuroimaging-data">MultiXplore: Visual Exploration Platform for Multimodal Neuroimaging Data</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=connectivity-analysis" rel="tag">Connectivity Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=diffusion-application" rel="tag">Diffusion Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=modeling" rel="tag">Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-analysis" rel="tag">Surface Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tractography" rel="tag">Tractography</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MultiXplore is a graphical user interface that has been implemented as a 3D Slicer plugin (scripted module). It serves to display corresponding set of cortical regions from functional connectivity matrix in an explorable 3D scene that represents brain anatomical environment. In addition to grey matter regions, MultiXplore automatically finds and extracts deterministic fiber bundles which exist between selected region(s) and adds them to the 3D environment. This is a helpful feature in generating region-based fiber bundles given a desired whole-brain tractography data. Please make sure to cite our paper if you find this tool useful in your own work:<br />
Bakhshmand, S. M., Khan, A. R., de Ribaupierre, S., &amp; Eagleson, R. (2017). &quot;MultiXplore: Visual exploration platform for multimodal neuroimaging data&quot;. Journal of Neuroscience Methods. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1100&screenshot_id=1061"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1100&screenshot_id=1061" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 11, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>3D Slicer License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/multixplore/">http://www.nitrc.org/projects/multixplore/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Western University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1, Nrrd</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1, Nrrd</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="n2a" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="N2A">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=n2a" rel="bookmark" name="n2a">N2A</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=multi-neuron-modeling" rel="tag">Multi-Neuron Modeling</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> A scalable object-oriented language for modeling neural systems, along with an IDE for writing models and controlling simulations. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 8, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/n2a/">http://www.nitrc.org/projects/n2a/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Sandia National Laboratories <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="n3---minc-b0-nonuniformity-correction" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="N3 - MINC B0 nonuniformity correction">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=n3---minc-b0-nonuniformity-correction" rel="bookmark" name="n3---minc-b0-nonuniformity-correction">N3 - MINC B0 nonuniformity correction</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> nu_correct implements a novel approach to correcting for intensity non-uniformity in MR data that achieves high performance without requiring supervision. By making relatively few assumptions about the data, the method can be applied at an early stage in an automated data analysis, before a tissue intensity or geometric model is available. Described as Non-parametric Non-uniform intensity Normalization (N3), the method is independent of pulse sequence and insensitive to pathological data that might otherwise violate model assumptions. To eliminate the dependence of the field estimate on anatomy, an iterative approach is employed to estimate both the multiplicative bias field and the distribution of the true tissue intensities. Preprocessing of MR data using N3 has been shown to substantially improve the accuracy of anatomical analysis techniques such as tissue classification and cortical surface extraction. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 22, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MINC Licence (BSD'ish)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/nu_correct/">http://www.nitrc.org/projects/nu_correct/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="ncaa-dod-grand-alliance-care-consortium" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NCAA-DOD Grand Alliance CARE Consortium">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=ncaa-dod-grand-alliance-care-consortium" rel="bookmark" name="ncaa-dod-grand-alliance-care-consortium">NCAA-DOD Grand Alliance CARE Consortium</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-data-resource" rel="tag">Other Data Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The CARE Consortium endeavors to provide necessary infrastructure and scientific expertise to study concussion. Together, we are united in our goal to gain a better understanding of the neurobiopsychosocial nature of concussive injury and recovery in order to ultimately enhance the safety and health of our student-athletes, service members, youth sports participants and the broader public. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1163&screenshot_id=1078"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1163&screenshot_id=1078" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 4, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/care/">http://www.nitrc.org/projects/care/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Indiana University, University of Michigan, Medical College of Wisconsin <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="ncanda:-data-integration-component" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NCANDA: Data Integration Component">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=ncanda:-data-integration-component" rel="bookmark" name="ncanda:-data-integration-component">NCANDA: Data Integration Component</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-information-resource" rel="tag">Other Information Resource</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-software-resource" rel="tag">Other Software Resource</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=platform-or-development-environment" rel="tag">Platform Or Development Environment</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This is the Data Integration, MRI, and Bioinformatics Component of the National Consortium on Alcohol and NeuroDevelopment in Adolescence (NCANDA), funded by the NIAAA.<br />
<br />
The NCANDA consortium consists of an Administrative Component at UC San Diego, the Data Integration Component at SRI International, and five data collection sites, Duke University, Oregon Health &amp; Sciences University, SRI International, University of Pittsburgh, and UC San Diego. Each collection site will collect data from about 150 adolescents, each of them seen for one baseline and three annual follow-up visits.<br />
<br />
Here, we are making available manuals, training materials, and computational tools developed by the NCANDA Data Component. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=672&screenshot_id=595"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=672&screenshot_id=595" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 20, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/ncanda-datacore/">http://www.nitrc.org/projects/ncanda-datacore/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>SRI International <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1, PDF - Portable Document Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1, PDF - Portable Document Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nc-igt-fast-imaging-library" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NC-IGT Fast Imaging Library">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nc-igt-fast-imaging-library" rel="bookmark" name="nc-igt-fast-imaging-library">NC-IGT Fast Imaging Library</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-reconstruction" rel="tag">Image Reconstruction</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This software provides algorithms for the reconstruction of raw MR data. In<br />
particular, it supports the reconstruction of accelerated data acquisitions<br />
where k-space is subsampled and the Fourier domain encoding is complemented<br />
by temporal encoding, spatial encoding, or and/or a constrained<br />
reconstruction. This library of functions provides a number of<br />
reconstruction algorithms that accurately employ advanced MR imaging methods<br />
including: UNFOLD; parallel imaging methods such as SENSE and GRAPPA;<br />
Homodyne processing of partial-Fourier data, and gradient field<br />
inhomogeneity correction (gradwarp); EPI Nyquist Ghost correction and<br />
ramp-sampling gridding. The target audience is research groups who may be<br />
interested in exploring or employing advanced MR reconstruction techniques,<br />
but don't have the necessary expertise in-house.<br />
<br />
Inquires may be directed to: ncigt-imaging-toolkit -at- bwh.harvard.edu </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 17, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>NC-ICT Fast Imaging Library Contribution and Software License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://ncigt.org/fast-imaging-library">http://ncigt.org/fast-imaging-library</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nemo:-neuron-morphological-analysis-tool" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NeMo: NEuron MOrphological analysis tool">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nemo:-neuron-morphological-analysis-tool" rel="bookmark" name="nemo:-neuron-morphological-analysis-tool">NeMo: NEuron MOrphological analysis tool</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NEMO (NEuron MOrphological analysis tool) is a new freeware for semi automated quantitative and dynamic analysis of neuron morphometry.<br />
NEMO was co-developed at the [Italian National Council of Research Institute of Clinical Physiology (IFC-CNR)] (<a href="https://www.ifc.cnr.it/index.php/it/" target="_new">https://www.ifc.cnr.it/index.php/it/</a>) and the [Research Center &quot;E. Piaggio&quot;, University of Pisa] (<a href="http://www.centropiaggio.unipi.it/" target="_new">http://www.centropiaggio.unipi.it/</a>).<br />
It incorporates the most important microstructural quantification methods, such as fractal and sholl analysis with statistical and classification tools to provide an integrated image processing environment which enables fast and easy feature identification.<br />
It includes:<br />
Friendly interactive graphical user interface<br />
Image pre-processing<br />
Morphological analysis<br />
Topological analysis<br />
Cell counting<br />
*3-way PCA analysis <br />
Plot of variables<br />
For additional information contact: [lucia.billeci@ifc.cnr.it] (mailto:lucia.billeci@ifc.cnr.it) or [chiara.magliaro@googlemail.com] (mailto:chiara.magliaro@googlemail.com) </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 2, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/nemo_microscopy/">http://www.nitrc.org/projects/nemo_microscopy/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pisa <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="neo" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Neo">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=neo" rel="bookmark" name="neo">Neo</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Neo is a package for representing electrophysiology data in Python, together with support for reading a wide range of neurophysiology file formats, including Spike2, NeuroExplorer, AlphaOmega, Axon, Blackrock, Plexon, Tdt, and support for writing to a subset of these formats plus non-proprietary formats including HDF5.<br />
<br />
The goal of Neo is to improve interoperability between Python tools for analyzing, visualizing and generating electrophysiology data (such as OpenElectrophy, Elephant, G-node, Helmholtz, PyNN) by providing a common, shared object model. In order to be as lightweight a dependency as possible, Neo is deliberately limited to represention of data, with no functions for data analysis or visualization. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1211&screenshot_id=1128"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1211&screenshot_id=1128" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 3, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://neuralensemble.org/neo/">http://neuralensemble.org/neo/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>NeuralEnsemble community <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="neoseg" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Neoseg">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=neoseg" rel="bookmark" name="neoseg">Neoseg</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This tool computes an automatic segmentation of neonatal brain MRI. Automatic segmentation of these images is a challenging task mainly due to the low intensity contrast and the non-uniformity of white matter intensities, where white matter can be divided into early myelination regions and non-myelinated regions. The degree of myelination is a fractional voxel property that represents regional changes of white matter as a function of age. Our method makes use of a registered probabilistic brain atlas to select training samples and to be used as a spatial prior. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 21, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/neoseg/">http://www.nitrc.org/projects/neoseg/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Utah <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="neosegpipeline---multisegpipeline" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NeoSegPipeline - MultisegPipeline">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=neosegpipeline---multisegpipeline" rel="bookmark" name="neosegpipeline---multisegpipeline">NeoSegPipeline - MultisegPipeline</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This tool allows the tissue segmentation of brain MRI using a subject-specific atlas, as well as a generic multi-atlas based segmentation of regional label definitions. For the tissue segmentation, it generates a subject-specific atlas based on an atlas population propagated to the individual subject image (using optionally also using diffusion images). Then we run a single atlas method with this atlas and obtain some good results. A quick demo on how to use the tool can be seen here: <a href="https://www.youtube.com/watch?v=yTr3NJZaMos" target="_new">https://www.youtube.com/watch?v=yTr3NJZa...</a> . The MultisegPipeline version allows also regional generic multi-atlas segmentation.<br />
<br />
Recent publication <a href="https://www.spiedigitallibrary.org/conference-proceedings-of-spie/10953/109530K/Multiseg-pipeline--automatic-tissue-segmentation-of-brain-MR-images/10.1117/12.2513237.short?SSO=1" target="_new">https://www.spiedigitallibrary.org/confe...</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=836&screenshot_id=789"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=836&screenshot_id=789" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 17, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/neosegpipeline/">http://www.nitrc.org/projects/neosegpipeline/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>NIRAL <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="netpet:-covariance-statistics-and-network-analysis-of-brain-pet-imaging-studies" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NetPET: Covariance statistics and network analysis of brain PET imaging studies">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=netpet:-covariance-statistics-and-network-analysis-of-brain-pet-imaging-studies" rel="bookmark" name="netpet:-covariance-statistics-and-network-analysis-of-brain-pet-imaging-studies">NetPET: Covariance statistics and network analysis of brain PET imaging studies</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The analysis of brain structural and functional imaging data using graph theory has increasingly become a popular approach since brain topological representations have proven to be helpful for visualising and understanding anatomical and functional relationships between different cerebral areas.<br />
<br />
NetPET is a tool for graph-based analysis of brain PET studies which exploits population-based covariance matrices. NetPET aims to explore topological tracer kinetics differences in cross-sectional investigations. Simulations, test-retest studies and applications to cross-sectional datasets from three different tracers ([18F]FDG, [18F]FDOPA and [11C]SB217045) and more than 400 PET scans have been used to test the applicability of NetPET in healthy controls and patients. Results showed good reproducibility and general applicability of the methods within the range of experimental settings typical of PET neuroimaging studies. <br />
<br />
NetPET is a Matlab-based software package. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1271&screenshot_id=1192"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1271&screenshot_id=1192" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 1, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/netpet_2018/">http://www.nitrc.org/projects/netpet_2018/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>King's College London <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="net-station" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Net Station">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=net-station" rel="bookmark" name="net-station">Net Station</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> APIs for Net Station data files. APIs are available for C++, C#, and Java. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 21, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.egi.com/clinical-division/clinical-division-geodesic-eeg-components/clinical-division-net-station">http://www.egi.com/clinical-division/clinical-division-geodesic-eeg-components/clinical-division-net-station</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Electrical Geodesics, Inc. <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="network-based-statistic-(nbs)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Network-Based Statistic (NBS)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=network-based-statistic-(nbs)" rel="bookmark" name="network-based-statistic-(nbs)">Network-Based Statistic (NBS)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=connectivity-analysis" rel="tag">Connectivity Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=statistical-operation" rel="tag">Statistical Operation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Matlab toolbox for testing hypotheses about the human connectome. NBS has been widely used to identify connections and networks comprising the connectome that are associated with an experimental effect or a between-group difference. User provides a series of connectivity matrices from different cohorts, or from the same subject during different experimental conditions. Connectivity matrices are inferred from neuroimaging data using other packages that, for example, count the number of tractography streamlines that interconnect each pair of regions (diffusion-MRI), or measure the extent of inter-regional correlation in BOLD response (fMRI). User specifies hypothesis to be tested at every connection with the general linear model. Features include: graphical user interface; NBSview, a basic network viewer modeled on SPMresults; exchange blocks for repeated measures; options to measure network size with intensity or extent; false discovery rate (FDR) option. Developed by Zalesky, Fornito, Cocchi and Bullmore. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=629&screenshot_id=548"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=629&screenshot_id=548" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 9, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/nbs/">http://www.nitrc.org/projects/nbs/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Melbourne <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="network-modification-(nemo)-tool-lite" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Network Modification (NeMo) Tool Lite">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=network-modification-(nemo)-tool-lite" rel="bookmark" name="network-modification-(nemo)-tool-lite">Network Modification (NeMo) Tool Lite</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The NeMo Tool is the first that associates localized white matter (WM) lesions with<br />
disruptions in gray matter connectivity as a step toward understanding the lesions’ functional<br />
implications. A Tractogram Reference Set (TRS), i.e. collections of white matter fibers, is<br />
constructed from 73 normal healthy individuals and coregistered to a common space (MNI). The<br />
NeMo Tool uses the TRS to assess structural network disruption due to a particular WM lesion<br />
mask on a region and network-wise level. This tool is an easy way for researchers and clinicians<br />
to investigate changes in the structural brain network without having to perform tractography on<br />
their own normal data or on diseased/injured brains where the results may not represent the<br />
underlying physiology. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 28, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/brainnet_2013/">http://www.nitrc.org/projects/brainnet_2013/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Weill Cornell Medical College <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="neural-electromagnetic-ontologies" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Neural ElectroMagnetic Ontologies">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=neural-electromagnetic-ontologies" rel="bookmark" name="neural-electromagnetic-ontologies">Neural ElectroMagnetic Ontologies</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=ontology" rel="tag">Ontology</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Neural ElectroMagnetic Ontologies (NEMO) is an NIH funded project that aims to create EEG and MEG ontologies and ontology based tools. Project documentation can be accessed on the NEMO website:<br />
<br />
<a href="http://aimlab.cs.uoregon.edu/NEMO/web/" target="_new">http://aimlab.cs.uoregon.edu/NEMO/web/</a><br />
<br />
Project resources (ontologies, database, portal, and pattern analysis tools) can be downloaded from Sourceforge:<br />
<br />
<a href="https://sourceforge.net/projects/nemoontologies/" target="_new">https://sourceforge.net/projects/nemoont...</a><br />
<br />
These resources will be used to support representation, classification, and meta-analysis of brain electromagnetic data. The three pillars of NEMO are: DATA, ONTOLOGY, and DATABASE. NEMO data consist of raw EEG, averaged EEG (ERPs), and ERP data analysis results. NEMO ontologies include concepts related to ERP data (including spatial and temporal features of ERP patterns), data provenance, and the cognitive and linguistic paradigms that were used to collect the data. The NEMO database is a large repository that stores NEMO consortium data, data analysis results, and data provenance. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 10, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>NEMO License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/nemo/">http://www.nitrc.org/projects/nemo/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Oregon <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="neuritetracer" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NeuriteTracer">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=neuritetracer" rel="bookmark" name="neuritetracer">NeuriteTracer</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NeuriteTracer is a set of ImageJ plugins for automated measurement of neurite outgrowth in fluorescence microscopy images of cultured neurons. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=879&screenshot_id=853"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=879&screenshot_id=853" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 24, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial Share Alike</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://fournierlab.mcgill.ca/styled-6/NeuriteTracer.html">http://fournierlab.mcgill.ca/styled-6/NeuriteTracer.html</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>McGill University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="neuroconstruct" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="neuroConstruct">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=neuroconstruct" rel="bookmark" name="neuroconstruct">neuroConstruct</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> neuroConstruct is being developed in the Silver Lab in the Department of Neuroscience, Physiology and Pharmacology at UCL. neuroConstruct has been designed to simplify development of complex networks of biologically realistic neurons, i.e. models incorporating dendritic morphologies and realistic cell membrane conductances. It is implemented in Java and generates script files for the NEURON and GENESIS simulators, with support for other simulation platforms (including PSICS, MOOSE and PyNN) in advanced stages of development. It uses the latest NeuroML specifications, including MorphML, ChannelML and NetworkML, and supports NeuroML v2.0. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=866&screenshot_id=841"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=866&screenshot_id=841" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 1, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.neuroconstruct.org/">http://www.neuroconstruct.org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University College London <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="neurodebian" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NeuroDebian">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=neurodebian" rel="bookmark" name="neurodebian">NeuroDebian</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-software-resource" rel="tag">Other Software Resource</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=platform-or-development-environment" rel="tag">Platform Or Development Environment</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This is a collaborative effort to package neuroscience-related software for the Debian operating system. The main goal of the project is to provide a versatile and convenient environment for neuroscientific research that is based on open-source software. To this end, the project offers a package repository that complements the main Debian (and Ubuntu) archive. NeuroDebian is not yet another Linux distribution, but rather an effort inside the Debian project itself. Software packages are fully integrated into the Debian system and from there will eventually migrate into Ubuntu as well.<br />
<br />
With NeuroDebian, installing and updating neuroscience software is no different from any other part of the operating system. Maintaining a research software environment becomes as easy as installing an editor. For example, installing FSL looks like: &quot;apt-get install fsl&quot;<br />
<br />
There is also virtual machine to test NeuroDebian on Windows or Mac OS.<br />
<br />
If you want to see your software packaged for Debian, please drop us a note. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=365&screenshot_id=236"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=365&screenshot_id=236" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 19, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://neuro.debian.net">http://neuro.debian.net</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>FreeBSD<br>Linux<br>Other UNIX-like</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>AFNI BRIK, DICOM, MINC2, NIfTI-1, Other Format, PDF - Portable Document Format, Philips PAR/REC, bshort/bfloat</td></tr>
			<tr class="even"><td>Output Formats</td><td>AFNI BRIK, DICOM, MINC2, NIfTI-1, Other Format, PDF - Portable Document Format, Philips PAR/REC, bshort/bfloat</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="neurodevelopmental-mri-database" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Neurodevelopmental MRI Database">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=neurodevelopmental-mri-database" rel="bookmark" name="neurodevelopmental-mri-database">Neurodevelopmental MRI Database</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This is a database of average MRIs and associated MRI volumes for developmental MRI work. It consists of average MRI templates, segmented partial volume estimate volumes for GM, WM, T2W-derived CSF. The database is separated into head-based and brain-based averages. The data are separated by ages in months, years, 6-month, or 5-year intervals. The templates are grouped into first year (2 weeks through 12 months), early childhood (15 months through 4 years), childhood (4 years through 10 years), adolescence (10.5 years through 17.5 years) and adults (18 years through 89 years). Tools for cortical source analysis of EEG and ERP are provided. These tools are based on the average MRI templates, segmenting, and atlases. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 5</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/neurodevdata/">http://www.nitrc.org/projects/neurodevdata/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of South Carolina <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="neurodocker" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Neurodocker">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=neurodocker" rel="bookmark" name="neurodocker">Neurodocker</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=application" rel="tag">Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=platform-or-development-environment" rel="tag">Platform Or Development Environment</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Generate Dockerfiles and Singularity recipes for neuroimaging with a simple command-line interface. Neurodocker supports a growing list of software, including AFNI, ANTs, Convert3D, Dcm2niix, FreeSurfer, FSL, Matlab Compiler Runtime, MINC, Miniconda, MRtrix3, NeuroDebian, PETPVC, and SPM12. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 20, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://hub.docker.com/r/kaczmarj/neurodocker/">https://hub.docker.com/r/kaczmarj/neurodocker/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="neuroelf" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NeuroElf">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=neuroelf" rel="bookmark" name="neuroelf">NeuroElf</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=format-conversion" rel="tag">Format Conversion</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-software-resource" rel="tag">Other Software Resource</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=statistical-operation" rel="tag">Statistical Operation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=time-domain-analysis" rel="tag">Time Domain Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NeuroElf is a Matlab-based toolbox initially created for reading, writing, and processing of BrainVoyager (QX) files in Matlab. Some additional routines are available, such as the scripting of preprocessing using SPM or second-level statistics. The toolbox is freely available. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=68&screenshot_id=775"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=68&screenshot_id=775" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 15, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://neuroelf.net/">http://neuroelf.net/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Columbia University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>AFNI BRIK, ANALYZE, DICOM, NIfTI-1, Other Format, Philips PAR/REC</td></tr>
			<tr class="even"><td>Output Formats</td><td>AFNI BRIK, ANALYZE, DICOM, NIfTI-1, Other Format, Philips PAR/REC</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="neuroimage-field-of-view-normalization-tool" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NeuroImage Field-of-View Normalization Tool">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=neuroimage-field-of-view-normalization-tool" rel="bookmark" name="neuroimage-field-of-view-normalization-tool">NeuroImage Field-of-View Normalization Tool</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Field-of-view mismatch in different subjects' images poses additional challenges to the streamline analysis of large-scale brain MRI datasets. <br />
<br />
We release an atlas-based FOV normalization tool. This tool automatically truncates the FOV to that of the MNI152 atlas', which includes the whole brain, the skull, and ends 10-15mm inferior to the most inferior voxel in the cerebellum. This tool runs in linux platform (command line). It has been extensively validated in hundreds of MRIs, acquired from various scanners (Philips 1.0T, Siemens/GE 1.5T/3T), in various imaging sites, with various scanning FOVs, and from subjects of various ages (infants, neonates, adolescents, young adults, elderly).<br />
<br />
We have quantitatively shown that this FOV normalization significantly improves the accuracy and success rate in streamline analysis of neuroimages in the very first steps (e.g., skull stripping, inter-subject registration, etc).<br />
<br />
The software runs in unix command line, single machine or clusters.<br />
<br />
Publication pending. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=934&screenshot_id=927"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=934&screenshot_id=927" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 25, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/normalizefov/">http://www.nitrc.org/projects/normalizefov/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Harvard Medical School <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="neuro-image-management-system-(nims)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Neuro-Image Management System (NIMS)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=neuro-image-management-system-(nims)" rel="bookmark" name="neuro-image-management-system-(nims)">Neuro-Image Management System (NIMS)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NIMS is a scientific data management system specifically designed for neuroimaging data. NIMS automatically reaps data from the measurement instrument (e.g., MR scanner), sorts and organizes the data based on header information, does some basic processing on the data, and makes the data available to authorized users through a web-based interface. The data are also available from the command-line through a FUSE-based filesystem. See a live demo at the CNI public data site: <a href="http://cni.stanford.edu/nims/pub/browse." target="_new">http://cni.stanford.edu/nims/pub/browse.</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 18, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MIT/X Consortium License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/nims/">http://www.nitrc.org/projects/nims/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Stanford University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="neuroimaging-analysis-kit-(niak)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NeuroImaging Analysis Kit (NIAK)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=neuroimaging-analysis-kit-(niak)" rel="bookmark" name="neuroimaging-analysis-kit-(niak)">NeuroImaging Analysis Kit (NIAK)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=artifact-removal" rel="tag">Artifact Removal</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=constrained-region-growing" rel="tag">Constrained Region Growing</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=filtering" rel="tag">Filtering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-to-template" rel="tag">Image-To-Template</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=independent-component-analysis" rel="tag">Independent Component Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=intensity-non-uniformity-correction" rel="tag">Intensity Non-Uniformity Correction</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=intermodal" rel="tag">Intermodal</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=intrasubject" rel="tag">Intrasubject</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=linear" rel="tag">Linear</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=principal-component-analysis" rel="tag">Principal Component Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quality-metrics" rel="tag">Quality Metrics</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=resampling" rel="tag">Resampling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NIAK is a library of modules and pipelines for fMRI processing with Octave that can run in parallel either locally or in a supercomputing environment. Windows/Mac OSX/Linux are supported, as well as NIFTI and MINC file formats. NIAK currently includes a preprocessing, a functional parcellation and a connectome pipelines. NIAK features powerful pipeline management capabilities, including parallel computing, generation of detailed logs and automatic handling of pipeline failures or updates.<br />
<a href="http://niak.simexp-lab.org/" target="_new">http://niak.simexp-lab.org/</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=411&screenshot_id=288"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=411&screenshot_id=288" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 26, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MIT/X Consortium License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://niak.simexp-lab.org/">http://niak.simexp-lab.org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Montreal <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>POSIX/UNIX-like</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>MINC, MINC2</td></tr>
			<tr class="even"><td>Output Formats</td><td>MINC, MINC2</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="neuroimaging-data-access-group" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Neuroimaging Data Access Group">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=neuroimaging-data-access-group" rel="bookmark" name="neuroimaging-data-access-group">Neuroimaging Data Access Group</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=community" rel="tag">Community</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The NeuroImaging Data Access Group (NIDAG) is an informal working group, dedicated to improving access to neuroimaging results in a free and open-access manner. Our current project involves the creation of a comprehensive database of neuroimaging results searchable based on standardized coordinates. Once complete, this will allow anyone to find all of the articles that report a coordinate, or set of coordinates, easily and without cost. Eventually, we hope to expand this database to include not only coordinates, but statistical parametric maps as well. Formation of such a database will increase the likelihood of relevant papers being found and cited, and also be a very useful tool for those interested in meta-analysis, and hopefully clarify structure-function relationships.<br />
<br />
Please visit our website (nidag.org) and sign up to join our mailing list. We are interested in hearing from people who might be willing to contribute to our projects, particularly those with programming experience. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=398&screenshot_id=298"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=398&screenshot_id=298" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 27, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>NIDAG</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nidag.org">http://www.nidag.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="neuroimaging-made-easy-blog" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Neuroimaging Made Easy Blog">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=neuroimaging-made-easy-blog" rel="bookmark" name="neuroimaging-made-easy-blog">Neuroimaging Made Easy Blog</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other" rel="tag">Other</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-software-resource" rel="tag">Other Software Resource</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The main aim of this blog is presenting some scripts that can be used to facilitate and automate processing and analysis of brain data. In addition, it could be helpful explaining non clear stages and steps of brain data processing using some software such as; Freesurfer, FSL, Brainvoyager QX... At the moment, there are more than 10 applescripts in the main website &quot;http://www.easyneuroimaging.com&quot; that control different tools and commands (aparcstats2table, asegstats2table, BET, dcm2nii, FIRST, fslsplit, fslswapdim, fslview, mri_convert, Qdec, Recon-all, SIENAX, tkmedit, tksurfer) 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=556&screenshot_id=468"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=556&screenshot_id=468" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 24, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/easy_neuroimage">http://www.nitrc.org/projects/easy_neuroimage</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, ASCII, DICOM, MGH/MGZ, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, ASCII, DICOM, MGH/MGZ, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="neuroinformatics---the-journal" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Neuroinformatics - The Journal">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=neuroinformatics---the-journal" rel="bookmark" name="neuroinformatics---the-journal">Neuroinformatics - The Journal</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=journal" rel="tag">Journal</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Neuroinformatics publishes original articles and reviews with an emphasis on data structure and software tools related to analysis, modeling, integration, and sharing in all areas of neuroscience research. Coverage extends to theory and methodology, including discussions on ontologies, modeling approaches, database design, and meta-analyses; descriptions of developed databases and software tools, and of the methods for their distribution; relevant experimental results, such as reports accompanied by the release of massive data sets; computational simulations of models integrating and organizing complex data; and neuroengineering approaches, including hardware, robotics, and information theory studies. Neuroinformatics also publishes independent &quot;tests and evaluations&quot; of available neuroscience databases and software tools, and fosters a commitment to the principles of tool and data sharing. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=163&screenshot_id=83"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=163&screenshot_id=83" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 15, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.springer.com/biomed/neuroscience/journal/12021">http://www.springer.com/biomed/neuroscience/journal/12021</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="neurolens" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NeuroLens">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=neurolens" rel="bookmark" name="neurolens">NeuroLens</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-to-image" rel="tag">Image-To-Image</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=linear" rel="tag">Linear</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=multivariate-analysis" rel="tag">Multivariate Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-rendering" rel="tag">Surface Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=temporal-convolution---deconvolution" rel="tag">Temporal Convolution - Deconvolution</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=three-dimensional-display" rel="tag">Three Dimensional Display</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NeuroLens is an integrated environment for the analysis and visualization of functional neuroimages. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 18, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MGH CSRL License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.neurolens.org/NeuroLens/Home.html">http://www.neurolens.org/NeuroLens/Home.html</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>AFNI BRIK, ANALYZE, COR, DICOM, MGH/MGZ, MINC, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>AFNI BRIK, ANALYZE, COR, DICOM, MGH/MGZ, MINC, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="neurolucida" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Neurolucida">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=neurolucida" rel="bookmark" name="neurolucida">Neurolucida</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Neurolucida is a powerful tool for creating and analyzing realistic, meaningful, and quantifiable neuron reconstructions from microscope images. Perform detailed morphometric analysis of neurons, such as: the number of dendrites, axons, nodes, synapses, and spines the length, width, and volume of dendrites and axons, area and volume of the soma, and the complexity and extension of neurons. <a href="http://www.mbfbioscience.com/neurolucida" target="_new">http://www.mbfbioscience.com/neurolucida</a><br />
 <br />
Researchers have reconstructed and analyzed tens of thousands of neurons using Neurolucida, leading to advances in areas of neuroscience including neurodegenerative diseases, neuropathy, memory, and behavior, and advances in research fields such as ophthalmology. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 15, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/neurolucida/">http://www.nitrc.org/projects/neurolucida/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>MBF Bioscience <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="neuroml" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NeuroML">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=neuroml" rel="bookmark" name="neuroml">NeuroML</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NeuroML is a standardized model description language for computational neuroscience. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1305&screenshot_id=1222"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1305&screenshot_id=1222" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 20, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Lesser General Public License (LGPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.neuroml.org">http://www.neuroml.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="neuromorpho.org" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NeuroMorpho.Org">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=neuromorpho.org" rel="bookmark" name="neuromorpho.org">NeuroMorpho.Org</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NeuroMorpho.Org is a centrally curated inventory of digitally reconstructed neurons associated with peer-reviewed publications. It contains contributions from over 100 laboratories worldwide and is continuously updated as new morphological reconstructions are collected, published, and shared. To date, NeuroMorpho.Org is the largest collection of publicly accessible 3D neuronal reconstructions and associated metadata.<br />
<br />
The goal of NeuroMorpho.Org is to provide dense coverage of available reconstruction data for the neuroscience community. Data sharing through NeuroMorpho.Org enables the full and continuing research potential of existing digital reconstruction data. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 15, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/neuromorpho_org/">http://www.nitrc.org/projects/neuromorpho_org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>George Mason University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="neuron" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NEURON">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=neuron" rel="bookmark" name="neuron">NEURON</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NEURON is a simulation environment for modeling individual neurons and networks of neurons. It provides tools for conveniently building, managing, and using models in a way that is numerically sound and computationally efficient. It is particularly well-suited to problems that are closely linked to experimental data, especially those that involve cells with complex anatomical and biophysical properties. For a more detailed description see <a href="http://www.neuron.yale.edu/neuron/what_is_neuron" target="_new">http://www.neuron.yale.edu/neuron/what_i...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 2, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://www.neuron.yale.edu/">https://www.neuron.yale.edu/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>OS/2<br>Other UNIX-like<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="neuron-c" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Neuron-C">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=neuron-c" rel="bookmark" name="neuron-c">Neuron-C</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Neuron-C is a simulation language for modeling biophysically realistic neural circuits. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 19, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial Share Alike</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/neuron-c/">http://www.nitrc.org/projects/neuron-c/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pennsylvania, Dept of Neuroscience <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="neuronj" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NeuronJ">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=neuronj" rel="bookmark" name="neuronj">NeuronJ</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NeuronJ is an ImageJ plugin for computer-aided delineation of elongated image structures and the measurement of basic length statistics. It was developed primarily to facilitate the tracing and quantification of neurites in 2D fluorescence microscopy images. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 16, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>NeuronJ License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/neuronj/">http://www.nitrc.org/projects/neuronj/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="neuropub-visualizer" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NeuroPub Visualizer">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=neuropub-visualizer" rel="bookmark" name="neuropub-visualizer">NeuroPub Visualizer</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NeuroPub is a NIfTI visualisation tool for the iPad. You can use it to store all your statistical images from your fMRI/VBM/TBSS studies and visualise them in 2D and 3D. Use NeuroPub as a library for your statistical images. You can download the app for free from the App Store! </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 2, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/neuropub/">http://www.nitrc.org/projects/neuropub/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="neuroscience-gateway-(nsg)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Neuroscience Gateway (NSG)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=neuroscience-gateway-(nsg)" rel="bookmark" name="neuroscience-gateway-(nsg)">Neuroscience Gateway (NSG)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> * Administratively Listed Project *<br />
The Neuroscience Gateway (NSG) portal facilitates access and use of National Science Foundation (NSF) High Performance Computing (HPC) resources by neuroscientists. It will offer free computer time to neuroscientists acquired via the supercomputer time allocation process managed by the Extreme Science and Engineering Discovery Environment (XSEDE) Resource Allocation Committee (XRAC). The portal provides access to the popular computational neuroscience tools installed on various HPC resources. It also provides a community mailing list for neuroscientists to collaborate and share ideas.<br />
<br />
* NOTE *<br />
This project is administratively added to NITRC for completeness; use with caution. <br />
The information was set from public content on 02-OCT-2017, and is not maintained.<br />
If you are (or know) an appropriate administrator, contact <a href="mailto:nitrc_info@nitrc.org" target="_new">nitrc_info@nitrc.org</a>. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1224&screenshot_id=1143"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1224&screenshot_id=1143" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 17, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://www.nsgportal.org/index.html">https://www.nsgportal.org/index.html</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UCSD, Yale University, University College London <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="neuroscope" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NeuroScope">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=neuroscope" rel="bookmark" name="neuroscope">NeuroScope</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NeuroScope is an advanced viewer for electrophysiological and behavioral data: it can display local field potentials (EEG), neuronal spikes, behavioral events, as well as the position of the animal in the environment. It also features limited editing capabilities. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=608&screenshot_id=715"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=608&screenshot_id=715" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 5, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/neuroscope/">http://www.nitrc.org/projects/neuroscope/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="neurosharelibrary" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NeuroshareLibrary">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=neurosharelibrary" rel="bookmark" name="neurosharelibrary">NeuroshareLibrary</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=format-conversion" rel="tag">Format Conversion</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This is MATLAB library to create Neuroshare data format.<br />
You can convert your own data into Neuroshare format file. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=488&screenshot_id=500"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=488&screenshot_id=500" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 5, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.cns.atr.jp/dni/en/downloads/tools-for-brain-behavior-data-sharing/">http://www.cns.atr.jp/dni/en/downloads/tools-for-brain-behavior-data-sharing/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Computational Neuroscience Labs <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="neurostars" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NeuroStars">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=neurostars" rel="bookmark" name="neurostars">NeuroStars</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=community" rel="tag">Community</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> A question and answer site for neuroinformatics. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1357&screenshot_id=1259"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1357&screenshot_id=1259" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 11</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://neurostars.org/">https://neurostars.org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Managed by International Neuroinformatics Coordinating Facility (INCF) <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="neurosynth" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NeuroSynth">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=neurosynth" rel="bookmark" name="neurosynth">NeuroSynth</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=knowledge-environment" rel="tag">Knowledge Environment</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=websites" rel="tag">Websites</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The NeuroSynth framework includes software tools for automated extraction and synthesis of data from published functional neuroimaging studies, a series of datasets generated using these tools, and a website ( <a href="http://neurosynth.org" target="_new">http://neurosynth.org</a> ) for visualizing and accessing many of the tools and results. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=508&screenshot_id=503"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=508&screenshot_id=503" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 11, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Open Software License 3.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://neurosynth.org/">http://neurosynth.org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="neurovault" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NeuroVault">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=neurovault" rel="bookmark" name="neurovault">NeuroVault</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NeuroVault makes uploading statistical maps easy and straightforward. In addition it provides permanent links to attractive volumetric and 3D (surface based) visualisations of your data. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 5, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://neurovault.org">http://neurovault.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="neuroviisas" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="neuroVIISAS">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=neuroviisas" rel="bookmark" name="neuroviisas">neuroVIISAS</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Functions of neuroVIISAS:<br />
1) Generation of terminologies from lists of regions up to complex ontologies.<br />
2) Linking external databases to nodes of ontologies<br />
3) Define and work with competing, developing or variable terminologies<br />
3) Defining stereotaxic coordinate systems<br />
4) Import stacks of images: atlas contour images, histological, MRI, DTI, virtual slides<br />
5) Definition of relations between ontologies and images<br />
6) Import lists of neuronal connections<br />
7) Visualizing sources and targets of connections in stereotaxic atlases<br />
8) Nested 3D-atlas visualizations<br />
10) Hierarchy based connectome navigation<br />
11) Matrix visualizations of hierarchical and non-hierarchical connectomes <br />
12) Interactive connectome exploration<br />
13) Differential connectomics<br />
14) Global and local connectome analysis<br />
15) Multivariate and graph theoretical analysis and visualization<br />
16) Phyton script generator for population simulations using atlas and connectome data.<br />
17) Numerical and 2D-, 3D-simulation visualizations </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 26, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>neuroVIISAS</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://neuroviisas.med.uni-rostock.de/neuroviisas.shtml">http://neuroviisas.med.uni-rostock.de/neuroviisas.shtml</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University Rostock <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="neuroweb---neuroimaging-database" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NeuroWeb - NeuroImaging Database">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=neuroweb---neuroimaging-database" rel="bookmark" name="neuroweb---neuroimaging-database">NeuroWeb - NeuroImaging Database</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NeuroWeb provides infrastructure for data aggregation, processing, and management in multi-dimensional medical imaging research (i.e., MRI, CT, PET). NeuroWeb is designed for rapid deployment on a small/moderate scale with limited hardware. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 19, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/neuroweb/">http://www.nitrc.org/projects/neuroweb/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nft:-neuroelectromag-forward-modeling" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NFT: Neuroelectromag Forward Modeling">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nft:-neuroelectromag-forward-modeling" rel="bookmark" name="nft:-neuroelectromag-forward-modeling">NFT: Neuroelectromag Forward Modeling</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=eeg-modeling" rel="tag">Eeg Modeling</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Neuroelectromagnetic Forward Head Modeling Toolbox (NFT) (<a href="http://sccn.ucsd.edu/nft" target="_new">http://sccn.ucsd.edu/nft</a>) is a MATLAB Toolbox for generating realistic head models from available data (MRI and/or electrode locations), for computing numerical solutions for the forward problem of electromagnetic source imaging and for single dipole source localization. The NFT includes tools for segmenting scalp, skull, cerebrospinal fluid (CSF) and brain tissues from T1-weighted magnetic resonance (MR) images. The Boundary Element Method (BEM) and Finite Element Method (FEM) are used for the numerical solution of the forward problem. When a subject MR image is not available a template head model can be warped to measured electrode locations to obtain an individualized head model. Toolbox functions may be called either from a graphic user interface compatible with EEGLAB or from the MATLAB command line. The toolbox is freely available under the GNU Public License for noncommercial use and open source development. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=565&screenshot_id=613"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=565&screenshot_id=613" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 24, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://sccn.ucsd.edu/wiki/NFT">http://sccn.ucsd.edu/wiki/NFT</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UCSD <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="niag-addiction-data" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NIAG Addiction Data">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=niag-addiction-data" rel="bookmark" name="niag-addiction-data">NIAG Addiction Data</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Addiction data </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 27, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://niag.ir/">http://niag.ir/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Tehran University of Medical Sciences <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="niagads---nia-genetics-of-alzheimer's-disease-data-storage-site" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NIAGADS - NIA Genetics of Alzheimer's Disease Data Storage Site">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=niagads---nia-genetics-of-alzheimer's-disease-data-storage-site" rel="bookmark" name="niagads---nia-genetics-of-alzheimer's-disease-data-storage-site">NIAGADS - NIA Genetics of Alzheimer's Disease Data Storage Site</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=next-generation-sequencing" rel="tag">Next Generation Sequencing</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-data-resource" rel="tag">Other Data Resource</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=snp's-&-genes" rel="tag">Snp'S & Genes</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=web-service" rel="tag">Web Service</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=websites" rel="tag">Websites</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NIAGADS (NIA Genetics of Alzheimer's Disease Data Storage Site) is a national genetics data repository that facilitates access of genotypic data to qualified investigators for the study of the genetics of late-onset Alzheimer's disease. It is the policy of the NIA that all genetic data derived from NIA-funded studies for the genetics of late-onset Alzheimer's disease be deposited at NIAGADS, another NIA-approved site, or both. NIAGADS, along with other NIA-approved sites, will make these Genetic Data and Associated Phenotypic Data available to qualified investigators in the scientific community for secondary analysis. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=850&screenshot_id=830"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=850&screenshot_id=830" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 31, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>NIH Data Access Policy</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.niagads.org">http://www.niagads.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pennsylvania <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>OS Independent<br>Other OS<br>Other UNIX-like<br>Windows Vista<br>Windows XP</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ASCII, Microsoft Word Document, PDF - Portable Document Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ASCII, Microsoft Word Document, PDF - Portable Document Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nice-sign" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NICE-SIGN">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nice-sign" rel="bookmark" name="nice-sign">NICE-SIGN</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NICE-SIGN is a nice sign of bias field correction (nonuniformity) in medical images. This tool is fast and efficient. Technical details can be found at <a href="http://zheng.vision.googlepages.com/biasCorrection_miccai09_Zheng.pdf" target="_new">http://zheng.vision.googlepages.com/bias...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 24, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/nicesign/">http://www.nitrc.org/projects/nicesign/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nidb---neuroinformatics-database" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NIDB - Neuroinformatics Database">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nidb---neuroinformatics-database" rel="bookmark" name="nidb---neuroinformatics-database">NIDB - Neuroinformatics Database</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database-application" rel="tag">Database Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NIDB is a powerful, easy to install neuroimaging database designed to allow simple importing, searching, and sharing of imaging data. NIDB associates any modality imaging or other binary data (MR, CT, US, EEG, etc) with a subject through any number of imaging studies or projects. Data is kept at your site, controlled by you, to be shared with other sites whenever you want. NIDB also provides automated pipelining with importing of results back into NIDB which can be searched along with imaging meta data. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=646&screenshot_id=562"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=646&screenshot_id=562" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 24, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://github.com/gbook/nidb">http://github.com/gbook/nidb</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Hartford Hospital <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>AFNI BRIK, ANALYZE, ASCII, CIFTI, COR, DICOM, EEG Formats, GIfTI, Interfile, MEG Formats, MGH/MGZ, MINC, Microsoft Word Document, bshort/bfloat, cff (Connectome File Format)</td></tr>
			<tr class="even"><td>Output Formats</td><td>AFNI BRIK, ANALYZE, ASCII, CIFTI, COR, DICOM, EEG Formats, GIfTI, Interfile, MEG Formats, MGH/MGZ, MINC, Microsoft Word Document, bshort/bfloat, cff (Connectome File Format)</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nifti" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NIfTI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nifti" rel="bookmark" name="nifti">NIfTI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-information-resource" rel="tag">Other Information Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Neuroimaging Informatics Technology Initiative (NIfTI) and NIfTI Data Format Working Group, creator and maintainer of the NIfTI-1 and NIfTI-2 data formats. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=26&screenshot_id=80"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=26&screenshot_id=80" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 17, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://nifti.nimh.nih.gov">http://nifti.nimh.nih.gov</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="niftyrec" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NiftyRec">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=niftyrec" rel="bookmark" name="niftyrec">NiftyRec</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The NiftyRec Tomography Toolbox includes reconstruction tools for emission and transmission imaging modalities, including Single Photon Emission Computed Tomography (SPECT), Positron Emission Tomography (PET), cone-beam X-Ray CT and parallel-beam X-Ray CT.<br />
<br />
At the core of NiftyRec are efficient, GPU accelerated, projection, back-projection and iterative reconstruction algorithms. <br />
<br />
The easy to use Matlab and Python interfaces of NiftyRec enable fast prototyping and development of reconstruction algorithms. NiftyRec includes standard iterative reconstruction algorithms such as Maximum Likelihood Expectation Maximisation (MLEM), Ordered Subsets Expectation Maximisation (OSEM) and One Step Late Maximum A Posteriori Expectation Maximisation (OSL-MAPEM), for multiple imaging modalities. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=695&screenshot_id=629"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=695&screenshot_id=629" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 9, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://niftyrec.scienceontheweb.net/">http://niftyrec.scienceontheweb.net/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University College London <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="niftyreg" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NiftyReg">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=niftyreg" rel="bookmark" name="niftyreg">NiftyReg</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=registration" rel="tag">Registration</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=resampling" rel="tag">Resampling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=warping" rel="tag">Warping</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NiftyReg includes tools for global and local image registration.<br />
<br />
The algorithm used for global registration is based on a block matching approach enabling robust registration (outliers rejection). The local registration implementation uses a cubic B-Spline parametrisation (Free-Form Deformation). All registration algorithms are based on symmetric approaches where forward and backward transformations can be optimised concurrently.<br />
<br />
NiftyReg has been implemented for both CPU and GPU (through the use of CUDA). 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=712&screenshot_id=645"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=712&screenshot_id=645" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 2, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://cmiclab.cs.ucl.ac.uk/mmodat/niftyreg">https://cmiclab.cs.ucl.ac.uk/mmodat/niftyreg</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University College London <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1, Nrrd</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1, Nrrd</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nifty-sim" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Nifty Sim">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nifty-sim" rel="bookmark" name="nifty-sim">Nifty Sim</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Nifty Sim is a high-performance nonlinear finite element solver, developed at University College London. A key feature is the option of GPU-based execution, which allows the solver to significantly out-perform equivalent commercial packages. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 1, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/niftysim/">http://www.nitrc.org/projects/niftysim/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Sheffield/University College London <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nighres" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Nighres">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nighres" rel="bookmark" name="nighres">Nighres</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Processing tools for high-resolution neuroimaging </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 22, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/nighres/">http://www.nitrc.org/projects/nighres/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nih-cidi-lung-segmentation-tool" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NIH-CIDI Lung Segmentation Tool">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nih-cidi-lung-segmentation-tool" rel="bookmark" name="nih-cidi-lung-segmentation-tool">NIH-CIDI Lung Segmentation Tool</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volumetric-analysis" rel="tag">Volumetric Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Open-source software for automatic segmentation of lung in CT images.<br />
<br />
The software is an segmentation tool for the segmentation of lung from CT images. The sofware can be run in two modes: (a) fully automatic and (b) semi-automatic with manual seeding by the user. The software also allows the user perform basic filtering operations and manual correction to the segmentation, if needed. The VTK-based rendering implemntation along with option to view in axial, coronal, and saggital provide user with better visualization of the segmented lung. A short video on the working of the software can be viewed here:<br />
<br />
<a href="http://www.youtube.com/watch?v=VD3GQ0O7weE" target="_new">http://www.youtube.com/watch?v=VD3GQ0O7w...</a><br />
<br />
The algorithm driving the software has been tested on a publically available database (LOLA Challenge 11, <a href="http://www.lola11.com" target="_new">http://www.lola11.com</a>). The results based on the overlap score can viewed on the challenge website. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=776&screenshot_id=700"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=776&screenshot_id=700" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 27, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/nihlungseg/">http://www.nitrc.org/projects/nihlungseg/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>National Institutes of Health <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nih-cidi-segmentation-of-pet-images-based-on-affinity-propagation-clustering" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NIH-CIDI Segmentation of PET Images based on Affinity Propagation Clustering">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nih-cidi-segmentation-of-pet-images-based-on-affinity-propagation-clustering" rel="bookmark" name="nih-cidi-segmentation-of-pet-images-based-on-affinity-propagation-clustering">NIH-CIDI Segmentation of PET Images based on Affinity Propagation Clustering</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quantification" rel="tag">Quantification</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Presented here is a MATLAB GUI for segmenting and quantifying PET images with multi-focal and diffuse uptakes. The segmentation algorithm was presented at the 2013 IEEE International Symposium on Biomedical Imaging and IEEE Transactions on Biomedical Engineering (In Press).<br />
<br />
The MATLAB GUI imports a PET image and allows the user to draw region of interests (ROIs) in 2D or 3D to roughly separate the object of interest from the background. Then, the areas are segmented using a PET image segmentation method based on Affinity Propagation clustering to cluster the image intensities into meaningful groups. <br />
<br />
For quantification, the Standardized Uptake Value measurements of the binary or the user defined ROI are SUVmax, SUVmean, and Volume (mm^3) and can be exported into an excel sheet. We believe that there is meaningful information in the secondary groups, not just the highest uptake group which is the current standard.<br />
<br />
Renderings with the functional information overlaid can also be made for visualization. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 26, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/ap_seg_2013_nih/">http://www.nitrc.org/projects/ap_seg_2013_nih/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>National Institutes of Health <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nih-marmoset-brain-atlas" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NIH Marmoset Brain Atlas">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nih-marmoset-brain-atlas" rel="bookmark" name="nih-marmoset-brain-atlas">NIH Marmoset Brain Atlas</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NIH Marmoset Brain Atlas </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 9, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/nih_marmoset/">http://www.nitrc.org/projects/nih_marmoset/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>NIH <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nih-pediatric-mri-data-repository" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NIH Pediatric MRI Data Repository">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nih-pediatric-mri-data-repository" rel="bookmark" name="nih-pediatric-mri-data-repository">NIH Pediatric MRI Data Repository</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=websites" rel="tag">Websites</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The NIH Pediatric MRI Data Repository contains longitudinal structural MRIs, spectroscopy, DTI and correlated clinical/behavioral data from approximately 500 healthy, normally developing children, ages newborn to young adult. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 7, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://pediatricmri.nih.gov">http://pediatricmri.nih.gov</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>DICOM, MINC, MINC2, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>DICOM, MINC, MINC2, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="niistat" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NiiStat">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=niistat" rel="bookmark" name="niistat">NiiStat</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NiiStat is a set of Matlab scripts for analyzing neuroimaging data from clinical populations. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=876&screenshot_id=844"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=876&screenshot_id=844" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 19, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/niistat/">http://www.nitrc.org/projects/niistat/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of South Carolina <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Microsoft<br>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nilearn" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Nilearn">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nilearn" rel="bookmark" name="nilearn">Nilearn</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Nilearn: Machine learning for Neuro-Imaging in Python, is a software package to facilitate the use of statistical learning on NeuroImaging data. <br />
<br />
Namely Nilearn leverages the scikit-learn Python toolbox for multivariate statistics with applications such as predictive modelling, classification, decoding, or connectivity analysis. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=728&screenshot_id=653"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=728&screenshot_id=653" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 11, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://nilearn.github.io">http://nilearn.github.io</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>INRIA <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nimh-data-archive-/-national-database-for-autism-research" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NIMH Data Archive / National Database for Autism Research">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nimh-data-archive-/-national-database-for-autism-research" rel="bookmark" name="nimh-data-archive-/-national-database-for-autism-research">NIMH Data Archive / National Database for Autism Research</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The NIMH Data Archive (NDA) and National Database for Autism Research (NDAR) is a secure research data repository promoting scientific data sharing and collaboration among autism spectrum disorder (ASD) investigators. The goal of the project is to accelerate scientific discovery through data sharing, data harmonization and the reporting of research results. NDAR currently has data from over 25,000 research participants available to qualified investigators.<br />
<br />
This resource also host the data releases for: Adolescent Brain Cognitive Development Study (ABCD), Research Domain Criteria Database (RDoCDb), and National Database for Clinical Trials Related to Mental Illness (NDCT). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 12, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Data License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://ndar.nih.gov/">http://ndar.nih.gov/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>National Institutes of Health <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>AFNI BRIK, ASCII, CIFTI, COR, DICOM, MINC, MINC2, NIfTI-1, Other Format, PDF - Portable Document Format, Philips PAR/REC, TeX, VRML, bshort/bfloat, cff (Connectome File Format)</td></tr>
			<tr class="even"><td>Output Formats</td><td>AFNI BRIK, ASCII, CIFTI, COR, DICOM, MINC, MINC2, NIfTI-1, Other Format, PDF - Portable Document Format, Philips PAR/REC, TeX, VRML, bshort/bfloat, cff (Connectome File Format)</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nipy-community" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NIPY Community">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nipy-community" rel="bookmark" name="nipy-community">NIPY Community</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The purpose of NIPY is to make it easier to do better brain imaging research. We believe that neuroscience ideas and analysis ideas develop together. Good ideas come from understanding; understanding comes from clarity, and clarity must come from well-designed teaching materials and well-designed software. The software must be designed as a natural extension of the underlying ideas.<br />
<br />
We aim to build software that is:<br />
<br />
 * clearly written<br />
 * clearly explained<br />
 * a good fit for the underlying ideas<br />
 * a natural home for collaboration </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 5, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://nipy.org">http://nipy.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nipy-diffusion-imaging-analysis" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NIPY Diffusion Imaging Analysis">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nipy-diffusion-imaging-analysis" rel="bookmark" name="nipy-diffusion-imaging-analysis">NIPY Diffusion Imaging Analysis</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=connectivity-analysis" rel="tag">Connectivity Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=platform-or-development-environment" rel="tag">Platform Or Development Environment</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=rendering" rel="tag">Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tensor-metric" rel="tag">Tensor Metric</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tractography" rel="tag">Tractography</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> DIPY is a free and open source software project for computational neuroanatomy. It focuses on diffusion magnetic resonance imaging (dMRI) analysis and tractography but also contains implementations of other computational imaging methods such as denoising and registration that are applicable to the greater medical imaging and image processing communities. Additionally, DIPY is an international project which brings together scientists across labs and countries to share their state-of-the-art code and expertise in the same codebase, accelerating scientific research in medical imaging.<br />
<br />
Twitter <a href="https://twitter.com/dipymri" target="_new">https://twitter.com/dipymri</a><br />
G+ <a href="http://bit.ly/dipymri" target="_new">http://bit.ly/dipymri</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=376&screenshot_id=932"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=376&screenshot_id=932" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 6, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://dipy.org">http://dipy.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nipy-file-io" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NIPY File IO">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nipy-file-io" rel="bookmark" name="nipy-file-io">NIPY File IO</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NiBabel is a Python package for reading and writing a variety of medical and neuroimaging file formats. This includes: ANALYZE (plain, SPM99, SPM2), NIfTI1, as well as MINC. NiBabel is the successor of PyNIfTI. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 6, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://nipy.org/nibabel/">http://nipy.org/nibabel/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nipy-pipeline-and-interfaces" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NIPY Pipeline and Interfaces">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nipy-pipeline-and-interfaces" rel="bookmark" name="nipy-pipeline-and-interfaces">NIPY Pipeline and Interfaces</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Current neuroimaging software offer users an incredible opportunity to analyze their data in different ways, with different underlying assumptions. However, this has resulted in a heterogeneous collection of specialized applications without transparent interoperability or a uniform operating interface. Nipype, an open-source, community-developed initiative under the umbrella of Nipy, is a Python project that solves these issues by providing a uniform interface to existing neuroimaging software and by facilitating interaction between these packages within a single workflow.<br />
<br />
Nipype provides an environment that encourages interactive exploration of algorithms from different packages (e.g., SPM, FSL), eases the design of workflows within and between packages, and reduces the learning curve necessary to use different packages. Nipype is creating a collaborative platform for neuroimaging software development in a high-level language and addressing limitations of existing pipeline systems. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 1, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://nipy.org/nipype/">http://nipy.org/nipype/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nipy-structural-and-functional-analysis" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NIPY Structural and Functional Analysis">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nipy-structural-and-functional-analysis" rel="bookmark" name="nipy-structural-and-functional-analysis">NIPY Structural and Functional Analysis</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Nipy aims to provide a complete Python environment for the analysis of structural and functional neuroimaging data. It currently has a full system for general linear modeling of functional magnetic resonance imaging (fMRI). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 3, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://nipy.org/nipy">http://nipy.org/nipy</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="niral-utilities" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NIRAL Utilities">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=niral-utilities" rel="bookmark" name="niral-utilities">NIRAL Utilities</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=application" rel="tag">Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NIRAL Utilities are open-source applications developped at UNC-Chapel Hill in the Neuro Image Research and Analysis Lab (NIRAL). These utilities are C++ based command line applications that allow image analysis and processing using ITK or VTK libraries.<br />
<br />
Specifically the following utilities are contained thus far:<br />
ImageMath - the swiss army knife image modification<br />
ImageStat - compute stats on images<br />
IntensityRescaler - rescale/normalize intensities using a prior brain tissue segmentation<br />
convertITKformats - convert 3D images in all ITK formats (NRRD, NIFTI, GIPL, Meta etc)<br />
DWI_NiftiNrrdConversion - convert DWI and DTI from/to NRRD and NIFTI, works with UNC DTI tools and FSL<br />
CropTools - crops 3D and 4D images<br />
PolydataMerge - Merges VTK polydata files<br />
PolydataTransform - Transforms polydata files<br />
TransformDeformationField - concatenates or average deformation fields (H-fields or displacement fields)<br />
DTIAtlasBuilder - Creates a DTI average from multiple DTI images 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=465&screenshot_id=335"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=465&screenshot_id=335" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 17, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/niral_utilities/">http://www.nitrc.org/projects/niral_utilities/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina at Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Microsoft</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, Nrrd, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, Nrrd, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nirep" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NIREP">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nirep" rel="bookmark" name="nirep">NIREP</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> We have started the Non-rigid Image Registration Evaluation Project (NIREP) to develop software tools and provide shared image validation databases for rigorous testing of non-rigid image registration algorithms. NIREP will extend the scope of prior validation projects by developing evaluation criteria and metrics using large image populations, using richly annotated image databases, using computer simulated data, and increasing the number and types of evaluation criteria.<br />
<br />
The goal of this project is to establish, maintain, and endorse a standardized set of relevant benchmarks and metrics for performance evaluation of nonrigid image registration algorithms. Furthermore, these standards will be incorporated into an exportable computer program to automatically evaluate the registration accuracy of nonrigid image registration algorithms. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 12, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/nirep/">http://www.nitrc.org/projects/nirep/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nirfast" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NIRFAST">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nirfast" rel="bookmark" name="nirfast">NIRFAST</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Nirfast is a software package for modeling Near-Infrared light transport in tissue and image reconstruction. This includes: Standard single wavelength absorption and reduced scatter, Multi-wavelength spectrally constrained models and Fluorescence models. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 12, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/nirfast/">http://www.nitrc.org/projects/nirfast/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Dartmouth College USA/University of Birmingham UK <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nirs-spm" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NIRS-SPM">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nirs-spm" rel="bookmark" name="nirs-spm">NIRS-SPM</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NIRS-SPM is a SPM and MATLAB-based software package for statistical analysis of near-infrared spectroscopy (NIRS) signals, developed at the Bio Imaging Signal Processing (BISP) lab. at KAIST in Korea. Based on the general linear model (GLM), and Sun's tube formula / Lipschitz-Killing curvature (LKC) based expected Euler characteristics, NIRS-SPM not only provides activation maps of oxy-, deoxy-, and total-hemoglobin, but also allows for super-resolution activation localization. Additional features, including a wavelet-minimum description length detrending algorithm and cerebral metabolic rate of oxygen (CMRO2) estimation without hypercapnia, were implemented in the NIRS-SPM software package. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 10, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://bisp.kaist.ac.kr/NIRS-SPM">http://bisp.kaist.ac.kr/NIRS-SPM</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Korea Advanced Institute of Science and Technology (KAIST) <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nirx2nirs:-a-nirx-to-.nirs-data-converter" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NIRx2nirs: A NIRx to .nirs data converter">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nirx2nirs:-a-nirx-to-.nirs-data-converter" rel="bookmark" name="nirx2nirs:-a-nirx-to-.nirs-data-converter">NIRx2nirs: A NIRx to .nirs data converter</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=nirs/fnir" rel="tag">Nirs/Fnir</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NIRx2nirs is a matlab script which takes near-infrared spectroscopy data recorded by NIRx system(s) and converts it to a .nirs file format for use with the HOMER2 NIRS processing pacakge. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 29, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/nirx2nirs/">http://www.nitrc.org/projects/nirx2nirs/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UCL <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nirx-nirs-neuroimaging" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NIRx NIRS Neuroimaging">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nirx-nirs-neuroimaging" rel="bookmark" name="nirx-nirs-neuroimaging">NIRx NIRS Neuroimaging</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=nirs/fnir" rel="tag">Nirs/Fnir</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tomography" rel="tag">Tomography</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NIRx Medical Technologies, LLC* is a leader in providing integrated solutions for NIRS neuroimaging. <br />
<br />
*Click on the company logo to see more images. <br />
<br />
We provide custom technology solutions to the investigative community for a wide range of near-infrared spectroscopy imaging applications. While many of our systems are in the field of neuroscience (infants to adults), they are also used for investigation of breast cancer, peripheral vascular disease and the study of small animals.<br />
<br />
Email: <a href="mailto:info@nirx.net" target="_new">info@nirx.net</a><br />
Web: <a href="http://www.nirx.net" target="_new">http://www.nirx.net</a><br />
<br />
NIRx is closely associated with The Optical Tomography Group (OTG) of SUNY: Downstate Medical Center. For more information on The OTG, visit:<br />
<br />
NITRC page: <a href="http://www.nitrc.org/projects/fnirs_downstate" target="_new">http://www.nitrc.org/projects/fnirs_down...</a><br />
Homepage: <a href="http://otg.downstate.edu/publications.htm" target="_new">http://otg.downstate.edu/publications.ht...</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=497&screenshot_id=805"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=497&screenshot_id=805" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 30, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Commerical License:NIRx</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nirx.net">http://www.nirx.net</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>NIRx Medical Technologies, LLC <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nitime" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="nitime">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nitime" rel="bookmark" name="nitime">nitime</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Nitime is a library for time-series analysis of data from neuroscience experiments.<br />
<br />
It contains a core of numerical algorithms for time-series analysis both in the time and spectral domains, a set of container objects to represent time-series, and auxiliary objects that expose a high level interface to the numerical machinery and make common analysis tasks easy to express with compact and semantically clear code. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 6, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://nipy.org/nitime/">http://nipy.org/nitime/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>NIPY <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nitrc-computational-environment-(nitrc-ce)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NITRC Computational Environment (NITRC-CE)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nitrc-computational-environment-(nitrc-ce)" rel="bookmark" name="nitrc-computational-environment-(nitrc-ce)">NITRC Computational Environment (NITRC-CE)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=platform-or-development-environment" rel="tag">Platform Or Development Environment</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NITRC Computational Environment (NITRC-CE) is an on-demand, cloud based computational virtual machine pre-installed with popular neuroimaging tools. We have leveraged NeuroDebian as well as manually added in a variety of popular software tools. NITRC-CE is available to use on your own infrastructure or via commercial cloud providers including Amazon and Microsoft. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=664&screenshot_id=1122"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=664&screenshot_id=1122" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 19, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/nitrc_es/">http://www.nitrc.org/projects/nitrc_es/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Neuroimaging Tools and Resources Collaboratory <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>DICOM, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>DICOM, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nitrc-gforge-extensions" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NITRC GForge Extensions">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nitrc-gforge-extensions" rel="bookmark" name="nitrc-gforge-extensions">NITRC GForge Extensions</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This project contains custom extensions to the GForge collaborative environment used to create NITRC. These features include: MediaWiki integration, ratings and reviews, Nutch Web crawling integration, improved project searching, BBCode extensions, and front and project summary page re-design. See our Release Notes page for specifics: <a href="http://www.nitrc.org/plugins/mwiki/index.php/nitrc:NITRC_Release_Notes" target="_new">http://www.nitrc.org/plugins/mwiki/index...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 6, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/nitrcext/">http://www.nitrc.org/projects/nitrcext/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Turner Consulting Group <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nitrc-image-repository-(nitrc-ir)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NITRC Image Repository (NITRC-IR)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nitrc-image-repository-(nitrc-ir)" rel="bookmark" name="nitrc-image-repository-(nitrc-ir)">NITRC Image Repository (NITRC-IR)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NITRC Image Repository offers a cloud-based federated neuroimaging data storage system for resource sharing of neuroimaging data in DlCOM and NIfTI formats. Available are thousands of subjects and imaging sessions searchable across over a dozen projects to promote re-use and integration of valuable NIH-funded data. <br />
<br />
Search for and freely download publicly available data sets including thousands of DICOM and NIfTI normal subjects and those with diagnoses such as: schizophrenia, ADHD, autism, and Parkinson's disease. From the search results you may select and download images. NITRC's Image Repository leverages XNAT software: Extensible Neuroimaging Archive Toolkit. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1194&screenshot_id=1113"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1194&screenshot_id=1113" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 27, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/nitrcir/">http://www.nitrc.org/projects/nitrcir/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>NeuroImaging Tools and Resources Collaboratory <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS<br>Microsoft<br>OS Independent<br>OS/2<br>Other OS<br>POSIX/UNIX-like</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="noddi-matlab-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NODDI Matlab Toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=noddi-matlab-toolbox" rel="bookmark" name="noddi-matlab-toolbox">NODDI Matlab Toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=diffusion-application" rel="tag">Diffusion Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This MATLAB toolbox implements a data fitting routine for Neurite Orientation Dispersion and Density Imaging (NODDI). NODDI is a new diffusion MRI technique for imaging brain tissue microstructure. Compared to DTI, it has the advantage of providing measures of tissue microstructure that are much more direct and hence more specific. It achieves this by adopting the model-based strategy which relates the signals from diffusion MRI to geometric models of tissue microstructure. In contrast to typical model-based techniques, NODDI is much more clinically feasible and can be acquired on standard MR scanners with an imaging time comparable to DTI. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=716&screenshot_id=783"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=716&screenshot_id=783" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 9, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Artistic License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/noddi_toolbox/">http://www.nitrc.org/projects/noddi_toolbox/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University College London <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nonfractal" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="nonfractal">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nonfractal" rel="bookmark" name="nonfractal">nonfractal</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> It is a MATLAB toolbox for estimating &quot;nonfractal connectivity&quot; and &quot;fractal connectivity&quot; as well as Hurst parameters from a set of time series with long-range dependence such as resting state fMRI BOLD or EEG signals. Please refer to the following publication for the underlying theories.<br />
<br />
Wonsang You, Sophie Achard, Joerg Stadler, Bernd Bruekner, and Udo Seiffert, &quot;Fractal analysis of resting state functional connectivity of the brain,&quot; in 2012 International Joint Conference on Neural Networks, 2012.<br />
<br />
For more details, please visit the wiki page:<br />
<a href="https://www.nitrc.org/plugins/mwiki/index.php/nonfractal:MainPage" target="_new">https://www.nitrc.org/plugins/mwiki/inde...</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1181&screenshot_id=1100"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1181&screenshot_id=1100" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 6, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial No Derivatives</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/nonfractal/">http://www.nitrc.org/projects/nonfractal/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Microsoft</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ASCII</td></tr>
			<tr class="even"><td>Output Formats</td><td>ASCII</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="non-redundant-connectivity-(nrc)-maps" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Non redundant connectivity (NRC) maps">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=non-redundant-connectivity-(nrc)-maps" rel="bookmark" name="non-redundant-connectivity-(nrc)-maps">Non redundant connectivity (NRC) maps</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Non-Redundant Connectivity (NRC) maps are summary maps of functional connectivity levels in the brain, estimated from fMRI datasets. They are similar to weighted Global Brain Connectivity (GBC) maps (Cole et al. 2010). However, NRC is based on estimates of multiple correlations instead of bivariate correlations as in GBC, allowing the exploration of the multivariate nature of brain connectivity. To avoid dimensionality problems it applies the method of Supervised Principal Components (Bair et al. 2006).<br />
<br />
To run a NRC analysis you can use nrcon, a 64 bit linux executable that can be downloaded from <a href="http://www.neuroimagen.es/webs/nrcon/" target="_new">http://www.neuroimagen.es/webs/nrcon/</a><br />
<br />
nrcon also calculates GBC and Degree Centrality (DC) images. A full description of the method is available in:<br />
<br />
Non redundant functional brain connectivity in schizophrenia.<br />
R. Salvador, R. Landín-Romero, M. Anguera, E. J. Canales-Rodríguez, J. Radua, A. Guerrero-Pedraza, S. Sarró, T. Maristany, P.J. McKenna, E. Pomarol-Clotet.<br />
Brain Imaging Behav. 2016 Mar 21 </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 18, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/nrcon/">http://www.nitrc.org/projects/nrcon/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="non-rigid-groupwise-registration-method" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Non-rigid groupwise registration method">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=non-rigid-groupwise-registration-method" rel="bookmark" name="non-rigid-groupwise-registration-method">Non-rigid groupwise registration method</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The goal of the project is to provide an open source implementation of a non-rigid groupwise registration method.<br />
<br />
This project is implemented by<br />
+ Serdar K Balci (serdar at csail.mit.edu)<br />
 and supervised by<br />
+ Polina Golland<br />
+ William M. Wells<br />
<br />
All metrics are implementing in a multi-threaded fashion. The algorithm will run faster on<br />
computers with multiple CPU's. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 27, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/groupwisereg/">http://www.nitrc.org/projects/groupwisereg/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nordicneurolab" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NordicNeuroLab">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nordicneurolab" rel="bookmark" name="nordicneurolab">NordicNeuroLab</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=diffusion-mr-fiber-tracking" rel="tag">Diffusion Mr Fiber Tracking</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=eye-tracking" rel="tag">Eye Tracking</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=perfusion" rel="tag">Perfusion</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=scanner" rel="tag">Scanner</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=stimulus-presentation" rel="tag">Stimulus Presentation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> With over a decade of experience, NordicNeuroLab (NNL) provides products and solutions that define the field of functional MR imaging. We understand the growing need for reliable and innovative tools in this growing field. As a result, we closely collaborate with research and clinical teams from both academic and medical centers, MR system manufacturers, and third party vendors to develop and manufacture hardware and software solutions that meet the needs of very experienced centers while developing training programs to make fMRI easy to adopt for more novice users.<br />
<br />
From state of the art post-processing and visualization software for BOLD, Diffusion/DTI, and Perfusion/DCE imaging to fMRI hardware for audio and visual stimulation, eye tracking, and patient response collection, NNL's products are used around the world by researchers and clinicians alike. Ultimately, we are dedicated to bringing the most advanced neuro-imaging tools to market while making functional MRI programs easy to implement. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=498&screenshot_id=375"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=498&screenshot_id=375" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 30, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Commercial: NordicNeuroLab</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nordicneurolab.com">http://www.nordicneurolab.com</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Windows Vista<br>Windows XP</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, ASCII, DICOM, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, ASCII, DICOM, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="normative-independent-component-analysis" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Normative Independent Component Analysis">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=normative-independent-component-analysis" rel="bookmark" name="normative-independent-component-analysis">Normative Independent Component Analysis</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=fourier-time-domain-analysis" rel="tag">Fourier Time-Domain Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=independent-component-analysis" rel="tag">Independent Component Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=spectral-analysis" rel="tag">Spectral Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This program, executable under any Windows32 OS, performs Group BSS (Blind Source Separation) analysis comparing two groups of individuals and it performs NICA (Normative ICA) analysis where individuals are compared individually to a (normative) group. All analysis is performed in the frequency domain, that is, for all frequencies. The program also performs all these analysis for qEEG, that is, at the electrode level, without any BSS. The program does all computations, saves and displays results.<br />
The rationale and methods used in this program are explained in all details in the following paper:<br />
<br />
Congedo M, John ER, De Ridder D, Prichep L (2010)<br />
Group Independent Component Analysis of Resting-State EEG in Large Normative Samples International Journal of Psychophysiology 78, 89-99. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 21, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://sites.google.com/site/marcocongedo/software/nica">http://sites.google.com/site/marcocongedo/software/nica</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>GIPSA-lab, Grenoble, FRANCE <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nornir" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Nornir">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nornir" rel="bookmark" name="nornir">Nornir</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-to-image" rel="tag">Image-To-Image</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Nornir’s takes large sets of overlapping images in 2D and produces registered (a.k.a. aligned) 2D mosaics and 3D volumes of any size and scale. Registered slices may be exported as a single large images or viewed/annoted with our Viking viewer.<br />
<br />
Nornir has been used successfully on transmission electron microscopy, scanning electron microscopy images, and light microscopy images. Nornir supports interleaving different imaging methods into the same volume. Support for SerialEM, Objective Imaging, and Digital Micrograph (DM4) raw data is available. Adding formats is not complicated and the author will consider requests.<br />
<br />
Nornir runs on fairly humble hardware for the task. A 32-core 64GB Xeon system built a ~60 TB 250um diameter 2.12nm/pixel volume from roughly 1400 slices. Nornir works incrementally, only updating data that has changed.<br />
<br />
Installation is fairly simple and primarily uses Python's PIP installer.<br />
 <br />
For further information: <a href="http://nornir.github.io/" target="_new">http://nornir.github.io/</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1307&screenshot_id=1223"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1307&screenshot_id=1223" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 20, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://nornir.github.io/">http://nornir.github.io/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Utah <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="northwestern-university-schizophrenia-data-and-software-tool-(nusdast)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Northwestern University Schizophrenia Data and Software Tool (NUSDAST)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=northwestern-university-schizophrenia-data-and-software-tool-(nusdast)" rel="bookmark" name="northwestern-university-schizophrenia-data-and-software-tool-(nusdast)">Northwestern University Schizophrenia Data and Software Tool (NUSDAST)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=federal-funding-resource" rel="tag">Federal Funding Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Northwestern University Schizophrenia Data and Software Tool (NUSDAST) is a repository of schizophrenia neuroimaging data collected from over 450 individuals with schizophrenia, healthy controls and their respective siblings, most with 2-year longitudinal follow-up. <br />
The data include:<br />
Neuroimaging data: Structural MR scans, landmarks and surface maps, and FreeSurfer parcellation and measurement)<br />
Cognitive data: domain scores for crystallized intelligence, working memory, episodic memory, and executive function<br />
Clinical data: demographic, sibling relationship, SAPS and SANS psychopathology<br />
Genetic data: 20 SNPs<br />
A neuroimaging mapping, analysis and visualization software tool, CAWorks, is also part of this resource. We have also established a web-based information retrieval portal that allows the user to efficiently search the collection. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 18, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Northwestern University Schizophrenia Data and Software Tool (NUSDAST)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://niacal.northwestern.edu/projects/9">http://niacal.northwestern.edu/projects/9</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Northwestern University Feinberg School of Medicine <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="notion-researchpacs" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Notion ResearchPACS">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=notion-researchpacs" rel="bookmark" name="notion-researchpacs">Notion ResearchPACS</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=application" rel="tag">Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Notion PACS<br />
===========<br />
<br />
Notion is a stand-alone [PACS](<a href="http://en.wikipedia.org/wiki/Picture_archiving_and_communication_system" target="_new">http://en.wikipedia.org/wiki/Picture_arc...</a>) designed to be used by radiology researchers for storage and anonymization of research images. <br />
<br />
### Why use Notion?<br />
<br />
If you have a need to:<br />
- store DICOM images, but do not want/have a dedicated research PACS<br />
- anonymize DICOM images<br />
- map Names, IDs and Accession numbers during anonymization<br />
- maintain separation of image data across projects<br />
- scale to 100's of independant research projects<br />
<br />
### Why *not* use Notion?<br />
<br />
If you:<br />
- already have a research PACS<br />
- do not need to anonymize data<br />
- do not care about isolation between research projects<br />
- are happy using manual anonymization tools 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=793&screenshot_id=721"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=793&screenshot_id=721" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 31, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/notion/">http://www.nitrc.org/projects/notion/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Mayo Clinic <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>DICOM</td></tr>
			<tr class="even"><td>Output Formats</td><td>DICOM</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="npbayes-fmri" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NPBayes-fMRI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=npbayes-fmri" rel="bookmark" name="npbayes-fmri">NPBayes-fMRI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=software" rel="tag">Software</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This is a user-friendly matlab GUI that implements a spatio-temporal nonparametric Bayesian linear regression modeling approach for the analysis of task-related fMRI data from multi-subject experiments. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1229&screenshot_id=1149"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1229&screenshot_id=1149" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 31, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://github.com/marinavannucci/NPBayes_fMRI">https://github.com/marinavannucci/NPBayes_fMRI</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Rice University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nptk" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NPTK">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nptk" rel="bookmark" name="nptk">NPTK</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=artifact-removal" rel="tag">Artifact Removal</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=intermodal" rel="tag">Intermodal</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=nonlinear-warp" rel="tag">Nonlinear Warp</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NPTK provides non-rigid registration/distortion correction tools for enhanced functional localization through the registration of EPI fMRI to high-resolution anatomical MRI. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 2, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/nptk/">http://www.nitrc.org/projects/nptk/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows XP</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nrm2018-pet-grand-challenge-dataset" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NRM2018 PET Grand Challenge Dataset">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nrm2018-pet-grand-challenge-dataset" rel="bookmark" name="nrm2018-pet-grand-challenge-dataset">NRM2018 PET Grand Challenge Dataset</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> For many years PET centres around the world have developed and optimised their own analysis pipelines, including a mixture of in-house and independent software, and have implemented different modelling choices for PET image processing and data quantification. As a result, many different methods and tools are available for PET image analysis.<br />
<br />
This dataset aims to provide a normative tool to assess the performance and consistency of PET modelling approaches on the same data for which the ground truth is known. <br />
It was created and released for the NRM2018 PET Grand Challenge. The challenge aimed at evaluating the performances of different PET analysis tools to identify areas and magnitude of receptor binding changes in a PET radioligand neurotransmission study. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1336&screenshot_id=1248"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1336&screenshot_id=1248" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 22, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution-NonCommercial-NoDerivatives 4.0 International (CC BY-NC-ND 4.0)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/nrm2018_petgc/">http://www.nitrc.org/projects/nrm2018_petgc/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>King's College London <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="ntu-dsi-122:-a-diffusion-spectrum-imaging-template-with-high-anatomical-matching-to-the-icbm-152-space" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NTU-DSI-122: A Diffusion Spectrum Imaging Template with High Anatomical Matching to the ICBM-152 Space">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=ntu-dsi-122:-a-diffusion-spectrum-imaging-template-with-high-anatomical-matching-to-the-icbm-152-space" rel="bookmark" name="ntu-dsi-122:-a-diffusion-spectrum-imaging-template-with-high-anatomical-matching-to-the-icbm-152-space">NTU-DSI-122: A Diffusion Spectrum Imaging Template with High Anatomical Matching to the ICBM-152 Space</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-data" rel="tag">Atlas Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=diffusion-spectrum" rel="tag">Diffusion Spectrum</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NTU-DSI-122 is a diffusion spectrum imaging (DSI) template constructed in the standard ICBM-152 space from 122 healthy adults. This template was built through incorporating the macroscopic anatomical information using high-resolution T1-weighted images and the microscopic structural information obtained from DSI datasets, rendering it to achieve a high anatomical matching to the ICBM-152 space. Therefore, this template can serve as a representative DSI dataset for a healthy adult population, and will be of potential value for brain research and clinical applications. This template is released in the original DWI format, so the users have the most freedom to perform their own advanced processing algorithms on NTU-DSI-122. NTU-DSI-122 is distributed under the terms of CC BY-NC-SA 4.0.<br />
<br />
Reference: Hsu Y-C, Lo Y-C, Chen Y-J, Wedeen VJ, Tseng W-YI (2015): NTU-DSI-122: A diffusion spectrum imaging template with high anatomical matching to the ICBM-152 space. Hum Brain Mapp doi: 10.1002/hbm.22860. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=784&screenshot_id=933"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=784&screenshot_id=933" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 25, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial Share Alike</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/ntu-dsi-122/">http://www.nitrc.org/projects/ntu-dsi-122/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>National Taiwan University College of Medicine <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="numerical-fibre-generator-(nfg)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Numerical Fibre Generator (NFG)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=numerical-fibre-generator-(nfg)" rel="bookmark" name="numerical-fibre-generator-(nfg)">Numerical Fibre Generator (NFG)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tractography" rel="tag">Tractography</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NFG is a collection of tools that generate numerical fiber structures with the complexity of human white matter and simulate Diffusion-Weighted MR images that would arise from them. Its primary use is to enable the testing of tracking algorithms 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=132&screenshot_id=48"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=132&screenshot_id=48" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 29, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.brain.org.au/software/">http://www.brain.org.au/software/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS<br>POSIX/UNIX-like<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nutil---neuroimaging-utilities" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Nutil - Neuroimaging utilities">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nutil---neuroimaging-utilities" rel="bookmark" name="nutil---neuroimaging-utilities">Nutil - Neuroimaging utilities</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Nutil aims to both simplify and streamline the mechanism of pre-and-post processing 2D brain image data. Nutil is developed as a stand-alone application that runs on all operating systems with a minimalistic user interface requiring little-to-no experience to execute. The user specifies the entire pipeline of pre-and post processing operations through a well-documented Excel template, and then feeds this document through Nutil. Pre-processing operations include conversion of images from JPEG/PNG format to tiled TIFF format, 2D transformations of extremely large tiled TIFF files (rotation, flipping and scaling), in addition to renaming, copying and downsizing. Post-processing is based on analysis of segmented images in the context of brain regions defined by a 3D reference atlase, such as the Allen Mouse Brain reference atlas or the Waxholm Space Atlas of the Sprague Dawley rat brain. All functions operate in batch mode, and operate in parallel on multiple CPUs. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 24</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/nutil/">http://www.nitrc.org/projects/nutil/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Oslo <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nutmeg" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NUTMEG">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nutmeg" rel="bookmark" name="nutmeg">NUTMEG</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=meg-modeling" rel="tag">Meg Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=multivariate-analysis" rel="tag">Multivariate Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=principal-component-analysis" rel="tag">Principal Component Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-rendering" rel="tag">Surface Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=three-dimensional-display" rel="tag">Three Dimensional Display</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NUTMEG (Neurodynamic Utility Toolbox for Magnetoencephalography) is an open-source MATLAB toolbox for reconstructing the spatiotemporal dynamics of neural activations and overlaying them onto structural MR images. More information can be found at <a href="http://www.nitrc.org/projects/nutmeg/" target="_new">http://www.nitrc.org/projects/nutmeg/</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=111&screenshot_id=389"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=111&screenshot_id=389" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 17, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/nutmeg/">http://www.nitrc.org/projects/nutmeg/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nvm" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NVM">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nvm" rel="bookmark" name="nvm">NVM</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=anatomic" rel="tag">Anatomic</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=morphology" rel="tag">Morphology</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volumetric-analysis" rel="tag">Volumetric Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volumetric-analysis" rel="tag">Volumetric Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NVM is an open-source software tool for locating and measuring neuroanatomy in structural volumetric image data. It is used to draw regions of interest for subsequent fMRI analysis. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 28, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>NVM License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://neuromorphometrics.com">http://neuromorphometrics.com</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, COR, DICOM, NIfTI-1, Other Format, bshort/bfloat</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, COR, DICOM, NIfTI-1, Other Format, bshort/bfloat</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="nyu-csc-testretest" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="NYU CSC TestRetest">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=nyu-csc-testretest" rel="bookmark" name="nyu-csc-testretest">NYU CSC TestRetest</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=test-data" rel="tag">Test Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The NYU CSC TestRetest resource includes EPI-images of 25 participants gathered during rest as well as anonymized anatomical images of the same participants. <br />
<br />
The resting-state fMRI images were collected on several occasions: <br />
1. the first resting-state scan in a scan session<br />
2. 5-11 months after the first resting-state scan<br />
3. about 30 (&lt; 45) minutes after 2.<br />
<br />
Each scan occasion is released as a new version release of the resource. Make sure to check all releases to get all files.<br />
<br />
Data use is unrestricted, but users should reference our original publications and NITRC (see Documentations).<br />
<br />
***Caution: Participants here are part of the NewYork_a contribution to the 1000 Functional Connectomes Project. DO NOT combine datasets.<br />
<br />
Acquisition was funded by Stavros S. Niarchos Foundation, the Leon Lowenstein Foundation, NARSAD (The Mental Health Research Association) grants to F.Xavier Castellanos; and Linda and Richard Schaps, Jill and Bob Smith, and the Taubman Foundation gifts to F.Xavier Castellanos. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 18, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/nyu_trt/">http://www.nitrc.org/projects/nyu_trt/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="oasis" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="OASIS">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=oasis" rel="bookmark" name="oasis">OASIS</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Open Access Series of Imaging Studies (OASIS) is a project aimed at making MRI data sets of the brain freely available to the scientific community. By compiling and freely distributing MRI data sets, we hope to facilitate future discoveries in basic and clinical neuroscience. OASIS is made available by the Washington University Alzheimer’s Disease Research Center, Dr. Randy Buckner at the Howard Hughes Medical Institute (HHMI) at Harvard University, the Neuroinformatics Research Group (NRG) at Washington University School of Medicine, and the Biomedical Informatics Research Network (BIRN). OASIS currently includes two data sets. A cross-sectional set includes 416 subjects aged 18 to 96, 100 of whom were clinically diagnosed with Alzheimer's disease. A longitudinal release includes 150 subjects aged 60 to 96, many diagnosed with Alzheimer's disease at some point during their participation. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 7, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>OASIS</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.oasis-brains.org/">http://www.oasis-brains.org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>DICOM</td></tr>
			<tr class="even"><td>Output Formats</td><td>DICOM</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="odin" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ODIN">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=odin" rel="bookmark" name="odin">ODIN</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-display" rel="tag">Image Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-reconstruction" rel="tag">Image Reconstruction</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=simulation" rel="tag">Simulation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ODIN is a C++ software framework to develop, simulate and run magnetic resonance sequences on different platforms. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 31, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://od1n.sourceforge.net">http://od1n.sourceforge.net</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="oei-fmri-compatible-olfactometer" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="OEI fMRI compatible olfactometer">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=oei-fmri-compatible-olfactometer" rel="bookmark" name="oei-fmri-compatible-olfactometer">OEI fMRI compatible olfactometer</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=stimulus-presentation" rel="tag">Stimulus Presentation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The OEI OLFACT-fMRI olfactometer is a computerized, odor delivery device that can be used for basic research applications including mapping olfactory centers, cognitive/learning research, neuro-marketing among other uses. Additional products including tests for odor threshold, odor identification, odor discrimination and odor memory. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 29, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Commercial License: OEI</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.osmicenterprises.com">http://www.osmicenterprises.com</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Osmic Enterprises, Inc. <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="offline-processing-mri" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Offline Processing MRI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=offline-processing-mri" rel="bookmark" name="offline-processing-mri">Offline Processing MRI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other" rel="tag">Other</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This is structural and functional MRI data from 35 healthy volunteers that accompanies Bursley et al. (2015), &quot;Awake, Offline Processing During Associative Learning.&quot; Subjects encoded paired associates and then performed a distractor task before being probed on associate pairs. Pattern analyses suggest that encoded memories were reactivated during the distractor task, and performance of the distractor task led to superior recall for the associate pairs, compared to a control condition in which no distractor task was performed. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 29, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/offprocmri/">http://www.nitrc.org/projects/offprocmri/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Carnegie Mellon University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="ogles---opengl/oiv-stereotactic-tool" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Ogles - OpenGL/OIV Stereotactic Tool">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=ogles---opengl/oiv-stereotactic-tool" rel="bookmark" name="ogles---opengl/oiv-stereotactic-tool">Ogles - OpenGL/OIV Stereotactic Tool</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Ogles2 is an interactive slice and volume visualization and analysis tool based on Open Inventor/Coin3D. Ogles2 allows for reproducing the work flow of frame based stereotactic neurosurgery.<br />
Ogles2 main focus is to combine slice derived volume data with surface meshes in the context of stereotactic neurosurgery. It allows for transformation of 3D brain atlas data to individual brain configurations and to estimate the effects of electrical fields of deep brain stimulation (DBS) electrodes. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 9, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://ogles.sourceforge.net/Ogles/ogles2-doc/index.html">http://ogles.sourceforge.net/Ogles/ogles2-doc/index.html</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Functional Neurosurgery Group <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>Windows XP</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="online-brain-atlas-reconciliation-tool" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Online Brain Atlas Reconciliation Tool">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=online-brain-atlas-reconciliation-tool" rel="bookmark" name="online-brain-atlas-reconciliation-tool">Online Brain Atlas Reconciliation Tool</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Online Brain Atlas Reconciliation Tool (OBART) aims to provide a quantitative solution to the so-called neuroanatomical nomenclature problem by comparing overlap relations between regions defined as spatial entities in different MRI-based human brain atlases. <br />
<br />
The tool currently allows users to: <br />
<br />
1. Explore the regions defined by individual atlases, choose individual regions to view in 2D projection images, and list other similar regions and the degree to which they overlap the chosen region<br />
<br />
2. Compare two atlases using a graph based visualization to understand &quot;higher-order&quot; neuroanatomical correspondences<br />
<br />
3. Simultaneously their own binary mask image based on ROIs from multiple atlases. This requires the user to upload an image registered to MNI-305 space, and returns a table of overlapping ROIs. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 1, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/obart/">http://www.nitrc.org/projects/obart/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Boston University and Cold Spring Harbor Laboratory <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="online-eeg-artifact-removal-by-adaptive-spatial-filtering" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Online EEG artifact removal by adaptive spatial filtering">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=online-eeg-artifact-removal-by-adaptive-spatial-filtering" rel="bookmark" name="online-eeg-artifact-removal-by-adaptive-spatial-filtering">Online EEG artifact removal by adaptive spatial filtering</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The performance of electroencephalography (EEG) data depends on the effective attenuation of artifacts that are mixed in the recordings. To address this problem, we have developed a novel online EEG artifact removal method for online applications, which combines Independent Component Analysis (ICA) and regression (REG) analysis. ICA-REG method relies on the availability of a calibration dataset of limited duration for the initialization of a spatial filter using ICA. Online artifact removal is implemented by dynamically adjusting the spatial filter in the actual experiment, based on linear regression.<br />
<br />
This software can be used to do a pseudo-online artifact removal (necessary for the validations in your project), or a real-time filtering, or to filter the whole signal offline.<br />
<br />
For support: <a href="mailto:roberto.guarnieri@kuleuven.be" target="_new">roberto.guarnieri@kuleuven.be</a><br />
<br />
To cite this software: <a href="http://iopscience.iop.org/article/10.1088/1741-2552/aacfdf" target="_new">http://iopscience.iop.org/article/10.108...</a><br />
Roberto Guarnieri et al 2018 J. Neural Eng. 15 056009 </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 28, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/ica_reg/">http://www.nitrc.org/projects/ica_reg/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>KU Leuven <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="onprc-infant-macaque-brain-atlas" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ONPRC Infant Macaque Brain Atlas">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=onprc-infant-macaque-brain-atlas" rel="bookmark" name="onprc-infant-macaque-brain-atlas">ONPRC Infant Macaque Brain Atlas</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This is a multi-modal (T1w and DTI) brain template of monkeys with atlas labels of a cohort of 6 rhesus macaques scanned longitudinally at ages 2, 4 and 6 months of age. All T1w and DTI templates are with the same coordinates. The reference publication is &quot;The effects of breastfeeding versus formula-feeding on cerebral cortex maturation in infant rhesus macaques&quot;, Neuroimage, 2018, in press (<a href="https://doi.org/10.1016/j.neuroimage.2018.09.015" target="_new">https://doi.org/10.1016/j.neuroimage.201...</a>). 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1315&screenshot_id=1227"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1315&screenshot_id=1227" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 7, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/brainscan_2017/">http://www.nitrc.org/projects/brainscan_2017/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Oregon Health and Science University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="open-connectome-project" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Open Connectome Project">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=open-connectome-project" rel="bookmark" name="open-connectome-project">Open Connectome Project</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Open Connectome Project (OCP: openconnectomeproject.org) provides access to high resolution neuroanatomical images that can be used to explore connectomes. We also provide programmatic access to this data for human and machine annotation, with a long-term goal of reconstructing the neural circuits comprising an entire brain. This project aims to bring the most state-of-the-art scientific data in the world to the hands of anybody with internet access, so collectively, we can begin to unravel connectomes. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 31, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/ocp/">http://www.nitrc.org/projects/ocp/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="openelectrophy" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="OpenElectrophy">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=openelectrophy" rel="bookmark" name="openelectrophy">OpenElectrophy</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The OpenElectrophy project aims to simplify data- and analysis-sharing for intra- and extra-cellular recordings. With OpenElectrophy you will be able to play with neural signals, spikes and oscillations.<br />
<br />
OpenElectrophy is a Python module for electrophysiology data analysis (intra- and extra-cellular). OpenElectrophy is built on top of Neo :<br />
<br />
- It includes the powerful Neo IO that can read a large number of data formats (Plexon, NeuroExplorer, Spike2, TDT, Axon, BlackRock, ...)<br />
- Neo objects ready for analyses (AnalogSIgnal, SpikeTrain, RecordingChannel, Segment, Block ...)<br />
<br />
But OpenElectrophy also provides:<br />
<br />
- A GUI for exploring datasets<br />
- A complete offline spikesorting tool chain = GUI and/or command line.<br />
- A timefrequency toolbox = fast wavelet scalogram plotting + transient oscillation in LFP detection.<br />
- Viewers for Neo objects.<br />
- A database for storage. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=605&screenshot_id=746"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=605&screenshot_id=746" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 21, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GPL-Style Open Source</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://neuralensemble.org/OpenElectrophy/">http://neuralensemble.org/OpenElectrophy/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>CNRS <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="openfmri" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="OpenFMRI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=openfmri" rel="bookmark" name="openfmri">OpenFMRI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> OpenFMRI is a data sharing resource for task-based fMRI data. It aims to provide a resource for any fMRI researcher who wishes to share their data openly. Currently it provides access to raw fMRI datasets along with associated metadata. In the future, it will also provide access to processed datasets. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 15, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Open Data Commons Public Domain Dedication and License 1.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://openfmri.org/">https://openfmri.org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Texas at Austin <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="openmeeg" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="OpenMEEG">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=openmeeg" rel="bookmark" name="openmeeg">OpenMEEG</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=eeg-modeling" rel="tag">Eeg Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=meg-modeling" rel="tag">Meg Modeling</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> A C++ package for low-frequency bio-electromagnetism <br />
solving forward problems in the field of EEG and MEG with very high accuracy. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=514&screenshot_id=496"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=514&screenshot_id=496" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 25, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>CeCill v2</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://openmeeg.gforge.inria.fr">http://openmeeg.gforge.inria.fr</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>INRIA <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS<br>POSIX/UNIX-like<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="open-neuroimaging-laboratory" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Open Neuroimaging Laboratory">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=open-neuroimaging-laboratory" rel="bookmark" name="open-neuroimaging-laboratory">Open Neuroimaging Laboratory</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=websites" rel="tag">Websites</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> OpenNeuroLab (<a href="http://openneu.ro" target="_new">http://openneu.ro</a>) is a web framework for collaborative applications for neuroimaging. So far, we host BrainBox, MetaSearch, Brainspell and MindControl. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 16, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://openneu.ro/">http://openneu.ro/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Instut Pasteur <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="open-science-framework" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Open Science Framework">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=open-science-framework" rel="bookmark" name="open-science-framework">Open Science Framework</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The OSF supports the scientific workflow. Create projects and use them to organize and archive research materials and data. Share those materials with just collaborators, or make them available publicly. Get credit for making materials available when others view, download, use, or extend them. Register projects to create frozen versions to mark the state of a project at a particular point in its history - onset of data collection, at manuscript submission, final version for publication. OSF is part network of research materials, part version control system, and part collaboration software. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 26, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://osf.io">http://osf.io</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Center for Open Science <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="open-vanderbilt-archive-of-the-temporal-lobe" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Open Vanderbilt Archive of the Temporal Lobe">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=open-vanderbilt-archive-of-the-temporal-lobe" rel="bookmark" name="open-vanderbilt-archive-of-the-temporal-lobe">Open Vanderbilt Archive of the Temporal Lobe</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Open Vanderbilt Archive of the Temporal Lobe is a tool for automated segmentation of the temporal lobe. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 20, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/oval/">http://www.nitrc.org/projects/oval/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Vanderbilt University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="openvibe" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="OpenViBE">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=openvibe" rel="bookmark" name="openvibe">OpenViBE</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=platform-or-development-environment" rel="tag">Platform Or Development Environment</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> OpenViBE is a software platform dedicated to designing, testing and using brain-computer interfaces (BCI). The package includes an Acquisition Server application to stream data from various hardware devices, and a Designer tool to create and run real-time signal-processing chains assembled with a graphical design language. OpenViBE package includes several BCI demo applications (based on e.g. P300, SSVEP, Motor Imagery, etc) which are ready for use. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=811&screenshot_id=750"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=811&screenshot_id=750" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 9, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Affero General Public License v3</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://openvibe.inria.fr">http://openvibe.inria.fr</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Inria <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>EDF/BDF/GDF</td></tr>
			<tr class="even"><td>Output Formats</td><td>EDF/BDF/GDF</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="openwalnut" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="OpenWalnut">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=openwalnut" rel="bookmark" name="openwalnut">OpenWalnut</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> OpenWalnut is an open source tool for multi-modal medical and brain data visualization. Its universality allows it to be easily extended and used in a large variety of application cases. It is both, a tool for the scientific user and a powerful framework for the visualization researcher. OpenWalnut is licensed under the terms of the GNU Lesser General Public License version 3. Written entirely in Standard C++ and using a number of portable libraries (e.g. Qt, Boost and OpenSceneGraph) it runs on common GNU/Linux operating systems, Mac OSX and Windows. For more information, including system requirements, see OpenWalnut. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=826&screenshot_id=777"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=826&screenshot_id=777" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 6, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Lesser General Public License (LGPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/openwalnut/">http://www.nitrc.org/projects/openwalnut/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Imaging and Signal Processing Group <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="optil.io" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Optil.io">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=optil.io" rel="bookmark" name="optil.io">Optil.io</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> OPTIL.io (<a href="http://www.optil.io" target="_new">http://www.optil.io</a>) is the online platform that can be used to organize programming challenges. It is dedicated to optimization challenges. However, it can be adapted to other problem types. Optil.io receives algorithmic solutions of optimization problems in the form of source code from the crowd of contestants, compiles it, executes in a homogeneous run-time environment and objectively evaluates using the set of test cases. Problems that are solved can be provided by external scientists or companies. Solutions can be submitted in many programming languages, and they are then carefully assessed taking into account resource limitations set for the run-time environment, such as CPU time or virtual memory. Each participant can continuously observe how his solution is being compared to others and improve it further. The platform is free for scientific and educational, non-commercial use. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 26, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/optilio/">http://www.nitrc.org/projects/optilio/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Poznan University of Technology <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="optseq---fmri-event-scheduler" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Optseq - fMRI Event Scheduler">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=optseq---fmri-event-scheduler" rel="bookmark" name="optseq---fmri-event-scheduler">Optseq - fMRI Event Scheduler</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=experimental-control" rel="tag">Experimental Control</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Optseq is a tool for automatically scheduling events for rapid-presentation event-related (RPER) fMRI experiments (the schedule is the order and timing of events). Events in RPER are presented closely enough in time that their hemodynamic responses will overlap. This requires that the onset times of the events be jittered in order to remove the overlap from the estimate of the hemodynamic response. RPER is highly resistant to habituation, expectation, and set because the subject does not know when the next stimulus will appear or which stimulus type it will be. RPER is also more efficient than fixed-interval event related (FIER) because more stimuli can be presented within a given scanning interval at the cost of assuming that the overlap in the hemodynamic responses will be linear. In SPM parlance, RPER is referred to as 'stochastic design'.<br />
<br />
This is a subproject of the Center for Functional Neuroimaging Techniques (<a href="http://www.nitrc.org/projects/cfnt/" target="_new">http://www.nitrc.org/projects/cfnt/</a>) </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 18, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://surfer.nmr.mgh.harvard.edu/optseq/">http://surfer.nmr.mgh.harvard.edu/optseq/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Other UNIX-like<br>Windows NT/2000</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="organization-for-human-brain-mapping" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Organization for Human Brain Mapping">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=organization-for-human-brain-mapping" rel="bookmark" name="organization-for-human-brain-mapping">Organization for Human Brain Mapping</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=community" rel="tag">Community</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-information-resource" rel="tag">Other Information Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Organization for Human Brain Mapping (OHBM) was created in 1995 and has since evolved in response to the explosion in the field of human functional neuroimaging and its movement into the scientific mainstream.<br />
<br />
One of the primary functions of the organization is to provide an educational forum for the exchange of up-to-the-minute and groundbreaking research across modalities exploring Human Brain Mapping. It does this through a growing membership and an annual conference, held in different locations throughout the world. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1109&screenshot_id=1063"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1109&screenshot_id=1063" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 24, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.humanbrainmapping.org">http://www.humanbrainmapping.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Organization for Human Brain Mapping <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="orion-i" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ORION I">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=orion-i" rel="bookmark" name="orion-i">ORION I</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ORION I is a software designed for automatic 3D morphological reconstruction of neurons. The input of a software is a 3D image stack (RAW format) representing the neuron or a sequence of 3D image stacks with the provide translation parameters to align the stacks. The output is the 3D morphological reconstruction of the neuron represented in a 3D acyclic graph as the standard SWC format. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 14, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/orion_1/">http://www.nitrc.org/projects/orion_1/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Houston <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="ors-visual-si" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ORS Visual SI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=ors-visual-si" rel="bookmark" name="ors-visual-si">ORS Visual SI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=active-contour" rel="tag">Active Contour</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=animation" rel="tag">Animation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=clipping" rel="tag">Clipping</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-reconstruction" rel="tag">Image Reconstruction</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quantitative-shape-analysis" rel="tag">Quantitative Shape Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=region-growing" rel="tag">Region Growing</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=region-of-interest" rel="tag">Region Of Interest</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=shape-analysis" rel="tag">Shape Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-analysis" rel="tag">Surface Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-rendering" rel="tag">Surface Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=three-dimensional-display" rel="tag">Three Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=two-dimensional-display" rel="tag">Two Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=vitual-endoscopy" rel="tag">Vitual Endoscopy</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volume-measurement" rel="tag">Volume Measurement</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volume-rendering" rel="tag">Volume Rendering</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ORS Visual SI's advanced visualization techniques and state-of-the-art volume rendering provide unparalleled insight into the details and properties of neurological data acquired by CT, micro-CT, MRI, PET, SPECT, microscopy and other modalities. With data fusion tools, intramodality and multimodality registration of MR/CT or PET/CT is easily accomplished, while semi-automatic VOI delineation on fused datasets can improve analysis. <br />
<br />
Standard formats, such as DICOM, RAW, JPEG, NIFTI, Analyze are supported and 3D/4D sequences can be played. Other features include MPR, oblique, CPR, volume clipping, and surface visualization of cortex, skull, and scalp models. Also standard are easy-to-use tools for voxel-based delineation of features and the measurement of properties, including areas, volumes, counts, and intensity profiles. Present your findings by creating annotated animations or high-resolution images for posters.<br />
<br />
An SDK is also available to create plug-ins that provide new workflows or functionalities. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 24, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>ORS Visual SI License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/orsvisual_si/">http://www.nitrc.org/projects/orsvisual_si/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Object Research Systems (ORS) Inc. <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="p-[(123)i]iodo-l-phenylalanine-(ipa)-spect-template-in-mni-coordinates" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="p-[(123)I]iodo-L-phenylalanine (IPA)-SPECT template in MNI-coordinates">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=p-[(123)i]iodo-l-phenylalanine-(ipa)-spect-template-in-mni-coordinates" rel="bookmark" name="p-[(123)i]iodo-l-phenylalanine-(ipa)-spect-template-in-mni-coordinates">p-[(123)I]iodo-L-phenylalanine (IPA)-SPECT template in MNI-coordinates</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> p-[(123)I]iodo-L-phenylalanine( IPA)-SPECT template in MNI-coordinates obtained from 12 human subjects </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 20, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/ipa-spect_templ/">http://www.nitrc.org/projects/ipa-spect_templ/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="palm---permutation-analysis-of-linear-models" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="PALM - Permutation Analysis of Linear Models">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=palm---permutation-analysis-of-linear-models" rel="bookmark" name="palm---permutation-analysis-of-linear-models">PALM - Permutation Analysis of Linear Models</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> PALM — Permutation Analysis of Linear Models — is a tool that allows inference using permutation methods, offering a number of features not available in other analysis software. For complete details, including documentation and User Guide, please visit: <a href="http://fsl.fmrib.ox.ac.uk/fsl/fslwiki/PALM" target="_new">http://fsl.fmrib.ox.ac.uk/fsl/fslwiki/PA...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 5, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU GPL v2</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/palm/">http://www.nitrc.org/projects/palm/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Oxford <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="panda:-a-pipeline-tool-for-diffusion-mri" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="PANDA: a pipeline tool for diffusion MRI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=panda:-a-pipeline-tool-for-diffusion-mri" rel="bookmark" name="panda:-a-pipeline-tool-for-diffusion-mri">PANDA: a pipeline tool for diffusion MRI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=connectivity-analysis" rel="tag">Connectivity Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=format-conversion" rel="tag">Format Conversion</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tensor-metric" rel="tag">Tensor Metric</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tractography" rel="tag">Tractography</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> PANDA (Pipeline for Analyzing braiN Diffusion imAges) is a matlab toolbox for pipeline processing of diffusion MRI images. For each subject, PANDA can provide outputs in 2 types: i) diffusion parameter data that is ready for statistical analysis; ii) brain anatomical networks constructed by using diffusion tractography. Particularly, there are 3 types of resultant diffusion parameter data: WM atlas-level, voxel-level and TBSS-level. The brain network generated by PANDA has various edge definitions, e.g. fiber number, length, or FA-weighted. <br />
<br />
The key advantages of PANDA are as follows: <br />
<br />
1.) fully-automatic processing from raw DICOM/NIFTI to final outputs; <br />
<br />
2.) Supporting both sequential and parallel computation. The parallel environment can be a single desktop with multiple-cores or a computing cluster with a SGE system;<br />
<br />
3.) A very friendly GUI (graphical user interface). 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=582&screenshot_id=509"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=582&screenshot_id=509" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 2, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/panda/ ">http://www.nitrc.org/projects/panda/ </a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Beijing Normal University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="papaya" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Papaya">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=papaya" rel="bookmark" name="papaya">Papaya</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=two-dimensional-display" rel="tag">Two Dimensional Display</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Papaya is a pure JavaScript medical research image viewer, supporting DICOM and NIFTI formats, compatible across a range of web browsers. This orthogonal viewer supports overlays, atlases, GIFTI surface data and DTI data. The Papaya UI is configurable with many display, menu and control options and can be run on a web server or as a local, shareable file.<br />
<br />
More info: <a href="http://rii-mango.github.io/Papaya/" target="_new">http://rii-mango.github.io/Papaya/</a><br />
Demo: <a href="http://rii.uthscsa.edu/mango/papaya/index.html" target="_new">http://rii.uthscsa.edu/mango/papaya/inde...</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=768&screenshot_id=694"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=768&screenshot_id=694" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 12, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://rii-mango.github.io/Papaya/">http://rii-mango.github.io/Papaya/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UTHSCSA <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS<br>OS Independent<br>POSIX/UNIX-like<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>DICOM, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>DICOM, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="paradigm" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Paradigm">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=paradigm" rel="bookmark" name="paradigm">Paradigm</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=eeg" rel="tag">Eeg</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=experimental-control" rel="tag">Experimental Control</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=eye-tracking" rel="tag">Eye Tracking</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=scanner" rel="tag">Scanner</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Flexible and millisecond accurate experimental control for cognitive neuroscience, psychology and linguistics research. Build your experiments using Paradigm's simple drag and drop interface. Presents text, images, sounds, movies, self-paced reading trials and rating scales. An integrated Python scripting API is available to ensure total flexibility and control. Joystick and microphone response are available. Supports button boxes from PST, Cedrus,<br />
fORP and custom built response boxes. Paradigm can detect fMRI triggers through serial and parallel ports. Includes sample experiments that implement many of the most popular experiment designs. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=460&screenshot_id=612"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=460&screenshot_id=612" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 9, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Paradigm License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.paradigmexperiments.com">http://www.paradigmexperiments.com</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Microsoft</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="parallel-graph-theoretical-analysis-toolkit" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Parallel Graph-theoretical Analysis Toolkit">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=parallel-graph-theoretical-analysis-toolkit" rel="bookmark" name="parallel-graph-theoretical-analysis-toolkit">Parallel Graph-theoretical Analysis Toolkit</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> PAGANI Toolkit is a complete software package which can implement all process including brain network construction and network analysis for fMRI data. A CPU-GPU hybrid framework is adopted to make PAGANI Toolkit faster in network analysis.Our PAGANI Toolkit has low hardware requirements for only a PC with a CUDA available GPU. Compared with PC clusters, it’s easier to set up and has comparable or better performance for human brain network analysis. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 5, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/pagani_toolkit/">http://www.nitrc.org/projects/pagani_toolkit/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Department of Electronic Engineering, Tsinghua University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="parallel-stochastic-ion-channel-simulator" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Parallel Stochastic Ion Channel Simulator">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=parallel-stochastic-ion-channel-simulator" rel="bookmark" name="parallel-stochastic-ion-channel-simulator">Parallel Stochastic Ion Channel Simulator</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> PSICS is software for efficient generation and simulation of models containing stochastic ion channels distributed across dendritic and axonal membranes </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 9, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.psics.org">http://www.psics.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Edinburgh <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="paraview" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ParaView">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=paraview" rel="bookmark" name="paraview">ParaView</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ParaView is an open-source, multi-platform data analysis and visualization application. ParaView users can quickly build visualizations to analyze their data using qualitative and quantitative techniques. The data exploration can be done interactively in 3D or programmatically using ParaView's batch processing capabilities.<br />
<br />
ParaView was developed to analyze extremely large datasets using distributed memory computing resources. It can be run on supercomputers to analyze datasets of terascale as well as on laptops for smaller data. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 22, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.paraview.org/">http://www.paraview.org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>KITWARE Inc. <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="parkinson's-disease-discovery-database" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Parkinson's Disease Discovery Database">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=parkinson's-disease-discovery-database" rel="bookmark" name="parkinson's-disease-discovery-database">Parkinson's Disease Discovery Database</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Tools will be available for biomedical data mining and visualization as well as linkages to Google Maps and other online resources. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 12, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>LONI Software License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/pd3/">http://www.nitrc.org/projects/pd3/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="pattern-recognition-based-on-joint-gam-sparsity-regression-model" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Pattern Recognition based on Joint GAM-Sparsity Regression Model">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=pattern-recognition-based-on-joint-gam-sparsity-regression-model" rel="bookmark" name="pattern-recognition-based-on-joint-gam-sparsity-regression-model">Pattern Recognition based on Joint GAM-Sparsity Regression Model</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This repository provides MATLAB toolbox for extracting meaningful patterns affected by a certain disease from a large data which are biased to other factors, e.g., age, gender, socioeconomic status, and scanner type. To find the meaningful patterns being able to distinguish group differences while suppressing the impact of other factors, we jointly parameterize a general additive model for desensitizing the image scores and a sparsity-constrained, logistic-regression model for classification by maximizing a likelihood. The software was developed by the Center for Health Sciences, SRI International.<br />
<br />
If you use this code, please cite the following publication: <br />
<br />
Park SH, Zhang Y, Kwon D, Zhao Q, Zahr N, Pfefferbaum A, Sullivan E, Pohl, KM: Alcohol use effect on adolescent brain development revealed by simultaneously removing confounding factors, identifying morphometric patterns, and classifying individuals, Scientific Reports, In press. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 1, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>PatternRecog_GAM_SparityRegression license</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/gam_sparityreg/">http://www.nitrc.org/projects/gam_sparityreg/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>SRI International <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="pbrain" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="pBrain">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=pbrain" rel="bookmark" name="pbrain">pBrain</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=event-related-potential" rel="tag">Event Related Potential</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-rendering" rel="tag">Surface Rendering</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Pbrain is a collection of applications for the analysis of EEG and medical image data. Currently there are two applications, eegview and loc3d. loc3d is a 3D image analysis application for localizing, identifying and labeling objects in image data, which we use primarily for locating electrodes in CT data. eegview is an eeg visualization and analysis application which has facilities for displaying the EEG mapped onto the spatial coordinates output from loc3d. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=893&screenshot_id=873"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=893&screenshot_id=873" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 17, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://nipy.sourceforge.net/pbrain/">http://nipy.sourceforge.net/pbrain/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Chicago <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="pdbp:-parkinson's-disease-biomarkers-program" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="PDBP: Parkinson's Disease Biomarkers Program">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=pdbp:-parkinson's-disease-biomarkers-program" rel="bookmark" name="pdbp:-parkinson's-disease-biomarkers-program">PDBP: Parkinson's Disease Biomarkers Program</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=knowledge-environment" rel="tag">Knowledge Environment</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=web-resource" rel="tag">Web Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The National Institute of Neurological Disorders and Stroke (NINDS) Parkinson's Disease Biomarkers Program (PDBP) was developed to accelerate the discovery of promising new diagnostic and progression biomarkers for Parkinson's Disease. The PDBP is focused on promoting the discovery of biomarker candidates for early detection and measurement of disease progression. The PDBP will coordinate the efforts of multiple stakeholders through a common data management resource and web portal. The PDBP will serve as a multi-faceted platform for:<br />
 - Integrating existing biomarker efforts<br />
 - Standardizing data collection and management across these efforts<br />
 - Accelerating the discovery of new biomarkers<br />
 - Fostering and expanding collaborative opportunities for all stakeholders. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 8, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>NIH Data Access Policy</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://pdbp.ninds.nih.gov/index.jsp">http://pdbp.ninds.nih.gov/index.jsp</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>National Institutes of Health <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="pd-rs-fmri-meta-and-validation" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="PD RS-fMRI meta and validation">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=pd-rs-fmri-meta-and-validation" rel="bookmark" name="pd-rs-fmri-meta-and-validation">PD RS-fMRI meta and validation</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Part-1 was for coordinated-based meta-analysis. Its details were from published papers of RS-fMRI on PD. Part-2 was for validation purpose. It contains ALFF maps and ReHo maps of all subjects (n = 49 for each PD and control group). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 19, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Web Application - Free to use</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/pdrsfmriwutao/">http://www.nitrc.org/projects/pdrsfmriwutao/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>XuanWu Hospital <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="peak_nii" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="peak_nii">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=peak_nii" rel="bookmark" name="peak_nii">peak_nii</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> peak_nii: Statistical image clustering, peak detection and data extraction.<br />
The Peak_nii toolbox was developed to allow the user to have flexibility of clustering their data. Based on your threshold, it will cluster your data and find the peaks within each cluster. Additionally, it has been combined with a data extraction tool that allows one to extract the data from all the scans of the analysis from all the clusters, along with several other extraction options, with a single command. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 22, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Personal License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/peak_nii/">http://www.nitrc.org/projects/peak_nii/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Massachusetts General Hospital <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="penn-chop-age-specific-neonate-atlas" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Penn-CHOP Age-Specific Neonate Atlas">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=penn-chop-age-specific-neonate-atlas" rel="bookmark" name="penn-chop-age-specific-neonate-atlas">Penn-CHOP Age-Specific Neonate Atlas</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Penn-CHOP age-specific neonate atlas package developed by Hao Huang lab (link1: <a href="http://www.haohuanglab.org/," target="_new">http://www.haohuanglab.org/,</a> link2: <a href="http://brainmrimap.org/penn-chop-age-specific-neonate-brain-atlas.html" target="_new">http://brainmrimap.org/penn-chop-age-spe...</a>) includes volume image of atlas labels, DTI fractional anisotropy (FA) map, DTI orientation-encoded colormap (OEC) and mean diffusivity (MD) map for the human neonate brains at 33, 36 and 39 postmenstrual weeks (PMW) at scan. All maps and atlas labels are in the template space and have the same dimension and resolution with details below. <br />
 <br />
Image dimension of DTI maps and atlas labels of all PMW: 180x220x180;<br />
Resolution: 0.6x0.6x0.6 mm3<br />
<br />
Publication:<br />
Feng L, Li H, Oishi K, Mishra V, Song L, Peng Q, Ouyang M, Wang J, Slinger M, Jeon T, Lee L, Heyne R, Chalak L, Peng Y, Liu S, Huang H. 2018. Age-specific gray and white matter DTI atlas for human brain at 33, 36 and 39 postmenstrual weeks. Neuroimage, 185: 685-698, 2019. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 26, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Penn-CHOP Age-Specific Neonate Atlas</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/pennchop_atlas/">http://www.nitrc.org/projects/pennchop_atlas/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pennsylvania, Children's Hospital of Philadelphia <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="penncnv" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="PennCNV">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=penncnv" rel="bookmark" name="penncnv">PennCNV</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> PennCNV is a free software tool for Copy Number Variation (CNV) detection from SNP genotyping arrays. Currently it can handle signal intensity data from Illumina and Affymetrix arrays. With appropriate preparation of file format, it can also handle other types of SNP arrays and oligonucleotide arrays.<br />
<br />
PennCNV implements a hidden Markov model (HMM) that integrates multiple sources of information to infer CNV calls for individual genotyped samples. It differs form segmentation-based algorithm in that it considered SNP allelic ratio distribution as well as other factors, in addition to signal intensity alone. In addition, PennCNV can optionally utilize family information to generate family-based CNV calls by several different algorithms. Furthermore, PennCNV can generate CNV calls given a specific set of candidate CNV regions, through a validation-calling algorithm. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 18, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Public Domain</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/penncnv/">http://www.nitrc.org/projects/penncnv/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>USC <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="penn-hippocampus-atlas" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Penn Hippocampus Atlas">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=penn-hippocampus-atlas" rel="bookmark" name="penn-hippocampus-atlas">Penn Hippocampus Atlas</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Penn Hippocampus Atlas is a resource consisting of segmented and normalized high-resolution ex vivo MRI and histology of the human hippocampus. The current atlas, consisting of MRI scans of 31 specimens and serial histology of 9 specimens is described in the 2018 paper by Adler, Wisse, et al. in PNAS. Imaging data, source code, and statistics for the 2018 paper can be accessed using the Download link on this page. <br />
<br />
A prior version of the atlas consisting of MRI scans of 5 specimens was described in Yushkevich et al., Neuroimage 44(2):385-398, 2009. Data for this atlas are also available on the Download page. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=109&screenshot_id=1174"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=109&screenshot_id=1174" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 30, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Penn Hippocampus Atlas</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/pennhippoatlas/">http://www.nitrc.org/projects/pennhippoatlas/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pennsylvania <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="personode" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Personode">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=personode" rel="bookmark" name="personode">Personode</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Personode is a user-friendly, open-source MatLab toolbox that semi-automatically guides through ICA components classification into resting-state networks (RSN), alleviating the selection process, and allows the robust definition of either group- or subject-specific ROIs derived from RSN, which could make network analysis more accurate and individually specific. This tool can produce higher activations than ROIs based on single-studies coordinates or meta-analysis studies for expected regions in a task-related ROI activation analysis and more accurate correlation values (either positive or negative) than clusters belonging to an atlas for expected interactions among networks in a resting state functional connectivity analysis. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 26, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/personode/">http://www.nitrc.org/projects/personode/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>USP/UZH <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="pestica-&-slomoco:-physio-and-motion-correction-tools" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="PESTICA & SLOMOCO: physio and motion correction tools">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=pestica-&-slomoco:-physio-and-motion-correction-tools" rel="bookmark" name="pestica-&-slomoco:-physio-and-motion-correction-tools">PESTICA & SLOMOCO: physio and motion correction tools</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=independent-component-analysis" rel="tag">Independent Component Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=multivariate-analysis" rel="tag">Multivariate Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=physiological-recording" rel="tag">Physiological Recording</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Physiologic and motion noise are major problems in BOLD imaging. Physiologic EStimation by Temporal ICA or PESTICA is an MRI-based pulse and respiration monitor. SLice-Oriented MOtion COrrection, or SLOMOCO, is a slice-oriented motion correction for 6DOF motion of every slice. These tools are retrospective and work for ASL, multiband and 7T data.<br />
<br />
PESTICA detects the pulse and the breathing cycles _from the data itself_, equivalent to a parallel monitored pulse signal and a respiratory chest-bellows signal. One can use parallel monitored or PESTICA-monitored pulse and respiration for correction using the model-based corrections RETROICOR or IRF-RETROICOR.<br />
<br />
The other half of PESTICA is an adaptive physiologic noise removal tool (Impulse Response Function or IRF-RETROICOR) that zooms in on noise with only 6 regressors, getting all the noise that 5th order RETROICOR gets (using 20 regressors). These tools will allow you to correct your data for physiologic noise with what you currently have. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 8, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/pestica/">http://www.nitrc.org/projects/pestica/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="phenolyer" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Phenolyer">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=phenolyer" rel="bookmark" name="phenolyer">Phenolyer</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=genomic-analysis" rel="tag">Genomic Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Phenolyzer is a software aimed at phenotype/disease specific gene prioritization and gene network visualization. It can be used alone to prioritize genes genome-widely, or from user-specified genomic regions or genes. It can also be used to further prioritize genes after the variant annotation and filtering step from other tools, like ANNOVAR. Phenolyzer has both a command line version and a web version: see <a href="http://phenolyzer.wglab.org/" target="_new">http://phenolyzer.wglab.org/</a> and <a href="https://github.com/WGLab/phenolyzer." target="_new">https://github.com/WGLab/phenolyzer.</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 13, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/phenolyer/">http://www.nitrc.org/projects/phenolyer/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Children's Hospital of Philadelphia <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="philadelphia-neurodevelopmental-cohort-(pnc)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Philadelphia Neurodevelopmental Cohort (PNC)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=philadelphia-neurodevelopmental-cohort-(pnc)" rel="bookmark" name="philadelphia-neurodevelopmental-cohort-(pnc)">Philadelphia Neurodevelopmental Cohort (PNC)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Philadelphia Neurodevelopmental Cohort (PNC) initiative focuses on characterizing brain and behavior interaction with genetics. This is a collaborative research effort between the Brain Behavior Laboratory at the University of Pennsylvania and the Center for Applied Genomics at the Children’s Hospital of Philadelphia.<br />
<br />
The PNC includes a population-based sample of over 9500 individuals from the greater Philadelphia area, ages 8-21 years who received medical care at the CHOP network. The participants presented for diverse medical conditions, ranging from a well child visit and minor problems to chronic condition management to potentially life threatening health problems. Participants were genotyped during the time of their clinical visit and provided written permission to be recontacted for studies of complex pediatric disorders. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1220&screenshot_id=1136"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1220&screenshot_id=1136" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 1, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.med.upenn.edu/bbl/philadelphianeurodevelopmentalcohort.html">http://www.med.upenn.edu/bbl/philadelphianeurodevelopmentalcohort.html</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Phillidelphia <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="philips-users-community" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Philips Users Community">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=philips-users-community" rel="bookmark" name="philips-users-community">Philips Users Community</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=community" rel="tag">Community</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This communnity project is to help support the efforts of investigators using Philips Healthcare systems. This clearingsite helps users find forums, mailinglists, etc. that support this community. If you have suggestions for inclusion, let the project admin know! </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 16, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/philips_users/">http://www.nitrc.org/projects/philips_users/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>DICOM, NIfTI-1, Philips PAR/REC</td></tr>
			<tr class="even"><td>Output Formats</td><td>DICOM, NIfTI-1, Philips PAR/REC</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="phoneme-syllable-tsinghua-university-cs&t-2011" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Phoneme-Syllable Tsinghua University CS&T 2011">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=phoneme-syllable-tsinghua-university-cs&t-2011" rel="bookmark" name="phoneme-syllable-tsinghua-university-cs&t-2011">Phoneme-Syllable Tsinghua University CS&T 2011</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> FMRI data set for phoneme coding in the auditory cortex with 7 subjects and 9 Chinese CV syllables. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 8, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Academic Free License ("AFL")</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/tsinghua_2011/">http://www.nitrc.org/projects/tsinghua_2011/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Tsinghua University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="phycaa+:-adaptive-physiological-noise-correction-for-bold-fmri" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="PHYCAA+: adaptive physiological noise correction for BOLD fMRI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=phycaa+:-adaptive-physiological-noise-correction-for-bold-fmri" rel="bookmark" name="phycaa+:-adaptive-physiological-noise-correction-for-bold-fmri">PHYCAA+: adaptive physiological noise correction for BOLD fMRI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The PHYCAA+ algorithm automatically estimates and removes physiological noise in BOLD fMRI data, including the effects of heartbeat and respiration. This algorithm (1) masks out high-variance CSF and vascular tracts that may otherwise confound analyses, and (2) regresses out noise timeseries in grey matter tissue, using an adaptive multivariate component decomposition (Canonical Autocorrelations Analysis). PHYCAA+ is an efficient, automated procedure that does NOT require external measures of physiology, nor does it require the user to manually identify noise components.<br />
<br />
Based on the peer-reviewed article:<br />
Churchill &amp; Strother (2013). &quot;PHYCAA+: An Optimized, Adaptive Procedure for <br />
Measuring and Controlling Physiological Noise in BOLD fMRI&quot;. NeuroImage 82: 306-325 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=751&screenshot_id=677"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=751&screenshot_id=677" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 5, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Lesser General Public License (LGPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/phycaa_plus/">http://www.nitrc.org/projects/phycaa_plus/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Rotman Research Institute <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="physio-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="PhysIO Toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=physio-toolbox" rel="bookmark" name="physio-toolbox">PhysIO Toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The general purpose of this Matlab toolbox is the model-based physiological noise correction of fMRI data using peripheral measures of respiration and cardiac pulsation. The PhysIO Toolbox can be downloaded as part of the TAPAS software collection of the TNU and integrates with SPM.<br />
<br />
Core design goals for the toolbox were: flexibility, robustness, and quality assurance to enable physiological noise correction for large-scale and multi-center studies. <br />
<br />
Some highlights:<br />
1) Robust preprocessing via iterative peak detection, shown for noisy data and patients.<br />
2) Flexible support of peripheral data formats (Siemens, Philips, GE, BIDS, Biopac, ...) and noise models (RETROICOR, RVHRCOR).<br />
3) Fully automated noise correction and performance assessment for group studies.<br />
4) Integration in fMRI pre-processing pipelines as SPM Toolbox (Batch Editor GUI).<br />
<br />
The accompanying technical paper about the toolbox concept and methodology can be found at: <a href="https://doi.org/10.1016/j.jneumeth.2016.10.019" target="_new">https://doi.org/10.1016/j.jneumeth.2016....</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 19, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/physio/">http://www.nitrc.org/projects/physio/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Institute for Biomedical Engineering, University of Zurich & ETH Zurich <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="picasso-skull-stripping-tool" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="PICASSO skull stripping tool">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=picasso-skull-stripping-tool" rel="bookmark" name="picasso-skull-stripping-tool">PICASSO skull stripping tool</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Personalized Congregation of Algorithms for Skull Stripping Optimization (PICASSO) tool is a general skull stripping for T1-weighted brain MRIs <br />
(a) in multiple imaging sites; <br />
(b) by scanners of various vendors (GE/Siemens/Philips); <br />
(c) by scanners of various magnetic field strengths (1.0T/1.5T/3T/7T); <br />
(d) by various T1-weighted pulse sequences (MPRAGE, SPGR, etc); <br />
(e) with various fields-of-view (even containing neck and shoulders);<br />
(f) from subjects of various ages (infants, pediatrics, young adults, elderly adults); <br />
(g) from subjects of different health conditions (normative, dementia, tumor-bearing); <br />
(h) with various image resolutions (from 1x1x1mm to 1.5x1.5x6mm); and <br />
(i) with various image contrasts (high or low SNRs).<br />
<br />
The PICASSO tool runs in the unix command line. It is compatible with a single linux/mac computer, or preferably with high-performance parallel clusters (PBS or SGE).<br />
<br />
Publication pending. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 29, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/picasso/">http://www.nitrc.org/projects/picasso/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Massachusetts General Hospital <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="picsl-multi-atlas-segmentation-tool" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="PICSL Multi-Atlas Segmentation Tool">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=picsl-multi-atlas-segmentation-tool" rel="bookmark" name="picsl-multi-atlas-segmentation-tool">PICSL Multi-Atlas Segmentation Tool</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This package contains a software implementation for joint label fusion and corrective learning, which were applied in MICCAI 2012 Grand Challenge on Multi-Atlas Labeling and finished in the first place.<br />
<br />
Joint label fusion is for combining candidate segmentations produced by registering and warping multiple atlases for a target image. Corrective learning can be applied to further reduce systematic errors produced by joint label fusion. In general, corrective learning can be applied to correct systematic errors produced by other segmentation methods as well. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 4, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/picsl_malf/">http://www.nitrc.org/projects/picsl_malf/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pennsylvania <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="pipeline-neuroimaging-virtualenvironment" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Pipeline Neuroimaging VirtualEnvironment">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=pipeline-neuroimaging-virtualenvironment" rel="bookmark" name="pipeline-neuroimaging-virtualenvironment">Pipeline Neuroimaging VirtualEnvironment</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The LONI Pipeline Neuroimaging Virtual Environment (PNVE), is a self-contained virtual machine that can be executed on a common laptop or desktop, enabling the Pipeline to run virtually anywhere. Neophytes to the Pipeline can have their own private server running in minutes, software engineers and workflow designers can use the PNVE as a sandbox, and those without access to grid computing facilities can now take full advantage of the Pipeline processing environment.<br />
<br />
PNVE is distributed by the Laboratory of Neuro Imaging (<a href="http://pipeline.loni.usc.edu/PNVE" target="_new">http://pipeline.loni.usc.edu/PNVE</a>) at USC. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=366&screenshot_id=415"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=366&screenshot_id=415" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 26, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>LONI Software License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/pnve/">http://www.nitrc.org/projects/pnve/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Southern California <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="pipeline-system-for-octave-and-matlab" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Pipeline system for Octave and Matlab">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=pipeline-system-for-octave-and-matlab" rel="bookmark" name="pipeline-system-for-octave-and-matlab">Pipeline system for Octave and Matlab</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The pipeline system for Octave and Matlab (PSOM) is a lightweight library to manage complex multi-stage data processing. A pipeline is a collection of jobs, i.e. Matlab or Octave codes that are using files for inputs and outputs, described in the form of a simple Matlab/Octave structure. <br />
<br />
See <a href="http://psom.simexp-lab.org" target="_new">http://psom.simexp-lab.org</a> for more info. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=316&screenshot_id=522"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=316&screenshot_id=522" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 19, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MIT/X Consortium License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://simexp.github.io/psom/">http://simexp.github.io/psom/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Montreal <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="plink" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="PLINK">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=plink" rel="bookmark" name="plink">PLINK</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> PLINK is a free, open-source whole genome association analysis toolset, designed to perform a range of basic, large-scale genetic analyses. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 24, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU GPL v2</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/plink/">http://www.nitrc.org/projects/plink/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="pliviewer" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="PLIViewer">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=pliviewer" rel="bookmark" name="pliviewer">PLIViewer</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?tag=interactive-visualization" rel="tag">Interactive Visualization</a>, <a href="https://hbp-hpc-platform.fz-juelich.de/?tag=interactive-visualization-and-analysis" rel="tag">Interactive Visualization And Analysis</a>, <a href="https://hbp-hpc-platform.fz-juelich.de/?tag=medical-imaging" rel="tag">Medical Imaging</a>, <a href="https://hbp-hpc-platform.fz-juelich.de/?tag=performance-visualization" rel="tag">Performance Visualization</a></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=application" rel="tag">Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=gui-tool" rel="tag">Gui Tool</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description>
      <p>The <strong><a href="https://devhub.vr.rwth-aachen.de/VR-Group/pli_vis">PLIViewer</a> </strong>is visualization software for 3D-Polarized Light Imaging (3D-PLI), to interactively explore the scalar and vector datasets; it provides additional methods to transform data, thus revealing new insights that are not available in the raw representations. The high resolution provided by 3D-PLI produces massive, terabyte-scale datasets, which makes visualization challenging.</p>
<p>The PLIViewer tackles this problem by providing functionality to select areas of interests from the dataset, and options for downscaling. It makes it possible to interactively compute and visualize Orientation Distribution Functions (ODFs) and polar plots from the vector field, which reveal mesoscopic and macroscopic scale information from the microscopic dataset without significant loss of detail. The PLIViewer equips the neuroscientist with specialized visualization tools needed to explore 3D-PLI datasets through direct and interactive visualization of the data.</p>

<figure aria-describedby="caption-attachment-1824" class="wp-caption alignnone" id="attachment_1824" style="width: 458px"><a href="https://hbp-hpc-platform.fz-juelich.de/wp-content/uploads/2018/04/Figure8a.png"><img alt="" class="size-full wp-image-1824" height="248" src="https://hbp-hpc-platform.fz-juelich.de/wp-content/uploads/2018/04/Figure8a.png" width="458"/></a><figcaption class="wp-caption-text" id="caption-attachment-1824">The original dataset: Fibre Orientation Maps rendered on top of Retardation map</figcaption></figure>
<figure aria-describedby="caption-attachment-1825" class="wp-caption alignnone" id="attachment_1825" style="width: 458px"><a href="https://hbp-hpc-platform.fz-juelich.de/wp-content/uploads/2018/04/Figure8b.png"><img alt="" class="size-full wp-image-1825" height="248" src="https://hbp-hpc-platform.fz-juelich.de/wp-content/uploads/2018/04/Figure8b.png" width="458"/></a><figcaption class="wp-caption-text" id="caption-attachment-1825">A full slice from the Fibre Orientation Map of a Vervet Monkey</figcaption></figure>
<figure aria-describedby="caption-attachment-1826" class="wp-caption alignnone" id="attachment_1826" style="width: 457px"><a href="https://hbp-hpc-platform.fz-juelich.de/wp-content/uploads/2018/04/Figure8c_v2.png"><img alt="" class="size-full wp-image-1826" height="248" src="https://hbp-hpc-platform.fz-juelich.de/wp-content/uploads/2018/04/Figure8c_v2.png" width="457"/></a><figcaption class="wp-caption-text" id="caption-attachment-1826">Orientation Distribution Functions (ODFs) rendered with Streamline Tractography</figcaption></figure>
<figure aria-describedby="caption-attachment-1827" class="wp-caption alignnone" id="attachment_1827" style="width: 458px"><a href="https://hbp-hpc-platform.fz-juelich.de/wp-content/uploads/2018/04/Figure8d_v2.png"><img alt="" class="size-full wp-image-1827" height="248" src="https://hbp-hpc-platform.fz-juelich.de/wp-content/uploads/2018/04/Figure8d_v2.png" width="458"/></a><figcaption class="wp-caption-text" id="caption-attachment-1827">Close-up view of the ODFs</figcaption></figure>
    </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>February 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>Internal</td></tr>
		<tbody><tr class="even"><td>License</td><td>Public Domain</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td>1.1.0</td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://devhub.vr.rwth-aachen.de/VR-Group/pli_vis">https://devhub.vr.rwth-aachen.de/VR-Group/pli_vis</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href="https://devhub.vr.rwth-aachen.de/VR-Group/pli_vis">https://devhub.vr.rwth-aachen.de/VR-Group/pli_vis</a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Ali Demiralp
     <a href="mailto:(demiralp@gmx.com)">(demiralp@gmx.com)</a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>HDF5</td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="polgui--matlab-gui-for-polhemus-fastrak" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="POLGUI- Matlab GUI for Polhemus Fastrak">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=polgui--matlab-gui-for-polhemus-fastrak" rel="bookmark" name="polgui--matlab-gui-for-polhemus-fastrak">POLGUI- Matlab GUI for Polhemus Fastrak</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-software-resource" rel="tag">Other Software Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> POLGUI is an interface between MATLAB and the Polhemus Fastrak digitizer used to digitize fiducial locations and scalp EEG electrode locations. <br />
 <br />
There are 5 versions all of which work under MATLAB R14 (on both linux and windows platforms), <br />
1. polgui_ver1_r14 : works with 1 receiver (stylus pen) <br />
2. polgui_ver2_r14 : works with 2 receivers (including the pen) <br />
3. polgui_ver3_r14 : works with 3 receivers(including the pen) <br />
4. polgui_ver4_r14 : works with 4 receivers (including the pen) <br />
5. polgui_ver5_r14 : Generic version which works with 1/2/3/4 receivers [WARNING: Ver 5 might be buggy; not fully tested] <br />
<br />
Requirements: MATLAB R14 (Linux/Windows)<br />
<br />
The project is hosted on Sourceforge at <a href="http://sourceforge.net/projects/polgui/" target="_new">http://sourceforge.net/projects/polgui/</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=174&screenshot_id=90"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=174&screenshot_id=90" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 31, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://sourceforge.net/projects/polgui/">http://sourceforge.net/projects/polgui/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="population-averaged-diffusion-tensor-imaging-atlas-of-the-sprague-dawley-rat-brain" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Population-averaged diffusion tensor imaging atlas of the Sprague Dawley rat brain">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=population-averaged-diffusion-tensor-imaging-atlas-of-the-sprague-dawley-rat-brain" rel="bookmark" name="population-averaged-diffusion-tensor-imaging-atlas-of-the-sprague-dawley-rat-brain">Population-averaged diffusion tensor imaging atlas of the Sprague Dawley rat brain</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The rat atlas can be downloaded in different data formats:<br />
1. Mat-file: a single mat-file - ratatlas.m - contains all data, and is structured as follows:<br />
DWI: Cell array containing all 13 diffusion weighted images<br />
DT: Cell array containing the 6 independent diffusion tensor elements<br />
FE: First eigenvectors<br />
eigval: Three eigenvalues of the diffusion tensors<br />
FA: Fractional anisotropy map<br />
FEFA: FE map scaled by the FA map.<br />
VDims: Voxel dimensions<br />
g: Diffusion gradient directions<br />
bval: Diffusion gradient strength<br />
labels: Label field<br />
T1: T1 image<br />
<br />
2. Nifti: all diffusion weighted images, the T1 image, and label field are saved as individual nifti files (DWI_0..13.nii, T1.nii, and labels.nii). Diffusion gradient information has been stored in grad.txt, while the delineated anatomical structures are enumerated in labels.txt.<br />
<br />
3. Amira: the label field is also provided in Amira format, labels.am. This file can be used for further segmentation, or (3D) visualization in Amira. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 11, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/visionlab/">http://www.nitrc.org/projects/visionlab/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Antwerp <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ASCII, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ASCII, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="pos2spm" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="POS2SPM">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=pos2spm" rel="bookmark" name="pos2spm">POS2SPM</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> POS2SPM is a MATLAB-based conversion and formatting tool that can be applied to a number of different probe arrays and channel configurations using either 1 or 2 POS measurement files from an electromagnetic 3D digitizer. This conversion tool outputs the appropriately formatted files necessary to complete spatial registration of functional near-infrared spectroscopy (fNIRS) data with the NIRS-SPM toolbox (SPM5 or SPM8), the SPM-fNIRS toolbox (SPM12), or both. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1022&screenshot_id=972"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1022&screenshot_id=972" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 25, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/pos2spm/">http://www.nitrc.org/projects/pos2spm/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="ppmi" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="PPMI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=ppmi" rel="bookmark" name="ppmi">PPMI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-data" rel="tag">Atlas Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> PPMI includes a large and comprehensive set of correlated clinical data and biospecimens that is made available to the entire scientific community to help accelerate biomarker verification research. PPMI data and specimens have been collected in a standardized manner under strict protocols developed by the steering committee. The PPMI study dataset includes clinical, biological and imaging data collected at PPMI clinical sites. The data have been aggregated into the PPMI study database, which is managed by the PPMI Bioinformatics Core, the Laboratory of NeuroImaging (LONI) at the University of Southern California.<br />
<br />
<a href="http://www.ppmi-info.org/access-data-specimens/" target="_new">http://www.ppmi-info.org/access-data-spe...</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=471&screenshot_id=416"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=471&screenshot_id=416" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 28, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/ppmi/">http://www.nitrc.org/projects/ppmi/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Southern California <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, MINC</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, MINC</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="predict" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="PREDiCT">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=predict" rel="bookmark" name="predict">PREDiCT</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> PREDiCT is the Patient Repository for EEG Data + Computational Tools. It offers one-stop shopping for EEG tasks, data storage, and analytic tools for pattern classification of clinical groups. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1187&screenshot_id=1108"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1187&screenshot_id=1108" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 27, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://predictsite.com">http://predictsite.com</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of New Mexico <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="predicting-alzheimer’s-conversion-in-mild-cognitive-impairment-patients-using-longitudinal-neuroimaging-and-clinical-markers" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Predicting Alzheimer’s conversion in mild cognitive impairment patients using longitudinal neuroimaging and clinical markers">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=predicting-alzheimer’s-conversion-in-mild-cognitive-impairment-patients-using-longitudinal-neuroimaging-and-clinical-markers" rel="bookmark" name="predicting-alzheimer’s-conversion-in-mild-cognitive-impairment-patients-using-longitudinal-neuroimaging-and-clinical-markers">Predicting Alzheimer’s conversion in mild cognitive impairment patients using longitudinal neuroimaging and clinical markers</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Patients with mild cognitive impairment (MCI) have a high risk for conversion to Alzheimer’s disease (AD). Selecting a set of relevant markers from multimodal data to predict conversion from MCI to AD has become a challenging task.<br />
The aim of this project is to quantify the impact of longitudinal predictive models with single- or multisource data for predicting MCI-to-AD conversion and identifying a very small subset of features that are highly predictive of conversion. We developed predictive models of MCI-to-AD progression that combine magnetic resonance imaging (MRI)-based markers (cortical thickness and volume of subcortical structures) with neuropsychological tests. A set of longitudinal features potentially discriminating between<br />
MCI subjects who convert to AD and those who remain stable over a period of 3 years was obtained. The proposed approach was developed, trained and evaluated using the<br />
Alzheimer’s Disease Neuroimaging Initiative (ADNI) dataset </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 3, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Non-Commercial Software License Agreement</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/predict_mci2ad/">http://www.nitrc.org/projects/predict_mci2ad/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Universidad Politécnica de Madrid <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="prediction-and-diagnosis-for-depression-and-schizophrenia" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Prediction and Diagnosis for Depression and Schizophrenia">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=prediction-and-diagnosis-for-depression-and-schizophrenia" rel="bookmark" name="prediction-and-diagnosis-for-depression-and-schizophrenia">Prediction and Diagnosis for Depression and Schizophrenia</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Researches have shown that Depression and Schizophrenia is a dynamic, nonstatic phenomenon. Accordingly, it appears to be more important to prevent the worsening of these diseases over time, and early assessment and prediction for these diseases can play an important role in achieving early intervention and reducing the incidence of them. In our research, we plan to explore the tool and resource about how to predict Depression and Schizophrenia based on Demographics and physiological information(EEG, ERPs, Genetics, MRI, fMRI, etc.). Ultimately, we hope this research can lay the theoretical foundation for the application of prediction for Depression and Schizophrenia. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 15, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>ADNI License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/zhangxw/">http://www.nitrc.org/projects/zhangxw/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Lanzhou University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="preprocessed-connectomes-project" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Preprocessed Connectomes Project">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=preprocessed-connectomes-project" rel="bookmark" name="preprocessed-connectomes-project">Preprocessed Connectomes Project</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Initiative to systematically preprocess neuroimaging data from open sharing repositories and to share the derivatives. Initiated in 2011 with the ADHD-200 Preprocessed initative, the PCP has grown to include the Beijing Enhanced DTI dataset and ABIDE. To enable the comparison of different preprocessing choices and to accomodate different opinions about the best preprocessings strategies, most of the data is preprocessed using a variety of tools and parameters.<br />
<br />
Data is currently hosted in an Amazon Web Services Public S3 Bucket and at NITRC. More information about downloading the data and about the various preprocessing pipelines and strategies employed can be found on the Datasets page. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=827&screenshot_id=1170"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=827&screenshot_id=1170" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 6, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://preprocessed-connectomes-project.org/">http://preprocessed-connectomes-project.org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="presentation" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Presentation">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=presentation" rel="bookmark" name="presentation">Presentation</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=platform-or-development-environment" rel="tag">Platform Or Development Environment</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Presentation® is a precise and powerful stimulus delivery and experimental control program for neuroscience. Presentation® runs on any Windows PC, and delivers auditory, visual and multimodal stimuli with sub-millisecond temporal precision. Presentation® includes special features for experiments using psychophysics, eye movements, fMRI, ERP, MEG, single neuron recording, and more. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 22, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Available by Purchase Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/presentation/">http://www.nitrc.org/projects/presentation/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="primatologist" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Primatologist">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=primatologist" rel="bookmark" name="primatologist">Primatologist</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Primatologist is a BrainVISA toolbox dedicated to the processing of structural MRIs acquired in macaques (for now) and more generally in non-human primates (NHP, in the future).<br />
<br />
Its main component is a modular segmentation pipeline for T1 or T2-weighted MRIs based on a generative model of intensities, with NHP-specific preprocessing tools. It allows segmenting the brain into 17 anatomical regions. The results can then easily be processed with Morphologist to extract meshes of the brain and sulci.<br />
<br />
It takes advantages of the features of BrainVISA including an ontological organization of the data, the possibility to easily distribute processing on a multicore workstation or on a computing cluster, and easy quality checking with dedicated viewers. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1198&screenshot_id=1119"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1198&screenshot_id=1119" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 3, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>CeCill v2</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/primatologist/">http://www.nitrc.org/projects/primatologist/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>CEA <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="principal-components-analysis-of-scalar,-vector,-and-mesh-vertex-data" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Principal Components Analysis of Scalar, Vector, and Mesh Vertex Data">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=principal-components-analysis-of-scalar,-vector,-and-mesh-vertex-data" rel="bookmark" name="principal-components-analysis-of-scalar,-vector,-and-mesh-vertex-data">Principal Components Analysis of Scalar, Vector, and Mesh Vertex Data</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> An implementation of standard PCA algorithms for use on<br />
scalar or vector data sets. Kernel PCA is implemented in this class as well, where the data sets are scalar or<br />
vector valued functions assigned at each of the points in a PointSet. A Gaussian Distance Kernel class is<br />
provided with the PCA class.<br />
 <br />
This contribution is part of a shape analysis software pipeline created at Johns Hopkins. PCA will be used<br />
to develop a set of basis vectors for use with Gaussian Random Field analysis. The output of PCA will be<br />
analyzed for significance with various statistical methods such as t-tests built upon the built-in statistical<br />
capabilities of ITK. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 4, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/pca-scalar-mesh/">http://www.nitrc.org/projects/pca-scalar-mesh/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Johns Hopkins University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="probabilisticbiascorrection" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ProbabilisticBiasCorrection">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=probabilisticbiascorrection" rel="bookmark" name="probabilisticbiascorrection">ProbabilisticBiasCorrection</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> A multichannel capable tool for probabilistic inhomogeneity correction implemented as both a standalone command line tool and a Slicer3 module. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 7, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>3D Slicer License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/probbiascor/">http://www.nitrc.org/projects/probbiascor/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="probability-associated-community-estimation" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Probability-Associated Community Estimation">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=probability-associated-community-estimation" rel="bookmark" name="probability-associated-community-estimation">Probability-Associated Community Estimation</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Understanding the modularity of fMRI derived brain networks or ‘connectomes’ can inform the study of brain function organization. However, fMRI connectomes additionally involve negative edges, which may not be optimally accounted for by existing approaches to modularity that variably threshold, binarize, or arbitrarily weight these connections. Consequently, many existing Q maximization-based modularity algorithms yield variable modular structures. Here we present an alternative complementary approach that exploits how frequent the BOLD-signal correlation between two nodes is negative. This approach a) permits a dual formulation, leading to equivalent solutions regardless of whether one considers positive or negative edges; b) is theoretically linked to the Ising model defined on the connectome, thus yielding modularity result that maximizes data likelihood. Please refer to Zhan et. al. 2017 J Comp Neurol. 525(15):3251-3265. doi: 10.1002/cne.24274 for details. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 28, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Open Software License 3.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/fmri_pace/">http://www.nitrc.org/projects/fmri_pace/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="proctseg" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ProCTSeg">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=proctseg" rel="bookmark" name="proctseg">ProCTSeg</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ProCTSeg is a prostate segmentation tool for the treatment CTs. This toolbox includes an image viewer, graphical user interface, and automatic algorithms for landmark detection and prostate segmentation. The tool can support the following functions: Open medical images with various formats (such as mhd, hdr, nii), control intensity range in visualization, zoom in/out, show prostate landmarks and segmentation results, load and save landmark positions and segmentation results, and so on. The software is developed by the IDEA group at the University of North Carolina at Chapel Hill (<a href="https://www.med.unc.edu/bric/ideagroup" target="_new">https://www.med.unc.edu/bric/ideagroup</a>).<br />
<br />
Windows operating system is required to run this software. The executable file was compiled on Windows 7 (64 bit). The graphical user interfaces and the overall framework of the software were implemented in C++. Parallelization technologies were also used to speed up landmark detection and prostate segmentation. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=892&screenshot_id=871"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=892&screenshot_id=871" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 4, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>ProCTSeg license</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/proctseg/">http://www.nitrc.org/projects/proctseg/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina at Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Windows XP</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="profound-hearing-loss-database" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Profound Hearing Loss Database">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=profound-hearing-loss-database" rel="bookmark" name="profound-hearing-loss-database">Profound Hearing Loss Database</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Profound hearing loss database<br />
Shared with a Creative Commons Attribution 4.0 International Licence (CC BY) License. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 20, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/phl_2019/">http://www.nitrc.org/projects/phl_2019/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>City University of Hong Kong and Johns Hopkins University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="program-for-optimal-design-of-blocked-fmri-experiments" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Program for optimal design of blocked fMRI experiments">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=program-for-optimal-design-of-blocked-fmri-experiments" rel="bookmark" name="program-for-optimal-design-of-blocked-fmri-experiments">Program for optimal design of blocked fMRI experiments</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-software-resource" rel="tag">Other Software Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The computer program POBE (program for optimal design of blocked experiments, version 1.1) provides a graphical user interface for fMRI researchers to easily and efficiently design their blocked experiments. The computer program POBE calculates the optimal number of subjects and the optimal scanning time for user specified experimental factors and model parameters so that the statistical efficiency is maximised for a given study budget. POBE can also be used to determine the minimum budget for a given power. Furthermore, a maximin design can be determined as efficient design for a possible range of values for the unknown model parameters. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 18, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/pobe/">http://www.nitrc.org/projects/pobe/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Liege <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="progression-score-model-toolkit" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Progression Score Model Toolkit">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=progression-score-model-toolkit" rel="bookmark" name="progression-score-model-toolkit">Progression Score Model Toolkit</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Progression Score Model Toolkit is a Matlab-based cross-platform statistical software that allows for the characterization of the temporal trajectories of voxelwise imaging measures from longitudinal data. The method can be applied to a variety of images including PET radiotracer binding maps, thickness maps, functional connectivity in fMRI maps, FA/MD maps, and determinant of Jacobian maps.<br />
<br />
The toolkit provides code for visualization of the estimated trajectories as a movie. The method also computes a summary score for each scan that indicates its position along the estimated trajectories. These summary scores can be used to explore associations with external variables such as genetic factors, disease risk scores, etc.<br />
<br />
Please cite:<br />
Murat Bilgel, Jerry L. Prince, Dean F. Wong, Susan M. Resnick, Bruno M. Jedynak, &quot;A multivariate nonlinear mixed effects model for longitudinal image analysis: Application to amyloid imaging&quot;, NeuroImage (2016), DOI: 10.1016/j.neuroimage.2016.04.001. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 1, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Lesser General Public License (LGPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/progscore/">http://www.nitrc.org/projects/progscore/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Johns Hopkins University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="pronto-–-pattern-recognition-for-neuroimaging-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="PRoNTo – Pattern Recognition for Neuroimaging Toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=pronto-–-pattern-recognition-for-neuroimaging-toolbox" rel="bookmark" name="pronto-–-pattern-recognition-for-neuroimaging-toolbox">PRoNTo – Pattern Recognition for Neuroimaging Toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> PRoNTo (Pattern Recognition for Neuroimaging Toolbox) is a software toolbox based on pattern recognition techniques for the analysis of neuroimaging data. Statistical pattern recognition is a field within the area of machine learning which is concerned with automatic discovery of regularities in data through the use of computer algorithms, and with the use of these regularities to take actions such as classifying the data into different categories. In PRoNTo, brain scans are treated as spatial patterns and statistical learning models are used to identify statistical properties of the data that can be used to discriminate between experimental conditions or groups of subjects (classification models) or to predict a continuous measure (regression models). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 26, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.mlnl.cs.ucl.ac.uk/pronto/">http://www.mlnl.cs.ucl.ac.uk/pronto/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University College London, Department of Computer Science <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="propixx-dlp-led-projector" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="PROPixx DLP LED projector">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=propixx-dlp-led-projector" rel="bookmark" name="propixx-dlp-led-projector">PROPixx DLP LED projector</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=eeg" rel="tag">Eeg</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=eye-tracking" rel="tag">Eye Tracking</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=meg" rel="tag">Meg</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=physiological-stimulation" rel="tag">Physiological Stimulation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=response-monitoring" rel="tag">Response Monitoring</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=scanner" rel="tag">Scanner</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=stimulus-presentation" rel="tag">Stimulus Presentation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The PROPixx is a unique DLP LED projector which has been designed to be the most flexible display solution for vision research and neuroscience research. The PROPixx features a native resolution of 1920 x 1080, and can be driven with refresh rate up to 500Hz with deterministic timing. The PROPixx uses high brightness LEDs as a light source, giving a wide colour gamut and much longer lifetime than halogen light sources. It features high-bit depth, up to 12-bit per color for high-frequency full colour stimulation. For stereo vision applications, our high-speed ferro-electric circular polarizer can project stereoscopic stimuli with the use of passive glasses at up to 400Hz. In addition the PROPixx includes an array of peripherals which often need to be synchronized to video during an experiment, and with perfect microsecond precision. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=732&screenshot_id=656"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=732&screenshot_id=656" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 20, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://vpixx.com/our-products/projector/">http://vpixx.com/our-products/projector/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>VPixx Technologies <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="prospective-acquisition-correction-(pace)-resting-state-epi-data" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Prospective Acquisition CorrEction (PACE) Resting State EPI data">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=prospective-acquisition-correction-(pace)-resting-state-epi-data" rel="bookmark" name="prospective-acquisition-correction-(pace)-resting-state-epi-data">Prospective Acquisition CorrEction (PACE) Resting State EPI data</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This data release contains the 4D NifTi pre-processed images corresponding to resting-state functional magnetic resonance imaging (Rs-fMRI) data acquired from 47 healthy individuals using the EPI-PACE sequence on a Siemens 3T Verio scanner at the Auburn University MRI Research Center. Along with the 4D images, the release contains voxel-wise frame-wise displacement of the head (FDvox) for each subject, MATLAB files containing the motion metrics (3 translations and 3 rotations) and summary motion statistics.<br />
<br />
We provide the 4D NifTi data for several combinations of nuisance signal regressors and retrospective motion correction approaches: (i) CSF (cerebro-spinal fluid) + WM (white matter) signal regression, (ii) CSF + WM + GS (global signal) regression, (iii) CSF + WM + Friston-24 motion regression, (iv) CSF + WM + GS + Friston-24 motion regression, (v) CSF + WM + Friston-24 motion regression + motion censoring, and (vi) CSF + WM + GS + Friston-24 motion regression + motion censoring. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 4, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/epi_pace_rest/">http://www.nitrc.org/projects/epi_pace_rest/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Auburn University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="psychopy" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="PsychoPy">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=psychopy" rel="bookmark" name="psychopy">PsychoPy</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=experimental-control" rel="tag">Experimental Control</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> PsychoPy - Psychology software in Python. PsychoPy is an open-source application to allow the presentation of stimuli and collection of data for a wide range of neuroscience, psychology and psychophysics experiments. It is intended as a free, powerful alternative to Presentation or e-Prime. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=369&screenshot_id=239"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=369&screenshot_id=239" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 4, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.psychopy.org">http://www.psychopy.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>FreeBSD<br>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="pubbrain-database" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="PubBrain Database">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=pubbrain-database" rel="bookmark" name="pubbrain-database">PubBrain Database</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> PubBrain is a literature search and visualization tool that allows end users to enter any PubMed query and see that query rendered as a heatmap illustrating which regions of interest are most commonly mentioned within the search results. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 30, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.pubbrain.org/">http://www.pubbrain.org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="pvs_enhance" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="PVS_Enhance">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=pvs_enhance" rel="bookmark" name="pvs_enhance">PVS_Enhance</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> PVS_Enhance is a toolbox for the enhancement of perivascular spaces (very thin and weak tubular structures) in MR image. Methods in the toolbox are based on Haar transformation of non-local cubes and 4D block-matching filtering. The software was implemented in MATLAB with cpp files (Mex compiling is required) and developed by the IDEA group at the University of North Carolina at Chapel Hill ( <a href="https://www.med.unc.edu/bric/ideagroup" target="_new">https://www.med.unc.edu/bric/ideagroup</a>).<br />
<br />
More detailed explanations can be found in the following paper. If you use this code, please include the following article as a reference.<br />
<br />
[Yingkun Hou*, Sang Hyun Park*, Qian Wang, Jun Zhang, Xiaopeng Zong, Weili Lin, Dinggang Shen, &quot;Enhancement of Perivascular Spaces in 7T MR Image using Haar Transform of Non-local Cubes and Block-matching Filtering,&quot; Scientific Reports 7, 8569, Aug. 2017.] 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1176&screenshot_id=1097"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1176&screenshot_id=1097" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 28, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>PVS_Enhance Software</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/pvs_enhance/">http://www.nitrc.org/projects/pvs_enhance/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina at Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="pydicom" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Pydicom">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=pydicom" rel="bookmark" name="pydicom">Pydicom</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> pydicom is a pure python package for working with DICOM files. It was made for inspecting and modifying DICOM data in an easy &quot;pythonic&quot; way. The modifications can be written again to a new file. As a pure python package, it should run anywhere python runs without any other requirements. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 21, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MIT/X Consortium License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://github.com/pydicom/pydicom">https://github.com/pydicom/pydicom</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>DICOM</td></tr>
			<tr class="even"><td>Output Formats</td><td>DICOM</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="pymvpa" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="PyMVPA">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=pymvpa" rel="bookmark" name="pymvpa">PyMVPA</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=frequency-domain" rel="tag">Frequency Domain</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=independent-component-analysis" rel="tag">Independent Component Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=linear" rel="tag">Linear</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=modeling" rel="tag">Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=multivariate-analysis" rel="tag">Multivariate Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=nonlinear" rel="tag">Nonlinear</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=principal-component-analysis" rel="tag">Principal Component Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=temporal-transformation" rel="tag">Temporal Transformation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> PyMVPA is a Python package intended to ease statistical learning analyses of large datasets. It offers an extensible framework with a high-level interface to a broad range of algorithms for classification, regression, feature selection, data import and export. While it is not limited to the neuroimaging domain, it is eminently suited for such datasets. PyMVPA is truly free software (in every respect) and additionally requires nothing but free-software to run. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=162&screenshot_id=221"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=162&screenshot_id=221" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 9, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MIT/X Consortium License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.pymvpa.org">http://www.pymvpa.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="pynifti" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="PyNIfTI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=pynifti" rel="bookmark" name="pynifti">PyNIfTI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The PyNIfTI module is a Python interface to the NIfTI I/O libraries. Using PyNIfTI, one can easily read and write NIfTI and ANALYZE images from within Python. The NiftiImage class provides pythonic access to the full header information and for a maximum of interoperability the image data is made available via NumPy arrays. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 6, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MIT/X Consortium License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/pynifti/">http://www.nitrc.org/projects/pynifti/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="pynn" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="PyNN">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=pynn" rel="bookmark" name="pynn">PyNN</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> PyNN (pronounced 'pine') is a simulator-independent language for building neuronal network models.<br />
<br />
In other words, you can write the code for a model once, using the PyNN API and the Python programming language, and then run it without modification on any simulator that PyNN supports (currently NEURON, NEST, and Brian), and on the SpiNNaker and BrainScaleS neuromorphic hardware systems.<br />
<br />
The PyNN API aims to support modelling at a high-level of abstraction (populations of neurons, layers, columns and the connections between them) while still allowing access to the details of individual neurons and synapses when required. PyNN provides a library of standard neuron, synapse and synaptic plasticity models, verified to work the same on the different supported simulators. PyNN also provides a set of commonly-used connectivity algorithms (e.g. all-to-all, random, distance-dependent) but makes it easy to provide your own connectivity in a simulator-independent way. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 4</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>CeCill v2</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/pynn/">http://www.nitrc.org/projects/pynn/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>The PyNN team <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="pysurfer" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="PySurfer">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=pysurfer" rel="bookmark" name="pysurfer">PySurfer</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> PySurfer is a Python based program for visualization and interaction with cortical surface representations of neuroimaging data from Freesurfer. It extends Mayavi’s powerful visualization engine with a high-level interface for working with MRI and MEG data. PySurfer offers both a command-line interface designed to broadly replicate Freesurfer’s Tksurfer program as well as a Python library for writing scripts to efficiently explore complex datasets. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=517&screenshot_id=388"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=517&screenshot_id=388" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 26, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://pysurfer.github.com">http://pysurfer.github.com</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="pythagorean-displacement-and-motion-regressors" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Pythagorean Displacement and Motion Regressors">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=pythagorean-displacement-and-motion-regressors" rel="bookmark" name="pythagorean-displacement-and-motion-regressors">Pythagorean Displacement and Motion Regressors</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This Matlab script uses the Pythagorean Theorem to calculate head motion and position, while preserving degrees of freedom. The motion parameters output by SPM (rp*.txt) estimate head position relative to the first volume in 3D translation and 3D rotation, which are often entered as a nuisance regressor during individual-level statistics. Regressing the total displacement and relative position can potentially explain more variance in voxel-level BOLD signals that is related to head movement during an fMRI experiment. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 27, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/pythagoras/">http://www.nitrc.org/projects/pythagoras/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Medical University of South Carolina <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="pyxnat" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="pyxnat">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=pyxnat" rel="bookmark" name="pyxnat">pyxnat</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> pyxnat is a simple python library that relies on the REST API provided by the XNAT platform since its 1.4 version. XNAT is an extensible database for neuroimaging data. The main objective is to ease communications with an XNAT server to plug-in external tools or python scripts to process the data. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 28, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/pyxnat/">http://www.nitrc.org/projects/pyxnat/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="qcqp" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="QCQP">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=qcqp" rel="bookmark" name="qcqp">QCQP</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Quadratically constrained quadratic programing (QCQP) technique in medical image analysis. QCQP based tools are provided for classification, segmentation, and bias field correction. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 12, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/qcqp/">http://www.nitrc.org/projects/qcqp/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="qmodeling" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="QModeling">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=qmodeling" rel="bookmark" name="qmodeling">QModeling</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quantification" rel="tag">Quantification</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> QModeling is a multiplatform toolbox for SPM to fit reference-region kinetic models (SRTM, SRTM2, Patlak Reference and Logan Reference plot are currently available in QModeling) to dynamic PET studies. The toolbox was developed in the bioengineering department of the Molecular Imaging Unit at CIMES (FGUMA) in collaboration with the department of Computer Science and Programming Languages at Malaga University (UMA).<br />
More information: <a href="http://www.uimcimes.es" target="_new">http://www.uimcimes.es</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1197&screenshot_id=1157"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1197&screenshot_id=1157" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 1, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License v3</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.uimcimes.es/contenidos/golink?p=1">http://www.uimcimes.es/contenidos/golink?p=1</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Fundación General de la Universidad de Málaga <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="quantification-of-structural-brain-connectivity-via-a-conductance-model" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Quantification of Structural Brain Connectivity via a Conductance Model">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=quantification-of-structural-brain-connectivity-via-a-conductance-model" rel="bookmark" name="quantification-of-structural-brain-connectivity-via-a-conductance-model">Quantification of Structural Brain Connectivity via a Conductance Model</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=connectivity-analysis" rel="tag">Connectivity Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=diffusion-application" rel="tag">Diffusion Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=diffusion-mr-fiber-tracking" rel="tag">Diffusion Mr Fiber Tracking</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=modeling" rel="tag">Modeling</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This toolbox computes the structural connectivity of the brain (connectome) from diffusion-weighted MRI, using a conductance-based mathematical model introduced by Frau-Pascual et al (NeuroImage, 2019). 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1335&screenshot_id=1245"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1335&screenshot_id=1245" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 15, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>FreeSurfer License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/conductance">http://www.nitrc.org/projects/conductance</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Massachusetts General Hospital / Harvard Medical School <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="quantitative-diffusion-tools" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Quantitative Diffusion Tools">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=quantitative-diffusion-tools" rel="bookmark" name="quantitative-diffusion-tools">Quantitative Diffusion Tools</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quantification" rel="tag">Quantification</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tractography" rel="tag">Tractography</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Quantitative Diffusion Tools provide Slicer3 modules for quantitative diffusion analysis. Modules include tools for clustering fiber tracts, summarizing measures over tract clusters, etc. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 23, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>3D Slicer License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/quantitativedti/">http://www.nitrc.org/projects/quantitativedti/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="quicknii---serial-section-aligner-to-volumetric-atlases" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="QuickNII - Serial section aligner to volumetric atlases">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=quicknii---serial-section-aligner-to-volumetric-atlases" rel="bookmark" name="quicknii---serial-section-aligner-to-volumetric-atlases">QuickNII - Serial section aligner to volumetric atlases</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> QuickNII is a tool for user guided affine registration (anchoring) of 2D experimental image data, typically high resolution microscopic images, to 3D atlas reference space, facilitating data integration through standardized coordinate systems.<br />
<br />
Key features:<br />
- generate user defined cut planes through the atlas templates, matching the orientation of the cut plane of the 2D experimental image data, as a first step towards anchoring of images to the relevant atlas template<br />
- propagate spatial transformations across series of sections following anchoring of selected images<br />
<br />
Multi-modality atlasing datasets provided with QuickNII:<br />
- Waxholm Space Atlas of the Sprague Dawley rat brain, version 2 (T2* MRI, DTI, and delineations at 39 μm resolution)<br />
- Allen Mouse Brain Atlas reference atlas, CCFv3 (SST average template, grayscale Nissl volume reconstruction and delineations from 2015 at 25 μm resolution)<br />
<br />
(Downloads, Documents and MediaWiki have content)<br />
Temporary reference: <a href="https://doi.org/10.3389/fnana.2018.00082" target="_new">https://doi.org/10.3389/fnana.2018.00082</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 4, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/quicknii/">http://www.nitrc.org/projects/quicknii/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Oslo <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="random-forest-for-prediction-of-ci_wrs" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Random forest for prediction of CI_WRS">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=random-forest-for-prediction-of-ci_wrs" rel="bookmark" name="random-forest-for-prediction-of-ci_wrs">Random forest for prediction of CI_WRS</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> To be added soon </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 20, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/ci_outcome_pred/">http://www.nitrc.org/projects/ci_outcome_pred/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Southern California <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="randomized-denoising-autoencoders-for-neuroimaging" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Randomized Denoising Autoencoders for Neuroimaging">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=randomized-denoising-autoencoders-for-neuroimaging" rel="bookmark" name="randomized-denoising-autoencoders-for-neuroimaging">Randomized Denoising Autoencoders for Neuroimaging</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The toolbox provides Matlab codes for learning randomized denoisiging autoencoders (rDA) based imaging marker for neuroimaing studies. rDA is an ensemble of neural networks (based on denoising autoencoders) that take imaging data as inputs and produce single/multi dimensional summary score. Parameters are learned from training data. rDA can be used for any learning task (classification, regression), for designing imaging disease markers. The unbiased and low variance of rDA's outputs are highly relevant for designing efficient clinical trials. Further details about the model are in the following paper (please cite it if the codes are used). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 16, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/rdacodes/">http://www.nitrc.org/projects/rdacodes/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of WIsconsin Madison <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="rapidart" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="RapidArt">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=rapidart" rel="bookmark" name="rapidart">RapidArt</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> RapidArt provides software for detecting artifacts and performing individual region-of-interest (ROI) based statistical analysis of fMRI data and enables users of fMRI technology to produce more detailed, consistent and reliable results. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 22, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MIT/X Consortium License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/rapidart/">http://www.nitrc.org/projects/rapidart/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="rapid-permutation-testing-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Rapid Permutation Testing Toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=rapid-permutation-testing-toolbox" rel="bookmark" name="rapid-permutation-testing-toolbox">Rapid Permutation Testing Toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> RapidPT is an open-source MATLAB toolbox for fast and efficient permutation testing. RapidPT is available as a stand-alone toolbox and can be downloaded here (nitrc) or from its Github page: <a href="https://github.com/felipegb94/RapidPT." target="_new">https://github.com/felipegb94/RapidPT.</a> For ease of use, RapidPT has also been integrated into the development version of SnPM. For a description of RapidPT and the usage within SnPM please refer to the documentation page: <a href="http://felipegb94.github.io/RapidPT/" target="_new">http://felipegb94.github.io/RapidPT/</a><br />
<br />
RapidPT is able to achieve substantial speedups over state of the art permutation testing implementations (e.g. SnPM) without compromising accuracy. For a thorough analysis of the scenarios were RapidPT performs best please refer to the paper:<br />
<br />
Accelerating Permutation Testing in Neuroimaging through Subspace Tracking: A new plugin for SnPM.<br />
F. Gutierrez-Barragan, V.K. Ithapu, C. Hinrichs, C. Maumet, S.C. Johnson, T.E. Nichols, V. Singh.<br />
Neuroimage, 2017. <a href="https://doi.org/10.1016/j.neuroimage.2017.07.025" target="_new">https://doi.org/10.1016/j.neuroimage.201...</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1310&screenshot_id=1225"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1310&screenshot_id=1225" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 27, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GPL-Style Open Source</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/rapidpt/">http://www.nitrc.org/projects/rapidpt/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Wisconsin-Madison <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="rapidtide" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="rapidtide">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=rapidtide" rel="bookmark" name="rapidtide">rapidtide</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The rapidtide suite is a set of python tools that we've developed and tested over the last several years in the McLean Hospital Opto Magnetic Group to perform rapid time delay analysis on functional imaging data to find time lagged correlations between the voxelwise time series and other time series. This technique can be used to detect, quantify, and display blood-borne physiological noise signals, as a method for quantifying hemodynamic parameters, and/or as a way to remove them from datasets to improve fMRI data quality. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1097&screenshot_id=1055"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1097&screenshot_id=1055" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 5, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/rapidtide/">http://www.nitrc.org/projects/rapidtide/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>McLean Hospital <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="rda2nifti" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="rda2nifti">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=rda2nifti" rel="bookmark" name="rda2nifti">rda2nifti</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> rda2nifti - useful MATLAB script for conversion from Siemens RDA file to NIFTI mask<br />
Dependencies: NIfTI_tools by Jimmy Sheen, MATLAB<br />
Credits to the author of read_rda102 function which has been already included in the ZIP file. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=991&screenshot_id=961"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=991&screenshot_id=961" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 1, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/rda2nifti/">http://www.nitrc.org/projects/rda2nifti/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Nencki Institute of Experimental Biology <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="recorder" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Recorder">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=recorder" rel="bookmark" name="recorder">Recorder</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=application" rel="tag">Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> BrainVision Recorder is a multifunctional recording software designed to provide our amplifier customers with a versatile and easy-to-use platform for recording setup and execution.<br />
<br />
•A wizard menu guides you through hardware setup and hard/software filter configuration. <br />
•Acquired data can be displayed in multiple ways. <br />
•Channel montages (original, bipolar, average) can be switched on the fly to adjust the channel view.<br />
•Channel by channel impedance check: Each electrode is placed at the topographic position and its impedance value is displayed with a color coding.<br />
•Acquisition parameters and impedance values are automatically stored and accessable any time. <br />
•An EP analysis can be performed in real-time; the segmented/averaged data can be stored together with the raw data. <br />
•A video module allows the capture of images of the subject; the video is synchronized with the EEG.<br />
•Data can be sent out to the network via the TCP/IP protocol (for real-time data analyses, e.g. in BrainVision RecView) 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1299&screenshot_id=1213"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1299&screenshot_id=1213" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 6, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Commercial License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://www.brainproducts.com/">https://www.brainproducts.com/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Brain Products GmbH <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>EEG Formats</td></tr>
			<tr class="even"><td>Output Formats</td><td>EEG Formats</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="recview" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="RecView">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=recview" rel="bookmark" name="recview">RecView</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=application" rel="tag">Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> BrainVision RecView is an advanced solution designed for real-time analysis of data received over the Ethernet network via TCP-IP directly from BrainVision Recorder.<br />
<br />
RecView at a glance<br />
•widely used in the EEG/fMRI field to remove both the gradient and the ballistocardiogram artifact (permitting experimental control during the scan)<br />
•The &quot;Template Drift Compensation&quot; algorithm remedies template jitter caused by imperfect synchronization between the EEG amplifier and the scanner clocks and thus ensures optimal data correction at any time.<br />
•uses the same history tree concept already implemented in BrainVision Analyzer<br />
•can also be used for FFT analysis, data filtering, mapping of the surface potentials as well as BCI and bio-/neurofeedback on the incoming data. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1300&screenshot_id=1216"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1300&screenshot_id=1216" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 6, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Commercial License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://www.brainproducts.com">https://www.brainproducts.com</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Brain Products GmbH <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>EEG Formats, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>EEG Formats, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="registry-builder-data-harmonization-and-aggregation-tool" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Registry Builder Data Harmonization and Aggregation Tool">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=registry-builder-data-harmonization-and-aggregation-tool" rel="bookmark" name="registry-builder-data-harmonization-and-aggregation-tool">Registry Builder Data Harmonization and Aggregation Tool</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data-resource" rel="tag">Data Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Remedy Informatics' platform aggregates data from multiple sources, harmonizes the data via Ontology, and provides data visualization and pattern recognition and querying tools. Our software helps harmonize the data that you have in different Excel files, databases, repositories, biospecimen applications, etc. and maps it to one common registry. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=705&screenshot_id=637"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=705&screenshot_id=637" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 8, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Mosaic License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.remedyinformatics.com/">http://www.remedyinformatics.com/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Remedy Informatics <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="repronim:-a-center-for-reproducible-neuroimaging-computation" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ReproNim: A Center for Reproducible Neuroimaging Computation">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=repronim:-a-center-for-reproducible-neuroimaging-computation" rel="bookmark" name="repronim:-a-center-for-reproducible-neuroimaging-computation">ReproNim: A Center for Reproducible Neuroimaging Computation</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=community" rel="tag">Community</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data-model" rel="tag">Data Model</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=platform-or-development-environment" rel="tag">Platform Or Development Environment</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ReproNim seeks to implement a shift in the way neuroimaging research is performed and reported. Through the development and implementation of technology that supports a comprehensive set of data management, analysis and utilization frameworks in support of both basic research and clinical activities, our overarching goal is to improve the reproducibility of neuroimaging science and extend the value of our national investment in neuroimaging research. Reproducibility is critical because the current literature is fraught with published results that are due to mistakes; or turn out to be false positive (contributed to by the lack of statistical power). More importantly, given the current publication system, it is exceedingly difficult to discern between false positive and true positive finding as data is hard to aggregate, and exact methods are hard to replicate. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1084&screenshot_id=1042"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1084&screenshot_id=1042" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 7, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://repronim.org">http://repronim.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="repronim-simple-reexecutable-publication" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ReproNim Simple Reexecutable Publication">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=repronim-simple-reexecutable-publication" rel="bookmark" name="repronim-simple-reexecutable-publication">ReproNim Simple Reexecutable Publication</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> In this project, we document a set of procedures, which include supplemental additions to a manuscript, that unambiguously define the data, workflow, execution environment and results of a neuroimaging analysis, in order to generate a verifiable re-executable publication. Re-executability provides a starting point for examination of the generalizability and reproducibility of a given finding.<br />
<br />
This project accompanies Ghosh SS, Poline JB, Keator DB et al. A very simple, re-executable neuroimaging publication. F1000Research 2017, 6:124 (doi: 10.12688/f1000research.10783.2) 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1258&screenshot_id=1190"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1258&screenshot_id=1190" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 9, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/simple1/">http://www.nitrc.org/projects/simple1/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>ReproNim <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="resource-ontology-discussion-group" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Resource Ontology Discussion Group">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=resource-ontology-discussion-group" rel="bookmark" name="resource-ontology-discussion-group">Resource Ontology Discussion Group</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=community" rel="tag">Community</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=ontology" rel="tag">Ontology</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This project will discuss, debate, develop and deploy ontological practices for the fMRI community. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 12, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/ontology/">http://www.nitrc.org/projects/ontology/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="rest:-a-toolkit-for-resting-state-fmri" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="REST: a toolkit for resting-state fMRI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=rest:-a-toolkit-for-resting-state-fmri" rel="bookmark" name="rest:-a-toolkit-for-resting-state-fmri">REST: a toolkit for resting-state fMRI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=cross-correlation" rel="tag">Cross-Correlation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=fourier-time-domain-analysis" rel="tag">Fourier Time-Domain Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=modeling" rel="tag">Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=resampling" rel="tag">Resampling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=spectral-analysis" rel="tag">Spectral Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=statistical-operation" rel="tag">Statistical Operation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=two-dimensional-display" rel="tag">Two Dimensional Display</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> REsting State fMRI data analysis Toolkit (REST) is a user-friendly convenient toolkit to calculate Functional Connectivity (FC), Regional Homogeneity (ReHo), Amplitude of Low-Frequency Fluctuation (ALFF), Fractional ALFF (fALFF), Gragner causality and perform statistical analysis. You also can use REST to view your data, perform Monte Carlo simulation similar to AlphaSim in AFNI, calculate your images, regress out covariates, extract ROI time courses, reslice images, and sort DICOM files. <br />
<br />
Most recent version and any questions could also be checked out in: <a href="http://www.restfmri.net." target="_new">http://www.restfmri.net.</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 31, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Academic Free License ("AFL")</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/rest/">http://www.nitrc.org/projects/rest/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>OS Independent<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="resting-state-fmri-from-healthy-young-subjects" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Resting-state fMRI from healthy young subjects">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=resting-state-fmri-from-healthy-young-subjects" rel="bookmark" name="resting-state-fmri-from-healthy-young-subjects">Resting-state fMRI from healthy young subjects</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This entry provides the original DICOM files of functional and structural MRI, obtained from resting-state healthy young subjects (100 males and 100 females), which were used in our research article (Donishi et al. 2017, <a href="https://doi.org/10.1002/brb3.890" target="_new">https://doi.org/10.1002/brb3.890</a>). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 1, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Wakayama_PsyRS</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/wakayama_psyrs/">http://www.nitrc.org/projects/wakayama_psyrs/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Wakayama Medical University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="resting-state-fmri-of-patients-with-pre-eclampsia-and-healthy-pregnant-women" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="resting-state fMRI of patients with pre-eclampsia and healthy pregnant women">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=resting-state-fmri-of-patients-with-pre-eclampsia-and-healthy-pregnant-women" rel="bookmark" name="resting-state-fmri-of-patients-with-pre-eclampsia-and-healthy-pregnant-women">resting-state fMRI of patients with pre-eclampsia and healthy pregnant women</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> resting-state fMRI data of patients with pre-eclapmsia and healthy pregnant women. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 7, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Wakayama_PE_study</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/fmri_pe_2012/">http://www.nitrc.org/projects/fmri_pe_2012/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Wakayama Medical University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="resting-state-hemodynamic-response-function-retrieval-and-deconvolution-(rs-hrf)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Resting State Hemodynamic Response Function Retrieval and Deconvolution (RS-HRF)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=resting-state-hemodynamic-response-function-retrieval-and-deconvolution-(rs-hrf)" rel="bookmark" name="resting-state-hemodynamic-response-function-retrieval-and-deconvolution-(rs-hrf)">Resting State Hemodynamic Response Function Retrieval and Deconvolution (RS-HRF)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=statistical-operation" rel="tag">Statistical Operation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=time-domain-analysis" rel="tag">Time Domain Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This toolbox is aimed to retrieve the onsets of pseudo-events triggering an hemodynamic response from resting state fMRI BOLD voxel-wise signal. It is based on point process theory, and fits a model to retrieve the optimal lag between the events and the HRF onset, as well as the HRF shape, using either the canonical shape with two derivatives, or a (smoothed) Finite Impulse Response.<br />
<br />
Once that the HRF has been retrieved for each voxel, it can be deconvolved from the time series (for example to improve lag-based connectivity estimates), or one can map the shape parameters everywhere in the brain (including white matter), and use it as a pathophysiological indicator.<br />
<br />
Input can be 3D or 4D nifti images, but also on time series matrices/vectors.<br />
The output are three HRF shape parameters for each voxel, plus the deconvolved time series, and the number of retrieved pseudo-events. All can be written back to nifti images.<br />
<br />
Please refer to the links on the left for the different versions and relative documentation. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1304&screenshot_id=1220"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1304&screenshot_id=1220" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 14, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD/MIT-Style Open Source License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://github.com/compneuro-da/rsHRF">https://github.com/compneuro-da/rsHRF</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Ghent University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="resting-state-meg-data-used-in-marek-et-al.,-2018,-plos-biology" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Resting state MEG data used in Marek et al., 2018, Plos Biology">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=resting-state-meg-data-used-in-marek-et-al.,-2018,-plos-biology" rel="bookmark" name="resting-state-meg-data-used-in-marek-et-al.,-2018,-plos-biology">Resting state MEG data used in Marek et al., 2018, Plos Biology</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Raw resting state MEG data used in Marek et al., 2018 Plos Biology. Dataset includes both raw resting state data (5 minutes), as well as raw empty room MEG data (2 minutes) in 89 subjects aged 14-31 years. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 10, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Academic Free License ("AFL")</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/rs_meg_2018/">http://www.nitrc.org/projects/rs_meg_2018/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Pittsburgh <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="resting-state-pediatric-imaging-template" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="resting-state pediatric imaging template">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=resting-state-pediatric-imaging-template" rel="bookmark" name="resting-state-pediatric-imaging-template">resting-state pediatric imaging template</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-data" rel="tag">Atlas Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Group ICA was used to generate spatial templates for 12 common resting-state networks in 62 typically-developing children, ages 9-15. We have made these available for those that will find them useful for masking and spatial template matching procedures. Basic demographic data on the sample is provided along with the protocol used to generate the templates. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 8, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Public Domain</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/r-spit/">http://www.nitrc.org/projects/r-spit/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="rex" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="REX">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=rex" rel="bookmark" name="rex">REX</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=time-domain-analysis" rel="tag">Time Domain Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> REX is a stand-alone MATLAB-based toolkit for the rapid and flexible exploration of ROI response waveforms and other signals from across large fMRI datasets. An alpha-release is currently available for use with an example dataset and tutorial. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 5, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.neuroimaging.org.au/nig/rex">http://www.neuroimaging.org.au/nig/rex</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="rft_fdr" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="RFT_FDR">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=rft_fdr" rel="bookmark" name="rft_fdr">RFT_FDR</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=modeling" rel="tag">Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=statistical-operation" rel="tag">Statistical Operation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-rendering" rel="tag">Surface Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=thickness" rel="tag">Thickness</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> So far there is a lack for Random Field Theory(RFT)-based multiple comparison correction for surfaces generated in Freesurfer software package. This set of Matlab-based functions can be used for that purpose. They are based on Worsley’s SurfStat toolbox. You also need to have installed Freesurfer software package and included the Freesurfer’s matlab subdirectory in the Matlab’s search path.<br />
<br />
In addition, this tool implements the RFT-FDR hierarchical correction that can be used for optimizing the amount of smoothing in cortical thickness analyses (Neuroimage 52, 158-171). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 17, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Academic Free License ("AFL")</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/rft_fdr/">http://www.nitrc.org/projects/rft_fdr/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>MGH/MGZ</td></tr>
			<tr class="even"><td>Output Formats</td><td>MGH/MGZ</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="rncut:-functional-brain-networks-parcellation" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="RNcut: functional brain networks parcellation">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=rncut:-functional-brain-networks-parcellation" rel="bookmark" name="rncut:-functional-brain-networks-parcellation">RNcut: functional brain networks parcellation</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The RNcut software freely disseminated below can be used for parcellating functional brain networks with resting-state BOLD fMRI datasets. This software package represents the improved version of conventional Ncut with two added regularization terms, smoothing term Ev and small-patch removal term Ep. The software package can generate robust and homogeneous functional parcellation of not only adult but also neonate brain networks. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 23, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>RNcut</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/rncut/">http://www.nitrc.org/projects/rncut/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Children's Hospital of Philadelphia/University of Pennsylvania <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="robust-biological-parametric-mapping" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Robust Biological Parametric Mapping">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=robust-biological-parametric-mapping" rel="bookmark" name="robust-biological-parametric-mapping">Robust Biological Parametric Mapping</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Biological parametric mapping (BPM) has extended the widely popular statistical parametric approach to enable application of the general linear model to multiple image modalities (both for regressors and regressands) along with scalar valued observations. This approach offers great promise for direct, voxelwise assessment of structural and functional relationships with multiple imaging modalities. However, as presented, the biological parametric mapping approach is not robust to outliers and may lead to invalid inferences (e.g., artifactual low p-values) due to slight mis-registration or variation in anatomy between subjects. To enable widespread application of this approach, we introduce robust regression and non-parametric regression in the neuroimaging context of application of the general linear model. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 30, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Lesser General Public License (LGPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/rbpm/">http://www.nitrc.org/projects/rbpm/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="robust-brain-extraction-(robex)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Robust Brain Extraction (ROBEX)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=robust-brain-extraction-(robex)" rel="bookmark" name="robust-brain-extraction-(robex)">Robust Brain Extraction (ROBEX)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ROBEX is an automatic whole-brain extraction tool for T1-weighted MRI data (commonly known as skull stripping). Whole-brain segmentation is often the first component in neuroimage pipelines and therefore, its robustness is critical for the overall performance of the system. Many methods have been proposed in the literature, but they often:<br />
- work well on certain datasets but fail on others.<br />
- require case-specific parameter tuning <br />
<br />
ROBEX aims for robust skull-stripping across datasets with no parameter settings. It fits a triangular mesh, constrained by a shape model, to the probabilistic output of a supervised brain boundary classifier. Because the shape model cannot perfectly accommodate unseen cases, a small free deformation is subsequently allowed. The deformation is optimized using graph cuts.<br />
<br />
The method ROBEX is based on was published in IEEE Transactions on Medical Imaging; please visit my website <a href="http://www.jeiglesias.com" target="_new">http://www.jeiglesias.com</a> to download the paper. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 11, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/robex/">http://www.nitrc.org/projects/robex/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UCLA <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="rockland-download-link-script" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Rockland Download Link Script">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=rockland-download-link-script" rel="bookmark" name="rockland-download-link-script">Rockland Download Link Script</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Script which points browser to NKI Rockland Sample </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 17, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dl_dataset/">http://www.nitrc.org/projects/dl_dataset/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="rodent-brain-extraction-tool" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Rodent Brain Extraction Tool">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=rodent-brain-extraction-tool" rel="bookmark" name="rodent-brain-extraction-tool">Rodent Brain Extraction Tool</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> A modified version of the Brain Extraction Tool (BET) that can process rodent brains. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=659&screenshot_id=577"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=659&screenshot_id=577" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 31, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://github.com/spinicist/QUIT">https://github.com/spinicist/QUIT</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>King's College London, Institute of Psychiatry <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="rodent-cortical-thickness-analysis" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Rodent Cortical Thickness Analysis">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=rodent-cortical-thickness-analysis" rel="bookmark" name="rodent-cortical-thickness-analysis">Rodent Cortical Thickness Analysis</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> RodentThickness is an automatic cortical thickness measurement tool for rat brains.<br />
<br />
The pipeline consists of four steps: preprocessing to create binary mask and label map, thickness measurement which produces laplacian field and thickness map in order, run particle correspondence followed by statistical analysis resulting in mean thickness color map and t-test result.<br />
<br />
By running RodentThickness, you will need to fill in informations in a Graphical User Interface, and then compute. You can also run the tool in command line without using the GUI.<br />
Using the GUI, you will be able to save or load a dataset file or a configuration file.<br />
<br />
The tool needs these other tools to work, so be sure to have these installed on your computer:<br />
<br />
 ImageMath<br />
 measureThicknessFilter<br />
 GenParaMeshCLP<br />
 ParaToSPHARMMeshCLP<br />
 ShapeWorksRun<br />
 ShapeWorksGroom<br />
 SegPostProcessCLP<br />
 BinaryToDistanceMap<br />
 MeshPointsIntensitysampling </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 26, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/rodentthickness/">http://www.nitrc.org/projects/rodentthickness/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="rosetta-bit" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Rosetta Bit">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=rosetta-bit" rel="bookmark" name="rosetta-bit">Rosetta Bit</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Rosetta Bit project will house public datasets that have been transcoded into multiple formats. This library of valid file format conversions (DICOM-&gt;NIFTI, DICOM-&gt;PAR/REC, etc.) will provide a reference for tool developers seeking to support multiple sources of data.<br />
<br />
Yvernault BC, Theobald CD Jr, Smith JC, Villalta V, Zald DH, Landman BA.<br />
Neuroinformatics. 2014 Oct;12(4):615-7. doi: 10.1007/s12021-014-9230-9.<br />
Validating DICOM Transcoding with an Open Multi-Format Resource.<br />
Pubmed: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24777387" target="_new">http://www.ncbi.nlm.nih.gov/pubmed/24777...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 17, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BIRN Data License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/rosetta/">http://www.nitrc.org/projects/rosetta/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Vanderbilt University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="r-package-for-adaptive-dwi-analysis" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="R-package for adaptive DWI analysis">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=r-package-for-adaptive-dwi-analysis" rel="bookmark" name="r-package-for-adaptive-dwi-analysis">R-package for adaptive DWI analysis</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The package dti provides methods for structural adaptive smoothing of diffusion weighted data in the context of the diffusion tensor model. Through its edge preserving properties they reduce data noise without compromizing significant structures. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 12, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/rdti/">http://www.nitrc.org/projects/rdti/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS<br>Microsoft<br>POSIX/UNIX-like</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>AFNI BRIK, ANALYZE, DICOM, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>AFNI BRIK, ANALYZE, DICOM, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="r-package-for-adaptive-fmri-analysis" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="R-package for adaptive fMRI analysis">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=r-package-for-adaptive-fmri-analysis" rel="bookmark" name="r-package-for-adaptive-fmri-analysis">R-package for adaptive fMRI analysis</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The package fmri provides fMRI analysis with R using structural adaptive smoothing methods. They allow smoothing especially at low SNR avoiding the apparent blurring of non-adapative smoothing and thus without reducing the effective spatial resolution. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 12, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/rfmri/">http://www.nitrc.org/projects/rfmri/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS<br>Microsoft<br>POSIX/UNIX-like</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>AFNI BRIK, ANALYZE, DICOM, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>AFNI BRIK, ANALYZE, DICOM, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="rt_image" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="RT_Image">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=rt_image" rel="bookmark" name="rt_image">RT_Image</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> RT_Image is an application developed in the Department of Radiation Oncology and MIPS at Stanford University to visualize, segment, and quantify three-dimensional images. Multiple datasets may be loaded, displayed, fused, processed, and quantitatively analyzed simultaneously. Data may be imported from any DICOM-compatible three dimensional imaging modality. Regions-of-interest may be defined using a number of manual, semi-automatic, and automated tools to segment three-dimensional pixel volumes. They may also be imported from and exported to DICOM structure sets. This software has been applied to preclinical and clinical computed tomography (CT), positron emission tomography (PET), single photon emission computed tomography (SPECT), magnetic resonance imaging (MRI), and optical imaging data. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 14, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/rt_image/">http://www.nitrc.org/projects/rt_image/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Stanford University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="ruby-nifti" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Ruby NIfTI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=ruby-nifti" rel="bookmark" name="ruby-nifti">Ruby NIfTI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Ruby NIfTI is a pure-ruby gem [library] for handling NIfTI data in the Ruby programming language.<br />
<br />
Ruby NIfTI currently supports basic read and write access to NIfTI files, including basic &amp; extended header information and image information. <br />
<br />
More information on ruby-nifti can be found on the github page; please use the Github Issue Tracker for any questions:<br />
<br />
<a href="https://github.com/brainmap/nifti" target="_new">https://github.com/brainmap/nifti</a><br />
<br />
INSTALLATION<br />
------------<br />
<br />
 gem install nifti<br />
<br />
or add this to your Gemfile and `bundle install`:<br />
<br />
 gem 'nifti', '~&gt;0.0.2'<br />
<br />
<br />
USAGE<br />
------<br />
<br />
Read file:<br />
<br />
 obj = NIFTI::NObject.new(&quot;T1.nii&quot;)<br />
<br />
Display some key information about the file:<br />
<br />
 puts obj.header['sform_code_descr']<br />
 =&gt; &quot;NIFTI_XFORM_SCANNER_ANAT&quot;<br />
<br />
Retrieve the pixel data in a Ruby Array:<br />
<br />
 image = obj.get_image<br />
<br />
<br />
Full Documentation is at: <br />
<br />
 <a href="http://rdoc.info/gems/nifti/frames" target="_new">http://rdoc.info/gems/nifti/frames</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 9, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Lesser General Public License (LGPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/ruby-nifti/">http://www.nitrc.org/projects/ruby-nifti/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Harvard University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="s3dl-sparse-dictionary-learning-based-mr-image-and-lesion-segmentation" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="S3DL Sparse Dictionary Learning based MR Image and Lesion Segmentation">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=s3dl-sparse-dictionary-learning-based-mr-image-and-lesion-segmentation" rel="bookmark" name="s3dl-sparse-dictionary-learning-based-mr-image-and-lesion-segmentation">S3DL Sparse Dictionary Learning based MR Image and Lesion Segmentation</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volumetric-analysis" rel="tag">Volumetric Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> S3DL (Subject Specific Sparse Dictionary Learning) is a software tool to generate whole brain segmentation as well as lesion segmentation from multi-contrast human brain MR images. It is a patch-based method, where similarities between patches from a subject and one or more atlases are exploited to create a segmentation of the subject.<br />
<br />
There are two parts of the tool, one to create multi-class segmentation from T1-w MR images, another to segment MS lesions from T1-w and FLAIR images. They can be used independently. In addition to the software, some training atlases and test data are provided.<br />
<br />
The toolbox is based on the following paper, <br />
S. Roy, Q. He, E. Sweeney, A. Carass, D. S. Reich, J.L. Prince, and D. L. Pham, &quot;Subject-Specific Sparse Dictionary Learning for Atlas-Based Brain MRI Segmentation&quot;, IEEE Journal of Biomedical and Health Informatics, 19(5):1598-1609, 2015. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1134&screenshot_id=1073"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1134&screenshot_id=1073" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 6, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/s3dl/">http://www.nitrc.org/projects/s3dl/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Henry Jackson Foundation <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="scalable-brain-atlas" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Scalable Brain Atlas">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=scalable-brain-atlas" rel="bookmark" name="scalable-brain-atlas">Scalable Brain Atlas</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-data" rel="tag">Atlas Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=three-dimensional-display" rel="tag">Three Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tractography" rel="tag">Tractography</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The INCF Scalable Brain Atlas (scalablebrainatlas.incf.org) is a web-based, interactive brain atlas viewer, containing a growing number of atlas templates for various species, including mouse, rat, macaque, marmoset, ferret and human. Standard features include fast brain region lookup, point and click to select a region and view its full 3D extent, mark a stereotaxic coordinate and view all regions in a hierarchy. Built-in extensions are the CoCoMac plugin to display Macaque connectivity, and a service to search for similar brain regions in other atlases or species. <br />
<br />
The SBA is designed to be customizable. External users can create plugins, hosted on their own servers, to interactively attach images or data to spatial atlas locations.<br />
<br />
In 2018, a new web-based application 'Scalable Brain Composer' is added, to display volumetric data and transparent meshes of brain regions together in 3D. Users can import their own data into this service to see how it fits within the atlas. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=434&screenshot_id=317"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=434&screenshot_id=317" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 10, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://scalablebrainatlas.incf.org">http://scalablebrainatlas.incf.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="schizconnect:-large-scale-schizophrenia-neuroimaging-data-mediation-&-federation" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="SchizConnect: Large-Scale Schizophrenia Neuroimaging Data Mediation & Federation">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=schizconnect:-large-scale-schizophrenia-neuroimaging-data-mediation-&-federation" rel="bookmark" name="schizconnect:-large-scale-schizophrenia-neuroimaging-data-mediation-&-federation">SchizConnect: Large-Scale Schizophrenia Neuroimaging Data Mediation & Federation</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database-application" rel="tag">Database Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-data-resource" rel="tag">Other Data Resource</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=websites" rel="tag">Websites</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Large-scale data sharing and integration is needed to further the state-of-the-art schizophrenia research, but is presently not possible due to practical limitations in the way in which data are being shared. SchizConnect is a data mediation and integration resource to overcome these limitations in a low-cost manner and deliver a web portal to interact with the federated databases. Access SchizConnect at <a href="http://schizconnect.org." target="_new">http://schizconnect.org.</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 19, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>SchizConnect</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.schizconnect.org/">http://www.schizconnect.org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Northwestern University Feinberg School of Medicine <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="scikit-learn" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="scikit-learn">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=scikit-learn" rel="bookmark" name="scikit-learn">scikit-learn</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=discriminant-analysis" rel="tag">Discriminant Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=independent-component-analysis" rel="tag">Independent Component Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=linear" rel="tag">Linear</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=modeling" rel="tag">Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=nonlinear" rel="tag">Nonlinear</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=principal-component-analysis" rel="tag">Principal Component Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> scikit-learn: machine learning in Python </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 25, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://scikit-learn.org/">http://scikit-learn.org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>INRIA <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS<br>POSIX/UNIX-like<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="scirun" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="SCIRun">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=scirun" rel="bookmark" name="scirun">SCIRun</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> SCIRun is a Problem Solving Environment (PSE), for modeling, simulation and visualization of scientific problems. SCIRun now includes the biomedical components formally released as BioPSE, as well as BioMesh3D. BioMesh3D is a free, easy to use program for generating quality meshes for the use in biological simulations. The most recent stable release is version 4.6. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 29, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MIT/X Consortium License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.scirun.org">http://www.scirun.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Utah <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="score" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="SCORE">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=score" rel="bookmark" name="score">SCORE</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> SCORE is a collection of methods for comparing the performance of different image algorithms. These methods generate quantitative scores that measure divergences to a standard (ground truth). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 7, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>3D Slicer License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/score/">http://www.nitrc.org/projects/score/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Utah <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="scp_taskfmri" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="SCP_TaskfMRI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=scp_taskfmri" rel="bookmark" name="scp_taskfmri">SCP_TaskfMRI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This repository holds the derived results of a task fMRI analysis software comparison project (Bowring et al, 2017). This project uses data from 3 carefully selected open source task fMRI datasets from OpenNeuro. Then, using the details in the published work, conducted three similar-as-possible parallel analyses in AFNI, FSL and SPM. The results are then all shared in the NeuroVault repository and all results are generated from freely available Python notebooks found <a href="https://github.com/NISOx-BDI/Software_Comparison_Analyses" target="_new">https://github.com/NISOx-BDI/Software_Co...</a> . The derived results comprise all first level and second results for AFNI, FSL and SPM parametric and nonparametric results.<br />
<br />
Bowring, A., Maumet, C., &amp; Nichols, T. (2018). Exploring the Impact of Analysis Software on Task fMRI Results. bioRxiv. <a href="https://www.biorxiv.org/content/early/2018/03/20/285585" target="_new">https://www.biorxiv.org/content/early/20...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 26, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Public Domain</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/scp_taskfmri/">http://www.nitrc.org/projects/scp_taskfmri/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Oxford <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="scralyze" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="SCRalyze">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=scralyze" rel="bookmark" name="scralyze">SCRalyze</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=modeling" rel="tag">Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-software-resource" rel="tag">Other Software Resource</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quantification" rel="tag">Quantification</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=time-domain-analysis" rel="tag">Time Domain Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> SCRalyze is a powerful software for model-based analysis of peripheral psychophysiology (e.g. skin conductance, heart rate, pupil size etc.). General linear modelling and dynamic causal modelling of these signals provide for inference on neural states/processes. SCRalyze includes flexible data import and display, statistical inference and results display and export. Easy programming of add-ons for new data formats, signal channels, and models. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=323&screenshot_id=215"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=323&screenshot_id=215" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 28, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://scralyze.sourceforge.net">http://scralyze.sourceforge.net</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="scribe" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Scribe">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=scribe" rel="bookmark" name="scribe">Scribe</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Scribe encodes papers to populate the BrainMap Database </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 5, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/scribe/">http://www.nitrc.org/projects/scribe/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="seecat" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="SeeCAT">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=seecat" rel="bookmark" name="seecat">SeeCAT</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Seed-based Connectivity Analysis Toolbox (SeeCAT) is developed in Matlab environment with GUI. The toolbox mainly includes four functional modules: 1) a preprocessing pipeline works with functional MRI data (work with SPM), which is enabled with fast parallel computing to improve processing speed; 2) a calculating module for seed-based structural or functional connectivity with various definitions of seeds; 3) a computation function for nodal degree in high-resolution functional brain networks (voxel-based) with flexible options, including selections for different thresholds, positive and negative connection, binary or weighted, anatomical distance, normalization, and smoothing; and 4) some useful tools for statistics, visualization and image calculation. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 16, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/seecat/">http://www.nitrc.org/projects/seecat/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Beijing Normal University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="seed-based-d-mapping-(sdm,-formerly-signed-differential-mapping)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Seed-based d Mapping (SDM, formerly Signed Differential Mapping)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=seed-based-d-mapping-(sdm,-formerly-signed-differential-mapping)" rel="bookmark" name="seed-based-d-mapping-(sdm,-formerly-signed-differential-mapping)">Seed-based d Mapping (SDM, formerly Signed Differential Mapping)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Seed-based d Mapping (formerly &quot;Signed Differential Mapping&quot;) is a statistical technique for meta-analyzing studies on differences in brain activity or structure that used neuroimaging techniques such as fMRI, VBM, DTI or PET. The methods have been fully validated in several studies, and meta-analyses using this method have been already published at the highest quality journals, such as Molecular Psychiatry, JAMA Psychiatry or the American Journal of Psychiatry. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=607&screenshot_id=506"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=607&screenshot_id=506" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 30, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.sdmproject.com/">http://www.sdmproject.com/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Translational Neuroimaging Group <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="seg3d" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Seg3D">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=seg3d" rel="bookmark" name="seg3d">Seg3D</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=three-dimensional-display" rel="tag">Three Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volume-rendering" rel="tag">Volume Rendering</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Seg3D is a free volume processing sgementing tool that combines a flexible manual interface with powerful image processing and segmentation algorithms. Users can explore and label image volumes using slice windows and 3D volume rendering. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 11, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MIT/X Consortium License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.seg3d.org">http://www.seg3d.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Utah <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, Nrrd, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, Nrrd, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="segediting" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="SegEditing">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=segediting" rel="bookmark" name="segediting">SegEditing</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> SegEditing is a segmentation editing tool using existing labels as references. It is useful when you would like to correct large errors with a few user interactions such as dots or rough scribbles using one or multiple reference labels of the target object. This toolbox includes an image viewer, graphical user interface, and semi-automatic algorithms for segmentation. The tool can support the following functions: Open medical images with various formats (such as mhd, hdr, nii), control intensity range in visualization, zoom in/out, show segmentation results, load and save user interactions and segmentation results, and so on. The software is developed by the IDEA group at the University of North Carolina at Chapel Hill (<a href="https://www.med.unc.edu/bric/ideagroup" target="_new">https://www.med.unc.edu/bric/ideagroup</a>).<br />
<br />
Windows operating system is required to run it. The executable file was compiled on Windows 7 (64 bit). The user interfaces and overall framework of the software were implemented in C++. Parallelization technologies were also used to speed up. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 27, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>ProCTSeg license</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/segediting/">http://www.nitrc.org/projects/segediting/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina at Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="segmentation-based-multimodal-rigid-image-registration" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Segmentation-based Multimodal Rigid Image Registration">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=segmentation-based-multimodal-rigid-image-registration" rel="bookmark" name="segmentation-based-multimodal-rigid-image-registration">Segmentation-based Multimodal Rigid Image Registration</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=intermodal" rel="tag">Intermodal</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This is an open-source Matlab function that allows rigid registration of 3D multi-modal images based on simultaneous segmentation, as introduced by Aganj and Fischl (2017). It is a modification of spm_coreg, and requires the publicly-available SPM12 toolbox. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 9, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License v3</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/sb-reg/">http://www.nitrc.org/projects/sb-reg/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Harvard Medical School / Massachusetts General Hospital <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="segmentation-validation-engine" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Segmentation Validation Engine">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=segmentation-validation-engine" rel="bookmark" name="segmentation-validation-engine">Segmentation Validation Engine</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=dice-coefficient" rel="tag">Dice Coefficient</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quality-metrics" rel="tag">Quality Metrics</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tanomoto-coefficient" rel="tag">Tanomoto Coefficient</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=web-service" rel="tag">Web Service</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Segmentation Validation Engine (SVE) provides an automated online framework for performing validation studies of skull-stripping methods. Registered users may download 40 T1 MRI volumes, skull-strip them with the algorithm of their choice, and upload their segmentation results to the SVE website (<a href="http://sve.bmap.ucla.edu" target="_new">http://sve.bmap.ucla.edu</a>), hosted at the UCLA Brain Mapping Center (<a href="http://www.bmap.ucla.edu" target="_new">http://www.bmap.ucla.edu</a>). The server will then compare the 40 skull-stripped results against a set of manually generated brain masks. The server computes a series of measures for the uploaded data, including Jaccard and Dice measures, and produces images of the spatial location of segmentation errors relative to a common space. Results are archived on the server, and the measures are viewable by visitors to the site. Details on the methodology can be found in:<br />
<br />
Shattuck DW, Prasad G, Mirza M, Narr KL, and Toga AW (2009) Online Resource for Validation of Brain Segmentation Algorithms, NeuroImage(doi:10.1016/j.neuroimage.2008.10.066). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 27, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>LONI/SVE Terms of Use</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://sve.bmap.ucla.edu/">http://sve.bmap.ucla.edu/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UCLA <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="self-threat-and-self-concept-data" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Self-threat and self-concept data">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=self-threat-and-self-concept-data" rel="bookmark" name="self-threat-and-self-concept-data">Self-threat and self-concept data</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The content includes the fMRI data of all of the 46 participants of the study mentioned above. You will be able to download the raw data, for preprocessing and analysing by yourself. Please be aware of the different conditions, which are mentioned in the connected paper and attached as logfiles in the repository. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 22, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/self01/">http://www.nitrc.org/projects/self01/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Graz <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>DICOM</td></tr>
			<tr class="even"><td>Output Formats</td><td>DICOM</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="serotonin-transporter-highresolution-pet-template" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Serotonin Transporter HighResolution PET Template">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=serotonin-transporter-highresolution-pet-template" rel="bookmark" name="serotonin-transporter-highresolution-pet-template">Serotonin Transporter HighResolution PET Template</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This PET template is the result of a recent study where we designed and validated an user-independent approach for a detailed in vivo quantification of serotonin transporter (5-HTT) availability in the human brainstem using a template-based approach. <br />
The template was generated with 3T-MR images and parametric binding potential (BPND) [11C]MADAM images of ten healthy subjects. Afterward on this template 5 brainstem regions of interest (Dorsal, Median and Caudal Raphe, Ventral Midbrain and Superior Colliculi) were delineated with a 3D Erosion approach. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 21, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/ki-5htt/">http://www.nitrc.org/projects/ki-5htt/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Karolinska Institutet, Department of Clinical Neuroscience <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="sfmproject" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="SFMProject">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=sfmproject" rel="bookmark" name="sfmproject">SFMProject</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Structure from motion algorithms repository. Common interface for various sfm algorithms. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 17, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/sfmproject/">http://www.nitrc.org/projects/sfmproject/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="shapeanalysismancova---spharm-tools" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="shapeAnalysisMANCOVA - SPHARM tools">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=shapeanalysismancova---spharm-tools" rel="bookmark" name="shapeanalysismancova---spharm-tools">shapeAnalysisMANCOVA - SPHARM tools</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=spherical-harmonics" rel="tag">Spherical Harmonics</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=statistical-operation" rel="tag">Statistical Operation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-analysis" rel="tag">Surface Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> shapeAnalysisMANCOVA offers statistical shape analysis based on a parametric boundary description (SPHARM) as the point-based model computing method. The point-based models will be analyzed with the methods here proposed using multivariate analysis of covariance (MANCOVA). Here, the number of variates being tested is the dimensionality of our observations. Each point of these observations is a three dimensional displacement vector from the mean. The number of contrasts is the number of equations involved in the null-hypothesis. In order to encompass varying numbers of variates and contrasts, and to account for independent variables, a matrix computation is performed. This matrix represents the multidimensional aspects of the correlation significance and it can be transformed into a scalar measure by manipulation of its eigenvalues.<br />
<br />
Details of the methods can be found in its Insight Journal publication: <a href="http://hdl.handle.net/10380/3124" target="_new">http://hdl.handle.net/10380/3124</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=294&screenshot_id=201"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=294&screenshot_id=201" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 16, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/shape_mancova/">http://www.nitrc.org/projects/shape_mancova/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina at Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS<br>POSIX/UNIX-like</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="shapecomplexatlas" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ShapeComplexAtlas">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=shapecomplexatlas" rel="bookmark" name="shapecomplexatlas">ShapeComplexAtlas</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This is a Matlab demo for constructing a neuro-anatomical shape complex atlas from 3D MRI brain structures, which is based on the paper &quot;Ting Chen, Anand Rangarajan, Stephan J. Eisenschenk and Baba C. Vemuri, Construction of a Neuroanatomical Shape Complex Atlas from 3D MRI Brain Structures. In NeuroImage, Volume 60, Page 1778-1787, 2012&quot;. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 8, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.cise.ufl.edu/~tichen/ShapeComplexAtlas.zip">http://www.cise.ufl.edu/~tichen/ShapeComplexAtlas.zip</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Florida <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="shapecomplexityindex" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ShapeComplexityIndex">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=shapecomplexityindex" rel="bookmark" name="shapecomplexityindex">ShapeComplexityIndex</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ShapeComplexityIndex tool calculate the local complexity as the difference between the observed distributions of local surface topology to its best-fit basic topology model within a given local kernel.<br />
<br />
The reference paper was published in <a href="http://proceedings.spiedigitallibrary.org/proceeding.aspx?articleid=2211519&amp;resultClick=1" target="_new">http://proceedings.spiedigitallibrary.or...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 6, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/shapecomplexity/">http://www.nitrc.org/projects/shapecomplexity/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina at Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="shapepopulationviewer" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ShapePopulationViewer">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=shapepopulationviewer" rel="bookmark" name="shapepopulationviewer">ShapePopulationViewer</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-analysis" rel="tag">Surface Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-rendering" rel="tag">Surface Rendering</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ShapePopulationViewer is a software that allows you to dynamically interact with multiple surfaces at the same time. It is very useful for visualisation and comparison of 3D surfaces by also displaying their scalars or vectors attributes stored in the points, and allowing the user to simply modify the colormap.<br />
ShapePopulationViewer is available as an extension of 3D Slicer (<a href="http://www.slicer.org" target="_new">http://www.slicer.org</a>) 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=759&screenshot_id=704"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=759&screenshot_id=704" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 25, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/shapepopviewer/">http://www.nitrc.org/projects/shapepopviewer/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina at Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="shapetools" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ShapeTools">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=shapetools" rel="bookmark" name="shapetools">ShapeTools</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The ShapeTools library is a collection of Java classes that enable Java programmers to model, manipulate and visualize geometric shapes and associated data values. It simplifies the creation of application programs by providing a ready-made set of support routines.<br />
<br />
ShapeTools are distributed by the Laboratory of Neuro Imaging (<a href="http://www.LONI.usc.edu/Software/ShapeTools" target="_new">http://www.LONI.usc.edu/Software/ShapeTo...</a>) at USC. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=299&screenshot_id=419"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=299&screenshot_id=419" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 8, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/shapetools/">http://www.nitrc.org/projects/shapetools/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Southern California <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="shapeviewer" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ShapeViewer">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=shapeviewer" rel="bookmark" name="shapeviewer">ShapeViewer</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The ShapeViewer (<a href="http://www.loni.usc.edu/Software/ShapeViewer" target="_new">http://www.loni.usc.edu/Software/ShapeVi...</a>) is a simple, portable geometry viewer that supports the file formats used by CCB researchers and provides their most commonly needed viewing functions. Since it is written in Java, it can run on a wide variety of computers.<br />
<br />
ShapeViewer is distributed by the Laboratory of Neuro Imaging (<a href="http://www.LONI.usc.edu/Software/ShapeViewer" target="_new">http://www.LONI.usc.edu/Software/ShapeVi...</a>) at USC. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=306&screenshot_id=418"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=306&screenshot_id=418" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 8, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/shapeviewer/">http://www.nitrc.org/projects/shapeviewer/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Southern California <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="shapeworks" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ShapeWorks">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=shapeworks" rel="bookmark" name="shapeworks">ShapeWorks</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=shape-analysis" rel="tag">Shape Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-rendering" rel="tag">Surface Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The ShapeWorks software is an open-source distribution of a new method for constructing compact statistical point-based models of ensembles of similar shapes that does not rely on any specific surface parameterization. The method requires very little preprocessing or parameter tuning, and is applicable to a wide range of shape analysis problems, including nonmanifold surfaces and objects of arbitrary topology. The proposed correspondence point optimization uses an entropy-based minimization that balances the simplicity of the model (compactness) with the accuracy of the surface representations. The ShapeWorks software includes tools for preprocessing data, computing point-based shape models, and visualizing the results. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=440&screenshot_id=319"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=440&screenshot_id=319" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 19, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MIT/X Consortium License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/shapeworks/">http://www.nitrc.org/projects/shapeworks/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="signalml" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="signalml">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=signalml" rel="bookmark" name="signalml">signalml</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Example implementation of SignaML metadescription of formats of biomedical time series, allowing to read data in new formats base upon standardized XML description.<br />
<br />
Also open platform for implementing advanced signal processing methods in user-friendly environment, at the moment interfacs for Java code, standalone executables and Matlab code via Matlab Builder for Java.<br />
<br />
<a href="http://signalml.org/" target="_new">http://signalml.org/</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1196&screenshot_id=1116"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1196&screenshot_id=1116" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 31, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://signalml.org/">http://signalml.org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="sigviewer" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="SigViewer">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=sigviewer" rel="bookmark" name="sigviewer">SigViewer</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> SigViewer is a viewing application for biosignals such as EEG or MEG time series. In addition to viewing raw data, SigViewer can also create, edit, and display event information (such as annotations or artifact selections). SigViewer supports many different biosignal data formats (including GDF, EDF, CNT, EEG, and many more). SigViewer also features basic signal processing modules such as computing the average over selected epochs (yielding event-related potentials) and power spectral densities of selected signals. SigViewer is written in standard C++ using the cross-platform graphical user interface (GUI) toolkit Qt. SigViewer is open source and available for all three major platforms (Windows, Mac OS X, and Linux). The application is licensed under the GNU GPL. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1110&screenshot_id=1066"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1110&screenshot_id=1066" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 30, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://github.com/cbrnr/sigviewer">https://github.com/cbrnr/sigviewer</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="simple-neurite-tracer" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Simple Neurite Tracer">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=simple-neurite-tracer" rel="bookmark" name="simple-neurite-tracer">Simple Neurite Tracer</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?tag=image-processing" rel="tag">Image Processing</a>, <a href="https://hbp-hpc-platform.fz-juelich.de/?tag=neurons" rel="tag">Neurons</a>, <a href="https://hbp-hpc-platform.fz-juelich.de/?tag=tracing" rel="tag">Tracing</a></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=plugin" rel="tag">Plugin</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description>
      Fiji's framework for semi-automated tracing of neurons and other tube-like structures.
    <p/><p/>
      This plugin is designed to allow easy semi-automatic tracing of neurons or other tube-like structures (e.g., blood vessels) through 3D image stacks. The plugin has built-in tools for analysis and hardware accelerated 3D visualization of the results. Data can be imported and exported in SWC files for interaction with other software, or details of the traces can be exported as CSV files for analysis in spreadsheets or statistical software. The native file format is open and XML-based. This plugin is Free Software (also known as Open Source software).
    </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>March 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GPLv3</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td>ImageJ</td></tr>
            <tr class="even"><td>Version of software</td><td>3.1.5</td></tr>
            <tr class="odd"><td>Version of documentation</td><td>3.1.3</td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://imagej.net/Simple_Neurite_Tracer">http://imagej.net/Simple_Neurite_Tracer</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href="https://github.com/fiji/Simple_Neurite_Tracer">https://github.com/fiji/Simple_Neurite_Tracer</a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Curtis Rueden <a href="mailto:"></a><br>Tiago Ferreira <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td>ImageJ, Java >= 1.8</td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>SWC, TRACES</td></tr>
			<tr class="even"><td>Output Formats</td><td>CSV, SWC, TRACES</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="simple-medical-imaging-library-interface-(smili)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Simple Medical Imaging Library Interface (SMILI)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=simple-medical-imaging-library-interface-(smili)" rel="bookmark" name="simple-medical-imaging-library-interface-(smili)">Simple Medical Imaging Library Interface (SMILI)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=animation" rel="tag">Animation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-display" rel="tag">Image Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=platform-or-development-environment" rel="tag">Platform Or Development Environment</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=shape-analysis" rel="tag">Shape Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-rendering" rel="tag">Surface Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volume-rendering" rel="tag">Volume Rendering</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Simple Medical Imaging Library Interface (SMILI), pronounced 'smilie', is an open-source, light-weight and easy-to-use medical imaging viewer and library for all major operating systems. The main sMILX application features for viewing n-D images, vector images, DICOMs, anonymizing, shape analysis and models/surfaces with easy drag and drop functions. It also features a number of standard processing algorithms for smoothing, thresholding, masking etc. images and models, both with graphical user interfaces and/or via the command-line. See our YouTube channel for tutorial videos via the homepage.<br />
<br />
The applications are all built out of a uniform user-interface framework that provides a very high level (Qt) interface to powerful image processing and scientific visualisation algorithms from the Insight Toolkit (ITK) and Visualisation Toolkit (VTK). The framework allows one to build stand-alone medical imaging applications quickly and easily, while having a multitude of processing and visualisations built-in. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=895&screenshot_id=877"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=895&screenshot_id=877" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 20, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://smili-project.sourceforge.net/">http://smili-project.sourceforge.net/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Queensland <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, NIfTI-1, Nrrd</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, NIfTI-1, Nrrd</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="simulated-diffusion-weighted-datasets" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Simulated Diffusion-Weighted Datasets">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=simulated-diffusion-weighted-datasets" rel="bookmark" name="simulated-diffusion-weighted-datasets">Simulated Diffusion-Weighted Datasets</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Simulated datasets generated using the framework described in the paper 'Realistic simulation of artefacts in diffusion MRI for validating post-processing correction techniques'. These can be used to validate your postprocessing correction techniques.<br />
<br />
Data containing susceptibility artefacts from the paper 'Quantitative assessment of the susceptibility artefact and its interaction with motion in diffusion MRI' is now also available for download.<br />
<br />
Code to simulate your own datasets can be found at:<br />
<a href="https://github.com/marksgraham/DW-POSSUM" target="_new">https://github.com/marksgraham/DW-POSSUM</a><br />
<br />
Available files:<br />
-SimulationFiles.zip contains DWI data with eddy-current and motion artefacts<br />
-Susceptibility datasets (static) contains DWIs with the susceptibility artefact<br />
-Susceptibility datasets (dynamic) contains data with the susceptibility artefact and motion (NB motion affects the susceptibility field)<br />
-Susceptibility datasets (other) contains a mask, T2-weighted images and a field-map for the static susceptibility data. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 12, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Public Domain</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/diffusionsim/">http://www.nitrc.org/projects/diffusionsim/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UCL <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="simulated-dw-mri-brain-data-sets-for-quantitative-evaluation-of-estimated-fiber-orientations" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Simulated DW-MRI Brain Data Sets for Quantitative Evaluation of Estimated Fiber Orientations">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=simulated-dw-mri-brain-data-sets-for-quantitative-evaluation-of-estimated-fiber-orientations" rel="bookmark" name="simulated-dw-mri-brain-data-sets-for-quantitative-evaluation-of-estimated-fiber-orientations">Simulated DW-MRI Brain Data Sets for Quantitative Evaluation of Estimated Fiber Orientations</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=test-data" rel="tag">Test Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tractography" rel="tag">Tractography</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This resource provides simulated DW-MRI brain images and quantitative tools for evaluating the performance of diffusion analysis methods in terms of fiber orientation estimation and false-positive/-negative fiber rates, which are of fundamental importance to tractography based studies. The data and tools aim to encourage greater understanding of the fiber estimation abilities of existing methods by use of standardized metrics for evaluation and comparison, and help development of improved diffusion analysis methods.<br />
<br />
DW data was generated using a multi-tensor model at SNRs of 9, 18 and 36, for sets of 20, 30, 40, 60, 90 and 120 gradient directions. For each combination of SNR and gradient direction set, 10 realizations of data are provided. All data is simulated with a diffusion-weighting of b=1000, as is common for clinical acquisitions.<br />
<br />
&gt;&gt; See &quot;Docs&quot; for all documentation and citation information.<br />
&gt;&gt; See &quot;Downloads&quot; for data and quantitative tools.<br />
<br />
&gt;&gt; Published in NeuroImage (Vol 109, April 2015). 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=769&screenshot_id=693"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=769&screenshot_id=693" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 12, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Data License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/sim_dwi_brain/">http://www.nitrc.org/projects/sim_dwi_brain/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Southern California (USC) <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="single-index-varying-coefficient-model-for-functional-responses" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Single-index Varying Coefficient Model for Functional Responses">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=single-index-varying-coefficient-model-for-functional-responses" rel="bookmark" name="single-index-varying-coefficient-model-for-functional-responses">Single-index Varying Coefficient Model for Functional Responses</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The aim of this tool is to implement a functional analysis pipeline, for the joint analysis of functional data and clinical data, for example age, gender and disease status. SIVC consists of a functional single index for characterizing the association of functional response with covariates of interest by incorporating complex spatial-temporal correlation structure, an efficient method for spatially smoothing varying coefficient functions, an estimation method for estimating the<br />
spatial-temporal correlation structure, a simultaneous confidence band for quantifying the uncertainty in the estimated coefficient functions. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 3, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Academic Free License ("AFL")</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/sivc/">http://www.nitrc.org/projects/sivc/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Department of Biostatistics <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="single-subject-resting-state-fmri-reproducibility-resource" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Single-subject Resting state fMRI Reproducibility Resource">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=single-subject-resting-state-fmri-reproducibility-resource" rel="bookmark" name="single-subject-resting-state-fmri-reproducibility-resource">Single-subject Resting state fMRI Reproducibility Resource</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> We have acquired a longitudinal single-subject dataset of a healthy volunteer (40 years of age at time of initial scan; male). A total of 156 sessions of MRI data was acquired on a weekly basis, over a span of 185 weeks (a little over 3 years).<br />
 <br />
The subject was scanned on a 3T Philips Achieva scanner (Philips HealthCare, Best, Netherlands). Rs-fMRI data of the subject was acquired using a multi-slice SENSE-EPI pulse sequence with TR/TE = 2000/30 ms, SENSE factor = 2, flip angle = 75°, 37 axial slices, nominal resolution = 3x3x3 mm3, 1 mm gap, 16 channel neuro-vascular coil, number of dynamics (frames) per run = 200. The subject was instructed to stay as still as possible with his eyes closed during the entire scan, and no other instruction was provided.<br />
<br />
(Please see the MediaWiki page for additional information) </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 29, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/kirbyweekly/">http://www.nitrc.org/projects/kirbyweekly/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Johns Hopkins University School of Medicine <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="sinomo-(singular-node-motifs)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="SINOMO (SIngular NOde MOtifs)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=sinomo-(singular-node-motifs)" rel="bookmark" name="sinomo-(singular-node-motifs)">SINOMO (SIngular NOde MOtifs)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Network nodes can be described by node-motifs--a combination of local network features. Certain node-motifs, such as highly connected nodes or hubs, have been shown to be important components of networks. Costa et al. (2009) have presented a technique to detect and specify more complex compound motifs, which are characterised by multiple features in combination. We described improvements to that method and showed how its parameters can be determined automatically [Echtermeyer et al. 2011]. SIMONO is our implementation of the enhanced workflow, which can be controlled via a graphical user interface or through the command-line for batch processing.<br />
<br />
Documentation is available at:<br />
<a href="http://www.biological-networks.org/p/sinomo/" target="_new">http://www.biological-networks.org/p/sin...</a><br />
You can also directly download the Matlab/Octave code following the link below.<br />
<br />
References:<br />
* L. D. F. Costa, F. A. Rodrigues, C. C. Hilgetag, and M. Kaiser, Europhys. Lett., 87, 1, 2009<br />
* C. Echtermeyer, L. Da Fontoura Costa, F. A. Rodrigues, M. Kaiser, PLoS ONE 6, 9, 2011 </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 28, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://www.biological-networks.org/pubs/suppl/sinomo/index.html">https://www.biological-networks.org/pubs/suppl/sinomo/index.html</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Newcastle University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="sinoplan" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Sinoplan">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=sinoplan" rel="bookmark" name="sinoplan">Sinoplan</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Sinoplan is a powerful software designed for planning of stereotactic implantation surgery, such as depth electrodes(SEEG) or DBS, It is also an useful neuroimaging tool.<br />
Main functions:<br />
Multi-model image fusion.<br />
Automatic image registration;<br />
3D reconstruction and visualization of Vessel, skin, cranial and bone; <br />
Easily trajectory planning and viewing, 8-16pins electrode avaliable.<br />
Calibration of the parameter of stereotactics frame, support both crw and leksell. <br />
<br />
Working environment:<br />
System: Windows 10 (64bits)<br />
CPU: Intel Core i7<br />
Memory: 8G<br />
Graphic: Nvidia GeForce GTX1050M or higher is recommended<br />
Network connection required.<br />
<br />
Attention: <br />
For intellectual property protection reason, this software must be worked in network environment and online verification may be required during operation. Users can use most of the software's features and register via email to access all functions.<br />
If have any questions or want to report bugs, feel free to leave us messages through forum or contact <a href="mailto:lz@sinovationmed.com" target="_new">lz@sinovationmed.com</a>. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1338&screenshot_id=1249"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1338&screenshot_id=1249" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 24, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/sinoplan_2018/">http://www.nitrc.org/projects/sinoplan_2018/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="sivic" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="SIVIC">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=sivic" rel="bookmark" name="sivic">SIVIC</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> SIVIC is an open-source, standards-based software framework and application suite for processing and visualization of DICOM MR Spectroscopy data. Through the use of DICOM, SIVIC aims to facilitate the application of MRS in medical imaging studies. Additional project information, binary downloads, documentations and tutorials are available at (<a href="https://sourceforge.net/projects/sivic/" target="_new">https://sourceforge.net/projects/sivic/</a>). 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1064&screenshot_id=1009"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1064&screenshot_id=1009" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 29, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/svk/">http://www.nitrc.org/projects/svk/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of California, San Francisco <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="skullstrippingtoolkit" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="SkullStrippingToolkit">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=skullstrippingtoolkit" rel="bookmark" name="skullstrippingtoolkit">SkullStrippingToolkit</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> A matlab-based code for skull stripping on infant and adult MR images.<br />
<br />
Please refer to below papers for details:<br />
<br />
&quot;LABEL: Pediatric Brain Extraction Using Learning-based Meta-algorithm&quot;, Neuroimage 62(3):1975–1986, Sep. 2012. [Feng Shi, Li Wang, Yakang Dai, John H Gilmore, Weili Lin, Dinggang Shen] </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 23, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>iBEAT</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/skulltoolkit/">http://www.nitrc.org/projects/skulltoolkit/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="slep:-sparse-learning-with-efficient-projections" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="SLEP: Sparse Learning with Efficient Projections">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=slep:-sparse-learning-with-efficient-projections" rel="bookmark" name="slep:-sparse-learning-with-efficient-projections">SLEP: Sparse Learning with Efficient Projections</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> SLEP contains a collection of algorithms for solving sparse learning problems, including LASSO, group LASSO, sparse group LASSO, hierarchical tree structure, and trace norm regularization. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 21, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License v3</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/slep/">http://www.nitrc.org/projects/slep/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="slic:-a-whole-brain-parcellation-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="SLIC: a whole brain parcellation toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=slic:-a-whole-brain-parcellation-toolbox" rel="bookmark" name="slic:-a-whole-brain-parcellation-toolbox">SLIC: a whole brain parcellation toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-data" rel="tag">Atlas Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quality-metrics" rel="tag">Quality Metrics</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=test-data" rel="tag">Test Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=two-dimensional-display" rel="tag">Two Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volumetric-analysis" rel="tag">Volumetric Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The SLIC toolbox contains five whole brain parcellation approaches that operates on resting-state fMRI data. Three of them are reproduced from the Ncut-based approaches in (Craddock et al., 2012, HBM) and (Shen et al., 2013, Neuroimage). The remaining two are the mean SLIC and two-level SLIC approaches that integrate Ncut and SLIC. <br />
<br />
The release includes a demo which reproduces the experiments in the paper (<a href="https://doi.org/10.3389/fnhum.2016.00659" target="_new">https://doi.org/10.3389/fnhum.2016.00659</a>), and atlases generated based on 190 subjects from the Beijing_Zang dataset. Matlab scripts, a set of test data and some instructions are included in the release. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1030&screenshot_id=1043"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1030&screenshot_id=1043" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 1, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/slic/">http://www.nitrc.org/projects/slic/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Southeast University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="slice:drop" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Slice:Drop">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=slice:drop" rel="bookmark" name="slice:drop">Slice:Drop</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description><a href="http://slicedrop.com" target="_new">http://slicedrop.com</a><br />
<br />
Slice:Drop is a viewer for medical imaging data. It supports a variety of scientific file formats out-of-the-box (see <a href="https://github.com/xtk/X/wiki/X:Fileformats" target="_new">https://github.com/xtk/X/wiki/X:Fileform...</a> for a complete list).<br />
<br />
We think that the best way to render your files is without any necessary conversions. Just drop'em on a website and they are ready to render.<br />
<br />
Getting started: Just drag'n'drop some medical imaging files on this website or try one of the four examples in the right corner. Then, play with the panels on the left and click, drag and rotate the 3d content.<br />
<br />
Slice:Drop uses WebGL and HTML5 Canvas to render the data in 2D and 3D. We use our own open-source toolkit to perform the rendering, called XTK ( <a href="http://goxtk.com" target="_new">http://goxtk.com</a> ). 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=635&screenshot_id=555"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=635&screenshot_id=555" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 11, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MIT/X Consortium License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://slicedrop.com">http://slicedrop.com</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Boston Children's Hospital <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ASCII, DICOM, MGH/MGZ, NIfTI-1, Nrrd, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ASCII, DICOM, MGH/MGZ, NIfTI-1, Nrrd, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="slicer3-example-modules" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Slicer3 Example Modules">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=slicer3-example-modules" rel="bookmark" name="slicer3-example-modules">Slicer3 Example Modules</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=application" rel="tag">Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=platform-or-development-environment" rel="tag">Platform Or Development Environment</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Example Slicer3 plugins that can be built against a Slicer3 build or a Slicer3 installation.<br />
<br />
Note: these are for 3D Slicer version 3. There is now a version 4 of 3D Slicer available. Information about extensions for version 4 can be found at the following links:<br />
<br />
<a href="http://www.slicer.org/slicerWiki/index.php/Documentation/Nightly/SlicerApplication/ExtensionsManager" target="_new">http://www.slicer.org/slicerWiki/index.p...</a><br />
<br />
<a href="http://www.slicer.org/slicerWiki/index.php/Documentation/Nightly/Developers/Tutorials/BuildTestPackageDistributeExtensions" target="_new">http://www.slicer.org/slicerWiki/index.p...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 31, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/slicer3examples/">http://www.nitrc.org/projects/slicer3examples/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows NT/2000<br>Windows Vista</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="slicer3-module-rician-noise-filter" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Slicer3 Module Rician noise filter">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=slicer3-module-rician-noise-filter" rel="bookmark" name="slicer3-module-rician-noise-filter">Slicer3 Module Rician noise filter</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=application" rel="tag">Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Two Slicer3 modules removing rician noise in diffusion tensor MRI </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 15, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dtiricianrem/">http://www.nitrc.org/projects/dtiricianrem/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="slicersalt" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="SlicerSALT">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=slicersalt" rel="bookmark" name="slicersalt">SlicerSALT</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> SlicerSALT (Shape AnaLysis Toolbox, SALT) continues the maintenance, expansion, validation and dissemination of the SPHARM-PDM (SPherical HARMonics Point Distributed Models) shape analysis toolbox. It is distributed as a tailor-made lighter version of 3DSlicer.<br />
<br />
SPHARM-PDM is a popular open-source tool that has been used by the biomedical research community to analyze the morphology of anatomical structures. SlicerSALT will maintain the core functionalities of SPHARM-PDM, as well as dramatically extending the toolbox into a general shape analysis toolbox. Novel, robust and validated methodologies will provide support for objects of non-spherical topology, skeletal models, four-dimensional models, and study-specific optimal model correspondence. SALT is being tested three application case scenarios to evaluate the efficacy of the proposed methods. <br />
<br />
SlicerSALT will continue providing excellent documentation, community resources, and user support. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1167&screenshot_id=1105"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1167&screenshot_id=1105" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 11, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD/MIT-Style Open Source License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://salt.slicer.org">http://salt.slicer.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Kitware Inc <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="smac:-spatial-multi-category-angle-based-classifier" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="SMAC: Spatial Multi-category Angle-based Classifier">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=smac:-spatial-multi-category-angle-based-classifier" rel="bookmark" name="smac:-spatial-multi-category-angle-based-classifier">SMAC: Spatial Multi-category Angle-based Classifier</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> We propose a novel Spatial Multi-category Angle-based Classifier (SMAC) for the efficient identification of biomarkers in the imaging data. The proposed SMAC not only utilizes the spatial structure of high-dimensional imaging data, but also handles both binary and multi-category classification problems. Besides, the SMAC also include the functions to deal with imaging based regression problems, with spatial and sparse penalty available. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 12, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BIAS License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/smac/">http://www.nitrc.org/projects/smac/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="small-effect-size-leads-to-reproducibility-failure-in-resting-state-fmri-studies" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Small effect size leads to reproducibility failure in resting-state fMRI studies">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=small-effect-size-leads-to-reproducibility-failure-in-resting-state-fmri-studies" rel="bookmark" name="small-effect-size-leads-to-reproducibility-failure-in-resting-state-fmri-studies">Small effect size leads to reproducibility failure in resting-state fMRI studies</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The amplitude of low frequency fluctuation (ALFF), regional homogeneity (ReHo), and degree centrality (DC) images of 5 datasets: PD-off, PD-on, ASD, MF, and EOEC were shared. Please see The amplitude of low frequency fluctuation (ALFF), regional homogeneity (ReHo), and degree centrality (DC) images of 5 datasets: PD-off, PD-on, ASD, MF, and EOEC Please see Jia et al. 2018 for the details about preprocessing and metrics calculation.<br />
<br />
Jia, Xi-Ze; Zhao, Na; Barton, Barek; Burciu, Roxana; Carriere, Nicolas; Cerasa, Antonio et al. (2018): Small effect size leads to reproducibility failure in resting-state fMRI studies. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 20</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Data License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/meta_metrics/">http://www.nitrc.org/projects/meta_metrics/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Hangzhou Normal University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="snpm" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="SnPM">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=snpm" rel="bookmark" name="snpm">SnPM</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=linear" rel="tag">Linear</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Statistical nonParametric Mapping toolbox provides an extensible framework for voxel level non-parametric permutation/randomisation tests of functional Neuroimaging experiments with independent observations. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=35&screenshot_id=800"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=35&screenshot_id=800" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 29, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Public Domain</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://warwick.ac.uk/snpm">http://warwick.ac.uk/snpm</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Warwick <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>SunOS/Solaris<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="sock" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="SOCK">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=sock" rel="bookmark" name="sock">SOCK</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=application" rel="tag">Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> SOCK (Spatially Organized Component Klassifikator) is a software toolbox that can automatically identify many of the artifact components that are often present in independent component analysis (ICA) of functional MRI (fMRI). The method:<br />
• Does not require temporal information about the fMRI paradigm.<br />
• Does not require the user to train the algorithm.<br />
• Requires only the EPI images (additional acquisition of anatomical images is not required).<br />
• Is able to identify a high proportion of artifact-related ICs without removing components that are likely to be of neuronal origin.<br />
• Can be applied to resting-state fMRI.<br />
• Is automated, requiring minimal or no human intervention.<br />
<br />
The method is described in the following paper:<br />
 Bhaganagarapu K, Jackson GD, Abbott DF. <br />
 An automated method for identifying artifact in <br />
 Independent Component Analysis of resting-state fMRI. <br />
 Frontiers in Human Neuroscience 7(343):1-16 (2013). <br />
 <a href="http://dx.doi.org/10.3389/fnhum.2013.00343" target="_new">http://dx.doi.org/10.3389/fnhum.2013.003...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 15, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/sock/">http://www.nitrc.org/projects/sock/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Florey Institute of Neuroscience and Mental Health, Melbourne, Australia <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="solar-eclipse-imaging-genetics-tools" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Solar Eclipse Imaging Genetics tools">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=solar-eclipse-imaging-genetics-tools" rel="bookmark" name="solar-eclipse-imaging-genetics-tools">Solar Eclipse Imaging Genetics tools</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=genetic-association" rel="tag">Genetic Association</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> We are developing software tools optimized for performing univariate and multivariate imaging genetics analyses while providing practical correction strategies for multiple testing.<br />
<br />
Official beta SOLAR Eclipse Version 8.3.X, adds multiple developments. This includes: Empirical pedigree from plink files, fast heritability (FPHI) and GWAS (NINGA) updates for CPU/GPU computing, more accurate p-value approximation for fast inference, polyclass_normalize function for multi-site homogenization and mega-analysis. <br />
<br />
Upcoming implements: CPU/GPU-based voxel-wise GWA using new HDF5 file format and cluster based statistical inference based on permutations.<br />
<br />
Source codes are distributed as a tarball or at github.com/kochunov/solar-eclipse<br />
<br />
The changes are described at <a href="http://solar-eclipse-genetics.org/" target="_new">http://solar-eclipse-genetics.org/</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 29, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GPL-Style Open Source</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://solar-eclipse-genetics.org">http://solar-eclipse-genetics.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Maryland and Texas Biomedical Foundation <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows XP</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>GIfTI, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>GIfTI, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="source-information-flow-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Source Information Flow Toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=source-information-flow-toolbox" rel="bookmark" name="source-information-flow-toolbox">Source Information Flow Toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=directed-transfer-analysis" rel="tag">Directed Transfer Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=granger-causality" rel="tag">Granger Causality</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=partial-directed-coherence" rel="tag">Partial Directed Coherence</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Source Information Flow Toolbox (SIFT) is an GUI-enabled EEGLAB plugin for modeling and visualizing dynamical interactions between electrophysiological signals (EEG, ECoG, MEG, etc), preferably after transforming signals into the source domain. The toolbox consists of four modules: (1) Data Preprocessing, (2) Model Fitting and Connectivity Estimation, (3) Statistical Analysis, (4) Visualization, with a fifth Group Analysis module in development. Module 2 currently includes several adaptive multivariate autoregressive modeling (AMVAR) algorithms, including segmentation AMVAR and Kalman filtering. This subsequently allows the user to validate the model and estimate (in the time-frequency domain) a wide range of multivariate Granger-causal and coherence measures published to date. Module 3 includes routines for parametric and non-parametric significance testing. Module 4 contains routines for interactive visualization of dynamical interactions across time, frequency and anatomical source location. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=549&screenshot_id=478"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=549&screenshot_id=478" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 15, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://sccn.ucsd.edu/wiki/SIFT">http://sccn.ucsd.edu/wiki/SIFT</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of California, San Diego <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="spanish-resting-state-network" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Spanish Resting State Network">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=spanish-resting-state-network" rel="bookmark" name="spanish-resting-state-network">Spanish Resting State Network</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Spanish Resting State Network (SRSN); Foro para compartir datos y conocimiento sobre esta red. Se constituye el Spanish Resting State Network como una colaboracion entre distintos grupos de investigacion de España y centros nacionales e internacionales. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=387&screenshot_id=261"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=387&screenshot_id=261" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 21, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/srsn/">http://www.nitrc.org/projects/srsn/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Microsoft Word Document, NIfTI-1, PDF - Portable Document Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>Microsoft Word Document, NIfTI-1, PDF - Portable Document Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="sparse-sampling-of-silence-database" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Sparse Sampling of Silence Database">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=sparse-sampling-of-silence-database" rel="bookmark" name="sparse-sampling-of-silence-database">Sparse Sampling of Silence Database</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Sparse Sampling of Silence </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 20, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/sparse_2018/">http://www.nitrc.org/projects/sparse_2018/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UNAM, CityU HK and JHU <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="spatial-analysis-3d" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Spatial Analysis 3D">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=spatial-analysis-3d" rel="bookmark" name="spatial-analysis-3d">Spatial Analysis 3D</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=position" rel="tag">Position</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=simulation" rel="tag">Simulation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=statistical-operation" rel="tag">Statistical Operation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Spatial Analysis 3D is a user-friendly, graphical user interface (GUI) that allows statistical and visual manipulations of real and simulated three-dimensional spatial point patterns. The analyses use files containing sets of X, Y, Z coordinates. These point patterns are frequently coordinates of cells of specific cell classes within in volumes of tissue derived from microscopy analyses. The analyses are scale independent so spatial analyses of coordinates from larger and smaller scale distributions are possible. The software can also generate sample sets of X, Y, Z coordinates for program exploration and modeling purposes. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=235&screenshot_id=140"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=235&screenshot_id=140" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 19, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/sa3d/">http://www.nitrc.org/projects/sa3d/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="spatially-constrained-parcellation" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Spatially Constrained Parcellation">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=spatially-constrained-parcellation" rel="bookmark" name="spatially-constrained-parcellation">Spatially Constrained Parcellation</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Source code and parcellations now available! Go to <a href="http://ccraddock.github.io/cluster_roi/" target="_new">http://ccraddock.github.io/cluster_roi/</a> for more information.<br />
<br />
Spatially constrained parcellation is a set of tools for deriving ROI atlases by whole brain clustering of task or resting state data. In addition to the tools, this resource also contains several atlases derived by parcellating publicly available resting state fMRI datasets. The initial release will include python scripts and ROI atlases developed to perform the analyses described in Craddock et. al., A whole brain fMRI atlas generated via spatially constrained spectral clustering, which is currently in revision in Human Brain Mapping. <br />
<br />
The scripts provide all of the tools necessary to derive an ROI atlases using spatially constrained Ncut spectral clustering. The scripts require python, numpy and scipy to run. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=427&screenshot_id=392"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=427&screenshot_id=392" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 30, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial Share Alike</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cluster_roi/">http://www.nitrc.org/projects/cluster_roi/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="spatial-statistical-parametric-mapping" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Spatial Statistical Parametric Mapping">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=spatial-statistical-parametric-mapping" rel="bookmark" name="spatial-statistical-parametric-mapping">Spatial Statistical Parametric Mapping</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> SSPM represents Spatial Statistical Parametric Mapping.This package includes two tools presently: MAGEE and FADTTS. <br />
MAGEE represents the Multiscale Adaptive Generalized Estimating Equation. It was developed specifically for analyzing multivariate neuroimaging data in 3-dimensional volume (or on 2-dimensional surface) from longitudinal neuroimaging studies. <br />
FADTTS represents Functional Analysis of Diffusion Tensor Tract Statistics. The aim of this tool is to implement a functional analysis pipeline, for delineating the structure of the variability of multiple diffusion properties along major white matter fiber bundles and their association with a set of covariates of interest, in various diffusion tensor imaging studies. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 26, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>NIH Data Access Policy</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/sspm/">http://www.nitrc.org/projects/sspm/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="spd-rsfmri-neu-zs" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="SPD-rsfMRI-NEU-ZS">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=spd-rsfmri-neu-zs" rel="bookmark" name="spd-rsfmri-neu-zs">SPD-rsfMRI-NEU-ZS</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> We provide MRI data of Schizotypal Personality Disorder(SPD) and Healthy controls. The total number of SPD subjects and HC are 36 (each 18). For each subject, T1 weighted and resting state functional MRI are included. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 9, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Academic Free License ("AFL")</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/spd_rsfmri/">http://www.nitrc.org/projects/spd_rsfmri/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Northeastern University and DaLian ZhongShan Hospital <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="spharm-mat" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="SPHARM-MAT">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=spharm-mat" rel="bookmark" name="spharm-mat">SPHARM-MAT</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=morphology" rel="tag">Morphology</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quantitative-shape-analysis" rel="tag">Quantitative Shape Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=spherical-harmonics" rel="tag">Spherical Harmonics</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-analysis" rel="tag">Surface Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> SPHARM-MAT is implemented based on a powerful 3D Fourier surface representation method called SPHARM, which creates parametric surface models using spherical harmonics. It is a matlab-based 3D shape modeling and analysis toolkit, and is designed to aid statistical shape analysis for identifying morphometric changes in 3D structures of interest related to different conditions. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=386&screenshot_id=249"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=386&screenshot_id=249" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 18, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.iu.edu/~spharm/">http://www.iu.edu/~spharm/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Indiana University School of Medicine <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>POSIX/UNIX-like<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="spharm-pdm-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="SPHARM-PDM Toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=spharm-pdm-toolbox" rel="bookmark" name="spharm-pdm-toolbox">SPHARM-PDM Toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quantitative-shape-analysis" rel="tag">Quantitative Shape Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=spherical-harmonics" rel="tag">Spherical Harmonics</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-analysis" rel="tag">Surface Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Shape analysis has become of increasing interest to the medical community due to its potential to precisely locate morphological changes between healthy and pathological structures. SPHARM-PDM is a tool that computes point-based models using a parametric boundary description for the computing of Shape analysis. <br />
<br />
Since July 2017, SPHARM-PDM is now available as a 3D Slicer extension (<a href="http://www.slicer.org" target="_new">http://www.slicer.org</a>) and as part of SlicerSALT (salt.slicer.org). Dissemination of SPHARM-PDM through SlicerSALT is now the major mode of dissemination for downloading new versions of SPHARM-PDM.<br />
<br />
For questions, please refer to the new SlicerSALT forum at <a href="https://discourse.slicer.org/c/community/slicer-salt" target="_new">https://discourse.slicer.org/c/community...</a><br />
<br />
Note: the modules available in the downloads section have not been updated since 2015 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=308&screenshot_id=1110"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=308&screenshot_id=1110" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 9, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/spharm-pdm/">http://www.nitrc.org/projects/spharm-pdm/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina at Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1, Nrrd, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1, Nrrd, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="spicodyn" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="SpiCoDyn">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=spicodyn" rel="bookmark" name="spicodyn">SpiCoDyn</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> SPICODYN is an automated and efficient open-source software for the analysis of multi-site neuronal spike signals. Accepted input data formats are HDF5, level 5 MAT and text files, containing recorded or generated time series spike signals data. SPICODYN processes such electrophysiological signals focusing on: spiking and bursting dynamics and functional-effective connectivity analysis. In particular, for inferring network connectivity, a new implementation of the transfer entropy (TE) method is presented dealing with multiple time delays (temporal extension) and with multiple binary patterns (high order extension). SPICODYN is specifically tailored to process data coming from different Multi-Electrode Arrays (MEAs) setups, guarantying, in those specific cases, automated processing. The optimized implementation of the Delayed Transfer Entropy (DTE) and the High-Order Transfer Entropy (HOTE) algorithms, allows performing accurate and rapid analysis on multiple spike trains from thousands of electrodes. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1212&screenshot_id=1260"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1212&screenshot_id=1260" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 14, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/spicodyn/">http://www.nitrc.org/projects/spicodyn/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>DIBRIS <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="spikecor:-fmri-tool-for-automated-correction-of-head-motion-spikes" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="SPIKECOR: fMRI tool for automated correction of head motion spikes">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=spikecor:-fmri-tool-for-automated-correction-of-head-motion-spikes" rel="bookmark" name="spikecor:-fmri-tool-for-automated-correction-of-head-motion-spikes">SPIKECOR: fMRI tool for automated correction of head motion spikes</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This algorithm corrects for &quot;spikes&quot; in fMRI data, typically caused by abrupt head motion during scanning. SPIKECOR identifies outliers using Principal Component Analysis (PCA) in a sliding time-window; it is sensitive to global motion artifact, and stable against non-stationary signal changes.<br />
Unlike typical &quot;scrubbing&quot; protocols it is automated, and identifies motion outliers using purely statistical criteria - no need to select arbitrary motion thresholds! It also replaces outlier volumes with interpolated values using cubic splines, which minimizes discontinuities in the data. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 27, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Lesser General Public License (LGPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/spikecor_fmri/">http://www.nitrc.org/projects/spikecor_fmri/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Rotman Research Institute <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="spinal-cord-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Spinal Cord Toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=spinal-cord-toolbox" rel="bookmark" name="spinal-cord-toolbox">Spinal Cord Toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Spinal Cord Toolbox is a comprehensive and open-source library of analysis tools for multi-parametric MRI of the spinal cord. The toolbox includes a template and several atlases (white and gray matter, spinal tracts, spinal and vertebral levels, etc.), along with state-of-the-art methods to register any data to the template. It also includes useful scripts for data preprocessing: extraction of centerline, automatic segmentation, slice-wise motion correction, etc. Check this out! 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=831&screenshot_id=791"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=831&screenshot_id=791" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 11, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MIT/X Consortium License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://sourceforge.net/projects/spinalcordtoolbox/">http://sourceforge.net/projects/spinalcordtoolbox/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Polytechnique Montreal, Université de Montréal <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="spinesegmentation-module-for-3dslicer" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="SpineSegmentation module for 3DSlicer">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=spinesegmentation-module-for-3dslicer" rel="bookmark" name="spinesegmentation-module-for-3dslicer">SpineSegmentation module for 3DSlicer</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> 3D Slicer module for automated segmentation of the spine. This is an implementation of a novel model-based segmentation algorithm. This work was presented at the NA-MIC Week in Salt Lake City, Jan 2010. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=244&screenshot_id=217"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=244&screenshot_id=217" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 3, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>3D Slicer License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://wiki.na-mic.org/Wiki/index.php/2010_Winter_Project_Week_Spine_Segmentation_Module_in_Slicer3">http://wiki.na-mic.org/Wiki/index.php/2010_Winter_Project_Week_Spine_Segmentation_Module_in_Slicer3</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="spm" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="SPM">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=spm" rel="bookmark" name="spm">SPM</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=forward---inverse" rel="tag">Forward - Inverse</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=haemodynamic-response" rel="tag">Haemodynamic Response</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=platform-or-development-environment" rel="tag">Platform Or Development Environment</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=rendering" rel="tag">Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=spatial-transformation" rel="tag">Spatial Transformation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=statistical-operation" rel="tag">Statistical Operation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=three-dimensional-display" rel="tag">Three Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=time-domain-analysis" rel="tag">Time Domain Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The SPM software package has been designed for the analysis of brain imaging data sequences. The sequences can be a series of images from different cohorts, or time-series from the same subject. The current release is designed for the analysis of fMRI, PET, SPECT, EEG and MEG. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 7, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.fil.ion.ucl.ac.uk/spm/">http://www.fil.ion.ucl.ac.uk/spm/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, GIfTI, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, GIfTI, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="spm-for-fnirs-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="SPM for fNIRS toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=spm-for-fnirs-toolbox" rel="bookmark" name="spm-for-fnirs-toolbox">SPM for fNIRS toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The SPM for fNIRS toolbox is the SPM12-based software for statistical analysis of functional near-infrared spectroscopy (fNIRS) signal. The toolbox allows for inferences about regionally specific hemodynamic effects with superresolution, by applying the general linear model and random field theory to fNIRS data. <br />
<br />
In the toolbox manual, we provide an illustrative statistical parametric mapping (SPM) analysis using fNIRS data acquired during Stroop task, and then describe detailed information about functions implemented in the toolbox.<br />
<br />
Manual: <a href="https://www.nitrc.org/docman/view.php/965/1995/manual_spm_fnirs.pdf" target="_new">https://www.nitrc.org/docman/view.php/96...</a><br />
<br />
Reference: <br />
Sensor space group analysis for fNIRS data. <br />
Tak, S, Uga, M, Flandin, G, Dan, I, Penny, WD, 2016. J. Neurosci. Meth. 264, 103-112. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 10, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/spm_fnirs/">http://www.nitrc.org/projects/spm_fnirs/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Wellcome Trust Centre for Neuroimaging <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="spm_ss---fmri-functional-localizers" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="SPM_SS - fMRI functional localizers">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=spm_ss---fmri-functional-localizers" rel="bookmark" name="spm_ss---fmri-functional-localizers">SPM_SS - fMRI functional localizers</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=linear" rel="tag">Linear</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=modeling" rel="tag">Modeling</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This spm-toolbox performs ROI-level and voxel-level between-subjects analyses of functional MRI data, restricting the analyses to those areas identified using subject-specific functional localizers. <br />
<br />
Methods: The toolbox implements ROI-level and voxel-level analyses, and it implements an automatic cross-validation procedure when the localizers are not orthogonal to the effects-of-interest. ROI-level analyses allow manually defined parcels of interest, as well as automatically-defined ones (GcSS procedure, Fedorenko et al. 2010). General linear model second-level analyses are implemented, including ReML and OLS estimation of population level effects. Hypothesis testing includes standard univariate tests as well as multivariate tests for mixed within- and between-subject designs (T, F, and Wilks' lambda statistics) <br />
<br />
This toolbox requires Matlab and SPM5/SPM8.<br />
<br />
<a href="http://web.mit.edu/evelina9/www/funcloc.html" target="_new">http://web.mit.edu/evelina9/www/funcloc....</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=415&screenshot_id=290"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=415&screenshot_id=290" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 6, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MIT/X Consortium License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://web.mit.edu/evelina9/www/funcloc.html">http://web.mit.edu/evelina9/www/funcloc.html</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>MIT <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="spm-workshop-hannover" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="SPM-Workshop Hannover">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=spm-workshop-hannover" rel="bookmark" name="spm-workshop-hannover">SPM-Workshop Hannover</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> In our workshops we offer a rich and pleasant environment to familiarize yourself with SPM and some useful extension under skilled supervision. We believe that the best way to learn how to operate SPM is by actually using it! <br />
<br />
We will hone your data analysis skills from preprocessing to group statistics. You will also train how to arrive at high-class graphical presentations of your results for use in your publications. Moreover you will benefit both from your own learning experience as well as from those of your peers.<br />
<br />
Go pro and learn how to analyze your own functional magnetic resonance imaging data from A to Z! </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 15, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>SPM-Workshop Hannover</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/spmws_hannover/">http://www.nitrc.org/projects/spmws_hannover/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="spot3d:-spatial-positioning-toolbox-for-head-markers-using-3d-scans" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="SPOT3D: Spatial positioning toolbox for head markers using 3D scans">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=spot3d:-spatial-positioning-toolbox-for-head-markers-using-3d-scans" rel="bookmark" name="spot3d:-spatial-positioning-toolbox-for-head-markers-using-3d-scans">SPOT3D: Spatial positioning toolbox for head markers using 3D scans</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> An accurate individual head model is fundamental for a reliable use of high-density electroencephalography (hdEEG) as brain imaging technique. Correct identification of sensor positions over the subject's scalp is thus necessary for precisely estimating neural activity. Recent studies suggest the use of colour-enhanced 3D scanner to acquire a three-dimensional point cloud of the whole surface of the subject's head.<br />
<br />
Here, we introduce the SPOT3D toolbox, which integrates a graphical user interface (GUI). This software performs alignment of the 3D scan in individual magnetic resonance (MR) space, sensor detection and labelling on the aligned scan, and positioning of facial landmarks. The editing tools accessible through the GUI allow the user to manually perform these processing steps from scratch, or to adjust their outcome after automated processing. User documentation specifies the software requirements and dependencies; sample datasets are also available.<br />
<br />
Contact: <a href="mailto:gaia.taberna@kuleuven.be" target="_new">gaia.taberna@kuleuven.be</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 29</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/spot3d/">http://www.nitrc.org/projects/spot3d/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>KU Leuven <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="s-rep-fitting,-statistics,-and-segmentation" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="S-rep Fitting, Statistics, and Segmentation">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=s-rep-fitting,-statistics,-and-segmentation" rel="bookmark" name="s-rep-fitting,-statistics,-and-segmentation">S-rep Fitting, Statistics, and Segmentation</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> S-reps are skeletal models for anatomic objects especially suited for computing probability distributions from populations of these objects and for providing object-related coordinates for the interior of these objects. They allow classification and hypothesis testing using their geometric features and physiological features derived from medical images. They also allow the definition of shape spaces, probability-based geometric typicality functions, and appearance models used for segmentation or registration. A variety of successful applications to objects in neuroimages have already been performed.<br />
<br />
The resource contains software to fit s-reps to segmented anatomic objects, to compute probability distributions on these s-reps, to train and to apply classifiers between two classes of anatomic objects, and to apply hypothesis testing to determine which geometric or physiological features vary significantly between two classes. Software for object segmentation from medical images may also be included. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=718&screenshot_id=647"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=718&screenshot_id=647" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 15, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/sreps/">http://www.nitrc.org/projects/sreps/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina at Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="sri24-atlas:-normal-adult-brain-anatomy" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="SRI24 Atlas: Normal Adult Brain Anatomy">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=sri24-atlas:-normal-adult-brain-anatomy" rel="bookmark" name="sri24-atlas:-normal-adult-brain-anatomy">SRI24 Atlas: Normal Adult Brain Anatomy</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data-model" rel="tag">Data Model</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> SRI24 is an MRI-based atlas of normal adult human brain anatomy, generated by template-free nonrigid registration from images of 24 normal control subjects.<br />
<br />
The atlas comprises T1, T2, and PD weighted structural MRI, tissue probability maps (GM, WM, CSF), maximum-likelihood tissue segmentation, DTI-based measures (FA, MD, longitudinal and transversal diffusivity), and two labels maps of cortical regions and subcortical structures.<br />
<br />
The atlas is provided at 1mm isotropic image resolution in Analyze, NIFTI, and Nrrd format. We are also providing an experimental packaging for use with SPM8.<br />
<br />
As per the CC-BY-SA license, please cite the following paper in publications using the SRI24 atlas:<br />
<br />
T. Rohlfing, N. M. Zahr, E. V. Sullivan, and A. Pfefferbaum, “The SRI24 multichannel atlas of normal adult human brain structure,” Human Brain Mapping, vol. 31, no. 5, pp. 798-819, 2010. DOI: 10.1002/hbm.20906<br />
<br />
Free article on PMC: <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2915788" target="_new">http://www.ncbi.nlm.nih.gov/pmc/articles...</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=214&screenshot_id=235"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=214&screenshot_id=235" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 2, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Share Alike</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/sri24/">http://www.nitrc.org/projects/sri24/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1, Nrrd</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1, Nrrd</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="sr-research-eyelink-eye-trackers" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="SR Research EyeLink Eye Trackers">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=sr-research-eyelink-eye-trackers" rel="bookmark" name="sr-research-eyelink-eye-trackers">SR Research EyeLink Eye Trackers</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=eye-tracking" rel="tag">Eye Tracking</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=response-monitoring" rel="tag">Response Monitoring</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> SR Research, makers of the world leading EyeLink High-Speed eye tracker line, have been developing advanced eye tracking technologies and serving world class support to our research user base since 1992. The EyeLink line provides eye tracking capabilities for behavioral labs as well as for MRI, MEG, and EEG environments. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=491&screenshot_id=369"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=491&screenshot_id=369" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 29, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Commercial License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.sr-research.com">http://www.sr-research.com</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>SR Research <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="stamp-atlases-for-brain-abnormality-detection" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="STAMP Atlases for Brain Abnormality Detection">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=stamp-atlases-for-brain-abnormality-detection" rel="bookmark" name="stamp-atlases-for-brain-abnormality-detection">STAMP Atlases for Brain Abnormality Detection</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> STandardized Abnormality-free Multi-Parametric (STAMP) atlases can help detect various lesions (e.g., MS lesions, stroke lesions, primary/recurrent tumors, post-surgery cavity, etc). <br />
<br />
STAMP atlases include 4 modalities (T1, T2, T1c, FLAIR) and 1 artificial modality (T1d:=T1c-T1, highlighting pure contrast). Here T1c is T1 MRI with gadolinium contrast. Multi-modal atlases help detect lesion components. The otherwise rarely available T1c/T1d atlases can help differentiate between normal vessels and enhancing tumor, both enhancing upon gadolinium injection.<br />
<br />
The STAMP atlases quantify the normal intensity variations at the voxel level, which adds to those at the image or tissue level. This may help abnormality detection, because normal variations and hence abnormal detection may depend on anatomical locations.<br />
<br />
The STAMP atlas can also be used to standardize intensity, to spatially normalize patient images, or to extract alignment-based features.<br />
<br />
Publication pending. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=971&screenshot_id=956"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=971&screenshot_id=956" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 29, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/stamp_atlases/">http://www.nitrc.org/projects/stamp_atlases/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>MGH <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="standard-amygdalar-fmri-probe-tasks" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Standard Amygdalar fMRI Probe Tasks">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=standard-amygdalar-fmri-probe-tasks" rel="bookmark" name="standard-amygdalar-fmri-probe-tasks">Standard Amygdalar fMRI Probe Tasks</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This resource provides MRI data on 32 control subjects who have been imaged with 4 standard amygdala mapping tasks. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 11, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BIRN Data License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/amygdalamapping/">http://www.nitrc.org/projects/amygdalamapping/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Vanderbilt University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="staple" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="STAPLE">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=staple" rel="bookmark" name="staple">STAPLE</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-display" rel="tag">Image Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quantification" rel="tag">Quantification</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-rendering" rel="tag">Surface Rendering</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> STAPLE is an algorithm for the Simultaneous Truth and Performance Level Estimation, which estimates a reference standard and segmentation generator performance from a set of segmentations. It has been widely applied for the validation of image segmentation algorithms, and to compare the performance of different algorithms and experts. It has also found application in the identification of a consensus segmentation, by combination of the output of a group of segmentation algorithms, and for segmentation by registration and template fusion. Further information is available at <a href="http://www.crl.med.harvard.edu" target="_new">http://www.crl.med.harvard.edu</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=220&screenshot_id=174"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=220&screenshot_id=174" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 24, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Open Software License 3.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.crl.med.harvard.edu/software/STAPLE/index.php">http://www.crl.med.harvard.edu/software/STAPLE/index.php</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, GIfTI, NIfTI-1, Nrrd</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, GIfTI, NIfTI-1, Nrrd</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="stark-cross-sectional-aging" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Stark Cross-Sectional Aging">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=stark-cross-sectional-aging" rel="bookmark" name="stark-cross-sectional-aging">Stark Cross-Sectional Aging</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> NOTE -Some, but not all of the data are are live. More will be added as time and disk space allow.<br />
<br />
Behavioral and imaging data from ~120 participants aged 18-89. Data were collected as part of a grant to use high-resolution imaging and advanced behavioral tasks to understand how aging affects the hippocampus and how this is related to age-related cognitive decline. The full dataset includes:<br />
- Traditional neuropsycholgical measures<br />
- Hippocampal-specific behavioral measures<br />
- Whole-brain DTI<br />
- High-resolution DTI of the medial temporal lobes<br />
- Structural MRI including segmentation of grey/white/CSF, of cortical regions (Freesurfer) and of hippocampal subfields </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 18, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial Share Alike</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/stark_aging/">http://www.nitrc.org/projects/stark_aging/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of California Irvine <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="startyourrecovery.org" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="StartYourRecovery.org">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=startyourrecovery.org" rel="bookmark" name="startyourrecovery.org">StartYourRecovery.org</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> StartYourRecovery.org offers people who are dealing with substance use issues a single source of reputable, objective information about signs, symptoms, conditions, treatment options, and resources — presented in a user-friendly format and in language that’s easy to understand. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 27, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>n/a</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/startyourrecov/">http://www.nitrc.org/projects/startyourrecov/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>StartYourRecovery.org <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="statistical-methods-for-heterogeneous-neuroimaging-data" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Statistical Methods for Heterogeneous Neuroimaging Data">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=statistical-methods-for-heterogeneous-neuroimaging-data" rel="bookmark" name="statistical-methods-for-heterogeneous-neuroimaging-data">Statistical Methods for Heterogeneous Neuroimaging Data</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The aim of this package is to present several statistical analysis pipelines for heterogeneous neuroimaging data. The Gaussian hidden Markov model (GHMM) toolbox is for dealing with the spatial heterogeneity of cartilage progression across both time and subjects. To estimate unknown parameters in GHMM, we employ a pseudo-likelihood function and optimize it by using an expectation-maximization (EM) algorithm. The proposed model can effectively detect diseased regions in each OA subject and present a localized analysis of longitudinal cartilage thickness within each latent subpopulation. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 14, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BIAS License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/ghmm/">http://www.nitrc.org/projects/ghmm/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="statistical-methods-for-manifold-valued-data" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Statistical Methods for Manifold-valued Data">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=statistical-methods-for-manifold-valued-data" rel="bookmark" name="statistical-methods-for-manifold-valued-data">Statistical Methods for Manifold-valued Data</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The aim of this package is to develop several statistical methods for manifold-valued data. Mixture of offset-normal shape factor analyzers (MOSFA) toolbox is a penalized model-based clustering framework to cluster landmark based planar shape data. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 14, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BIAS License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/smmd/">http://www.nitrc.org/projects/smmd/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="statistics-online-computational-resource" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Statistics Online Computational Resource">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=statistics-online-computational-resource" rel="bookmark" name="statistics-online-computational-resource">Statistics Online Computational Resource</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=web-resource" rel="tag">Web Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Statistics Online Computational Resources (SOCR) include the following suite of web-based Java applets: <br />
* Distributions (interactive graphs and calculators)<br />
* Experiments (virtual computer-generated games and processes)<br />
* Analyses (collection of common web-accessible tools for statistical data analysis)<br />
* Games (interfaces and simulations to real-life processes)<br />
* Modeler (tools for distribution, polynomial and spectral model-fitting and simulation)<br />
* Graphs, Plots and Charts (comprehensive web-based tools for exploratory data analysis), <br />
* Additional Tools (other statistical tools and resources)<br />
* SOCR Java-based Statistical Computing Libraries.<br />
<br />
In addition, SOCR provides a suite of tools for volume-based statistical mapping (<a href="http://wiki.stat.ucla.edu/socr/index.php/SOCR_EduMaterials_AnalysesCommandLine" target="_new">http://wiki.stat.ucla.edu/socr/index.php...</a>) via command-line execution and via the LONI Pipeline workflows (<a href="http://www.nitrc.org/projects/pipeline" target="_new">http://www.nitrc.org/projects/pipeline</a>). 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=178&screenshot_id=660"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=178&screenshot_id=660" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 6, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Lesser General Public License (LGPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/socr/">http://www.nitrc.org/projects/socr/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Michigan <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="stdpc" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="StdpC">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=stdpc" rel="bookmark" name="stdpc">StdpC</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> StdpC is dynamic clamp software that allows users to use dynamic clamp without in depth knowledge of real time interfaces or indeed programming. StdpC has a graphical user interface based on the popular QT framework and defines a good number of common synapse and ion channel models, including plasticity rules such as STDP. Advanced features include a comprehensive scripting mechanism that allows full experimental automation and the inclusion of spike generator and simulated neuron modules. StdpC also supports Active Electrode Compensation methods, as introduced by the Brette lab, for Dynamic Clamp with a single, high impedance electrode. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 6, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU GPL v2</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/stdpc/">http://www.nitrc.org/projects/stdpc/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="stereo-investigator" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Stereo Investigator">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=stereo-investigator" rel="bookmark" name="stereo-investigator">Stereo Investigator</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=stereology" rel="tag">Stereology</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> A Stereo Investigator system for stereology gives you accurate, unbiased estimates of the number, length, area, and volume of cells or biological structures in a tissue specimen. It is a key research tool that has helped lead advances in numerous areas of neuroscience including neurodegenerative diseases, neuropathy, memory, and behavior, as well as other research fields including pulmonary research, spinal cord research, and toxicology.<br />
<br />
In addition to stereology software, a Stereo Investigator system includes any hardware you need, such as a microscope, computer, motorized stage, camera, etc., and comes with MBF’s technical support for the entire system as well as research support to optimize your experimental design. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 15, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/si_stereology/">http://www.nitrc.org/projects/si_stereology/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="stereoscopic-atlas-of-intrinsic-brain-networks-(saibn)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Stereoscopic Atlas of Intrinsic Brain Networks (SAIBN)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=stereoscopic-atlas-of-intrinsic-brain-networks-(saibn)" rel="bookmark" name="stereoscopic-atlas-of-intrinsic-brain-networks-(saibn)">Stereoscopic Atlas of Intrinsic Brain Networks (SAIBN)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This tool is a 3D stereoscopic (anaglyph method) full brain functional connectivity atlas created using a parcellation atlas published by Craddock et al. (2012). Using 3D Slicer 3.6.3 and the two hundred ROI version of the Craddock atlas, 200 grayscale surface models were created using a z-stat threshold &gt; 2.3, and each surface model was processed with a surface decimation algorithm, smoothed with the Taubin algorithm and without surface normals.<br />
<br />
For improved visualization of the functional connectivity networks and their relative anatomical position, the surface model of five subcortical anatomical structures (corpus callosum, bilateral caudate, pallidum, putamen, thalamus, amygdala and hippocampus) were included in SAIBN. These surfaces were created with 3D Slicer using the segmentation computed with Freesurfer v. 5.1.<br />
<br />
The viewer should use red-cyan glasses to see the 3D stereoscopic effect using 3D Slicer (version 3.6.3, <a href="http://www.slicer.org/pages/Special:SlicerDownloads" target="_new">http://www.slicer.org/pages/Special:Slic...</a>). 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=650&screenshot_id=580"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=650&screenshot_id=580" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 27, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/saibn/">http://www.nitrc.org/projects/saibn/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="stochastic-tractography-system" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Stochastic Tractography System">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=stochastic-tractography-system" rel="bookmark" name="stochastic-tractography-system">Stochastic Tractography System</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=application" rel="tag">Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Stochastic Tractography applies a Bayesian approach towards the estimation of nerve fiber tracts from DWMRI images. This project provides tools which can perform Stochastic Tractography and related analysis on DWMRI data. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 29, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/stfilter/">http://www.nitrc.org/projects/stfilter/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="structural-asymmetries" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Structural Asymmetries">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=structural-asymmetries" rel="bookmark" name="structural-asymmetries">Structural Asymmetries</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The package of scripts available here enables users to measure structural asymmetries at different scales of resolution using deformation fields that are obtained at 3 increasingly finer scales of normalization resolution. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 11</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/structural_asym/">http://www.nitrc.org/projects/structural_asym/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Medical University of South Carolina <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="studyforrest" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="studyforrest">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=studyforrest" rel="bookmark" name="studyforrest">studyforrest</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> We provide extensive functional brain imaging data from natural stimulation, a rich set of auxiliary data, (such as structural brain scans, measurements of physiological, and technical confounds), as well as stimulus annotations. <a href="http://studyforrest.org" target="_new">http://studyforrest.org</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 12, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MIT/X Consortium License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://studyforrest.org">http://studyforrest.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Magdeburg University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="subject-library" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Subject Library">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=subject-library" rel="bookmark" name="subject-library">Subject Library</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database-application" rel="tag">Database Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-software-resource" rel="tag">Other Software Resource</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> UC Davis IDeA Lab Applications for Management Of Neuroimaging Data -- <br />
<br />
A collection of tools used for processing and organizing MRI data. The Dicom Importer allows you to to view, assemble, and organize dicom files. Subject Library is a filesystem-based search and reporting tool that can be configured to work with many different organization schemes. This package also contains a python library that can be used to write scripts for custom tasks. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 19, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://idealab.ucdavis.edu/software/">http://idealab.ucdavis.edu/software/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="subject-order-independent-group-ica" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Subject Order-Independent Group ICA">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=subject-order-independent-group-ica" rel="bookmark" name="subject-order-independent-group-ica">Subject Order-Independent Group ICA</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=independent-component-analysis" rel="tag">Independent Component Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> While the traditional temporally concatenated Group ICA (TC-GICA) adopting three steps of PCA reduction, it could result in inconsistent and variable components when different subject orders were used, both for the group- and individual-level results. Such instability can further cause instable and thus unreliable statistical results. Subject Order-Independent Group ICA (SOI-GICA) aims to fix this problem by producing stable and reliable GICA results. For details please see the paper &quot;Subject Order-Independent Group ICA (SOI-GICA) for Functional MRI Data Analysis&quot; (Zhang et al., 2010, NeuroImage)(<a href="http://dx.doi.org/10.1016/j.neuroimage.2010.03.039" target="_new">http://dx.doi.org/10.1016/j.neuroimage.2...</a>). MICA is the toolbox inplemented SOI-GICA for convenience of usage. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 2, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cogicat/">http://www.nitrc.org/projects/cogicat/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="sumsdb-database" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="SumsDB Database">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=sumsdb-database" rel="bookmark" name="sumsdb-database">SumsDB Database</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database-application" rel="tag">Database Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=registration" rel="tag">Registration</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-analysis" rel="tag">Surface Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-rendering" rel="tag">Surface Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volume-rendering" rel="tag">Volume Rendering</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> SumsDB is a repository of neuroimaging data (surfaces &amp; volumes; structural &amp; functional data), mainly from humans and macaques but also mouse, rat, and great apes. WebCaret is an online visualization tool for viewing SumsDB datasets. Visit the Tool/Resource Home Page for more information. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 28, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://brainvis.wustl.edu/wiki/index.php/Sums:About">http://brainvis.wustl.edu/wiki/index.php/Sums:About</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>AFNI BRIK, ANALYZE, NIfTI-1, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>AFNI BRIK, ANALYZE, NIfTI-1, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="surface-based-processing-and-analysis-of-mri" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Surface-based processing and analysis of MRI">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=surface-based-processing-and-analysis-of-mri" rel="bookmark" name="surface-based-processing-and-analysis-of-mri">Surface-based processing and analysis of MRI</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The SPAMRI (surface-based processing &amp; analysis for MRI) aims to provide automatic and friendly-used GUI for analysis of structural MRI, such as cortical thickness analysis. Any questions or suggestions, please concact the author: <a href="mailto:longzhiliang@swu.edu.cn" target="_new">longzhiliang@swu.edu.cn</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 17, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>SPAMRI license</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/longzhiliangmri/">http://www.nitrc.org/projects/longzhiliangmri/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Southwest University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, MGH/MGZ</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, MGH/MGZ</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="surf-ice" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Surf Ice">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=surf-ice" rel="bookmark" name="surf-ice">Surf Ice</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-rendering" rel="tag">Surface Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=three-dimensional-display" rel="tag">Three Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tractography" rel="tag">Tractography</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Surf Ice is a tool for surface rendering the cortex with overlays to illustrate tractography, network connections, anatomical atlases and statistical maps. While there are many alternatives, Surf Ice is easy to use and uses advances shaders to generate stunning images. It supports many popular mesh formats [3ds, ac3d, BrainVoyager (srf), ctm, Collada (dae), dfs, dxf, FreeSurfer (Asc, Srf, Curv, gcs, Pial, W), GIfTI (gii), gts, lwo, ms3d, mz3, nv, obj, off, ply, stl, vtk], connectome formats (edge/node) and tractography formats [bfloat, pdb, tck, trk, vtk]. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=984&screenshot_id=950"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=984&screenshot_id=950" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 19, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/surfice/">http://www.nitrc.org/projects/surfice/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of South Carolina <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>GIfTI, NIfTI-1, bshort/bfloat</td></tr>
			<tr class="even"><td>Output Formats</td><td>GIfTI, NIfTI-1, bshort/bfloat</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="surfstat" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="SurfStat">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=surfstat" rel="bookmark" name="surfstat">SurfStat</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=linear" rel="tag">Linear</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=modeling" rel="tag">Modeling</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> SurfStat is a Matlab toolbox for the statistical analysis of univariate and multivariate surface data using linear mixed effects models and random field theory. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 23, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.math.mcgill.ca/keith/surfstat">http://www.math.mcgill.ca/keith/surfstat</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>AFNI BRIK, ANALYZE, MINC, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>AFNI BRIK, ANALYZE, MINC, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="svv" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="SVV">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=svv" rel="bookmark" name="svv">SVV</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-rendering" rel="tag">Surface Rendering</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This project is a simple Surface and Volumetric Visualization application that has been designed to facilitate rapid and flexible visualization from neuroanatomically segmented results. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=7&screenshot_id=70"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=7&screenshot_id=70" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 13, 2006</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/dave1/">http://www.nitrc.org/projects/dave1/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Massachusetts General Hospital <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="synchronized-histological-image-view-arc" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Synchronized Histological Image View Arc">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=synchronized-histological-image-view-arc" rel="bookmark" name="synchronized-histological-image-view-arc">Synchronized Histological Image View Arc</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Synchronized Histological Image Viewing Architecture (<a href="http://www.loni.usc.edu/Software/SHIVA" target="_new">http://www.loni.usc.edu/Software/SHIVA</a>) is a Java-based visualization and analysis application. SHIVA can process 2D and 3D image files and provides convenient methods for users to overlay multiple datasets.<br />
<br />
SHIVA is distributed by the Laboratory of Neuro Imaging (<a href="http://www.loni.usc.edu/Software/SHIVA" target="_new">http://www.loni.usc.edu/Software/SHIVA</a>) at USC. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=303&screenshot_id=420"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=303&screenshot_id=420" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 8, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/shiva/">http://www.nitrc.org/projects/shiva/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Southern California <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="talairach-daemon" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Talairach Daemon">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=talairach-daemon" rel="bookmark" name="talairach-daemon">Talairach Daemon</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database-application" rel="tag">Database Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Talairach Daemon database contains anatomical names for brain areas using x-y-z coordinates defined by the 1988 Talairach atlas. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 31, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BrainMap License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.talairach.org/">http://www.talairach.org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="tapas---translational-algorithms-for-psychiatry-advancing-science" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="TAPAS - Translational Algorithms for Psychiatry-Advancing Science">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=tapas---translational-algorithms-for-psychiatry-advancing-science" rel="bookmark" name="tapas---translational-algorithms-for-psychiatry-advancing-science">TAPAS - Translational Algorithms for Psychiatry-Advancing Science</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> TAPAS is a collection of algorithms and software tools developed by the Translational Neuromodeling Unit (TNU, Zurich) and collaborators. The goal of these tools is to support clinical neuromodeling, particularly computational psychiatry, computational neurology, and computational psychosomatics.<br />
<br />
Currently, TAPAS includes the following packages:<br />
<br />
HGF: Hierarchical Gaussian Filter; Bayesian inference on computational processes from observed behavior<br />
<br />
HUGE: Variational Bayesian inversion for hierarchical unsupervised generative embedding<br />
 <br />
MICP: Bayesian Mixed-effects Inference for Classification Studies<br />
<br />
MPDCM: Massively Parallel DCM; Efficient numerical integration<br />
<br />
PhysIO: Physiological Noise Correction for fMRI<br />
<br />
rDCM: Efficient inference on effective brain connectivity for fMRI.<br />
<br />
SEM: SERIA Model for (anti-)Saccadic Eye Movements and Reaction Times<br />
<br />
VBLM: Variational Bayesian Linear Regression<br />
<br />
TAPAS is written in MATLAB and distributed as open source code under GNU General Public License 3.0. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1259&screenshot_id=1172"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1259&screenshot_id=1172" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 9, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License v3</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.translationalneuromodeling.org/tapas">http://www.translationalneuromodeling.org/tapas</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Zurich and ETH Zurich <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1, Philips PAR/REC</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1, Philips PAR/REC</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="tapir" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="TAPIR">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=tapir" rel="bookmark" name="tapir">TAPIR</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> TAPIR (Tools for Advanced Parameterized Image Registration) is a set of command line tools allowing 2D and 3D image registration, mainly for medical imaging (although also relevant to other image registration problems). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 27, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Mozilla Public License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/tapir/">http://www.nitrc.org/projects/tapir/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="tarquin-mrs-analysis-package" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="TARQUIN MRS analysis package">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=tarquin-mrs-analysis-package" rel="bookmark" name="tarquin-mrs-analysis-package">TARQUIN MRS analysis package</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> TARQUIN is an analysis tool for automatically determining the quantities of molecules present in NMR spectroscopic data. The intended purpose of TARQUIN is to aid the characterisation of pathologies, in particular brain tumours, both non-invasively with in-vivo 1H MRS and ex-vivo with 1H HR-MAS. TARQUIN has the following features:<br />
<br />
Free to use and modify under the GPL licence.<br />
Based on a flexible time-domain fitting routine designed to give accurate rapid and automated quantitation for routine analysis.<br />
Cross platform, works on Windows, Linux and OSX.<br />
Comes packaged with a quantum mechanically based metabolite simulator to allow basis set construction optimised for the investigation of particular pathologies sequence parameters.<br />
Includes both GUI and command line interface for one-off and batch analyses. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 17, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://tarquin.sourceforge.net/">http://tarquin.sourceforge.net/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="task-based-fmri-atlases" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Task-based fMRI atlases">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=task-based-fmri-atlases" rel="bookmark" name="task-based-fmri-atlases">Task-based fMRI atlases</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=anatomic" rel="tag">Anatomic</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The 200 ROI Task-based fMRI atlas and 222 ROI Task-based fMRI atlas are fully labeled, whole brain atlas derived via spatially constrained n-cut parcellation of both task-based and resting-state fMRI data. (James et al, Magn Reson Imaging, in press; PMID 26523655). Atlases is provided in NIFTI format. Anatomic labels are provided in BrainViewer format and comma-delimited text format. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=989&screenshot_id=963"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=989&screenshot_id=963" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 18, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial Share Alike</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/task_fmri_atlas/">http://www.nitrc.org/projects/task_fmri_atlas/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Arkansas for Medical Sciences <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="task-independent-fluctuations-discussion" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Task Independent Fluctuations Discussion">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=task-independent-fluctuations-discussion" rel="bookmark" name="task-independent-fluctuations-discussion">Task Independent Fluctuations Discussion</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=community" rel="tag">Community</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data-model" rel="tag">Data Model</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-software-resource" rel="tag">Other Software Resource</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=web-resource" rel="tag">Web Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The methodology and applications of task independent fluctuation measures including: connectivity maps of fMRI resting state scans, research using EEG/MEG/PET etc, methods to remove non-neural fluctuations, and applications to clinical populations. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 19, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/fluctuations/">http://www.nitrc.org/projects/fluctuations/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="template-image-processing-library" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Template Image Processing Library">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=template-image-processing-library" rel="bookmark" name="template-image-processing-library">Template Image Processing Library</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=format-conversion" rel="tag">Format Conversion</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=spatial-transformation" rel="tag">Spatial Transformation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Template Image Processing Library (TIPL) is a lightweight C++ template library designed mainly for medical imaging processing. The design paradigm follows generic programming, and the purpose is to provide an &quot;easy-to-use&quot; and also &quot;ready-to-use&quot; library. <br />
<br />
The code is template-based, and only header files are needed to be included to the source code.<br />
<br />
This library provides the following functions:<br />
1.DICOM (r), Analyze(r), Nifti (r/w), and MATLAB MAT V4 (r/w)<br />
2.numerical: add, multiply, gradient.<br />
3.interpolation: linear, gaussian radial basis<br />
3.filters: mean, gaussian, laplacian, sobel, anisotropic diffusion<br />
4.morphological processing: erosion, expansion, opening, closing<br />
5.template-based Fourier transform<br />
6.linear coregistration: rigid body, affine transform, least square fit, mutual information<br />
7.nonlinear coregistration: The Large Deformation Diffeomorphic Metric Mapping (LDDMM) </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 15, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://tipl.labsolver.org">http://tipl.labsolver.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, NIfTI-1, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, NIfTI-1, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="templates-for-in-vivo-mouse-brain" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Templates for In vivo Mouse Brain">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=templates-for-in-vivo-mouse-brain" rel="bookmark" name="templates-for-in-vivo-mouse-brain">Templates for In vivo Mouse Brain</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-data" rel="tag">Atlas Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The templates for in vivo mouse brain are intended for SPM normalization and segmentation (including skull-stripping). The population-averaged, stereotaxic, and tissue segmented template were created from in vivo T1WI (an isotropic resolution of 80μm) of C57Bl/6(n=30, male), BALB/cBy(n=10, male), C3H/He(n=10, male), and DBA/2 (n=10, male) mice. The voxel size of templates were multiplied by 10 to be able to use SPM directly.<br />
Keywords: atlas, mouse, probability map, template, inbred strain<br />
<br />
Please refer to the following article for the atlases: <br />
Hikishima K, Komaki Y, Seki F, Ohnishi Y, Okano HJ, Okano H. <br />
In vivo microscopic voxel-based morphometry with a brain template to characterize strain-specific structures in the mouse brain. Sci Rep. 2017 Dec;7(1):85. doi:10.1038/s41598-017-00148-1. PubMed PMID: 28273899.<br />
<br />
Authors: Keigo Hikishima, Hideyuki Okano<br />
Contact person: <a href="mailto:keigo.hikishima@oist.jp" target="_new">keigo.hikishima@oist.jp</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 26, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial Share Alike</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/tpm_mouse/">http://www.nitrc.org/projects/tpm_mouse/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="templates-for-long-evans-rat-brain" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Templates for Long-Evans Rat Brain">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=templates-for-long-evans-rat-brain" rel="bookmark" name="templates-for-long-evans-rat-brain">Templates for Long-Evans Rat Brain</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-data" rel="tag">Atlas Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The average signal intensity, stereotaxic templates and TPMs of rat brain are intended for SPM normalization and segmentation (including skull-stripping) of rat head MRI data. Ex-vivo T2WI (an isotropic resolution of 118μm) of Long Evans (n=4, male, 8wo) rat head were acquired and DARTEL in SPM8 was used for the creation of templates and TPMs. The bounding box (a three dimensional space) of templates encompassed the followings x, y, z dimensions (-18,18; -20, 7; -14, 6) and the origin was (0, 0, 0) in mm. The horizontal slice (z=0) passed through bregma and lambda (the flat-skull position) and the coronal slice (y=0) lay perpendicular to the horizontal plane and passes through the bregma, and the sagittal slice (x=0) was located at the median sagittal plane. The locations of bregma and lambda were determined from the image contrasts of T2WI and CT data.<br />
<br />
Authors: Keigo Hikishima, Susumu Setogawa, Hiroshi Mizuma, Yilong Cui, Kazuto Kobayashi, Hirotaka Onoe<br />
<br />
Contact person: <a href="mailto:keigo.hikishima@oist.jp" target="_new">keigo.hikishima@oist.jp</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 8, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Share Alike</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/tpm_rat/">http://www.nitrc.org/projects/tpm_rat/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="temporal-lobe.com" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Temporal-lobe.com">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=temporal-lobe.com" rel="bookmark" name="temporal-lobe.com">Temporal-lobe.com</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Existing knowledge of hippocampal - parahippocampal connections was integrated into an interactive diagram in which any connection can be turned on or off at the level of cortical layers.<br />
<br />
Project Goals:<br />
* To provide an overview of all known anatomical projections of the rat hippocampal - parahippocampal network.<br />
* To provide a graphical interface with which users can turn on or off any connections at regional level down to the level of cortical layers.<br />
* To make it easy to find references for a particular projection. <br />
<br />
Download the diagram for free at <a href="http://www.temporal-lobe.com." target="_new">http://www.temporal-lobe.com.</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 29, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/connectivity/">http://www.nitrc.org/projects/connectivity/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="temporal-lobe-epilpsy-multiple-modality-data" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="TEMPORAL LOBE EPILPSY MULTIPLE MODALITY DATA">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=temporal-lobe-epilpsy-multiple-modality-data" rel="bookmark" name="temporal-lobe-epilpsy-multiple-modality-data">TEMPORAL LOBE EPILPSY MULTIPLE MODALITY DATA</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The data-set consists of multiple modalities from three temporal lobe epilepsy studies.<br />
<br />
•Study 1- HFH_T1_HP_SEG_TLE_1<br />
<br />
The purpose of this study was to develop a systematic and statistical approach using a large dataset for the evaluation of automated methods and establish a method that would achieve results better approximating those attained by manual tracing in the epileptogenic hippocampus.<br />
<br />
<br />
•Study 2- HFH_FLAIR_T1_HP_SEG_TLE<br />
<br />
<br />
The purpose of this study was to develop a systematic and statistical approach using a large dataset for the evaluation of automated methods and establish a method that would achieve results better approximating those attained by manual tracing in the epileptogenic hippocampus.<br />
<br />
•Study 3- HFH_SPECT_T1_HP_SEG_TLE<br />
<br />
This study assesses the utility of compartmental analysis of SPECT data in lateralizing ictal onset in cases of a putative mesial temporal lobe epilepsy (mTLE).<br />
<br />
PS: CLICK ON EACH STUDY NAME ON THE DOWNLOAD PAGE FOR MORE INFO BEFORE DOWNLOADING. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 18, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/hfh_t1_hp_seg1/">http://www.nitrc.org/projects/hfh_t1_hp_seg1/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Henry Ford Health System <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="tetrametrix" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="TetraMetrix">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=tetrametrix" rel="bookmark" name="tetrametrix">TetraMetrix</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> TetraMetrix is an open-source software project for distributed Tetrahedral Mesh Modeling and Analysis (TMMA) of multidimensional data. Tetrahedra are 3D, space filling, geometric objects that can be used to form representations and partitions of objects like volumes and shapes obtained from biomedical images (e.g., 3D MRI brain images). Tetrahedra are a natural extension of lines (1D) and surface triangulations (2D) which enable finite element analysis of irregular shapes, and permit a greater range of morphometric characterization of multidimensional objects. The project is being developed by researchers at USC Laboratory of Neuro Imaging (<a href="http://www.LONI.usc.edu" target="_new">http://www.LONI.usc.edu</a>) and Department of Mathematics (<a href="http://www.math.ucla.edu" target="_new">http://www.math.ucla.edu</a>).<br />
<br />
TetraMetrix is distributed by the Laboratory of Neuro Imaging at USC.<br />
<br />
Update: A new version tetrahedral meshing software along with new related image registration software was made available in the fall of 2015. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=408&screenshot_id=947"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=408&screenshot_id=947" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 4, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Lesser General Public License (LGPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/tmma">http://www.nitrc.org/projects/tmma</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Southern California <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="textpresso---literature-search-engine" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Textpresso - literature search engine">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=textpresso---literature-search-engine" rel="bookmark" name="textpresso---literature-search-engine">Textpresso - literature search engine</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=bibliographic-resource" rel="tag">Bibliographic Resource</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database-application" rel="tag">Database Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=knowledge-environment" rel="tag">Knowledge Environment</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=web-service" rel="tag">Web Service</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=websites" rel="tag">Websites</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Textpresso is a text-mining system for scientific literature. Textpresso's two major elements are (1) access to full text, so that entire articles can be searched, and (2) introduction of categories of biological concepts and classes that relate two objects (e.g., association, regulation, etc.) or describe one (e.g., methods, etc). A search engine enables the user to search for one or a combination of these categories and/or keywords within an entire literature.<br />
Textpresso is useful as a search engine for researchers as well as a curation tool. It was developed as a part of WormBase and is used extensively by C. elegans curators. Textpresso has currently been implemented for 17 different literatures, among them Neuroscience, and can readily be extended to other corpora of text.<br />
Textpresso can be used online (<a href="http://www.textpresso.org" target="_new">http://www.textpresso.org</a>), but also installed<br />
locally via a downloadable software package (<a href="http://www.textpresso.org/downloads.html" target="_new">http://www.textpresso.org/downloads.html</a>) </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 31, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Textpresso License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/textpresso-2-0/">http://www.nitrc.org/projects/textpresso-2-0/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>PDF - Portable Document Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>PDF - Portable Document Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="the-brain-catalogue" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="The Brain Catalogue">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=the-brain-catalogue" rel="bookmark" name="the-brain-catalogue">The Brain Catalogue</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Our aim is to celebrate the diversity of the vertebrate brain by making high quality data, open and freely available to everyone. Do you have data that you would like to share? Do not hesitate to contact us!<br />
<br />
The Brain Catalogue is developed by Florencia Grisanti (Taxidermy Workshop of the Natural History Museum in Paris) and Roberto Toro (Neuroscience Department of the Institut Pasteur). Many of our specimens come from the Vertebrate Brain Collection of the Jardin des Plantes, curated by Marc Herbin, and are scanned at the Institut du Cerveau et de la Moelle (ICM) by Mathieu Santin and Alexandra Petiet, from the CENIR laboratory, with financial and methodological support kindly provided by Olivier Colliot, head of the Cogimage team at the ICM. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 21, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://braincatalogue.org/">http://braincatalogue.org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Institut Pasteur <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="the-brain-coactivation-map" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="The Brain Coactivation Map">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=the-brain-coactivation-map" rel="bookmark" name="the-brain-coactivation-map">The Brain Coactivation Map</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Brain Coactivation Map describes all the coactivation networks in the human brain based on the meta-analysis of more than 5,400 neuroimaging articles (from NeuroSynth) containing more than 16,000 individual experiments. It also provides a list of the MeSH tags that more closely match the structure of each network.<br />
You can use our viewer CoactivationMap (<a href="https://github.com/r03ert0/CoactivationMap.app" target="_new">https://github.com/r03ert0/CoactivationM...</a>) to interactively browse the map, or our command line cmtool (<a href="https://github.com/r03ert0/cmtool" target="_new">https://github.com/r03ert0/cmtool</a>) to query the map from a shell. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 31, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial Share Alike</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/cmap/">http://www.nitrc.org/projects/cmap/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Institut Pasteur <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="the-cancer-imaging-archive-(tcia)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="The Cancer Imaging Archive (TCIA)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=the-cancer-imaging-archive-(tcia)" rel="bookmark" name="the-cancer-imaging-archive-(tcia)">The Cancer Imaging Archive (TCIA)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=community" rel="tag">Community</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database" rel="tag">Database</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=test-data" rel="tag">Test Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=web-service" rel="tag">Web Service</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Cancer Imaging Archive is a large archive of medical images of cancer accessible for public download. Registering is free. All images are stored in DICOM file format. The images are organized as &quot;Collections&quot;, typically patients related by a common disease (e.g. lung cancer), image modality (MRI, CT, etc) or research focus. A full listing of the available data sets can be found at <a href="https://wiki.cancerimagingarchive.net/display/Public/Collections" target="_new">https://wiki.cancerimagingarchive.net/di...</a><br />
<br />
There are a number of neuroimaging data sets available. Most contain substantial metadata generated by The Cancer Genome Atlas project which includes clinical outcomes, pathology, and genomics in addition to the DICOM images. Learn more about the CIP TCGA Radiology Initiative to investigate imaging genomics correlations at <a href="https://wiki.cancerimagingarchive.net/display/Public/CIP+TCGA+Radiology+Initiative" target="_new">https://wiki.cancerimagingarchive.net/di...</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=677&screenshot_id=605"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=677&screenshot_id=605" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 14, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://cancerimagingarchive.net">http://cancerimagingarchive.net</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>National Cancer Institute <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>DICOM, NIfTI-1, Nrrd, Other Format, PDF - Portable Document Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>DICOM, NIfTI-1, Nrrd, Other Format, PDF - Portable Document Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="the-cognitive-atlas" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="The Cognitive Atlas">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=the-cognitive-atlas" rel="bookmark" name="the-cognitive-atlas">The Cognitive Atlas</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=knowledge-environment" rel="tag">Knowledge Environment</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=ontology" rel="tag">Ontology</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=websites" rel="tag">Websites</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Cognitive Atlas, <a href="http://www.cognitiveatlas.org," target="_new">http://www.cognitiveatlas.org,</a> is a collaborative knowledge building project that aims to develop a knowledge base (or ontology) that characterizes the state of current thought in cognitive neuroscience. The goal is to develop a knowledge base that will support annotation of data in databases, as well as supporting improved discourse in the community. It is open to all interested researchers. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=396&screenshot_id=502"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=396&screenshot_id=502" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 26, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.cognitiveatlas.org">http://www.cognitiveatlas.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Stanford University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="the-de-identification-toolbox---a-data-sharing-tool-for-neuroimaging-studies" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="The De-identification Toolbox - A data sharing tool for neuroimaging studies">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=the-de-identification-toolbox---a-data-sharing-tool-for-neuroimaging-studies" rel="bookmark" name="the-de-identification-toolbox---a-data-sharing-tool-for-neuroimaging-studies">The De-identification Toolbox - A data sharing tool for neuroimaging studies</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The De-identification Toolbox (formerly DeID) is a Java program that users can use to remove identifying information in neuroimaging datasets. This software provides a series of user interaction wizards to allow users to select data variables to be de-identified, implements functions for auditing and validation of de-identified data, and enables the user to share the de-identified data in a single compressed package through various communication protocols. The software runs with Windows, Linux, and Mac operating systems. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 9, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Lesser General Public License (LGPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/de-identification/">http://www.nitrc.org/projects/de-identification/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="the-ebds-infant-dti-fiber-atlases" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="The EBDS Infant DTI Fiber Atlases">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=the-ebds-infant-dti-fiber-atlases" rel="bookmark" name="the-ebds-infant-dti-fiber-atlases">The EBDS Infant DTI Fiber Atlases</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Neonatal and 1-2 year old DTI fiber atlases for studies of brain development at birth to 2 years of age from the UNC Early Brain Development Studies (EBDS) group and the NIRAL (Neuro Image Research and Analysis Laboratory)<br />
<br />
Infant DTI atlases with a comprehensive set of template fibers for semi-automatic tract based analysis that represents a typically developing human brain during the first few weeks up to 2 years of life. To the best of our knowledge, this the first such population atlases with this magnitude of quality and sample size.<br />
<br />
These resources enable widespread application of a set of template fibers for atlas based along-tract analysis supporting an adequate and reliable analysis of DTI in newborns in both practice and in clinical research settings and address a critical gap in the current research community. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1162&screenshot_id=1123"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1162&screenshot_id=1123" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 3, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial Share Alike</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/uncebds_neodti/">http://www.nitrc.org/projects/uncebds_neodti/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina at Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="the-fmri-data-center" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="The fMRI Data Center">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=the-fmri-data-center" rel="bookmark" name="the-fmri-data-center">The fMRI Data Center</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> A free database of fMRI data used in peer reviewed studies (fMRIDC). The database of studies can be browsed and searched, and data is provided via download links that are emailed to users in response to specific requests. All of the data has been rendered anonymous, is free, and can be used in further studies. Any published findings based on these datasets should credit the authors of the original study as well as the fMRI Data Center including the accession number. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 26, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/fmridatacenter/">http://www.nitrc.org/projects/fmridatacenter/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="the-grid-code-analysis-toolbox-(gridcat)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="The Grid Code Analysis Toolbox (GridCAT)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=the-grid-code-analysis-toolbox-(gridcat)" rel="bookmark" name="the-grid-code-analysis-toolbox-(gridcat)">The Grid Code Analysis Toolbox (GridCAT)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Matlab-based Grid Code Analysis Toolbox (GridCAT) allows researchers to carry out automated analysis of the putative firing of grid cells (i.e., the grid code) in human fMRI data. All analysis steps, from estimation and fitting of the grid code in the general linear model, to the generation of grid code metrics and plots, can be performed by means of a simple and user-friendly graphical user interface. Researchers confident with programming can also edit the open-source code and use example scripts accompanying the GridCAT to implement their own analysis pipelines. Moreover, an example dataset is provided together with a detailed manual, so that users can explore the GridCAT’s functionality.<br />
<br />
<br />
Reference:<br />
<br />
Stangl, M.*, Shine, J.*, &amp; Wolbers, T. (2017). The GridCAT: A toolbox for automated analysis of human grid cell codes in fMRI. Frontiers in Neuroinformatics, 11:47. [* equal contribution]<br />
<a href="https://doi.org/10.3389/fninf.2017.00047" target="_new">https://doi.org/10.3389/fninf.2017.00047</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1115&screenshot_id=1075"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1115&screenshot_id=1075" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 3, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/gridcat/">http://www.nitrc.org/projects/gridcat/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="the-medical-image-computing-and-computer-assisted-intervention-society" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="The Medical Image Computing and Computer Assisted Intervention Society">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=the-medical-image-computing-and-computer-assisted-intervention-society" rel="bookmark" name="the-medical-image-computing-and-computer-assisted-intervention-society">The Medical Image Computing and Computer Assisted Intervention Society</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=community" rel="tag">Community</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-information-resource" rel="tag">Other Information Resource</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=websites" rel="tag">Websites</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Medical Image Computing and Computer Assisted Intervention Society (the MICCAI Society) is dedicated to the promotion, preservation and facilitation of research, education and practice in the field of medical image computing and computer assisted medical interventions including biomedical imaging and robotics, through the organization and operation of regular high quality international conferences and publications which promote and foster the exchange and dissemination of advanced knowledge, expertise and experience in the field produced by leading institutions and outstanding scientists, physicians and educators around the world. The MICCAI Society is committed to maintaining high academic standards and independence from any personal, political or commercial vested interests. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1355&screenshot_id=1257"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1355&screenshot_id=1257" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 10</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://www.miccai.org/">https://www.miccai.org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>The MICCAI Society <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="the-multiscale-object-oriented-simulation-environment" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="the Multiscale Object Oriented Simulation Environment">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=the-multiscale-object-oriented-simulation-environment" rel="bookmark" name="the-multiscale-object-oriented-simulation-environment">the Multiscale Object Oriented Simulation Environment</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=monte-carlo" rel="tag">Monte Carlo</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=neuronal-modeling" rel="tag">Neuronal Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=simulation" rel="tag">Simulation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MOOSE is the Multiscale Object-Oriented Simulation Environment. It is the base and numerical core for large, detailed simulations including Computational Neuroscience and Systems Biology.<br />
<br />
MOOSE spans the range from single molecules to subcellular networks, from single cells to neuronal networks, and to still larger systems.<br />
MOOSE uses Python for scripting compatibility with a large range of software and analysis tools. It recognizes model definition standards including SBML, NeuroML, and GENESIS model file formats. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=832&screenshot_id=782"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=832&screenshot_id=782" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 12, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://moose.ncbs.res.in">http://moose.ncbs.res.in</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Tata Institute of Fundamental Research <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="the-neuro-bureau" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="The Neuro Bureau">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=the-neuro-bureau" rel="bookmark" name="the-neuro-bureau">The Neuro Bureau</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=community" rel="tag">Community</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-information-resource" rel="tag">Other Information Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Neuro Bureau is neuro-collaboration in action.<br />
<br />
More specifically the Neuro Bureau is a neuroscience collaboratory that supports open neuroscience. To join the Neuro Bureau use the &quot;join the team&quot; link in the yellow box to the right. For more information, please contact: <a href="mailto:theneurobureau@gmail.com" target="_new">theneurobureau@gmail.com</a><br />
<br />
Preprocessed imaging data can be downloaded using the Downloads link on the left, for more information on the preprocessed data use the MediaWiki link on the left.<br />
<br />
Brain-Art Competition 2011 is now over. All of the submissions and the winners are posted at the Brain-Art Gallery.<br />
neurobureau.projects.nitrc.org/BrainArt/Gallery 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=383&screenshot_id=349"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=383&screenshot_id=349" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 10, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://neurobureau.org">http://neurobureau.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="the-neurophysiological-biomarker-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="The Neurophysiological Biomarker Toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=the-neurophysiological-biomarker-toolbox" rel="bookmark" name="the-neurophysiological-biomarker-toolbox">The Neurophysiological Biomarker Toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database-application" rel="tag">Database Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=fourier-time-domain-analysis" rel="tag">Fourier Time-Domain Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=statistical-operation" rel="tag">Statistical Operation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=temporal-wavelet-analysis" rel="tag">Temporal Wavelet Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=web-service" rel="tag">Web Service</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Neurophysiological Biomarker Toolbox (NBT) is an open source Matlab toolbox for the computation and integration of neurophysiological biomarkers. NBT offers a pipeline from data storage to statistics including artefact rejection, signal visualization, biomarker computation, and statistical testing. NBT allows for easy implementation of new biomarkers, and incorporates an online wiki (<a href="http://www.nbtwiki.net/" target="_new">http://www.nbtwiki.net/</a>) that facilitates collaboration among NBT users including extensive help and tutorials. <br />
NBT is specialized in analyzing EEG data, however it allows the processing of any kind of signal. <br />
NBT can, e.g., be used to analyze ongoing oscillation between:<br />
<br />
- Eyes-closed rest of subject populations (e.g., healthy subjects and patients, males vs. females, young vs. old, etc.).<br />
- Two experimental condition (e.g., classical eyes-closed rest vs. meditation, or before vs. after consumption of a CNS-active substance (a drug, coffee, nicotine, alcohol, etc.).<br />
<br />
More details can be found on <a href="http://www.nbtwiki.net/" target="_new">http://www.nbtwiki.net/</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=612&screenshot_id=530"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=612&screenshot_id=530" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 24, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nbtwiki.net/">http://www.nbtwiki.net/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Neuronal Oscillations and Cognition group <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Brain Vision Analyzer Data Exchange Format, EDF/BDF/GDF, EGI</td></tr>
			<tr class="even"><td>Output Formats</td><td>Brain Vision Analyzer Data Exchange Format, EDF/BDF/GDF, EGI</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="the-role-of-v6-in-the-interaction-between-dorsal-and-ventral-streams-in-near-and-far-space-processing" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="The role of V6 in the interaction between dorsal and ventral streams in near and far space processing">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=the-role-of-v6-in-the-interaction-between-dorsal-and-ventral-streams-in-near-and-far-space-processing" rel="bookmark" name="the-role-of-v6-in-the-interaction-between-dorsal-and-ventral-streams-in-near-and-far-space-processing">The role of V6 in the interaction between dorsal and ventral streams in near and far space processing</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Using functional MRI, aimming at investigated the neural interface which integrated information between the dorsal and the ventral streams. The results showed that the parietal-occipital junction (POJ) and bilateral superior occipital gyrus (SOG) showed relative increased activity when responded to a target presented in near space than in far space, which was independent of the retinotopic or perceived size of the target. Furthermore, the POJ showed enhanced functional connectivity with both the dorsal and ventral streams during far space processing irrespective of the target size, supporting the role of the POJ as an interface between the two streams. In contrast, the SOG showed enhanced functional connectivity only with the ventral stream if retinotopic sizes of the targets in near and far space were matched, which suggested the functional dissociation between the POJ and SOG. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 28, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>spatial attention in 3D license</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/visuospatial/">http://www.nitrc.org/projects/visuospatial/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>South China Normal University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="the-striatal-subregional-voimap" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="The Striatal Subregional VOImap">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=the-striatal-subregional-voimap" rel="bookmark" name="the-striatal-subregional-voimap">The Striatal Subregional VOImap</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Striatal Subregional VOImap is intended to provide accurate data in terms of specific uptake location to make the BP quantitation. Available in .obj format.<br />
<br />
The VOIs were manually drawn with software Analyze 9,0 (Mayo Clinic) in 18F-DOPA brain image after spatial normalization with a 18F-DOPA Template developed by our group (available at: <a href="https://www.nitrc.org/projects/spmtemplates" target="_new">https://www.nitrc.org/projects/spmtempla...</a>). Each striatum was divided into 6 sub-regions: ventral caudate, anterior dorsal caudate, posterior dorsal caudate, ventral putamen, anterior dorsal putamen and posterior dorsal putamen, based on Mawlawi et al. (J Cereb Blood Flow Metab.2001;21:1034–57) and Oh et al. (J Nucl Med.2012;53:399–406) segmentation. A nonspecific background volume was drawn in the occipital cortex.<br />
<br />
Terms of use:<br />
The Striatal Subregional VOImap is intended for free use by the neuroimaging community. However, we do ask that you please cite.<br />
<br />
J.A. Lojo-Ramírez, F.J. García-Gómez &amp; D. García-Solís.<br />
Contact: <a href="mailto:jalojoram@gmail.com" target="_new">jalojoram@gmail.com</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=801&screenshot_id=727"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=801&screenshot_id=727" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 11, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/striatalvoimap/">http://www.nitrc.org/projects/striatalvoimap/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="the-virtual-brain-(tvb)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="The Virtual Brain (TVB)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=the-virtual-brain-(tvb)" rel="bookmark" name="the-virtual-brain-(tvb)">The Virtual Brain (TVB)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=modeling" rel="tag">Modeling</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> TVB allows the reproduction and evaluation of personalized configurations of the brain by using individual subject data. This personalization facilitates an exploration of the consequences of pathological changes in the system, permitting to investigate potential ways to counteract such unfavourable processes. The architecture supports interaction with MATLAB/octave packages, for example, the well known Brain Connectivity Toolbox. TVB can be used in a client-server configuration, such that it can be remotely accessed through the Internet thanks to its web-based HTML5, JS and WebGL graphical user interface. TVB is also accessible as a standalone cross-platform Python library and application, and users can interact with the scientific core through the scripting interface using IPython Notebook, enabling easy modelling, development and debugging of the scientific kernel. This second interface makes TVB extensible by combining it with other libraries and modules developed by the Python scientific community. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=855&screenshot_id=833"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=855&screenshot_id=833" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 11, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.thevirtualbrain.org/">http://www.thevirtualbrain.org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>GIfTI, NIfTI-1, Other Format, cff (Connectome File Format)</td></tr>
			<tr class="even"><td>Output Formats</td><td>GIfTI, NIfTI-1, Other Format, cff (Connectome File Format)</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="time-encoding-and-decoding-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Time Encoding and Decoding Toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=time-encoding-and-decoding-toolbox" rel="bookmark" name="time-encoding-and-decoding-toolbox">Time Encoding and Decoding Toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Time Encoding Machines (TEMs) are asynchronous signal processors that encode analog information in the time domain. TEMs play a key role in modeling the sensing of the natural world by biological sensory systems.<br />
<br />
Time Decoding Machines (TDMs) recover stimuli encoded as time (spike) sequences by TEMs. Assuming that Nyquist-type rate conditions are satisfied by the encoding architecture, arbitrarily precise stimulus recovery of time-encoded 1D and 2D (video) signals can be achieved. <br />
<br />
The Time Encoding and Decoding Toolbox contains Python implementations of a selection of SISO, SIMO, and MIMO TEMs and corresponding TDMs that include encoding circuits utilizing classical neuron models (LIF, Hodgkin-Huxley, etc.), feedback, random thresholds, and/or Asynchronous Sigma/Delta Modulators. These can be used to develop new models of information encoding and processing in neural circuits. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 11, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/ted-python/">http://www.nitrc.org/projects/ted-python/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Bionet Group, Columbia University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="tissue-thickness-estimation-via-minimum-line-integrals" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Tissue Thickness Estimation via Minimum Line Integrals">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=tissue-thickness-estimation-via-minimum-line-integrals" rel="bookmark" name="tissue-thickness-estimation-via-minimum-line-integrals">Tissue Thickness Estimation via Minimum Line Integrals</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=medial-axis" rel="tag">Medial Axis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=thickness" rel="tag">Thickness</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volumetric-analysis" rel="tag">Volumetric Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This open-source toolbox contains Matlab codes for the estimation of tissue (such as cortical) thickness from soft segmentation of 3D medical images, using minimum line integrals as described by Aganj et al (HBM 2009). A short tutorial is included in EXAMPLE.m. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1202&screenshot_id=1121"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1202&screenshot_id=1121" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 10, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/thickness">http://www.nitrc.org/projects/thickness</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Minnesota, and Harvard Medical School / MGH <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="tns-data-repository" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="TNS Data Repository">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=tns-data-repository" rel="bookmark" name="tns-data-repository">TNS Data Repository</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This project provides an umbrella for all releases of imaging data from the Translational Neuroradiology Section at the National Institutes of Health. This data will mainly focus on neuroimaging data collected on healthy controls and patients collect in various studies performed by the TNS. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 22, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>TNS Data Access</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/tns_data/">http://www.nitrc.org/projects/tns_data/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>National Institutes of Health <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="tns-lymphatics-dataset" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="TNS Lymphatics Dataset">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=tns-lymphatics-dataset" rel="bookmark" name="tns-lymphatics-dataset">TNS Lymphatics Dataset</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This dataset provides deidentified imaging data for 5 subjects from an imaging study designed to image the lymphatic system non-invasively using MRI. The associated data was published as a part of an article in Elife (&quot;Human and nonhuman primate meninges harbor lymphatic vessels that can be visualized noninvasively by MRI&quot;) and publicized by the NIH Director as a part of his blog (<a href="https://directorsblog.nih.gov/2017/10/17/new-imaging-approach-reveals-lymph-system-in-brain/" target="_new">https://directorsblog.nih.gov/2017/10/17...</a>). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 24, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>TNS Data Access</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/tns_lymph/">http://www.nitrc.org/projects/tns_lymph/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>National Institutes of Health <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="toads-cruise-brain-segmentation-tools" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="TOADS-CRUISE Brain Segmentation Tools">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=toads-cruise-brain-segmentation-tools" rel="bookmark" name="toads-cruise-brain-segmentation-tools">TOADS-CRUISE Brain Segmentation Tools</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The TOADS-CRUISE Brain Segmentation Tools are a collection of software plug-ins developed for the automatic segmentation of magnetic resonance brain images. The tools include multiple published algorithms developed at Johns Hopkins University. The SPECTRE algorithm performs brain extraction. The TOADS algorithm generates a topology-preserving tissue classification into cortical, subcortical, and cerebellar structures. The CRUISE algorithm produces inner, central, and outer cortical surfaces suitable for computing thickness and other geometric measures. Tools are also included for performing gyral labeling, lesion segmentation, thickness computation, surface visualization, and surface file conversion. All tools are released as plug-ins for the MIPAV software package and were developed using the Java Image Science Toolkit (both available on NITRC). They are therefore cross-platform and compatible with a wide variety of file formats. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 14, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/toads-cruise/">http://www.nitrc.org/projects/toads-cruise/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="toolconnect" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ToolConnect">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=toolconnect" rel="bookmark" name="toolconnect">ToolConnect</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ToolConnect is an open source software for the functional connectivity analysis of in vitro neural networks. In the current version, ToolConnect implements correlation- (cross-correlation, partial-correlation) and information theory (joint entropy, transfer entropy) based algorithms, as well as useful and practical add-ons to visualize functional connectivity graphs and extract some topological features derived from graph theory. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 10, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/toolconnect/">http://www.nitrc.org/projects/toolconnect/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>DIBRIS <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="topographica" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Topographica">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=topographica" rel="bookmark" name="topographica">Topographica</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Topographica is a software package for computational modeling of neural maps, developed as part of the NIMH Human Brain Project under grant 1R01-MH66991. The goal is to help researchers understand brain function at the level of the topographic maps that make up sensory and motor systems.<br />
<br />
Topographica is intended to complement the many good low-level neuron simulators that are available, such as Genesis and Neuron. Those simulators focus on modeling the detailed internal behavior of neurons and small networks of them. Topographica instead focuses on the large-scale structure and function that is visible only when many thousands of such neurons are connected into topographic maps containing millions of connections. Topographica provides a general-purpose framework for building models at this level, at an appropriate level of detail and complexity, as determined by the available computing power, phenomena of interest, and amount of biological data available for validation. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 19, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/topographica/">http://www.nitrc.org/projects/topographica/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Edinburgh <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="topographic-tract-filtering" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Topographic Tract Filtering">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=topographic-tract-filtering" rel="bookmark" name="topographic-tract-filtering">Topographic Tract Filtering</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=connectivity-analysis" rel="tag">Connectivity Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tractography" rel="tag">Tractography</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The package contains the conference version of the TTF tool for filtering white matter tracts reconstructed from diffusion MRI based on a notion called topographic regularity. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1247&screenshot_id=1180"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1247&screenshot_id=1180" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 16, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>TTF Lisence</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/ttf/">http://www.nitrc.org/projects/ttf/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>USC <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="tortoise" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="TORTOISE">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=tortoise" rel="bookmark" name="tortoise">TORTOISE</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tensor-metric" rel="tag">Tensor Metric</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The TORTOISE software package is for processing diffusion MRI data. It contains two main modules, DIFF_PREP and DIFF_CALC.<br />
<br />
DIFF_PREP - software for image resampling, motion, eddy current distortion and susceptibility induced EPI distortion corrections, and for re-orientation of data to a common space<br />
<br />
DIFF_CALC - software for tensor fitting, error analysis, color map visualization and ROI analysis<br />
<br />
Software and sample data downloads, as well as documentation for the software, are available at <a href="http://www.tortoisedti.org." target="_new">http://www.tortoisedti.org.</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 29, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>TORTOISE License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.tortoisedti.org">http://www.tortoisedti.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="tractography-based-on-riemannian-geodesics-and-tensor-adjugates" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Tractography based on Riemannian geodesics and tensor adjugates">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=tractography-based-on-riemannian-geodesics-and-tensor-adjugates" rel="bookmark" name="tractography-based-on-riemannian-geodesics-and-tensor-adjugates">Tractography based on Riemannian geodesics and tensor adjugates</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This tool provides a way to trace fiber tracts based on Riemannian geodesics, the local cost function being defined as the adjugate (as opposed to the commonly used inverse) of the diffusion tensor. The minimization of the cost function is globally (not just locally) attained via dynamic programming, using the &quot;Fast sweeping&quot; algorithm for the sake of efficiency. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 8, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>3D Slicer License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/riemantract/">http://www.nitrc.org/projects/riemantract/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="tractor:-tractography-with-r" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="TractoR: Tractography with R">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=tractor:-tractography-with-r" rel="bookmark" name="tractor:-tractography-with-r">TractoR: Tractography with R</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=modeling" rel="tag">Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quantification" rel="tag">Quantification</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=shape-analysis" rel="tag">Shape Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=statistical-operation" rel="tag">Statistical Operation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=tractography" rel="tag">Tractography</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The TractoR (Tractography with R) project includes R packages for reading, writing and visualising magnetic resonance images stored in Analyze, NIfTI and DICOM file formats (DICOM support is read only). It also contains functions specifically designed for working with diffusion MRI, tractography and graph theory, including a standard implementation of the neighbourhood tractography approach to white matter tract segmentation. A shell interface is also provided, to run experiments with TractoR without interacting with R. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=224&screenshot_id=1102"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=224&screenshot_id=1102" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 5, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.tractor-mri.org.uk">http://www.tractor-mri.org.uk</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University College London <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Other UNIX-like</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, MGH/MGZ, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, MGH/MGZ, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="trafic---tracts-fibers-classification-using-deep-learning" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="TRAFIC - Tracts Fibers Classification Using Deep Learning">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=trafic---tracts-fibers-classification-using-deep-learning" rel="bookmark" name="trafic---tracts-fibers-classification-using-deep-learning">TRAFIC - Tracts Fibers Classification Using Deep Learning</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=diffusion-application" rel="tag">Diffusion Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=diffusion-mr-fiber-tracking" rel="tag">Diffusion Mr Fiber Tracking</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=neural-network" rel="tag">Neural Network</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> We present TRAFIC, a fully automated tool for the labeling and classification of brain fiber tracts. TRAFIC classifies new fibers using a neural network trained using shape features computed from previously traced and manually corrected fiber tracts. It is independent from a DTI Atlas as it is applied to already traced fibers. This work is motivated by medical applications where the process of extracting fibers from a DTI atlas, or classifying fibers manually is time consuming and requires knowledge about brain anatomy. With this new approach we were able to classify traced fiber tracts obtaining encouraging results. In this report we will present in detail the methods used and the results achieved with our approach. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1325&screenshot_id=1252"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1325&screenshot_id=1252" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 22, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/trafic/">http://www.nitrc.org/projects/trafic/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="trees-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="TREES toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=trees-toolbox" rel="bookmark" name="trees-toolbox">TREES toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> 1. Tools to automatically reconstruct neuronal branching from microscopy image stacks and to generate synthetic axonal and dendritic trees.<br />
2. The basic tools to edit, visualize and analyze dendritic and axonal trees.<br />
3. Methods for quantitatively comparing branching structures between neurons.<br />
4. Tools for exploring how dendritic and axonal branching depends on local optimization of total wiring and conduction distance.<br />
<br />
This software package is written in Matlab (Mathworks, Natick, MA), the most widely used scientific programming language. We hope that other groups will benefit from this package and that they will add their own code to the TREES toolbox based on their own specific applications. The code is therefore freely distributed. When publishing scientific work using this toolbox please cite the paper:<br />
<br />
&quot;Cuntz H, Forstner F, Borst A, Häusser M (2010). One rule to grow them all: A general theory of neuronal branching and its practical application. PLoS Comput Biol 6(8): e1000877.&quot; </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 13, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.treestoolbox.org">http://www.treestoolbox.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="trentool" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="TRENTOOL">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=trentool" rel="bookmark" name="trentool">TRENTOOL</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=information-theory" rel="tag">Information Theory</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surrogate-data-analysis" rel="tag">Surrogate Data Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=transfer-entropy-analysis" rel="tag">Transfer Entropy Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> TRENTOOL (TRansfer ENtropy TOOLbox) is an open source GPLv3 Matlab® toolbox for the analysis of information transfer in time series data.<br />
<br />
Transfer entropy (TE) is an information-theoretic measure of directed information transfer; it offers an approach to the detection of neuronal interactions that is free of an explicit model of the interactions and thus offers the power to analyze linear and nonlinear interactions alike.<br />
<br />
TRENTOOL provides user friendly routines for the estimation and statistical testing of TE in time series data. For the use with neural data TRENTOOL seamlessly integrates with the popular FieldTrip toolbox. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1095&screenshot_id=1056"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1095&screenshot_id=1056" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 5, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License v3</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://trentool.github.io/TRENTOOL3/">http://trentool.github.io/TRENTOOL3/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Goethe-University, Frankfurt, Germany <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="triangle-biosystems" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Triangle BioSystems">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=triangle-biosystems" rel="bookmark" name="triangle-biosystems">Triangle BioSystems</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=eeg" rel="tag">Eeg</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-software-resource" rel="tag">Other Software Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> TBSI is a biomedical device company focused on developing and manufacturing neural hardware solutions for application in medical research using animals when bio-monitoring, recording and stimulation functions are needed.<br />
<br />
- Neural Recording Equipment<br />
- Neural Stimulation Equipment<br />
- Data Acquisition Hardware and Software 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=592&screenshot_id=494"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=592&screenshot_id=494" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 2, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Commercial License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.trianglebiosystems.com/">http://www.trianglebiosystems.com/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>TBSI <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="tumorsim" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="TumorSim">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=tumorsim" rel="bookmark" name="tumorsim">TumorSim</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> TumorSim is a cross-platform simulation software that generates pathological ground truth from a healthy ground truth (e.g., the BrainWeb data). The software requires an input directory that describes a healthy anatomy (anatomical probabilities, mesh, diffusion tensor image, etc). Example output simulation images are available for download for validating various image analysis algorithms such as registration and segmentation algorithms.<br />
<br />
The simulation method is described in the following paper:<br />
Marcel Prastawa, Elizabeth Bullitt, and Guido Gerig. Simulation of Brain Tumors in MR Images for Evaluation of Segmentation Efficacy. Medical Image Analysis (MedIA), Vol 13, No 2, April 2009, Pages 297-311.<br />
<br />
Research efforts on the simulator was supported by NIH grant NIBIB R01 EB000219 (PI: Elizabeth Bullitt). 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=546&screenshot_id=458"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=546&screenshot_id=458" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 13, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/tumorsim/">http://www.nitrc.org/projects/tumorsim/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="turbo-brainvoyager" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Turbo-BrainVoyager">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=turbo-brainvoyager" rel="bookmark" name="turbo-brainvoyager">Turbo-BrainVoyager</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=application" rel="tag">Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Turbo-BrainVoyager (TBV) is a highly optimized, easy to use software package for the real-time analysis and dynamic visualization of functional magnetic resonance imaging data sets. Turbo-BrainVoyager allows to observe the working brain &quot;online&quot; by incrementally computing statistical maps as contrasts of a General Linear Model (GLM). The program also performs real-time pre-processing, including 3D motion correction, spatial Gaussian smoothing and temporal filtering (drift removal). 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=770&screenshot_id=701"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=770&screenshot_id=701" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 17, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Commercial License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.brainvoyager.com/products/turbobrainvoyager.html">http://www.brainvoyager.com/products/turbobrainvoyager.html</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Brain Innovation B.V. <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Microsoft</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>DICOM</td></tr>
			<tr class="even"><td>Output Formats</td><td>DICOM</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="turbofire" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="TurboFIRE">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=turbofire" rel="bookmark" name="turbofire">TurboFIRE</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=artifact-removal" rel="tag">Artifact Removal</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=connectivity-analysis" rel="tag">Connectivity Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=experimental-control" rel="tag">Experimental Control</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=intramodal" rel="tag">Intramodal</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=intrasubject" rel="tag">Intrasubject</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=linear" rel="tag">Linear</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quality-metrics" rel="tag">Quality Metrics</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Real-time task-based and resting-state fMRI analysis software </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 27, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Commerical Software</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.neurinsight.com/">http://www.neurinsight.com/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>NeurInsight LLC <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="turtleseg" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="TurtleSeg">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=turtleseg" rel="bookmark" name="turtleseg">TurtleSeg</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=intensity-contour" rel="tag">Intensity Contour</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> TurtleSeg is an interactive segmentation tool originally designed for 3D medical images. Accurate and automatic 3D medical image segmentation remains an elusive goal and manual intervention is often unavoidable. TurtleSeg implements techniques that allow the user to provide intuitive yet minimal interaction for guiding the 3D segmentation process. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=448&screenshot_id=325"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=448&screenshot_id=325" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 8, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>TurtleSeg License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.turtleseg.org">http://www.turtleseg.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Windows NT/2000<br>Windows Vista<br>Windows XP</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, MINC, NIfTI-1, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, MINC, NIfTI-1, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="ucla-multimodal-connectivity-database" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="UCLA Multimodal Connectivity Database">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=ucla-multimodal-connectivity-database" rel="bookmark" name="ucla-multimodal-connectivity-database">UCLA Multimodal Connectivity Database</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-display" rel="tag">Image Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=rendering" rel="tag">Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=web-resource" rel="tag">Web Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The UCLA Multimodal Connectivity Database - <a href="http://umcd.humanconnectomeproject.org" target="_new">http://umcd.humanconnectomeproject.org</a> - is a web-based analysis site and data repository for connectivity matrices that have been derived from neuroimaging data. The site is powered by the MGH/UCLA Human Connectome Project. Anyone can browse and analyze connectivity matrices that researchers have shared on this site. The data comes from different imaging modalities (fMRI, DTI, structural MRI, EEG), subject groups, and studies. The website allows users to choose any network shared by another user, compute graph theoretical metrics on the fly, and visualize the results. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=519&screenshot_id=393"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=519&screenshot_id=393" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 28, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Web Application - Free to use</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://umcd.humanconnectomeproject.org">http://umcd.humanconnectomeproject.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UCLA <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="uk-biobank---multimodal-imaging" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="UK Biobank - multimodal imaging">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=uk-biobank---multimodal-imaging" rel="bookmark" name="uk-biobank---multimodal-imaging">UK Biobank - multimodal imaging</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> UK Biobank is a large prospective population-based study of half a million men and women aged 40-69. The cohort combines a large population with an extensive range of measurements, including lifestyle, environmental and health-related factors, cohort-wide genome-wide genotyping, biomarkers and linkage to longitudinal medical records.<br />
<br />
UK Biobank is currently in the process of developing the worlds largest multimodal imaging resource which will feature 100,000 participants who have undergone brain, body and heart MRI, carotid ultrasound and DXA. This project started in 2015 and data is already available for &gt;10,000 participants. The NITRC community might be particularly interested in the neuroimaging data which consists of T1 and T2 scans, diffusion imaging, resting and task fMRI in additional to susceptibility-weighted imaging. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 26, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/uk_biobank/">http://www.nitrc.org/projects/uk_biobank/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UK Biobank <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="ultra-high-field-atlas-for-dbs-planning" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Ultra-high field atlas for DBS planning">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=ultra-high-field-atlas-for-dbs-planning" rel="bookmark" name="ultra-high-field-atlas-for-dbs-planning">Ultra-high field atlas for DBS planning</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> 7T T1-w and T2-w average atlas (ANTS) from 12 healthy controls, with manual labelling of deep and mid brain structures. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1102&screenshot_id=1062"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1102&screenshot_id=1062" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 12, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.khanlab.ca">http://www.khanlab.ca</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Western University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="umanitoba-jhu-functionally-defined-human-white-matter-atlas" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="UManitoba-JHU Functionally-Defined Human White Matter Atlas">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=umanitoba-jhu-functionally-defined-human-white-matter-atlas" rel="bookmark" name="umanitoba-jhu-functionally-defined-human-white-matter-atlas">UManitoba-JHU Functionally-Defined Human White Matter Atlas</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-data" rel="tag">Atlas Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This is a probabilistic atlas of human white matter tracts/regions underlying several well-known resting state brain networks (e.g., dorsal and ventral Default Mode, left and right Executive Control, anterior and posterior Salience, Basal Ganglia, Language, Sensorimotor, Visuospatial, etc.). These were derived by performing fMRI-guided DTI tractography on a group of 32 neurologically-healthy, adult control subjects. The atlas includes group probability maps for each network, as well as each individual tract (i.e., 1-mm isotropic NIFTI images) that are aligned to both the SPM and MRIStudio ICBM templates. In accordance with the CC BY-NC-ND 4.0 License, this atlas may be used for academic purposes, but users should cite the following two papers:<br />
<br />
1. Figley TD, Bhullar N, Courtney SM and Figley CR (2015). Front. Hum. Neurosci. 9:585.<br />
<br />
2. Figley TD, Mortazavi Moghadam B, Bhullar N, Kornelsen J, Courtney SM, and Figley CR (2017). Front. Hum. Neurosci. 11:306. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=983&screenshot_id=951"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=983&screenshot_id=951" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 16, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution-NonCommercial-NoDerivatives 4.0 International (CC BY-NC-ND 4.0)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/uofm_jhu_atlas/">http://www.nitrc.org/projects/uofm_jhu_atlas/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Manitoba and Johns Hopkins University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="umeå-brain-arteries" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Umeå Brain Arteries">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=umeå-brain-arteries" rel="bookmark" name="umeå-brain-arteries">Umeå Brain Arteries</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Umeå Brain Arteries (UBA24 and UBA167) are stereotactic atlases with probabilistic values describing the location of the main cerebral arteries. <br />
<br />
Data collection is described in:<br />
COBRA: A prospective multimodal imaging<br />
study of dopamine, brain structure and function, and cognition, Nevalainen et al., Brain Research, 2015 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1070&screenshot_id=1027"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1070&screenshot_id=1027" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 2, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial Share Alike</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/brainarteries/">http://www.nitrc.org/projects/brainarteries/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Umeå University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="unbiased-atlas-construction-via-population-(group-wise)-dramms-registration" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Unbiased Atlas Construction via Population (group-wise) DRAMMS Registration">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=unbiased-atlas-construction-via-population-(group-wise)-dramms-registration" rel="bookmark" name="unbiased-atlas-construction-via-population-(group-wise)-dramms-registration">Unbiased Atlas Construction via Population (group-wise) DRAMMS Registration</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Unbiased atlas construction by group-wise dramms registration. <br />
<br />
Can be used to construct a normal atlas from either a normal population or an abnormal population. <br />
<br />
The tool will find a virtual space that represents the mean geometry, mean anatomy and voxel-wise mean intensity of the input population (hence unbiased). <br />
<br />
If the input images are from an abnormality-bearing population, the above happens to normal regions only, and will create an unbiased normal-appearing atlas. The software can either take as input the manually annotated abnormality masks, or automatically detect and exclude abnormalities from atlas construction. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 3, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/popdramms/">http://www.nitrc.org/projects/popdramms/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>MGH <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="unbiased,-deformable-spatiotemporal-atlas-of-the-fetal-brain" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Unbiased, Deformable Spatiotemporal Atlas of the Fetal Brain">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=unbiased,-deformable-spatiotemporal-atlas-of-the-fetal-brain" rel="bookmark" name="unbiased,-deformable-spatiotemporal-atlas-of-the-fetal-brain">Unbiased, Deformable Spatiotemporal Atlas of the Fetal Brain</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The CRL has developed a mathematical framework for the generation of an unbiased, deformable spatiotemporal atlas of the fetal brain from magnetic resonance imaging (MRI) of normal fetuses scanned prenatally. <br />
<br />
Our atlas serves to capture the inter-subject anatomic variability of the fetal brain over the fetal brain growth period and is currently available between 27 weeks gestational age to 35 weeks. The atlas has been constructed following an unbiased minimum distance template estimation approach which utilizes symmetric diffeomorphic deformation and the cross-correlation (CC) similarity metric integrated with kernel regression in age. <br />
<br />
Visit <a href="http://crl.med.harvard.edu/research/fetal_brain_atlas/" target="_new">http://crl.med.harvard.edu/research/feta...</a> for more information. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 16, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Open Software License 3.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/crl_fetal_atlas/">http://www.nitrc.org/projects/crl_fetal_atlas/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Computational Radiology Laboratory <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="unc-0-1-2-infant-atlases" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="UNC 0-1-2 Infant Atlases">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=unc-0-1-2-infant-atlases" rel="bookmark" name="unc-0-1-2-infant-atlases">UNC 0-1-2 Infant Atlases</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Keyword: Infant; Pediatric; Template; Atlas; Longitudinal<br />
There are 3 atlases dedicated for neonates, 1-year-olds, and 2-year-olds. Each atlas comprises a set of 3D images made up of the intensity model, tissue probability maps, and anatomical parcellation map. These atlases are constructed with the help of state-of-the-art infant MR segmentation and groupwise registration methods, on a set of longitudinal images acquired from 95 normal infants (56 males and 39 females) at neonate, 1-year-old, and 2-year-old.<br />
<br />
Please refer to the following article for the atlases:<br />
Feng Shi, Pew-Thian Yap, Guorong Wu, Hongjun Jia, John H. Gilmore, Weili Lin, Dinggang Shen,&quot;Infant Brain Atlases from Neonates to 1- and 2-year-olds&quot;, PLoS ONE, 6(4): e18746, 2011.<br />
<br />
For more information and download link, please visit:<br />
<a href="http://www.med.unc.edu/bric/ideagroup/free-softwares/unc-infant-0-1-2-atlases" target="_new">http://www.med.unc.edu/bric/ideagroup/fr...</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=204&screenshot_id=682"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=204&screenshot_id=682" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 20, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/pediatricatlas/">http://www.nitrc.org/projects/pediatricatlas/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UNC <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Microsoft</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="unc-cedars-infant-atlas" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="UNC CEDARS INFANT ATLAS">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=unc-cedars-infant-atlas" rel="bookmark" name="unc-cedars-infant-atlas">UNC CEDARS INFANT ATLAS</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=spatial-transformation" rel="tag">Spatial Transformation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=time-domain-analysis" rel="tag">Time Domain Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> We derived the first set of normative functional brain atlases for infants (neonates, 1- and 2-year olds) by parcellating the infant brain into functionally homogeneous regions based on a novel hybrid iterative normalized cut approach. Infant-specific structural (i.e., AAL) atlases were used as spatial constraint. <br />
<br />
Please refer to the following article for the atlases:<br />
Feng Shi, Andrew P. Salzwedel, Weili Lin, John H. Gilmore, Wei Gao, &quot;Functional Brain Parcellations of the Infant Brain and the Associated Developmental Trends&quot;, Cerebral Cortex, 2017. <a href="https://doi.org/10.1093/cercor/bhx062" target="_new">https://doi.org/10.1093/cercor/bhx062</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 30, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/functionalatlas/">http://www.nitrc.org/projects/functionalatlas/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="unc-detail-preserved-longitudinal-0-3-6-9-12-months-old-atlas" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="UNC detail-preserved longitudinal 0-3-6-9-12 months-old atlas">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=unc-detail-preserved-longitudinal-0-3-6-9-12-months-old-atlas" rel="bookmark" name="unc-detail-preserved-longitudinal-0-3-6-9-12-months-old-atlas">UNC detail-preserved longitudinal 0-3-6-9-12 months-old atlas</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> We present a series of detail-preserved MRI longitudinal infant atlases. The atlases are 0-3-6-9-12 months-old respectively. For each time point, we built both T1-weighted &amp; T2-weighted atlases, and the probabilistic segmentation maps. Data are in NIFTI format. Please see the related publication for further detials.<br />
Zhang, Y., Shi, F., Wu, G., Wang, L., Yap, P.T. and Shen, D., 2016. Consistent spatial-temporal longitudinal atlas construction for developing infant brains. IEEE transactions on medical imaging, 35(12), pp.2568-2577. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1195&screenshot_id=1114"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1195&screenshot_id=1114" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 31, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/infant_atlas_4d/">http://www.nitrc.org/projects/infant_atlas_4d/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="unc-human-brain-atlas" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="UNC Human Brain Atlas">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=unc-human-brain-atlas" rel="bookmark" name="unc-human-brain-atlas">UNC Human Brain Atlas</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-data" rel="tag">Atlas Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Human brain atlases and databases generated at UNC-Chapel Hill. <br />
<br />
Current available atlases:<br />
- UNC/UCI Neonate hippocampus amygdala multi-atlas database<br />
- UNC/UCI Neonate scan rescan database : 6 subjects with each two scans of T1- and T2-weighted MR images<br />
- UNC Neonate multi/population atlas: Multi-atlas population for tissue segmentation in neonates, includes left-right mirrored images for symmetric analysis<br />
 - UNC-MNI Pediatric 1-year-old atlas: Symmetric atlas generated from 104 1-year-old subjects, combining children at high familial risk of autism and controls.<br />
 - Pediatric 4-year-old atlas: Symmetric atlas generated from 10 4-year-old healthy subjects.<br />
 - Adult atlas: Symmetric atlas generated from 50+ healthy adult subjects (20-59 year old).<br />
 - Elderly atlas: Atlas generated from 27 healthy elderly subjects (60+ years old).<br />
<br />
Additional information and acknowledgment for their usage can be found by clicking on the release notes. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=288&screenshot_id=197"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=288&screenshot_id=197" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 23, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial Share Alike</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/unc_brain_atlas">http://www.nitrc.org/projects/unc_brain_atlas</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina at Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Nrrd</td></tr>
			<tr class="even"><td>Output Formats</td><td>Nrrd</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="unc-neonate-and-infant-cortical-surface-atlases" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="UNC Neonate and Infant Cortical Surface Atlases">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=unc-neonate-and-infant-cortical-surface-atlases" rel="bookmark" name="unc-neonate-and-infant-cortical-surface-atlases">UNC Neonate and Infant Cortical Surface Atlases</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-data" rel="tag">Atlas Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> UNC infant surface atlases of cortical structures include 2 atlases, 1) The UNC 4D infant cortical surface atlas, which includes surface atlases at 1, 3, 6, 9, 12, 18, 24, 36, 48, 60 and 72 months, thus densely covering and well characterizing the critical stages of the dynamic infant brain development; 2) The spatiotemporal neonatal cortical surface atlas, which mainly focuses on the neonatal cortical surface, includes surface atlases at 39, 40, 41, 42, 43, and 44 gestational weeks. Please refer to the following article for the atlases: Li, G., Wang, L., Shi, F., Gilmore, J. H., Lin, W., &amp; Shen, D. (2015). Construction of 4D high-definition cortical surface atlases of infants: Methods and applications. Medical image analysis, 25(1). Wu Z., Li, G., Wang L., Lin, W., Gilmore, J.H., &amp; Shen, D. (2018). Construction of spatiotemporal neonatal cortical surface atlas using a large-scale dataset. IEEE intern. Symp. on Biomed. Imaging (ISBI). 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1208&screenshot_id=1129"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1208&screenshot_id=1129" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 28, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/infantsurfatlas/">http://www.nitrc.org/projects/infantsurfatlas/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UNC Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="unc-primate-brain-atlas" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="UNC Primate Brain Atlas">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=unc-primate-brain-atlas" rel="bookmark" name="unc-primate-brain-atlas">UNC Primate Brain Atlas</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-data" rel="tag">Atlas Data</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This symmetric atlas of the primate brain has been created using 18 cases of rhesus macaques aged 16-34 months. It includes the T1-weighted image (with and without skull), and also tissue segmentation probability maps (white matter, gray matter, CSF, rest), subcortical structures segmentation (amygdala, caudate, hippocampus, pallidus, putamen), and a lobar parcellation map.<br />
<br />
You can find more details about the creation of this atlas in the following paper : <br />
M. Styner, R. Knickmeyer, S. Joshi, C. Coe, S. J. Short, and J. Gilmore. Automatic brain segmentation in rhesus monkeys. In Proc SPIE Vol 6512, Medical Imaging, 2007, pp. 65122 L1-8 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=197&screenshot_id=103"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=197&screenshot_id=103" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 11, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Share Alike</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/primate_atlas/">http://www.nitrc.org/projects/primate_atlas/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina at Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Nrrd</td></tr>
			<tr class="even"><td>Output Formats</td><td>Nrrd</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="unc/utah-namic-dti-fiber-analysis-framework" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="UNC/Utah NAMIC DTI Fiber Analysis Framework">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=unc/utah-namic-dti-fiber-analysis-framework" rel="bookmark" name="unc/utah-namic-dti-fiber-analysis-framework">UNC/Utah NAMIC DTI Fiber Analysis Framework</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=diffusion-application" rel="tag">Diffusion Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This project can be seens as a master project encompassing several current NITRC projects into a coherent set. The project will host binary packaged distributions, scripts, example datasets, and corresponding results of analysis using our UNC/Utah NAMIC DTI Fiber Analysis Framework. <br />
<br />
Our workflow utilizes tools already available on NITRC including: <br />
-DTIPrep (available as a 3D Slicer extension <a href="http://www.slicer.org" target="_new">http://www.slicer.org</a>)<br />
-DTIAtlasBuilder (available as a 3D Slicer extension <a href="http://www.slicer.org" target="_new">http://www.slicer.org</a>)<br />
-FiberViewerLight (available as a 3D Slicer extension <a href="http://www.slicer.org" target="_new">http://www.slicer.org</a>)<br />
-DTIAtlasFiberAnalyzer (available as a 3D Slicer extension <a href="http://www.slicer.org" target="_new">http://www.slicer.org</a>)<br />
-FADTTS with the updated GUI FADTTSter </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 8, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/namicdtifiber/">http://www.nitrc.org/projects/namicdtifiber/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina at Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="unc-wisconsin-neurodevelopment-rhesus-mri-database" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="UNC-Wisconsin Neurodevelopment Rhesus MRI Database">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=unc-wisconsin-neurodevelopment-rhesus-mri-database" rel="bookmark" name="unc-wisconsin-neurodevelopment-rhesus-mri-database">UNC-Wisconsin Neurodevelopment Rhesus MRI Database</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This is a macaque brain MRI database characterizing the normal postnatal macaque brain development. This longitudinal primate database was acquired from a cohort of healthy macaque monkeys ranging from a few week olds up to 3-year-old adolescents.<br />
<br />
Each scan consists of structural (both T1 and T2) and diffusion MRI. T1 and T2 are provided as NRRD format in the original scanner space as well as in a common atlas space where they are rigidly aligned. Diffusion MRI is provided as raw diffusion weighted images. In addition, pre-processing was performed including motion and eddy current distortion correction on the diffusion weighted MR and tensors were calculated with diffusion property maps (FA, MD, Baseline and iDWI).<br />
<br />
For further information, see also Young JT et al. The UNC-Wisconsin Rhesus Macaque Neurodevelopment Database: A Structural MRI and DTI Database of Early Postnatal Development. Frontiers in Neuroscience. 2017 Feb 2;11:292–11. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=794&screenshot_id=726"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=794&screenshot_id=726" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 3, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/uncuw_macdevmri/">http://www.nitrc.org/projects/uncuw_macdevmri/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of North Carolina at Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="unfold-toolbox-for-regression-based-eeg-analysis" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Unfold toolbox for regression-based EEG analysis">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=unfold-toolbox-for-regression-based-eeg-analysis" rel="bookmark" name="unfold-toolbox-for-regression-based-eeg-analysis">Unfold toolbox for regression-based EEG analysis</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> unfold is a MATLAB tool for regression-based EEG analysis that was written to facilitate the use of advanced deconvolution models and spline regression in event-related potential (ERP) research. In experiments that involve temporally overlapping brain responses, unfold can be used to isolate the responses to the individual events (linear deconvolution). Simultaneously, it can model influences of various linear or non-linear covariates on the neural response (using spline regression/generalized additive modeling). The toolbox is compatible with EEGLAB and offers built-in functions to visualize the model coefficients (betas) of each predictor as waveforms or scalp topographies (&quot;regression-ERPs&quot;). Alternatively, results can be easily exported as plain text or transferred to other toolboxes (e.g. EEGLAB or Fieldtrip). In addition to EEG, unfold can be used to analyze other overlapping biometric signals, such as electrodermal activity, or pupil responses. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1290&screenshot_id=1204"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1290&screenshot_id=1204" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 9, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.unfoldtoolbox.org">http://www.unfoldtoolbox.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Universität Osnabrück <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="university-of-illinois-cognitive-neuroimaging-laboratory-optical-brain-imaging-software" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="University of Illinois Cognitive Neuroimaging Laboratory Optical Brain Imaging Software">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=university-of-illinois-cognitive-neuroimaging-laboratory-optical-brain-imaging-software" rel="bookmark" name="university-of-illinois-cognitive-neuroimaging-laboratory-optical-brain-imaging-software">University of Illinois Cognitive Neuroimaging Laboratory Optical Brain Imaging Software</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-software-resource" rel="tag">Other Software Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> “CNL Optical Brain Imaging Software” is a suite of tools for designing and analyzing whole head optical data, both &quot;slow (NIRS) and &quot;fast&quot; (EROS). These tools are designed to work together to simplify, as much as possible, the co-registration of digitized source and detector locations (“OCP”), correction of pulse artifacts, filtering and averaging (“p_pod”), motion artifacts (kbWF) and 3D reconstruction and statistical analysis (“opt3d”). Most of these steps are implemented as Matlab subroutines, which can be called by batch controllers or used independently. Currently, the final stages (“opt3d”) are implemented in Lahey FORTRAN. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=868&screenshot_id=976"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=868&screenshot_id=976" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 2, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution-NonCommercial-NoDerivs CC BY-NC-ND</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/uofi-cnl-opt/">http://www.nitrc.org/projects/uofi-cnl-opt/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Illinois <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="user-friendly-functional-connectivity---uf²c" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="User Friendly Functional Connectivity - UF²C">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=user-friendly-functional-connectivity---uf²c" rel="bookmark" name="user-friendly-functional-connectivity---uf²c">User Friendly Functional Connectivity - UF²C</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-software-resource" rel="tag">Other Software Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> UF²C is an open source software developed at the Neuroimaging Laboratory at Unicamp that aims to simplify and organize functional studies in neuroimaging through clean and validated methodologies.<br />
The graphical user interface makes the processing and analysis options accessible for neuroscientists, with reasonable choices of default settings. UF²C allows the user to study functional connectivity both through a quantitative view that provides detailed values of average connectivity and through a spatial view that provides statistical maps that can be directly used for further analyses. All results are carefully organized in a distinct folder for each subject, and a common folder is generated with a log file reporting the quantitative results of all the analyzed subjects.<br />
Several UF²C modalities and tools runs combined with Statistical Parametric Mapping functions.<br />
 Coordinator and developer<br />
 Brunno M. de Campos, Ph.D.<br />
 Collaborator Developer<br />
 Raphael Fernandes Casseb, Ph.D. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1319&screenshot_id=1236"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1319&screenshot_id=1236" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 1, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License, GNU Lesser General Public License and GNU General Public License, v</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/uf2c/">http://www.nitrc.org/projects/uf2c/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Campinas <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Microsoft</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="using-make-for-neuroimaging-workflow:-manual-and-examples" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Using Make for Neuroimaging Workflow: Manual and Examples">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=using-make-for-neuroimaging-workflow:-manual-and-examples" rel="bookmark" name="using-make-for-neuroimaging-workflow:-manual-and-examples">Using Make for Neuroimaging Workflow: Manual and Examples</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=platform-or-development-environment" rel="tag">Platform Or Development Environment</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Make Pipelines is a set of examples of neuroimaging workflows, implemented using the popular UNIX utility &quot;make&quot;, documented in an extensive manual. These examples span structural MRI processing, fMRI processing (resting and task), diffusion tensor imaging, quantitative perfusion imaging, and reporting of fMRI results.<br />
<br />
See documentation link for related paper and manual. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 21, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/makepipelines/">http://www.nitrc.org/projects/makepipelines/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Washington <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="vaa3d" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Vaa3D">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=vaa3d" rel="bookmark" name="vaa3d">Vaa3D</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?tag=big-image-data" rel="tag">Big Image Data</a>, <a href="https://hbp-hpc-platform.fz-juelich.de/?tag=image-processing" rel="tag">Image Processing</a>, <a href="https://hbp-hpc-platform.fz-juelich.de/?tag=image-visualization" rel="tag">Image Visualization</a></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=application" rel="tag">Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description><b><a href="http://vaa3d.org">Vaa3D</a></b> is an open source 3D/4D/5D image visualization and analysis software
      for bioimage analysis. It is a cross-platform (Mac, Linux, and Windows) tool for visualizing large-scale (gigabytes, and 64-bit data) image volumes and surface objects.
      It is also a container of powerful modules/plugins for high dimensional image analysis (cell segmentation, neuron tracing, brain registration, annotation, quantitative measurement and statistics, etc).
      This makes Vaa3D suitable for various bioimage informatics applications, and a nice platform to develop new 3D image analysis algorithms for high-throughput processing.
      In short, Vaa3D streamlines the workflow of visualization-assisted analysis.</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>March 2019</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>MIT license</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td>v3.597</td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://vaa3d.org">http://vaa3d.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href="https://github.com/Vaa3D">https://github.com/Vaa3D</a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Konrad Rokicki <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop<br>VR</td></tr>
			<tr class="odd"><td>Input Formats</td><td>TIFF, Vaa3D raw</td></tr>
			<tr class="even"><td>Output Formats</td><td>APO, SWC, TIFF, Vaa3D raw</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="vaa3d-and-vaa3d-neuron" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Vaa3D and Vaa3D-Neuron">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=vaa3d-and-vaa3d-neuron" rel="bookmark" name="vaa3d-and-vaa3d-neuron">Vaa3D and Vaa3D-Neuron</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=fiber-tracking" rel="tag">Fiber Tracking</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-display" rel="tag">Image Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=neuronal-characterization" rel="tag">Neuronal Characterization</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=platform-or-development-environment" rel="tag">Platform Or Development Environment</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=position" rel="tag">Position</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=registration" rel="tag">Registration</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=rendering" rel="tag">Rendering</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=resampling" rel="tag">Resampling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=shape-analysis" rel="tag">Shape Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=surface-analysis" rel="tag">Surface Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=volumetric-analysis" rel="tag">Volumetric Analysis</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=warping" rel="tag">Warping</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Vaa3D (3D Visualization-Assisted Analysis) is a handy, fast, and versatile 3D/4D/5D Image Visualization &amp; Analysis System for Bioimages &amp; Surface Objects. Vaa3D is a cross-platform (Mac, Linux, and Windows) tool for visualizing large-scale (gigabytes, and 64-bit data) 3D/4D/5D image stacks and various surface data. It is also a container of powerful modules for 3D image analysis (cell segmentation, neuron tracing, brain registration, annotation, quantitative measurement and statistics, etc) and data management. <br />
<br />
<br />
<br />
Vaa3D is very easy to be extended via a powerful plugin interface. For example, many ITK tools are being converted to Vaa3D Plugins.<br />
<br />
Vaa3D-Neuron is built upon Vaa3D to make 3D neuron reconstruction much easier. In a recent Nature Biotechnology paper (2010, 28(4), pp.348-353) about Vaa3D and Vaa3D-Neuron, an order of magnitude of performance improvement (both reconstruction accuracy and speed) was achieved compared to other tools. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=379&screenshot_id=921"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=379&screenshot_id=921" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 3, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.vaa3d.org">http://www.vaa3d.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Cygwin<br>Linux<br>MacOS<br>Windows 95/98/2000<br>Windows NT/2000<br>Windows Vista<br>Windows XP</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="validate29-squirrel-monkey-brain-atlas" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="VALiDATe29 squirrel monkey brain atlas">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=validate29-squirrel-monkey-brain-atlas" rel="bookmark" name="validate29-squirrel-monkey-brain-atlas">VALiDATe29 squirrel monkey brain atlas</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-data" rel="tag">Atlas Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The VALiDATe29 squirrel monkey brain atlas was created from MRI scans of 29 squirrel monkey brains. The atlas is currently comprised of multiple anatomical templates (proton density, T1, T2* weighted), diffusion MRI templates (fractional anisotropy, mean diffusivity), and ex vivo templates (fractional anisotropy and a structural MRI). In addition, the templates are combined with histologically defined cortical labels, and diffusion tractography defined white matter labels. The atlas was created by researchers at Vanderbilt University Institute of Imaging Science. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1114&screenshot_id=1070"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1114&screenshot_id=1070" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 1, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/validate29/">http://www.nitrc.org/projects/validate29/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Vanderbilt University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="valmap:-simple-statistical-mapping-tool" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ValMap: simple statistical mapping tool">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=valmap:-simple-statistical-mapping-tool" rel="bookmark" name="valmap:-simple-statistical-mapping-tool">ValMap: simple statistical mapping tool</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=linear" rel="tag">Linear</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=morphology" rel="tag">Morphology</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> valmap is a command line voxel-wise statistical analysis program for images. Images can be gray matter density, jacobian images, etc.<br />
<br />
The linear model is implemented, i.e. designs that can be modeled as Y=AB, where Y is a vector or matrix of dependent variables, B is a vector or matrix of parameters to be estimated, and A is a design matrix. <br />
<br />
Why use valmap?<br />
<br />
1. Do not need a Matlab license to run.<br />
2. Can incorporate a spatially varying independent variable (e.g., you have a perfusion map as your dependent variable, and you want to co-vary for gray matter at each voxel, so use a gray matter map as an independent variable).<br />
3. Can use spatially invariant independent variables (e.g., you can have a cognitive test score as the dependent variable, and use jacobian maps as the independent variable).<br />
4. Can have multiple dependent variables and do multivariate analyses (e.g., want to know the overall effect of disease on perfusion and structure, so use perfusion maps and jacobian maps as dependent variables). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 6, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/valmap/">http://www.nitrc.org/projects/valmap/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>Windows XP</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="vamca-cortical-meta-analysis-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="VAMCA Cortical Meta-analysis Toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=vamca-cortical-meta-analysis-toolbox" rel="bookmark" name="vamca-cortical-meta-analysis-toolbox">VAMCA Cortical Meta-analysis Toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database-application" rel="tag">Database Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> VAMCA: Visualization And Meta-analysis on Cortical Anatomy is a stand-alone, open source human cortical meta-analysis and visualization toolbox for MatLab. It projects stereotaxic coordinates to a mean cortical surface by using an anatomical database of 60 young adults to provide multiple mappings of normalized cortical surfaces into MNI space. VAMCA performs the following analyses:<br />
<br />
1) Multi-Fiducial Projection Mapping: Map stereotaxic 3D coordinates to the normalized cortical location for each of 60 database subjects.<br />
2) Computing Centroid Locations for groups of foci both on a mean cortical surface and in MNI space.<br />
3) Comparing Two Groups of Foci for differences in location (surface or 3D) of their group centroids and computing the groups' overlap extent using permutation tests.<br />
4) Detecting Significant Densities of Foci or Density Differences of Two Groups within anatomical ROIs on a mean cortical surface by using Monte Carlo analyses. Coordinate weights allow fixed or random effects type analyses. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=234&screenshot_id=142"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=234&screenshot_id=142" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 18, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.ebire.org/hcnlab/software/vamca.html">http://www.ebire.org/hcnlab/software/vamca.html</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>Windows 95/98/2000<br>Windows NT/2000<br>Windows XP</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="vandpire" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="VANDPIRE">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=vandpire" rel="bookmark" name="vandpire">VANDPIRE</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quantification" rel="tag">Quantification</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> VANDPIRE is intuitive and comprehensive software for quantifying cerebral blood flow (CBF) and calculating CBF territories using arterial spin labeling (ASL) MRI data. CBF analysis, particularly CBF territory analysis, is time-consuming and requires advanced knowledge of scripting and up-to-date mathematical models. This toolbox addresses these concerns, and is available on multiple platforms as a graphical user interface.<br />
<br />
<br />
<br />
<br />
Features:<br />
- Available on Windows, Mac, and Linux platforms<br />
- Runs from single folder and thus capable of being run online in scanning environment<br />
- Comes with complete help tutorial in HTML format and example data<br />
- Numerous customizable options that are responsive to user input<br />
- Output data formats: Nifti, Dicom<br />
- Includes modifiable motion correction and coregistration options using FSL or Elastix<br />
- Currently tested with Philips Par/Rec and Philips Dicom pseudocontinuous ASL input data; however, support available for other data formats upon request<br />
<br />
<br />
Created by Daniel F. Arteaga. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=962&screenshot_id=944"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=962&screenshot_id=944" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 31, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>VANDPIRE software license agreement</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/vandpire/">http://www.nitrc.org/projects/vandpire/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Vanderbilt University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS<br>Microsoft<br>POSIX/UNIX-like</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>DICOM, NIfTI-1, Philips PAR/REC</td></tr>
			<tr class="even"><td>Output Formats</td><td>DICOM, NIfTI-1, Philips PAR/REC</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="vartbx-–-variability-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="VarTbx – Variability Toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=vartbx-–-variability-toolbox" rel="bookmark" name="vartbx-–-variability-toolbox">VarTbx – Variability Toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=time-domain-analysis" rel="tag">Time Domain Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> VarTbx measures within-voxel time series variability in fMRI data. The VarTbx structure is intended to be similar to a standard SPM first-level analysis. You can then proceed to pass those first-level, variability-based images to a level-2 SPM analysis in order to model group effects of interest. However, the first-level NIFTI output files could also be used within other statistics programs of your choice (e.g., FSL, AFNI, PLS).<br />
<br />
VarTbx currently supports modeling block designs with a boxcar model, and computes temporal variability using measures such as: detrended variance (VAR), detrended standard deviation (SD), mean squared successive difference (MSSD), and SQRT(MSSD). We plan to add additional modeling approaches and variability measures in future releases. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 16, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/vartbx/">http://www.nitrc.org/projects/vartbx/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Max Planck Institute for Human Development <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="vectorvaluedhistogramnormalizer" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="VectorValuedHistogramNormalizer">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=vectorvaluedhistogramnormalizer" rel="bookmark" name="vectorvaluedhistogramnormalizer">VectorValuedHistogramNormalizer</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Implement the methods defined in <a href="http://www5.informatik.uni-erlangen.de/Forschung/Publikationen/2006/Jaeger06-ANM.pdf" target="_new">http://www5.informatik.uni-erlangen.de/F...</a> as and ITK filter. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 29, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/vvhistomatch/">http://www.nitrc.org/projects/vvhistomatch/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="vervet-probabilistic-atlas" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Vervet Probabilistic Atlas">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=vervet-probabilistic-atlas" rel="bookmark" name="vervet-probabilistic-atlas">Vervet Probabilistic Atlas</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-data" rel="tag">Atlas Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The vervet (Chlorocebus aethiops sabaeus) probabilistic atlas defines an anatomical space (template) with associated tissue and regional prior probability maps. The atlas was produced from whole head MRI of 10 normal adult animal subjects. The package consists of two atlases. The &quot;Biased&quot; directory contains the average template and probabilistic atlases for selected tissue classes constructed by registering the training population to one subject. The &quot;Unbiased&quot; directory contains the atlas constructed using unbiased estimation. The atlas is suitable for use in any segmentation tool using a probabilistic atlas, for example those in Slicer. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=388&screenshot_id=297"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=388&screenshot_id=297" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 2, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Share Alike</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/vervet_atlas/">http://www.nitrc.org/projects/vervet_atlas/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="viewpixx" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="VIEWPixx">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=viewpixx" rel="bookmark" name="viewpixx">VIEWPixx</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=physiological-recording" rel="tag">Physiological Recording</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=physiological-stimulation" rel="tag">Physiological Stimulation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=response-monitoring" rel="tag">Response Monitoring</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=stimulus-presentation" rel="tag">Stimulus Presentation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The VIEWPixx is a complete display toolbox which has been conceived specifically to replace CRTs in vision science labs. The VIEWPixx features high-performance industrial LCD glass, and a panel controller which has been custom designed to support vision research.Our innovative LED backlight design features superior display uniformity, and a wide color gamut exceeding that of any CRT. In addition, it includes an array of peripherals which often need to be synchronized to video during an experiment, including a stereo audio stimulator, a button box port for precise reaction-time measurement, triggers for electrophysiology equipment, and even a complete analog I/O subsystem. Because we implemented the video controller and peripheral control on the same circuit board, you can now successfully synchronize all of your subject I/O to video refresh with microsecond precision. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=593&screenshot_id=495"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=593&screenshot_id=495" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Apr 3, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Commercial License: VPixx</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://vpixx.com/our-products/crt-replacement/">http://vpixx.com/our-products/crt-replacement/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>VPixx Technologies <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="viewpixx-/3d" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="VIEWPixx /3D">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=viewpixx-/3d" rel="bookmark" name="viewpixx-/3d">VIEWPixx /3D</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=physiological-recording" rel="tag">Physiological Recording</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=physiological-stimulation" rel="tag">Physiological Stimulation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=response-monitoring" rel="tag">Response Monitoring</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=stimulus-presentation" rel="tag">Stimulus Presentation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The VIEWPixx /3D is a complete display toolbox which has been developed specifically to replace CRTs in vision science labs. The VIEWPixx /3D incorporates industrial LCD glass, and a controller which has been custom designed to support vision research. Our innovative LED backlight design features superior display uniformity, and a wide color gamut exceeding that of any CRT. In addition, it includes an array of peripherals which often need to be synchronized (with microsecond precision) to video during an experiment, including a stereo audio stimulator, a button box port for precise reaction-time measurement, triggers for electrophysiology equipment, and even a complete analog I/O subsystem. It is perfectly suited for presentation of dynamic stimuli, it features 1920 x 1080 resolution, 10-bit intensity on each colour and true 120Hz refresh rate. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=731&screenshot_id=655"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=731&screenshot_id=655" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 20, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://vpixx.com/products/viewpixx-3d/">http://vpixx.com/products/viewpixx-3d/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>VPixx Technologies <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="viking-family-of-viewing/annotation/analysis-tools-for-connectomics" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Viking family of Viewing/Annotation/Analysis tools for Connectomics">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=viking-family-of-viewing/annotation/analysis-tools-for-connectomics" rel="bookmark" name="viking-family-of-viewing/annotation/analysis-tools-for-connectomics">Viking family of Viewing/Annotation/Analysis tools for Connectomics</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Viking is a multi-user modular viewing and annotation system which functions with any type of stacked 2D imagery registered into a 3D volume. Originally designed for transmission electron microscopy connectome data Viking has been used for confocal, bright field, and OCT images. <br />
<br />
Viking provides multi-user concurrent annotation of data sets using a variety of 1D or 2D shapes, including arbitrary polygons. Annotations are stored in a spatial database. The largest current database has well over a million annotations.<br />
<br />
A paper describing Viking is here:<br />
<a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2818.2010.03402.x/abstract" target="_new">http://onlinelibrary.wiley.com/doi/10.11...</a><br />
<br />
Collaborating labs host images, typically registered with the Nornir tools, locally. Annotations are either stored on the Viking servers or optionally labs deploy local back-end servers with assistance from the developer. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=419&screenshot_id=1224"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=419&screenshot_id=1224" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 15, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/viking_viewer/">http://www.nitrc.org/projects/viking_viewer/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Utah <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="vimeapy" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ViMEAPy">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=vimeapy" rel="bookmark" name="vimeapy">ViMEAPy</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Virtual MEA measurements in Python (ViMEAPy) is a Python toolbox for calculating the extracellular potential arising from a known current source distribution, in the in vitro slice setting, where a thin slice of neural tissue is immersed in saline on top of a Micro Electrode Array (MEA). For this experimental set-up, calculating the extracellular potentials arising from a known distribution of current sources is a non-trivial problem because of the heterogeneous conductivity of the system. We provide here a Python toolbox that facilitates such modelling by use of the Method of Images.<br />
<br />
For specific details, validations, and usage, please go to the article &quot;Modelling and Analysis of Electrical Potentials Recorded in Microelectrode Arrays (MEAs)&quot; which is available here: <a href="http://link.springer.com/article/10.1007%2Fs12021-015-9265-6" target="_new">http://link.springer.com/article/10.1007...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 6, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/vimeapy/">http://www.nitrc.org/projects/vimeapy/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Norwegian University of Life Sciences <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="viola" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="VIOLA">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=viola" rel="bookmark" name="viola">VIOLA</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?tag=image-visualization" rel="tag">Image Visualization</a>, <a href="https://hbp-hpc-platform.fz-juelich.de/?tag=medical-visualization" rel="tag">Medical Visualization</a>, <a href="https://hbp-hpc-platform.fz-juelich.de/?tag=neurons" rel="tag">Neurons</a></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=application" rel="tag">Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description><b><a href="https://github.com/HBPVIS/VIOLA">VIOLA</a></b> (Visualization Of Layer Activity) is an interactive, web-based tool to visualize activity data in multiple 2D layers such as the simulation output of neuronal networks with 2D geometry.
<p/><p/>
A usage example demonstrates the visualization of spike data resulting from a NEST simulation of a spatially structured point-neuron network with excitatory and inhibitory neuron populations and an external stimulus.</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>November 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GPLv2</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://hbpvis.github.io/VIOLA/">http://hbpvis.github.io/VIOLA/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href="https://github.com/HBPVIS/VIOLA">https://github.com/HBPVIS/VIOLA</a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Benjamin Weyers <a href="mailto:"></a><br>Espen Hagen <a href="mailto:"></a><br>Johanna Senk <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Web</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>DAT, GDF, JSON, LFP</td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="virtual-brain-generator" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Virtual Brain Generator">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=virtual-brain-generator" rel="bookmark" name="virtual-brain-generator">Virtual Brain Generator</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> MATLAB code to reverse-engineer neural networks (Virtual Brains) whose dynamics reproduce large-scale activity patterns of real brains observed with EEG and MEG in health and disease. The model of the virtual brain was originally presented in:<br />
<br />
G. Karl Steinke and Roberto F. Galán (2011). Brain rhythms reveal a hierarchical network organization. PLoS Computational Biology, 7(10): e1002207. <br />
Key words: EEG, neural complexity, inverse-eigenvalue problem, network oscillations, brain connectivity.<br />
<br />
<a href="http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1002207" target="_new">http://journals.plos.org/ploscompbiol/ar...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 13, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://case.edu/med/galanlab/software.html">https://case.edu/med/galanlab/software.html</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="virtual-digital-brain" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Virtual digital brain">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=virtual-digital-brain" rel="bookmark" name="virtual-digital-brain">Virtual digital brain</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=directed-transfer-analysis" rel="tag">Directed Transfer Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This software can be freely downloaded from the website: <a href="https://www.nitrc.org" target="_new">https://www.nitrc.org</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 4, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Free For Non-Commercial Use Only</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/vdb/">http://www.nitrc.org/projects/vdb/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Taishan medical university <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Windows Vista</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="virtual-electrode-recording-tool-for-extracellular-potentials-(vertex)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Virtual Electrode Recording Tool for EXtracellular potentials (VERTEX)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=virtual-electrode-recording-tool-for-extracellular-potentials-(vertex)" rel="bookmark" name="virtual-electrode-recording-tool-for-extracellular-potentials-(vertex)">Virtual Electrode Recording Tool for EXtracellular potentials (VERTEX)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Virtual Electrode Recording Tool for EXtracellular potentials (VERTEX) is a Matlab tool for simulating extracellular potential recordings in spiking neural network (SNN) models. VERTEX is designed to facilitate the simulation of extracellular potentials generated by activity in SNNs; in particular, spatially-organised networks containing thousands or hundreds of thousands of neurons. VERTEX’s interface and model specification options were designed with this particular task in mind. It is therefore less flexible than other neural simulators, but the limited scope has allowed us to simplify the user interface so that a simulation can be specified simply by setting some parameters and run using a few function calls. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 26, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Non-Commercial Software License Agreement</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.vertexsimulator.org/">http://www.vertexsimulator.org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Newcastle University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="virtual-reality-brain-network-viewer" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Virtual Reality Brain Network Viewer">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=virtual-reality-brain-network-viewer" rel="bookmark" name="virtual-reality-brain-network-viewer">Virtual Reality Brain Network Viewer</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The Virtual Reality Brain Network Viewer provides a new and innovative way to interact with brain connectome data. By utilizing the Oculus Rift headset and the Oculus Touch controllers the software allows you to physically interact with your data sets. This includes the ability to pick up the brain model and move it with your hands, as well as the ability to slice open the brain mesh to further examine the nodes and connections within. The software also comes with analysis tools that allow control over the connection threshold, the ability to isolate connections on a given node, and the overlay of MRI data on the brain model. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1244&screenshot_id=1159"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1244&screenshot_id=1159" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 5, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/vrnetworkviewer/">http://www.nitrc.org/projects/vrnetworkviewer/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UNC Chapel Hill <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="visapy" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="ViSAPy">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=visapy" rel="bookmark" name="visapy">ViSAPy</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ViSAPy (Virtual Spiking Activity in Python) is a tool for generation of biophysically realistic benchmark data for evaluation of spike sorting algorithms. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 5, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/visapy/">http://www.nitrc.org/projects/visapy/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="vismotionqcdb" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="VisMotionQCDB">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=vismotionqcdb" rel="bookmark" name="vismotionqcdb">VisMotionQCDB</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This is a resource to train researchers to visually evaluate the degree of motion artifact present in structural MR images. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 14, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Apache License 2.0</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/vismotionqcdb/">http://www.nitrc.org/projects/vismotionqcdb/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>UNC School of Medicine <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="vist/e" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="vIST/e">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=vist/e" rel="bookmark" name="vist/e">vIST/e</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> vIST/e is a C++ application that allows interactive visualization of various kinds of imaging datasets, both simple scalar data as well as complex high-dimensional data such as diffusion tensor imaging (DTI) and high angular resolution imaging data (HARDI). vIST/e is based on VTK, Qt and is designed as a plugin system which allows it to be extended in a flexible way. vIST/e is the main research tool of Imaging Science and Technology Eindhoven (IST/e), a major imaging group Eindhoven University of Technology, the Netherlands which brings together all university faculties involved in medical or industrial imaging. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 16, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/viste/">http://www.nitrc.org/projects/viste/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Eindhoven University of Technology <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="visualization-toolkit" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Visualization Toolkit">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=visualization-toolkit" rel="bookmark" name="visualization-toolkit">Visualization Toolkit</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?tag=image-visualization" rel="tag">Image Visualization</a></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=application" rel="tag">Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description>
      <b><a href="https://www.vtk.org">The Visualization Toolkit (VTK)</a></b> is open source software for manipulating and displaying scientific data.
      It comes with state-of-the-art tools for 3D rendering, a suite of widgets for 3D interaction, and extensive 2D plotting capability.
<p/><p/>
VTK is part of Kitware's collection of supported platforms for software development.
      The platform is used worldwide in commercial applications, as well as in research and development.
    </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>January 2019</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>3-Clause BSD</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td>8.2.0</td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://www.vtk.org">https://www.vtk.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href="https://gitlab.kitware.com/vtk/vtk">https://gitlab.kitware.com/vtk/vtk</a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Kitware Inc.
    
     <a href="mailto:(kitware@kitware.com)">(kitware@kitware.com)</a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Android<br>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>HPC<br>Mobile<br>Standalone<br>Web</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>3D Studio, AVS "UCD" format, CAD STL, DICOM, Ensight, Facet, Facet, Fluent GAMIT ASCII, Gaussian, Marching Cubes, Movie BYU, OBJ, Open Inventor 2.0, PDB, PLOT3D, PLY, Particle, VTK Poly Data, VTK Structured Grid, XYZ</td></tr>
			<tr class="even"><td>Output Formats</td><td>CAD STL, CGM, Facet, Marching Cubes, Movie BYU, OBJ, Open Inventor 2.0, PLY, Renderman, VRML 2.0, VTK Poly Data, VTK Structured Grid</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="visually-evoked-potential-eeg" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Visually Evoked Potential EEG">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=visually-evoked-potential-eeg" rel="bookmark" name="visually-evoked-potential-eeg">Visually Evoked Potential EEG</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=eeg-modeling" rel="tag">Eeg Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=event-related-potentials" rel="tag">Event-Related Potentials</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> An 18-subject EEG dataset from an experiment in which subjects performed a standard visual oddball task. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1223&screenshot_id=1173"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1223&screenshot_id=1173" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 10, 2017</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/vep_eeg_raw/">http://www.nitrc.org/projects/vep_eeg_raw/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Texas at San Antonio <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="vis:-visualized-statistics-toolbox-(spm5/8/12)" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="vis: Visualized statistics toolbox (SPM5/8/12)">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=vis:-visualized-statistics-toolbox-(spm5/8/12)" rel="bookmark" name="vis:-visualized-statistics-toolbox-(spm5/8/12)">vis: Visualized statistics toolbox (SPM5/8/12)</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=quantification" rel="tag">Quantification</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=statistical-operation" rel="tag">Statistical Operation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> A simple, menu-driven toolbox for exploratory data analysis of brain images in SPM5, SPM8, or SPM12; features include: <br />
<br />
(1) Histograms or kernel density estimates of voxel values; <br />
(2) Scatter, Bland-Altman, or quantile-quantile plots comparing two images; <br />
(3) Surface plot of all voxel values at a particular axial slice; <br />
(4) ROI-based extraction of voxel value point estimates (mean, std, etc.)<br />
<br />
For **CURRENT VERSION** download and documentation, visit <a href="http://tools.robjellis.net" target="_new">http://tools.robjellis.net</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 18, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://tools.robjellis.net/">http://tools.robjellis.net/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>BIDMC / Harvard Medical School <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="vmtk-in-3d-slicer" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="VMTK in 3D Slicer">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=vmtk-in-3d-slicer" rel="bookmark" name="vmtk-in-3d-slicer">VMTK in 3D Slicer</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=modeling" rel="tag">Modeling</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=segmentation" rel="tag">Segmentation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=visualization" rel="tag">Visualization</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This project provides a series of modules which enable functions of the Vascular Modeling Toolkit (<a href="http://www.vmtk.org" target="_new">http://www.vmtk.org</a>) in 3D Slicer (<a href="http://www.slicer.org" target="_new">http://www.slicer.org</a>). <br />
<br />
The functionality includes vessel enhancement filtering, level set segmentation, centerline computation, network extraction and branch splitting.<br />
<br />
Installation notes and documentation are available at the official project page:<br />
<br />
<a href="http://www.vmtk.org/Main/VmtkIn3DSlicer" target="_new">http://www.vmtk.org/Main/VmtkIn3DSlicer</a>
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=196&screenshot_id=269"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=196&screenshot_id=269" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 11, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.vmtk.org/Main/VmtkIn3DSlicer">http://www.vmtk.org/Main/VmtkIn3DSlicer</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>MacOS<br>POSIX/UNIX-like<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>DICOM, Nrrd, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>DICOM, Nrrd, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="volbrain:-online-mri-segmentation" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="volBrain: online MRI segmentation">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=volbrain:-online-mri-segmentation" rel="bookmark" name="volbrain:-online-mri-segmentation">volBrain: online MRI segmentation</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> volBrain is an online MRI brain segmentation system. It is intended to help researchers all over the world to obtain automatically volumetric brain information from their MRI data without the need for any infrastructure in their local sites. volBrain works in a fully automatic manner and is able to provide brain structure volumes without any human interaction. We encourage you to use the system hoping you find it useful. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=932&screenshot_id=924"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=932&screenshot_id=924" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 23, 2015</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>volBrain is free for non-commercial and non-medical purposes</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/volbrain/">http://www.nitrc.org/projects/volbrain/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Universidad Politécnica de Valencia, Bordeaux University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="voxbo" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="VoxBo">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=voxbo" rel="bookmark" name="voxbo">VoxBo</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=format-conversion" rel="tag">Format Conversion</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=linear" rel="tag">Linear</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=overlap-metrics" rel="tag">Overlap Metrics</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=sinc-function-interpolation" rel="tag">Sinc Function Interpolation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=three-dimensional-display" rel="tag">Three Dimensional Display</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> VoxBo is a highly modular and interoperable collection of free tools for brain image manipulation and analysis, focusing on fMRI and lesion analysis. VoxBo can be used independently or in conjunction with other packages. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=73&screenshot_id=238"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=73&screenshot_id=238" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 29, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://voxbo.org/">http://voxbo.org/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>MacOS<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, NIfTI-1, Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, NIfTI-1, Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="vutools" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="vuTools">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=vutools" rel="bookmark" name="vutools">vuTools</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This site provides access to the VUIIS Image and Data Analysis Core's data processing tools written for MATLAB. These tools are written for ease of use from within MATLAB. Unless stated otherwise, each of the tools is capable of processing 2D/3D images (matrices). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 28, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD/MIT-Style Open Source License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/vutools/">http://www.nitrc.org/projects/vutools/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="wannovar" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="wANNOVAR">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=wannovar" rel="bookmark" name="wannovar">wANNOVAR</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=genomic-analysis" rel="tag">Genomic Analysis</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> ANNOVAR is a rapid, efficient tool to annotate functional consequences of genetic variation from high-throughput sequencing data. wANNOVAR provides easy and intuitive web-based access to the most popular functionalities of the ANNOVAR software. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 13, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>wANNOVAR Terms of Use</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/wannovar/">http://www.nitrc.org/projects/wannovar/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Children's Hospital of Philadelphia <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="wavelet-based-image-fusion" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Wavelet-based Image Fusion">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=wavelet-based-image-fusion" rel="bookmark" name="wavelet-based-image-fusion">Wavelet-based Image Fusion</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-reconstruction" rel="tag">Image Reconstruction</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=wavelet-transformation" rel="tag">Wavelet Transformation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This is a Matlab toolbox that implements the wavelet-based image fusion technique for orthogonal images, introduced in (Aganj et al, MRM 2012). 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=513&screenshot_id=382"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=513&screenshot_id=382" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 18, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/wlfusion/">http://www.nitrc.org/projects/wlfusion/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Minnesota <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="waxholm-space-atlas-of-the-c57bl/6j-mouse-brain" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Waxholm Space Atlas of the C57BL/6J Mouse Brain">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=waxholm-space-atlas-of-the-c57bl/6j-mouse-brain" rel="bookmark" name="waxholm-space-atlas-of-the-c57bl/6j-mouse-brain">Waxholm Space Atlas of the C57BL/6J Mouse Brain</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Here you can download image volumes representing the canonical Waxholm Space (WHS) adult C57BL/6J mouse brain. These images include T1-, T2*-, and T2-Weighted MR volumes (generated at the Duke Center for In-Vivo Microscopy), Nissl-stained optical histology (acquired at Drexel University), and a volume of labels. All volumes are represented at 21.5μ isotropic resolution. <br />
<br />
The label volume represents phase one delineations of the WHS mouse brain. The atlas contains 26 structures (+ inner ear) organized in accordance with the NeuroLex Brain Partonomy scheme found at: <a href="http://neurolex.org/wiki/Brain_Partonomy_(general_mammalian)." target="_new">http://neurolex.org/wiki/Brain_Partonomy...</a> The delineations are intended as a communication aid for finding brain locations. Delineations are based on visually distinguishable features and are not intended to impose a specific view of brain structure or function. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 14, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/incfwhsmouse/">http://www.nitrc.org/projects/incfwhsmouse/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>INCF <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="waxholm-space-atlas-of-the-sprague-dawley-rat-brain" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Waxholm Space Atlas of the Sprague Dawley Rat Brain">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=waxholm-space-atlas-of-the-sprague-dawley-rat-brain" rel="bookmark" name="waxholm-space-atlas-of-the-sprague-dawley-rat-brain">Waxholm Space Atlas of the Sprague Dawley Rat Brain</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Open access volumetric atlas offering comprehensive anatomical delineations of the rat brain based on structural contrast in isotropic magnetic resonance (39 μm) and diffusion tensor (78 μm) images acquired ex vivo from an 80 day old male Sprague Dawley rat at the Duke Center for In Vivo Microscopy. Spatial reference is provided by the Waxholm Space coordinate system. The location of bregma and lambda are identified as anchors towards stereotaxic space. Application areas include localization of signal in non-structural images e.g. functional or pharmacological MRI and PET.<br />
<br />
The atlas, the MRI/DTI volumes, and the diffusion tensor data are shared in NIfTI format with labels and configuration files for use with ITK-SNAP, the Mouse BIRN Atlasing Toolkit, and PMOD. Delineation criteria are described in Papp et al. 2014 &amp; Kjonigsen et al. 2015 (see Docs). The latest atlas version contains 79 structures, intended as a basis for further development by researchers in the community. Missing a region? Get involved! 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1081&screenshot_id=1028"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1081&screenshot_id=1028" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 22, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/whs-sd-atlas/">http://www.nitrc.org/projects/whs-sd-atlas/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Oslo <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="web-game-for-collaborative-labeling" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Web Game for Collaborative Labeling">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=web-game-for-collaborative-labeling" rel="bookmark" name="web-game-for-collaborative-labeling">Web Game for Collaborative Labeling</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Statistical atlases of regional brain anatomy have proven to be extremely useful in characterizing the relationship between the structure and function of the human nervous system. Typically, an expert human rater manually examines each slice of a three-dimensional volume. This approach can be exceptionally time and resource intensive, so cost severely limits the clinical studies where subject-specific labeling is feasible. Methods for improved efficiency and reliability of manual labeling would be of immense benefit for clinical investigation into morphological correlates of brain function. The goal of the proposed work is to enable an alternative to expert raters for medical image labeling through statistical analysis of the collaborative efforts of many, minimally-trained raters. The proposed research investigates extension of established practices for volumetric labeling and web- based collaboration to create an innovative infrastructure for labeling. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=242&screenshot_id=187"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=242&screenshot_id=187" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 30, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU Lesser General Public License (LGPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/webmill/">http://www.nitrc.org/projects/webmill/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="web-interfaces-for-multiscale-images" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Web Interfaces for Multiscale Images">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=web-interfaces-for-multiscale-images" rel="bookmark" name="web-interfaces-for-multiscale-images">Web Interfaces for Multiscale Images</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This is a venue for discussing and defining standard web interfaces for sharing images, annotations, and analyses of multiscale biological images. The goal is to increase interoperability of code to share the burden of infrastructure, increase code reuse, and allow us to spend more time focused on scientific questions.<br />
<br />
Please visit our Wiki to start participating.<br />
<br />
Together we can develop a small group of interfaces which are easy to implement, extensible, and cover the major tasks of developing tools for multiscale data on the web. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Nov 26, 2010</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Public Domain</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/plugins/mwiki/index.php/webmscaleapi:MainPage">http://www.nitrc.org/plugins/mwiki/index.php/webmscaleapi:MainPage</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="wfu_bpm" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="WFU_BPM">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=wfu_bpm" rel="bookmark" name="wfu_bpm">WFU_BPM</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=statistical-operation" rel="tag">Statistical Operation</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The WFU Biological Parametric Mapping (WFU_BPM) toolbox performs SPM analysis with voxel-wise imaging covariates. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 7, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>WFU ANSIR License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://fmri.wfubmc.edu/cms/software#BPM">http://fmri.wfubmc.edu/cms/software#BPM</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>POSIX/UNIX-like<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="wfu_mmnet---multivariate-modeling-of-brain-networks-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="WFU_MMNET - Multivariate Modeling of Brain Networks Toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=wfu_mmnet---multivariate-modeling-of-brain-networks-toolbox" rel="bookmark" name="wfu_mmnet---multivariate-modeling-of-brain-networks-toolbox">WFU_MMNET - Multivariate Modeling of Brain Networks Toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This toolbox provides a framework for multivariate modeling of brain networks. It allows assessing brain network differences between study populations as well as assessing the effects of phenotypic variabls such as aging, task performance, and disease states on the density and strength of connections in global and local brain networks while controlling for confounding variables. A variety of neuroimaging data such as fMRI, EEG, and DTI can be analyzed with this toolbox. The toolbox has been developed in MATLAB with a graphical user interface but uses SAS, R, or Python (depending on software availability) to perform the statistical modeling. <br />
We have made all efforts to avoid errors but users are strongly urged to post the bugs or suggested improvements on the provided forum. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 20, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/wfu_mmnet/">http://www.nitrc.org/projects/wfu_mmnet/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Virginia Tech - Wake Forest School of Biomedical Engineering and Sciences <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="wfu_pickatlas" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="WFU_PickAtlas">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=wfu_pickatlas" rel="bookmark" name="wfu_pickatlas">WFU_PickAtlas</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-application" rel="tag">Atlas Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> WFU_PickAtlas provides a method for generating ROI masks based on the Talairach Daemon database. The atlases include Brodmann area, Lobar, Hemisphere, Anatomic Label and Tissue Type. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 7, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>WFU ANSIR License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://fmri.wfubmc.edu/software/PickAtlas">http://fmri.wfubmc.edu/software/PickAtlas</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>POSIX/UNIX-like<br>Windows</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="wfu_pipeline" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="WFU_Pipeline">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=wfu_pipeline" rel="bookmark" name="wfu_pipeline">WFU_Pipeline</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=workflow" rel="tag">Workflow</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The WFU SPM5 Pipeline is a fully automated method for the processing of fMRI data using SPM. It is fully automated from the point of data acquisition at the MRI scanner. It incorporates tools for automated data transfer, archiving, real-time SPM5 batch script generation with distributed grid processing, automated error-recovery procedures, full data-provenance, email notifications, optional conversion back to DICOM (Digital Imaging and Communications in Medicine), and picture archiving and communications systems (PACS) insertion. The architecture allows for an infinite number of easily definable analyses that are fully automated from the point of acquisition. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Feb 10, 2009</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>WFU ANSIR License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/wfu_pipeline/">http://www.nitrc.org/projects/wfu_pipeline/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, DICOM, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, DICOM, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="whitetext---annotated-neuroscience-text" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="WhiteText - annotated neuroscience text">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=whitetext---annotated-neuroscience-text" rel="bookmark" name="whitetext---annotated-neuroscience-text">WhiteText - annotated neuroscience text</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=bibliographic-resource" rel="tag">Bibliographic Resource</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=knowledge-environment" rel="tag">Knowledge Environment</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=web-resource" rel="tag">Web Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> WhiteText is a corpus of manually annotated brain region mentions. It was created to facilitate text mining of neuroscience literature. The corpus contains 1,377 abstracts with 17,585 brain region annotations. Interannotator agreement was evaluated for a subset of the documents, and was 90.7% and 96.7% for strict and lenient matching respectively. We observed a large vocabulary of over 6,000 unique brain region terms and 17,000 words. <br />
<br />
The corpus can be found at <a href="http://www.chibi.ubc.ca/WhiteText/" target="_new">http://www.chibi.ubc.ca/WhiteText/</a><br />
<br />
Previous evaluation of automated recognition of the mentions is described in:<br />
&quot;Automated Recognition of Brain Region Mentions in Neuroscience Literature&quot;<br />
by French, Lane, Xu and Pavlidis.<br />
<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2741206/" target="_new">http://www.ncbi.nlm.nih.gov/pmc/articles...</a></description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jan 15, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://whitetext.msl.ubc.ca/">https://whitetext.msl.ubc.ca/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="wisconsin-cortical-thickness-analysis-(cta)-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Wisconsin Cortical Thickness Analysis (CTA) Toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=wisconsin-cortical-thickness-analysis-(cta)-toolbox" rel="bookmark" name="wisconsin-cortical-thickness-analysis-(cta)-toolbox">Wisconsin Cortical Thickness Analysis (CTA) Toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=thickness" rel="tag">Thickness</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> Wisconsin CTA toolbox is a Matlab tool to perform statistical analysis on cortical thickness signals on brain surfaces obtained from Freesurfer. In particular, this tool is useful for multi-resolutional analysis of such cortical thickness signals and detecting group differences. We find that compared to standard analysis, this method is much more sensitive in detecting such differences. It is based on the Spectral Graph Wavelet Transform (SGWT) toolbox (<a href="http://wiki.epfl.ch/sgwt" target="_new">http://wiki.epfl.ch/sgwt</a>), and provides plug and play methods for deriving Wavelet Multiscale Descriptor (WMD), cortical thickness smoothing using SGWT, Multivariate General Linear Model (MGLM), False Discovery Rate (FDR) and so on.<br />
<br />
Won Hwa Kim (<a href="http://pages.cs.wisc.edu/~wonhwa" target="_new">http://pages.cs.wisc.edu/~wonhwa</a>) </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Mar 13, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://pages.cs.wisc.edu/~wonhwa/CTA_toolbox">http://pages.cs.wisc.edu/~wonhwa/CTA_toolbox</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Wisconsin - Madison <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="wisconsin-rhesus-brain-parcellation" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Wisconsin Rhesus Brain Parcellation">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=wisconsin-rhesus-brain-parcellation" rel="bookmark" name="wisconsin-rhesus-brain-parcellation">Wisconsin Rhesus Brain Parcellation</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=atlas-data" rel="tag">Atlas Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This dataset includes 286 of the Paxinos et al. rhesus brain regions aligned to the McLaren et al. rhesus MRI brain template. It also includes composite regions and regions drawn by the authors. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1313&screenshot_id=1226"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1313&screenshot_id=1226" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Aug 29, 2018</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Public Domain</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/uw_rhesus_brain/">http://www.nitrc.org/projects/uw_rhesus_brain/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Wisconsin-Madison <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="wisconsin-white-matter-hyperintensities-segmentation-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Wisconsin White Matter Hyperintensities Segmentation Toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=wisconsin-white-matter-hyperintensities-segmentation-toolbox" rel="bookmark" name="wisconsin-white-matter-hyperintensities-segmentation-toolbox">Wisconsin White Matter Hyperintensities Segmentation Toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> W2MHS is an open source MATLAB toolbox designed for detecting and quantifying White Matter Hyperintensities(WMH) in Alzheimer’s and aging related neurological disorders.Our toolbox provides a self-sufficient set of tools for segmenting these WMHs reliably and further quantifying their burden for down-processing studies. WMHs arise as bright regions on T2-weighted FLAIR images. They reflect comorbid neural injury or cerebral vascular disease burden. Their precise detection is of interest in Alzheimer’s disease (AD) with regard to its prognosis.<br />
<br />
Please cite the following paper upon usage of this toolbox. <br />
Ithapu, V., Singh, V., Lindner, C., Austin, B. P., Hinrichs, C., Carlsson, C. M., Bendlin, B. B. and Johnson, S. C. (2014), Extracting and summarizing white matter hyperintensities using supervised segmentation methods in Alzheimer's disease risk and aging studies. Hum. Brain Mapp.. doi: 10.1002/hbm.22472 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=719&screenshot_id=657"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=719&screenshot_id=657" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 19, 2013</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Academic Free License ("AFL")</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/w2mhs/">http://www.nitrc.org/projects/w2mhs/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Wisconsin - Madison <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="working-memory-trainer" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Working Memory Trainer">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=working-memory-trainer" rel="bookmark" name="working-memory-trainer">Working Memory Trainer</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This is a stand-alone axecutable under all main Windows OS. It is a program for training the working memory. The WM Trainer looks and behaves a little bit like a video-game and has been specifically conceived for children attending the primary school. However, it can be used purposefully by people of any age, including adult and elderly.<br />
<br />
This application features highest graphic quality, a powerful adaptive engine for the difficulty level, a database of users and statistical tools to evaluate the progress.<br />
<br />
Currently English, French and Italian are supported, but any language can be easily supported. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 21, 2012</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/wmtrainer/">http://www.nitrc.org/projects/wmtrainer/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>GIPSA-lab, Grenoble, FRANCE <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="xcede-schema" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="XCEDE Schema">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=xcede-schema" rel="bookmark" name="xcede-schema">XCEDE Schema</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The XML-Based Clinical Experiment Data Exchange Schema provides an extensive metadata hierarchy for describing and documenting research and clinical studies. XCEDE was originally designed in the context of neuroimaging studies and complements the BIRN-HID </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jun 19, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="https://github.com/incf-nidash/XCEDE">https://github.com/incf-nidash/XCEDE</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="xfsl:-an-fsl-toolbox" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="XFSL: An FSL toolbox">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=xfsl:-an-fsl-toolbox" rel="bookmark" name="xfsl:-an-fsl-toolbox">XFSL: An FSL toolbox</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=other-software-resource" rel="tag">Other Software Resource</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> A set of BASH scripts for MRI data management, FSL automation and web application 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=890&screenshot_id=914"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=890&screenshot_id=914" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 7, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/xfsl/">http://www.nitrc.org/projects/xfsl/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>USC & BNU <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="xjview,-a-viewing-tool-for-spm" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="xjView, a viewing tool for SPM">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=xjview,-a-viewing-tool-for-spm" rel="bookmark" name="xjview,-a-viewing-tool-for-spm">xjView, a viewing tool for SPM</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=image-display" rel="tag">Image Display</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> xjView is a viewing program for SPM2 and SPM5. p-value slider, displays multiple images at a time and can be used to build ROI masks. For a given region you can find the anatomical name and search the selected region in online database (wiki, google scholar and pubmed). </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Dec 5, 2008</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Freeware</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.alivelearn.net/xjview8/">http://www.alivelearn.net/xjview8/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Linux<br>OS Independent<br>Windows XP</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>ANALYZE, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>ANALYZE, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="xnat" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="XNAT">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=xnat" rel="bookmark" name="xnat">XNAT</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database-application" rel="tag">Database Application</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=platform-or-development-environment" rel="tag">Platform Or Development Environment</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> XNAT is an open source imaging informatics platform designed to facilitate management and exploration of medical imaging and related data. XNAT includes a DICOM workflow, a secure database backend, and a rich web-based user interface. XNAT's web services interface enables external applications to easily access XNAT-hosted data. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=60&screenshot_id=531"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=60&screenshot_id=531" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 7, 2007</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>BSD License</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.xnat.org">http://www.xnat.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Washington University <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>OS Independent</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>DICOM, NIfTI-1</td></tr>
			<tr class="even"><td>Output Formats</td><td>DICOM, NIfTI-1</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="xnat-extras" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="XNAT Extras">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=xnat-extras" rel="bookmark" name="xnat-extras">XNAT Extras</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=database-application" rel="tag">Database Application</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> User contributions for XNAT. </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Sep 8, 2011</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td></td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.xnat.org">http://www.xnat.org</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>POSIX/UNIX-like</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="xnbc" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="XNBC">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=xnbc" rel="bookmark" name="xnbc">XNBC</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=animation" rel="tag">Animation</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=event-related-potential" rel="tag">Event Related Potential</a> <a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=multi-neuron-modeling" rel="tag">Multi-Neuron Modeling</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> XNBC is a full featured application for computer naive neuroscientists. It simulates biological neural networks using graphic tools to edit neurons and networks, to run the simulation and to analyze results. Written in C, it runs on Unix and Windows. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=848&screenshot_id=825"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=848&screenshot_id=825" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Jul 28, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>GNU General Public License (GPL)</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/xnbc/">http://www.nitrc.org/projects/xnbc/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Université Pierre et Marie Curie <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td>Cygwin<br>Linux<br>Other UNIX-like<br>Windows 95/98/2000<br>Windows XP</td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td>Other Format</td></tr>
			<tr class="even"><td>Output Formats</td><td>Other Format</td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="ymdti:-diffusion-tensor-images-of-healthy-young-males" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="YMDTI: Diffusion Tensor Images of Healthy Young Males">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=ymdti:-diffusion-tensor-images-of-healthy-young-males" rel="bookmark" name="ymdti:-diffusion-tensor-images-of-healthy-young-males">YMDTI: Diffusion Tensor Images of Healthy Young Males</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=data" rel="tag">Data</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> This dataset contains diffusion tensor images of 93 healthy, young male subjects.<br />
<br />
Subject infomration:<br />
 age: 18-30, mean age 23<br />
 86 right-handed, 7 left-handed<br />
 no psychiatric or neurological diseases<br />
<br />
DTI-images were recorded on a 3-T Philips Achieva scanner with the following specifications:<br />
 8-channel head coil<br />
 32 gradient directions<br />
 70 axial slices <br />
 slice thickness 2 mm<br />
 single-shot EPI sequence<br />
 TR = 7852 ms<br />
 TE = 60 ms<br />
 flip angle 90°<br />
 FOV = 224 mm<br />
 matrix = 112 x 112 </description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>May 29, 2014</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Attribution Non-Commercial</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/ymdti/">http://www.nitrc.org/projects/ymdti/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>University of Luebeck <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
<hr><article id="zebrafish-heart-segmentation" class="post-1869 hbp_software type-hbp_software status-publish hentry " name="software" software-name="Zebrafish heart segmentation">
	<header class="entry-header">

				<h1 class="entry-title">
			<a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=zebrafish-heart-segmentation" rel="bookmark" name="zebrafish-heart-segmentation">Zebrafish heart segmentation</a>
		</h1>

		<div class="entry-meta">

				<span class="tags-links"></span>
				<br><span class="tags-links"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software_cat=algorithm-or-reusable-library" rel="tag">Algorithm Or Reusable Library</a> </span>					</div><!-- .entry-meta -->
	</header><!-- .entry-header -->

		<div class="entry-content">
            <description> The package aims to perform image segmentation of the adult zebrafish cardiac ventricle obtained by light-sheet fluorescent imaging (also known as single plane illumination microscopy). <br />
<br />
Please refer to the following article for the package:<br />
Rene Packard, Kyung In Baek, Tyler Beebe, Nelson Jen, Yichen Ding, Feng Shi, Peng Fei, Bong Jin Kang, Po-Heng Chen, Jonathan Gau, Michael Chen, Jonathan Tang, Yu-Huan Shih, Yonghe Ding, Debiao Li, Xiaolei Xu, and Tzung Hsiai, &quot;Automated Segmentation of Light-Sheet Fluorescent Imaging to Characterize Experimental Doxorubicin-Induced Cardiac Injury and Repair&quot;. Scientific Reports, in press, 2017. 
<p/>
<a href="https://www.nitrc.org/project/screenshot.php?group_id=1065&screenshot_id=1010"><img class="size-full wp-image-1824" src="https://www.nitrc.org/project/screenshot.php?group_id=1065&screenshot_id=1010" alt="" width="458" height="248"/></a>
</description>
        <table>
            <tbody><tr class="even"><td>Date of Release</td><td>Oct 7, 2016</td></tr>
			<tbody><tr class="odd"><td>Availability</td><td>External</td></tr>
		<tbody><tr class="even"><td>License</td><td>Academic Free License ("AFL")</td></tr>
		<tbody><tr class="odd"><td>Framework</td><td></td></tr>
            <tr class="even"><td>Version of software</td><td></td></tr>
            <tr class="odd"><td>Version of documentation</td><td></td></tr>
            <tr class="even"><td>Software available</td><td><a href="http://www.nitrc.org/projects/zebrafishhearts/">http://www.nitrc.org/projects/zebrafishhearts/</a></td></tr>
            <tr class="odd"><td>Documentation</td><td><a href=""></a>
</td></tr>
            <tr class="even"><td>Responsible</td><td>Cedars Sinai Medical Center & UCLA <a href="mailto:"></a></td></tr>
	    <tr class="odd"><td>Requirements &amp; dependencies</td><td></td></tr>
            <tr class="even"><td>Platforms</td><td></td></tr>
			<tbody><tr class="odd"><td>Deployment</td><td>Standalone</td></tr>
		<tbody><tr class="even"><td>Displaytype</td><td>Desktop</td></tr>
			<tr class="odd"><td>Input Formats</td><td></td></tr>
			<tr class="even"><td>Output Formats</td><td></td></tr>
        </tbody></table>


    </div><!-- .entry-content -->


	<footer class="entry-meta">

			</footer><!-- .entry-meta -->
</article><!-- #post -->
                </div>
                <footer id="colophon" class="site-footer" role="contentinfo">

                    <div class="site-info">

                        <p>
                            <img src="https://hbp-hpc-platform.fz-juelich.de/wp-content/themes/hbp-hpc/img/flag_yellow_low.jpg"
                                 style="float: right; width: 6em; margin-left: 5em; opacity: 0.6;">
                            This project has received funding from the European Union Seventh Framework Programme
                            (FP7/2007-2013) under grant agreement no. 604102 (Human Brain Project), and from the
                            European Unionâ€™s Horizon 2020 Framework Programme for Research and Innovation under the
                            Specific Grant Agreements No. 20270 (Human Brain Project SGA1) and No. 785907 (Human Brain
                            Project SGA2).
                        </p>
                        <a href="http://www.fz-juelich.de/portal/EN/Service/LegalNotice/_node.html"
                           title="Legal Notice">Legal Notice</a> | <a
                            href="https://hbp-hpc-platform.fz-juelich.de/?page_id=1974" title="Privacy Policy">Data
                        protection</a>
                    </div><!-- .site-info -->
                </footer><!-- #colophon -->
            </div><!-- #content -->
        </div><!-- #primary -->


    </div><!-- #main -->

</div><!-- #page -->


<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.12.0/lodash.js"></script>
<script>
    window.onscroll = function () {
        scrollFunction()
    };

    function scrollFunction() {
        if (document.body.scrollTop > 20 || document.documentElement.scrollTop > 20) {
            document.getElementById("myBtn").style.display = "block";
        } else {
            document.getElementById("myBtn").style.display = "none";
        }
    }

    // When the user clicks on the button, scroll to the top of the document
    function topFunction() {
        document.body.scrollTop = 0; // For Safari
        document.documentElement.scrollTop = 0; // For Chrome, Firefox, IE and Opera
    }

    function clearFilters() {
        document.getElementsByName("filterCheckbox").forEach(cb => {
            cb.checked = false;
        });
        handleClick();
    }

    function handleClick() {
        var cbList = [];
        document.getElementsByName("filterCheckbox").forEach(cb => {
            if (cb.checked)
                cbList.push(cb);
        });

        if (document.getElementById("softwareDIV").innerHTML)
            document.getElementById("softwareDIV").innerHTML = "";

        if (cbList.length === 0) {
            document.getElementsByName("software").forEach(value => {
                value.style.display = "block";
                document.getElementById("softwareDIV").innerHTML += "<li><a href=\"#" + value.id + "\">" + document.getElementById(value.id).attributes.getNamedItem("software-name").value + "</a></li><br/>"
            });
            document.getElementById("filterDIV").innerHTML = "&nbsp;";

            return;
        }

        document.getElementsByName("software").forEach(value => value.style.display = "none");

        var softwareList = [];
        var filterList = [];
        cbList.forEach(cb => {
            softwareList.push(cb.dataset.software.split(" "));
            filterList.push(cb.nextSibling.nodeValue);
        });

        document.getElementById("filterDIV").innerHTML = filterList.join(", ");

        var intersect = _.intersection.apply(_, softwareList);
        intersect.forEach(s => {
            document.getElementById(s).style.display = "block";
            document.getElementById("softwareDIV").innerHTML += "<li><a href=\"#" + s + "\">" + document.getElementById(s).attributes.getNamedItem("software-name").value + "</a></li><br/>"
        })
    }
    scrollFunction();
    handleClick();
</script>
<script>
    (function (body) {
        'use strict';
        body.className = body.className.replace(/\btribe-no-js\b/, 'tribe-js');
    })(document.body);
</script>
<script> /* <![CDATA[ */
var tribe_l10n_datatables = {
    "aria": {
        "sort_ascending": ": activate to sort column ascending",
        "sort_descending": ": activate to sort column descending"
    },
    "length_menu": "Show _MENU_ entries",
    "empty_table": "No data available in table",
    "info": "Showing _START_ to _END_ of _TOTAL_ entries",
    "info_empty": "Showing 0 to 0 of 0 entries",
    "info_filtered": "(filtered from _MAX_ total entries)",
    "zero_records": "No matching records found",
    "search": "Search:",
    "all_selected_text": "All items on this page were selected. ",
    "select_all_link": "Select all pages",
    "clear_selection": "Clear Selection.",
    "pagination": {"all": "All", "next": "Next", "previous": "Previous"},
    "select": {"rows": {"0": "", "_": ": Selected %d rows", "1": ": Selected 1 row"}},
    "datepicker": {
        "dayNames": ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"],
        "dayNamesShort": ["Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat"],
        "dayNamesMin": ["S", "M", "T", "W", "T", "F", "S"],
        "monthNames": ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"],
        "monthNamesShort": ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"],
        "monthNamesMin": ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"],
        "nextText": "Next",
        "prevText": "Prev",
        "currentText": "Today",
        "closeText": "Done",
        "today": "Today",
        "clear": "Clear"
    }
};
var tribe_system_info = {
    "sysinfo_optin_nonce": "114c91ae53",
    "clipboard_btn_text": "Copy to clipboard",
    "clipboard_copied_text": "System info copied",
    "clipboard_fail_text": "Press \"Cmd + C\" to copy"
};/* ]]> */ </script>
<script type="text/javascript">
    /* <![CDATA[ */
    var wpcf7 = {
        "apiSettings": {
            "root": "https:\/\/hbp-hpc-platform.fz-juelich.de\/index.php?rest_route=\/contact-form-7\/v1",
            "namespace": "contact-form-7\/v1"
        }
    };
    /* ]]> */
</script>
<script type="text/javascript"
        src="https://hbp-hpc-platform.fz-juelich.de/wp-content/plugins/contact-form-7/includes/js/scripts.js?ver=5.1.1"></script>
<script type="text/javascript"
        src="https://hbp-hpc-platform.fz-juelich.de/wp-content/plugins/vegas-fullscreen-background-slider/methods/../js/jquery.vegas.js"></script>
<script type="text/javascript"
        src="https://hbp-hpc-platform.fz-juelich.de/wp-content/plugins/wpmbytplayer/js/jquery.mb.YTPlayer.js?ver=3.2.5"></script>
<script type="text/javascript"
        src="https://hbp-hpc-platform.fz-juelich.de/wp-content/themes/hbp-hpc/js/scrollspy.js?ver=1.0.0"></script>
<script type="text/javascript"
        src="https://hbp-hpc-platform.fz-juelich.de/wp-content/themes/hbp-hpc/js/functions.js?ver=1.0.0"></script>
<script type="text/javascript"
        src="https://hbp-hpc-platform.fz-juelich.de/wp-content/themes/twentythirteen/js/functions.js?ver=20160717"></script>
<script type="text/javascript"
        src="https://hbp-hpc-platform.fz-juelich.de/wp-includes/js/wp-embed.min.js?ver=5.1.1"></script>


</body>
</html>